<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ejhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5315515/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>SNP variants at the MAP3K1/SETD9 locus 5q11.2 associate with somatic PIK3CA variants in breast cancers</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5315515/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="European Journal of Human Genetics"><meta name="citation_title" content="SNP variants at the MAP3K1/SETD9 locus 5q11.2 associate with somatic PIK3CA variants in breast cancers"><meta name="citation_authors" content="Roberto Puzone, Ulrich Pfeffer"><meta name="citation_date" content="March 2017"><meta name="citation_issue" content="3"><meta name="citation_volume" content="25"><meta name="citation_firstpage" content="384"><meta name="citation_doi" content="10.1038/ejhg.2016.179"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5315515/?report=abstract"><meta name="citation_pmid" content="28029147"><meta name="DC.Title" content="SNP variants at the MAP3K1/SETD9 locus 5q11.2 associate with somatic PIK3CA variants in breast cancers"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Nature Publishing Group"><meta name="DC.Contributor" content="Roberto Puzone"><meta name="DC.Contributor" content="Ulrich Pfeffer"><meta name="DC.Date" content="2017 Mar"><meta name="DC.Identifier" content="10.1038/ejhg.2016.179"><meta name="DC.Language" content="en"><meta property="og:title" content="SNP variants at the MAP3K1/SETD9 locus 5q11.2 associate with somatic PIK3CA variants in breast cancers"><meta property="og:type" content="article"><meta property="og:description" content="Genome-wide association studies have revealed many breast cancer (BC) risk-associated genetic variants that might functionally interact with other molecular determinants of BC. We analysed the association of 21 known risk-associated single-nucleotide ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5315515/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ejhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5315515/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5315515/pdf/ejhg2016179a.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89C678DD592AF1000000000A520619.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5315515%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1300/">Eur J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/287993/">v.25(3); 2017 Mar</a></li><li class="accid">PMC5315515</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ejhg.gif" alt="Logo of ejhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/ejhg/" target="pmc_ext" ref="reftype=publisher&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Eur J Hum Genet</a></span>. 2017 Mar; 25(3): 384–387. </span></div><div><span class="fm-vol-iss-date">Published online 2016 Dec 28. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fejhg.2016.179" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ejhg.2016.179</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5315515</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/28029147">28029147</a></div></div></div></div><h1 class="content-title">SNP variants at the <em>MAP3K1/SETD9</em> locus 5q11.2 associate with somatic <em>PIK3CA</em> variants in breast cancers</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Puzone%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28029147" class="affpopup" co-rid="_co_idm140390892674224" co-class="co-affbox">Roberto Puzone</a><sup>1,</sup><sup>*</sup> and  <a href="/pubmed/?term=Pfeffer%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28029147" class="affpopup" co-rid="_co_idm140390817548064" co-class="co-affbox">Ulrich Pfeffer</a><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140390892674224"><h3 class="no_margin">Roberto Puzone</h3><p><sup>1</sup>Integrated Oncology Therapies Department, Clinical Epidemiology, IRCCS AOU San Martino – IST Istituto Nazionale per la Ricerca sul Cancro, Genova, Italy</p><div>Find articles by <a href="/pubmed/?term=Puzone%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28029147">Roberto Puzone</a></div></div><div id="_co_idm140390817548064"><h3 class="no_margin">Ulrich Pfeffer</h3><p><sup>2</sup>Integrated Oncology Therapies Department, Molecular Pathology, IRCCS AOU San Martino – IST Istituto Nazionale per la Ricerca sul Cancro, Genova, Italy</p><div>Find articles by <a href="/pubmed/?term=Pfeffer%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28029147">Ulrich Pfeffer</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140390892007264_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140390892007264_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140390892007264_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140390892007264_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Integrated Oncology Therapies Department, Clinical Epidemiology, IRCCS AOU San Martino – IST Istituto Nazionale per la Ricerca sul Cancro, Genova, Italy</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Integrated Oncology Therapies Department, Molecular Pathology, IRCCS AOU San Martino – IST Istituto Nazionale per la Ricerca sul Cancro, Genova, Italy</div><div id="caf1"><sup>*</sup>Integrated Oncology Therapies Department, Clinical Epidemiology, IRCCS AOU San Martino – IST Istituto Nazionale per la Ricerca sul Cancro, 16132 Genova, Italy. Tel: +39 010 5558429; Fax: +39 010 354103; E-mail: <a href="mailto:dev@null" data-email="ti.onitramnash@enozup.otrebor" class="oemail">ti.onitramnash@enozup.otrebor</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140390892007264_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2016 Jan 11; Revised 2016 Oct 12; Accepted 2016 Nov 22.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140390892007264_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2017 Macmillan Publishers Limited, part of Springer Nature.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC5315515/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Information.</strong></div><div><a href="/pmc/articles/PMC5315515/bin/ejhg2016179x1.txt" data-ga-action="click_feat_suppl">ejhg2016179x1.txt</a><span style="color:gray"> (370 bytes)</span></div><div class="small guid">GUID:&nbsp;B77DF431-F94F-487B-9C1E-A6948BEDCE88</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Information.</strong></div><div><a href="/pmc/articles/PMC5315515/bin/ejhg2016179x2.doc" data-ga-action="click_feat_suppl">ejhg2016179x2.doc</a><span style="color:gray"> (367K)</span></div><div class="small guid">GUID:&nbsp;5CDDA5B0-5606-49D4-AD62-C4F8105D0850</div></div></dd></dl></div><div id="idm140390815345856" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140390815345856title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Genome-wide association studies have revealed many breast cancer (BC) risk-associated genetic variants that might functionally interact with other molecular determinants of BC. We analysed the association of 21 known risk-associated single-nucleotide variants (SNVs) with recurrent somatic variants in two cohorts of 77 and 754 oestrogen receptor <em>α</em>-positive BCs. Four SNVs located at 5q11.2 were found to be associated with the somatic PIK3CA variant status in the pilot cohort of 77 cases with odds ratio (OR) up to 6.5 indicating strong effects, and were selected for the validation phase. Two of these SNVs, rs252913 and rs331499, located in the MAP3K1/SETD9 gene boundary, were confirmed to be associated with somatic PIK3CA variants in the large cohort with OR 2.97 (1.17–7.75) and 1.76 (1.11–2.77), respectively, notably higher than their BC risk-associated values, both around 1.1. In the presence of the SNV or of somatic PIK3CA variants, cancers express significantly elevated levels of MAP3K1 and SETD9, with synergy of SNV and PIK3CA variants in <em>MAP3K1</em> gene overexpression, consistent with a preferential PIK3CA-dependent regulation of the variant alleles.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p2" class="p p-first">Less than 10% of human breast cancers (BCs) show pronounced familiarity that can be explained by high penetrance germline variants, but sporadic BCs are also co-determined by low penetrance variants.<sup><a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770460">1</a>, <a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770471">2</a>, <a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_667179242">3</a></sup> Genome-wide association studies in BC cohorts compared with the general population have identified almost a 100 single-nucleotide variants (SNVs) associated with BC with odds ratio (OR) below 1.2 for most single SNVs.<sup><a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770482">4</a>, <a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770468">5</a></sup> However, differently from high penetrance variants, how SNVs functionally increase BC risk is difficult to establish and mostly unknown.</p><p id="__p3" class="p p-last">In sporadic BC, next-generation sequencing has revealed a mutational landscape characterised by a large number of somatic variants (SM), but few recurrently mutated genes: <em>PIK3CA</em> (25–35%), <em>TP53</em> (20–30%), and to a lesser extent <em>MAP3K1</em>, <em>GATA3</em> and <em>CDH1</em>, with a certain preference for specific BC (sub)types.<sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770470">6</a>, <a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770472">7</a>, <a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770477">8</a></sup> Recurrent somatic variants drive tumour progression, but little is known about what leads to the development of tumours carrying these variants. We reasoned that risk-associated genetic variants might modify driver gene penetrance, and investigated the issue by analysing the association of a small series of known BC risk-associated SNVs with the occurrence of SM in the frequently mutated genes <em>PIK3CA</em>, <em>TP53</em> and <em>MAP3K1</em>.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Methods</h2><p id="__p4" class="p p-first">Twenty-one SNVs were selected (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S1</a>): 11 from O'Brien <em>et al.</em>,<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770476">9</a></sup> rs889312, an expression quantitative trait locus (eQTL) SNV,<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770478">10</a>, <a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770457">11</a></sup> and three further SNVs in high linkage disequilibrium (LD) with the former all located at 5q11.2 (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S2</a>), as well as seven further SNVs were selected from Rhie <em>et al.</em><sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770463">12</a></sup> We preferred SNVs associated with oestrogen receptor <em>α</em>-positive (ER+) BC risk (often higher than for all BC) and close to coding sequences, to maximise the use of genotyped data without imputations of allelic variant status.</p><p id="__p5" class="p p-last">Because no previous OR assessment was available, we started by analysing a small pilot data set powered enough to observe strong association, Ellis <em>et al.</em>,<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770464">13</a></sup> whole-genome and exome sequences of samples from 77 ER+ BC patients of CEU ethnicity (genomic data from dbGap<sup><a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770474">14</a></sup> repository <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000472.v1.p1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000472.v1.p1</a>, authorisation #7444; SM data from Ellis <em>et al.</em><sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770484">13</a></sup>). To confirm the observed associations, we used the 754 ER+ BC patient data from the BRCA data set of the TCGA consortium collection<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770467">7</a></sup> (genomic data <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000178.v9.p8" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000178.v9.p8</a>, available from CGHub<sup><a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770458">15</a></sup> repository, authorisation #7821; public data at <a href="https://tcga-data.nci.nih.gov" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://tcga-data.nci.nih.gov</a>, and many research portals such as CBIO (<a href="http://www.cbioportal.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.cbioportal.org/</a>)<sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770475">16</a></sup>). Clinical and protected SRR/BAM files of normal blood samples, collecting selected sequences that included the 21 SNVs for the Ellis 2012 data set or the four 5q11.2 SNVs for the BRCA data set, were downloaded by using dbGap's SRA-Toolkit service for Elllis 2012 and beta2 version of CGHub's BAM-Slicer service for TCGA-BRCA. Selected sequence downloading allowed a crucial reduction of the multi-terabytes secure file handling required for full data sets; to our knowledge, this is the first report of an analysis entirely performed by using selected downloads. Data were processed by Python and R scripts for realignments, filtering and allele calls, then linked to public (level 3) SM data of BC samples. For the Ellis 2012 pilot data set, ORs were calculated for each allele (dominant model) of the 21 SNVs <em>vs</em> the SM status of each of <em>PIK3CA</em>, <em>MAP3k1</em> and <em>TP53</em>. Uncorrected Fisher's exact test <em>P</em>&lt;0.1 and OR &gt;3 were used to select the resulting associations. Allele correlations were assessed by the Pearson's <em>r</em> coefficient. For the BRCA data set, ER+ samples were selected and a similar analysis was performed on the four 5q11.2 SNVs <em>vs</em>
<em>PIK3CA</em> SM only. Fisher's <em>P</em> were reported without multiple test correction owing to the high correlations among the four SNVs.<sup><a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770462">17</a></sup> Logistic regression and ethnicity strata calculation with forest plots and Breslow–Day homogeneity test were also performed. Public (level 3) log2 normalised gene expression data for the BRCA BC samples were merged with the SNV/SM/clinical data and difference of expression significances were assessed by using the Student's <em>t</em>-test. Statistical analyses were performed by using R(x64) 3.1.3 (<a href="http://www.R-project.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.R-project.org</a>). Result data were submitted to the GWAS Central repository (<a href="http://www.gwascentral.org/study/HGVST1843" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.gwascentral.org/study/HGVST1843</a>).</p></div><div id="__sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec3title">Results</h2><p id="__p6" class="p p-first">In the Ellis 2012 pilot data set, we found strong indication that four SNVs in the 5q11.2 locus were associated with the mutational status of <em>PIK3CA</em> with high OR (<a href="/pmc/articles/PMC5315515/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). As expected by their LD, high correlation was found among these variants. Most of the other high OR SNV/SM associations had very low significance levels (full results in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S3</a>). We focused on the associations of the four 5q11.2 SNVs with <em>PIK3CA</em> SM and verified them in the 754 ER+ BC patient data from the TCGA-BRCA data set. Two variants, rs331499 (hg19.chr5:g.56210923A&gt;G) and rs252913 (hg19.chr5:g.56195846G&gt;A), located in the boundary of <em>MAP3K1</em> and <em>SETD9</em> genes, were confirmed to be correlated with <em>PIK3CA</em> SM with high OR; a third, rs832552(hg19.chr5:g.56113850T&gt;G) inside <em>MAP3K1</em>, had few valid samples but a similar OR trend (<a href="/pmc/articles/PMC5315515/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). High correlation (Pearson's <em>r</em> range: 0.73–0.94) among the variants was confirmed (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S4</a>). In logistic regression, no evidence of significant heterogeneity for ethnicity was found (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Tables S4 and S5</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Association of SNVs close to the <em>MAP3K1</em> gene with somatic PIK3CA variants, two cohorts</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140390853458656" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th colspan="2" charoff="50" rowspan="1" valign="top" align="center"><em>SNV</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th colspan="4" charoff="50" rowspan="1" valign="top" align="center"><em>Ellis 2012 pilot data set</em></th><th colspan="4" charoff="50" rowspan="1" valign="top" align="center"><em>TCGA-BRCA (ER+) data set</em></th></tr><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Name/allele</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>HGVS ID</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Distance (kbases) from MAP3K1</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>OR (95% CI)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P</em><em>-value</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Patients</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Allele freq</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>OR (95% CI)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P</em><em>-value</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Patients</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Allele freq</em></th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs889312 A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">hg19.chr5:g.56031884A&gt;C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−79.0</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">Inf (0.69, Inf)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.09</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">45</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.64</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.90 (0.33, 2.51)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">112</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.66</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs832552 T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">hg19.chr5:g.56113850T&gt;G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&lt;&gt;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6.48 (1.18, 48.3)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.03</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">45</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.55 (0.58, 4.24)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.48</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">91</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.5</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs252913 G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">hg19.chr5:g.56195846G&gt;A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.9</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5.29 (0.88, 40.0)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.06</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">45</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.51</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.97 (1.17, 7.75)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.01</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">177</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.57</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs331499 A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">hg19.chr5:g.56210923A&gt;G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">18.9</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.40 (0.70, 17.7)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.15</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">45</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.51</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.75 (1.11, 2.77)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.02</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">541</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.56</td></tr></tbody></table></div><div id="largeobj_idm140390853458656" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5315515/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t1-fn1"><p id="__p17" class="p p-first-last">Abbreviations: &lt;&gt;, inside; Distance, SNV distance from gene start or end (the shortest); OR, odds ratio; CI, confidence interval for OR; <em>P</em>-value, <em>P</em>-value from Fisher's exact test.</p></div></div></div><p id="__p7" class="p p-last">The three 5q11.2 variants were found to be associated with the overexpression of one or both of their nearest genes <em>MAP3K1</em> and <em>SETD9</em>; for <em>MAP3K1</em>, associations were stronger in <em>PI3KCA</em> SM than in wild type (WT; <a href="/pmc/articles/PMC5315515/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a> and <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Tables S6–S8</a>). Furthermore, we found a direct association between both <em>MAP3K1</em> and <em>SETD9</em> overexpression, and <em>PIK3CA</em> SM status – <em>MAP3K1</em> expression at <em>PIK3CA</em> SM/WT, difference of means: 0.38 (95% CI: 0.19, 0.57), <em>P</em>=4e−4; <em>SETD9</em> expression at <em>PIK3CA</em> SM/WT, difference of means: 1.44 (95% CI: 1.24, 1.63), <em>P</em>=2e−16.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>5q11.2 SNV association with MAP3K1 and SETD9 gene expression in TCGA-BRCA (ER+) data set</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140390855999216" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th colspan="3" charoff="50" rowspan="1" valign="top" align="center"><em>SNV</em></th><th colspan="4" charoff="50" rowspan="1" valign="top" align="center"><em>MAP3K1 gene expression</em></th><th colspan="4" charoff="50" rowspan="1" valign="top" align="center"><em>SETD9 gene expression</em></th></tr><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Name/allele</em></th><th colspan="2" charoff="50" rowspan="1" valign="top" align="center"><em>Distance (kbases) from</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>PIK3CA SM</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>PIK3CA WT</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>PIK3CA SM</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>PIK3CA WT</em></th></tr><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>MAP3K1</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>SETD9</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Diff. allele +/</em>− <em>(95% CI)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P</em><em>-value</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Diff. allele +/</em>− <em>(95% CI)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P</em><em>-value</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Diff. allele +/</em>− <em>(95% CI)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P</em><em>-value</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Diff. allele +/</em>− <em>(95% CI)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>P</em><em>-value</em></th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs832552 T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&lt;&gt;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−91.2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.90 (0.46, 1.33)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.0E−04</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.20 (−0.56, 0.96)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.57</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.51 (−0.05, 1.08)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.07</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.06 (−0.91, 0.79)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.87</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs252913 G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.9</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">−9.2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.77 (0.36, 1.18)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.0E−03</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.51 (−0.23, 1.25)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.17</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.64 (0.05, 1.23)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.04</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.72 (0.19, 1.25)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.01</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs331499 A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">18.9</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">&lt;&gt;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.09 (−0.45, 0.61)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.74</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.30 (−0.06, 0.66)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.69 (0.30, 1.08)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.02</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.81 (0.50, 1.13)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.0E−06</td></tr></tbody></table></div><div id="largeobj_idm140390855999216" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5315515/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t2-fn1"><p id="__p18" class="p p-first-last">Abbreviations: &lt;&gt;, inside; Distance, SNV distance from gene start or end (the shortest); PIK3CA SM/WT, in samples with PIK3CA somatic variants/wild type; Diff. allele +/− (95% confidence interval (CI)), difference of the means of gene expression in samples with presence/samples with absence of the allele, with 95% CI; <em>P</em>-value, <em>P</em>-value from Student's <em>t</em>-test.</p></div></div></div></div><div id="__sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec4title">Discussion</h2><p id="__p8" class="p p-first">In this short report, we show that germline SNVs located near the <em>MAP3K1/SETD9</em> genes associate with <em>PIK3CA</em> SM in ER+ BC with OR values (1.75 and 2.97 for rs331499 and rs252913, respectively) much higher than their OR of association with BC or BC subtypes (OR about 1.1,<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770456">18</a></sup> as the OR of most cancer-risk SNV<sup><a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770480">4</a></sup>). SNV data are coherent with gene expression data: the SNV associations with <em>MAP3K1/SETD9</em> overexpression are increased when the distance from the target gene is reduced, and, for <em>MAP3K1</em>, are stronger in <em>PIK3CA</em> SM BC samples. The overall picture is compatible with a <em>MAP3K1/SETD9</em> variant-dependent overexpression affecting <em>PIK3CA</em> SM penetrance. Moreover, we found a clear direct association of <em>PIK3CA</em> SM with <em>MAP3K1</em> and <em>SETD9</em> overexpression. Indeed, inter-regulation between PI3K and MAP-kinase pathways has been described in <em>in vitro</em> experiments and computer simulation,<sup><a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770479">19</a></sup> and combination of drugs targeting both pathways is under clinical investigation.<sup><a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770459">20</a></sup> A possible <em>SETD9</em> involvement is suggested by the strong SNV associations with <em>SETD9</em> overexpression; moreover, 5q11.2 SNV eQTL to <em>SETD9</em> has been reported also in normal blood.<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770473">18</a></sup> However, we found a synergy of <em>PI3KCA</em> SM and SNV only for <em>MAP3K1</em> overexpression.</p><p id="__p9">Two of our findings indicate that a complex BC risk SNV structure is present in the 5q11.2 region. First, only the SNV in the boundary of <em>MAP3K1/SETD9</em> genes (but not the reference risk SNP rs889312, which they are in high LD with) were found associated with <em>PI3KCA</em> SM. Second, phasing data showed that the SNV alleles associated with increased <em>PIK3CA</em> SM (and <em>MAP3k1/SETD9</em> overexpression) are actually correlated with the reduced BC risk allele (A) of rs889312<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770469">11</a></sup>(<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S10</a>). Hence, their opposite alleles should be associated with BCs in which PIK3CA is not mutated to build up to their overall increased BC risk.<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770465">18</a></sup> This ‘reverse' phase should not surprise because the SNV/<em>PIK3CA</em> SM associations found have an allele unbalancing effect one order of magnitude stronger than the reported SNV/BC risk. However, it predicts the presence of multiple classes of SNV BC risk in the 5q11.2 segment that split when probed for PIK3CA somatic variants.</p><p id="__p10">This multiplicity could be a consequence of MAP3K1 ubiquitinase activity in addition to its kinase activity, which can therefore both activate and destabilise MAP-kinases.<sup><a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770481">21</a>, <a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770483">22</a></sup> The complex BC risk SNV structure has been confirmed by a recent fine scale analysis of 5q11.2 region in a large cohort of patients (not available when we started our investigation) that identified, by logistic regression, four BC risk-associated haplo-blocks.<sup><a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_590770461">17</a></sup> By analysing in the BRCA data set, four genotyped SNV representative of the haplo-blocks, we found that only one SNV allele correlated with enriched <em>PIK3CA</em> SM, and it was associated with a reduced BC risk (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table S9</a>).</p><p id="__p11" class="p p-last">In conclusion, the germline 5q11.2 variants, rs331499 allele A and rs252913 allele G, are associated with <em>MAP3K1</em> and <em>SETD9</em> overexpression, and correlate with increased <em>PIK3CA</em> SM frequency in ER+ BC. Genome-wide analysis of SNV/SM associations can increase our understanding of tumour biology with relevant information for precision medicine.</p></div><div id="idm140390853843424" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140390853843424title">Acknowledgments</h2><div class="sec"><p id="__p12">The results shown here are in part based upon data generated by the TCGA Research Network: <a href="http://cancergenome.nih.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cancergenome.nih.gov/</a>. We thank Paola Ghiorzo, Genova, for helpful comments.</p></div></div><div id="idm140390853841824" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140390853841824title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140390853841696"></p><p id="__p13" class="p p-first-last"><a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Information</a> accompanies this paper on European Journal of Human Genetics website (http://www.nature.com/ejhg)</p><p></p><p class="fn sec" id="idm140390790417216"></p><p id="__p14" class="p p-first-last">The authors declare no conflict of interest.</p><p></p></div></div><div id="sup1" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sup1title">Supplementary Material</h2><!--/article/back/sec/--><div class="sec suppmat" id="xob1"><h4>Supplementary Information</h4><div class="sup-box half_rhythm" id="idm140390819814480"><a href="/pmc/articles/PMC5315515/bin/ejhg2016179x1.txt" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(370 bytes, txt)</sup></div></div><div class="sec suppmat" id="xob2"><h4>Supplementary Information</h4><div class="sup-box half_rhythm" id="idm140390814029408"><a href="/pmc/articles/PMC5315515/bin/ejhg2016179x2.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(367K, doc)</sup></div></div></div><div id="idm140390790416352" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140390790416352title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="bib1"><span class="mixed-citation">Foulkes WD: <span class="ref-title">Inherited susceptibility to common cancers</span>. <span class="ref-journal">N Engl J Med</span>
2008; <span class="ref-vol">359</span>: 2143–2153. [<a href="/pubmed/19005198" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Inherited+susceptibility+to+common+cancers&amp;volume=359&amp;publication_year=2008&amp;pages=2143&amp;pmid=19005198&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib2"><span class="mixed-citation">Pharoah PD, Antoniou A, Bobrow M, Zimmern RL, Easton DF, Ponder BA: <span class="ref-title">Polygenic susceptibility to breast cancer and implications for prevention</span>. <span class="ref-journal">Nat Genet</span>
2002; <span class="ref-vol">31</span>: 33–36. [<a href="/pubmed/11984562" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Polygenic+susceptibility+to+breast+cancer+and+implications+for+prevention&amp;volume=31&amp;publication_year=2002&amp;pages=33&amp;pmid=11984562&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib3"><span class="mixed-citation">Mucci LA, Hjelmborg JB, Harris JR et al: <span class="ref-title">Familial risk and heritability of cancer among twins in Nordic countries</span>. <span class="ref-journal">JAMA</span>
2016; <span class="ref-vol">315</span>: 68–76. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5498110/">PMC free article</a>]</span> [<a href="/pubmed/26746459" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Familial+risk+and+heritability+of+cancer+among+twins+in+Nordic+countries&amp;volume=315&amp;publication_year=2016&amp;pages=68&amp;pmid=26746459&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib4"><span class="mixed-citation">Michailidou K, Beesley J, Lindstrom S et al: <span class="ref-title">Genome-wide association analysis of more than 120,000 individuals identifies 15 new susceptibility loci for breast cancer</span>. <span class="ref-journal">Nat Genet</span>
2015; <span class="ref-vol">47</span>: 373–380. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4549775/">PMC free article</a>]</span> [<a href="/pubmed/25751625" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+analysis+of+more+than+120,000+individuals+identifies+15+new+susceptibility+loci+for+breast+cancer&amp;volume=47&amp;publication_year=2015&amp;pages=373&amp;pmid=25751625&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib5"><span class="mixed-citation">Harlid S, Ivarsson MI, Butt S et al: <span class="ref-title">Combined effect of low-penetrant SNVs on breast cancer risk</span>. <span class="ref-journal">Br J Cancer</span>
2012; <span class="ref-vol">106</span>: 389–396. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3261688/">PMC free article</a>]</span> [<a href="/pubmed/22045194" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Cancer&amp;title=Combined+effect+of+low-penetrant+SNVs+on+breast+cancer+risk&amp;volume=106&amp;publication_year=2012&amp;pages=389&amp;pmid=22045194&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib6"><span class="mixed-citation">Stephens PJ, Tarpey PS, Davies H et al: <span class="ref-title">The landscape of cancer genes and mutational processes in breast cancer</span>. <span class="ref-journal">Nature</span>
2012; <span class="ref-vol">486</span>: 400–404. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3428862/">PMC free article</a>]</span> [<a href="/pubmed/22722201" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+landscape+of+cancer+genes+and+mutational+processes+in+breast+cancer&amp;volume=486&amp;publication_year=2012&amp;pages=400&amp;pmid=22722201&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib7"><span class="mixed-citation">Cancer Genome Atlas Network: <span class="ref-title">Comprehensive molecular portraits of human breast tumours</span>. <span class="ref-journal">Nature</span>
2012; <span class="ref-vol">490</span>: 61–70. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3465532/">PMC free article</a>]</span> [<a href="/pubmed/23000897" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Comprehensive+molecular+portraits+of+human+breast+tumours&amp;volume=490&amp;publication_year=2012&amp;pages=61&amp;pmid=23000897&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib8"><span class="mixed-citation">Stratton MR, Campbell PJ, Futreal PA: <span class="ref-title">The cancer genome</span>. <span class="ref-journal">Nature</span>
2009; <span class="ref-vol">458</span>: 719–724. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2821689/">PMC free article</a>]</span> [<a href="/pubmed/19360079" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+cancer+genome&amp;volume=458&amp;publication_year=2009&amp;pages=719&amp;pmid=19360079&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib9"><span class="mixed-citation">O'Brien KM, Cole SR, Engel LS et al: <span class="ref-title">Breast cancer subtypes and previously established genetic risk factors: a bayesian approach</span>. <span class="ref-journal">Cancer Epidemiol Biomarkers Prev</span>
2014; <span class="ref-vol">23</span>: 84–97. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3947131/">PMC free article</a>]</span> [<a href="/pubmed/24177593" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Breast+cancer+subtypes+and+previously+established+genetic+risk+factors:+a+bayesian+approach&amp;volume=23&amp;publication_year=2014&amp;pages=84&amp;pmid=24177593&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib10"><span class="mixed-citation">Li Q, Seo JH, Stranger B et al: <span class="ref-title">Integrative eQTL-based analyses reveal the biology of breast cancer risk loci</span>. <span class="ref-journal">Cell</span>
2013; <span class="ref-vol">152</span>: 633–641. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4165609/">PMC free article</a>]</span> [<a href="/pubmed/23374354" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Integrative+eQTL-based+analyses+reveal+the+biology+of+breast+cancer+risk+loci&amp;volume=152&amp;publication_year=2013&amp;pages=633&amp;pmid=23374354&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib11"><span class="mixed-citation">Lu PH, Yang J, Li C et al: <span class="ref-title">Association between mitogen-activated protein kinase kinase kinase 1 rs889312 polymorphism and breast cancer risk: evidence from 59977 subjects</span>. <span class="ref-journal">Breast Cancer Res Treat</span>
2011; <span class="ref-vol">126</span>: 663–670. [<a href="/pubmed/20809358" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Breast+Cancer+Res+Treat&amp;title=Association+between+mitogen-activated+protein+kinase+kinase+kinase+1+rs889312+polymorphism+and+breast+cancer+risk:+evidence+from+59977+subjects&amp;volume=126&amp;publication_year=2011&amp;pages=663&amp;pmid=20809358&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib12"><span class="mixed-citation">Rhie SK, Coetzee SG, Noushmehr H et al: <span class="ref-title">Comprehensive functional annotation of seventy-one breast cancer risk Loci</span>. <span class="ref-journal">PLoS One</span>
2013; <span class="ref-vol">8</span>: e63925. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3661550/">PMC free article</a>]</span> [<a href="/pubmed/23717510" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Comprehensive+functional+annotation+of+seventy-one+breast+cancer+risk+Loci&amp;volume=8&amp;publication_year=2013&amp;pages=e63925&amp;pmid=23717510&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib13"><span class="mixed-citation">Ellis MJ, Ding L, Shen D et al: <span class="ref-title">Whole-genome analysis informs breast cancer response to aromatase inhibition</span>. <span class="ref-journal">Nature</span>
2012; <span class="ref-vol">486</span>: 353–360. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3383766/">PMC free article</a>]</span> [<a href="/pubmed/22722193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Whole-genome+analysis+informs+breast+cancer+response+to+aromatase+inhibition&amp;volume=486&amp;publication_year=2012&amp;pages=353&amp;pmid=22722193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib14"><span class="mixed-citation">Tryka KA, Hao L, Sturcke A et al: <span class="ref-title">NCBI's Database of Genotypes and Phenotypes: dbGaP</span>. <span class="ref-journal">Nucleic Acids Res</span>
2014; <span class="ref-vol">42</span>: D975–D979. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3965052/">PMC free article</a>]</span> [<a href="/pubmed/24297256" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=NCBI's+Database+of+Genotypes+and+Phenotypes:+dbGaP&amp;volume=42&amp;publication_year=2014&amp;pages=D975&amp;pmid=24297256&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib15"><span class="mixed-citation">Wilks C, Cline MS, Weiler E et al: <span class="ref-title">The Cancer Genomics Hub (CGHub): overcoming cancer through the power of torrential data</span>. <span class="ref-journal">Database</span>
2014, e-pub ahead of print 29 September 2014 doi:10.1093/database/bau093. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4178372/">PMC free article</a>]</span> [<a href="/pubmed/25267794" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></li><li id="bib16"><span class="mixed-citation">Cerami, Gao J, Dogrusoz U et al: <span class="ref-title">The cBio Cancer Genomics Portal: an open platform for exploring multidimensional cancer genomics data</span>. <span class="ref-journal">Cancer Discov</span>
2012; <span class="ref-vol">2</span>: 401. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3956037/">PMC free article</a>]</span> [<a href="/pubmed/22588877" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Discov&amp;title=The+cBio+Cancer+Genomics+Portal:+an+open+platform+for+exploring+multidimensional+cancer+genomics+data&amp;volume=2&amp;publication_year=2012&amp;pages=401&amp;pmid=22588877&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib17"><span class="mixed-citation">Nyholt DR: <span class="ref-title">A simple correction for multiple testing for single-nucleotide polymorphisms in linkage disequilibrium with each other</span>. <span class="ref-journal">Am J Hum Genet</span>
2004; <span class="ref-vol">74</span>: 765–769. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1181954/">PMC free article</a>]</span> [<a href="/pubmed/14997420" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+simple+correction+for+multiple+testing+for+single-nucleotide+polymorphisms+in+linkage+disequilibrium+with+each+other&amp;volume=74&amp;publication_year=2004&amp;pages=765&amp;pmid=14997420&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib18"><span class="mixed-citation">Glubb DM, Maranian MJ, Michailidou K et al: <span class="ref-title">Fine-scale mapping of the 5q11.2 breast cancer locus reveals at least three independent risk variants regulating MAP3K1</span>. <span class="ref-journal">Am J Hum Genet</span>
2015; <span class="ref-vol">96</span>: 5–20. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4289692/">PMC free article</a>]</span> [<a href="/pubmed/25529635" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Fine-scale+mapping+of+the+5q11.2+breast+cancer+locus+reveals+at+least+three+independent+risk+variants+regulating+MAP3K1&amp;volume=96&amp;publication_year=2015&amp;pages=5&amp;pmid=25529635&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib19"><span class="mixed-citation">Aksamitiene E, Kiyatkin A, Kholodenko BN: <span class="ref-title">Cross-talk between mitogenic Ras/MAPK and survival PI3K/Akt pathways: a fine balance</span>. <span class="ref-journal">Biochem Soc Trans</span>
2012; <span class="ref-vol">40</span>: 139–146. [<a href="/pubmed/22260680" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Soc+Trans&amp;title=Cross-talk+between+mitogenic+Ras/MAPK+and+survival+PI3K/Akt+pathways:+a+fine+balance&amp;volume=40&amp;publication_year=2012&amp;pages=139&amp;pmid=22260680&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib20"><span class="mixed-citation">Britten CD: <span class="ref-title">PI3K and MEK inhibitor combinations: examining the evidence in selected tumor types</span>. <span class="ref-journal">Cancer Chemother Pharmacol</span>
2013; <span class="ref-vol">71</span>: 1395–1409. [<a href="/pubmed/23443307" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Chemother+Pharmacol&amp;title=PI3K+and+MEK+inhibitor+combinations:+examining+the+evidence+in+selected+tumor+types&amp;volume=71&amp;publication_year=2013&amp;pages=1395&amp;pmid=23443307&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib21"><span class="mixed-citation">Lu Z, Xu S, Joazeiro C, Cobb MH, Hunter T: <span class="ref-title">The PHD domain of MEKK1 acts as an E3 ubiquitin ligase and mediates ubiquitination and degradation of ERK1/2</span>. <span class="ref-journal">Mol Cell</span>
2002; <span class="ref-vol">9</span>: 945–956. [<a href="/pubmed/12049732" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell&amp;title=The+PHD+domain+of+MEKK1+acts+as+an+E3+ubiquitin+ligase+and+mediates+ubiquitination+and+degradation+of+ERK1/2&amp;volume=9&amp;publication_year=2002&amp;pages=945&amp;pmid=12049732&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib22"><span class="mixed-citation">Xia Y, Wang J, Xu S, Johnson GL, Hunter T, Lu Z: <span class="ref-title">MEKK1 mediates the ubiquitination and degradation of c-Jun in response to osmotic stress</span>. <span class="ref-journal">Mol Cell Biol</span>
2007; <span class="ref-vol">27</span>: 510–517. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1800814/">PMC free article</a>]</span> [<a href="/pubmed/17101801" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=MEKK1+mediates+the+ubiquitination+and+degradation+of+c-Jun+in+response+to+osmotic+stress&amp;volume=27&amp;publication_year=2007&amp;pages=510&amp;pmid=17101801&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5315515&amp;issue-id=287993&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140390815345856title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Methods</a></li><li><a href="#__sec3title">Results</a></li><li><a href="#__sec4title">Discussion</a></li><li><a href="#idm140390853843424title">Acknowledgments</a></li><li><a href="#idm140390853841824title">Footnotes</a></li><li><a href="#sup1title">Supplementary Material</a></li><li><a href="#idm140390790416352title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">European Journal of Human Genetics</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5315515/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5315515/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5315515/pdf/ejhg2016179a.pdf">PDF (112K)</a></li> | <li><a href="#" data-citationid="PMC5315515" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5315515%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5315515%2F&amp;text=SNP%20variants%20at%20the%20MAP3K1%2FSETD9%20locus%205q11.2%20associate%20with%20somatic%20PIK3CA%20variants%20in%20breast%20cancers" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5315515%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="28029147" data-aiid="5315515" data-aid="5315515" data-iid="287993" data-domainid="1300" data-domain="ejhg" data-accid="PMC5315515" data-md5="b38de130024a86aaa4b10e975083fb3d"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T15:30:24-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89C678DD592AF1000000000A520619&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5315515%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ejhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5315515/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>