<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5975608/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association studies in women of African ancestry identified 3q26.21 as a novel susceptibility locus for oestrogen receptor negative breast cancer</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5975608/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association studies in women of African ancestry identified 3q26.21 as a novel susceptibility locus for oestrogen receptor negative breast cancer"><meta name="citation_authors" content="Dezheng Huo, Ye Feng, Stephen Haddad, Yonglan Zheng, Song Yao, Yoo-Jeong Han, Temidayo O. Ogundiran, Clement Adebamowo, Oladosu Ojengbede, Adeyinka G. Falusi, Wei Zheng, William Blot, Qiuyin Cai, Lisa Signorello, Esther M. John, Leslie Bernstein, Jennifer J. Hu, Regina G. Ziegler, Sarah Nyante, Elisa V. Bandera, Sue A. Ingles, Michael F. Press, Sandra L. Deming, Jorge L. Rodriguez-Gil, Katherine L. Nathanson, Susan M. Domchek, Timothy R. Rebbeck, Edward A. Ruiz-Narváez, Lara E. Sucheston-Campbell, Jeannette T. Bensen, Michael S. Simon, Anselm Hennis, Barbara Nemesure, M. Cristina Leske, Stefan Ambs, Lin S. Chen, Frank Qian, Eric R. Gamazon, Kathryn L. Lunetta, Nancy J. Cox, Stephen J. Chanock, Laurence N. Kolonel, Andrew F. Olshan, Christine B. Ambrosone, Olufunmilayo I. Olopade, Julie R. Palmer, Christopher A. Haiman"><meta name="citation_date" content="1 November 2016"><meta name="citation_issue" content="21"><meta name="citation_volume" content="25"><meta name="citation_firstpage" content="4835"><meta name="citation_doi" content="10.1093/hmg/ddw305"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5975608/?report=abstract"><meta name="citation_pmid" content="28171663"><meta name="DC.Title" content="Genome-wide association studies in women of African ancestry identified 3q26.21 as a novel susceptibility locus for oestrogen receptor negative breast cancer"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Dezheng Huo"><meta name="DC.Contributor" content="Ye Feng"><meta name="DC.Contributor" content="Stephen Haddad"><meta name="DC.Contributor" content="Yonglan Zheng"><meta name="DC.Contributor" content="Song Yao"><meta name="DC.Contributor" content="Yoo-Jeong Han"><meta name="DC.Contributor" content="Temidayo O. Ogundiran"><meta name="DC.Contributor" content="Clement Adebamowo"><meta name="DC.Contributor" content="Oladosu Ojengbede"><meta name="DC.Contributor" content="Adeyinka G. Falusi"><meta name="DC.Contributor" content="Wei Zheng"><meta name="DC.Contributor" content="William Blot"><meta name="DC.Contributor" content="Qiuyin Cai"><meta name="DC.Contributor" content="Lisa Signorello"><meta name="DC.Contributor" content="Esther M. John"><meta name="DC.Contributor" content="Leslie Bernstein"><meta name="DC.Contributor" content="Jennifer J. Hu"><meta name="DC.Contributor" content="Regina G. Ziegler"><meta name="DC.Contributor" content="Sarah Nyante"><meta name="DC.Contributor" content="Elisa V. Bandera"><meta name="DC.Contributor" content="Sue A. Ingles"><meta name="DC.Contributor" content="Michael F. Press"><meta name="DC.Contributor" content="Sandra L. Deming"><meta name="DC.Contributor" content="Jorge L. Rodriguez-Gil"><meta name="DC.Contributor" content="Katherine L. Nathanson"><meta name="DC.Contributor" content="Susan M. Domchek"><meta name="DC.Contributor" content="Timothy R. Rebbeck"><meta name="DC.Contributor" content="Edward A. Ruiz-Narváez"><meta name="DC.Contributor" content="Lara E. Sucheston-Campbell"><meta name="DC.Contributor" content="Jeannette T. Bensen"><meta name="DC.Contributor" content="Michael S. Simon"><meta name="DC.Contributor" content="Anselm Hennis"><meta name="DC.Contributor" content="Barbara Nemesure"><meta name="DC.Contributor" content="M. Cristina Leske"><meta name="DC.Contributor" content="Stefan Ambs"><meta name="DC.Contributor" content="Lin S. Chen"><meta name="DC.Contributor" content="Frank Qian"><meta name="DC.Contributor" content="Eric R. Gamazon"><meta name="DC.Contributor" content="Kathryn L. Lunetta"><meta name="DC.Contributor" content="Nancy J. Cox"><meta name="DC.Contributor" content="Stephen J. Chanock"><meta name="DC.Contributor" content="Laurence N. Kolonel"><meta name="DC.Contributor" content="Andrew F. Olshan"><meta name="DC.Contributor" content="Christine B. Ambrosone"><meta name="DC.Contributor" content="Olufunmilayo I. Olopade"><meta name="DC.Contributor" content="Julie R. Palmer"><meta name="DC.Contributor" content="Christopher A. Haiman"><meta name="DC.Date" content="2016 Nov 1"><meta name="DC.Identifier" content="10.1093/hmg/ddw305"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association studies in women of African ancestry identified 3q26.21 as a novel susceptibility locus for oestrogen receptor negative breast cancer"><meta property="og:type" content="article"><meta property="og:description" content="Multiple breast cancer loci have been identified in previous genome-wide association studies, but they were mainly conducted in populations of European ancestry. Women of African ancestry are more likely to have young-onset and oestrogen receptor (ER) ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5975608/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5975608/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5975608/pdf/ddw305.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE88C218DD5A2411000000000271016C.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathEvents.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathZoom.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathMenu.js?rev=2.6.1"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5975608%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC5975608</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2016 Nov 1; 25(21): 4835–4846. </span></div><div><span class="fm-vol-iss-date">Published online 2016 Sep 4. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddw305" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddw305</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5975608</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/28171663">28171663</a></div></div></div></div><h1 class="content-title">Genome-wide association studies in women of African ancestry identified 3q26.21 as a novel susceptibility locus for oestrogen receptor negative breast cancer</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Huo%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015760693760" co-class="co-affbox">Dezheng Huo</a>,<sup>1</sup> <a href="/pubmed/?term=Feng%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015760679216" co-class="co-affbox">Ye Feng</a>,<sup>2</sup> <a href="/pubmed/?term=Haddad%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015760676832" co-class="co-affbox">Stephen Haddad</a>,<sup>3</sup> <a href="/pubmed/?term=Zheng%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015760674512" co-class="co-affbox">Yonglan Zheng</a>,<sup>4</sup> <a href="/pubmed/?term=Yao%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759251600" co-class="co-affbox">Song Yao</a>,<sup>5</sup> <a href="/pubmed/?term=Han%20YJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759249616" co-class="co-affbox">Yoo-Jeong Han</a>,<sup>4</sup> <a href="/pubmed/?term=Ogundiran%20TO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759247632" co-class="co-affbox">Temidayo O. Ogundiran</a>,<sup>6</sup> <a href="/pubmed/?term=Adebamowo%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759245648" co-class="co-affbox">Clement Adebamowo</a>,<sup>7</sup> <a href="/pubmed/?term=Ojengbede%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759243664" co-class="co-affbox">Oladosu Ojengbede</a>,<sup>8</sup> <a href="/pubmed/?term=Falusi%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759241680" co-class="co-affbox">Adeyinka G. Falusi</a>,<sup>9</sup> <a href="/pubmed/?term=Zheng%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015762057488" co-class="co-affbox">Wei Zheng</a>,<sup>10</sup> <a href="/pubmed/?term=Blot%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015762055248" co-class="co-affbox">William Blot</a>,<sup>10</sup> <a href="/pubmed/?term=Cai%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015762053040" co-class="co-affbox">Qiuyin Cai</a>,<sup>10</sup> <a href="/pubmed/?term=Signorello%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015762050864" co-class="co-affbox">Lisa Signorello</a>,<sup>11</sup> <a href="/pubmed/?term=John%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015762048592" co-class="co-affbox">Esther M. John</a>,<sup>12,</sup><sup>13</sup> <a href="/pubmed/?term=Bernstein%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015762045424" co-class="co-affbox">Leslie Bernstein</a>,<sup>14</sup> <a href="/pubmed/?term=Hu%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015762043152" co-class="co-affbox">Jennifer J. Hu</a>,<sup>15</sup> <a href="/pubmed/?term=Ziegler%20RG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015762040912" co-class="co-affbox">Regina G. Ziegler</a>,<sup>16</sup> <a href="/pubmed/?term=Nyante%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015762038640" co-class="co-affbox">Sarah Nyante</a>,<sup>17</sup> <a href="/pubmed/?term=Bandera%20EV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015762036368" co-class="co-affbox">Elisa V. Bandera</a>,<sup>18</sup> <a href="/pubmed/?term=Ingles%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015762034096" co-class="co-affbox">Sue A. Ingles</a>,<sup>2</sup> <a href="/pubmed/?term=Press%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759091360" co-class="co-affbox">Michael F. Press</a>,<sup>19</sup> <a href="/pubmed/?term=Deming%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759089376" co-class="co-affbox">Sandra L. Deming</a>,<sup>10</sup> <a href="/pubmed/?term=Rodriguez-Gil%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759087392" co-class="co-affbox">Jorge L. Rodriguez-Gil</a>,<sup>15</sup> <a href="/pubmed/?term=Nathanson%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759085408" co-class="co-affbox">Katherine L. Nathanson</a>,<sup>20</sup> <a href="/pubmed/?term=Domchek%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759083424" co-class="co-affbox">Susan M. Domchek</a>,<sup>20</sup> <a href="/pubmed/?term=Rebbeck%20TR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759081440" co-class="co-affbox">Timothy R. Rebbeck</a>,<sup>21</sup> <a href="/pubmed/?term=Ruiz-Narv%26%23x000e1%3Bez%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759079456" co-class="co-affbox">Edward A. Ruiz-Narváez</a>,<sup>3</sup> <a href="/pubmed/?term=Sucheston-Campbell%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759077472" co-class="co-affbox">Lara E. Sucheston-Campbell</a>,<sup>22</sup> <a href="/pubmed/?term=Bensen%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759075488" co-class="co-affbox">Jeannette T. Bensen</a>,<sup>17</sup> <a href="/pubmed/?term=Simon%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759073504" co-class="co-affbox">Michael S. Simon</a>,<sup>23</sup> <a href="/pubmed/?term=Hennis%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759071520" co-class="co-affbox">Anselm Hennis</a>,<sup>24,</sup><sup>25</sup> <a href="/pubmed/?term=Nemesure%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759068800" co-class="co-affbox">Barbara Nemesure</a>,<sup>25</sup> <a href="/pubmed/?term=Leske%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759066816" co-class="co-affbox">M. Cristina Leske</a>,<sup>25</sup> <a href="/pubmed/?term=Ambs%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759064832" co-class="co-affbox">Stefan Ambs</a>,<sup>26</sup> <a href="/pubmed/?term=Chen%20LS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759062848" co-class="co-affbox">Lin S. Chen</a>,<sup>1</sup> <a href="/pubmed/?term=Qian%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759060864" co-class="co-affbox">Frank Qian</a>,<sup>4</sup> <a href="/pubmed/?term=Gamazon%20ER%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759058880" co-class="co-affbox">Eric R. Gamazon</a>,<sup>27,</sup><sup>28</sup> <a href="/pubmed/?term=Lunetta%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759056160" co-class="co-affbox">Kathryn L. Lunetta</a>,<sup>29</sup> <a href="/pubmed/?term=Cox%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759054176" co-class="co-affbox">Nancy J. Cox</a>,<sup>27</sup> <a href="/pubmed/?term=Chanock%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759052192" co-class="co-affbox">Stephen J. Chanock</a>,<sup>16</sup> <a href="/pubmed/?term=Kolonel%20LN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759050208" co-class="co-affbox">Laurence N. Kolonel</a>,<sup>30</sup> <a href="/pubmed/?term=Olshan%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759048224" co-class="co-affbox">Andrew F. Olshan</a>,<sup>17</sup> <a href="/pubmed/?term=Ambrosone%20CB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759046240" co-class="co-affbox">Christine B. Ambrosone</a>,<sup>5</sup> <a href="/pubmed/?term=Olopade%20OI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759044256" co-class="co-affbox">Olufunmilayo I. Olopade</a>,<sup>4</sup><sup></sup> <a href="/pubmed/?term=Palmer%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759041536" co-class="co-affbox">Julie R. Palmer</a>,<sup>3</sup><sup></sup> and  <a href="/pubmed/?term=Haiman%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663" class="affpopup" co-rid="_co_idm140015759038816" co-class="co-affbox">Christopher A. Haiman</a><sup>2</sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140015760693760"><h3 class="no_margin">Dezheng Huo</h3><p>  <sup>1</sup>Department of Public Health Sciences, University of Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Huo%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Dezheng Huo</a></div></div><div id="_co_idm140015760679216"><h3 class="no_margin">Ye Feng</h3><p>  <sup>2</sup>Department of Preventive Medicine, Keck School of Medicine and Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Feng%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Ye Feng</a></div></div><div id="_co_idm140015760676832"><h3 class="no_margin">Stephen Haddad</h3><p>  <sup>3</sup>Slone Epidemiology Center, Boston University, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Haddad%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Stephen Haddad</a></div></div><div id="_co_idm140015760674512"><h3 class="no_margin">Yonglan Zheng</h3><p>  <sup>4</sup>Center for Clinical Cancer Genetics and Global Health, Department of Medicine, University of Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Zheng%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Yonglan Zheng</a></div></div><div id="_co_idm140015759251600"><h3 class="no_margin">Song Yao</h3><p>  <sup>5</sup>Roswell Park Cancer Institute, Buffalo, NY, USA</p><div>Find articles by <a href="/pubmed/?term=Yao%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Song Yao</a></div></div><div id="_co_idm140015759249616"><h3 class="no_margin">Yoo-Jeong Han</h3><p>  <sup>4</sup>Center for Clinical Cancer Genetics and Global Health, Department of Medicine, University of Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Han%20YJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Yoo-Jeong Han</a></div></div><div id="_co_idm140015759247632"><h3 class="no_margin">Temidayo O. Ogundiran</h3><p>  <sup>6</sup>Department of Surgery, College of Medicine, University of Ibadan, Ibadan, Nigeria</p><div>Find articles by <a href="/pubmed/?term=Ogundiran%20TO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Temidayo O. Ogundiran</a></div></div><div id="_co_idm140015759245648"><h3 class="no_margin">Clement Adebamowo</h3><p>  <sup>7</sup>Department of Epidemiology &amp; Preventive Medicine, University of Maryland, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Adebamowo%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Clement Adebamowo</a></div></div><div id="_co_idm140015759243664"><h3 class="no_margin">Oladosu Ojengbede</h3><p>  <sup>8</sup>Center for Population and Reproductive Health, College of Medicine, University of Ibadan, Ibadan, Nigeria</p><div>Find articles by <a href="/pubmed/?term=Ojengbede%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Oladosu Ojengbede</a></div></div><div id="_co_idm140015759241680"><h3 class="no_margin">Adeyinka G. Falusi</h3><p>  <sup>9</sup>Institute for Medical Research and Training, College of Medicine, University of Ibadan, Ibadan, Nigeria</p><div>Find articles by <a href="/pubmed/?term=Falusi%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Adeyinka G. Falusi</a></div></div><div id="_co_idm140015762057488"><h3 class="no_margin">Wei Zheng</h3><p>  <sup>10</sup>Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology Center, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=Zheng%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Wei Zheng</a></div></div><div id="_co_idm140015762055248"><h3 class="no_margin">William Blot</h3><p>  <sup>10</sup>Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology Center, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=Blot%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">William Blot</a></div></div><div id="_co_idm140015762053040"><h3 class="no_margin">Qiuyin Cai</h3><p>  <sup>10</sup>Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology Center, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=Cai%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Qiuyin Cai</a></div></div><div id="_co_idm140015762050864"><h3 class="no_margin">Lisa Signorello</h3><p>  <sup>11</sup>Cancer Prevention Fellowship Program, National Cancer Institute, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Signorello%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Lisa Signorello</a></div></div><div id="_co_idm140015762048592"><h3 class="no_margin">Esther M. John</h3><p>  <sup>12</sup>Cancer Prevention Institute of California, Fremont, CA, USA</p><p>  <sup>13</sup>Department of Health Research &amp; Policy and Stanford Cancer Institute, Stanford University School of Medicine Stanford, CA, USA</p><div>Find articles by <a href="/pubmed/?term=John%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Esther M. John</a></div></div><div id="_co_idm140015762045424"><h3 class="no_margin">Leslie Bernstein</h3><p>  <sup>14</sup>Division of Cancer Etiology, Department of Population Sciences, Beckman Research Institute of City of Hope, Duarte, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Bernstein%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Leslie Bernstein</a></div></div><div id="_co_idm140015762043152"><h3 class="no_margin">Jennifer J. Hu</h3><p>  <sup>15</sup>Sylvester Comprehensive Cancer Center and Department of Public Health Sciences, University of Miami Miller School of Medicine, Miami, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Hu%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Jennifer J. Hu</a></div></div><div id="_co_idm140015762040912"><h3 class="no_margin">Regina G. Ziegler</h3><p>  <sup>16</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, Bethesda, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Ziegler%20RG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Regina G. Ziegler</a></div></div><div id="_co_idm140015762038640"><h3 class="no_margin">Sarah Nyante</h3><p>  <sup>17</sup>Department of Epidemiology, Gillings School of Global Public Health, and Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Nyante%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Sarah Nyante</a></div></div><div id="_co_idm140015762036368"><h3 class="no_margin">Elisa V. Bandera</h3><p>  <sup>18</sup>The Cancer Institute of New Jersey, New Brunswick, NJ, USA</p><div>Find articles by <a href="/pubmed/?term=Bandera%20EV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Elisa V. Bandera</a></div></div><div id="_co_idm140015762034096"><h3 class="no_margin">Sue A. Ingles</h3><p>  <sup>2</sup>Department of Preventive Medicine, Keck School of Medicine and Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Ingles%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Sue A. Ingles</a></div></div><div id="_co_idm140015759091360"><h3 class="no_margin">Michael F. Press</h3><p>  <sup>19</sup>Department of Pathology, Keck School of Medicine and Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Press%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Michael F. Press</a></div></div><div id="_co_idm140015759089376"><h3 class="no_margin">Sandra L. Deming</h3><p>  <sup>10</sup>Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology Center, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=Deming%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Sandra L. Deming</a></div></div><div id="_co_idm140015759087392"><h3 class="no_margin">Jorge L. Rodriguez-Gil</h3><p>  <sup>15</sup>Sylvester Comprehensive Cancer Center and Department of Public Health Sciences, University of Miami Miller School of Medicine, Miami, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Rodriguez-Gil%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Jorge L. Rodriguez-Gil</a></div></div><div id="_co_idm140015759085408"><h3 class="no_margin">Katherine L. Nathanson</h3><p>  <sup>20</sup>Department of Medicine, University of Pennsylvania, Philadelphia, PA, USA</p><div>Find articles by <a href="/pubmed/?term=Nathanson%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Katherine L. Nathanson</a></div></div><div id="_co_idm140015759083424"><h3 class="no_margin">Susan M. Domchek</h3><p>  <sup>20</sup>Department of Medicine, University of Pennsylvania, Philadelphia, PA, USA</p><div>Find articles by <a href="/pubmed/?term=Domchek%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Susan M. Domchek</a></div></div><div id="_co_idm140015759081440"><h3 class="no_margin">Timothy R. Rebbeck</h3><p>  <sup>21</sup>Dana Farber Cancer Institute &amp; Harvard T.H. Chan School of Public Health, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Rebbeck%20TR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Timothy R. Rebbeck</a></div></div><div id="_co_idm140015759079456"><h3 class="no_margin">Edward A. Ruiz-Narváez</h3><p>  <sup>3</sup>Slone Epidemiology Center, Boston University, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Ruiz-Narv%26%23x000e1%3Bez%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Edward A. Ruiz-Narváez</a></div></div><div id="_co_idm140015759077472"><h3 class="no_margin">Lara E. Sucheston-Campbell</h3><p>  <sup>22</sup>Colleges of Pharmacy and Veterinary Medicine, Ohio State University, Columbus, OH, USA</p><div>Find articles by <a href="/pubmed/?term=Sucheston-Campbell%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Lara E. Sucheston-Campbell</a></div></div><div id="_co_idm140015759075488"><h3 class="no_margin">Jeannette T. Bensen</h3><p>  <sup>17</sup>Department of Epidemiology, Gillings School of Global Public Health, and Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Bensen%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Jeannette T. Bensen</a></div></div><div id="_co_idm140015759073504"><h3 class="no_margin">Michael S. Simon</h3><p>  <sup>23</sup>Karmanos Cancer Institute, Department of Oncology, Wayne State University, Detroit, MI, USA</p><div>Find articles by <a href="/pubmed/?term=Simon%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Michael S. Simon</a></div></div><div id="_co_idm140015759071520"><h3 class="no_margin">Anselm Hennis</h3><p>  <sup>24</sup>Chronic Disease Research Centre, Tropical Medicine Research Institute, University of the West Indies, Bridgetown, Barbados</p><p>  <sup>25</sup>Department of Preventive Medicine, State University of New York at Stony Brook, Stony Brook, NY, USA</p><div>Find articles by <a href="/pubmed/?term=Hennis%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Anselm Hennis</a></div></div><div id="_co_idm140015759068800"><h3 class="no_margin">Barbara Nemesure</h3><p>  <sup>25</sup>Department of Preventive Medicine, State University of New York at Stony Brook, Stony Brook, NY, USA</p><div>Find articles by <a href="/pubmed/?term=Nemesure%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Barbara Nemesure</a></div></div><div id="_co_idm140015759066816"><h3 class="no_margin">M. Cristina Leske</h3><p>  <sup>25</sup>Department of Preventive Medicine, State University of New York at Stony Brook, Stony Brook, NY, USA</p><div>Find articles by <a href="/pubmed/?term=Leske%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">M. Cristina Leske</a></div></div><div id="_co_idm140015759064832"><h3 class="no_margin">Stefan Ambs</h3><p>  <sup>26</sup>Laboratory of Human Carcinogenesis, National Cancer Institute, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Ambs%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Stefan Ambs</a></div></div><div id="_co_idm140015759062848"><h3 class="no_margin">Lin S. Chen</h3><p>  <sup>1</sup>Department of Public Health Sciences, University of Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Chen%20LS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Lin S. Chen</a></div></div><div id="_co_idm140015759060864"><h3 class="no_margin">Frank Qian</h3><p>  <sup>4</sup>Center for Clinical Cancer Genetics and Global Health, Department of Medicine, University of Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Qian%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Frank Qian</a></div></div><div id="_co_idm140015759058880"><h3 class="no_margin">Eric R. Gamazon</h3><p>  <sup>27</sup>Division of Genetic Medicine, Department of Medicine, Vanderbilt University, Nashville, TN, USA</p><p>  <sup>28</sup>Academic Medical Center, University of Amsterdam, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Gamazon%20ER%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Eric R. Gamazon</a></div></div><div id="_co_idm140015759056160"><h3 class="no_margin">Kathryn L. Lunetta</h3><p>  <sup>29</sup>Department of Biostatistics, Boston University School of Public Health, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Lunetta%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Kathryn L. Lunetta</a></div></div><div id="_co_idm140015759054176"><h3 class="no_margin">Nancy J. Cox</h3><p>  <sup>27</sup>Division of Genetic Medicine, Department of Medicine, Vanderbilt University, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=Cox%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Nancy J. Cox</a></div></div><div id="_co_idm140015759052192"><h3 class="no_margin">Stephen J. Chanock</h3><p>  <sup>16</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, Bethesda, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Chanock%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Stephen J. Chanock</a></div></div><div id="_co_idm140015759050208"><h3 class="no_margin">Laurence N. Kolonel</h3><p>  <sup>30</sup>Epidemiology Program, University of Hawaii Cancer Center, Honolulu, HI, USA</p><div>Find articles by <a href="/pubmed/?term=Kolonel%20LN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Laurence N. Kolonel</a></div></div><div id="_co_idm140015759048224"><h3 class="no_margin">Andrew F. Olshan</h3><p>  <sup>17</sup>Department of Epidemiology, Gillings School of Global Public Health, and Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Olshan%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Andrew F. Olshan</a></div></div><div id="_co_idm140015759046240"><h3 class="no_margin">Christine B. Ambrosone</h3><p>  <sup>5</sup>Roswell Park Cancer Institute, Buffalo, NY, USA</p><div>Find articles by <a href="/pubmed/?term=Ambrosone%20CB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Christine B. Ambrosone</a></div></div><div id="_co_idm140015759044256"><h3 class="no_margin">Olufunmilayo I. Olopade</h3><p>  <sup>4</sup>Center for Clinical Cancer Genetics and Global Health, Department of Medicine, University of Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Olopade%20OI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Olufunmilayo I. Olopade</a></div></div><div id="_co_idm140015759041536"><h3 class="no_margin">Julie R. Palmer</h3><p>  <sup>3</sup>Slone Epidemiology Center, Boston University, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Palmer%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Julie R. Palmer</a></div></div><div id="_co_idm140015759038816"><h3 class="no_margin">Christopher A. Haiman</h3><p>  <sup>2</sup>Department of Preventive Medicine, Keck School of Medicine and Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Haiman%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171663">Christopher A. Haiman</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140015762835168_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140015762835168_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140015762835168_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140015762835168_ai" style="display:none"><div class="fm-affl" id="ddw305-aff1" lang="en">  <sup>1</sup>Department of Public Health Sciences, University of Chicago, Chicago, IL, USA</div><div class="fm-affl" id="ddw305-aff2" lang="en">  <sup>2</sup>Department of Preventive Medicine, Keck School of Medicine and Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, CA, USA</div><div class="fm-affl" id="ddw305-aff3" lang="en">  <sup>3</sup>Slone Epidemiology Center, Boston University, Boston, MA, USA</div><div class="fm-affl" id="ddw305-aff4" lang="en">  <sup>4</sup>Center for Clinical Cancer Genetics and Global Health, Department of Medicine, University of Chicago, Chicago, IL, USA</div><div class="fm-affl" id="ddw305-aff5" lang="en">  <sup>5</sup>Roswell Park Cancer Institute, Buffalo, NY, USA</div><div class="fm-affl" id="ddw305-aff6" lang="en">  <sup>6</sup>Department of Surgery, College of Medicine, University of Ibadan, Ibadan, Nigeria</div><div class="fm-affl" id="ddw305-aff7" lang="en">  <sup>7</sup>Department of Epidemiology &amp; Preventive Medicine, University of Maryland, Baltimore, MD, USA</div><div class="fm-affl" id="ddw305-aff8" lang="en">  <sup>8</sup>Center for Population and Reproductive Health, College of Medicine, University of Ibadan, Ibadan, Nigeria</div><div class="fm-affl" id="ddw305-aff9" lang="en">  <sup>9</sup>Institute for Medical Research and Training, College of Medicine, University of Ibadan, Ibadan, Nigeria</div><div class="fm-affl" id="ddw305-aff10" lang="en">  <sup>10</sup>Division of Epidemiology, Department of Medicine, Vanderbilt Epidemiology Center, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, TN, USA</div><div class="fm-affl" id="ddw305-aff11" lang="en">  <sup>11</sup>Cancer Prevention Fellowship Program, National Cancer Institute, Bethesda, MD, USA</div><div class="fm-affl" id="ddw305-aff12" lang="en">  <sup>12</sup>Cancer Prevention Institute of California, Fremont, CA, USA</div><div class="fm-affl" id="ddw305-aff13" lang="en">  <sup>13</sup>Department of Health Research &amp; Policy and Stanford Cancer Institute, Stanford University School of Medicine Stanford, CA, USA</div><div class="fm-affl" id="ddw305-aff14" lang="en">  <sup>14</sup>Division of Cancer Etiology, Department of Population Sciences, Beckman Research Institute of City of Hope, Duarte, CA, USA</div><div class="fm-affl" id="ddw305-aff15" lang="en">  <sup>15</sup>Sylvester Comprehensive Cancer Center and Department of Public Health Sciences, University of Miami Miller School of Medicine, Miami, FL, USA</div><div class="fm-affl" id="ddw305-aff16" lang="en">  <sup>16</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, Bethesda, MA, USA</div><div class="fm-affl" id="ddw305-aff17" lang="en">  <sup>17</sup>Department of Epidemiology, Gillings School of Global Public Health, and Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, NC, USA</div><div class="fm-affl" id="ddw305-aff18" lang="en">  <sup>18</sup>The Cancer Institute of New Jersey, New Brunswick, NJ, USA</div><div class="fm-affl" id="ddw305-aff19" lang="en">  <sup>19</sup>Department of Pathology, Keck School of Medicine and Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, CA, USA</div><div class="fm-affl" id="ddw305-aff20" lang="en">  <sup>20</sup>Department of Medicine, University of Pennsylvania, Philadelphia, PA, USA</div><div class="fm-affl" id="ddw305-aff21" lang="en">  <sup>21</sup>Dana Farber Cancer Institute &amp; Harvard T.H. Chan School of Public Health, Boston, MA, USA</div><div class="fm-affl" id="ddw305-aff22" lang="en">  <sup>22</sup>Colleges of Pharmacy and Veterinary Medicine, Ohio State University, Columbus, OH, USA</div><div class="fm-affl" id="ddw305-aff23" lang="en">  <sup>23</sup>Karmanos Cancer Institute, Department of Oncology, Wayne State University, Detroit, MI, USA</div><div class="fm-affl" id="ddw305-aff24" lang="en">  <sup>24</sup>Chronic Disease Research Centre, Tropical Medicine Research Institute, University of the West Indies, Bridgetown, Barbados</div><div class="fm-affl" id="ddw305-aff25" lang="en">  <sup>25</sup>Department of Preventive Medicine, State University of New York at Stony Brook, Stony Brook, NY, USA</div><div class="fm-affl" id="ddw305-aff26" lang="en">  <sup>26</sup>Laboratory of Human Carcinogenesis, National Cancer Institute, Bethesda, MD, USA</div><div class="fm-affl" id="ddw305-aff27" lang="en">  <sup>27</sup>Division of Genetic Medicine, Department of Medicine, Vanderbilt University, Nashville, TN, USA</div><div class="fm-affl" id="ddw305-aff28" lang="en">  <sup>28</sup>Academic Medical Center, University of Amsterdam, Amsterdam, The Netherlands</div><div class="fm-affl" id="ddw305-aff29" lang="en">  <sup>29</sup>Department of Biostatistics, Boston University School of Public Health, Boston, MA, USA</div><div class="fm-affl" id="ddw305-aff30" lang="en">  <sup>30</sup>Epidemiology Program, University of Hawaii Cancer Center, Honolulu, HI, USA</div><div id="ddw305-cor1"><sup>*</sup>To whom correspondence should be addressed at: Olufunmilayo I. Olopade, MBBS, Center for Clinical Genetics &amp; Innovation in Global Health, University of Chicago, Chicago, IL, USA. Email: <a href="mailto:dev@null" data-email="ude.ogacihcu.dsb.enicidem@edapolof" class="oemail">ude.ogacihcu.dsb.enicidem@edapolof</a>, Julie R. Palmer, ScD, Slone Epidemiology Center, Boston University, Boston MA, USA. Email: <a href="mailto:dev@null" data-email="ude.ub@remlapj" class="oemail">ude.ub@remlapj</a>, Christopher A. Haiman, PhD, Department of Preventive Medicine, Keck School of Medicine and Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, CA, USA. Email: <a href="mailto:dev@null" data-email="ude.csu.dem@namiaH.rehpotsirhC" class="oemail">ude.csu.dem@namiaH.rehpotsirhC</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140015762835168_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2016 Apr 1; Revised 2016 Jul 21; Accepted 2016 Aug 26.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140015762835168_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2016. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC5975608/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Data.</strong></div><div><a href="/pmc/articles/PMC5975608/bin/ddw305_supp.docx" data-ga-action="click_feat_suppl">ddw305_supp.docx</a><span style="color:gray"> (7.0M)</span></div><div class="small guid">GUID:&nbsp;2C7D40A6-BD43-47C2-9A8E-D74C39AC0E64</div></div></dd></dl></div><div id="idm140015758985232" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140015758985232title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Multiple breast cancer loci have been identified in previous genome-wide association studies, but they were mainly conducted in populations of European ancestry. Women of African ancestry are more likely to have young-onset and oestrogen receptor (ER) negative breast cancer for reasons that are unknown and understudied. To identify genetic risk factors for breast cancer in women of African descent, we conducted a meta-analysis of two genome-wide association studies of breast cancer; one study consists of 1,657 cases and 2,029 controls genotyped with Illumina’s HumanOmni2.5 BeadChip and the other study included 3,016 cases and 2,745 controls genotyped using Illumina Human1M-Duo BeadChip. The top 18,376 single nucleotide polymorphisms (SNP) from the meta-analysis were replicated in the third study that consists of 1,984 African Americans cases and 2,939 controls. We found that SNP rs13074711, 26.5 Kb upstream of <em>TNFSF10</em> at 3q26.21, was significantly associated with risk of oestrogen receptor (ER)-negative breast cancer (odds ratio [OR]=1.29, 95% CI: 1.18-1.40; <em>P </em>=<em> </em>1.8 × 10 <sup>−</sup> <sup>8</sup>). Functional annotations suggest that the <em>TNFSF10</em> gene may be involved in breast cancer aetiology, but further functional experiments are needed. In addition, we confirmed SNP rs10069690 was the best indicator for ER-negative breast cancer at 5p15.33 (OR = 1.30; <em>P </em>=<em> </em>2.4 × 10 <sup>−</sup> <sup>10</sup>) and identified rs12998806 as the best indicator for ER-positive breast cancer at 2q35 (OR = 1.34; <em>P </em>=<em> </em>2.2 × 10 <sup>−</sup> <sup>8</sup>) for women of African ancestry. These findings demonstrated additional susceptibility alleles for breast cancer can be revealed in diverse populations and have important public health implications in building race/ethnicity-specific risk prediction model for breast cancer.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p2" class="p p-first-last">As the most common cancer among women both in the United States and in the world (<a href="#ddw305-B1" rid="ddw305-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384109">1</a>), breast cancer is a heterogeneous disease with several molecular subtypes defined by biomarkers such as oestrogen receptor (ER). Oestrogen receptor-negative breast cancer is more likely to be early-onset and is associated with worse clinical outcomes. African Americans are more likely to have ER-negative breast cancer, for reasons that remain unknown and understudied (<a href="#ddw305-B2" rid="ddw305-B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384104">2</a>). Indigenous Africans have an even higher proportion of ER-negative breast cancer than African Americans (<a href="#ddw305-B3" rid="ddw305-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384075">3</a>,<a href="#ddw305-B4" rid="ddw305-B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384074">4</a>), suggesting that previously unrecognized genetic factors play an important role in this racial difference. Multiple genome-wide association studies (GWAS) of breast cancer have been conducted but mainly in populations of European ancestry, which have revealed more than 90 common susceptibility loci (<a href="#ddw305-B5" rid="ddw305-B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384106">5–21</a>). However, only a few of these loci appear to be specific for ER-negative disease (<a href="#ddw305-B19" rid="ddw305-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384098">19</a>,<a href="#ddw305-B21" rid="ddw305-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384097">21</a>). Because of its aggressive nature, there is an unmet clinical need for predictive genetic markers of susceptibility to ER-negative breast cancer, especially among women of African ancestry.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Results</h2><p id="__p3" class="p p-first">The study consists of a discovery stage and a validation stage. In the discovery stage, we conducted a meta-analysis of two GWAS (<a href="/pmc/articles/PMC5975608/table/ddw305-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw305-T1" rid-ob="ob-ddw305-T1" co-legend-rid=""><span>Table 1</span></a>). One study (GWAS of Breast Cancer in the African Diaspora, ROOT consortium) consists of 1,657 cases and 2,029 controls, while the other study (African American Breast Cancer Consortium, AABC) consists of 3,016 cases and 2,745 controls. Genotyping was conducted using the Illumina HumanOmni2.5-8v1 and Illumina Human1M-Duo BeadChip in ROOT and AABC studies, respectively. To combine genotyping results from the two studies, we imputed genotypes for single nucleotide polymorphisms in the 1000 Genomes Project using IMPUTE2 (<a href="#ddw305-B22" rid="ddw305-B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384101">22</a>). Quantile-quantile plots of P-values showed little inflation of test statistics in ROOT, AABC, or in the meta-analysis (all genomic control inflation factor λ’s &lt; 1.04; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Fig. S1</a>), indicating that there was no residual cryptic population substructure in the analysis. Results of the meta-analysis are summarized in the Manhattan plots (<a href="/pmc/articles/PMC5975608/figure/ddw305-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw305-F1" rid-ob="ob-ddw305-F1" co-legend-rid="lgnd_ddw305-F1"><span>Fig. 1</span></a>). The top 20,000 SNPs from the meta-analysis of overall breast cancer or ER-specific disease were considered for stage 2 replication genotyping in an independent sample of 1,984 invasive cases and 2,939 controls assembled by the African American Breast Cancer Epidemiology and Risk (AMBER) consortium (<a href="/pmc/articles/PMC5975608/table/ddw305-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw305-T1" rid-ob="ob-ddw305-T1" co-legend-rid=""><span>Table 1</span></a>). Of the ∼20,000 SNPs, 18,376 SNPs passed the design stage for the Illumina custom chip and were successfully genotyped.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddw305-F1" co-legend-rid="lgnd_ddw305-F1"><a href="/pmc/articles/PMC5975608/figure/ddw305-F1/" target="figure" rid-figpopup="ddw305-F1" rid-ob="ob-ddw305-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140015758930176" class="figure"><a href="/pmc/articles/PMC5975608/figure/ddw305-F1/" target="figure" rid-figpopup="ddw305-F1" rid-ob="ob-ddw305-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5975608_ddw305f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddw305f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5975608/bin/ddw305f1.jpg"></a></div><div id="largeobj_idm140015758930176" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5975608/figure/ddw305-F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddw305-F1"><div><a class="figpopup" href="/pmc/articles/PMC5975608/figure/ddw305-F1/" target="figure" rid-figpopup="ddw305-F1" rid-ob="ob-ddw305-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p4">Manhattan plots of –log10 of <em>P-</em>values in the meta-analysis of ROOT and AABC for overall, ER+ and ER- breast cancer risk.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw305-T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p5">Characteristics of study participants in the GWAS of breast cancer for women of African ancestry</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140015758927488" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="2" rowspan="1"><strong>No. of subjects</strong><hr></th><th colspan="2" rowspan="1"><strong>Mean (SD) of age in years</strong><hr></th><th colspan="2" rowspan="1"><strong>Oestrogen receptor (n, %)</strong><hr></th></tr><tr><th rowspan="1" colspan="1">Study</th><th rowspan="1" colspan="1">Study Name</th><th rowspan="1" colspan="1">Case</th><th rowspan="1" colspan="1">Control</th><th rowspan="1" colspan="1">Case</th><th rowspan="1" colspan="1">Control</th><th rowspan="1" colspan="1">Positive</th><th rowspan="1" colspan="1">Negative</th></tr></thead><tbody><tr><td colspan="8" rowspan="1"><strong><em>ROOT consortium</em></strong></td></tr><tr><td rowspan="1" colspan="1">NBCS</td><td rowspan="1" colspan="1">Nigerian Breast Cancer Study</td><td rowspan="1" colspan="1">711</td><td rowspan="1" colspan="1">624</td><td rowspan="1" colspan="1">48 (12)</td><td rowspan="1" colspan="1">45 (12)</td><td rowspan="1" colspan="1">99 (70)</td><td rowspan="1" colspan="1">42 (30)</td></tr><tr><td rowspan="1" colspan="1">BNCS</td><td rowspan="1" colspan="1">Barbados National Cancer Study</td><td rowspan="1" colspan="1">92</td><td rowspan="1" colspan="1">229</td><td rowspan="1" colspan="1">57 (15)</td><td rowspan="1" colspan="1">55 (13)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">RVGBC</td><td rowspan="1" colspan="1">Racial Variability in Genotypic Determinants of Breast Cancer Risk Study</td><td rowspan="1" colspan="1">145</td><td rowspan="1" colspan="1">257</td><td rowspan="1" colspan="1">46 (11)</td><td rowspan="1" colspan="1">41 (12)</td><td rowspan="1" colspan="1">25 (48)</td><td rowspan="1" colspan="1">27 (52)</td></tr><tr><td rowspan="1" colspan="1">CCPS</td><td rowspan="1" colspan="1">Chicago Cancer Prone Study</td><td rowspan="1" colspan="1">394</td><td rowspan="1" colspan="1">387</td><td rowspan="1" colspan="1">47 (12)</td><td rowspan="1" colspan="1">45 (11)</td><td rowspan="1" colspan="1">140 (45)</td><td rowspan="1" colspan="1">171 (55)</td></tr><tr><td rowspan="1" colspan="1">BBCS</td><td rowspan="1" colspan="1">Baltimore Breast Cancer Study</td><td rowspan="1" colspan="1">95</td><td rowspan="1" colspan="1">102</td><td rowspan="1" colspan="1">54 (14)</td><td rowspan="1" colspan="1">52 (13)</td><td rowspan="1" colspan="1">44 (49)</td><td rowspan="1" colspan="1">45 (51)</td></tr><tr><td rowspan="1" colspan="1">SCCS</td><td rowspan="1" colspan="1">Southern Community Cohort Study</td><td rowspan="1" colspan="1">220</td><td rowspan="1" colspan="1">430</td><td rowspan="1" colspan="1">57 (9)</td><td rowspan="1" colspan="1">57 (9)</td><td rowspan="1" colspan="1">66 (36)</td><td rowspan="1" colspan="1">118 (64)</td></tr><tr><td rowspan="1" colspan="1">subtotal</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><strong>1657</strong></td><td rowspan="1" colspan="1"><strong>2029</strong></td><td rowspan="1" colspan="1"><strong>49 (12)</strong></td><td rowspan="1" colspan="1"><strong>48 (13)</strong></td><td rowspan="1" colspan="1"><strong>374 (48)</strong></td><td rowspan="1" colspan="1"><strong>403 (52)</strong></td></tr><tr><td colspan="8" rowspan="1"><strong><em>AABC consortium</em></strong></td></tr><tr><td rowspan="1" colspan="1">MEC</td><td rowspan="1" colspan="1">Multiethnic Cohort Study</td><td rowspan="1" colspan="1">694</td><td rowspan="1" colspan="1">990</td><td rowspan="1" colspan="1">66 (9)</td><td rowspan="1" colspan="1">67 (9)</td><td rowspan="1" colspan="1">408 (70)</td><td rowspan="1" colspan="1">176 (30)</td></tr><tr><td rowspan="1" colspan="1">CARE</td><td rowspan="1" colspan="1">Women’s Contraceptive and Reproductive Experiences Study, Log Angeles component</td><td rowspan="1" colspan="1">357</td><td rowspan="1" colspan="1">215</td><td rowspan="1" colspan="1">49 (8)</td><td rowspan="1" colspan="1">48 (8)</td><td rowspan="1" colspan="1">183 (58)</td><td rowspan="1" colspan="1">130 (42)</td></tr><tr><td rowspan="1" colspan="1">WCHS</td><td rowspan="1" colspan="1">Women’s Circle of Health Study</td><td rowspan="1" colspan="1">261</td><td rowspan="1" colspan="1">239</td><td rowspan="1" colspan="1">50 (10)</td><td rowspan="1" colspan="1">50 (9)</td><td rowspan="1" colspan="1">131 (62)</td><td rowspan="1" colspan="1">80 (38)</td></tr><tr><td rowspan="1" colspan="1">SFBCS</td><td rowspan="1" colspan="1">San Francisco Bay Area Breast Cancer Study</td><td rowspan="1" colspan="1">165</td><td rowspan="1" colspan="1">220</td><td rowspan="1" colspan="1">56 (12)</td><td rowspan="1" colspan="1">55 (12)</td><td rowspan="1" colspan="1">84 (63)</td><td rowspan="1" colspan="1">50 (37)</td></tr><tr><td rowspan="1" colspan="1">NC-BCFR</td><td rowspan="1" colspan="1">Northern California site of the Breast Cancer Family Registry</td><td rowspan="1" colspan="1">424</td><td rowspan="1" colspan="1">50</td><td rowspan="1" colspan="1">50 (9)</td><td rowspan="1" colspan="1">49 (9)</td><td rowspan="1" colspan="1">219 (64)</td><td rowspan="1" colspan="1">121 (36)</td></tr><tr><td rowspan="1" colspan="1">CBCS</td><td rowspan="1" colspan="1">Carolina Breast Cancer Study</td><td rowspan="1" colspan="1">635</td><td rowspan="1" colspan="1">589</td><td rowspan="1" colspan="1">51 (12)</td><td rowspan="1" colspan="1">52 (11)</td><td rowspan="1" colspan="1">272 (46)</td><td rowspan="1" colspan="1">317 (54)</td></tr><tr><td rowspan="1" colspan="1">PLCO</td><td rowspan="1" colspan="1">Prostate, Lungs, Colorectal and Ovarian Cancer Screening Trial Cohort</td><td rowspan="1" colspan="1">56</td><td rowspan="1" colspan="1">116</td><td rowspan="1" colspan="1">68 (7)</td><td rowspan="1" colspan="1">68 (6)</td><td rowspan="1" colspan="1">14 (70)</td><td rowspan="1" colspan="1">6 (30)</td></tr><tr><td rowspan="1" colspan="1">NBHS</td><td rowspan="1" colspan="1">Nashville Breast Health Study</td><td rowspan="1" colspan="1">304</td><td rowspan="1" colspan="1">182</td><td rowspan="1" colspan="1">54 (11)</td><td rowspan="1" colspan="1">52 (10)</td><td rowspan="1" colspan="1">143 (69)</td><td rowspan="1" colspan="1">65 (31)</td></tr><tr><td rowspan="1" colspan="1">WFBC</td><td rowspan="1" colspan="1">Wake Forest University Breast Cancer Study</td><td rowspan="1" colspan="1">120</td><td rowspan="1" colspan="1">144</td><td rowspan="1" colspan="1">55 (12)</td><td rowspan="1" colspan="1">55 (11)</td><td rowspan="1" colspan="1">66 (61)</td><td rowspan="1" colspan="1">43 (39)</td></tr><tr><td rowspan="1" colspan="1">subtotal</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><strong>3016</strong></td><td rowspan="1" colspan="1"><strong>2745</strong></td><td rowspan="1" colspan="1"><strong>55 (12)</strong></td><td rowspan="1" colspan="1"><strong>58 (13)</strong></td><td rowspan="1" colspan="1"><strong>1520 (61)</strong></td><td rowspan="1" colspan="1"><strong>988 (39)</strong></td></tr><tr><td colspan="8" rowspan="1"><strong><em>AMBER consortium</em></strong></td></tr><tr><td rowspan="1" colspan="1">BWHS</td><td rowspan="1" colspan="1">Black Women's Health Study</td><td rowspan="1" colspan="1">752</td><td rowspan="1" colspan="1">2249</td><td rowspan="1" colspan="1">54 (10)</td><td rowspan="1" colspan="1">53 (11)</td><td rowspan="1" colspan="1">412(65)</td><td rowspan="1" colspan="1">218(35)</td></tr><tr><td rowspan="1" colspan="1">CBCS</td><td rowspan="1" colspan="1">Carolina Breast Cancer Study</td><td rowspan="1" colspan="1">727</td><td rowspan="1" colspan="1">61</td><td rowspan="1" colspan="1">51 (11)</td><td rowspan="1" colspan="1">53 (9)</td><td rowspan="1" colspan="1">444(64)</td><td rowspan="1" colspan="1">252(36)</td></tr><tr><td rowspan="1" colspan="1">WCHS</td><td rowspan="1" colspan="1">Women’s Circle of Health Study</td><td rowspan="1" colspan="1">505</td><td rowspan="1" colspan="1">629</td><td rowspan="1" colspan="1">54 (11)</td><td rowspan="1" colspan="1">51 (10)</td><td rowspan="1" colspan="1">237(70)</td><td rowspan="1" colspan="1">103(30)</td></tr><tr><td rowspan="1" colspan="1">subtotal</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><strong>1984</strong></td><td rowspan="1" colspan="1"><strong>2939</strong></td><td rowspan="1" colspan="1"><strong>53 (11)</strong></td><td rowspan="1" colspan="1"><strong>53 (10)</strong></td><td rowspan="1" colspan="1"><strong>1093 (66)</strong></td><td rowspan="1" colspan="1"><strong>573 (34)</strong></td></tr><tr><td rowspan="1" colspan="1">Total</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><strong>6657</strong></td><td rowspan="1" colspan="1"><strong>7713</strong></td><td rowspan="1" colspan="1"><strong>53 (12)</strong></td><td rowspan="1" colspan="1"><strong>53 (13)</strong></td><td rowspan="1" colspan="1"><strong>2987 (60)</strong></td><td rowspan="1" colspan="1"><strong>1964 (40)</strong></td></tr></tbody></table></div><div id="largeobj_idm140015758927488" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5975608/table/ddw305-T1/?report=objectonly">Open in a separate window</a></div></div><p id="__p6">In the combined analysis of GWAS discovery and validation stages, associations of SNPs in three regions attained genome-wide significance (<a href="/pmc/articles/PMC5975608/table/ddw305-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw305-T2" rid-ob="ob-ddw305-T2" co-legend-rid=""><span>Table 2</span></a>), with one of these regions (3q26.21) not identified in previous GWAS. All three SNPs were significantly associated with overall risk of breast cancer, but the strength of the association varied by breast cancer subtype. SNP rs12998806 at 2q35 was associated with ER-positive breast cancer (G-allele, OR = 1.34; <em>P </em>=<em> </em>2.2 × 10 <sup>−</sup> <sup>8</sup>), but not with ER-negative disease (OR = 0.99). SNP rs13074711 at 3q26.21 was significantly associated with ER-negative breast cancer (T-allele, OR = 1.29; <em>P </em>=<em> </em>1.8 × 10 <sup>−</sup> <sup>8</sup>), but only weakly with ER-positive breast cancer (OR = 1.10, <em>P </em>=<em> </em>0.0094). Similarly, rs10069690 at 5p15.33 was associated with ER-negative breast cancer (T-allele, OR = 1.30; <em>P </em>=<em> </em>2.4 × 10 <sup>−</sup> <sup>10</sup>), but only weakly associated with ER-positive disease (OR = 1.08, <em>P </em>=<em> </em>0.03). SNP rs12998806 was imputed in ROOT and AABC (imputation information score <em>r</em><sup>2 </sup>&gt;<sup> </sup>0.99) and rs13074711 was imputed in ROOT (<em>r</em><sup>2 </sup>=<sup> </sup>0.987). We genotyped these two SNPs in 96 samples from ROOT using sequencing technique, and found that the genotyping results were 100% and 99% in concordance with imputation for rs12998806 and rs13074711, respectively, suggesting that the results based on imputed SNPs were reliable.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw305-T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p7">Association between three loci and breast cancer risk in African ancestry, by type of breast cancer</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140015758752624" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1">Cytoband, gene</th><th colspan="2" rowspan="1"></th><th rowspan="2" colspan="1">Alleles (reference/ test)</th><th rowspan="1" colspan="1"></th><th rowspan="2" colspan="1">Imputation score</th><th rowspan="1" colspan="1"></th><th colspan="2" rowspan="1">Overall breast cancer<hr></th><th colspan="2" rowspan="1">ER-positive breast cancer<hr></th><th colspan="2" rowspan="1">ER-negative breast cancer<hr></th></tr><tr><th rowspan="1" colspan="1">SNP</th><th rowspan="1" colspan="1">Position<sup>*</sup></th><th rowspan="1" colspan="1">Study</th><th rowspan="1" colspan="1">RAF</th><th rowspan="1" colspan="1">OR (95% CI)</th><th rowspan="1" colspan="1">P value</th><th rowspan="1" colspan="1">OR (95% CI)</th><th rowspan="1" colspan="1">P value</th><th rowspan="1" colspan="1">OR (95% CI)</th><th rowspan="1" colspan="1">P value</th></tr></thead><tbody><tr><td rowspan="5" colspan="1">2q35</td><td rowspan="5" colspan="1">rs12998806</td><td rowspan="5" colspan="1">217893763</td><td rowspan="5" colspan="1">A/G</td><td rowspan="1" colspan="1">ROOT</td><td rowspan="1" colspan="1">0.995</td><td rowspan="1" colspan="1">0.87</td><td rowspan="1" colspan="1">1.09 (0.95-1.26)</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">1.21 (0.95-1.54)</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.88 (0.70-1.12)</td><td rowspan="1" colspan="1">0.30</td></tr><tr><td rowspan="1" colspan="1">AABC</td><td rowspan="1" colspan="1">0.992</td><td rowspan="1" colspan="1">0.86</td><td rowspan="1" colspan="1">1.20 (1.08-1.34)</td><td rowspan="1" colspan="1">0.0011</td><td rowspan="1" colspan="1">1.45 (1.26-1.67)</td><td rowspan="1" colspan="1">3.0E-07</td><td rowspan="1" colspan="1">1.00 (0.86-1.16)</td><td rowspan="1" colspan="1">0.99</td></tr><tr><td rowspan="1" colspan="1">AMBER</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1">0.86</td><td rowspan="1" colspan="1">1.14 (0.99-1.31)</td><td rowspan="1" colspan="1">0.076</td><td rowspan="1" colspan="1">1.24 (1.03-1.49)</td><td rowspan="1" colspan="1">0.022</td><td rowspan="1" colspan="1">1.07 (0.85-1.34)</td><td rowspan="1" colspan="1">0.56</td></tr><tr><td rowspan="1" colspan="1">Combined</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.86</td><td rowspan="1" colspan="1">1.15 (1.07-1.24)</td><td rowspan="1" colspan="1">0.00017</td><td rowspan="1" colspan="1"><strong>1.34 (1.21-1.48)</strong></td><td rowspan="1" colspan="1"><strong>2.2E-08</strong></td><td rowspan="1" colspan="1">0.99 (0.88-1.10)</td><td rowspan="1" colspan="1">0.84</td></tr><tr><td colspan="2" rowspan="1"><em>P for heterogeneity</em></td><td colspan="3" rowspan="1"><em>0.57</em></td><td colspan="2" rowspan="1"><em>0.27</em></td><td colspan="2" rowspan="1"><em>0.51</em></td></tr><tr><td rowspan="5" colspan="1">3q26.21 <em>TNFSF10</em></td><td rowspan="5" colspan="1">rs13074711</td><td rowspan="5" colspan="1">172267803</td><td rowspan="5" colspan="1">C/T</td><td rowspan="1" colspan="1">ROOT</td><td rowspan="1" colspan="1">0.987</td><td rowspan="1" colspan="1">0.67</td><td rowspan="1" colspan="1">1.03 (0.93-1.14)</td><td rowspan="1" colspan="1">0.53</td><td rowspan="1" colspan="1">0.86 (0.73-1.02)</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">1.11 (0.93-1.33)</td><td rowspan="1" colspan="1">0.24</td></tr><tr><td rowspan="1" colspan="1">AABC</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1">0.70</td><td rowspan="1" colspan="1">1.23 (1.13-1.34)</td><td rowspan="1" colspan="1">1.9E-06</td><td rowspan="1" colspan="1">1.18 (1.06-1.31)</td><td rowspan="1" colspan="1">0.0017</td><td rowspan="1" colspan="1">1.37 (1.21-1.55)</td><td rowspan="1" colspan="1">4.1E-07</td></tr><tr><td rowspan="1" colspan="1">AMBER</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1">0.69</td><td rowspan="1" colspan="1">1.15 (1.03-1.28)</td><td rowspan="1" colspan="1">0.015</td><td rowspan="1" colspan="1">1.15 (1.00-1.32)</td><td rowspan="1" colspan="1">0.049</td><td rowspan="1" colspan="1">1.30 (1.09-1.56)</td><td rowspan="1" colspan="1">0.0038</td></tr><tr><td rowspan="1" colspan="1">Combined</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.69</td><td rowspan="1" colspan="1">1.14 (1.08-1.21)</td><td rowspan="1" colspan="1">2.1E-06</td><td rowspan="1" colspan="1">1.10 (1.02-1.19)</td><td rowspan="1" colspan="1">0.0094</td><td rowspan="1" colspan="1"><strong>1.29 (1.18-1.40)</strong></td><td rowspan="1" colspan="1"><strong>1.8E-08</strong></td></tr><tr><td colspan="2" rowspan="1"><em>P for heterogeneity</em></td><td colspan="3" rowspan="1"><em>0.04</em></td><td colspan="2" rowspan="1"><em>0.007</em></td><td colspan="2" rowspan="1"><em>0.16</em></td></tr><tr><td rowspan="5" colspan="1">5p15.33 <em>TERT</em></td><td rowspan="5" colspan="1">rs10069690</td><td rowspan="5" colspan="1">1279790</td><td rowspan="5" colspan="1">C/T</td><td rowspan="1" colspan="1">ROOT</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1">0.63</td><td rowspan="1" colspan="1">1.15 (1.04-1.26)</td><td rowspan="1" colspan="1">0.0049</td><td rowspan="1" colspan="1">1.11 (0.94-1.31)</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">1.27 (1.08-1.50)</td><td rowspan="1" colspan="1">0.0048</td></tr><tr><td rowspan="1" colspan="1">AABC</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1">0.59</td><td rowspan="1" colspan="1">1.13 (1.04-1.22)</td><td rowspan="1" colspan="1">0.0022</td><td rowspan="1" colspan="1">1.08 (0.98-1.19)</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">1.31 (1.17-1.47)</td><td rowspan="1" colspan="1">2.3E-06</td></tr><tr><td rowspan="1" colspan="1">AMBER</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1">0.60</td><td rowspan="1" colspan="1">1.11 (1.00-1.23)</td><td rowspan="1" colspan="1">0.049</td><td rowspan="1" colspan="1">1.06 (0.93-1.20)</td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">1.31 (1.11-1.54)</td><td rowspan="1" colspan="1">0.0016</td></tr><tr><td rowspan="1" colspan="1">Combined</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.60</td><td rowspan="1" colspan="1">1.13 (1.07-1.19)</td><td rowspan="1" colspan="1">4.8E-06</td><td rowspan="1" colspan="1">1.08 (1.01-1.16)</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1"><strong>1.30 (1.20-1.41)</strong></td><td rowspan="1" colspan="1"><strong>2.4E-10</strong></td></tr><tr><td colspan="2" rowspan="1"><em>P for heterogeneity</em></td><td colspan="3" rowspan="1"><em>0.88</em></td><td colspan="2" rowspan="1"><em>0.91</em></td><td colspan="2" rowspan="1"><em>0.94</em></td></tr></tbody></table></div><div id="largeobj_idm140015758752624" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5975608/table/ddw305-T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn1"><sup>*</sup>NCBI Build 37 position.</div><div id="tblfn2"><p id="__p9" class="p p-first-last">Abbreviation: ER, oestrogen receptor; OR, allelic odds ratio; CI, confidence interval; RAF, risk (test) allele frequency.</p></div></div></div><p id="__p10">The rs13074711 SNP is located 26.5 Kb upstream of the Tumour Necrosis Factor Superfamily, Member 10 (<em>TNFSF10)</em> gene and the association signal for ER-negative breast cancer maps to a linkage disequilibrium (LD) block of ∼100 Kb between 172.2 Mb and 172.3 Mb on chromosome 3, which covers the <em>TNFSF10</em> gene (<a href="/pmc/articles/PMC5975608/figure/ddw305-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw305-F2" rid-ob="ob-ddw305-F2" co-legend-rid="lgnd_ddw305-F2"><span>Fig. 2A</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Fig S2</a>). The <em>TNFSF10</em> gene, also known as <em>TRAIL</em> or <em>APO2L</em>, encodes a cytokine that belongs to the tumour necrosis factor ligand family. TNFSF10 cytokine preferentially induces apoptosis in tumour cells or transformed cells, but is not toxic to normal cells (<a href="#ddw305-B23" rid="ddw305-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384079">23</a>). The <em>TNFSF10</em> gene is expressed in a wide range of tissues, including blood lymphocytes, breast, prostate, lungs, minor salivary grand and spleen (<a href="#ddw305-B24" rid="ddw305-B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384095">24</a>). Interestingly, triple-negative breast cancer cell lines have been shown to be very sensitive to TNFSF10-induced apoptosis, while ER-positive cell lines were resistant to TNFSF10-induced apoptosis (<a href="#ddw305-B25" rid="ddw305-B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384085">25</a>). Because of its important role in tumour apoptosis, several TNFSF10 receptor agonists or recombinant forms of TNFSF10 are in clinical development as molecularly targeted therapy for cancers (<a href="#ddw305-B26" rid="ddw305-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384096">26</a>). Thus, <em>TNFSF10</em> gene is a plausible causal gene in the region.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddw305-F2" co-legend-rid="lgnd_ddw305-F2"><a href="/pmc/articles/PMC5975608/figure/ddw305-F2/" target="figure" rid-figpopup="ddw305-F2" rid-ob="ob-ddw305-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140015758599232" class="figure"><a href="/pmc/articles/PMC5975608/figure/ddw305-F2/" target="figure" rid-figpopup="ddw305-F2" rid-ob="ob-ddw305-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5975608_ddw305f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddw305f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5975608/bin/ddw305f2.jpg"></a></div><div id="largeobj_idm140015758599232" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5975608/figure/ddw305-F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddw305-F2"><div><a class="figpopup" href="/pmc/articles/PMC5975608/figure/ddw305-F2/" target="figure" rid-figpopup="ddw305-F2" rid-ob="ob-ddw305-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p11">The 3q26.21/<em>TNFSF10</em> locus tagged by rs13074711. <strong>(A)</strong> The regional Manhattan plot shows the –log10 P values for testing SNPs in 250kb region with ER- breast cancer risk in the meta-analysis of the ROOT, AABC and AMBER studies. The colours depict the strength of the correlation (<em>r</em><sup>2</sup>) between SNP rs13074711 and the SNPs tested in the region. The correlation is estimated using the African panel of the 1000 Genomes Project data. The blue line indicates the recombination rates in centimorgans per megabase. <strong>(B)</strong> Analysis of regulation enhancer with data from ENCODE through UCSC Genome Browser, including histone modification marks for H3K4Me1 and H3K27Ac of seven cell types, transcription factor binding sites and DNase hypersensitivity sites of human mammary epithelial cells (HMEC), breast cancer cells (MCF7, T-47D)<strong>.</strong> Chromosomal coordinates are in NCBI build 37.</p></div></div></div><p id="__p12">There appeared to be a second association signal in the 3q26.21/<em>TNFSF10</em> region: rs9833271 was associated with ER-negative breast cancer (A-allele, OR = 1.39, <em>P =</em> 3.7 × 10 <sup>−</sup> <sup>5</sup>) and it was in weak LD with the top signal SNP, rs13074711 (<em>r</em><sup>2 </sup>=<sup> </sup>0.06). After adjusting rs13074711, the association between rs9833271 and ER-negative breast cancer remained statistically significant (p = 0.002) though the adjusted odds ratio was smaller (<a href="/pmc/articles/PMC5975608/table/ddw305-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw305-T3" rid-ob="ob-ddw305-T3" co-legend-rid=""><span>Table 3</span></a>). Interestingly, the risk allele of rs9833271 exists only in African populations. Furthermore, a missense variant in exon 1 of <em>TNFSF10</em> (rs6763816, c.97G &gt; A, Val33Ile) has a frequency of 0.04 in African populations, but is monomorphic in other populations; the minor allele of this missense variant was associated with reduced risk of ER-negative breast cancer (OR = 0.75; <em>P </em>=<em> </em>0.0012). Other SNPs that were in moderate LD with rs13074711 were no longer statistically significant after adjusting for rs13074711 (<a href="/pmc/articles/PMC5975608/table/ddw305-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw305-T3" rid-ob="ob-ddw305-T3" co-legend-rid=""><span>Table 3</span></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw305-T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p13">Selected variants in 3q26/<em>TNFSF10</em> region: their association with oestrogen-receptor negative breast cancer, and their linkage disequilibrium with rs13074711 (the top SNP in the region)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140015758587424" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Variant</th><th rowspan="1" colspan="1">Position<sup>*</sup></th><th rowspan="1" colspan="1">Allele†</th><th rowspan="1" colspan="1"><em>r</em><sup>2</sup> in YRI</th><th rowspan="1" colspan="1"><em>r</em><sup>2</sup> in CEU</th><th rowspan="1" colspan="1">Study</th><th rowspan="1" colspan="1">RAF</th><th rowspan="1" colspan="1">OR (95% CI)</th><th rowspan="1" colspan="1">P value</th><th rowspan="1" colspan="1">AOR (95% CI)<sup>‡</sup></th><th rowspan="1" colspan="1">P value<sup>‡</sup></th></tr></thead><tbody><tr><td rowspan="4" colspan="1">rs6763816</td><td rowspan="1" colspan="1">Val33Ile</td><td rowspan="4" colspan="1">C/T</td><td rowspan="4" colspan="1">0.08</td><td rowspan="4" colspan="1">NA</td><td rowspan="1" colspan="1">ROOT</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.92 (0.66-1.28)</td><td rowspan="1" colspan="1">0.62</td><td rowspan="1" colspan="1">0.98 (0.68-1.39)</td><td rowspan="1" colspan="1">0.90</td></tr><tr><td rowspan="3" colspan="1">Exon 1</td><td rowspan="1" colspan="1">AABC</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.73 (0.56-0.94)</td><td rowspan="1" colspan="1">0.014</td><td rowspan="1" colspan="1">0.85 (0.65-1.11)</td><td rowspan="1" colspan="1">0.22</td></tr><tr><td rowspan="1" colspan="1">AMBER</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.60 (0.41-0.87)</td><td rowspan="1" colspan="1">0.0079</td><td rowspan="1" colspan="1">0.65 (0.44-0.98)</td><td rowspan="1" colspan="1">0.038</td></tr><tr><td rowspan="1" colspan="1">Combined</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1"><strong>0.75 (0.62-0.89)</strong></td><td rowspan="1" colspan="1"><strong>0.0012</strong></td><td rowspan="1" colspan="1"><strong>0.83 (0.69-1.01)</strong></td><td rowspan="1" colspan="1"><strong>0.058</strong></td></tr><tr><td rowspan="3" colspan="1">rs16845798</td><td rowspan="3" colspan="1">13767</td><td rowspan="3" colspan="1">G/A</td><td rowspan="3" colspan="1">0.44</td><td rowspan="3" colspan="1">0.75</td><td rowspan="1" colspan="1">ROOT</td><td rowspan="1" colspan="1">0.73</td><td rowspan="1" colspan="1">1.03 (0.85-1.24)</td><td rowspan="1" colspan="1">0.79</td><td rowspan="1" colspan="1">0.88 (0.67-1.16)</td><td rowspan="1" colspan="1">0.37</td></tr><tr><td rowspan="1" colspan="1">AABC</td><td rowspan="1" colspan="1">0.74</td><td rowspan="1" colspan="1">1.25 (1.10-1.42)</td><td rowspan="1" colspan="1">6.7E-04</td><td rowspan="1" colspan="1">0.95 (0.78-1.15)</td><td rowspan="1" colspan="1">0.57</td></tr><tr><td rowspan="1" colspan="1">Combined</td><td rowspan="1" colspan="1">0.74</td><td rowspan="1" colspan="1">1.17 (1.06-1.30)</td><td rowspan="1" colspan="1">0.0031</td><td rowspan="1" colspan="1">0.92 (0.79-1.08)</td><td rowspan="1" colspan="1">0.33</td></tr><tr><td rowspan="3" colspan="1">rs17601879</td><td rowspan="3" colspan="1">16833</td><td rowspan="3" colspan="1">C/T</td><td rowspan="3" colspan="1">0.25</td><td rowspan="3" colspan="1">0.32</td><td rowspan="1" colspan="1">ROOT</td><td rowspan="1" colspan="1">0.81</td><td rowspan="1" colspan="1">1.00 (0.81-1.22)</td><td rowspan="1" colspan="1">0.98</td><td rowspan="1" colspan="1">0.87 (0.67-1.14)</td><td rowspan="1" colspan="1">0.32</td></tr><tr><td rowspan="1" colspan="1">AABC</td><td rowspan="1" colspan="1">0.81</td><td rowspan="1" colspan="1">1.24 (1.08-1.43)</td><td rowspan="1" colspan="1">0.0025</td><td rowspan="1" colspan="1">0.97 (0.80-1.16)</td><td rowspan="1" colspan="1">0.71</td></tr><tr><td rowspan="1" colspan="1">Combined</td><td rowspan="1" colspan="1">0.81</td><td rowspan="1" colspan="1">1.16 (1.03-1.30)</td><td rowspan="1" colspan="1">0.014</td><td rowspan="1" colspan="1">0.93 (0.80-1.09)</td><td rowspan="1" colspan="1">0.38</td></tr><tr><td rowspan="3" colspan="1">rs7650827</td><td rowspan="3" colspan="1">18258</td><td rowspan="3" colspan="1">C/T</td><td rowspan="3" colspan="1">0.36</td><td rowspan="3" colspan="1">0.32</td><td rowspan="1" colspan="1">ROOT</td><td rowspan="1" colspan="1">0.78</td><td rowspan="1" colspan="1">1.04 (0.85-1.27)</td><td rowspan="1" colspan="1">0.69</td><td rowspan="1" colspan="1">0.92 (0.70-1.22)</td><td rowspan="1" colspan="1">0.58</td></tr><tr><td rowspan="1" colspan="1">AABC</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">1.27 (1.10-1.46)</td><td rowspan="1" colspan="1">7.9E-04</td><td rowspan="1" colspan="1">0.98 (0.81-1.19)</td><td rowspan="1" colspan="1">0.86</td></tr><tr><td rowspan="1" colspan="1">Combined</td><td rowspan="1" colspan="1">0.79</td><td rowspan="1" colspan="1">1.19 (1.06-1.34)</td><td rowspan="1" colspan="1">0.0029</td><td rowspan="1" colspan="1">0.96 (0.82-1.13)</td><td rowspan="1" colspan="1">0.65</td></tr><tr><td rowspan="3" colspan="1">rs9809402</td><td rowspan="3" colspan="1">22074</td><td rowspan="3" colspan="1">C/T</td><td rowspan="3" colspan="1">0.25</td><td rowspan="3" colspan="1">0.09</td><td rowspan="1" colspan="1">ROOT</td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">1.14 (0.96-1.35)</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">1.10 (0.92-1.33)</td><td rowspan="1" colspan="1">0.30</td></tr><tr><td rowspan="1" colspan="1">AABC</td><td rowspan="1" colspan="1">0.32</td><td rowspan="1" colspan="1">1.19 (1.06-1.34)</td><td rowspan="1" colspan="1">0.0031</td><td rowspan="1" colspan="1">1.05 (0.92-1.20)</td><td rowspan="1" colspan="1">0.43</td></tr><tr><td rowspan="1" colspan="1">Combined</td><td rowspan="1" colspan="1">0.31</td><td rowspan="1" colspan="1">1.17 (1.07-1.29)</td><td rowspan="1" colspan="1">0.0011</td><td rowspan="1" colspan="1">1.07 (0.96-1.19)</td><td rowspan="1" colspan="1">0.22</td></tr><tr><td rowspan="3" colspan="1">rs7627427</td><td rowspan="3" colspan="1">23005</td><td rowspan="3" colspan="1">G/C</td><td rowspan="3" colspan="1">0.25</td><td rowspan="3" colspan="1">0.09</td><td rowspan="1" colspan="1">ROOT</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">1.04 (0.87-1.25)</td><td rowspan="1" colspan="1">0.66</td><td rowspan="1" colspan="1">1.00 (0.83-1.21)</td><td rowspan="1" colspan="1">0.99</td></tr><tr><td rowspan="1" colspan="1">AABC</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">1.19 (1.05-1.35)</td><td rowspan="1" colspan="1">0.0053</td><td rowspan="1" colspan="1">1.07 (0.94-1.22)</td><td rowspan="1" colspan="1">0.29</td></tr><tr><td rowspan="1" colspan="1">Combined</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">1.14 (1.03-1.26)</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">1.05 (0.94-1.17)</td><td rowspan="1" colspan="1">0.38</td></tr><tr><td rowspan="3" colspan="1">rs9833915</td><td rowspan="3" colspan="1">24723</td><td rowspan="3" colspan="1">G/A</td><td rowspan="3" colspan="1">0.75</td><td rowspan="3" colspan="1">1.0</td><td rowspan="1" colspan="1">ROOT</td><td rowspan="1" colspan="1">0.60</td><td rowspan="1" colspan="1">1.06 (0.90-1.26)</td><td rowspan="1" colspan="1">0.47</td><td rowspan="1" colspan="1">0.93 (0.69-1.26)</td><td rowspan="1" colspan="1">0.64</td></tr><tr><td rowspan="1" colspan="1">AABC</td><td rowspan="1" colspan="1">0.61</td><td rowspan="1" colspan="1">1.26 (1.12-1.42)</td><td rowspan="1" colspan="1">8.1E-05</td><td rowspan="1" colspan="1">0.95 (0.77-1.15)</td><td rowspan="1" colspan="1">0.58</td></tr><tr><td rowspan="1" colspan="1">Combined</td><td rowspan="1" colspan="1">0.61</td><td rowspan="1" colspan="1">1.20 (1.09-1.32)</td><td rowspan="1" colspan="1">2.5E-04</td><td rowspan="1" colspan="1">0.94 (0.80-1.11)</td><td rowspan="1" colspan="1">0.47</td></tr><tr><td rowspan="4" colspan="1">rs13074711</td><td rowspan="4" colspan="1">26505</td><td rowspan="4" colspan="1">C/T</td><td rowspan="4" colspan="1">1.0</td><td rowspan="4" colspan="1">1.0</td><td rowspan="1" colspan="1">ROOT</td><td rowspan="1" colspan="1">0.67</td><td rowspan="1" colspan="1">1.11 (0.93-1.33)</td><td rowspan="1" colspan="1">0.24</td><td rowspan="4" colspan="1" align="center">NA</td><td rowspan="4" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">AABC</td><td rowspan="1" colspan="1">0.69</td><td rowspan="1" colspan="1">1.37 (1.21-1.55)</td><td rowspan="1" colspan="1">4.1E-07</td></tr><tr><td rowspan="1" colspan="1">AMBER</td><td rowspan="1" colspan="1">0.69</td><td rowspan="1" colspan="1">1.30 (1.09-1.56)</td><td rowspan="1" colspan="1">0.0038</td></tr><tr><td rowspan="1" colspan="1">Combined</td><td rowspan="1" colspan="1">0.69</td><td rowspan="1" colspan="1"><strong>1.29 (1.18-1.40)</strong></td><td rowspan="1" colspan="1"><strong>1.8E-08</strong></td></tr><tr><td rowspan="4" colspan="1">rs9833271</td><td rowspan="4" colspan="1">37244</td><td rowspan="4" colspan="1">G/A</td><td rowspan="4" colspan="1">0.06</td><td rowspan="4" colspan="1">NA</td><td rowspan="1" colspan="1">ROOT</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">1.63 (1.19-2.24)</td><td rowspan="1" colspan="1">0.0024</td><td rowspan="1" colspan="1">1.59 (1.15-2.18)</td><td rowspan="1" colspan="1">0.0046</td></tr><tr><td rowspan="1" colspan="1">AABC</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">1.32 (1.06-1.64)</td><td rowspan="1" colspan="1">0.012</td><td rowspan="1" colspan="1">1.18 (0.94-1.47)</td><td rowspan="1" colspan="1">0.15</td></tr><tr><td rowspan="1" colspan="1">AMBER</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">1.32 (0.95-1.82)</td><td rowspan="1" colspan="1">0.095</td><td rowspan="1" colspan="1">1.23 (0.89-1.71)</td><td rowspan="1" colspan="1">0.21</td></tr><tr><td rowspan="1" colspan="1">Combined</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1"><strong>1.39 (1.19-1.63)</strong></td><td rowspan="1" colspan="1"><strong>3.7E-05</strong></td><td rowspan="1" colspan="1"><strong>1.28 (1.09-1.50)</strong></td><td rowspan="1" colspan="1"><strong>0.002</strong></td></tr></tbody></table></div><div id="largeobj_idm140015758587424" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5975608/table/ddw305-T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn3"><sup>*</sup>base pair upstream of the transcription starting site of <em>TNFSF10</em>.</div><div id="tblfn4"><sup>†</sup>reference allele/test allele.</div><div id="tblfn5"><sup>‡</sup>conditional analysis adjusted for SNP rs13074711.</div><div id="tblfn6"><p id="__p17" class="p p-first-last">Abbreviation: NA, not applicable; OR, allelic odds ratio; CI, confidence interval; RAF, risk (test) allele frequency; YRI, Yoruba in Ibadan, Nigeria; CEU, Utah residents with Northern and Western European ancestry.</p></div></div></div><p id="__p18">The expression quantitative trait locus (eQTL) analysis showed that several significant SNPs in the 3q26.21/<em>TNFSF10</em> region were associated with the expression of <em>TNFSF10</em> in peripheral blood (<a href="#ddw305-B27" rid="ddw305-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384092">27</a>) or brain (<a href="#ddw305-B28" rid="ddw305-B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384087">28</a>), but not in breast tumour (<a href="/pmc/articles/PMC5975608/table/ddw305-T4/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw305-T4" rid-ob="ob-ddw305-T4" co-legend-rid=""><span>Table 4</span></a>, <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Fig. S3</a>). We also examined the possibility that the breast cancer-associated SNPs influence <em>TNFSF10</em> expression by serving as enhancer variants. The top associated SNP, rs13074711, and its LD-linked SNPs were located in regions of high histone H3 lysine 4 monomethylation (H3K4Me1) and histone H3 lysine 27 acetylation (H3K27Ac) features that are characteristic of enhancer elements (<a href="/pmc/articles/PMC5975608/figure/ddw305-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw305-F2" rid-ob="ob-ddw305-F2" co-legend-rid="lgnd_ddw305-F2"><span>Fig. 2B</span></a>). The second associated SNP, rs9833271, was also located in an enhancer element. Notably, strong enhancer features of these SNPs were found in human mammary epithelial cells (HMEC) and breast myoepithelial cells (MYO) (<a href="/pmc/articles/PMC5975608/table/ddw305-T4/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw305-T4" rid-ob="ob-ddw305-T4" co-legend-rid=""><span>Table 4</span></a><strong>)</strong>. The three SNPs close to the transcriptional start site of <em>TNFSF10</em> (rs6763816, rs16845798, and rs17601879) also resided in transcription factor (GATA3) binding sites in a ductal breast cancer cell line (T47D). The minor allele of rs13074711 could alter multiple motifs. Taken together, these lines of evidences suggest that the association between variants in the 3q26.21 region and breast cancer risk may be mediated by regulating <em>TNFSF10</em> gene expression through enhancer activities.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw305-T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p19">Correlation of selected variants with expression of <em>TNFSF10</em> (from eQTL analysis (<a href="#ddw305-B27" rid="ddw305-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384083">27</a>,<a href="#ddw305-B28" rid="ddw305-B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384110">28</a>)) and regulatory element annotation of selected variants in 3q26/<em>TNFSF10</em> region using HaploReg (<a href="#ddw305-B37" rid="ddw305-B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384082">37</a>)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140015758372816" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1">Variant</th><th rowspan="2" colspan="1"><strong>Position</strong><sup>*</sup></th><th rowspan="1" colspan="1">Alleles</th><th colspan="4" rowspan="1"><strong>Minor allele frequency</strong><hr></th><th rowspan="2" colspan="1">eQTL</th><th rowspan="2" colspan="1">Promoter histone marks<sup>†</sup></th><th rowspan="2" colspan="1">Enhancer histone marks</th><th rowspan="2" colspan="1">DNAse hypersensitivity</th><th rowspan="2" colspan="1">Proteins bound</th><th rowspan="2" colspan="1">Motifs changed by the variant</th></tr><tr><th rowspan="1" colspan="1">Ref/Alt</th><th rowspan="1" colspan="1">AFR</th><th rowspan="1" colspan="1">AMR</th><th rowspan="1" colspan="1">ASN</th><th rowspan="1" colspan="1">EUR</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs6763816</td><td rowspan="1" colspan="1">Exon 1, Val33Ile</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">Brain</td><td rowspan="1" colspan="1">24 tissues (HMEC.35, MYO, HMEC)</td><td rowspan="1" colspan="1">6 tissues (HMEC.35, MYO)</td><td rowspan="1" colspan="1">26 tissues (HMEC.35, HEMC)</td><td rowspan="1" colspan="1">7 bound proteins (ERALPHA_A and GATA3 in T-47D)</td><td rowspan="1" colspan="1">Irf, Pax-5, XBP-1</td></tr><tr><td rowspan="1" colspan="1">rs16845798</td><td rowspan="1" colspan="1">13767</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.34</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">Brain</td><td rowspan="1" colspan="1">3 tissues</td><td rowspan="1" colspan="1">7 tissues (HMEC.35, MYO, HMEC)</td><td rowspan="1" colspan="1">9 tissues (HEME.35, HMEC)</td><td rowspan="1" colspan="1">GATA1,GATA3 (T-47D)</td><td rowspan="1" colspan="1">Pou2f2, Pou5f1</td></tr><tr><td rowspan="1" colspan="1">rs17601879</td><td rowspan="1" colspan="1">16833</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">0.1</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">Blood, brain</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">Blood, GI and breast (HEMC.35, MYO)</td><td rowspan="1" colspan="1">Cervix (HELAS3)</td><td rowspan="1" colspan="1">FOXA1 (T-47D) GATA3 (T-47D)</td><td rowspan="1" colspan="1">SEF-1</td></tr><tr><td rowspan="1" colspan="1">rs7650827</td><td rowspan="1" colspan="1">18258</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">0.1</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">Blood, brain</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">10 tissues (HMEC.35, MYO, HMEC)</td><td rowspan="1" colspan="1">Skin, Cervix</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">HNF4, Mef2, Pax-2</td></tr><tr><td rowspan="1" colspan="1">rs9809402</td><td rowspan="1" colspan="1">22074</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">0.34</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.44</td><td rowspan="1" colspan="1">Blood, brain</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">(HMEC, MYO)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">6 altered motifs</td></tr><tr><td rowspan="1" colspan="1">rs7627427</td><td rowspan="1" colspan="1">23005</td><td rowspan="1" colspan="1">G/C</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.44</td><td rowspan="1" colspan="1">Blood, brain</td><td rowspan="1" colspan="1">4 tissues</td><td rowspan="1" colspan="1">12 tissues (HMEC.35, MYO, HMEC)</td><td rowspan="1" colspan="1">6 tissues (HMEC.35, HMEC)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs9833915</td><td rowspan="1" colspan="1">24723</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.45</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">Brain</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">3 tissues (HMEC.35, MYO)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">8 altered motifs</td></tr><tr><td rowspan="1" colspan="1">rs13074711</td><td rowspan="1" colspan="1">26505</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.39</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">Brain</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">7 tissues (HMEC.35, MYO, HMEC)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">29 altered motifs</td></tr><tr><td rowspan="1" colspan="1">rs9833271</td><td rowspan="1" colspan="1">37244</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">8 tissues (HMEC.35, MYO, HMEC)</td><td rowspan="1" colspan="1">4 tissues (HMEC)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">DMRT2</td></tr></tbody></table></div><div id="largeobj_idm140015758372816" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5975608/table/ddw305-T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn7"><sup>*</sup>base pair upstream of the transcription starting site of <em>TNFSF10</em>.</div><div id="tblfn8"><sup>†</sup>Normal mammary or breast cancer cell lines are indicated in parenthesis.</div><div id="tblfn9"><p id="__p22" class="p p-first-last">HMEC.35: breast variant human mammary epithelial cells; MYO: breast myoepithelial primary cells; HMEC: mammary epithelial primary cells; T-47D: mammary ductal carcinoma cells.</p></div></div></div><p id="__p23">The association signal around rs12998806 was confined to a region of ∼87 Kb between 217.855 Mb and 217.942 Mb on chromosome 2q35 (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Fig S4</a>). Variant rs13387042 in this region has been associated with ER-positive breast cancer in previous GWAS conducted mainly in populations of European ancestry (<a href="#ddw305-B9" rid="ddw305-B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384090">9</a>,<a href="#ddw305-B12" rid="ddw305-B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384107">12</a>). rs13387042 was associated with breast cancer risk in some studies among African Americans (<a href="#ddw305-B29" rid="ddw305-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384089">29</a>,<a href="#ddw305-B30" rid="ddw305-B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384081">30</a>), but not in other studies among women of African ancestry (<a href="#ddw305-B9" rid="ddw305-B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384100">9</a>,<a href="#ddw305-B31" rid="ddw305-B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384080">31</a>,<a href="#ddw305-B32" rid="ddw305-B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716915495">32</a>) or Asians (<a href="#ddw305-B33" rid="ddw305-B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384099">33</a>). In the current study, the A-allele of rs13387042 was significantly associated with ER-positive breast cancer (OR = 1.16; <em>P </em>=<em> </em>0.002), but the association was weaker than with rs12998806. The two SNPs are in the same LD block, separated by 12 Kb and correlated in both YRI (<em>r</em><sup>2 </sup>=<sup> </sup>0.63) and CEU (<em>r</em><sup>2 </sup>=<sup> </sup>0.31) populations in the 1000 Genomes Project. No gene was found in the LD block and the nearest known genes include <em>TNP1</em> (169 Kb), <em>IGFBP5</em> (334 Kb) and <em>IGFBP2</em> (365 Kb). One study proposed that rs4442975 (G-allele) regulation of <em>IGFBP5</em> gene expression is the underlying mechanism for 2q35 region’s effect on ER-positive breast cancer risk (<a href="#ddw305-B33" rid="ddw305-B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384111">33</a>), but rs4442975 was not associated with breast cancer risk in Asians (<a href="#ddw305-B33" rid="ddw305-B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384094">33</a>) and only weakly associated with breast cancer in our study (OR = 1.12, <em>P </em>=<em> </em>0.009). For the 2q35 locus, the LD pattern in African populations is weaker than in European populations, which may help to pinpoint the causal variants (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Fig. S5</a>).</p><p id="__p24" class="p p-last">The rs10069690 SNP is located in intron 4 of the <em>TERT</em> gene and the association signal was confined to a small region on chromosome 5p15 (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Fig. S6</a>). The LD pattern surrounding rs10069690 was weak in both African and European populations, and few SNPs were correlated with rs10069690 at <em>r</em><sup>2 </sup>&gt;<sup> </sup>0.5 (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Fig. S7</a>). The rs10069690 SNP was previously identified in a GWAS meta-analysis of ER-negative breast cancer which included data from the AABC consortium (<a href="#ddw305-B21" rid="ddw305-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384091">21</a>). The strength of association between rs10069690 and ER-negative breast cancer in ROOT and AMBER is very similar to that observed in AABC, suggesting that the T-allele of rs10069690 is the best indicator of ER-negative breast cancer at 5p15/<em>TERT</em> for women of African ancestry.</p></div><div id="__sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec3title">Discussion</h2><p id="__p25" class="p p-first">In the largest breast cancer GWAS conducted in women of African ancestry with more than 14,000 cases and controls, we have identified 3q26.31 locus as a novel risk region for ER-negative breast cancer. The second signal SNP and a missense variant in 3q26.31 locus only exists in African ancestry populations, which may explain why this locus had not been detected in previous GWAS of other populations. Functional annotation of the associated variants suggests that the nearby <em>TNFSF10</em> gene may be the causal gene underlying this association. We also found and confirmed breast cancer risk variants that are specific for African Americans at two known susceptibility regions. These findings showed that genome-wide scans of common and rare genetic variants carried out in diverse populations can reveal additional susceptibility alleles for breast cancer, and population-specific risk variants are important for building appropriate risk prediction models.</p><p id="__p26">Interestingly, the associations with the three breast cancer risk susceptibility loci depend on ER status, which reinforces the need for even larger discovery efforts for etiologically distinct breast cancer subtypes. The risk allele frequency for rs10069690 in 5p15/TERT locus is greater in African populations (0.62) than in European ancestry populations (0.27), and the risk allele of rs9833271 exists only in African populations. These may contribute to the higher proportion of ER-negative breast cancer in women of African ancestry. By contrast, the risk allele frequency for rs13074711 is smaller in African populations (0.61) than in European ancestry populations (0.89), and the risk allele frequency for rs12998806 is greater in African populations (0.78) than in women of European ancestry (0.66), suggesting that these two loci may not account for racial differences in subtype distribution but more definitive studies are warranted.</p><p id="__p27" class="p p-last">The aetiology of ER-negative breast cancer is largely unknown, and our study findings that the association for all three top loci varies by ER status provide important insight into distinct biological pathways for ER-positive and ER-negative breast cancers, which will benefit both breast cancer treatment and prevention. In particular, theses data can be used to improve risk prediction model for aggressive forms of breast cancer in diverse populations. We provide several lines of evidence that the association at 3q26.31 region may be mediated by regulation of <em>TNFSF10</em> gene expression, and <em>TNFSF10</em> is important for tumour apoptosis. Further functional experiments will be needed to confirm whether the <em>TNFSF10</em> gene is a susceptibility gene for ER-negative breast cancer and to fully understand how genetic variants affect risk of breast cancer in diverse populations and why the associations vary by breast cancer subtype.</p></div><div id="__sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec4title">Materials and Methods</h2><div id="__sec5" class="sec sec-first"><h3 id="__sec5title">Study populations</h3><p id="__p28" class="p p-first-last">All the studies included in the GWAS have been approved by their corresponding Institutional Review Boards. Informed consent has been obtained from the participants. The discovery stage (stage 1) of the study is a meta-analysis of genome-wide association studies in two breast cancer consortia among women of African ancestry. The ROOT (GWAS in Breast Cancer in the African Diaspora) consortium consists of six epidemiologic studies and the AABC (African American Breast Cancer) consortium consists of nine epidemiological studies (<a href="#ddw305-B34" rid="ddw305-B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384076">34</a>). The replication stage (stage 2) of the study is based on the AMBER (African American Breast Cancer Epidemiology and Risk) consortium, which consists of four epidemiologic studies (<a href="#ddw305-B35" rid="ddw305-B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384103">35</a>). Selected characteristics of study participants for each study are summarized in <a href="/pmc/articles/PMC5975608/table/ddw305-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw305-T1" rid-ob="ob-ddw305-T1" co-legend-rid=""><span>Table 1</span></a>. The study design for each study is described in the <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Materials</a>.</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Genome-wide genotyping and quality control</h3><p id="__p29" class="p p-first">Genotyping in the ROOT consortium was conducted using the Illumina HumanOmni2.5-8v1 array. Genotyping was attempted for a total of 3,909 study samples, of which 3,859 were successful. The 3,859 study samples were derived from 3,774 subjects, with 85 subjects with duplicate samples. After genotyping, quality control analysis was carried out by checking chromosomal anomalies, participant relativeness, population structure, missing call rates, batch effects, duplicate sample discordance, Mendelian errors, Hardy-Weinberg equilibrium, and duplicate SNP probes. First, duplicate samples from 85 pairs were excluded. Chromosomal anomalies &gt;5 Mb and anomalies on chromosomes where the sum of the anomaly lengths &gt;10 Mb were identified. No subjects were excluded for this reason but all genotypes were set to missing for regions with chromosome anomalies. The relatedness between each pair of participants was evaluated by the identical by descent (IBD) analysis. According to the IBD analysis, samples from 59 subjects were excluded because they were possibly relatives of other subjects. Additionally, 20 subjects (18 African Americans and 2 African Barbadian) were excluded because principal components analysis showed that they were clustered with the CEU/TSI HapMaps (i.e. high percentage of European ancestry). We further filtered out 7 samples with a missing call rate &gt; 2% and 2 samples with possible tumour contamination. After these exclusions, a total of 3,686 subjects (1,657 cases and 2,029 controls) were included for the final analysis.</p><p id="__p30">A total of 2,379,855 SNP probes were attempted in Illumina HumanOmni2.5-8v1 array. In the SNP level quality control analysis, 32,192 SNPs with technical errors, 119,924 SNPs with minor allele frequency = 0 and 5,631 duplicate SNPs were excluded. We further filtered out 99,511 SNPs with missing call rate  &gt; = 2%, 563 SNPs with &gt;1 discordant calls, 1,078 SNPs with &gt;1 Mendelian errors and 4,591 SNPs with Hardy-Weinberg equilibrium test <em>P</em> &lt; 10 <sup>−</sup> <sup>4</sup>. After these exclusions, 2,116,365 SNP remained.</p><p id="__p31">Genotyping in the AABC consortium was conducted using the IlluminaHuman1M-Duo BeadChip. Of the 5,984 samples available to the AABC consortium (3,153 cases and 2,831 controls), we attempted genotyping of 5,932, removing samples (<em>n =</em> 52) with DNA concentrations &lt;20 ng/ul. Following genotyping, we removed samples based on the following exclusion criteria: 1) unknown replicates (≥98.9% genetically identical) that we were able to confirm (only one of each duplicate was removed, <em>n =</em> 15); 2) unknown replicates that we were not able to confirm through discussions with study investigators (pair or triplicate removed, <em>n =</em> 14); 3) samples with call rates &lt;95% after a second attempt (<em>n =</em> 100); 4) samples with ≤ 5% African ancestry (<em>n =</em> 36); and, 5) samples with &lt;15% mean heterozygosity of SNPs in the X chromosome and/or similar mean allele intensities of SNPs on the X and Y chromosomes (<em>n =</em> 6) (these are likely to be males).</p><p id="__p32" class="p p-last">In the analysis, we removed SNPs with &lt;95% call rate (<em>n =</em> 21,732) or minor allele frequencies (MAFs) &lt;1% (<em>n =</em> 80,193). To assess genotyping reproducibility, we included 138 replicate samples; the average concordance rate was 99.95% (&gt;99.93% for all pairs). We also eliminated SNPs with genotyping concordance rates &lt;98% based on the replicates (<em>n =</em> 11,701). The final analysis dataset included 1,043,036 SNPs genotyped on 3,016 cases and 2,745 controls, with an average SNP call rate of 99.7% and average sample call rate of 99.8%. Hardy-Weinberg equilibrium (HWE) was not used as a criterion for removing SNPs at this stage.</p></div><div id="__sec7" class="sec"><h3 id="__sec7title">Imputation and technical validation by sequencing</h3><p id="__p33" class="p p-first">Genotype imputation for the ROOT consortium was conducted by the University of Washington Genetics Coordinating Center (GCC) using the <em>IMPUTE2</em> software (<a href="#ddw305-B23" rid="ddw305-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384102">23</a>). Following filters for genotype imputation, including no ambiguity in strand alignment and ability to be mapped to NCDB build 37, 2,018,833 SNPs were selected as the basis for imputation. Using the 1000 Genomes Project phase I integrated variant set as the reference panel (October 2011 release), 20,109,249 SNPs were imputed. Of these, 16,147,413 have passed the imputation quality filter (imputation score &gt;0.3) and were included along with the genotyped SNPs in the final association analysis. Genotype imputation in AABC was conducted using <em>IMPUTE2</em> software (<a href="#ddw305-B23" rid="ddw305-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384086">23</a>) to a cosmopolitan panel of all 1000 Genome Project subjects (March 2012 release).</p><p id="__p34" class="p p-last">To provide technical validation, we sequenced 96 samples that have been included in the final analysis of the ROOT GWAS. We conducted whole genome sequencing on the Illumina platform, with average depth of 30x. The concordance rate between whole genome sequencing and array-based genotyping was 99.5%.</p></div><div id="__sec8" class="sec"><h3 id="__sec8title">Genotyping in the replication stage</h3><p id="__p35" class="p p-first">Genotyping of the AMBER cases and controls was performed on the Illumina Human Exome Beadchip v1.1 with additional custom content at the Center for Inherited Disease Research (CIDR). The standard content of this array includes more than 240,000 coding variants, as well as ancestry informative markers (AIMs). Twenty thousand variants identified in the AABC/ROOT meta-analysis, including 2 or more variants per associated region for the top ∼5000 loci, were included as part of the custom content.</p><p id="__p36" class="p p-last">Of 405,555 SNPs attempted for genotyping, 381,212 were released by CIDR and 299,873 of these remained after removing SNPs that were monomorphic (<em>n =</em> 70,761), were positional duplicates, were on the Y chromosome, had Hardy-Weinberg Equilibrium (HWE) <em>P &lt;</em> 10 <sup>−</sup> <sup>4</sup>, had call rate &lt; 0.98, had &gt; 1 Mendelian errors in trios from HapMap (<a href="http://hapmap.ncbi.nlm.nih.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://hapmap.ncbi.nlm.nih.gov/</a>), or had &gt; 2 discordant calls in duplicate samples. IMPUTE2 was used to impute additional variants using the 1000 Genomes Phase I reference panel (5/21/2011 1000G data, December 2013 haplotype release in IMPUTE2 site). For the meta-analysis, 18,376 SNPs passed quality control and included in the study.</p></div><div id="__sec9" class="sec"><h3 id="__sec9title">Statistical analysis</h3><p id="__p37" class="p p-first">In the ROOT GWAS, we analysed genotyped SNPs and imputed SNPs with imputation score &gt;0.3 using SNPTEST software to account for uncertainty in imputation (<a href="#ddw305-B36" rid="ddw305-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716915494">36</a>). We examined the association of each SNP and breast cancer risk using unconditional logistic regression, adjusting for age, study site and the first four eigenvectors from principal component analysis. The first four eigenvectors were used to control for population stratification as only the first 4 eigenvectors were associated with case status. Odds ratios (OR) and 95% confidence intervals (CI) were calculated from the multivariable logistic regressions. All tests of statistical significance were two-sided. Using similar methods, we conducted an analysis to compare ER-positive breast cancers with controls and compare ER-negative breast cancers with controls, in order to identify subtype-specific SNPs.</p><p id="__p38">In the AABC GWAS, we analysed genotyped SNPs and imputed SNPs with imputation score &gt;0.3 and minor allele frequency &gt;0.01. Unconditional logistic regression was used to examine the association between each SNP and overall breast cancer risk, ER-positive and ER-negative breast cancer risk separately. Age, the first 10 eigenvectors from principal component analysis, and study site were adjusted for in the logistic regression. To be conservative, all the first 10 eigenvectors were used to control population stratification. The SFBCS and NC-BCFR studies were conducted in the same San Francisco Bay Area population and were combined as one study site in the analyses.</p><p id="__p39" class="p p-last">In the replication study, principal components were computed using the smartpca program in the EIGENSOFT package. No eigenvectors from the principal component analysis were strongly associated with case status after controlling for the matching factors, DNA source and study. ORs and 95% CIs for each SNP were estimated in the analyses adjusted for study, age, geographic region, DNA source and eigenvectors 5, 6 and 8, which were associated with cases status at <em>P &lt;</em> 0.10. Regression coefficients (log odds ratios) and their standard errors, from the two GWAS and the replication study were combined using a fixed effect meta-analysis method.</p></div><div id="__sec10" class="sec sec-last"><h3 id="__sec10title">Functional annotation</h3><p id="__p40" class="p p-first">For each top variant and region identified in this study, we used HaploReg (<a href="#ddw305-B37" rid="ddw305-B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384088">37</a>) and USCS Genome Browser to explore functional annotations of noncoding variants. Chromatin states (promoters and enhancers), conserved regions, variant effect on regulatory motifs, and protein binding sites were assessed from available data from the ENCODE (<a href="#ddw305-B38" rid="ddw305-B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384078">38</a>) and the Roadmap Epigenomics Consortium (<a href="#ddw305-B39" rid="ddw305-B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384073">39</a>). Data from breast tumour cell lines (MCF-7, T-47D) and normal mammary epithelial cells (HMEC, MYO, vMHEC) were emphasized.</p><p id="__p41" class="p p-last">Data on expression quantitative trait locus (eQTL) analysis in blood were obtained from Westra <em>et al</em>. (<a href="#ddw305-B27" rid="ddw305-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384084">27</a>), in which eQTLs were identified in a meta-analysis in 5,311 peripheral untransformed blood samples and replication analysis in another 2,775 samples (<a href="#ddw305-B27" rid="ddw305-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384077">27</a>). Data on eQTL analysis in brain were obtained from ∼400 autopsied subjects (<a href="#ddw305-B28" rid="ddw305-B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716384105">28</a>). Analysis on eQTL in breast tumours was conducted using 46 ER+ breast tumours and 45 ER- breast tumours from The Cancer Genome Atlas (TCGA). Only samples from African Americans were included. Copy number change and methylation status were controlled for in the eQTL analysis in tumours with MatrixEQTL package in Bioconductor.</p></div></div><div id="__sec11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec11title">Supplementary Material</h2><p id="__p42" class="p p-first">
<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material</a> is available at <em>HMG</em> online.</p><p id="__p43" class="p p-last">
<em>Conflict of Interest statement</em>. None declared. </p></div><div id="__sec12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec12title">Funding </h2><p id="__p44" class="p p-first-last">The ROOT Consortium was supported by National Cancer Institute grants R01-CA142996, P50-CA125183, R01-CA89085 and U01-CA161032. DH and OIO were also supported by American Cancer Society (MRSG-13-063-01-TBG and CRP-10-119-01-CCE). AABC was supported by a Department of Defense Breast Cancer Research Program Era of Hope Scholar Award to CAH [W81XWH-08-1-0383] and the Norris Foundation. Each of the participating AABC studies was supported by the following grants: MEC (National Institutes of Health grants R01-CA63464 and R37-CA54281); CARE (National Institute for Child Health and Development grant NO1-HD-3-3175, K05 CA136967); WCHS (U.S. Army Medical Research and Material Command (USAMRMC) grant DAMD-17-01-0-0334, the National Institutes of Health grant R01-CA100598, and the Breast Cancer Research Foundation; SFBCS (National Institutes of Health grant R01-CA077305 and United States Army Medical Research Program grant DAMD17-96-6071); NC-BCFR (National Institutes of Health grant U01-CA069417); CBCS (National Institutes of Health Specialized Program of Research Excellence in Breast Cancer, grant number P50-CA58223 and Center for Environmental Health and Susceptibility National Institute of Environmental Health Sciences, National Institutes of Health, grant number P30-ES10126); PLCO (Intramural Research Program, National Cancer Institute, National Institutes of Health); NBHS (National Institutes of Health grant R01-CA100374). The Breast Cancer Family Registry (BCFR) was supported by the National Cancer Institute, National Institutes of Health under RFA-CA-06-503 and through cooperative agreements with members of the Breast Cancer Family Registry and Principal Investigators. The content of this manuscript does not necessarily reflect the views or policies of the National Cancer Institute or any of the collaborating centres in the BCFR, nor does mention of trade names, commercial products, or organizations imply endorsement by the U.S. Government or the BCFR. AMBER was supported by the National Cancer Institute grants P01-CA151135, R01-CA098663, R01-CA058420, UM1-CA164974, R01-CA100598, UM1-CA164973, R01-CA54281, P50-CA58223, and the University Cancer Research Fund of North Carolina. Pathology data were obtained from numerous state cancer registries (Arizona, California, Colorado, Connecticut, Delaware, District of Columbia, Florida, Georgia, Hawaii, Illinois, Indiana, Kentucky, Louisiana, Maryland, Massachusetts, Michigan, New Jersey, New York, North Carolina, Oklahoma, Pennsylvania, South Carolina, Tennessee, Texas, Virginia). The results reported do not necessarily represent their views or the views of the National Institutes of Health.</p></div><div id="__sec13" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec13title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="sup1"><h4>Supplementary Data</h4><div class="sup-box half_rhythm" id="idm140015758159728"><a href="/pmc/articles/PMC5975608/bin/ddw305_supp.docx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(7.0M, docx)</sup></div></div></div><div id="idm140015758157824" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140015758157824title">References </h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="ddw305-B1">1. <span class="mixed-citation">
Torre L.A., Bray F., Siegel R.L., Ferlay J., Lortet-Tieulent J., Jemal A. (2015) 
<span class="ref-title">Global cancer statistics, 2012</span>. <span class="ref-journal">CA. Cancer J. Clin.</span>, <span class="ref-vol">65</span>, 87–108. [<a href="/pubmed/25651787" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CA.+Cancer+J.+Clin.&amp;title=Global+cancer+statistics,+2012&amp;author=L.A.+Torre&amp;author=F.+Bray&amp;author=R.L.+Siegel&amp;author=J.+Ferlay&amp;author=J.+Lortet-Tieulent&amp;volume=65&amp;publication_year=2015&amp;pages=87-108&amp;pmid=25651787&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B2">2. <span class="mixed-citation">
Carey L.A., Perou C.M., Livasy C.A., Dressler L.G., Cowan D., Conway K., Karaca G., Troester M.A., Tse C.K., Edmiston S., et al. (2006) 
<span class="ref-title">Race, breast cancer subtypes, and survival in the Carolina Breast Cancer Study</span>. <span class="ref-journal">JAMA</span>, <span class="ref-vol">295</span>, 2492–2502. [<a href="/pubmed/16757721" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Race,+breast+cancer+subtypes,+and+survival+in+the+Carolina+Breast+Cancer+Study&amp;author=L.A.+Carey&amp;author=C.M.+Perou&amp;author=C.A.+Livasy&amp;author=L.G.+Dressler&amp;author=D.+Cowan&amp;volume=295&amp;publication_year=2006&amp;pages=2492-2502&amp;pmid=16757721&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B3">3. <span class="mixed-citation">
Huo D., Ikpatt F., Khramtsov A., Dangou J.M., Nanda R., Dignam J., Zhang B., Grushko T., Zhang C., Oluwasola O., et al. (2009) 
<span class="ref-title">Population differences in breast cancer: survey in indigenous african women reveals over-representation of triple-negative breast cancer</span>. <span class="ref-journal">J. Clin. Oncol.</span>, <span class="ref-vol">27</span>, 4515–4521. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2754904/">PMC free article</a>]</span> [<a href="/pubmed/19704069" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Oncol.&amp;title=Population+differences+in+breast+cancer:+survey+in+indigenous+african+women+reveals+over-representation+of+triple-negative+breast+cancer&amp;author=D.+Huo&amp;author=F.+Ikpatt&amp;author=A.+Khramtsov&amp;author=J.M.+Dangou&amp;author=R.+Nanda&amp;volume=27&amp;publication_year=2009&amp;pages=4515-4521&amp;pmid=19704069&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B4">4. <span class="mixed-citation">
Eng A., McCormack V., dos-Santos-Silva I. (2014) 
<span class="ref-title">Receptor-defined subtypes of breast cancer in indigenous populations in Africa: a systematic review and meta-analysis</span>. <span class="ref-journal">PLoS Med.</span>, <span class="ref-vol">11</span>, e1001720.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4159229/">PMC free article</a>]</span> [<a href="/pubmed/25202974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Med.&amp;title=Receptor-defined+subtypes+of+breast+cancer+in+indigenous+populations+in+Africa:+a+systematic+review+and+meta-analysis&amp;author=A.+Eng&amp;author=V.+McCormack&amp;author=I.+dos-Santos-Silva&amp;volume=11&amp;publication_year=2014&amp;pages=e1001720.&amp;pmid=25202974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B5">5. <span class="mixed-citation">
Michailidou K., Hall P., Gonzalez-Neira A., Ghoussaini M., Dennis J., Milne R.L., Schmidt M.K., Chang-Claude J., Bojesen S.E., Bolla M.K., et al. (2013) 
<span class="ref-title">Large-scale genotyping identifies 41 new loci associated with breast cancer risk</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">45</span>, 353–361. 361e351-352. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3771688/">PMC free article</a>]</span> [<a href="/pubmed/23535729" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Large-scale+genotyping+identifies+41+new+loci+associated+with+breast+cancer+risk&amp;author=K.+Michailidou&amp;author=P.+Hall&amp;author=A.+Gonzalez-Neira&amp;author=M.+Ghoussaini&amp;author=J.+Dennis&amp;volume=45&amp;publication_year=2013&amp;pages=353-361&amp;pmid=23535729&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B6">6. <span class="mixed-citation">
Michailidou K., Beesley J., Lindstrom S., Canisius S., Dennis J., Lush M.J., Maranian M.J., Bolla M.K., Wang Q., Shah M., et al. (2015) 
<span class="ref-title">Genome-wide association analysis of more than 120,000 individuals identifies 15 new susceptibility loci for breast cancer</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">47</span>, 373–380. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4549775/">PMC free article</a>]</span> [<a href="/pubmed/25751625" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+analysis+of+more+than+120,000+individuals+identifies+15+new+susceptibility+loci+for+breast+cancer&amp;author=K.+Michailidou&amp;author=J.+Beesley&amp;author=S.+Lindstrom&amp;author=S.+Canisius&amp;author=J.+Dennis&amp;volume=47&amp;publication_year=2015&amp;pages=373-380&amp;pmid=25751625&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B7">7. <span class="mixed-citation">
Easton D.F., Pooley K.A., Dunning A.M., Pharoah P.D., Thompson D., Ballinger D.G., Struewing J.P., Morrison J., Field H., Luben R., et al. (2007) 
<span class="ref-title">Genome-wide association study identifies novel breast cancer susceptibility loci</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">447</span>, 1087–1093. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2714974/">PMC free article</a>]</span> [<a href="/pubmed/17529967" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+identifies+novel+breast+cancer+susceptibility+loci&amp;author=D.F.+Easton&amp;author=K.A.+Pooley&amp;author=A.M.+Dunning&amp;author=P.D.+Pharoah&amp;author=D.+Thompson&amp;volume=447&amp;publication_year=2007&amp;pages=1087-1093&amp;pmid=17529967&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B8">8. <span class="mixed-citation">
Hunter D.J., Kraft P., Jacobs K.B., Cox D.G., Yeager M., Hankinson S.E., Wacholder S., Wang Z., Welch R., Hutchinson A., et al. (2007) 
<span class="ref-title">A genome-wide association study identifies alleles in FGFR2 associated with risk of sporadic postmenopausal breast cancer</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">39</span>, 870–874. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3493132/">PMC free article</a>]</span> [<a href="/pubmed/17529973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+genome-wide+association+study+identifies+alleles+in+FGFR2+associated+with+risk+of+sporadic+postmenopausal+breast+cancer&amp;author=D.J.+Hunter&amp;author=P.+Kraft&amp;author=K.B.+Jacobs&amp;author=D.G.+Cox&amp;author=M.+Yeager&amp;volume=39&amp;publication_year=2007&amp;pages=870-874&amp;pmid=17529973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B9">9. <span class="mixed-citation">
Stacey S.N., Manolescu A., Sulem P., Rafnar T., Gudmundsson J., Gudjonsson S.A., Masson G., Jakobsdottir M., Thorlacius S., Helgason A., et al. (2007) 
<span class="ref-title">Common variants on chromosomes 2q35 and 16q12 confer susceptibility to estrogen receptor-positive breast cancer</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">39</span>, 865–869. [<a href="/pubmed/17529974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+on+chromosomes+2q35+and+16q12+confer+susceptibility+to+estrogen+receptor-positive+breast+cancer&amp;author=S.N.+Stacey&amp;author=A.+Manolescu&amp;author=P.+Sulem&amp;author=T.+Rafnar&amp;author=J.+Gudmundsson&amp;volume=39&amp;publication_year=2007&amp;pages=865-869&amp;pmid=17529974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B10">10. <span class="mixed-citation">
Stacey S.N., Manolescu A., Sulem P., Thorlacius S., Gudjonsson S.A., Jonsson G.F., Jakobsdottir M., Bergthorsson J.T., Gudmundsson J., Aben K.K., et al. (2008) 
<span class="ref-title">Common variants on chromosome 5p12 confer susceptibility to estrogen receptor-positive breast cancer</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">40</span>, 703–706. [<a href="/pubmed/18438407" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+on+chromosome+5p12+confer+susceptibility+to+estrogen+receptor-positive+breast+cancer&amp;author=S.N.+Stacey&amp;author=A.+Manolescu&amp;author=P.+Sulem&amp;author=S.+Thorlacius&amp;author=S.A.+Gudjonsson&amp;volume=40&amp;publication_year=2008&amp;pages=703-706&amp;pmid=18438407&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B11">11. <span class="mixed-citation">
Ahmed S., Thomas G., Ghoussaini M., Healey C.S., Humphreys M.K., Platte R., Morrison J., Maranian M., Pooley K.A., Luben R., et al. (2009) 
<span class="ref-title">Newly discovered breast cancer susceptibility loci on 3p24 and 17q23.2</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">41</span>, 585–590. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2748125/">PMC free article</a>]</span> [<a href="/pubmed/19330027" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Newly+discovered+breast+cancer+susceptibility+loci+on+3p24+and+17q23.2&amp;author=S.+Ahmed&amp;author=G.+Thomas&amp;author=M.+Ghoussaini&amp;author=C.S.+Healey&amp;author=M.K.+Humphreys&amp;volume=41&amp;publication_year=2009&amp;pages=585-590&amp;pmid=19330027&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B12">12. <span class="mixed-citation">
Zheng W., Long J., Gao Y.T., Li C., Zheng Y., Xiang Y.B., Wen W., Levy S., Deming S.L., Haines J.L., et al. (2009) 
<span class="ref-title">Genome-wide association study identifies a new breast cancer susceptibility locus at 6q25.1</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">41</span>, 324–328. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2754845/">PMC free article</a>]</span> [<a href="/pubmed/19219042" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+identifies+a+new+breast+cancer+susceptibility+locus+at+6q25.1&amp;author=W.+Zheng&amp;author=J.+Long&amp;author=Y.T.+Gao&amp;author=C.+Li&amp;author=Y.+Zheng&amp;volume=41&amp;publication_year=2009&amp;pages=324-328&amp;pmid=19219042&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B13">13. <span class="mixed-citation">
Thomas G., Jacobs K.B., Kraft P., Yeager M., Wacholder S., Cox D.G., Hankinson S.E., Hutchinson A., Wang Z., Yu K., et al. (2009) 
<span class="ref-title">A multistage genome-wide association study in breast cancer identifies two new risk alleles at 1p11.2 and 14q24.1 (RAD51L1)</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">41</span>, 579–584. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928646/">PMC free article</a>]</span> [<a href="/pubmed/19330030" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+multistage+genome-wide+association+study+in+breast+cancer+identifies+two+new+risk+alleles+at+1p11.2+and+14q24.1+(RAD51L1)&amp;author=G.+Thomas&amp;author=K.B.+Jacobs&amp;author=P.+Kraft&amp;author=M.+Yeager&amp;author=S.+Wacholder&amp;volume=41&amp;publication_year=2009&amp;pages=579-584&amp;pmid=19330030&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B14">14. <span class="mixed-citation">
Turnbull C., Ahmed S., Morrison J., Pernet D., Renwick A., Maranian M., Seal S., Ghoussaini M., Hines S., Healey C.S., et al. (2010) 
<span class="ref-title">Genome-wide association study identifies five new breast cancer susceptibility loci</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">42</span>, 504–507. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3632836/">PMC free article</a>]</span> [<a href="/pubmed/20453838" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+identifies+five+new+breast+cancer+susceptibility+loci&amp;author=C.+Turnbull&amp;author=S.+Ahmed&amp;author=J.+Morrison&amp;author=D.+Pernet&amp;author=A.+Renwick&amp;volume=42&amp;publication_year=2010&amp;pages=504-507&amp;pmid=20453838&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B15">15. <span class="mixed-citation">
Antoniou A.C., Wang X., Fredericksen Z.S., McGuffog L., Tarrell R., Sinilnikova O.M., Healey S., Morrison J., Kartsonaki C., Lesnick T., et al. (2010) 
<span class="ref-title">A locus on 19p13 modifies risk of breast cancer in BRCA1 mutation carriers and is associated with hormone receptor-negative breast cancer in the general population</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">42</span>, 885–892. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3130795/">PMC free article</a>]</span> [<a href="/pubmed/20852631" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+locus+on+19p13+modifies+risk+of+breast+cancer+in+BRCA1+mutation+carriers+and+is+associated+with+hormone+receptor-negative+breast+cancer+in+the+general+population&amp;author=A.C.+Antoniou&amp;author=X.+Wang&amp;author=Z.S.+Fredericksen&amp;author=L.+McGuffog&amp;author=R.+Tarrell&amp;volume=42&amp;publication_year=2010&amp;pages=885-892&amp;pmid=20852631&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B16">16. <span class="mixed-citation">
Fletcher O., Johnson N., Orr N., Hosking F.J., Gibson L.J., Walker K., Zelenika D., Gut I., Heath S., Palles C., et al. (2011) 
<span class="ref-title">Novel breast cancer susceptibility locus at 9q31.2: results of a genome-wide association study</span>. <span class="ref-journal">J. Natl Cancer Inst.</span>, <span class="ref-vol">103</span>, 425–435. [<a href="/pubmed/21263130" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Natl+Cancer+Inst.&amp;title=Novel+breast+cancer+susceptibility+locus+at+9q31.2:+results+of+a+genome-wide+association+study&amp;author=O.+Fletcher&amp;author=N.+Johnson&amp;author=N.+Orr&amp;author=F.J.+Hosking&amp;author=L.J.+Gibson&amp;volume=103&amp;publication_year=2011&amp;pages=425-435&amp;pmid=21263130&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B17">17. <span class="mixed-citation">
Ghoussaini M., Fletcher O., Michailidou K., Turnbull C., Schmidt M.K., Dicks E., Dennis J., Wang Q., Humphreys M.K., Luccarini C., et al. (2012) 
<span class="ref-title">Genome-wide association analysis identifies three new breast cancer susceptibility loci</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">44</span>, 312–318. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3653403/">PMC free article</a>]</span> [<a href="/pubmed/22267197" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+analysis+identifies+three+new+breast+cancer+susceptibility+loci&amp;author=M.+Ghoussaini&amp;author=O.+Fletcher&amp;author=K.+Michailidou&amp;author=C.+Turnbull&amp;author=M.K.+Schmidt&amp;volume=44&amp;publication_year=2012&amp;pages=312-318&amp;pmid=22267197&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B18">18. <span class="mixed-citation">
Siddiq A., Couch F.J., Chen G.K., Lindstrom S., Eccles D., Millikan R.C., Michailidou K., Stram D.O., Beckmann L., Rhie S.K., et al. (2012) 
<span class="ref-title">A meta-analysis of genome-wide association studies of breast cancer identifies two novel susceptibility loci at 6q14 and 20q11</span>. <span class="ref-journal">Hum. Mol. Genet.</span>, <span class="ref-vol">21</span>, 5373–5384. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3510753/">PMC free article</a>]</span> [<a href="/pubmed/22976474" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=A+meta-analysis+of+genome-wide+association+studies+of+breast+cancer+identifies+two+novel+susceptibility+loci+at+6q14+and+20q11&amp;author=A.+Siddiq&amp;author=F.J.+Couch&amp;author=G.K.+Chen&amp;author=S.+Lindstrom&amp;author=D.+Eccles&amp;volume=21&amp;publication_year=2012&amp;pages=5373-5384&amp;pmid=22976474&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B19">19. <span class="mixed-citation">
Garcia-Closas M., Couch F.J., Lindstrom S., Michailidou K., Schmidt M.K., Brook M.N., Orr N., Rhie S.K., Riboli E., Feigelson H.S., et al. (2013) 
<span class="ref-title">Genome-wide association studies identify four ER negative-specific breast cancer risk loci</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">45</span>, 392–398. 398e391-392. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3771695/">PMC free article</a>]</span> [<a href="/pubmed/23535733" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+studies+identify+four+ER+negative-specific+breast+cancer+risk+loci&amp;author=M.+Garcia-Closas&amp;author=F.J.+Couch&amp;author=S.+Lindstrom&amp;author=K.+Michailidou&amp;author=M.K.+Schmidt&amp;volume=45&amp;publication_year=2013&amp;pages=392-398&amp;pmid=23535733&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B20">20. <span class="mixed-citation">
Bojesen S.E., Pooley K.A., Johnatty S.E., Beesley J., Michailidou K., Tyrer J.P., Edwards S.L., Pickett H.A., Shen H.C., Smart C.E., et al. (2013) 
<span class="ref-title">Multiple independent variants at the TERT locus are associated with telomere length and risks of breast and ovarian cancer</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">45</span>, 371–384. 384e371-372. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3670748/">PMC free article</a>]</span> [<a href="/pubmed/23535731" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Multiple+independent+variants+at+the+TERT+locus+are+associated+with+telomere+length+and+risks+of+breast+and+ovarian+cancer&amp;author=S.E.+Bojesen&amp;author=K.A.+Pooley&amp;author=S.E.+Johnatty&amp;author=J.+Beesley&amp;author=K.+Michailidou&amp;volume=45&amp;publication_year=2013&amp;pages=371-384&amp;pmid=23535731&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B21">21. <span class="mixed-citation">
Haiman C.A., Chen G.K., Vachon C.M., Canzian F., Dunning A., Millikan R.C., Wang X., Ademuyiwa F., Ahmed S., Ambrosone C.B., et al. (2011) 
<span class="ref-title">A common variant at the TERT-CLPTM1L locus is associated with estrogen receptor-negative breast cancer</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">43</span>, 1210–1214. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3279120/">PMC free article</a>]</span> [<a href="/pubmed/22037553" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+common+variant+at+the+TERT-CLPTM1L+locus+is+associated+with+estrogen+receptor-negative+breast+cancer&amp;author=C.A.+Haiman&amp;author=G.K.+Chen&amp;author=C.M.+Vachon&amp;author=F.+Canzian&amp;author=A.+Dunning&amp;volume=43&amp;publication_year=2011&amp;pages=1210-1214&amp;pmid=22037553&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B22">22. <span class="mixed-citation">
Howie B.N., Donnelly P., Marchini J. (2009) 
<span class="ref-title">A flexible and accurate genotype imputation method for the next generation of genome-wide association studies</span>. <span class="ref-journal">PLoS Genetics</span>, <span class="ref-vol">5</span>, e1000529.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genetics&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=B.N.+Howie&amp;author=P.+Donnelly&amp;author=J.+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529.&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B23">23. <span class="mixed-citation">
Wiley S.R., Schooley K., Smolak P.J., Din W.S., Huang C.P., Nicholl J.K., Sutherland G.R., Smith T.D., Rauch C., Smith C.A., et al. (1995) 
<span class="ref-title">Identification and characterization of a new member of the TNF family that induces apoptosis</span>. <span class="ref-journal">Immunity</span>, <span class="ref-vol">3</span>, 673–682. [<a href="/pubmed/8777713" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Identification+and+characterization+of+a+new+member+of+the+TNF+family+that+induces+apoptosis&amp;author=S.R.+Wiley&amp;author=K.+Schooley&amp;author=P.J.+Smolak&amp;author=W.S.+Din&amp;author=C.P.+Huang&amp;volume=3&amp;publication_year=1995&amp;pages=673-682&amp;pmid=8777713&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B24">24. <span class="mixed-citation">
Consortium G.T. (2015) 
<span class="ref-title">Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans</span>. <span class="ref-journal">Science</span>, <span class="ref-vol">348</span>, 648–660. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4547484/">PMC free article</a>]</span> [<a href="/pubmed/25954001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Human+genomics.+The+Genotype-Tissue+Expression+(GTEx)+pilot+analysis:+multitissue+gene+regulation+in+humans&amp;author=G.T.+Consortium&amp;volume=348&amp;publication_year=2015&amp;pages=648-660&amp;pmid=25954001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B25">25. <span class="mixed-citation">
Rahman M., Davis S.R., Pumphrey J.G., Bao J., Nau M.M., Meltzer P.S., Lipkowitz S. (2009) 
<span class="ref-title">TRAIL induces apoptosis in triple-negative breast cancer cells with a mesenchymal phenotype</span>. <span class="ref-journal">Breast Cancer Res. Treat.</span>, <span class="ref-vol">113</span>, 217–230. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2615075/">PMC free article</a>]</span> [<a href="/pubmed/18266105" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Breast+Cancer+Res.+Treat.&amp;title=TRAIL+induces+apoptosis+in+triple-negative+breast+cancer+cells+with+a+mesenchymal+phenotype&amp;author=M.+Rahman&amp;author=S.R.+Davis&amp;author=J.G.+Pumphrey&amp;author=J.+Bao&amp;author=M.M.+Nau&amp;volume=113&amp;publication_year=2009&amp;pages=217-230&amp;pmid=18266105&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B26">26. <span class="mixed-citation">
Lemke J., von Karstedt S., Zinngrebe J., Walczak H. (2014) 
<span class="ref-title">Getting TRAIL back on track for cancer therapy</span>. <span class="ref-journal">Cell Death Differ.</span>, <span class="ref-vol">21</span>, 1350–1364. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4131183/">PMC free article</a>]</span> [<a href="/pubmed/24948009" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Death+Differ.&amp;title=Getting+TRAIL+back+on+track+for+cancer+therapy&amp;author=J.+Lemke&amp;author=S.+von+Karstedt&amp;author=J.+Zinngrebe&amp;author=H.+Walczak&amp;volume=21&amp;publication_year=2014&amp;pages=1350-1364&amp;pmid=24948009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B27">27. <span class="mixed-citation">
Westra H.J., Peters M.J., Esko T., Yaghootkar H., Schurmann C., Kettunen J., Christiansen M.W., Fairfax B.P., Schramm K., Powell J.E., et al. (2013) 
<span class="ref-title">Systematic identification of trans eQTLs as putative drivers of known disease associations</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">45</span>, 1238–1243. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3991562/">PMC free article</a>]</span> [<a href="/pubmed/24013639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Systematic+identification+of+trans+eQTLs+as+putative+drivers+of+known+disease+associations&amp;author=H.J.+Westra&amp;author=M.J.+Peters&amp;author=T.+Esko&amp;author=H.+Yaghootkar&amp;author=C.+Schurmann&amp;volume=45&amp;publication_year=2013&amp;pages=1238-1243&amp;pmid=24013639&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B28">28. <span class="mixed-citation">
Zou F., Chai H.S., Younkin C.S., Allen M., Crook J., Pankratz V.S., Carrasquillo M.M., Rowley C.N., Nair A.A., Middha S., et al. (2012) 
<span class="ref-title">Brain expression genome-wide association study (eGWAS) identifies human disease-associated variants</span>. <span class="ref-journal">PLoS Genetics</span>, <span class="ref-vol">8</span>, e1002707.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3369937/">PMC free article</a>]</span> [<a href="/pubmed/22685416" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genetics&amp;title=Brain+expression+genome-wide+association+study+(eGWAS)+identifies+human+disease-associated+variants&amp;author=F.+Zou&amp;author=H.S.+Chai&amp;author=C.S.+Younkin&amp;author=M.+Allen&amp;author=J.+Crook&amp;volume=8&amp;publication_year=2012&amp;pages=e1002707.&amp;pmid=22685416&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B29">29. <span class="mixed-citation">
Chen F., Chen G.K., Millikan R.C., John E.M., Ambrosone C.B., Bernstein L., Zheng W., Hu J.J., Ziegler R.G., Deming S.L., et al. (2011) 
<span class="ref-title">Fine-mapping of breast cancer susceptibility loci characterizes genetic risk in African Americans</span>. <span class="ref-journal">Hum. Mol. Genet.</span>, <span class="ref-vol">20</span>, 4491–4503. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3196889/">PMC free article</a>]</span> [<a href="/pubmed/21852243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Fine-mapping+of+breast+cancer+susceptibility+loci+characterizes+genetic+risk+in+African+Americans&amp;author=F.+Chen&amp;author=G.K.+Chen&amp;author=R.C.+Millikan&amp;author=E.M.+John&amp;author=C.B.+Ambrosone&amp;volume=20&amp;publication_year=2011&amp;pages=4491-4503&amp;pmid=21852243&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B30">30. <span class="mixed-citation">
Zheng W., Cai Q., Signorello L.B., Long J., Hargreaves M.K., Deming S.L., Li G., Li C., Cui Y., Blot W.J. (2009) 
<span class="ref-title">Evaluation of 11 breast cancer susceptibility loci in African-American women</span>. <span class="ref-journal">Cancer. Epidemiol. Biomarkers Prev.</span>, <span class="ref-vol">18</span>, 2761–2764. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2759857/">PMC free article</a>]</span> [<a href="/pubmed/19789366" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer.+Epidemiol.+Biomarkers+Prev.&amp;title=Evaluation+of+11+breast+cancer+susceptibility+loci+in+African-American+women&amp;author=W.+Zheng&amp;author=Q.+Cai&amp;author=L.B.+Signorello&amp;author=J.+Long&amp;author=M.K.+Hargreaves&amp;volume=18&amp;publication_year=2009&amp;pages=2761-2764&amp;pmid=19789366&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B31">31. <span class="mixed-citation">
Huo D., Zheng Y., Ogundiran T.O., Adebamowo C., Nathanson K.L., Domchek S.M., Rebbeck T.R., Simon M.S., John E.M., Hennis A., et al. (2012) 
<span class="ref-title">Evaluation of 19 susceptibility loci of breast cancer in women of African ancestry</span>. <span class="ref-journal">Carcinogenesis</span>, <span class="ref-vol">33</span>, 835–840. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3324445/">PMC free article</a>]</span> [<a href="/pubmed/22357627" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Evaluation+of+19+susceptibility+loci+of+breast+cancer+in+women+of+African+ancestry&amp;author=D.+Huo&amp;author=Y.+Zheng&amp;author=T.O.+Ogundiran&amp;author=C.+Adebamowo&amp;author=K.L.+Nathanson&amp;volume=33&amp;publication_year=2012&amp;pages=835-840&amp;pmid=22357627&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B32">32. <span class="mixed-citation">
Palmer J.R., Ruiz-Narvaez E.A., Rotimi C.N., Cupples L.A., Cozier Y.C., Adams-Campbell L.L., Rosenberg L. (2013) 
<span class="ref-title">Genetic susceptibility loci for subtypes of breast cancer in an African American population. <em>Cancer.</em> </span><span class="ref-journal">Epidemiol. Biomarkers Prev.</span>, <span class="ref-vol">22</span>, 127–134. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3538887/">PMC free article</a>]</span> [<a href="/pubmed/23136140" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Epidemiol.+Biomarkers+Prev.&amp;title=Genetic+susceptibility+loci+for+subtypes+of+breast+cancer+in+an+African+American+population.+Cancer.&amp;author=J.R.+Palmer&amp;author=E.A.+Ruiz-Narvaez&amp;author=C.N.+Rotimi&amp;author=L.A.+Cupples&amp;author=Y.C.+Cozier&amp;volume=22&amp;publication_year=2013&amp;pages=127-134&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B33">33. <span class="mixed-citation">
Ghoussaini M., Edwards S.L., Michailidou K., Nord S., Cowper-Sal Lari R., Desai K., Kar S., Hillman K.M., Kaufmann S., Glubb D.M., et al. (2014) 
<span class="ref-title">Evidence that breast cancer risk at the 2q35 locus is mediated through IGFBP5 regulation</span>. <span class="ref-journal">Nat. Commun.</span>, <span class="ref-vol">4</span>, 4999.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4321900/">PMC free article</a>]</span> [<a href="/pubmed/25248036" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Evidence+that+breast+cancer+risk+at+the+2q35+locus+is+mediated+through+IGFBP5+regulation&amp;author=M.+Ghoussaini&amp;author=S.L.+Edwards&amp;author=K.+Michailidou&amp;author=S.+Nord&amp;author=R.+Cowper-Sal+Lari&amp;volume=4&amp;publication_year=2014&amp;pages=4999.&amp;pmid=25248036&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B34">34. <span class="mixed-citation">
Chen F., Chen G.K., Stram D.O., Millikan R.C., Ambrosone C.B., John E.M., Bernstein L., Zheng W., Palmer J.R., Hu J.J., et al. (2013) 
<span class="ref-title">A genome-wide association study of breast cancer in women of African ancestry</span>. <span class="ref-journal">Hum. Genet.</span>, <span class="ref-vol">132</span>, 39–48. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3749077/">PMC free article</a>]</span> [<a href="/pubmed/22923054" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=A+genome-wide+association+study+of+breast+cancer+in+women+of+African+ancestry&amp;author=F.+Chen&amp;author=G.K.+Chen&amp;author=D.O.+Stram&amp;author=R.C.+Millikan&amp;author=C.B.+Ambrosone&amp;volume=132&amp;publication_year=2013&amp;pages=39-48&amp;pmid=22923054&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B35">35. <span class="mixed-citation">
Palmer J.R., Ambrosone C.B., Olshan A.F. (2014) 
<span class="ref-title">A collaborative study of the etiology of breast cancer subtypes in African American women: the AMBER consortium</span>. <span class="ref-journal">Cancer Causes Control</span>, <span class="ref-vol">25</span>, 309–319. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3945713/">PMC free article</a>]</span> [<a href="/pubmed/24343304" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Causes+Control&amp;title=A+collaborative+study+of+the+etiology+of+breast+cancer+subtypes+in+African+American+women:+the+AMBER+consortium&amp;author=J.R.+Palmer&amp;author=C.B.+Ambrosone&amp;author=A.F.+Olshan&amp;volume=25&amp;publication_year=2014&amp;pages=309-319&amp;pmid=24343304&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B36">36. <span class="mixed-citation">
Marchini J., Howie B. (2010) 
<span class="ref-title">Genotype imputation for genome-wide association studies</span>. <span class="ref-journal">Nat. Rev.</span>, <span class="ref-vol">11</span>, 499–511. [<a href="/pubmed/20517342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.&amp;title=Genotype+imputation+for+genome-wide+association+studies&amp;author=J.+Marchini&amp;author=B.+Howie&amp;volume=11&amp;publication_year=2010&amp;pages=499-511&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B37">37. <span class="mixed-citation">
Ward L.D., Kellis M. (2012) 
<span class="ref-title">HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants</span>. <span class="ref-journal">Nucleic Acids Res.</span>, <span class="ref-vol">40</span>, D930–D934. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3245002/">PMC free article</a>]</span> [<a href="/pubmed/22064851" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=HaploReg:+a+resource+for+exploring+chromatin+states,+conservation,+and+regulatory+motif+alterations+within+sets+of+genetically+linked+variants&amp;author=L.D.+Ward&amp;author=M.+Kellis&amp;volume=40&amp;publication_year=2012&amp;pages=D930-D934&amp;pmid=22064851&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B38">38. <span class="mixed-citation">
Encode Project Consortium (2011) 
<span class="ref-title">A user's guide to the encyclopedia of DNA elements (ENCODE)</span>. <span class="ref-journal">PLoS Biol.</span>, <span class="ref-vol">9</span>, e1001046.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3079585/">PMC free article</a>]</span> [<a href="/pubmed/21526222" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol.&amp;title=A+user's+guide+to+the+encyclopedia+of+DNA+elements+(ENCODE)&amp;volume=9&amp;publication_year=2011&amp;pages=e1001046.&amp;pmid=21526222&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw305-B39">39. <span class="mixed-citation">
Roadmap Epigenomics C., Kundaje A., Meuleman W., Ernst J., Bilenky M., Yen A., Heravi-Moussavi A., Kheradpour P., Zhang Z., Wang J., et al. (2015) 
<span class="ref-title">Integrative analysis of 111 reference human epigenomes</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">518</span>, 317–330. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4530010/">PMC free article</a>]</span> [<a href="/pubmed/25693563" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrative+analysis+of+111+reference+human+epigenomes&amp;author=C.+Roadmap+Epigenomics&amp;author=A.+Kundaje&amp;author=W.+Meuleman&amp;author=J.+Ernst&amp;author=M.+Bilenky&amp;volume=518&amp;publication_year=2015&amp;pages=317-330&amp;pmid=25693563&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5975608&amp;issue-id=292035&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140015758985232title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Results</a></li><li><a href="#__sec3title">Discussion</a></li><li><a href="#__sec4title">Materials and Methods</a></li><li><a href="#__sec11title">Supplementary Material</a></li><li><a href="#__sec12title">Funding </a></li><li><a href="#__sec13title">Supplementary Material</a></li><li><a href="#idm140015758157824title">References </a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5975608/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5975608/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5975608/pdf/ddw305.pdf">PDF (768K)</a></li> | <li><a href="#" data-citationid="PMC5975608" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5975608%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5975608%2F&amp;text=Genome-wide%20association%20studies%20in%20women%20of%20African%20ancestry%20identified%203q26.21%20as%20a%20novel%20susceptibility%20locus%20for%20oestrogen%20receptor%20negative%20breast%20cancer" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5975608%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="28171663" data-aiid="5975608" data-aid="5975608" data-iid="292035" data-domainid="740" data-domain="hmg" data-accid="PMC5975608" data-md5="a237627ede5afe1c0c7e60ee11100366"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T15:46:10-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE88C218DD5A2411000000000271016C&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5975608%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5975608/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>