<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC6215702/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association reveals that common genetic variation in the kallikrein-kinin system is associated with serum L-arginine levels</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6215702/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Thrombosis and haemostasis"><meta name="citation_title" content="Genome-wide association reveals that common genetic variation in the kallikrein-kinin system is associated with serum L-arginine levels"><meta name="citation_authors" content="Weihua Zhang, Fredrik Jernerén, Benjamin C Lehne, Ming-Huei Chen, Robert N Luben, Carole Johnston, Amany Elshorbagy, Ruben N Eppinga, William R Scott, Elizabeth Adeyeye, James Scott, Rainer H. Böger, Kay-Tee Khaw, Pim van der Harst, Nicholas J Wareham, Ramachandran S Vasan, John C Chambers, Helga Refsum, Jaspal S Kooner"><meta name="citation_date" content="30 November 2016"><meta name="citation_issue" content="6"><meta name="citation_volume" content="116"><meta name="citation_firstpage" content="1041"><meta name="citation_doi" content="10.1160/TH16-02-0151"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC6215702/?report=abstract"><meta name="citation_pmid" content="27656708"><meta name="DC.Title" content="Genome-wide association reveals that common genetic variation in the kallikrein-kinin system is associated with serum L-arginine levels"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Weihua Zhang"><meta name="DC.Contributor" content="Fredrik Jernerén"><meta name="DC.Contributor" content="Benjamin C Lehne"><meta name="DC.Contributor" content="Ming-Huei Chen"><meta name="DC.Contributor" content="Robert N Luben"><meta name="DC.Contributor" content="Carole Johnston"><meta name="DC.Contributor" content="Amany Elshorbagy"><meta name="DC.Contributor" content="Ruben N Eppinga"><meta name="DC.Contributor" content="William R Scott"><meta name="DC.Contributor" content="Elizabeth Adeyeye"><meta name="DC.Contributor" content="James Scott"><meta name="DC.Contributor" content="Rainer H. Böger"><meta name="DC.Contributor" content="Kay-Tee Khaw"><meta name="DC.Contributor" content="Pim van der Harst"><meta name="DC.Contributor" content="Nicholas J Wareham"><meta name="DC.Contributor" content="Ramachandran S Vasan"><meta name="DC.Contributor" content="John C Chambers"><meta name="DC.Contributor" content="Helga Refsum"><meta name="DC.Contributor" content="Jaspal S Kooner"><meta name="DC.Date" content="2016 Nov 30"><meta name="DC.Identifier" content="10.1160/TH16-02-0151"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association reveals that common genetic variation in the kallikrein-kinin system is associated with serum L-arginine levels"><meta property="og:type" content="article"><meta property="og:description" content="L-arginine is the essential precursor of nitric oxide, and is involved in multiple key physiological processes, including vascular and immune function. The genetic regulation of blood L-arginine levels is largely unknown. We performed a genome-wide association ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6215702/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC6215702/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC6215702/pdf/emss-80283.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B08AEDD59D85100000000003B0023.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6215702%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC6215702</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Thromb Haemost</a></span>. Author manuscript; available in PMC 2018 Nov 4.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=27656708" target="pmc_ext" ref="reftype=publisher&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Thromb Haemost. 2016 Nov 30; 116(6): 1041–1049. </span></a></div><span class="fm-vol-iss-date">Published online 2016 Sep 22. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1160%2FTH16-02-0151" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1160/TH16-02-0151</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC6215702</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>EMS80283</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/27656708">27656708</a></div></div></div></div><h1 class="content-title">Genome-wide association reveals that common genetic variation in the kallikrein-kinin system is associated with serum L-arginine levels</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Zhang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037232622960" co-class="co-affbox">Weihua Zhang</a>,<sup>1,</sup><sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Jerner%26%23x000e9%3Bn%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037241924800" co-class="co-affbox">Fredrik Jernerén</a>,<sup>3</sup> <a href="/pubmed/?term=Lehne%20BC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037245723776" co-class="co-affbox">Benjamin C Lehne</a>,<sup>1</sup> <a href="/pubmed/?term=Chen%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037245721904" co-class="co-affbox">Ming-Huei Chen</a>,<sup>4</sup> <a href="/pubmed/?term=Luben%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037236455088" co-class="co-affbox">Robert N Luben</a>,<sup>5</sup> <a href="/pubmed/?term=Johnston%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037245440016" co-class="co-affbox">Carole Johnston</a>,<sup>3</sup> <a href="/pubmed/?term=Elshorbagy%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037245438144" co-class="co-affbox">Amany Elshorbagy</a>,<sup>3,</sup><sup>6</sup> <a href="/pubmed/?term=Eppinga%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037238900720" co-class="co-affbox">Ruben N Eppinga</a>,<sup>7,</sup><sup>8</sup> <a href="/pubmed/?term=Scott%20WR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037240265392" co-class="co-affbox">William R Scott</a>,<sup>1,</sup><sup>9</sup> <a href="/pubmed/?term=Adeyeye%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037238760640" co-class="co-affbox">Elizabeth Adeyeye</a>,<sup>2,</sup><sup>10</sup> <a href="/pubmed/?term=Scott%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037315336384" co-class="co-affbox">James Scott</a>,<sup>9</sup> <a href="/pubmed/?term=B%26%23x000f6%3Bger%20RH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037236035296" co-class="co-affbox">Rainer H. Böger</a>,<sup>11</sup> <a href="/pubmed/?term=Khaw%20KT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037236033312" co-class="co-affbox">Kay-Tee Khaw</a>,<sup>5</sup> <a href="/pubmed/?term=van%20der%20Harst%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037243299872" co-class="co-affbox">Pim van der Harst</a>,<sup>7,</sup><sup>8</sup> <a href="/pubmed/?term=Wareham%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037243921504" co-class="co-affbox">Nicholas J Wareham</a>,<sup>12</sup> <a href="/pubmed/?term=Vasan%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037243919520" co-class="co-affbox">Ramachandran S Vasan</a>,<sup>13,</sup><sup>14</sup> <a href="/pubmed/?term=Chambers%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037277710848" co-class="co-affbox">John C Chambers</a>,<sup>1,</sup><sup>2,</sup><sup>15,</sup><sup>*</sup> <a href="/pubmed/?term=Refsum%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037238709936" co-class="co-affbox">Helga Refsum</a>,<sup>3,</sup><sup>16,</sup><sup>*</sup> and  <a href="/pubmed/?term=Kooner%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708" class="affpopup" co-rid="_co_idm140037243330624" co-class="co-affbox">Jaspal S Kooner</a><sup>2,</sup><sup>9,</sup><sup>15,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140037232622960"><h3 class="no_margin">Weihua Zhang</h3><p><sup>1</sup>Department of Epidemiology and Biostatistics, School of Public Health, Imperial College London, Norfolk Place, London W2 1PG, UK</p><p><sup>2</sup>Department of Cardiology, Ealing Hospital NHS Trust, Uxbridge Road, Southall, Middlesex UB1 3HW, UK</p><div>Find articles by <a href="/pubmed/?term=Zhang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Weihua Zhang</a></div></div><div id="_co_idm140037241924800"><h3 class="no_margin">Fredrik Jernerén</h3><p><sup>3</sup>Department of Pharmacology, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Jerner%26%23x000e9%3Bn%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Fredrik Jernerén</a></div></div><div id="_co_idm140037245723776"><h3 class="no_margin">Benjamin C Lehne</h3><p><sup>1</sup>Department of Epidemiology and Biostatistics, School of Public Health, Imperial College London, Norfolk Place, London W2 1PG, UK</p><div>Find articles by <a href="/pubmed/?term=Lehne%20BC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Benjamin C Lehne</a></div></div><div id="_co_idm140037245721904"><h3 class="no_margin">Ming-Huei Chen</h3><p><sup>4</sup>Population Sciences Branch, National Heart, Lung, Blood Institute, National Institutes of Health, Framingham, MA 01702, USA</p><div>Find articles by <a href="/pubmed/?term=Chen%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Ming-Huei Chen</a></div></div><div id="_co_idm140037236455088"><h3 class="no_margin">Robert N Luben</h3><p><sup>5</sup>Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Luben%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Robert N Luben</a></div></div><div id="_co_idm140037245440016"><h3 class="no_margin">Carole Johnston</h3><p><sup>3</sup>Department of Pharmacology, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Johnston%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Carole Johnston</a></div></div><div id="_co_idm140037245438144"><h3 class="no_margin">Amany Elshorbagy</h3><p><sup>3</sup>Department of Pharmacology, University of Oxford, Oxford, UK</p><p><sup>6</sup>Department of Physiology, Faculty of Medicine, University of Alexandria, Alexandria, Egypt</p><div>Find articles by <a href="/pubmed/?term=Elshorbagy%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Amany Elshorbagy</a></div></div><div id="_co_idm140037238900720"><h3 class="no_margin">Ruben N Eppinga</h3><p><sup>7</sup>Department of Cardiology, University of Groningen, University Medical Center Groningen, 9700 RB Groningen, the Netherlands</p><p><sup>8</sup>Department of Genetics, University of Groningen, University Medical Center Groningen, 9700 RB Groningen, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Eppinga%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Ruben N Eppinga</a></div></div><div id="_co_idm140037240265392"><h3 class="no_margin">William R Scott</h3><p><sup>1</sup>Department of Epidemiology and Biostatistics, School of Public Health, Imperial College London, Norfolk Place, London W2 1PG, UK</p><p><sup>9</sup>National Heart and Lung Institute, Imperial College London, Hammersmith Hospital Campus, London, UK</p><div>Find articles by <a href="/pubmed/?term=Scott%20WR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">William R Scott</a></div></div><div id="_co_idm140037238760640"><h3 class="no_margin">Elizabeth Adeyeye</h3><p><sup>2</sup>Department of Cardiology, Ealing Hospital NHS Trust, Uxbridge Road, Southall, Middlesex UB1 3HW, UK</p><p><sup>10</sup>Faculty of Medicine, Imperial College London, South Kensington Campus, London, SW7 2AZ</p><div>Find articles by <a href="/pubmed/?term=Adeyeye%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Elizabeth Adeyeye</a></div></div><div id="_co_idm140037315336384"><h3 class="no_margin">James Scott</h3><p><sup>9</sup>National Heart and Lung Institute, Imperial College London, Hammersmith Hospital Campus, London, UK</p><div>Find articles by <a href="/pubmed/?term=Scott%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">James Scott</a></div></div><div id="_co_idm140037236035296"><h3 class="no_margin">Rainer H. Böger</h3><p><sup>11</sup>Department of Clinical Pharmacology and Toxicology, University Medical Center Hamburg-Eppendorf, 20246 Hamburg, Germany</p><div>Find articles by <a href="/pubmed/?term=B%26%23x000f6%3Bger%20RH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Rainer H. Böger</a></div></div><div id="_co_idm140037236033312"><h3 class="no_margin">Kay-Tee Khaw</h3><p><sup>5</sup>Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Khaw%20KT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Kay-Tee Khaw</a></div></div><div id="_co_idm140037243299872"><h3 class="no_margin">Pim van der Harst</h3><p><sup>7</sup>Department of Cardiology, University of Groningen, University Medical Center Groningen, 9700 RB Groningen, the Netherlands</p><p><sup>8</sup>Department of Genetics, University of Groningen, University Medical Center Groningen, 9700 RB Groningen, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20der%20Harst%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Pim van der Harst</a></div></div><div id="_co_idm140037243921504"><h3 class="no_margin">Nicholas J Wareham</h3><p><sup>12</sup>Medical Research Council Epidemiology Unit, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Wareham%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Nicholas J Wareham</a></div></div><div id="_co_idm140037243919520"><h3 class="no_margin">Ramachandran S Vasan</h3><p><sup>13</sup>Section of Preventive Medicine and Epidemiology and Cardiology, Boston University School of Medicine, Boston, MA 02118, USA</p><p><sup>14</sup>Boston University and National Heart, Lung &amp; Blood Institute’s Framingham Heart Study, Framingham, MA 01702, USA</p><div>Find articles by <a href="/pubmed/?term=Vasan%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Ramachandran S Vasan</a></div></div><div id="_co_idm140037277710848"><h3 class="no_margin">John C Chambers</h3><p><sup>1</sup>Department of Epidemiology and Biostatistics, School of Public Health, Imperial College London, Norfolk Place, London W2 1PG, UK</p><p><sup>2</sup>Department of Cardiology, Ealing Hospital NHS Trust, Uxbridge Road, Southall, Middlesex UB1 3HW, UK</p><p><sup>15</sup>Imperial College Healthcare NHS Trust, London, UK</p><div>Find articles by <a href="/pubmed/?term=Chambers%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">John C Chambers</a></div></div><div id="_co_idm140037238709936"><h3 class="no_margin">Helga Refsum</h3><p><sup>3</sup>Department of Pharmacology, University of Oxford, Oxford, UK</p><p><sup>16</sup>Department of Nutrition, Institute of Basic Medical Science, Faculty of Medicine, University of Oslo, Oslo, Norway</p><div>Find articles by <a href="/pubmed/?term=Refsum%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Helga Refsum</a></div></div><div id="_co_idm140037243330624"><h3 class="no_margin">Jaspal S Kooner</h3><p><sup>2</sup>Department of Cardiology, Ealing Hospital NHS Trust, Uxbridge Road, Southall, Middlesex UB1 3HW, UK</p><p><sup>9</sup>National Heart and Lung Institute, Imperial College London, Hammersmith Hospital Campus, London, UK</p><p><sup>15</sup>Imperial College Healthcare NHS Trust, London, UK</p><div>Find articles by <a href="/pubmed/?term=Kooner%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27656708">Jaspal S Kooner</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140037246969024_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140037246969024_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140037246969024_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Epidemiology and Biostatistics, School of Public Health, Imperial College London, Norfolk Place, London W2 1PG, UK</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Cardiology, Ealing Hospital NHS Trust, Uxbridge Road, Southall, Middlesex UB1 3HW, UK</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Pharmacology, University of Oxford, Oxford, UK</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Population Sciences Branch, National Heart, Lung, Blood Institute, National Institutes of Health, Framingham, MA 01702, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Physiology, Faculty of Medicine, University of Alexandria, Alexandria, Egypt</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Cardiology, University of Groningen, University Medical Center Groningen, 9700 RB Groningen, the Netherlands</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Genetics, University of Groningen, University Medical Center Groningen, 9700 RB Groningen, The Netherlands</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>National Heart and Lung Institute, Imperial College London, Hammersmith Hospital Campus, London, UK</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Faculty of Medicine, Imperial College London, South Kensington Campus, London, SW7 2AZ</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Clinical Pharmacology and Toxicology, University Medical Center Hamburg-Eppendorf, 20246 Hamburg, Germany</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Medical Research Council Epidemiology Unit, University of Cambridge, Cambridge, UK</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Section of Preventive Medicine and Epidemiology and Cardiology, Boston University School of Medicine, Boston, MA 02118, USA</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Boston University and National Heart, Lung &amp; Blood Institute’s Framingham Heart Study, Framingham, MA 01702, USA</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Imperial College Healthcare NHS Trust, London, UK</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Department of Nutrition, Institute of Basic Medical Science, Faculty of Medicine, University of Oslo, Oslo, Norway</div><div id="CR1"><strong>Corresponding author</strong>: Weihua Zhang, Ph.D, Department of Epidemiology and Biostatistics, School of Public Health, Faculty of Medicine, Imperial College London, Norfolk Place, London W2 1PG, UK, Tel: +44 (0)20 8242 5926, Fax: +44 (0)20 8967 5007, <a href="mailto:dev@null" data-email="ku.ca.lairepmi@gnahz.auhiew" class="oemail">ku.ca.lairepmi@gnahz.auhiew</a></div><div id="FN1"><sup>*</sup>Equal joint authorship</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140037246969024_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=27656708" target="pmc_ext" ref="reftype=publisher&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Thromb Haemost</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC6215702/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary information.</strong></div><div><a href="/pmc/articles/PMC6215702/bin/NIHMS80283-supplement-Supplementary_information.docx" data-ga-action="click_feat_suppl">NIHMS80283-supplement-Supplementary_information.docx</a><span style="color:gray"> (185K)</span></div><div class="small guid">GUID:&nbsp;BB1DDEED-407A-4EE9-806E-3CB3AE68EC8A</div></div></dd></dl></div><div id="idm140037238970944" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140037238970944title">Summary</h2><!--article-meta--><div><p id="P2" class="p p-first-last">L-arginine is the essential precursor of nitric oxide, and is involved in multiple key physiological processes, including vascular and immune function. The genetic regulation of blood L-arginine levels is largely unknown. We performed a genome-wide association study (GWAS) to identify genetic factors determining serum L-arginine levels, amongst 901 Europeans and 1,394 Indian Asians. We show that common genetic variations at the <em>KLKB1</em> and <em>F12</em> loci are strongly associated with serum L-arginine levels. The G allele of SNP rs71640036 (T/G) in <em>KLKB1</em> is associated with lower serum L-arginine concentrations (10 μmol/L per allele copy, p=1×10<sup>-24</sup>), while allele T of rs2545801 (T/C) near the <em>F12</em> gene is associated with lower serum L-arginine levels (7 μmol/L per allele copy, p=7×10<sup>-12</sup>). Together these two loci explain 7% of the total variance in serum L-arginine concentrations. The associations at both loci were replicated in independent cohorts with plasma L-arginine measurements (p&lt;0.004). The two sentinel SNPs are in nearly complete LD with the nonsynonymous SNP rs3733402 at <em>KLKB1</em> and the 5’-UTR SNP rs1801020 at <em>F12</em>, respectively. SNPs at both loci are associated with blood pressure. Our findings provide new insight into the genetic regulation of L-arginine and its potential relationship with cardiovascular risk.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Serum L-arginine concentration, genome-wide association, coagulation, kallikrein-kinin system</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P3" class="p p-first">L-arginine is a naturally occurring semi-essential amino acid, which plays a key biological role in cardiovascular physiology, immune function and hormonal signalling. L-arginine is the substrate for the synthesis of the potent vasodilator nitric oxide (NO) by the endothelial NO synthase (eNOS) in blood vessels. NO is crucial in maintaining vascular homeostasis, and its depletion may contribute to the development of hypertension, atherosclerosis, endothelial dysfunction and vascular inflammation (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970840">1</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970851">5</a>). L-arginine bioavailability is a rate limiting factor in the synthesis of NO (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970869">3</a>, <a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970847">4</a>) and low levels of circulating L-arginine are reported in a number of diseases and conditions (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970875">6</a>). Oral administration or infusion of L-arginine has shown the beneficial effect of improved endothelia function and reduced blood pressure (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970835">7</a>, <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970853">8</a>).</p><p id="P4">Recent studies suggest that L-arginine concentrations are heritable, and associated with genetic variation in <em>ARG1</em> (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970867">9</a>). In contrast, several studies using metabolome wide profiling, with semi-quantitative measurement of L-arginine, fail to identify genetic variation influencing L-arginine concentrations (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970868">10</a>–<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970838">16</a>). The genetic factors regulating L-arginine levels are not well understood.</p><p id="P5" class="p p-last">We performed a GWAS of serum L-arginine levels using targeted quantification of L-arginine by mass spectrometry (MS), to identify the genetic factors influencing L-arginine and investigate the potential relationships between L-arginine metabolism and cardiovascular traits.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Materials and methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Samples and phenotypes at the discovery stage</h3><p id="P6" class="p p-first">We investigated 2,487 participants selected at random from the London Life Sciences Prospective Population (LOLIPOP) Study (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970854">17</a>, <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970874">18</a>), comprising 998 Europeans and 1,489 Indian Asians. The LOLIPOP study is approved by the local Research Ethics Committee, and all participants provided written consent for genetic studies.</p><p id="P7" class="p p-last">Participants were interviewed by trained research nurses according to a standardized protocol. An interviewer-administered questionnaire was used to collect data on demographic factors, tobacco and alcohol consumption, as well as personal and family medical history. Indian Asians had all four grandparents born on the Indian subcontinent (India, Pakistan, Sri Lanka, or Bangladesh). Europeans were of self-reported white ancestry. Anthropometric (height, weight, waist and hip circumference) and blood pressure measurements were recorded. Blood samples were collected after an 8 hour fast for measurement of fasting glucose, hemoglobin A1c (HbA1c), lipid profile, creatinine, and high-sensitivity C-reactive protein (hsCRP). Aliquots of whole blood and serum were stored at -80°C.</p></div><div id="S4" class="sec"><h3 id="S4title">Serum L-arginine measurement at the discovery stage</h3><p id="P8" class="p p-first-last">Serum concentrations of L-arginine were measured by liquid chromatography-tandem mass spectrometry (LC-MS/MS) using a Prominence LC-20AD<sub>XR</sub> binary pump (Shimadzu, Kyoto, Japan) coupled to a QTRAP 5500 hybrid triple quadropole mass spectrometer (AB Sciex, Framingham, MA, US). Serum (25 μL) was mixed with equal volumes of distilled water, 50 mM ammonium hydroxide, and a solution containing isotopically labelled internal standard, d<sub>7</sub>-Arg (Cambridge Isotopes Laboratories, Andover, MA, US). Dithioerythritol (25 μL, 100 mM) was added and the samples were left for 15 min followed by protein precipitation using 25 μL perchloric acid (3.3 M) and centrifugation at 14’000 rpm for 10 min. Supernatant (5 μL) was then added to 195 μL NH<sub>4</sub><sup>+</sup> (10 mM), and 7 μL was subsequently injected onto a Kinetex PFP column (100 × 2.10 mm, 2.6 μm; Phenomenex, Torrance, CA, US) kept at 35°C. Gradient elution at 300 μL/min was employed with mobile phases A (0.05 % formic acid in water, v/v) and B (0.1 % formic acid in methanol, v/v). 0% B was initially held at for 1.5 min, followed by a linear gradient reaching 10% B at 4.0 min, and 100% B at 4.3 min, and held there for 1.5 min, followed by equilibration at 0% B for 2 min. Gas 1 and 2 were set at 50, collision gas at high, and the curtain gas, ion spray voltage, and source temperature were set at 35, 4000 V, and 700°C, respectively. The following ion transitions were used for quantitation in multiple reaction monitoring (MRM) mode: m/z 175 → m/z 70 for L-arginine, and m/z 182 → m/z 77 for d<sub>7</sub>-Arg. Optimisation of MRM transitions, data acquisition, and analysis were performed with Analyst 1.6.1 (AB Sciex, Framingham, MA, US). L-arginine was quantified by comparing the area ratio against the internal standard with calibration curves constructed in control plasma. For each batch, a number of randomly chosen samples, as well as all identified outliers, were identified and repeated to ensure accurate quantitation. Inter- and intra-assay coefficients of variation for L-arginine assessed at three QC levels were ~7%, well below the accepted limit recommended by the European Medicines Agency (see <a href="#FN2" rid="FN2" class=" fn">URL</a>).</p></div><div id="S5" class="sec"><h3 id="S5title">Genotyping and imputation at the discovery stage</h3><p id="P9" class="p p-first">DNA was extracted from whole blood. Genome-wide genotyping was carried out using the Illumina Human610 array, as previously described (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970877">17</a>), in 901 Europeans and 1,394 Indian Asians from amongst the 2,487 LOLIPOP participants with L-arginine measurements. SNPs with minor allele frequencies &lt;1%, p-values for Hardy-Weinberg equilibrium (p-HWE) &lt;1x10<sup>-6</sup>, or missing rates &gt;5% were removed. Exclusion criteria for samples were duplicates, ethnic outliers, cryptic relatedness, gender discrepancy, extreme heterogeneity, missing rate &gt;5%, or incomplete phenotype data.</p><p id="P10" class="p p-last">Imputation of un-genotyped variants was performed in European (544,620 directly genotyped SNPs) and Indian Asian (541,497 SNPs) samples separately with IMPUTE2 v2.3.0 (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970832">19</a>), using the cosmopolitan 1000 Genomes haplotypes (phase 1, March 2012 version, genomic build 37) as reference (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970885">20</a>). The reference haplotypes most relevant to the study population by allelic consistency are chosen automatically and it has shown that the cosmopolitan reference haplotypes produce high imputation accuracy (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970871">20</a>).</p></div><div id="S6" class="sec"><h3 id="S6title">Replication studies</h3><p id="P11" class="p p-first">The lead SNPs identified at the discovery stage were tested in two independent cohorts with plasma L-arginine measurements. All participants in the replication studies provided written informed consent for genetic studies, and the study protocols were approved by local Institutional review boards and ethical committees.</p><p id="P12">The European Prospective Investigation into Cancer, Norfolk (EPIC-Norfolk) is a prospective population cohort of men and women of European origin aged 40-79 years living in Norfolk, UK, recruited from general practices between 1993 and 1998 (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970834">21</a>). A nested case-control study of incident stroke was performed with an average follow-up of eleven years (N = 1,000). Controls were matched to cases on age and gender. Clinical and lifestyle data and blood samples were collected at baseline when all the cases and controls were stroke free. Baseline plasma L-arginine levels were measured with a modified approach as in our discovery study. Genotyping was performed on the Affymetrix 500K arrays, followed by quality control and imputation to Phase 1 1000 Genomes reference panel.</p><p id="P13" class="p p-last">The Framingham Heart Study (FHS) is a longitudinal observational, community-based cohort initiated in 1948 in Framingham, MA, USA, to investigate risk factors prospectively for cardiovascular diseases (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970864">22</a>). In this study, 2,981 participants of the Framingham Offspring Cohort (FOC), who had complete genotypic (Affymetrix 500K and 50K) and plasma L-arginine measurements were analysed after exclusions. The FOC consists of children (and children’s spouses) of the original Framingham cohort participants. Details for plasma L-arginine measurement, genotyping and imputation were described previously (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970856">9</a>).</p></div><div id="S7" class="sec"><h3 id="S7title">Statistical analysis</h3><p id="P14" class="p p-first">In the discovery cohorts, outliers with serum L-arginine levels exceeding 3 standard deviations from the means in males or females separately were removed from further analysis. Continuous and discrete traits were compared between Europeans and Indian Asians using one-way ANOVA and Chi-square tests respectively with R (version 2.15.0, <a href="/pmc/articles/PMC6215702/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Linear regression was used to investigate the associations between serum L-arginine levels and other phenotypic traits, with adjustment for age, gender and ethnicity (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p3">Characteristics of participants for the discovery cohorts. Results are presented as mean (SD) or % amongst Europeans (n = 998) and Indian Asians (N = 1,489). P value is for the comparison between the two populations.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140037236770800" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center">Europeans</th><th rowspan="1" colspan="1" valign="top" align="center">Indian Asians</th><th rowspan="1" colspan="1" valign="top" align="center">P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Age (years)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">55.4 (10.0)</td><td rowspan="1" colspan="1" valign="top" align="center">54.4 (10.0)</td><td rowspan="1" colspan="1" valign="top" align="center">1.8×10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Female (%)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">29.3</td><td rowspan="1" colspan="1" valign="top" align="center">36.6</td><td rowspan="1" colspan="1" valign="top" align="center">1.7×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Current smoking (%)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">17.0</td><td rowspan="1" colspan="1" valign="top" align="center">7.2</td><td rowspan="1" colspan="1" valign="top" align="center">3.3×10<sup>−14</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Alcohol drinking (%)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">73.0</td><td rowspan="1" colspan="1" valign="top" align="center">31.6</td><td rowspan="1" colspan="1" valign="top" align="center">&lt; 2×10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Type 2 diabetes (%)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">8.3</td><td rowspan="1" colspan="1" valign="top" align="center">20.5</td><td rowspan="1" colspan="1" valign="top" align="center">4.0×10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Coronary heart disease (%)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">2.4</td><td rowspan="1" colspan="1" valign="top" align="center">6.4</td><td rowspan="1" colspan="1" valign="top" align="center">8.3×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Hypertension (%)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">43.5</td><td rowspan="1" colspan="1" valign="top" align="center">50.5</td><td rowspan="1" colspan="1" valign="top" align="center">6.9×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Body mass index (kg/m<sup>2</sup>)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">27.6 (4.75)</td><td rowspan="1" colspan="1" valign="top" align="center">27.2 (4.2)</td><td rowspan="1" colspan="1" valign="top" align="center">3.9×10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Waist hip ratio</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.92 (0.08)</td><td rowspan="1" colspan="1" valign="top" align="center">0.95 (0.08)</td><td rowspan="1" colspan="1" valign="top" align="center">&lt; 2×10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Diastolic blood pressure (mmHg)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">80.8 (10.2)</td><td rowspan="1" colspan="1" valign="top" align="center">80.0 (10.3)</td><td rowspan="1" colspan="1" valign="top" align="center">7.7×10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Systolic blood pressure (mmHg)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">132.5 (19.0)</td><td rowspan="1" colspan="1" valign="top" align="center">131.1 (19.4)</td><td rowspan="1" colspan="1" valign="top" align="center">7.3×10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Total cholesterol (mmol/L)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">5.6 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">5.3 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">1.9×10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Triglycerides (mmol/L)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">1.6 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">1.7 (1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">1.5×10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>LDL cholesterol (mmol/L)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">3.5 (0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">3.3 (0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">3.9×10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>HDL cholesterol (mmol/L)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">1.4 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">1.3 (0.3)</td><td rowspan="1" colspan="1" valign="top" align="center">&lt; 2×10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Glucose (mmol/L)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">5.5 (1.5)</td><td rowspan="1" colspan="1" valign="top" align="center">5.9 (1.9)</td><td rowspan="1" colspan="1" valign="top" align="center">5.2×10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Insulin (IU/L)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">9.8 (9.1)</td><td rowspan="1" colspan="1" valign="top" align="center">12.1 (12.4)</td><td rowspan="1" colspan="1" valign="top" align="center">7.5×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>HbA1c (%)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">5.5 (0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">6.1 (1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">&lt; 2×10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>HOMA-IR</strong></td><td rowspan="1" colspan="1" valign="top" align="center">2.6 (3.2)</td><td rowspan="1" colspan="1" valign="top" align="center">3.3 (4.3)</td><td rowspan="1" colspan="1" valign="top" align="center">2.6×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>hsCRP (mg/L)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">3.7 (6.9)</td><td rowspan="1" colspan="1" valign="top" align="center">4.0 (5.3)</td><td rowspan="1" colspan="1" valign="top" align="center">2.1×10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Creatinine (μmol/L)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">92.0 (14.9)</td><td rowspan="1" colspan="1" valign="top" align="center">90.3 (25.1)</td><td rowspan="1" colspan="1" valign="top" align="center">4.8×10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>L-arginine (μmol/L)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">131.7 (31.6)</td><td rowspan="1" colspan="1" valign="top" align="center">135.0 (33.5)</td><td rowspan="1" colspan="1" valign="top" align="center">1.3×10<sup>−2</sup></td></tr></tbody></table></div><div id="largeobj_idm140037236770800" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6215702/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P50" class="p p-first-last">Footnote: Alcohol drinking is defined as any consumption of alcohol in the previous week. Coronary heart disease is defined as history of myocardial infarction, coronary revascularization, or angiographically severe disease. Hypertension is defined as history of hypertension on medical treatment. HOMA-IR, homeostatic model assessment-estimated insulin resistance index. hsCRP, high-sensitivity C-reactive protein.</p></div></div></div><p id="P15">Principal components (PCs) of the genotype data were obtained with smartpca in EIGENSOFT using ~100K linkage disequilibrium (LD)-pruned genome-wide SNPs. Linear regression with additive genetic effect for association and conditional analysis were performed with SNPTEST (v2.4.1), adjusted for age, gender, and PCs 1-5 (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970850">23</a>). No data transformation was applied to serum L-arginine as it conformed approximately to a normal distribution. Markers with minor allele frequency (MAF) &lt;0.01, p-HWE &lt;1×10<sup>-6</sup>, or imputation quality score &lt;0.4 were excluded at the discovery stage. A total of 28.7M markers were obtained before quality control, in which about 8M SNPs retained after filtering for further analysis. The association of SNPs with L-arginine concentrations was tested in Europeans and Indian Asians separately, and association test results corrected for the genomic inflation factors. We then carried out inverse variance meta-analyses across the two ethnic groups using a fixed effects model as implemented in METAL (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970837">24</a>). Regional plots for the associated loci were generated using LOCUSZOOM using genome build GRCh37/hg19 and 1000 Genome Nov 2014 European (EUR) population for linkage disequilibrium (LD) (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970833">25</a>). Trait variance explained by the sentinel (lead) SNP(s) was calculated in Europeans or Indian Asians separately, using linear regression model with covariates adjusted as above. The r<sup>2</sup> from the regression model was taken as the estimates of variance explained.</p><p id="P16" class="p p-last">In replication studies, association analyses of the sentinel or proxy SNPs and plasma L-arginine were carried out in FHS, using a linear mixed effects model with relationship coefficient matrix to account for familial correlation, adjusted for sex, age, diabetes mellitus, systolic and diastolic blood pressure, smoking, body mass index, and serum creatinine levels. In EPIC-Norfolk, association analyses were performed in cases (N = 500) and controls (N = 500) separately, under the additive genetic model with adjustment for age and gender. In meta-analyses of combined discovery and replication studies, we used the weighted Z-score method in METAL along with a random-effects model in METASOFT to account for heterogeneity between discovery and replication studies (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970857">26</a>).</p></div><div id="S8" class="sec"><h3 id="S8title">Expression quantitative trait loci (eQTLs) analysis</h3><p id="P17" class="p p-first">We analysed the relationship between the sentinel SNPs influencing L-arginine concentrations and the expression of genes within 1Mb either side of the sentinel SNPs in peripheral blood leucocytes amongst 816 Indian Asian participants of the LOLIPOP study. Bonferroni correction was applied for testing multiple genes. In brief, transcriptome-wide gene expression in peripheral blood leucocytes was measured using the Illumina Human HT-12 v4 expression BeadChip according to manufacturer’s protocol in two LOLIPOP cohorts of Indian Asian participants. Negative control-probes were used to calculate detection p-value threshold (p&lt;10<sup>-9</sup>, corresponding to p&lt;0.05 adjusted for the number of tests). Transcripts with call-rates &lt; 10% were removed. Expression intensities were read and processed using the R packages Illuminaio and Limma. Gene expression values were subjected to background correction, quantile normalisation and log<sub>2</sub> transformation. Linear regression was applied to the processed expression data adjusted for age, gender, RNA integrity number, RNA conversion batch, scanning batch, and array position. Residuals from the regression were obtained for the following genetic association analysis.</p><p id="P18">Genotyping of the two cohorts was carried out using Illumina OmniExpress (N=612) or Illumina OmniExpressExome (N=204) whole-genome arrays. Standard quality control of GWAS was applied and genotype imputation was performed using the 1000 Genomes cosmopolitan reference panel as described above. Association between the residuals of the gene expression and the sentinel SNPs (rs71640036 vs <em>KLKB1</em> and rs2545801 vs <em>F12</em>) was carried out with Quicktest separately in the two datasets, adjusted for age, gender, and PCs 1-5. METAL was used to combine the effects of both cohorts by inverse-variance meta-analysis. Association results were corrected for genomic inflation factors before and after meta-analysis.</p><p id="P19" class="p p-last">Finally we examined the association of our sentinel or proxy SNPs with gene expression using published data for liver and other tissues (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970878">27</a>–<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970844">29</a>), using the Genotype-Tissue Expression (GTEx) data (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970852">30</a>).</p></div><div id="S9" class="sec sec-last"><h3 id="S9title">Association with cardiovascular and metabolic traits</h3><p id="P20" class="p p-first-last">We examined the relationship of the sentinel SNPs influencing L-arginine concentrations with blood pressure, cardiovascular disease and metabolic traits using public databases and the NHGRI GWAS Catalog (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970830">31</a>). Where the sentinel SNP was not available in the respective dataset, we used the best available proxy SNP, selected based on LD (r<sup>2</sup>&gt;0.9).</p></div></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">Results</h2><div id="S11" class="sec sec-first"><h3 id="S11title">Characteristics of participants</h3><p id="P21" class="p p-first-last">The characteristics of the discovery and replication study samples are described in <a href="/pmc/articles/PMC6215702/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">S2</a>, respectively. Serum L-arginine levels are higher among Indian Asians than Europeans (p=0.01). Indian Asians also have higher rates of coronary heart disease and type 2 diabetes, and higher measurements of waist-hip ratio and glycemic traits (fasting glucose, fasting insulin, and HbA1c), and lower levels of total and HDL cholesterol, compared to Europeans (p&lt;0.05, <a href="/pmc/articles/PMC6215702/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p></div><div id="S12" class="sec"><h3 id="S12title">Genome-wide association in the discovery study</h3><p id="P22" class="p p-first-last">The LOLIPOP GWAS genomic inflation factors were 1.008 for Europeans, 1.004 for Indian Asians, and 1.002 for meta-analysis in the discovery study. Quantile-quantile (QQ) plots show evidence for association between genetic variation and serum L-arginine levels in both ethnic groups and in combined analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S1</a>). Twenty-four SNPs in the European samples and fifty-nine SNPs in the Indian Asian samples reach genome-wide significance for association with serum L-arginine levels (p&lt;5×10<sup>-8</sup>, <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2</a>), all on chromosome 4. In combined analysis, 82 SNPs are associated with L-arginine levels, comprising 77 SNPs at the <em>KLKB1</em> locus (kallikrein B, plasma (Fletcher Factor) 1) on chromosome 4, and 5 SNPs at the <em>F12</em> locus (Coagulation Factor XII (Hageman Factor)) on chromosome 5 (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S3</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2</a>).</p></div><div id="S13" class="sec"><h3 id="S13title">KLKB1 locus</h3><p id="P23" class="p p-first">The SNP with the lowest p-value (rs71640036, p=9.7×10<sup>-25</sup>, the sentinel SNP) is an intron variant in the <em>KLKB1</em> gene (<a href="/pmc/articles/PMC6215702/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Tables 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">S3</a>). The G allele of rs71640036 is associated with decreased serum L-arginine levels at a rate 10 μmol/L per allele copy (<a href="/pmc/articles/PMC6215702/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The sentinel SNP is highly significantly associated with serum L-arginine levels in both ethnic groups and there is no significant heterogeneity between the two groups (<a href="/pmc/articles/PMC6215702/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Tables 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">S3</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p4">Association of sentinel SNPs identified with serum or plasma L-arginine levels in genome-wide association.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140037236063328" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP (EA/AA) / Stage</th><th rowspan="1" colspan="1" valign="top" align="center">Data</th><th rowspan="1" colspan="1" valign="top" align="center">EAF</th><th rowspan="1" colspan="1" valign="top" align="center">Effect (SE)</th><th rowspan="1" colspan="1" valign="top" align="center">P</th><th rowspan="1" colspan="1" valign="top" align="center">N</th><th rowspan="1" colspan="1" valign="top" align="center">P_het</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>rs71640036 (G/T)</strong></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="3" colspan="1" valign="top" align="right"><strong>Discovery</strong></td><td rowspan="1" colspan="1" valign="top" align="left">LOLIPOP European</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td><td rowspan="1" colspan="1" valign="top" align="center">-9.1 (1.5)</td><td rowspan="1" colspan="1" valign="top" align="center">1.4×10<sup>-09</sup></td><td rowspan="1" colspan="1" valign="top" align="center">901</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LOLIPOP Indian Asian</td><td rowspan="1" colspan="1" valign="top" align="center">0.61</td><td rowspan="1" colspan="1" valign="top" align="center">-11.2 (1.3)</td><td rowspan="1" colspan="1" valign="top" align="center">4.7×10<sup>-17</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1,394</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Discovery combined</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>-</strong></td><td rowspan="1" colspan="1" valign="top" align="center">-10.3 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">9.7×10<sup>-25</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2,295</td><td rowspan="1" colspan="1" valign="top" align="center">3.1×10<sup>-01</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="4" colspan="1" valign="top" align="right"><strong>Replication</strong></td><td rowspan="1" colspan="1" valign="top" align="left">EPIC-Norfolk-cases</td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="center">-0.87 (1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">4.8×10<sup>-01</sup></td><td rowspan="1" colspan="1" valign="top" align="center">450</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">EPIC-Norfolk-controls</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td><td rowspan="1" colspan="1" valign="top" align="center">-1.51 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">1.4×10<sup>-01</sup></td><td rowspan="1" colspan="1" valign="top" align="center">450</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">FHS</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td><td rowspan="1" colspan="1" valign="top" align="center">-1.4 (0.5)</td><td rowspan="1" colspan="1" valign="top" align="center">9.5×10<sup>-03</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2,981</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Replication combined</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-1.4 (0.5)</td><td rowspan="1" colspan="1" valign="top" align="center">2.4×10<sup>-03</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3,881</td><td rowspan="1" colspan="1" valign="top" align="center">8.3×10<sup>-01</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="right"><strong>All combined</strong></td><td rowspan="1" colspan="1" valign="top" align="left">WZ method</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">3.8×10<sup>-18</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6,176</td><td rowspan="1" colspan="1" valign="top" align="center">7.9×10<sup>-09</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">RE method</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">1.6×10-<sup>22</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6,176</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>rs2545801 (T/C)</strong></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="3" colspan="1" valign="top" align="right"><strong>Discovery</strong></td><td rowspan="1" colspan="1" valign="top" align="left">LOLIPOP European</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">-7.9 (1.8)</td><td rowspan="1" colspan="1" valign="top" align="center">9.0×10<sup>-06</sup></td><td rowspan="1" colspan="1" valign="top" align="center">901</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LOLIPOP Indian Asian</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">-6.8 (1.3)</td><td rowspan="1" colspan="1" valign="top" align="center">1.3×10<sup>-07</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1,394</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Discovery combined</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-7.1 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">6.8×10<sup>-12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2,295</td><td rowspan="1" colspan="1" valign="top" align="center">6.0×10<sup>-01</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="4" colspan="1" valign="top" align="right"><strong>Replication</strong></td><td rowspan="1" colspan="1" valign="top" align="left">EPIC-Norfolk-cases</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">-3.6 (1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">4.0×10<sup>-03</sup></td><td rowspan="1" colspan="1" valign="top" align="center">457</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">EPIC-Norfolk-controls</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">0.7 (1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">6.0×10<sup>-01</sup></td><td rowspan="1" colspan="1" valign="top" align="center">457</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">FHS</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">-4.3 (1.5)</td><td rowspan="1" colspan="1" valign="top" align="center">3.4×10<sup>-03</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2,981</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Replication combined</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-2.2 (0.7)</td><td rowspan="1" colspan="1" valign="top" align="center">3.5×10<sup>-03</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3,895</td><td rowspan="1" colspan="1" valign="top" align="center">1.3×10<sup>-01</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="2" colspan="1" valign="top" align="right"><strong>All combined</strong></td><td rowspan="1" colspan="1" valign="top" align="left">WZ method</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">6.3×10<sup>-12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6,190</td><td rowspan="1" colspan="1" valign="top" align="center">1.0×10<sup>-02</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">RE method</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">9.8×10<sup>-13</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6,190</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr></tbody></table></div><div id="largeobj_idm140037236063328" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6215702/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><p id="P51" class="p p-first-last">Footnote: EA, effect allele; AA, alternate allele; EAF, effect allele frequency; Effect, the change of L-arginine concentration per copy of the effect allele (μmol/L); SE, standard error; R<sup>2</sup>, proportion of variance explained by the respective SNP; R<sup>2</sup> for combined effect of rs71640036 and rs2545801 was 0.075 for European and 0.066 for Indian; EPIC-Norfolk, the European Prospective Investigation into Cancer, Norfolk; FHS, Framingham Heart Study; WZ, weighted Z-score method for meta-analysis; RE, Random effects model for meta-analysis accounting for heterogeneity. Imputation quality scores for both sentinel SNPs in discovery and replication datasets were &gt; 0.9, except for rs2545801 in FHS (oevar_imp = 0.25). Removing FHS from meta-analyses of rs2545801, there was a weak association in the replication cohort (EPIC-Norfolk study with adjustment for age, gender and stroke status, p = 5.1×10<sup>-2</sup>) and a significant association in combined discovery and replication analysis (WZ method, p = 6.1×10<sup>-12</sup>). In FHS, SNP rs3733402 was used as a proxy to rs71640036. Serum samples were used in the discovery studies and plasma samples were used in the replication studies.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p5">Serum L-arginine concentrations (mean [SD], μmol/L) in relation to the genotypes of sentinel SNPs.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140037241186800" class="xtable"><table frame="above" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="left"></th><th colspan="2" rowspan="1" align="left"><span class="underline">Europeans</span></th><th colspan="2" rowspan="1" align="left"><span class="underline">Indian Asians</span></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Genotype</th><th rowspan="1" colspan="1" valign="top" align="center">%</th><th rowspan="1" colspan="1" valign="top" align="center">L-arginine</th><th rowspan="1" colspan="1" valign="top" align="center">%</th><th rowspan="1" colspan="1" valign="top" align="center">L-arginine</th></tr><tr><th colspan="6" rowspan="1" align="left"><hr></th></tr></thead><tbody><tr><td rowspan="3" colspan="1" valign="top" align="left"><strong>rs71640036</strong></td><td rowspan="1" colspan="1" valign="top" align="center">TT</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">140.1 (31.8)</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">149.6 (31.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">TG</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td><td rowspan="1" colspan="1" valign="top" align="center">134.3 (31.3)</td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="center">137.7 (34.8)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">GG</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">121.2 (29.7)</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">126.8 (30.6)</td></tr><tr><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="3" colspan="1" valign="top" align="left"><strong>rs2545801</strong></td><td rowspan="1" colspan="1" valign="top" align="center">CC</td><td rowspan="1" colspan="1" valign="top" align="center">0.58</td><td rowspan="1" colspan="1" valign="top" align="center">134.8 (31.8)</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">141.1 (34.1)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">TC</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">129.6 (30.9)</td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="center">134.5 (33.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">TT</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">116.8 (23.7)</td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center">126.7 (31.1)</td></tr></tbody></table></div><div id="largeobj_idm140037241186800" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6215702/table/T3/?report=objectonly">Open in a separate window</a></div></div><p id="P24" class="p p-last">A missense non-synonymous SNP rs3733402 (Asn143Ser), which is 3,086 base pairs upstream of the sentinel SNP is also highly significantly associated with serum L-arginine levels (p=1.0×10<sup>-23</sup>, <a href="#SD1" rid="SD1" class=" supplementary-material">Table S3</a>). SNP rs3733402 is in high LD with the sentinel SNP (r<sup>2</sup> = 0.964 in Europeans and 0.981 in Indian Asians). The G allele of rs3733402 is associated with decreased serum L-arginine levels. Conditional analyses on either the sentinel SNP rs71640036 or rs3733402 showed that none of the other SNPs are significantly associated with serum L-arginine levels (p&gt;0.05 after Bonferroni correction) in this region (±500 kb from the sentinel SNP, covering the <em>KLKB1</em>, <em>CYP4V2</em>, <em>F11</em> and other nearby genes, data not shown).</p></div><div id="S14" class="sec"><h3 id="S14title">F12 locus</h3><p id="P25" class="p p-first-last">The five SNPs significantly associated with serum L-arginine levels on chromosome 5 are in or near the <em>F12</em> gene. The sentinel SNP rs2545801 (p=6.8×10<sup>-12</sup>) is intergenic, but is in high LD with rs1801020 (r<sup>2</sup> = 0.993 in Europeans and 0.996 in Indian Asians) in the highly conserved 5’ untranslated (promoter) region of <em>F12</em> gene, itself strongly associated with L-arginine levels (p=1.2×10<sup>-11</sup>, <a href="#SD1" rid="SD1" class=" supplementary-material">Table S3</a>). The minor allele T of the sentinel SNP is associated with decreased serum L-arginine levels (7 μmol/L per allele copy, <a href="/pmc/articles/PMC6215702/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). There is no significant heterogeneity between the two ethnic groups. Conditional analyses on the sentinel SNP rs2545801 showed that there are no secondary associations with serum L-arginine levels in this region (±500 kb from the sentinel SNP, data not shown). Together the sentinel SNPs from both <em>KLKB1</em> and <em>F12</em> loci explain approximately 7% of the total variance of serum L-arginine after adjusting for age, gender, and PCs 1-5 (<a href="/pmc/articles/PMC6215702/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p></div><div id="S15" class="sec"><h3 id="S15title">Replication studies</h3><p id="P26" class="p p-first-last">Combined analyses of the replication cohorts showed significant associations at both the <em>KLKB1</em> and <em>F12</em> loci (p&lt;0.004, <a href="/pmc/articles/PMC6215702/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Meta-analysis combining discovery and replication studies also showed strong associations with circulating L-arginine levels (<a href="/pmc/articles/PMC6215702/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p></div><div id="S16" class="sec"><h3 id="S16title">Suggestive associations and other analyses</h3><p id="P27" class="p p-first">In combined analysis of the European and Indian Asian samples in the discovery study, 129 SNPs showed p-values &lt;1×10<sup>-5</sup> but &gt;5×10<sup>-8</sup> with consistent direction of effects in both ethnic groups (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S4</a>). Two intron SNPs in <em>ARG1</em> gene and 7 SNPs in the nearby <em>MED23</em> gene on chromosome 6 are associated with serum L-arginine levels (lowest p=5 ×10<sup>-7</sup>, <a href="#SD1" rid="SD1" class=" supplementary-material">Table S4</a>). Other regions with suggestive associations are in non-gene regions of chromosomes 2 and 11, <em>SOX6</em> (11p15.3), <em>PRH1-PRR4</em> (12p), <em>GPR133</em> (12q24.33), <em>HS3ST4</em> (16p11.2), <em>RP11-638</em> (18q22.1), and <em>TSHZ2</em> (20q13.2) (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S4</a>).</p><p id="P28" class="p p-last">We separately performed association analyses with dominant or recessive models; no additional loci reached genome-wide significance. Gene-gene interaction analysis did not show any synergistic effects for the SNPs between the <em>KLKB1</em> and <em>F12</em> genes beyond their additive effect (data not shown).</p></div><div id="S17" class="sec"><h3 id="S17title">Gene expression studies</h3><p id="P29" class="p p-first-last">SNP rs2545801 is associated with expression of <em>F12</em> (p=0.04 after Bonferroni correction), and SNP rs71640036 is associated with expression of <em>CYP4V2</em> (p=0.01, underlying mechanisms unknown) in our peripheral blood leucocyte samples (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S5</a>). In contrast there was no significant association of rs71640036 with expression of <em>KLKB1</em> (p=0.53). The sentinel SNPs were not significantly associated with expression levels of any other genes within the regions (±1Mb). Both <em>KLKB1</em> and <em>F12</em> genes are predominantly expressed in liver (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970876">30</a>). SNP rs2731672, which is in high LD (r<sup>2</sup> = 0.994 in Europeans and 0.987 in Indian Asians) with the sentinel SNP rs2545801 at the <em>F12</em> locus is strongly associated with <em>F12</em> gene expression in liver (p=1.2×10<sup>-23</sup>) (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970886">27</a>).</p></div><div id="S18" class="sec"><h3 id="S18title">Association of sentinel SNPs with phenotypic traits</h3><p id="P30" class="p p-first-last">Published studies show that the sentinel SNPs at the <em>KLKB1</em> and <em>F12</em> loci are also strongly associated with other phenotypic traits including levels of phenylalanine (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970863">32</a>, <a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970842">33</a>), des-arg(<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970860">9</a>) bradykinin (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970839">12</a>, <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970866">15</a>), active plasma renin (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970843">34</a>), midregional-proadrenomedullin (MR-pro-ADM) and C-terminal-pro-endothelin-1 (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970836">35</a>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S6</a>). In addition, the <em>KLKB1</em> SNPs are associated with histidine (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970855">32</a>), leucylphenylalanine (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970831">14</a>), HXGXA (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970879">14</a>), plasma B-Type Natriuretic Peptide (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970859">36</a>) and ratios of serum amino acid levels (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970882">32</a>), while <em>F12</em> SNPs are highly associated with blood FXII antigen levels (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970845">37</a>) and prolonged activated partial thromboplastin time (aPTT) (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970829">38</a>) and plasma renin activity (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970872">39</a>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S6</a>).</p></div><div id="S19" class="sec sec-last"><h3 id="S19title">Association of sentinel SNPs with cardiovascular disease</h3><p id="P31" class="p p-first-last">SNP rs71640036 at the <em>KLKB1</em> locus is associated with both systolic and diastolic blood pressure amongst 134,773 participants of UK Biobank (p=9x10<sup>-5</sup> and p=8x10<sup>-3</sup> respectively, <a href="/pmc/articles/PMC6215702/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). SNP rs2545801 at the <em>F12</em> locus is also associated with systolic blood pressure (p=8x10<sup>-3</sup>) but not with diastolic blood pressure. There were no associations of rs71640036 or rs2545801 with coronary heart disease or hsCRP, after Bonferroni correction for multiple testing (<a href="/pmc/articles/PMC6215702/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Tables 4</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">S6</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p6">Association of sentinel or proxy SNPs with cardiovascular disease.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140037239298704" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">EA/AA</th><th rowspan="1" colspan="1" valign="top" align="center">Trait</th><th rowspan="1" colspan="1" valign="top" align="center">N</th><th rowspan="1" colspan="1" valign="top" align="center">Effect</th><th rowspan="1" colspan="1" valign="top" align="center">P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong><em><span class="underline">KLKB1</span></em></strong></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="3" colspan="1" valign="top" align="left">rs71640036</td><td rowspan="3" colspan="1" valign="top" align="center">G/T</td><td rowspan="1" colspan="1" valign="top" align="center">Coronary heart disease (<a href="#R48" rid="R48" class=" bibr popnode">48</a>)</td><td rowspan="1" colspan="1" valign="top" align="center">185,000</td><td rowspan="1" colspan="1" valign="top" align="center">0.023</td><td rowspan="1" colspan="1" valign="top" align="center">2×10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Diastolic blood pressure<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="center">134,773</td><td rowspan="1" colspan="1" valign="top" align="center">-0.152</td><td rowspan="1" colspan="1" valign="top" align="center">9×10<sup>-5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Systolic blood pressure<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="center">134,773</td><td rowspan="1" colspan="1" valign="top" align="center">-0.180</td><td rowspan="1" colspan="1" valign="top" align="center">8×10<sup>-3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong><em><span class="underline">F12</span></em></strong></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="3" colspan="1" valign="top" align="left">rs2545801</td><td rowspan="3" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">Coronary heart disease (<a href="#R48" rid="R48" class=" bibr popnode">48</a>)</td><td rowspan="1" colspan="1" valign="top" align="center">185,000</td><td rowspan="1" colspan="1" valign="top" align="center">0.017</td><td rowspan="1" colspan="1" valign="top" align="center">1×10<sup>-1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Diastolic blood pressure<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="center">134,773</td><td rowspan="1" colspan="1" valign="top" align="center">0.067</td><td rowspan="1" colspan="1" valign="top" align="center">1×10<sup>-1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Systolic blood pressure<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="center">134,773</td><td rowspan="1" colspan="1" valign="top" align="center">0.201</td><td rowspan="1" colspan="1" valign="top" align="center">7×10<sup>-3</sup></td></tr></tbody></table></div><div id="largeobj_idm140037239298704" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6215702/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><p id="P52" class="p p-first-last">Footnote: EA, effect allele (minor allele, associated with reduced serum L-arginine levels); AA, alternate allele; Logistic and linear regressions (for coronary heart disease and blood pressure, respectively) under the genetic model of additive effect were applied. Effect was calculated as the logarithm of the odds ratio for coronary heart disease, or change of blood pressure (mmHg) per copy of the effect allele. *Data from UK Biobank. The critical p-value for significance after Bonferroni correction for six tests is 8×10<sup>-3</sup> at the familywise error rate of 0.05.</p></div></div></div></div></div><div id="S20" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S20title">Discussion</h2><p id="P32" class="p p-first">We show that common genetic variation at the <em>KLKB1</em> and <em>F12</em> loci is an important determinant of serum L-arginine concentrations. <em>KLKB1</em> and <em>F12</em> both encode components of the kinin system. <em>KLKB1</em> (4q35.2) encodes prekallikrein, which is a precursor of the serine protease plasma kallikrein. <em>F12</em> (5q35.3) encodes the coagulation Factor XII (FXII), a serine protease involved in coagulation activation through the intrinsic pathway. Kallikrein and FXII are closely linked biologically and activate each other in a reciprocal fashion.</p><p id="P33">We found that nonsynonymous SNP rs3733402 in the <em>KLKB1</em> gene, which is in high LD with the sentinel SNP rs71640036, is associated with decreased serum L-arginine levels. This SNP leads to the replacement of asparagine with serine at codon 143 in exome 5 of <em>KLKB1</em> gene, located in apple domain A2 of plasma kallikrein. Previous studies show that this non-synonymous SNP reduces the binding activity of apple domain A2 to circulating high molecular weight kininogen (HMWK), and is associated with low levels of plasma prekallikrein and bradykinin metabolites (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970862">40</a>).</p><p id="P34">The minor A allele of SNP rs1801020 in the promoter region of the <em>F12</em> gene, which was nearly in complete LD with the sentinel SNP rs2545801 and rs2731672, is associated with reduced serum L-arginine levels in our study, decreased plasma FXII concentrations (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970880">37</a>, <a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970848">41</a>, <a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970865">42</a>) and prolonged aPTT (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970861">38</a>). The SNP is located 4 bases upstream of the translation initiation codon of factor XII. Experiments using reporter constructs suggest that the altered upstream open reading frame (uORF) through the minor T allele is responsible for &gt;50% reduction of FXII protein levels and &lt;30% reduction of mRNA levels (<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970873">43</a>). The eQTL data showed that these genetic variants are associated with reduced <em>F12</em> gene expression in liver and blood leucocytes.</p><p id="P35">The mechanisms linking genetic variants in <em>KLKB1</em> and <em>F12</em> to serum L-arginine concentrations remain to be determined. We note that both <em>KLKB1</em> and <em>F12</em> are involved in bradykinin release from HMWK. Bradykinin is a nonapeptide with the amino acid sequence Arg-Pro-Pro-Gly-Phe-Ser-Pro-Phe-Arg and is rapidly degraded once it has been generated in cardiovascular system. Aminopeptidase P (APP) cleaves positions 1-2, while carboxypeptidase N (CPN) cleaves positions 8-9, both enzymatic reactions lead to the release of L-arginine (<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970841">44</a>–<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970846">46</a>). These observations raise the possibility that genetic variation at <em>KLKB1</em> and <em>F12</em> may influence blood L-arginine levels through an effect on bradykinin release and degradation.</p><p id="P36">Although the role of nitric oxide in human physiology is well described, the relationship of circulating L-arginine metabolism with human disease is uncertain. We found that SNP rs71640036 in the <em>KLKB1</em> gene, which is strongly associated with lower L-arginine levels, is also associated with lower blood pressure and the <em>F12</em> variant is associated with higher systolic blood pressure. These findings raise the possibility that L-arginine may be involved in blood pressure regulation. However, since plasma kallikrein and FXII participate in multiple key signalling pathways (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970849">35</a>, <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970884">37</a>, <a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970870">47</a>), the association between their genetic variants and blood pressure through L-arginine needs further investigation.</p><p id="P37">There is increasing evidence showing associations between variants in the <em>KLKB1</em> and <em>F12</em> loci and cardiovascular biomarkers (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970887">35</a>, <a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970858">36</a>). The sentinel SNPs, however, showed only borderline or non-significant associations with CAD in the recent large-scale CARDIoGRAMplusC4D study (p=0.02 for the <em>KLKB1</em> variant) (<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970881">48</a>). This reflects the complex and multifactorial aetiology of CAD and these loci may contribute to the risk of CAD through intermediate traits, such as blood pressure and thrombosis. The same applies to hsCRP, a marker of systemic inflammation. Given that hsCRP is highly associated with serum L-arginine concentrations in our data and the <em>KLKB1</em>/<em>F12</em> genes are involved in inflammation, further studies are required to explore the relation between L-arginine and hsCRP.</p><p id="P38">The kinin system also plays an important role in the contact activation of the intrinsic pathway in coagulation and fibrinolysis. A recent study indicated that the intrinsic pathway-dependent thrombin generation may be a risk factor for ischemic stroke and the <em>F12</em> variant is associated with thrombin generation (<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_734970883">49</a>). Factor XI (FXI), encoded by the <em>F11</em> gene, is a key factor in the intrinsic pathway and consistently associated with thrombosis risk. The <em>KLKB1</em> and <em>F11</em> genes are closely located but with moderate LD (r<sup>2</sup> up to 0.3). Our study revealed no association of <em>F11</em> variants with serum L-arginine levels after exclusion of the effects from the <em>KLKB1</em> locus. How the L-arginine metabolism affects thrombosis risk remains unclear.</p><p id="P39" class="p p-last">In conclusion, we have identified genetic variation in the <em>KLKB1</em> and <em>F12</em> genes as important determinants of serum levels of L-arginine, and provide potential evidence for their role in blood pressure regulation. Our findings provide new insight into the pathways influencing metabolism of L-arginine, the essential substrate in the synthesis of NO, and may help resolve the understanding of the pathways linking L-arginine with endothelial function and cardiovascular health.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary information</h4><div class="sup-box half_rhythm" id="d36e1082"><a href="/pmc/articles/PMC6215702/bin/NIHMS80283-supplement-Supplementary_information.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(185K, docx)</sup></div></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">Acknowledgements</h2><div class="sec"><p id="__p1">The LOLIPOP study is supported by the National Institute for Health Research (NIHR) Comprehensive Biomedical Research Centre Imperial College Healthcare NHS Trust, the British Heart Foundation (SP/04/002), the Medical Research Council (G0601966, G0700931), the Wellcome Trust (084723/Z/08/Z), the NIHR (RP-PG-0407-10371), European Union FP7 (EpiMigrant, 279143), and Action on Hearing Loss (G51). Fredrik Jernerén is supported by the Norwegian research Council. This work was also supported by the Norwegian research Council (to Helga Refsum) to fund the experiments. We thank the participants and research staff who made the study possible, and in particular the staff at University of Oxford (Cynthia Prendergast, Carol Johnston, and Maria Valdivia-Garcia) for performing the amino acid measurements. The Framingham Heart Study was supported by the National Heart, Lung and Blood Institute’s Framingham Heart Study (Contract No. N01-HC-25195) and its contract with Affymetrix Inc. for genotyping services (Contract No. N02-HL-6-4278). EPIC-Norfolk is supported by the Medical Research Council programme grants (G0401527, G1000143) and Cancer Research UK programme grant (C864/A8257). The authors would like to acknowledge the contribution of the staff and participants of the Framingham Offspring Cohort Study and the EPIC-Norfolk Study. The work was carried out in part at the NIHR/Wellcome Trust Imperial Clinical Research Facility. We thank Dr Abbas Dehghan and the CHARGE Inflammation working group for providing the hsCRP and SNP association results. This research has been conducted using the UK Biobank Resource. The views expressed are those of the author(s) and not necessarily those of the Imperial College Healthcare NHS Trust, the NIHR or the Department of Health.</p><p id="__p2">Financial support: the British Heart Foundation (SP/04/002), the Medical Research Council (G0601966, G0700931, G0401527, G1000143), the Wellcome Trust (084723/Z/08/Z), the NIHR (RP-PG-0407-10371), European Union FP7 (EpiMigrant, 279143), Action on Hearing Loss (G51), Cancer Research UK (C864/A8257), the Norwegian research Council, and the National Heart, Lung and Blood Institute’s Framingham Heart Study (Contract No. N01-HC-25195) and its contract with Affymetrix Inc. for genotyping services (Contract No. N02-HL-6-4278).</p></div></div><div id="idm140037240175424" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140037240175424title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P40" class="p p-first"><strong>URLs</strong></p><p id="P41" class="p">Guideline on bioanalytical method validation, European Medicines Agency, <a href="http://www.ema.europa.eu/docs/en_GB/document_library/Scientific_guideline/2011/08/WC500109686.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ema.europa.eu/docs/en_GB/document_library/Scientific_guideline/2011/08/WC500109686.pdf</a></p><p id="P42" class="p">R-project, <a href="https://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.r-project.org/</a></p><p id="P43" class="p">EIGENSOFT, <a href="http://genetics.med.harvard.edu/reich/Reich_Lab/Software.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genetics.med.harvard.edu/reich/Reich_Lab/Software.html</a></p><p id="P44" class="p">IMPUTE2, <a href="https://mathgen.stats.ox.ac.uk/impute/impute_v2.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/impute/impute_v2.html</a></p><p id="P45" class="p">SNPTEST, <a href="https://mathgen.stats.ox.ac.uk/genetics_software/snptest/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/genetics_software/snptest/snptest.html</a></p><p id="P46" class="p">METAL, <a href="http://genome.sph.umich.edu/wiki/METAL_Documentation" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genome.sph.umich.edu/wiki/METAL_Documentation</a></p><p id="P47" class="p">LOCUSROOM, <a href="http://locuszoom.sph.umich.edu/locuszoom/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://locuszoom.sph.umich.edu/locuszoom/</a></p><p id="P48" class="p">NHGRI GWAS Catalog, <a href="http://www.ebi.ac.uk/gwas/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ebi.ac.uk/gwas/</a></p><p id="P49" class="p p-last">GTEx Portal, <a href="http://www.broadinstitute.org/gtex/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/gtex/</a></p><p></p></div></div><div id="idm140037246518048" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140037246518048title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Popolo A, Adesso S, Pinto A, et al.  L-Arginine and its metabolites in kidney and cardiovascular disease. <span><span class="ref-journal">Amino Acids. </span>2014;<span class="ref-vol">46</span>:2271–2286.</span> [<a href="/pubmed/25161088" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Amino+Acids&amp;title=L-Arginine+and+its+metabolites+in+kidney+and+cardiovascular+disease&amp;author=A+Popolo&amp;author=S+Adesso&amp;author=A+Pinto&amp;volume=46&amp;publication_year=2014&amp;pages=2271-2286&amp;pmid=25161088&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Boger RH. The pharmacodynamics of L-arginine. <span><span class="ref-journal">Altern Ther Health Med. </span>2014;<span class="ref-vol">20</span>:48–54.</span> [<a href="/pubmed/24755570" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Altern+Ther+Health+Med&amp;title=The+pharmacodynamics+of+L-arginine&amp;author=RH+Boger&amp;volume=20&amp;publication_year=2014&amp;pages=48-54&amp;pmid=24755570&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Lorin J, Zeller M, Guilland JC, et al.  Arginine and nitric oxide synthase: regulatory mechanisms and cardiovascular aspects. <span><span class="ref-journal">Mol Nutr Food Res. </span>2014;<span class="ref-vol">58</span>:101–116.</span> [<a href="/pubmed/23740826" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Nutr+Food+Res&amp;title=Arginine+and+nitric+oxide+synthase:+regulatory+mechanisms+and+cardiovascular+aspects&amp;author=J+Lorin&amp;author=M+Zeller&amp;author=JC+Guilland&amp;volume=58&amp;publication_year=2014&amp;pages=101-116&amp;pmid=23740826&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Luiking YC, Ten Have GA, Wolfe RR, et al.  Arginine de novo and nitric oxide production in disease states. <span><span class="ref-journal">Am J Physiol Endocrinol Metab. </span>2012;<span class="ref-vol">303</span>:E1177–1189.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3517635/">PMC free article</a>]</span> [<a href="/pubmed/23011059" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol+Endocrinol+Metab&amp;title=Arginine+de+novo+and+nitric+oxide+production+in+disease+states&amp;author=YC+Luiking&amp;author=GA+Ten+Have&amp;author=RR+Wolfe&amp;volume=303&amp;publication_year=2012&amp;pages=E1177-1189&amp;pmid=23011059&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Tousoulis D, Kampoli AM, Tentolouris C, et al.  The role of nitric oxide on endothelial function. <span><span class="ref-journal">Curr Vasc Pharmacol. </span>2012;<span class="ref-vol">10</span>:4–18.</span> [<a href="/pubmed/22112350" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Vasc+Pharmacol&amp;title=The+role+of+nitric+oxide+on+endothelial+function&amp;author=D+Tousoulis&amp;author=AM+Kampoli&amp;author=C+Tentolouris&amp;volume=10&amp;publication_year=2012&amp;pages=4-18&amp;pmid=22112350&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Morris SM., Jr Arginases and arginine deficiency syndromes. <span><span class="ref-journal">Curr Opin Clin Nutr Metab Care. </span>2012;<span class="ref-vol">15</span>:64–70.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3268370/">PMC free article</a>]</span> [<a href="/pubmed/22037011" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Clin+Nutr+Metab+Care&amp;title=Arginases+and+arginine+deficiency+syndromes&amp;author=SM+Morris&amp;volume=15&amp;publication_year=2012&amp;pages=64-70&amp;pmid=22037011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Gokce N. L-arginine and hypertension. <span><span class="ref-journal">J Nutr. </span>2004;<span class="ref-vol">134</span>:2807S–2811S.</span> discussion 2818S–2819S. [<a href="/pubmed/15465790" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Nutr&amp;title=L-arginine+and+hypertension&amp;author=N+Gokce&amp;volume=134&amp;publication_year=2004&amp;pages=2807S-2811S&amp;pmid=15465790&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Dong JY, Qin LQ, Zhang Z, et al.  Effect of oral L-arginine supplementation on blood pressure: a meta-analysis of randomized, double-blind, placebo-controlled trials. <span><span class="ref-journal">Am Heart J. </span>2011;<span class="ref-vol">162</span>:959–965.</span> [<a href="/pubmed/22137067" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+Heart+J&amp;title=Effect+of+oral+L-arginine+supplementation+on+blood+pressure:+a+meta-analysis+of+randomized,+double-blind,+placebo-controlled+trials&amp;author=JY+Dong&amp;author=LQ+Qin&amp;author=Z+Zhang&amp;volume=162&amp;publication_year=2011&amp;pages=959-965&amp;pmid=22137067&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Luneburg N, Lieb W, Zeller T, et al.  Genome-wide association study of L-arginine and dimethylarginines reveals novel metabolic pathway for symmetric dimethylarginine. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2014;<span class="ref-vol">7</span>:864–872.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4797637/">PMC free article</a>]</span> [<a href="/pubmed/25245031" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Genome-wide+association+study+of+L-arginine+and+dimethylarginines+reveals+novel+metabolic+pathway+for+symmetric+dimethylarginine&amp;author=N+Luneburg&amp;author=W+Lieb&amp;author=T+Zeller&amp;volume=7&amp;publication_year=2014&amp;pages=864-872&amp;pmid=25245031&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Gieger C, Geistlinger L, Altmaier E, et al.  Genetics meets metabolomics: a genome-wide association study of metabolite profiles in human serum. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000282.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2581785/">PMC free article</a>]</span> [<a href="/pubmed/19043545" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genetics+meets+metabolomics:+a+genome-wide+association+study+of+metabolite+profiles+in+human+serum&amp;author=C+Gieger&amp;author=L+Geistlinger&amp;author=E+Altmaier&amp;volume=4&amp;publication_year=2008&amp;pages=e1000282&amp;pmid=19043545&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Illig T, Gieger C, Zhai G, et al.  A genome-wide perspective of genetic variation in human metabolism. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:137–141.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3773904/">PMC free article</a>]</span> [<a href="/pubmed/20037589" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+perspective+of+genetic+variation+in+human+metabolism&amp;author=T+Illig&amp;author=C+Gieger&amp;author=G+Zhai&amp;volume=42&amp;publication_year=2010&amp;pages=137-141&amp;pmid=20037589&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Suhre K, Shin SY, Petersen AK, et al.  Human metabolic individuality in biomedical and pharmaceutical research. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">477</span>:54–60.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3832838/">PMC free article</a>]</span> [<a href="/pubmed/21886157" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Human+metabolic+individuality+in+biomedical+and+pharmaceutical+research&amp;author=K+Suhre&amp;author=SY+Shin&amp;author=AK+Petersen&amp;volume=477&amp;publication_year=2011&amp;pages=54-60&amp;pmid=21886157&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Rhee EP, Ho JE, Chen MH, et al.  A genome-wide association study of the human metabolome in a community-based cohort. <span><span class="ref-journal">Cell Metab. </span>2013;<span class="ref-vol">18</span>:130–143.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3973158/">PMC free article</a>]</span> [<a href="/pubmed/23823483" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Metab&amp;title=A+genome-wide+association+study+of+the+human+metabolome+in+a+community-based+cohort&amp;author=EP+Rhee&amp;author=JE+Ho&amp;author=MH+Chen&amp;volume=18&amp;publication_year=2013&amp;pages=130-143&amp;pmid=23823483&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Yu B, Zheng Y, Alexander D, et al.  Genetic determinants influencing human serum metabolome among African Americans. <span><span class="ref-journal">PLoS Genet. </span>2014;<span class="ref-vol">10</span>:e1004212.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3952826/">PMC free article</a>]</span> [<a href="/pubmed/24625756" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genetic+determinants+influencing+human+serum+metabolome+among+African+Americans&amp;author=B+Yu&amp;author=Y+Zheng&amp;author=D+Alexander&amp;volume=10&amp;publication_year=2014&amp;pages=e1004212&amp;pmid=24625756&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Shin SY, Fauman EB, Petersen AK, et al.  An atlas of genetic influences on human blood metabolites. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:543–550.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4064254/">PMC free article</a>]</span> [<a href="/pubmed/24816252" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=An+atlas+of+genetic+influences+on+human+blood+metabolites&amp;author=SY+Shin&amp;author=EB+Fauman&amp;author=AK+Petersen&amp;volume=46&amp;publication_year=2014&amp;pages=543-550&amp;pmid=24816252&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Draisma HH, Pool R, Kobl M, et al.  Genome-wide association study identifies novel genetic variants contributing to variation in blood metabolite levels. <span><span class="ref-journal">Nat Commun. </span>2015;<span class="ref-vol">6</span>:7208.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4745136/">PMC free article</a>]</span> [<a href="/pubmed/26068415" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Genome-wide+association+study+identifies+novel+genetic+variants+contributing+to+variation+in+blood+metabolite+levels&amp;author=HH+Draisma&amp;author=R+Pool&amp;author=M+Kobl&amp;volume=6&amp;publication_year=2015&amp;pages=7208&amp;pmid=26068415&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Chambers JC, Zhang W, Zabaneh D, et al.  Common genetic variation near melatonin receptor MTNR1B contributes to raised plasma glucose and increased risk of type 2 diabetes among Indian Asians and European Caucasians. <span><span class="ref-journal">Diabetes. </span>2009;<span class="ref-vol">58</span>:2703–2708.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2768158/">PMC free article</a>]</span> [<a href="/pubmed/19651812" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Common+genetic+variation+near+melatonin+receptor+MTNR1B+contributes+to+raised+plasma+glucose+and+increased+risk+of+type+2+diabetes+among+Indian+Asians+and+European+Caucasians&amp;author=JC+Chambers&amp;author=W+Zhang&amp;author=D+Zabaneh&amp;volume=58&amp;publication_year=2009&amp;pages=2703-2708&amp;pmid=19651812&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Chambers JC, Elliott P, Zabaneh D, et al.  Common genetic variation near MC4R is associated with waist circumference and insulin resistance. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:716–718.</span> [<a href="/pubmed/18454146" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+genetic+variation+near+MC4R+is+associated+with+waist+circumference+and+insulin+resistance&amp;author=JC+Chambers&amp;author=P+Elliott&amp;author=D+Zabaneh&amp;volume=40&amp;publication_year=2008&amp;pages=716-718&amp;pmid=18454146&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Marchini J, Howie B, Myers S, et al.  A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Howie B, Marchini J, Stephens M. Genotype imputation with thousands of genomes. <span><span class="ref-journal">G3 (Bethesda) </span>2011;<span class="ref-vol">1</span>:457–470.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3276165/">PMC free article</a>]</span> [<a href="/pubmed/22384356" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=G3+(Bethesda)&amp;title=Genotype+imputation+with+thousands+of+genomes&amp;author=B+Howie&amp;author=J+Marchini&amp;author=M+Stephens&amp;volume=1&amp;publication_year=2011&amp;pages=457-470&amp;pmid=22384356&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Day N, Oakes S, Luben R, et al.  EPIC-Norfolk: study design and characteristics of the cohort. European Prospective Investigation of Cancer. <span><span class="ref-journal">Br J Cancer. </span>1999;<span class="ref-vol">80</span>(Suppl 1):95–103.</span> [<a href="/pubmed/10466767" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Cancer&amp;title=EPIC-Norfolk:+study+design+and+characteristics+of+the+cohort.+European+Prospective+Investigation+of+Cancer&amp;author=N+Day&amp;author=S+Oakes&amp;author=R+Luben&amp;volume=80&amp;issue=Suppl+1&amp;publication_year=1999&amp;pages=95-103&amp;pmid=10466767&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Kannel WB, Feinleib M, McNamara PM, et al.  An investigation of coronary heart disease in families. The Framingham offspring study. <span><span class="ref-journal">Am J Epidemiol. </span>1979;<span class="ref-vol">110</span>:281–290.</span> [<a href="/pubmed/474565" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=An+investigation+of+coronary+heart+disease+in+families.+The+Framingham+offspring+study&amp;author=WB+Kannel&amp;author=M+Feinleib&amp;author=PM+McNamara&amp;volume=110&amp;publication_year=1979&amp;pages=281-290&amp;pmid=474565&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Marchini J, Howie B. Genotype imputation for genome-wide association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2010;<span class="ref-vol">11</span>:499–511.</span> [<a href="/pubmed/20517342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genotype+imputation+for+genome-wide+association+studies&amp;author=J+Marchini&amp;author=B+Howie&amp;volume=11&amp;publication_year=2010&amp;pages=499-511&amp;pmid=20517342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Pruim RJ, Welch RP, Sanna S, et al.  LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–2337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Han B, Eskin E. Random-effects model aimed at discovering associations in meta-analysis of genome-wide association studies. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>:586–598.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3146723/">PMC free article</a>]</span> [<a href="/pubmed/21565292" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Random-effects+model+aimed+at+discovering+associations+in+meta-analysis+of+genome-wide+association+studies&amp;author=B+Han&amp;author=E+Eskin&amp;volume=88&amp;publication_year=2011&amp;pages=586-598&amp;pmid=21565292&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Schadt EE, Molony C, Chudin E, et al.  Mapping the genetic architecture of gene expression in human liver. <span><span class="ref-journal">PLoS Biol. </span>2008;<span class="ref-vol">6</span>:e107.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2365981/">PMC free article</a>]</span> [<a href="/pubmed/18462017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Mapping+the+genetic+architecture+of+gene+expression+in+human+liver&amp;author=EE+Schadt&amp;author=C+Molony&amp;author=E+Chudin&amp;volume=6&amp;publication_year=2008&amp;pages=e107&amp;pmid=18462017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Innocenti F, Cooper GM, Stanaway IB, et al.  Identification, replication, and functional fine-mapping of expression quantitative trait loci in primary human liver tissue. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002078.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3102751/">PMC free article</a>]</span> [<a href="/pubmed/21637794" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Identification,+replication,+and+functional+fine-mapping+of+expression+quantitative+trait+loci+in+primary+human+liver+tissue&amp;author=F+Innocenti&amp;author=GM+Cooper&amp;author=IB+Stanaway&amp;volume=7&amp;publication_year=2011&amp;pages=e1002078&amp;pmid=21637794&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Greenawalt DM, Dobrin R, Chudin E, et al.  A survey of the genetics of stomach, liver, and adipose gene expression from a morbidly obese cohort. <span><span class="ref-journal">Genome Res. </span>2011;<span class="ref-vol">21</span>:1008–1016.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3129244/">PMC free article</a>]</span> [<a href="/pubmed/21602305" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=A+survey+of+the+genetics+of+stomach,+liver,+and+adipose+gene+expression+from+a+morbidly+obese+cohort&amp;author=DM+Greenawalt&amp;author=R+Dobrin&amp;author=E+Chudin&amp;volume=21&amp;publication_year=2011&amp;pages=1008-1016&amp;pmid=21602305&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Consortium GT. The Genotype-Tissue Expression (GTEx) project. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:580–585.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4010069/">PMC free article</a>]</span> [<a href="/pubmed/23715323" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+Genotype-Tissue+Expression+(GTEx)+project&amp;author=GT+Consortium&amp;volume=45&amp;publication_year=2013&amp;pages=580-585&amp;pmid=23715323&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Welter D, MacArthur J, Morales J, et al.  The NHGRI GWAS Catalog, a curated resource of SNP-trait associations. <span><span class="ref-journal">Nucleic Acids Res. </span>2014;<span class="ref-vol">42</span>:D1001–1006.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3965119/">PMC free article</a>]</span> [<a href="/pubmed/24316577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+NHGRI+GWAS+Catalog,+a+curated+resource+of+SNP-trait+associations&amp;author=D+Welter&amp;author=J+MacArthur&amp;author=J+Morales&amp;volume=42&amp;publication_year=2014&amp;pages=D1001-1006&amp;pmid=24316577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Kettunen J, Tukiainen T, Sarin AP, et al.  Genome-wide association study identifies multiple loci influencing human serum metabolite levels. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:269–276.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3605033/">PMC free article</a>]</span> [<a href="/pubmed/22286219" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+multiple+loci+influencing+human+serum+metabolite+levels&amp;author=J+Kettunen&amp;author=T+Tukiainen&amp;author=AP+Sarin&amp;volume=44&amp;publication_year=2012&amp;pages=269-276&amp;pmid=22286219&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Inouye M, Ripatti S, Kettunen J, et al.  Novel Loci for metabolic networks and multi-tissue expression studies reveal genes for atherosclerosis. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002907.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3420921/">PMC free article</a>]</span> [<a href="/pubmed/22916037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Novel+Loci+for+metabolic+networks+and+multi-tissue+expression+studies+reveal+genes+for+atherosclerosis&amp;author=M+Inouye&amp;author=S+Ripatti&amp;author=J+Kettunen&amp;volume=8&amp;publication_year=2012&amp;pages=e1002907&amp;pmid=22916037&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Biswas N, Maihofer AX, Mir SA, et al.  Polymorphisms at the F12 and KLKB1 loci have significant trait association with activation of the renin-angiotensin system. <span><span class="ref-journal">BMC Med Genet. </span>2016;<span class="ref-vol">17</span>:21.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4788869/">PMC free article</a>]</span> [<a href="/pubmed/26969407" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genet&amp;title=Polymorphisms+at+the+F12+and+KLKB1+loci+have+significant+trait+association+with+activation+of+the+renin-angiotensin+system&amp;author=N+Biswas&amp;author=AX+Maihofer&amp;author=SA+Mir&amp;volume=17&amp;publication_year=2016&amp;pages=21&amp;pmid=26969407&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Verweij N, Mahmud H, Mateo Leach I, et al.  Genome-wide association study on plasma levels of midregional-proadrenomedullin and C-terminal-pro-endothelin-1. <span><span class="ref-journal">Hypertension. </span>2013;<span class="ref-vol">61</span>:602–608.</span> [<a href="/pubmed/23381795" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hypertension&amp;title=Genome-wide+association+study+on+plasma+levels+of+midregional-proadrenomedullin+and+C-terminal-pro-endothelin-1&amp;author=N+Verweij&amp;author=H+Mahmud&amp;author=I+Mateo+Leach&amp;volume=61&amp;publication_year=2013&amp;pages=602-608&amp;pmid=23381795&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Musani SK, Fox ER, Kraja A, et al.  Genome-wide association analysis of plasma B-type natriuretic peptide in blacks: the Jackson Heart Study. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2015;<span class="ref-vol">8</span>:122–130.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4426827/">PMC free article</a>]</span> [<a href="/pubmed/25561047" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Genome-wide+association+analysis+of+plasma+B-type+natriuretic+peptide+in+blacks:+the+Jackson+Heart+Study&amp;author=SK+Musani&amp;author=ER+Fox&amp;author=A+Kraja&amp;volume=8&amp;publication_year=2015&amp;pages=122-130&amp;pmid=25561047&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Williams FM, Carter AM, Hysi PG, et al.  Ischemic stroke is associated with the ABO locus: the EuroCLOT study. <span><span class="ref-journal">Ann Neurol. </span>2013;<span class="ref-vol">73</span>:16–31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3582024/">PMC free article</a>]</span> [<a href="/pubmed/23381943" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Ischemic+stroke+is+associated+with+the+ABO+locus:+the+EuroCLOT+study&amp;author=FM+Williams&amp;author=AM+Carter&amp;author=PG+Hysi&amp;volume=73&amp;publication_year=2013&amp;pages=16-31&amp;pmid=23381943&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Tang W, Schwienbacher C, Lopez LM, et al.  Genetic associations for activated partial thromboplastin time and prothrombin time, their gene expression profiles, and risk of coronary artery disease. <span><span class="ref-journal">Am J Hum Genet. </span>2012;<span class="ref-vol">91</span>:152–162.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3397273/">PMC free article</a>]</span> [<a href="/pubmed/22703881" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genetic+associations+for+activated+partial+thromboplastin+time+and+prothrombin+time,+their+gene+expression+profiles,+and+risk+of+coronary+artery+disease&amp;author=W+Tang&amp;author=C+Schwienbacher&amp;author=LM+Lopez&amp;volume=91&amp;publication_year=2012&amp;pages=152-162&amp;pmid=22703881&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Lieb W, Chen MH, Teumer A, et al.  Genome-wide meta-analyses of plasma renin activity and concentration reveal association with the kininogen 1 and prekallikrein genes. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2015;<span class="ref-vol">8</span>:131–140.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4354880/">PMC free article</a>]</span> [<a href="/pubmed/25477429" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Genome-wide+meta-analyses+of+plasma+renin+activity+and+concentration+reveal+association+with+the+kininogen+1+and+prekallikrein+genes&amp;author=W+Lieb&amp;author=MH+Chen&amp;author=A+Teumer&amp;volume=8&amp;publication_year=2015&amp;pages=131-140&amp;pmid=25477429&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Katsuda I, Maruyama F, Ezaki K, et al.  A new type of plasma prekallikrein deficiency associated with homozygosity for Gly104Arg and Asn124Ser in apple domain 2 of the heavy-chain region. <span><span class="ref-journal">Eur J Haematol. </span>2007;<span class="ref-vol">79</span>:59–68.</span> [<a href="/pubmed/17598838" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Haematol&amp;title=A+new+type+of+plasma+prekallikrein+deficiency+associated+with+homozygosity+for+Gly104Arg+and+Asn124Ser+in+apple+domain+2+of+the+heavy-chain+region&amp;author=I+Katsuda&amp;author=F+Maruyama&amp;author=K+Ezaki&amp;volume=79&amp;publication_year=2007&amp;pages=59-68&amp;pmid=17598838&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Kanaji T, Okamura T, Osaki K, et al.  A common genetic polymorphism (46 C to T substitution) in the 5'-untranslated region of the coagulation factor XII gene is associated with low translation efficiency and decrease in plasma factor XII level. <span><span class="ref-journal">Blood. </span>1998;<span class="ref-vol">91</span>:2010–2014.</span> [<a href="/pubmed/9490684" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=A+common+genetic+polymorphism+(46+C+to+T+substitution)+in+the+5'-untranslated+region+of+the+coagulation+factor+XII+gene+is+associated+with+low+translation+efficiency+and+decrease+in+plasma+factor+XII+level&amp;author=T+Kanaji&amp;author=T+Okamura&amp;author=K+Osaki&amp;volume=91&amp;publication_year=1998&amp;pages=2010-2014&amp;pmid=9490684&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Calafell F, Almasy L, Sabater-Lleal M, et al.  Sequence variation and genetic evolution at the human F12 locus: mapping quantitative trait nucleotides that influence FXII plasma levels. <span><span class="ref-journal">Hum Mol Genet. </span>2010;<span class="ref-vol">19</span>:517–525.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2798724/">PMC free article</a>]</span> [<a href="/pubmed/19933701" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Sequence+variation+and+genetic+evolution+at+the+human+F12+locus:+mapping+quantitative+trait+nucleotides+that+influence+FXII+plasma+levels&amp;author=F+Calafell&amp;author=L+Almasy&amp;author=M+Sabater-Lleal&amp;volume=19&amp;publication_year=2010&amp;pages=517-525&amp;pmid=19933701&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Calvo SE, Pagliarini DJ, Mootha VK. Upstream open reading frames cause widespread reduction of protein expression and are polymorphic among humans. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2009;<span class="ref-vol">106</span>:7507–7512.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2669787/">PMC free article</a>]</span> [<a href="/pubmed/19372376" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Upstream+open+reading+frames+cause+widespread+reduction+of+protein+expression+and+are+polymorphic+among+humans&amp;author=SE+Calvo&amp;author=DJ+Pagliarini&amp;author=VK+Mootha&amp;volume=106&amp;publication_year=2009&amp;pages=7507-7512&amp;pmid=19372376&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Dendorfer A, Wolfrum S, Wagemann M, et al.  Pathways of bradykinin degradation in blood and plasma of normotensive and hypertensive rats. <span><span class="ref-journal">Am J Physiol Heart Circ Physiol. </span>2001;<span class="ref-vol">280</span>:H2182–2188.</span> [<a href="/pubmed/11299220" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol+Heart+Circ+Physiol&amp;title=Pathways+of+bradykinin+degradation+in+blood+and+plasma+of+normotensive+and+hypertensive+rats&amp;author=A+Dendorfer&amp;author=S+Wolfrum&amp;author=M+Wagemann&amp;volume=280&amp;publication_year=2001&amp;pages=H2182-2188&amp;pmid=11299220&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Kuoppala A, Lindstedt KA, Saarinen J, et al.  Inactivation of bradykinin by angiotensin-converting enzyme and by carboxypeptidase N in human plasma. <span><span class="ref-journal">Am J Physiol Heart Circ Physiol. </span>2000;<span class="ref-vol">278</span>:H1069–1074.</span> [<a href="/pubmed/10749699" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol+Heart+Circ+Physiol&amp;title=Inactivation+of+bradykinin+by+angiotensin-converting+enzyme+and+by+carboxypeptidase+N+in+human+plasma&amp;author=A+Kuoppala&amp;author=KA+Lindstedt&amp;author=J+Saarinen&amp;volume=278&amp;publication_year=2000&amp;pages=H1069-1074&amp;pmid=10749699&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Sheikh IA, Kaplan AP. Mechanism of digestion of bradykinin and lysylbradykinin (kallidin) in human serum. Role of carboxypeptidase, angiotensin-converting enzyme and determination of final degradation products. <span><span class="ref-journal">Biochem Pharmacol. </span>1989;<span class="ref-vol">38</span>:993–1000.</span> [<a href="/pubmed/2539165" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Pharmacol&amp;title=Mechanism+of+digestion+of+bradykinin+and+lysylbradykinin+(kallidin)+in+human+serum.+Role+of+carboxypeptidase,+angiotensin-converting+enzyme+and+determination+of+final+degradation+products&amp;author=IA+Sheikh&amp;author=AP+Kaplan&amp;volume=38&amp;publication_year=1989&amp;pages=993-1000&amp;pmid=2539165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Bjorkqvist J, Jamsa A, Renne T. Plasma kallikrein: the bradykinin-producing enzyme. <span><span class="ref-journal">Thromb Haemost. </span>2013;<span class="ref-vol">110</span>:399–407.</span> [<a href="/pubmed/23846131" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thromb+Haemost&amp;title=Plasma+kallikrein:+the+bradykinin-producing+enzyme&amp;author=J+Bjorkqvist&amp;author=A+Jamsa&amp;author=T+Renne&amp;volume=110&amp;publication_year=2013&amp;pages=399-407&amp;pmid=23846131&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Nikpay M, Goel A, Won HH, et al.  A comprehensive 1,000 Genomes-based genome-wide association meta-analysis of coronary artery disease. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:1121–1130.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4589895/">PMC free article</a>]</span> [<a href="/pubmed/26343387" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+comprehensive+1,000+Genomes-based+genome-wide+association+meta-analysis+of+coronary+artery+disease&amp;author=M+Nikpay&amp;author=A+Goel&amp;author=HH+Won&amp;volume=47&amp;publication_year=2015&amp;pages=1121-1130&amp;pmid=26343387&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Olson NC, Butenas S, Lange LA, et al.  Coagulation factor XII genetic variation, ex vivo thrombin generation, and stroke risk in the elderly: results from the Cardiovascular Health Study. <span><span class="ref-journal">J Thromb Haemost. </span>2015;<span class="ref-vol">13</span>:1867–1877.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4946166/">PMC free article</a>]</span> [<a href="/pubmed/26286125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=Coagulation+factor+XII+genetic+variation,+ex+vivo+thrombin+generation,+and+stroke+risk+in+the+elderly:+results+from+the+Cardiovascular+Health+Study&amp;author=NC+Olson&amp;author=S+Butenas&amp;author=LA+Lange&amp;volume=13&amp;publication_year=2015&amp;pages=1867-1877&amp;pmid=26286125&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6215702&amp;issue-id=323434&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140037238970944title">Summary</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Materials and methods</a></li><li><a href="#S10title">Results</a></li><li><a href="#S20title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S21title">Acknowledgements</a></li><li><a href="#idm140037240175424title">Footnotes</a></li><li><a href="#idm140037246518048title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC6215702/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC6215702/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC6215702/pdf/emss-80283.pdf">PDF (334K)</a></li> | <li><a href="#" data-citationid="PMC6215702" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6215702%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6215702%2F&amp;text=Genome-wide%20association%20reveals%20that%20common%20genetic%20variation%20in%20the%20kallikrein-kinin%20system%20is%20associated%20with%20serum%20L-arginine%20levels" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6215702%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="27656708" data-aiid="6215702" data-aid="6215702" data-iid="323434" data-domainid="358" data-domain="wtpa" data-accid="PMC6215702" data-md5="3d0312e0885137b6c21cd0e2e4675e30"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T15:11:13-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B08AEDD59D85100000000003B0023&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6215702%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC6215702/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>