<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC6697541/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genomic atlas of the human plasma proteome</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6697541/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature"><meta name="citation_title" content="Genomic atlas of the human plasma proteome"><meta name="citation_authors" content="Benjamin B. Sun, Joseph C. Maranville, James E. Peters, David Stacey, James R. Staley, James Blackshaw, Stephen Burgess, Tao Jiang, Ellie Paige, Praveen Surendran, Clare Oliver-Williams, Mihir A. Kamat, Bram P. Prins, Sheri K. Wilcox, Erik S. Zimmerman, An Chi, Narinder Bansal, Sarah L. Spain, Angela M. Wood, Nicholas W. Morrell, John R. Bradley, Nebojsa Janjic, David J. Roberts, Willem H. Ouwehand, John A. Todd, Nicole Soranzo, Karsten Suhre, Dirk S. Paul, Caroline S. Fox, Robert M. Plenge, John Danesh, Heiko Runz, Adam S. Butterworth"><meta name="citation_date" content="1 June 2018"><meta name="citation_issue" content="7708"><meta name="citation_volume" content="558"><meta name="citation_firstpage" content="73"><meta name="citation_doi" content="10.1038/s41586-018-0175-2"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC6697541/?report=abstract"><meta name="citation_pmid" content="29875488"><meta name="DC.Title" content="Genomic atlas of the human plasma proteome"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Benjamin B. Sun"><meta name="DC.Contributor" content="Joseph C. Maranville"><meta name="DC.Contributor" content="James E. Peters"><meta name="DC.Contributor" content="David Stacey"><meta name="DC.Contributor" content="James R. Staley"><meta name="DC.Contributor" content="James Blackshaw"><meta name="DC.Contributor" content="Stephen Burgess"><meta name="DC.Contributor" content="Tao Jiang"><meta name="DC.Contributor" content="Ellie Paige"><meta name="DC.Contributor" content="Praveen Surendran"><meta name="DC.Contributor" content="Clare Oliver-Williams"><meta name="DC.Contributor" content="Mihir A. Kamat"><meta name="DC.Contributor" content="Bram P. Prins"><meta name="DC.Contributor" content="Sheri K. Wilcox"><meta name="DC.Contributor" content="Erik S. Zimmerman"><meta name="DC.Contributor" content="An Chi"><meta name="DC.Contributor" content="Narinder Bansal"><meta name="DC.Contributor" content="Sarah L. Spain"><meta name="DC.Contributor" content="Angela M. Wood"><meta name="DC.Contributor" content="Nicholas W. Morrell"><meta name="DC.Contributor" content="John R. Bradley"><meta name="DC.Contributor" content="Nebojsa Janjic"><meta name="DC.Contributor" content="David J. Roberts"><meta name="DC.Contributor" content="Willem H. Ouwehand"><meta name="DC.Contributor" content="John A. Todd"><meta name="DC.Contributor" content="Nicole Soranzo"><meta name="DC.Contributor" content="Karsten Suhre"><meta name="DC.Contributor" content="Dirk S. Paul"><meta name="DC.Contributor" content="Caroline S. Fox"><meta name="DC.Contributor" content="Robert M. Plenge"><meta name="DC.Contributor" content="John Danesh"><meta name="DC.Contributor" content="Heiko Runz"><meta name="DC.Contributor" content="Adam S. Butterworth"><meta name="DC.Date" content="2018 Jun 1"><meta name="DC.Identifier" content="10.1038/s41586-018-0175-2"><meta name="DC.Language" content="en"><meta property="og:title" content="Genomic atlas of the human plasma proteome"><meta property="og:type" content="article"><meta property="og:description" content="Although plasma proteins have important roles in biological processes and
are the direct targets of many drugs, the genetic factors that control
inter-individual variation in plasma protein levels are not well understood.
Here we characterize the genetic ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6697541/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC6697541/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC6697541/pdf/EMS83968.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8878C9DD5B8D110000000000C5006D.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6697541%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="nlmcatalog">NLM Catalog</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC6697541</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nature</a></span>. Author manuscript; available in PMC 2019 Aug 16.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=29875488" target="pmc_ext" ref="reftype=publisher&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nature. 2018 Jun 1; 558(7708): 73–79. </span></a></div><span class="fm-vol-iss-date">Published online 2018 Jun 6. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fs41586-018-0175-2" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/s41586-018-0175-2</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC6697541</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>EMS83968</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/29875488">29875488</a></div></div></div></div><h1 class="content-title">Genomic atlas of the human plasma proteome</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Sun%20BB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142947751568" co-class="co-affbox">Benjamin B. Sun</a>,<sup>#</sup><sup>1</sup> <a href="/pubmed/?term=Maranville%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142946318704" co-class="co-affbox">Joseph C. Maranville</a>,<sup>#</sup><sup>2,</sup><sup>20</sup> <a href="/pubmed/?term=Peters%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142947746864" co-class="co-affbox">James E. Peters</a>,<sup>#</sup><sup>1,</sup><sup>3</sup> <a href="/pubmed/?term=Stacey%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142947743760" co-class="co-affbox">David Stacey</a>,<sup>1</sup> <a href="/pubmed/?term=Staley%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142982929872" co-class="co-affbox">James R. Staley</a>,<sup>1</sup> <a href="/pubmed/?term=Blackshaw%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142982927840" co-class="co-affbox">James Blackshaw</a>,<sup>1</sup> <a href="/pubmed/?term=Burgess%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142982925808" co-class="co-affbox">Stephen Burgess</a>,<sup>1,</sup><sup>4</sup> <a href="/pubmed/?term=Jiang%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142982922944" co-class="co-affbox">Tao Jiang</a>,<sup>1</sup> <a href="/pubmed/?term=Paige%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142932200448" co-class="co-affbox">Ellie Paige</a>,<sup>1,</sup><sup>5</sup> <a href="/pubmed/?term=Surendran%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142932197536" co-class="co-affbox">Praveen Surendran</a>,<sup>1</sup> <a href="/pubmed/?term=Oliver-Williams%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142932195504" co-class="co-affbox">Clare Oliver-Williams</a>,<sup>1,</sup><sup>6</sup> <a href="/pubmed/?term=Kamat%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142932192592" co-class="co-affbox">Mihir A. Kamat</a>,<sup>1</sup> <a href="/pubmed/?term=Prins%20BP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142943681888" co-class="co-affbox">Bram P. Prins</a>,<sup>1</sup> <a href="/pubmed/?term=Wilcox%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142943679856" co-class="co-affbox">Sheri K. Wilcox</a>,<sup>7</sup> <a href="/pubmed/?term=Zimmerman%20ES%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142943677824" co-class="co-affbox">Erik S. Zimmerman</a>,<sup>7</sup> <a href="/pubmed/?term=Chi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142943675840" co-class="co-affbox">An Chi</a>,<sup>2</sup> <a href="/pubmed/?term=Bansal%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142933585568" co-class="co-affbox">Narinder Bansal</a>,<sup>1,</sup><sup>8</sup> <a href="/pubmed/?term=Spain%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142933582960" co-class="co-affbox">Sarah L. Spain</a>,<sup>9</sup> <a href="/pubmed/?term=Wood%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142933580976" co-class="co-affbox">Angela M. Wood</a>,<sup>1</sup> <a href="/pubmed/?term=Morrell%20NW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142944916496" co-class="co-affbox">Nicholas W. Morrell</a>,<sup>3,</sup><sup>10</sup> <a href="/pubmed/?term=Bradley%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142944913776" co-class="co-affbox">John R. Bradley</a>,<sup>11</sup> <a href="/pubmed/?term=Janjic%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142944911792" co-class="co-affbox">Nebojsa Janjic</a>,<sup>7</sup> <a href="/pubmed/?term=Roberts%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142944909808" co-class="co-affbox">David J. Roberts</a>,<sup>12,</sup><sup>13</sup> <a href="/pubmed/?term=Ouwehand%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142935210144" co-class="co-affbox">Willem H. Ouwehand</a>,<sup>3,</sup><sup>14,</sup><sup>15,</sup><sup>16,</sup><sup>17</sup> <a href="/pubmed/?term=Todd%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142935205216" co-class="co-affbox">John A. Todd</a>,<sup>18</sup> <a href="/pubmed/?term=Soranzo%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142935203232" co-class="co-affbox">Nicole Soranzo</a>,<sup>3,</sup><sup>14,</sup><sup>16,</sup><sup>17</sup> <a href="/pubmed/?term=Suhre%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142948300944" co-class="co-affbox">Karsten Suhre</a>,<sup>19</sup> <a href="/pubmed/?term=Paul%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142948298960" co-class="co-affbox">Dirk S. Paul</a>,<sup>1</sup> <a href="/pubmed/?term=Fox%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142948296976" co-class="co-affbox">Caroline S. Fox</a>,<sup>2</sup> <a href="/pubmed/?term=Plenge%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142948294992" co-class="co-affbox">Robert M. Plenge</a>,<sup>2,</sup><sup>20</sup> <a href="/pubmed/?term=Danesh%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142933316096" co-class="co-affbox">John Danesh</a>,<sup>1,</sup><sup>3,</sup><sup>16,</sup><sup>17,</sup><sup>*</sup> <a href="/pubmed/?term=Runz%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142933311168" co-class="co-affbox">Heiko Runz</a>,<sup>2,</sup><sup>21,</sup><sup>23</sup> and  <a href="/pubmed/?term=Butterworth%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488" class="affpopup" co-rid="_co_idm140142930659024" co-class="co-affbox">Adam S. Butterworth</a><sup>1,</sup><sup>17,</sup><sup>23,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140142947751568"><h3 class="no_margin">Benjamin B. Sun</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Sun%20BB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Benjamin B. Sun</a></div></div><div id="_co_idm140142946318704"><h3 class="no_margin">Joseph C. Maranville</h3><p>  <sup>2</sup>MRL, Merck &amp; Co., Inc., Kenilworth, NJ, USA</p><div>Find articles by <a href="/pubmed/?term=Maranville%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Joseph C. Maranville</a></div></div><div id="_co_idm140142947746864"><h3 class="no_margin">James E. Peters</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><p>  <sup>3</sup>British Heart Foundation Cambridge Centre of Excellence, Division of Cardiovascular Medicine, Addenbrooke’s Hospital, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Peters%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">James E. Peters</a></div></div><div id="_co_idm140142947743760"><h3 class="no_margin">David Stacey</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Stacey%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">David Stacey</a></div></div><div id="_co_idm140142982929872"><h3 class="no_margin">James R. Staley</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Staley%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">James R. Staley</a></div></div><div id="_co_idm140142982927840"><h3 class="no_margin">James Blackshaw</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Blackshaw%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">James Blackshaw</a></div></div><div id="_co_idm140142982925808"><h3 class="no_margin">Stephen Burgess</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><p>  <sup>4</sup>MRC Biostatistics Unit, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Burgess%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Stephen Burgess</a></div></div><div id="_co_idm140142982922944"><h3 class="no_margin">Tao Jiang</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Jiang%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Tao Jiang</a></div></div><div id="_co_idm140142932200448"><h3 class="no_margin">Ellie Paige</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><p>  <sup>5</sup>National Centre for Epidemiology and Population Health, The Australian National University, Canberra, Australian Capital Territory, Australia</p><div>Find articles by <a href="/pubmed/?term=Paige%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Ellie Paige</a></div></div><div id="_co_idm140142932197536"><h3 class="no_margin">Praveen Surendran</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Surendran%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Praveen Surendran</a></div></div><div id="_co_idm140142932195504"><h3 class="no_margin">Clare Oliver-Williams</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><p>  <sup>6</sup>Homerton College, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Oliver-Williams%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Clare Oliver-Williams</a></div></div><div id="_co_idm140142932192592"><h3 class="no_margin">Mihir A. Kamat</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Kamat%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Mihir A. Kamat</a></div></div><div id="_co_idm140142943681888"><h3 class="no_margin">Bram P. Prins</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Prins%20BP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Bram P. Prins</a></div></div><div id="_co_idm140142943679856"><h3 class="no_margin">Sheri K. Wilcox</h3><p>  <sup>7</sup>SomaLogic Inc, Boulder, CO, USA</p><div>Find articles by <a href="/pubmed/?term=Wilcox%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Sheri K. Wilcox</a></div></div><div id="_co_idm140142943677824"><h3 class="no_margin">Erik S. Zimmerman</h3><p>  <sup>7</sup>SomaLogic Inc, Boulder, CO, USA</p><div>Find articles by <a href="/pubmed/?term=Zimmerman%20ES%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Erik S. Zimmerman</a></div></div><div id="_co_idm140142943675840"><h3 class="no_margin">An Chi</h3><p>  <sup>2</sup>MRL, Merck &amp; Co., Inc., Kenilworth, NJ, USA</p><div>Find articles by <a href="/pubmed/?term=Chi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">An Chi</a></div></div><div id="_co_idm140142933585568"><h3 class="no_margin">Narinder Bansal</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><p>  <sup>8</sup>Population Health Sciences, Bristol Medical School, University of Bristol, Bristol, UK</p><div>Find articles by <a href="/pubmed/?term=Bansal%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Narinder Bansal</a></div></div><div id="_co_idm140142933582960"><h3 class="no_margin">Sarah L. Spain</h3><p>  <sup>9</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Spain%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Sarah L. Spain</a></div></div><div id="_co_idm140142933580976"><h3 class="no_margin">Angela M. Wood</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Wood%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Angela M. Wood</a></div></div><div id="_co_idm140142944916496"><h3 class="no_margin">Nicholas W. Morrell</h3><p>  <sup>3</sup>British Heart Foundation Cambridge Centre of Excellence, Division of Cardiovascular Medicine, Addenbrooke’s Hospital, Cambridge, UK</p><p>  <sup>10</sup>Division of Respiratory Medicine, Department of Medicine, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Morrell%20NW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Nicholas W. Morrell</a></div></div><div id="_co_idm140142944913776"><h3 class="no_margin">John R. Bradley</h3><p>  <sup>11</sup>NIHR Cambridge Biomedical Research Centre/BioResource, Cambridge University Hospitals, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Bradley%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">John R. Bradley</a></div></div><div id="_co_idm140142944911792"><h3 class="no_margin">Nebojsa Janjic</h3><p>  <sup>7</sup>SomaLogic Inc, Boulder, CO, USA</p><div>Find articles by <a href="/pubmed/?term=Janjic%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Nebojsa Janjic</a></div></div><div id="_co_idm140142944909808"><h3 class="no_margin">David J. Roberts</h3><p>  <sup>12</sup>National Health Service (NHS) Blood and Transplant and Radcliffe Department of Medicine, NIHR Oxford Biomedical Research Centre, University of Oxford, John Radcliffe Hospital, Oxford, UK</p><p>  <sup>13</sup>BRC Haematology Theme and Department of Haematology, Churchill Hospital, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Roberts%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">David J. Roberts</a></div></div><div id="_co_idm140142935210144"><h3 class="no_margin">Willem H. Ouwehand</h3><p>  <sup>3</sup>British Heart Foundation Cambridge Centre of Excellence, Division of Cardiovascular Medicine, Addenbrooke’s Hospital, Cambridge, UK</p><p>  <sup>14</sup>Department of Haematology, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK</p><p>  <sup>15</sup>National Health Service (NHS) Blood and Transplant, Cambridge Biomedical Campus, Cambridge, UK</p><p>  <sup>16</sup>Department of Human Genetics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK</p><p>  <sup>17</sup>NIHR Blood and Transplant Research Unit in Donor Health and Genomics, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Ouwehand%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Willem H. Ouwehand</a></div></div><div id="_co_idm140142935205216"><h3 class="no_margin">John A. Todd</h3><p>  <sup>18</sup>JDRF/Wellcome Trust Diabetes and Inflammation Laboratory, Wellcome Trust Centre for Human Genetics, Nuffield Department of Medicine, NIHR Oxford Biomedical Research Centre, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Todd%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">John A. Todd</a></div></div><div id="_co_idm140142935203232"><h3 class="no_margin">Nicole Soranzo</h3><p>  <sup>3</sup>British Heart Foundation Cambridge Centre of Excellence, Division of Cardiovascular Medicine, Addenbrooke’s Hospital, Cambridge, UK</p><p>  <sup>14</sup>Department of Haematology, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK</p><p>  <sup>16</sup>Department of Human Genetics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK</p><p>  <sup>17</sup>NIHR Blood and Transplant Research Unit in Donor Health and Genomics, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Soranzo%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Nicole Soranzo</a></div></div><div id="_co_idm140142948300944"><h3 class="no_margin">Karsten Suhre</h3><p>  <sup>19</sup>Department of Physiology and Biophysics, Weill Cornell Medicine–Qatar, Doha, Qatar</p><div>Find articles by <a href="/pubmed/?term=Suhre%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Karsten Suhre</a></div></div><div id="_co_idm140142948298960"><h3 class="no_margin">Dirk S. Paul</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Paul%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Dirk S. Paul</a></div></div><div id="_co_idm140142948296976"><h3 class="no_margin">Caroline S. Fox</h3><p>  <sup>2</sup>MRL, Merck &amp; Co., Inc., Kenilworth, NJ, USA</p><div>Find articles by <a href="/pubmed/?term=Fox%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Caroline S. Fox</a></div></div><div id="_co_idm140142948294992"><h3 class="no_margin">Robert M. Plenge</h3><p>  <sup>2</sup>MRL, Merck &amp; Co., Inc., Kenilworth, NJ, USA</p><div>Find articles by <a href="/pubmed/?term=Plenge%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Robert M. Plenge</a></div></div><div id="_co_idm140142933316096"><h3 class="no_margin">John Danesh</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><p>  <sup>3</sup>British Heart Foundation Cambridge Centre of Excellence, Division of Cardiovascular Medicine, Addenbrooke’s Hospital, Cambridge, UK</p><p>  <sup>16</sup>Department of Human Genetics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK</p><p>  <sup>17</sup>NIHR Blood and Transplant Research Unit in Donor Health and Genomics, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Danesh%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">John Danesh</a></div></div><div id="_co_idm140142933311168"><h3 class="no_margin">Heiko Runz</h3><p>  <sup>2</sup>MRL, Merck &amp; Co., Inc., Kenilworth, NJ, USA</p><div>Find articles by <a href="/pubmed/?term=Runz%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Heiko Runz</a></div></div><div id="_co_idm140142930659024"><h3 class="no_margin">Adam S. Butterworth</h3><p>  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><p>  <sup>17</sup>NIHR Blood and Transplant Research Unit in Donor Health and Genomics, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Butterworth%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29875488">Adam S. Butterworth</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140142935289936_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140142935289936_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140142935289936_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">  <sup>1</sup>MRC/BHF Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</div><div class="fm-affl" id="A2" lang="en">  <sup>2</sup>MRL, Merck &amp; Co., Inc., Kenilworth, NJ, USA</div><div class="fm-affl" id="A3" lang="en">  <sup>3</sup>British Heart Foundation Cambridge Centre of Excellence, Division of Cardiovascular Medicine, Addenbrooke’s Hospital, Cambridge, UK</div><div class="fm-affl" id="A4" lang="en">  <sup>4</sup>MRC Biostatistics Unit, University of Cambridge, Cambridge, UK</div><div class="fm-affl" id="A5" lang="en">  <sup>5</sup>National Centre for Epidemiology and Population Health, The Australian National University, Canberra, Australian Capital Territory, Australia</div><div class="fm-affl" id="A6" lang="en">  <sup>6</sup>Homerton College, Cambridge, UK</div><div class="fm-affl" id="A7" lang="en">  <sup>7</sup>SomaLogic Inc, Boulder, CO, USA</div><div class="fm-affl" id="A8" lang="en">  <sup>8</sup>Population Health Sciences, Bristol Medical School, University of Bristol, Bristol, UK</div><div class="fm-affl" id="A9" lang="en">  <sup>9</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK</div><div class="fm-affl" id="A10" lang="en">  <sup>10</sup>Division of Respiratory Medicine, Department of Medicine, University of Cambridge, Cambridge, UK</div><div class="fm-affl" id="A11" lang="en">  <sup>11</sup>NIHR Cambridge Biomedical Research Centre/BioResource, Cambridge University Hospitals, Cambridge, UK</div><div class="fm-affl" id="A12" lang="en">  <sup>12</sup>National Health Service (NHS) Blood and Transplant and Radcliffe Department of Medicine, NIHR Oxford Biomedical Research Centre, University of Oxford, John Radcliffe Hospital, Oxford, UK</div><div class="fm-affl" id="A13" lang="en">  <sup>13</sup>BRC Haematology Theme and Department of Haematology, Churchill Hospital, Oxford, UK</div><div class="fm-affl" id="A14" lang="en">  <sup>14</sup>Department of Haematology, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK</div><div class="fm-affl" id="A15" lang="en">  <sup>15</sup>National Health Service (NHS) Blood and Transplant, Cambridge Biomedical Campus, Cambridge, UK</div><div class="fm-affl" id="A16" lang="en">  <sup>16</sup>Department of Human Genetics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK</div><div class="fm-affl" id="A17" lang="en">  <sup>17</sup>NIHR Blood and Transplant Research Unit in Donor Health and Genomics, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</div><div class="fm-affl" id="A18" lang="en">  <sup>18</sup>JDRF/Wellcome Trust Diabetes and Inflammation Laboratory, Wellcome Trust Centre for Human Genetics, Nuffield Department of Medicine, NIHR Oxford Biomedical Research Centre, University of Oxford, Oxford, UK</div><div class="fm-affl" id="A19" lang="en">  <sup>19</sup>Department of Physiology and Biophysics, Weill Cornell Medicine–Qatar, Doha, Qatar</div><div><sup>#</sup>Contributed equally.</div><div id="CR1"><sup>*</sup><strong>Correspondence and requests for materials</strong> should be addressed to J.D. or A.S.B. <a href="mailto:dev@null" data-email="ku.ca.mac.lhcsdem@292d" class="oemail">ku.ca.mac.lhcsdem@292d</a>; <a href="mailto:dev@null" data-email="ku.ca.mac.lhcsdem@83bsa" class="oemail">ku.ca.mac.lhcsdem@83bsa</a></div><div id="FN1"><sup>20</sup>Present address: Celgene Inc., Cambridge, MA, USA</div><div id="FN2"><sup>21</sup>Present address: Biogen Inc., Cambridge, MA, USA.</div><div id="FN3"><sup>23</sup>These authors jointly supervised this work: Heiko Runz, Adam S.
Butterworth.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140142935289936_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=29875488" target="pmc_ext" ref="reftype=publisher&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nature</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC6697541/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="3" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary tables.</strong></div><div><a href="/pmc/articles/PMC6697541/bin/EMS83968-supplement-Supplementary_tables.xlsx" data-ga-action="click_feat_suppl">EMS83968-supplement-Supplementary_tables.xlsx</a><span style="color:gray"> (2.1M)</span></div><div class="small guid">GUID:&nbsp;895024BA-F2DB-4086-8E09-0FF6C56B10B7</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary figure 1.</strong></div><div><a href="/pmc/articles/PMC6697541/bin/EMS83968-supplement-Supplementary_figure_1.html" data-ga-action="click_feat_suppl">EMS83968-supplement-Supplementary_figure_1.html</a><span style="color:gray"> (2.7M)</span></div><div class="small guid">GUID:&nbsp;5165C71D-FDD7-41AA-9432-CF112E3237AA</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary file.</strong></div><div><a href="/pmc/articles/PMC6697541/bin/EMS83968-supplement-Supplementary_file.pdf" data-ga-action="click_feat_suppl">EMS83968-supplement-Supplementary_file.pdf</a><span style="color:gray"> (417K)</span></div><div class="small guid">GUID:&nbsp;E317AE57-641B-4812-92B3-F5B0B1B37E73</div></div></dd></dl><dl data-length="477" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last"><strong>Data availability.</strong> Participant-level genotype and protein
data, and full summary association results from the genetic analysis, are
available through the European Genotype Archive (accession number
EGAS00001002555). Summary association results are also publically available at
<a href="http://www.phpc.cam.ac.uk/ceu/proteins/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.phpc.cam.ac.uk/ceu/proteins/</a>, through PhenoScanner
(<a href="http://www.phenoscanner.medschl.cam.ac.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.phenoscanner.medschl.cam.ac.uk</a>) and from the
NHGRI-EBI GWAS Catalog (<a href="https://www.ebi.ac.uk/gwas/downloads/summary-statistics" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ebi.ac.uk/gwas/downloads/summary-statistics</a>).</p></dd></dl></div><div id="idm140142931798448" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140142931798448title">Abstract</h2><!--article-meta--><div><p id="P3" class="p p-first-last">Although plasma proteins have important roles in biological processes and
are the direct targets of many drugs, the genetic factors that control
inter-individual variation in plasma protein levels are not well understood.
Here we characterize the genetic architecture of the human plasma proteome in
healthy blood donors from the INTERVAL study. We identify 1,927 genetic
associations with 1,478 proteins, a fourfold increase on existing knowledge,
including <em>trans</em> associations for 1,104 proteins. To understand
the consequences of perturbations in plasma protein levels, we apply an
integrated approach that links genetic variation with biological pathway,
disease, and drug databases. We show that protein quantitative trait loci
overlap with gene expression quantitative trait loci, as well as with
disease-associated loci, and find evidence that protein biomarkers have causal
roles in disease using Mendelian randomization analysis. By linking genetic
factors to diseases via specific proteins, our analyses highlight potential
therapeutic targets, opportunities for matching existing drugs with new disease
indications, and potential safety concerns for drugs under development.</p></div></div><div id="idm140142931795856" class="tsec sec headless whole_rhythm"><p id="P4" class="p p-first">Plasma proteins have key roles in various biological processes, including
signalling, transport, growth, repair, and defence against infection. These proteins are
frequently dysregulated in disease and are important drug targets. Identifying factors
that determine inter-individual protein variability should, therefore, furnish
biological and medical insights<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750399" role="button" aria-expanded="false" aria-haspopup="true">1</a></sup>.
Despite evidence for the heritability of plasma protein abundance<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750388" role="button" aria-expanded="false" aria-haspopup="true">2</a></sup>, however, systematic assessment of how
genetic variation influences plasma protein levels has been limited<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750456" role="button" aria-expanded="false" aria-haspopup="true">3</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750431" role="button" aria-expanded="false" aria-haspopup="true">5</a></sup>. Studies have examined intra-cellular protein quantitative trait
loci (pQTLs)<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750438" role="button" aria-expanded="false" aria-haspopup="true">6</a>,<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750454" role="button" aria-expanded="false" aria-haspopup="true">7</a></sup>, but these studies have tended to be small and involved cell
lines rather than primary human tissues.</p><p id="P5" class="p">Here we create and interrogate a genetic atlas of the human plasma proteome,
using an expanded version of an aptamer-based multiplex protein assay
(SOMAscan)<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750413" role="button" aria-expanded="false" aria-haspopup="true">8</a></sup> to quantify 3,622
plasma proteins in 3,301 healthy participants from the INTERVAL study, a genomic
bioresource of 50,000 blood donors from 25 centres across England recruited into a
randomized trial of blood donation frequency<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750443" role="button" aria-expanded="false" aria-haspopup="true">9</a>,<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750429" role="button" aria-expanded="false" aria-haspopup="true">10</a></sup>. We identify 1,927
genotype–protein associations (pQTLs), including
<em>trans</em>-associated loci for 1,104 proteins, providing new understanding
of the genetic control of protein regulation. Eighty-eight pQTLs overlap with disease
susceptibility loci, suggesting the molecular effects of disease-associated variants.
Using the principle of Mendelian randomization<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750378" role="button" aria-expanded="false" aria-haspopup="true">11</a></sup>, we find evidence to support causal roles in disease for
several protein pathways, and cross-reference our data with disease and drug databases
to highlight potential therapeutic targets.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Genetic architecture of the plasma proteome</h2><p id="P6" class="p p-first">We performed genome-wide testing of 10.6 million imputed autosomal variants
against levels of 2,994 plasma proteins in 3,301 individuals of European descent
(<a href="#S8" rid="S8" class=" sec">Methods</a>, <a href="/pmc/articles/PMC6697541/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Extended Data Fig. 1</span></a>). We demonstrated the robustness of protein
measurements in several ways (<a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Note</a>, <a href="/pmc/articles/PMC6697541/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Extended Data Fig.
2</span></a>), including: highly consistent measurements in replicate samples;
temporal consistency of protein levels within individuals over two years (<a href="/pmc/articles/PMC6697541/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Fig. 3b</span></a>); and replication of known
associations with non-genetic factors (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 1, 2</a>). To assess potential off-target
cross-reactivity, we tested 920 aptamers (SOMAmers) for detection of proteins with
at least 40% sequence homology to the target protein (<a href="#S8" rid="S8" class=" sec">Methods</a>). Although 126 (14%) SOMAmers showed comparable binding with a
homologous protein (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary
Table 3</a>), nearly half of these were binding to alternative forms of the
same protein.</p><p id="P7">We found 1,927 significant (<em>P</em> &lt; 1.5 ×
10<sup>−11</sup>) associations between 1,478 proteins and 764 genomic
regions (<a href="/pmc/articles/PMC6697541/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1a</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>, <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Fig. 1</a>, <a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Note Table 1</a>),
with 89% of these pQTLs being previously unreported. Of the 764 associated regions,
502 (66%) had local-acting (<em>cis</em>) associations only, 228 (30%)
<em>trans</em> only, and 34 (4%) both <em>cis</em> and
<em>trans</em> (<a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Note Table 1</a>). Of the <em>cis</em> pQTL sentinel
variants, 95% and 87% were located within 200 kb and 100 kb, respectively, of the
relevant gene’s canonical transcription start site (TSS), and 44% were within
the gene itself. The <em>P</em> values for <em>cis</em> associations
increased with distance from the TSS (<a href="/pmc/articles/PMC6697541/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1b</span></a>),
mirroring findings for gene expression QTLs (eQTLs)<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750386" role="button" aria-expanded="false" aria-haspopup="true">12</a></sup>. Of proteins with a significant pQTL, 88% had
either <em>cis</em> (<em>n</em> = 374) or <em>trans</em>
(<em>n</em> = 925) associations only, and 12% (<em>n</em> = 179)
had both (<a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Note Table
1</a>). The majority of significantly associated proteins (75%;
<em>n</em> = 1,113) had a single pQTL, while 20% had two and 5% had more
than two (<a href="/pmc/articles/PMC6697541/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1c</span></a>). To detect multiple
independent associations at the same locus, we used stepwise conditional analysis,
identifying 2,658 conditionally significant associations (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). Of
the 1,927 pQTLs, 414 (21%) had multiple conditionally significant associations
(<a href="/pmc/articles/PMC6697541/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1d</span></a>), of which 255 were
<em>cis</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC6697541/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142935317200" class="figure"><a href="/pmc/articles/PMC6697541/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f001.jpg"></a></div><div id="largeobj_idm140142935317200" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6697541/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC6697541/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>The genetic architecture of plasma protein levels.</strong><p id="__p2"><em>n</em> = 3,301 participants. <strong>a</strong>, Genomic locations of
pQTLs. Red, <em>cis</em>; blue, <em>trans</em>. The
<em>x</em>- and <em>y</em>-axes indicate the positions of
the sentinel variant and the gene encoding the associated protein, respectively.
Highly pleiotropic genomic regions are annotated. <strong>b</strong>, Significance
of <em>cis</em> associations (linear regression) versus distance of
sentinel variant from TSS. <strong>c</strong>, Number of significantly associated
loci per protein. <strong>d</strong>, Number of conditionally significant
associations within each pQTL. <strong>e</strong>, Histogram of variance explained
by conditionally significant variants. <strong>f</strong>, Effect size versus MAF.
<strong>g</strong>, Distributions of the predicted functional annotation classes
of sentinel pQTL variants versus null sets of variants from permutation. Bar
height represents the mean proportion of variants within each class and error
bars reflect one s.d. from the mean. *Significant enrichment (permutation test,
Bonferroni-corrected threshold, <em>P</em> &lt; 0.005).</p></div></div></div><p id="P8">We tested replication of 163 pQTLs in 4,998 individuals using an alternative
protein assay (Olink, see <a href="#S8" rid="S8" class=" sec">Methods</a>)<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750462" role="button" aria-expanded="false" aria-haspopup="true">13</a></sup>. Effect-size estimates were
strongly correlated between the SOMAscan and Olink platforms (<em>r</em> =
0.83; <a href="/pmc/articles/PMC6697541/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Fig. 3c</span></a>). One-hundred and
six out of one-hundred and sixty-three (65% overall; 81% <em>cis</em>, 52%
<em>trans</em>) pQTLs were replicated after Bonferroni correction (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 4, 6</a>).
The lower replication rate of <em>trans</em> associations may reflect
various factors, including differences between protein assays (for example,
detection of free versus complexed proteins, <a href="/pmc/articles/PMC6697541/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Extended
Data Fig. 4</span></a>) and the higher ‘biological prior’ for
<em>cis</em> associations.</p><p id="P9">Of 1,927 pQTLs, 549 (28%) were <em>cis</em>-acting (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>).
Genetic variants that change protein structure may result in apparent
<em>cis</em> pQTLs owing to altered aptamer binding rather than true
quantitative differences in protein levels. We found evidence against such
artefactual associations for 371 (68%) <em>cis</em> pQTLs (<a href="#S8" rid="S8" class=" sec">Methods</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 4, 7, 8</a>). The results were materially
unchanged when we repeated downstream analyses but excluded pQTLs without evidence
against binding effects.</p><p id="P10">The median variation in protein levels explained by pQTLs was 5.8%
(interquartile range: 2.6–12.4%, <a href="/pmc/articles/PMC6697541/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig.
1e</span></a>). For 193 proteins, genetic variants explained more than 20% of the
variation. There was a strong inverse relationship between effect size and minor
allele frequency (MAF) (<a href="/pmc/articles/PMC6697541/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1f</span></a>), consistent
with previous genome-wide association studies (GWAS) of quantitative
traits<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750389" role="button" aria-expanded="false" aria-haspopup="true">7</a>,<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750406" role="button" aria-expanded="false" aria-haspopup="true">10</a>,<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750452" role="button" aria-expanded="false" aria-haspopup="true">14</a></sup>. We
found 23 and 208 associations with rare (MAF &lt;1%) and low-frequency (MAF
1–5%) variants, respectively (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). Of the 36 strongest associations
(per-allele effect size &gt;1.5 standard deviation (s.d.)), 29 were with rare or
low-frequency variants.</p><p id="P11" class="p p-last">Both <em>cis</em> and <em>trans</em> pQTLs were strongly
enriched for missense variants (<em>P</em> &lt; 0.0001) and for location
in 3′ untranslated (<em>P</em> = 0.0025) or splice sites
(<em>P</em> = 0.0004) (<a href="/pmc/articles/PMC6697541/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1g</span></a>,
<a href="/pmc/articles/PMC6697541/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Extended Data Fig. 5a</span></a>). We found at least
threefold enrichment (<em>P</em> &lt; 5 × 10<sup>−5</sup>)
of pQTLs at features indicative of transcriptional activation in blood cells and at
hepatocyte regulatory elements, consistent with the role of the liver in protein
synthesis and secretion (<a href="#S8" rid="S8" class=" sec">Methods</a>, <a href="/pmc/articles/PMC6697541/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Extended Data Fig. 6</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a>).</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Overlap of eQTLs and pQTLs</h2><p id="P12" class="p p-first">To help evaluate the extent to which genetic associations with plasma
protein levels are driven by effects at the transcriptional level rather than other
mechanisms (for example, altered protein clearance or secretion), we
cross-referenced our <em>cis</em> pQTLs with previous eQTL studies (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 10</a>),
initially defining overlap between an eQTL and pQTL as high linkage disequilibrium
(LD) (<em>r</em><sup>2</sup> ≥ 0.8) between the lead pQTL and eQTL
variants. Forty per cent (<em>n</em> = 224) of <em>cis</em> pQTLs
were eQTLs for the same gene in one or more tissue or cell type (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>). The
greatest overlaps were in whole blood (<em>n</em> = 117), liver
(<em>n</em> = 70) and lymphoblastoid cell lines (LCLs)
(<em>n</em> = 52), consistent with biological expectation, but also
probably driven by the larger eQTL study sample sizes for these cell types. To
investigate whether the same causal variant was likely to underlie overlapping eQTLs
and pQTLs, we performed colocalization testing (see <a href="#S8" rid="S8" class=" sec">Methods</a>). Of 228 pQTLs outside the human leukocyte antigen (HLA) region
for which testing was possible, colocalization in one or more tissue or cell type
was highly likely (posterior probability (PP) &gt; 0.8) for 179 (78.5%) and the
most likely explanation (PP &gt; 0.5) for 197 (86.4%) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a>).
<em>cis</em> pQTLs were significantly enriched for eQTLs for the
corresponding gene (<em>P</em> &lt; 0.0001) (<a href="#S8" rid="S8" class=" sec">Methods</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 11</a>). To address the converse (that is, to what
extent are eQTLs also pQTLs), we selected well-powered eQTL studies in relevant
tissues (whole blood, LCLs, liver and monocytes<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750416" role="button" aria-expanded="false" aria-haspopup="true">15</a>–<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750419" role="button" aria-expanded="false" aria-haspopup="true">18</a></sup>). Of the
strongest <em>cis</em> eQTLs (<em>P</em> &lt; 1.5 ×
10<sup>−11</sup>) in whole blood, LCLs, liver and monocytes, 12.2%,
21.3%, 14.8% and 14.7%, respectively, were plasma <em>cis</em> pQTLs.</p><p id="P13" class="p p-last">Comparisons between eQTL and pQTL studies have inherent limitations,
including differences in the tissues, sample sizes and technological platforms used.
Moreover, plasma protein levels may not reflect levels within tissues or cells.
Nevertheless, our data suggest that genetic effects on plasma protein abundance are
often, but not exclusively, driven by regulation of mRNA. <em>cis</em> pQTLs
without corresponding <em>cis</em> eQTLs may reflect genetic effects on
processes other than transcription, including protein degradation, binding,
secretion, or clearance from circulation.</p></div><div id="S3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S3title"><em>trans</em> pQTLs identify pathways to disease</h2><p id="P14" class="p p-first">Of the 764 protein-associated regions, 262 had <em>trans</em>
associations with 1,104 proteins (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 4, 12</a>). There was no enrichment of cross-reactivity
in SOMAmers with a <em>trans</em> pQTL versus those without (<a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Note</a>). We
replicated known <em>trans</em> associations, including
<em>TMPRSS6</em> with transferrin receptor protein 1<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750434" role="button" aria-expanded="false" aria-haspopup="true">19</a></sup> and <em>SORT1</em> with
granulins<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750397" role="button" aria-expanded="false" aria-haspopup="true">20</a></sup>, and
identified several novel and biologically plausible <em>trans</em>
associations (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table
13</a>), including known or presumed ligand–receptor pairs (for
example, the <em>CD320</em> locus, encoding the transcobalamin receptor, was
associated with transcobalamin-2 levels).</p><p id="P15">Most <em>trans</em> loci (82%) were associated with fewer than four
proteins, but twelve ‘hotspot’ regions were associated with more than
twenty (<a href="/pmc/articles/PMC6697541/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1a</span></a>, <a href="/pmc/articles/PMC6697541/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Extended Data Fig. 5b</span></a>), including well-known pleiotropic loci (for
example, <em>ABO</em>, <em>CFH</em>, <em>APOE</em> and
<em>KLKB1</em>) and loci associated with many correlated proteins (for
example, the <em>ZFPM2</em> locus, which encodes the transcription factor
FOG2). Similar pleiotropy at these loci has been seen in other plasma pQTL
studies<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750391" role="button" aria-expanded="false" aria-haspopup="true">3</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750411" role="button" aria-expanded="false" aria-haspopup="true">5</a></sup>, albeit with fewer proteins owing to
limited assay breadth. A missense variant (rs28929474:T) in
<em>SERPINA1</em> was associated with 13 proteins at <em>P</em>
&lt; 1.5 × 10<sup>−11</sup> and a further six at
<em>P</em> &lt; 5 × 10<sup>−8</sup> (<a href="/pmc/articles/PMC6697541/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). This variant (the
‘Z-allele’) results in defective secretion and intracellular
accumulation of α1-antitrypsin (A1AT), an anti-protease. Individuals
homozygous for the Z allele have a deficiency of circulating A1AT and an increased
risk of emphysema, liver cirrhosis and vasculitis. The
‘protease–antiprotease’ hypothesis posits that the pulmonary
manifestations of A1AT deficiency result from unchecked protease activity. Our
discovery of multiple <em>trans</em>-associated proteins at this locus
highlights additional pathways that might be relevant to pathogenesis, a hypothesis
supported by accumulating data<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750404" role="button" aria-expanded="false" aria-haspopup="true">21</a></sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC6697541/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142935310112" class="figure"><a href="/pmc/articles/PMC6697541/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f002.jpg"></a></div><div id="largeobj_idm140142935310112" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6697541/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC6697541/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Missense variant rs28929474:T in <em>SERPINA1</em> is a
<em>trans</em> pQTL hotspot.</strong><p id="__p3">Outermost numbers indicate chromosomes. Lines link the genomic location of
rs28929474 with genes encoding significantly associated proteins. Associations
with and without asterisks indicate significance at <em>P</em> &lt;
5 × 10<sup>−8</sup> and <em>P</em> &lt; 1.5 ×
10<sup>−11</sup>, respectively. Line thickness is proportional to
effect size (red, positive; blue, negative); <em>n</em> = 3,301
participants.</p></div></div></div><p id="P16">GWAS have identified thousands of loci associated with common diseases, but
the mechanisms by which most variants influence disease susceptibility are unknown.
To identify intermediate links between genotype and disease, we overlapped pQTLs
with disease-associated variants from GWAS. Eighty-eight of our sentinel pQTL
variants were in high LD (<em>r</em><sup>2</sup> ≥ 0.8) with sentinel
disease-associated variants (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 14</a>), including 30 with <em>cis</em>
associations, 54 with <em>trans</em>, and 4 with both. As some genetic loci
are associated with multiple diseases, these 88 variants represent 253 distinct
genotype–disease associations. Overlap of a pQTL and a disease association
does not necessarily imply that the same genetic variant underlies both traits,
because there may be distinct causal variants for each trait that are in LD. We
therefore performed colocalization testing (see <a href="#S8" rid="S8" class=" sec">Methods</a>). Of 108 locus–disease associations outside the major
histocompatibility (MHC) region for which testing was possible, colocalization was
highly likely (PP &gt; 0.8) for 96 (88.9%), and the most likely explanation (PP
&gt; 0.5) for 106 (98.1%) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 14</a>).</p><p id="P17"><em>trans</em> pQTLs that overlap with disease associations can
highlight previously unsuspected candidate proteins through which genetic loci may
influence disease risk. To help to identify such candidates, we applied the ProGeM
framework<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750417" role="button" aria-expanded="false" aria-haspopup="true">22</a></sup> (<a href="#S8" rid="S8" class=" sec">Methods</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 12</a>, <a href="/pmc/articles/PMC6697541/figure/F12/" target="figure" class="fig-table-link figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Extended
Data Fig. 7</span></a>). We show that an inflammatory bowel disease (IBD) risk
allele<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750414" role="button" aria-expanded="false" aria-haspopup="true">23</a></sup> (rs3197999:A,
p.Arg703Cys) in <em>MST1</em> on chromosome 3, which decreases plasma MST1
levels<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750450" role="button" aria-expanded="false" aria-haspopup="true">24</a></sup>, is a
<em>trans</em> pQTL for eight additional proteins (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>, <a href="/pmc/articles/PMC6697541/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). Notably, genes that encode three of
these proteins (<em>PRDM1</em>, <em>FASLG</em> and
<em>DOCK9</em>) each lie within 500 kb of IBD GWAS loci at which the
causal gene is ambiguous<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750457" role="button" aria-expanded="false" aria-haspopup="true">25</a></sup>. For
instance, the IBD-associated variant rs6911490 lies on chromosome 6 in the
intergenic region near <em>PRDM1</em> (encoding BLIMP1, a master regulator
of immune cell differentiation) and <em>ATG5</em> (involved in autophagy)
(<a href="/pmc/articles/PMC6697541/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3c</span></a>). Neither fine-mapping nor eQTL
colocalization analyses have unequivocally resolved the causal gene at this
locus<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750422" role="button" aria-expanded="false" aria-haspopup="true">25</a></sup>; both
<em>PRDM1</em>and <em>ATG5</em> are plausible candidates. Our
data provide support for <em>PRDM1</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC6697541/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142947669904" class="figure"><a href="/pmc/articles/PMC6697541/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f003.jpg"></a></div><div id="largeobj_idm140142947669904" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6697541/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC6697541/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig. 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong><em>trans</em> pQTL for BLIMP1 at an inflammatory bowel disease (IBD)
associated missense variant (rs3197999:A) in <em>MST1</em>.</strong><p id="__p4"><strong>a</strong>, rs3197999:A is associated with multiple proteins. Lines link
rs3197999 and the genes encoding significantly associated proteins. Line
thickness is proportional to effect size of the IBD risk allele (red = positive,
blue = negative). <em>n</em> = 3,301 participants. Asterisks indicate
genes in IBD GWAS loci. <strong>b</strong>, Regional association plots at
<em>MST1</em>, showing IBD association (top) and
<em>trans</em> pQTLs for BLIMP1, DOCK9 and FASLG. Colour key
indicates <em>r</em><sup>2</sup> with rs3197999. <strong>c</strong>,
Regional association plot of the IBD susceptibility locus at
<em>PRDM1</em>, which encodes BLIMP1. IBD association data are for
European participants from a GWAS meta-analysis<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750426" role="button" aria-expanded="false" aria-haspopup="true">23</a></sup>.</p></div></div></div><p id="P18">Anti-neutrophil cytoplasmic antibody-associated vasculitis (AAV) is an
autoimmune disease characterized by vascular inflammation and autoantibodies to the
neutrophil proteases proteinase-3 (PR3) or myeloperoxidase. GWAS have revealed
distinct genetic associations according to antibody specificity<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750461" role="button" aria-expanded="false" aria-haspopup="true">26</a></sup>, with variants near
<em>PRTN3</em> (encoding PR3) and at the Z-allele of
<em>SERPINA1</em> (encoding A1AT, which inhibits PR3) associated
specifically with PR3-antibody positive AAV. The SOMAscan assay has two SOMAmers
that target PR3; we identified a <em>cis</em> pQTL immediately upstream of
<em>PRTN3</em> for both, and replicated it with the Olink assay (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>, <a href="/pmc/articles/PMC6697541/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4a, b</span></a>). Conditional analysis revealed
multiple independently associated variants (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>), one of which (rs7254911) was in high LD
with the previously reported<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750463" role="button" aria-expanded="false" aria-haspopup="true">26</a>,<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750395" role="button" aria-expanded="false" aria-haspopup="true">27</a></sup> PR3+ vasculitis-associated
variants in the <em>PRTN3</em> region (<a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Note</a>). We show
that the vasculitis risk allele at <em>PRTN3</em> is associated with higher
plasma levels of PR3 (<a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Note Table 4</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC6697541/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142931174800" class="figure"><a href="/pmc/articles/PMC6697541/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f004.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f004.jpg"></a></div><div id="largeobj_idm140142931174800" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6697541/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC6697541/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Fig. 4</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Proteinase-3, <em>SERPINA1</em> and vasculitis.</strong><p id="__p5"><strong>a</strong>, Manhattan plots for plasma PR3 measured with two SOMAmers and the
Olink assay.</p><p id="__p6"><strong>b</strong>, <em>PRTN3</em> regional association plots. Colour key
indicates <em>r</em><sup>2</sup> with sentinel variant rs10425544.
‘Vasculitis GWAS’: previously reported vasculitis-associated
variants (see <a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary
Note</a>). EVGC, rs62132295 (from European Vasculitis Genetics
Consortium<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750393" role="button" aria-expanded="false" aria-haspopup="true">26</a></sup>); VCRCi,
rs138303849 and VCRCt, rs62132293 (most significant imputed and genotyped
variants, respectively, from Vasculitis Clinical Research Consortium<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750447" role="button" aria-expanded="false" aria-haspopup="true">27</a></sup>). ‘Independent
pQTLs’: conditionally independent PR3 pQTL variants (black lettering
shows lead variant for both SOMAmers; purple and green show conditionally
independent variants for SOMAmers PRTN3.3514.49.2 and PRTN3.13720.95.3,
respectively). <strong>c</strong>, Proposed mechanisms by which
<em>PRTN3</em> and <em>SERPINA1</em> affect PR3 levels and
thus vasculitis risk. Left, individuals without either the
<em>PRTN3</em> or <em>SERPINA1</em> vasculitis risk alleles.
Middle, <em>SERPINA1</em> Z-allele carriers have lower circulating A1AT,
resulting in higher free plasma PR3. Right, <em>cis</em>-acting variant
at the <em>PRTN3</em> locus results in higher total plasma PR3.
Increases in either free or total PR3 predispose individuals to loss of immune
tolerance.</p></div></div></div><p id="P19">For one PR3 SOMAmer, we also found a <em>trans</em> pQTL at
<em>SERPINA1</em>, with the Z-allele being associated with reduced
levels of plasma PR3 (<a href="/pmc/articles/PMC6697541/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4a</span></a>). To understand
the SOMAmer-specific nature of this association, we assayed the relative affinity of
these SOMAmers for the free and complexed forms of PR3 and A1AT. We found that the
SOMAmer showing <em>cis</em> and <em>trans</em> associations
predominantly measured the PR3–A1AT complex rather than free PR3, whereas the
SOMAmer with only a <em>cis</em> association measured both the free and
complexed forms (<a href="/pmc/articles/PMC6697541/figure/F13/" target="figure" class="fig-table-link figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Extended Data Fig. 8</span></a>, <a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Note</a>). Notably,
neither SOMAmer bound free A1AT, demonstrating that the <em>SERPINA1</em>
pQTL did not reflect non-specific cross-reactivity (<a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Note</a>).</p><p id="P20" class="p p-last">These data show that the vasculitis risk allele at <em>PRTN3</em>
increases total PR3 plasma levels, consistent with its effect on
<em>PRTN3</em> mRNA abundance in whole blood in GTEx data<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750446" role="button" aria-expanded="false" aria-haspopup="true">28</a></sup>. The <em>SERPINA1</em>
Z-allele results in a reduced proportion of PR3 bound to A1AT. We thus demonstrate
that altered availability of PR3, conferred by two independent genetic mechanisms,
is a key susceptibility factor for breaking immune tolerance to PR3 and the
development of PR3+ vasculitis (<a href="/pmc/articles/PMC6697541/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4c</span></a>).</p></div><div id="S4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S4title">Causal evaluation of candidate proteins in disease</h2><p id="P21" class="p p-first">Association of plasma protein levels with disease risk does not necessarily
imply causation. To help to establish causality, we used Mendelian randomization
(MR) analysis<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750439" role="button" aria-expanded="false" aria-haspopup="true">11</a></sup>, which uses
genetic variants as instrumental variables to avoid confounding and reverse
causation (<a href="/pmc/articles/PMC6697541/figure/F14/" target="figure" class="fig-table-link figpopup" rid-figpopup="F14" rid-ob="ob-F14" co-legend-rid="lgnd_F14"><span>Extended Data Fig. 9</span></a>). If a genetic
variant is specifically associated with levels of a protein and is also associated
with disease risk, then this provides evidence of the protein’s causal role.
For example, serum levels of PSP-94 (also known as MSMB) are lower in men who go on
to develop prostate cancer<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750435" role="button" aria-expanded="false" aria-haspopup="true">29</a></sup>, but
it is unclear whether this association is correlative or causal. We identified a
<em>cis</em> pQTL associated with lower PSP-94 plasma levels that
overlaps with the prostate cancer susceptibility variant rs10993994<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750427" role="button" aria-expanded="false" aria-haspopup="true">30</a></sup>, supporting a protective role for
PSP-94 in prostate cancer (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 14</a>).</p><p id="P22">Next, we leveraged multi-variant MR analysis methods to identify causal
proteins among multiple plausible candidates, exemplified by the
<em>IL1RL1–IL18R1</em> locus, which is associated with multiple
immunemediated diseases including atopic dermatitis<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750430" role="button" aria-expanded="false" aria-haspopup="true">31</a></sup>. We identified four proteins that each had
<em>cis</em> pQTLs at this locus (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>), and
created a genetic score for each protein (see <a href="#S8" rid="S8" class=" sec">Methods</a>). Initial ‘one-protein-at-a-time’ analysis
identified associations of the scores for IL18R1 (<em>P</em> = 9.3 ×
10<sup>−72</sup>) and IL1RL1 (<em>P</em> = 5.7 ×
10<sup>−27</sup>) with atopic dermatitis risk (<a href="/pmc/articles/PMC6697541/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5a</span></a>), and a weak association for IL1RL2 (<em>P</em>
= 0.013). We then mutually adjusted these associations for one another to account
for the effects of the variants on multiple proteins. Whereas the association of
IL18R1 remained significant (<em>P</em> = 1.5 ×
10<sup>−28</sup>), the association of IL1RL1 (<em>P</em> = 0.01)
was attenuated. In contrast, the association of IL1RL2 (<em>P</em> = 1.1
× 10<sup>−69</sup>) became much stronger, suggesting that IL1RL2 and
IL18R1 underlie atopic dermatitis risk at this locus.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC6697541/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142931168816" class="figure"><a href="/pmc/articles/PMC6697541/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f005.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f005.jpg"></a></div><div id="largeobj_idm140142931168816" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6697541/figure/F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC6697541/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Fig. 5</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Evaluation of causal role of proteins in disease.</strong><p id="__p7"><em>n</em> = 3,301 participants. <strong>a</strong>, MR estimates with 95%
confidence intervals (CIs) (instrumental variable analysis) for proteins encoded
in the <em>IL1RL1</em>– <em>IL18R1</em> locus and risk of
atopic dermatitis (AD) risk. Univariable MR not possible for IL1R1 and IL18RAP
(no significant pQTLs to select as ‘genetic instruments’).
<strong>b</strong>, MMP-12 levels and risk of coronary heart disease (CHD). Top,
MR estimates with 95% CIs. Bottom, estimated effect sizes (with 95% CIs) on
plasma MMP-12 (from linear regression) and CHD risk (from logistic regression)
for each variant used in the genetic score.</p></div></div></div><p id="P23" class="p p-last">MMP-12 plays a key role in lung tissue damage, and MMP-12 inhibitors are
being tested as treatments for chronic obstructive pulmonary disease<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750383" role="button" aria-expanded="false" aria-haspopup="true">32</a></sup>. We created a multi-allelic
genetic score that explains 14% of the variation in plasma MMP-12 levels (see <a href="#S8" rid="S8" class=" sec">Methods</a>). Observational studies reveal that
higher levels of plasma MMP-12 are associated with recurrent cardiovascular
events<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750382" role="button" aria-expanded="false" aria-haspopup="true">33</a></sup>, stimulating
interest in the use of MMP-12 inhibitors to treat cardiovascular disease. However,
we found that genetic predisposition to higher MMP-12 levels is associated with
decreased coronary disease risk (<em>P</em> = 2.8 ×
10<sup>−13</sup>) (<a href="/pmc/articles/PMC6697541/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5b</span></a>) and
decreased large artery atherosclerotic stroke risk<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750407" role="button" aria-expanded="false" aria-haspopup="true">34</a></sup>. It will be important to understand the
discordance between the observational epidemiology and the genetic risk score, given
the therapeutic interest in this target.</p></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Drug target prioritization</h2><p id="P24" class="p p-first">Drugs directed at targets with human genetic support have a greater
likelihood of therapeutic success than those directed at unsupported
targets<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750425" role="button" aria-expanded="false" aria-haspopup="true">35</a></sup>. Of the proteins
for which we identified a pQTL, 244 (17%) are established drug targets in the
Informa Pharmaprojects database (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 15</a>). Thirty-one pQTLs for drug target proteins were
highly likely to colocalize (PP &gt; 0.8) with a GWAS disease locus, including
some that are targets of approved drugs such as tocilizumab (anti-IL6R) and
ustekinumab (anti-IL12/23) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 16a</a>).</p><p id="P25">To identify additional indications for existing drugs, we investigated
disease associations of pQTLs for proteins already targeted by licensed drugs. Our
results suggest potential drug repurposing opportunities. For example, we identified
a <em>cis</em> pQTL for RANK (encoded by <em>TNFRSF11A</em>) at
rs884205, a variant associated with Paget’s disease<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750400" role="button" aria-expanded="false" aria-haspopup="true">36</a></sup>, which is characterized by excessive bone
turnover, deformity and fracture (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 16b</a>). The standard treatment for Paget’s
disease is osteoclast inhibition with bisphosphonates, originally developed as
anti-osteoporotic drugs. Denosumab, another anti-osteoporosis drug, is a monoclonal
antibody targeting RANKL, the ligand for RANK. Our data suggest that denosumab may
be an alternative treatment for Paget’s disease when bisphosphonates are
contraindicated, a hypothesis supported by clinical case reports<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750442" role="button" aria-expanded="false" aria-haspopup="true">37</a></sup>.</p><p id="P26" class="p p-last">Next we evaluated targets of drugs currently under development. Drugs
targeting GP1BA, a receptor for von Willebrand factor, are in preclinical
development as anti-thrombotic agents and in phase 2 trials for thrombotic
thrombocytopenic purpura. We found a <em>cis</em> pQTL associated with both
higher GP1BA abundance and higher platelet count, suggesting a link between GP1BA
and platelet count (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary
Table 16</a>). Furthermore, we identified a <em>trans</em> pQTL for
GP1BA at the <em>SH2B3</em>–<em>BRAP</em> locus, which
colocalized with associations with platelet count<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750440" role="button" aria-expanded="false" aria-haspopup="true">10</a></sup>, myocardial infarction and stroke (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 16b</a>). The
risk allele for cardiovascular disease increases both plasma GP1BA and platelet
count, suggesting that GP1BA influences vascular risk via platelets. Collectively,
these results support targeting GP1BA in conditions characterized by platelet
aggregation such as arterial thrombosis. More generally, our data provide a
substrate for generating hypotheses about potential therapeutic targets through
linking genetic factors to disease via specific proteins.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Discussion</h2><p id="P27" class="p p-first-last">This study elucidates the genetic control of the human plasma proteome and
uncovers intermediate molecular pathways that connect the genome to disease
endpoints. We applied our discoveries to evaluate causal roles for proteins in human
diseases using the principle of Mendelian randomization. Proteins provide an ideal
paradigm for MR analysis because they are under proximal genetic control. However,
application of protein-based MR has been constrained by limited availability of
suitable genetic instruments, a bottleneck remedied by our approach. Our study
provides a resource for understanding complex traits and an example of the
application of novel bioassay technologies to population biobanks.</p></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">Online content</h2><p id="P28" class="p p-first-last">Any Methods, including any statements of data availability and Nature
Research reporting summaries, along with any additional references and Source Data
files, are available in the online version of the paper at <a href="https://doi.org/10.1038/s41586-018-0175-2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://doi.org/10.1038/s41586-018-0175-2</a></p></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">Methods</h2><div id="S9" class="sec sec-first"><h3 id="S9title">Study participants</h3><p id="P29" class="p p-first-last">The INTERVAL study comprises about 50,000 participants nested within a
randomized trial of varying blood donation intervals<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750377" role="button" aria-expanded="false" aria-haspopup="true">9</a></sup>. Between mid-2012 and mid-2014, blood donors
aged 18 years and older were recruited at 25 centres of England’s
National Health Service Blood and Transplant (NHSBT). All participants gave
informed consent before joining the study and the National Research Ethics
Service approved this study (11/EE/0538). Participants completed an online
questionnaire including questions about demographic characteristics (for
example, age, sex, ethnicity), anthropometry (height, weight), lifestyle (for
example, alcohol and tobacco consumption) and diet. Participants were generally
in good health because blood donation criteria exclude people with a history of
major diseases (such as myocardial infarction, stroke, cancer, HIV, and
hepatitis B or C) and those who have had recent illness or infection. For
SomaLogic assays, we randomly selected two non-overlapping subcohorts of 2,731
and 831 participants from INTERVAL. After genetic quality control, 3,301
participants (2,481 and 820 in the two subcohorts) remained for analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 17</a>).
No statistical methods were used to determine sample size. The experiments were
not randomized. Laboratory staff conducting proteomic assays were blinded to the
genotypes of participants.</p></div><div id="S10" class="sec"><h3 id="S10title">Plasma sample preparation</h3><p id="P30" class="p p-first-last">Sample collection procedures for INTERVAL have been described
previously<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750387" role="button" aria-expanded="false" aria-haspopup="true">38</a></sup>. In
brief, blood samples for research purposes were collected in 6-ml EDTA tubes
using standard venepuncture protocols. The tubes were inverted three times and
transferred at ambient temperature to UK Biocentre (Stockport, UK) for
processing. Plasma was extracted into two 0.8-ml plasma aliquots by
centrifugation and subsequently stored at −80 °C before use.</p></div><div id="S11" class="sec"><h3 id="S11title">Protein measurements</h3><p id="P31" class="p p-first">We used a multiplexed, aptamer-based approach (SOMAscan assay) to
measure the relative concentrations of 3,622 plasma proteins or protein
complexes assayed using 4,034 modified aptamers (‘SOMAmer
reagents’, hereafter referred to as SOMAmers; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 18</a>).
The assay extends the lower limit of detectable protein abundance afforded by
conventional approaches (for example, immunoassays), measuring both
extracellular and intracellular proteins (including soluble domains of
membrane-associated proteins), with a bias towards proteins likely to be found
in the human secretome<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750432" role="button" aria-expanded="false" aria-haspopup="true">8</a>,<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750381" role="button" aria-expanded="false" aria-haspopup="true">39</a></sup> (<a href="/pmc/articles/PMC6697541/figure/F15/" target="figure" class="fig-table-link figpopup" rid-figpopup="F15" rid-ob="ob-F15" co-legend-rid="lgnd_F15"><span>Extended Data Fig. 10a</span></a>). The proteins cover a wide range of
molecular functions (<a href="/pmc/articles/PMC6697541/figure/F15/" target="figure" class="fig-table-link figpopup" rid-figpopup="F15" rid-ob="ob-F15" co-legend-rid="lgnd_F15"><span>Extended Data Fig.
10b</span></a>). The selection of proteins on the platform reflects both the
availability of purified protein targets and a focus on proteins suspected to be
involved in the pathophysiology of human disease.</p><p id="P32">Aliquots of 150 μl of plasma were sent on dry ice to SomaLogic
Inc. (Boulder, Colorado, US) for protein measurement. Assay details have been
previously described<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750441" role="button" aria-expanded="false" aria-haspopup="true">39</a>,<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750390" role="button" aria-expanded="false" aria-haspopup="true">40</a></sup> and a technical white paper
with further information can be found at the manufacturer’s website
(<a href="http://somalogic.com/wp-content/uploads/2017/06/SSM-002-Technical-White-Paper_010916_LSM1.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://somalogic.com/wp-content/uploads/2017/06/SSM-002-Technical-White-Paper_010916_LSM1.pdf</a>).
In brief, modified single-stranded DNA SOMAmers are used to bind to specific
protein targets that are then quantified using a DNA microarray. Protein
concentrations are quantified as relative fluorescent units.</p><p id="P33">Quality control (QC) was performed at the sample and SOMAmer levels
using control aptamers and calibrator samples. At the sample level,
hybridization controls on the microarray were used to correct for systematic
variability in hybridization, while the median signal over all features assigned
to one of three dilution sets (40%, 1% and 0.005%) was used to correct for
within-run technical variability. The resulting hybridization scale factors and
median scale factors were used to normalize data across samples within a run.
The acceptance criteria for these values are between 0.4 and 2.5 based on
historical runs. SOMAmer-level QC made use of replicate calibrator samples using
the same study matrix (plasma) to correct for between-run variability. The
acceptance criterion for each SOMAmer was that the calibration scale factor be
less than 0.4 from the median for each of the plates run. In addition, at the
plate level, the acceptance criteria were that the median of the calibration
scale factors be between 0.8 and 1.2, and that 95% of individual SOMAmers be
less than 0.4 from the median within the plate.</p><p id="P34">In addition to QC processes routinely conducted by SomaLogic, we
measured protein levels of 30 and 10 pooled plasma samples randomly distributed
across plates for subcohort 1 and subcohort 2, respectively. Laboratory
technicians were blinded to the presence of pooled samples. This approach
enabled estimation of the reproducibility of the protein assays. We calculated
the coefficient of variation (CV) for each SOMAmer within each subcohort by
dividing the standard deviation by the mean of the pooled plasma sample protein
read-outs. In addition to passing SomaLogic QC processes, we required SOMAmers
to have a CV ≤ 20% in both subcohorts. Eight non-human protein targets
were also excluded, leaving 3,283 SOMAmers (mapping to 2,994 unique proteins or
protein complexes) for inclusion in the GWAS.</p><p id="P35" class="p p-last">Protein mapping to UniProt identifiers and gene names was provided by
SomaLogic. Mapping to Ensembl gene IDs and genomic positions was performed using
Ensembl Variant Effect Predictor v83 (VEP)<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750455" role="button" aria-expanded="false" aria-haspopup="true">41</a></sup>. Protein subcellular locations were determined by
exporting the subcellular location annotations from UniProt<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750398" role="button" aria-expanded="false" aria-haspopup="true">42</a></sup>. If the term
‘membrane’ was included in the descriptor, the protein was
considered to be a membrane protein, whereas if the term
‘secreted’ (but not ‘membrane’) was included in the
descriptor, the protein was considered to be a secreted protein. Proteins not
annotated as either membrane or secreted proteins were classified (by inference)
as intracellular proteins. Proteins were mapped to molecular functions using
gene ontology annotations<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750453" role="button" aria-expanded="false" aria-haspopup="true">43</a></sup>
from UniProt.</p></div><div id="S12" class="sec"><h3 id="S12title">Non-genetic associations of proteins</h3><p id="P36" class="p p-first-last">To provide confidence in the reproducibility of the protein assays, we
attempted to replicate the associations with age or sex of 45 proteins
previously reported by Ngo et al. and 40 reported by Menni et al.<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750380" role="button" aria-expanded="false" aria-haspopup="true">44</a>,<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750423" role="button" aria-expanded="false" aria-haspopup="true">45</a></sup>. We used Bonferroni-corrected <em>P</em> value
thresholds of <em>P</em> = 1.1 × 10<sup>−3</sup> (0.05/45)
and <em>P</em> = 1.2 × 10<sup>−3</sup> (0.05/40),
respectively. Relative protein abundances were rank-inverse normalized within
each subcohort and linear regression was performed using age, sex, body mass
index, natural log of estimated glomerular filtration rate (eGFR) and subcohort
as independent variables.</p></div><div id="S13" class="sec"><h3 id="S13title">Genotyping and imputation</h3><p id="P37" class="p p-first">The genotyping protocol and QC for the INTERVAL samples
(<em>n</em> ≈ 50,000) have been described previously in
detail<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750403" role="button" aria-expanded="false" aria-haspopup="true">10</a></sup>. DNA
extracted from buffy coat was used to assay approximately 830,000 variants on
the Affymetrix Axiom UK Biobank genotyping array at Affymetrix (Santa Clara,
California, US). Genotyping was performed in multiple batches of approximately
4,800 samples each. Sample QC was performed including exclusions for sex
mismatches, low call rates, duplicate samples, extreme heterozygosity and
non-European descent. Relatedness was removed by excluding one participant from
each pair of close (first- or second-degree) relatives, defined as
<span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M1" overflow="scroll" role="math" style="width: 5.453em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.494em; height: 0px; font-size: 121%;"><span style="position: absolute; clip: rect(1.313em, 1004.42em, 2.416em, -1000em); top: -2.221em; left: 0em;"><span class="mrow" id="MathJax-Span-2"><span class="mrow" id="MathJax-Span-3"><span class="mover" id="MathJax-Span-4"><span style="display: inline-block; position: relative; width: 0.591em; height: 0px;"><span style="position: absolute; clip: rect(3.391em, 1000.57em, 4.143em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-5" style="font-family: MathJax_Math; font-style: italic;">π<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.078em, 1000.56em, 3.581em, -1000em); top: -3.986em; left: 0.035em;"><span class="mo" id="MathJax-Span-6" style=""><span style="font-family: MathJax_Size1;">ˆ</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-7" style="font-family: MathJax_Main; padding-left: 0.278em;">&gt;</span><span class="mn" id="MathJax-Span-8" style="font-family: MathJax_Main; padding-left: 0.278em;">0.187.</span></span></span><span style="display: inline-block; width: 0px; height: 2.221em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.111em; border-left: 0px solid; width: 0px; height: 1.085em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M1" overflow="scroll"><mrow><mover accent="true"><mi>π</mi><mo>^</mo></mover><mo>&gt;</mo><mn>0.187.</mn></mrow></math></script> Identity-by-descent was estimated using a
subset of variants with a call rate &gt;99% and MAF &gt; 5% in the
merged data set of both subcohorts, pruned for linkage disequilibrium (LD) using
PLINK v1.9<sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750418" role="button" aria-expanded="false" aria-haspopup="true">46</a></sup>. Numbers of
participants excluded at each stage of the genetic QC are summarized in <a href="/pmc/articles/PMC6697541/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Extended Data Fig. 1</span></a>. Multi-dimensional
scaling was performed using PLINK v1.9 to create components to account for
ancestry in genetic analyses.</p><p id="P38">Prior to imputation, additional variant filtering steps were performed
to establish a high-quality imputation scaffold. In summary, 654,966
high-quality variants (auto-somal, non-monomorphic, bi-allelic variants with
Hardy–Weinberg Equilibrium (HWE) <em>P</em> &gt; 5 ×
10<sup>−6</sup>, with a call rate of &gt;99% across the INTERVAL
genotyping batches in which a variant passed QC, and a global call rate of
&gt;75% across all INTERVAL genotyping batches) were used for imputation.
Variants were phased using SHAPEIT3 and imputed using a combined 1000 Genomes
Phase 3-UK10K reference panel. Imputation was performed via the Sanger
Imputation Server (<a href="https://imputation.sanger.ac.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://imputation.sanger.ac.uk</a>) and resulted in 87,696,888
imputed variants.</p><p id="P39" class="p p-last">Prior to genetic association testing, variants were filtered in each
subcohort separately using the following exclusion criteria: (1) imputation
quality (INFO) score &lt;0.7; (2) minor allele count &lt;8; (3) HWE
<em>P</em> &lt; 5 × 10<sup>−6</sup>. In the small
number of cases in which imputed variants had the same genomic position (GRCh37)
and alleles, the variant with the lowest INFO score was removed. 10,572,788
variants passing all filters in both subcohorts were taken forward for analysis
(<a href="/pmc/articles/PMC6697541/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Extended Data Fig. 1</span></a>).</p></div><div id="S14" class="sec"><h3 id="S14title">Genome-wide association study</h3><p id="P40" class="p p-first-last">Within each subcohort, relative protein abundances were first natural
log-transformed. Log-transformed protein levels were then adjusted in a linear
regression for age, sex, duration between blood draw and processing (binary,
≤1 day/&gt;1day) and the first three principal components of ancestry
from multi-dimensional scaling. The protein residuals from this linear
regression were then rank-inverse normalized and used as phenotypes for
association testing. Simple linear regression using an additive genetic model
was used to test genetic associations. Association tests were carried out on
allelic dosages to account for imputation uncertainty (‘-method
expected’ option) using SNPTEST v2.5.2<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750402" role="button" aria-expanded="false" aria-haspopup="true">47</a></sup>.</p></div><div id="S15" class="sec"><h3 id="S15title">Meta-analysis and statistical significance</h3><p id="P41" class="p p-first-last">Association results from the two subcohorts were combined via
fixed-effects inverse-variance meta-analysis combining the betas and standard
errors using METAL<sup><a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750394" role="button" aria-expanded="false" aria-haspopup="true">48</a></sup>. Genetic
associations were considered to be genome-wide significant based on a
conservative strategy requiring associations to have (i) a meta-analysis
<em>P</em> value &lt; 1.5 × 10<sup>−11</sup>
(genome-wide threshold of <em>P</em> = 5 × 10<sup>−8</sup>
Bonferroni-corrected for 3,283 aptamers tested), (ii) at least nominal
significance (<em>P</em> &lt; 0.05) in both subcohorts, and (iii)
consistent direction of effect across subcohorts. We did not observe significant
genomic inflation (mean inflation factor was 1.0, standard deviation = 0.01)
(<a href="/pmc/articles/PMC6697541/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Extended Data Fig. 3d</span></a>).</p></div><div id="S16" class="sec"><h3 id="S16title">Refinement of significant regions</h3><p id="P42" class="p p-first-last">To identify distinct non-overlapping regions associated with a given
SOMAmer, we first defined a 1-Mb region around each significant variant for that
SOMAmer. Starting with the region containing the variant with the smallest
<em>P</em> value, any overlapping regions were then merged and this
process was repeated until no more overlapping 1-Mb regions remained. The
variant with the lowest <em>P</em> value for each region was assigned as
the ‘regional sentinel variant’. Owing to the complexity of the
MHC region, we treated the extended MHC region (chr6:25.5–34.0Mb) as one
region. To identify whether a region was associated with multiple SOMAmers, we
used an LD-based clumping approach. Regional sentinel variants in high LD
(<em>r</em><sup>2</sup> ≥ 0.8) with each other were combined
together into a single region.</p></div><div id="S17" class="sec"><h3 id="S17title">Conditional analyses</h3><p id="P43" class="p p-first-last">To identify conditionally significant associations, we performed
approximate genome-wide stepwise conditional analysis using GCTA
v1.25.2<sup><a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750415" role="button" aria-expanded="false" aria-haspopup="true">49</a></sup> using the
‘cojo-slct’ option. We used the same conservative significance
threshold of <em>P</em> = 1.5 × 10<sup>−11</sup> as for
the univariable analysis. As inputs for GCTA, we used the summary statistics
(that is, betas and standard errors) from the meta-analysis. Correlation between
variants was estimated using the ‘hard-called’ genotypes (where a
genotype was called if it had a posterior probability of &gt;0.9 following
imputation or set to missing otherwise) in the merged genetic data set, and only
variants also passing the univariable genome-wide threshold (<em>P</em>
&lt; 1.5 × 10<sup>−11</sup>) were considered for stepwise
selection. As the conditional analyses use different data inputs to the
univariable analysis (that is, summarized rather than individual-level data),
there were some instances where the conditional analysis failed to include in
the stepwise selection sentinel variants that were only just statistically
significant in the univariable analysis. In these instances (<em>n</em>
= 28), we re-conducted the joint model estimation without stepwise selection in
GCTA, using the variants identified by the conditional analysis in addition to
the regional sentinel variant. We report and highlight these cases in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table
5</a>.</p></div><div id="S18" class="sec"><h3 id="S18title">Replication of previous pQTLs</h3><p id="P44" class="p p-first">We attempted to identify all previously reported pQTLs from GWAS and to
assess whether they replicated in our study. We used the NCBI Entrez programming
utility in R (rentrez) to perform a literature search for pQTL studies published
from 2008 onwards. We searched for the following terms: ‘pQTL’,
‘pQTLs’, and ‘protein quantitative trait locus’. We
supplemented this search by filtering out GWAS associations from the NHGRI-EBI
GWAS Catalog v.1.0.1<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750444" role="button" aria-expanded="false" aria-haspopup="true">50</a></sup>
(<a href="https://www.ebi.ac.uk/gwas/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ebi.ac.uk/gwas/</a>, downloaded November 2017), which
has all phenotypes mapped to the Experimental Factor Ontology (EFO)<sup><a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750448" role="button" aria-expanded="false" aria-haspopup="true">51</a></sup>, by restricting to those with
EFO annotations relevant to protein biomarkers (for example, ‘protein
measurement’, EFO_0004747). Studies identified through both approaches
were manually filtered to include only studies that profiled plasma or serum
samples and to exclude studies not assessing proteins. We recorded basic summary
information for each study including the assay used, sample size and number of
proteins with pQTLs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 19</a>). To reduce the impact of ethnic differences
in allele frequencies on replication rate estimates, we filtered studies to
include only associations reported in European-ancestry populations. We then
manually extracted summary data on all reported associations from the manuscript
or the <a href="#SM" rid="SM" class=" supplementary-material">supplementary
material</a>. This included rsID, protein UniProt ID, <em>P</em>
values, and whether the association was <em>cis</em> or
<em>trans</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 20</a>).</p><p id="P45" class="p p-last">To assess replication we first identified the set of unique UniProt IDs
that were also assayed on the SOMAscan panel. For previous studies that used
SomaLogic technology, we refined this match to the specific aptamer used. We
then clumped associations into distinct loci using the same method that we
applied to our pQTLs (see ‘Refinement of significant regions’).
For each locus, we asked whether the sentinel SNP or a proxy
(<em>r</em><sup>2</sup> &gt; 0.6) was associated with the same
protein (or aptamer) in our study at a defined significance threshold. For our
primary assessment, we used a <em>P</em> value threshold of
10<sup>−4</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 21</a>). We also performed sensitivity analyses to
explore factors that influence replication rate (<a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Note</a>).</p></div><div id="S19" class="sec"><h3 id="S19title">Replication study using Olink assay</h3><p id="P46" class="p p-first-last">To test replication of 163 pQTLs for 116 proteins, we performed protein
measurements using an alternative assay, that is, a proximity extension assay
method (Olink Bioscience, Uppsala, Sweden)<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750408" role="button" aria-expanded="false" aria-haspopup="true">13</a></sup> in an additional subcohort of 4,998 INTERVAL
participants. Proteins were measured using three 92-protein
‘panels’ – ‘inflammatory’,
‘cvd2’ and ‘cvd3’ (10 proteins were assayed on more
than 1 panel). 4,902, 4,947 and 4,987 samples passed quality control for the
‘inflammatory’, ‘cvd2’ and ‘cvd3’
panels, respectively, of which 712, 715 and 721 samples were from individuals
included in our primary pQTL analysis using the SOMAscan assay. Normalized
protein levels (‘NPX’) were regressed on age, sex, plate, time
from blood draw to processing (in days), and season (categorical:
‘Spring’, ‘Summer’, ‘Autumn’,
‘Winter’). The residuals were then rank-inverse normalized.
Genotype data was processed as described earlier. Linear regression of the
rank-inversed normalized residuals on genotype was carried out in SNPTEST with
the first three components of multi-dimensional scaling as covariates to adjust
for ancestry. pQTLs were considered to have replicated if they met a
<em>P</em> value threshold Bonferroni-corrected for the number of
tests (<em>P</em> &lt; 3.1 × 10<sup>−4</sup>;
0.05/163) and had a directionally concordant beta estimate with the SOMAscan
estimate.</p></div><div id="S20" class="sec"><h3 id="S20title">Candidate gene annotation</h3><p id="P47" class="p p-first">We defined a pQTL as <em>cis</em> when the most significantly
associated variant in the region was located within 1 Mb of the TSS of the
gene(s) encoding the protein. pQTLs lying outside of this region were defined as
<em>trans</em>. When considering the distance of the lead
<em>cis</em>-associated variant from the relevant TSS, only proteins
that mapped to single genes on the primary assembly in Ensembl v83 were
considered.</p><p id="P48">For <em>trans</em> pQTLs, we sought to prioritize candidate
genes in the region that might underpin the genotype–protein association.
We applied the ProGeM framework<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750384" role="button" aria-expanded="false" aria-haspopup="true">22</a></sup>, which leverages a combination of databases of molecular
pathways, protein–protein interaction networks, and variant annotation,
as well as functional genomic data including eQTL and chromosome conformation
capture. In addition to reporting the nearest gene to the sentinel variant,
ProGeM employs complementary ‘bottom up’ and ‘top
down’ approaches, starting from the variant and protein respectively. For
the ‘bottom up’ approach, the sentinel variant and corresponding
proxies (<em>r</em><sup>2</sup> &gt; 0.8) for each
<em>trans</em> pQTL were first annotated using Ensembl VEP v83
(using the ‘pick’ option) to determine whether variants were (1)
protein-altering coding variants; (2) synonymous coding or 5′/3′
untranslated region (UTR); (3) intronic or up/downstream; or (4) intergenic.
Second, we queried all sentinel variants and proxies against significant
<em>cis</em> eQTL variants (defined by beta distribution-adjusted
empirical <em>P</em> values using an FDR threshold of 0.05, see
<a href="http://www.gtex-portal.org/home/documentationPage" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.gtex-portal.org/home/documentationPage</a> for details)
in any cell type or tissue from the Genotype-Tissue Expression (GTEx) project
v6<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750436" role="button" aria-expanded="false" aria-haspopup="true">28</a></sup> (<a href="http://www.gtexportal.org/home/datasets" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.gtexportal.org/home/datasets</a>). Third, we also queried
promoter capture Hi-C data in 17 human primary haematopoietic cell
types<sup><a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750396" role="button" aria-expanded="false" aria-haspopup="true">52</a></sup> to identify
contacts (with a CHiCAGO score &gt;5 in at least one cell type) involving
chromosomal regions containing a sentinel variant. We considered gene promoters
annotated on either fragment (that is, the fragment containing the sentinel
variant or the other corresponding fragment) as potential candidate genes. Using
these three sources of information, we generated a list of candidate genes for
the <em>trans</em> pQTLs. A gene was considered a candidate if it
fulfilled at least one of the following criteria: (1) it was proximal
(intragenic or ± 5 kb from the gene) or nearest to the sentinel variant;
(2) it contained a sentinel or proxy variant (<em>r</em><sup>2</sup>
&gt; 0.8) that was protein-altering; (3) it had a significant
<em>cis</em> eQTL in at least one GTEx tissue overlapping with a
sentinel pQTL variant (or proxy); or (4) it was regulated by a promoter
annotated on either fragment of a chromosomal contact<sup><a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750376" role="button" aria-expanded="false" aria-haspopup="true">52</a></sup> involving a sentinel variant.</p><p id="P49">For the ‘top down’ approach, we first identified all genes
with a TSS located within the corresponding pQTL region using the GenomicRanges
Bioconductor package<sup><a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750401" role="button" aria-expanded="false" aria-haspopup="true">53</a></sup> with
annotation from a GRCh37 GTF file from Ensembl
(ftp://ftp.ensembl.org/pub/grch37/update/gtf/homo_sapiens; file:
‘Homo_sapiens. GRCh37.82.gtf.gz’, downloaded June 2016). We then
identified any local genes that had previously been linked with the
corresponding <em>trans</em>-associated protein(s) according to the
following open source databases: (1) the Online Mendelian Inheritance in Man
(OMIM) catalogue<sup><a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750459" role="button" aria-expanded="false" aria-haspopup="true">54</a></sup> (<a href="http://www.omim.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.omim.org/</a>); (2) the Kyoto Encyclopedia of Genes and
Genomes (KEGG)<sup><a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750421" role="button" aria-expanded="false" aria-haspopup="true">55</a></sup> (<a href="http://www.genome.jp/kegg/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.jp/kegg/</a>); and (3) STRINGdb<sup><a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750437" role="button" aria-expanded="false" aria-haspopup="true">56</a></sup> (<a href="http://string-db.org/;v10.0" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://string-db.org/;v10.0</a>). We accessed OMIM data via
HumanMine web tool<sup><a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750385" role="button" aria-expanded="false" aria-haspopup="true">57</a></sup>
(<a href="http://www.humanmine.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.humanmine.org/</a>; accessed June 2016), whereby we
extracted all OMIM IDs for (i) our <em>trans</em>-affected proteins and
(ii) genes local (± 500 kb) to the corresponding
<em>trans</em>-acting variant. We extracted all human KEGG pathway IDs
using the KEGGREST Bioconductor package (<a href="https://bioconductor.org/packages/release/bioc/html/KEGGREST.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://bioconductor.org/packages/release/bioc/html/KEGGREST.html</a>).
In cases where a <em>trans</em>-associated protein shared either an OMIM
ID or a KEGG pathway ID with a gene local to the corresponding
<em>trans</em>-acting variant, we took this as evidence of a
potential functional involvement of that gene. We interrogated
protein–protein interaction data by accessing STRINGdb data using the
STRINGdb Bioconductor package<sup><a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750420" role="button" aria-expanded="false" aria-haspopup="true">58</a></sup>, whereby we extracted all pairwise interaction scores for
each <em>trans</em>-affected protein and all proteins with genes local
to the corresponding <em>trans</em>-acting variants. We took the default
interaction score of 400 as evidence of an interaction between the proteins,
therefore indicating a possible functional involvement for the local gene. In
addition to using data from open source databases in our top down approach, we
also adopted a ‘guilt-by-association’ (GbA) approach using the
same plasma proteomic data used to identify our pQTLs. We first generated a
matrix containing all possible pairwise Pearson’s correlation
coefficients between our 3,283 SOMAmers. We then extracted the coefficients
relating to our <em>trans</em>-associated proteins and any proteins
encoded by genes local to their corresponding <em>trans</em>-acting
variants (where available). Where the correlation coefficient was ≥0.5 we
prioritized the relevant local genes as being potential mediators of the
<em>trans</em> association(s) at that locus.</p><p id="P50" class="p p-last">We report the potential candidate genes for our <em>trans</em>
pQTLs from both the ‘bottom up’ and ‘top down’
approaches, highlighting cases where the same gene was highlighted by both
approaches.</p></div><div id="S21" class="sec"><h3 id="S21title">Functional annotation of pQTLs</h3><p id="P51" class="p p-first-last">Functional annotation of variants was performed using Ensembl VEP v83
using the ‘pick’ option. We tested the enrichment of significant
pQTL variants for certain functional classes by comparing to permuted sets of
variants showing no significant association with any protein (<em>P</em>
&gt; 0.0001 for all proteins tested). First, the regional sentinel variants
were LD-pruned at <em>r</em><sup>2</sup> of 0.1. Each time the sentinel
variants were LD-pruned, one of the pairs of correlated variants was removed at
random and for each set of LD-pruned sentinel variants, 100 equally sized sets
of null permuted variants were sampled matching for MAF (bins of 5%), distance
to TSS (bins of 0–0.5 kb, 0.5–2 kb, 2–5 kb, 5–10 kb,
10–20 kb, 20–100 kb and &gt;100 kb in each direction) and LD
(± half the number of variants in LD with the sentinel variant at
<em>r</em><sup>2</sup> of 0.8). This procedure was repeated 100
times resulting in 10,000 permuted sets of variants. An empirical
<em>P</em> value was calculated as the proportion of permuted
variant sets where the proportion that is classified as a particular functional
group exceeded that of the test set of sentinel pQTL variants, and we used a
significance threshold of <em>P</em> = 0.005 (0.05/10 functional classes
tested).</p></div><div id="S22" class="sec"><h3 id="S22title">Evidence against aptamer-binding effects at <em>cis</em>
pQTLs</h3><p id="P52" class="p p-first-last">All protein assays that rely on binding (for example, of antibodies or
SOMAmers) are susceptible to the possibility of binding-affinity effects, where
protein-altering variants (PAVs) (or their proxies in LD) are associated with
protein measurements owing to differential binding rather than differences in
protein abundance. To account for this potential effect, we performed
conditional analysis at all <em>cis</em> pQTLs where the sentinel
variant was in LD (<em>r</em><sup>2</sup> ≥ 0.1 and
<em>r</em><sup>2</sup> ≤ 0.9) with a PAV in the gene(s)
encoding the associated protein. First, variants were annotated with Ensembl VEP
v83 using the ‘per-gene’ option. Variant annotations were
considered protein-altering if they were annotated as coding sequence variant,
frameshift variant, in-frame deletion, in-frame insertion, missense variant,
protein altering variant, splice acceptor variant, splice donor variant, splice
region variant, start lost, stop gained, or stop lost. To avoid
multi-collinearity, PAVs were LD-pruned (<em>r</em><sup>2</sup> &gt;
0.9) using PLINK v1.9 before including them as covariates in the conditional
analysis on the meta-analysis summary statistics using GCTA v1.25.2. Coverage of
known common (MAF &gt;5%) PAVs in our data was checked by comparison with
exome sequences from ~60,000 individuals in the Exome Aggregation
Consortium (ExAC (<a href="http://exac.broadinstitute.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://exac.broadinstitute.org</a>), downloaded June 2016)<sup><a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750451" role="button" aria-expanded="false" aria-haspopup="true">59</a></sup>.</p></div><div id="S23" class="sec"><h3 id="S23title">Testing for regulatory and functional enrichment</h3><p id="P53" class="p p-first">We tested whether our pQTLs were enriched for functional and regulatory
characteristics using GARFIELD v1.2.0<sup><a href="#R60" rid="R60" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">60</a></sup>. GARFIELD is a non-parametric permutation-based
enrichment method that compares input variants to permuted sets matched for
number of proxies (<em>r</em><sup>2</sup> ≥ 0.8), MAF and
distance to the closest TSS. It first applies ‘greedy pruning’
(<em>r</em><sup>2</sup> &lt; 0.1) within a 1-Mb region of the
most significant variant. GARFIELD annotates variants with more than a thousand
features, drawn predominantly from the GENCODE, ENCODE and ROADMAP projects,
which includes genic annotations, histone modifications, chromatin states and
other regulatory features across a wide range of tissues and cell types.</p><p id="P54" class="p p-last">The enrichment analysis was run using all variants that passed our
Bonferroni-adjusted significance threshold (<em>P</em> &lt; 1.5
× 10<sup>−11</sup>) for association with any protein. For each of
the matching criteria (MAF, distance to TSS, number of LD proxies), we used five
bins. In total we tested 25 combinations of features (classified as
transcription factor binding sites, FAIRE-seq, chromatin states, histone
modifications, footprints, hotspots, or peaks) with up to 190 cell types from 57
tissues, leading to 998 tests. Hence, we considered enrichment with
<em>P</em> &lt; 5 × 10<sup>−5</sup> (0.05/998) to
be statistically significant.</p></div><div id="S24" class="sec"><h3 id="S24title">Disease annotation</h3><p id="P55" class="p p-first-last">To identify diseases with which our pQTLs have been associated, we
queried our sentinel variants and their strong proxies
(<em>r</em><sup>2</sup> ≥ 0.8) against publicly available
disease GWAS data using PhenoScanner<sup><a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750405" role="button" aria-expanded="false" aria-haspopup="true">61</a></sup>. A list of data sets queried is available at <a href="http://www.phenoscanner.medschl.cam.ac.uk/information.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.phenoscanner.medschl.cam.ac.uk/information.html</a>. For
disease GWAS, results were filtered to <em>P</em> &lt; 5 ×
10<sup>−8</sup> and then manually curated to retain only the entry
with the strongest evidence for association (that is, smallest
<em>P</em> value) per disease. Non-disease phenotypes such as
anthropometric traits, intermediate biomarkers and lipids were excluded
manually.</p></div><div id="S25" class="sec"><h3 id="S25title"><em>cis</em> eQTL overlap and enrichment of <em>cis</em>
pQTLs for <em>cis</em> eQTLs</h3><p id="P56" class="p p-first">For each regional sentinel <em>cis</em> pQTL variant, its strong
proxies (<em>r</em><sup>2</sup> ≥ 0.8) were queried against
publicly available eQTL association data using PhenoScanner.
<em>cis</em> eQTL results were filtered to retain only variants with
<em>P</em> &lt; 1.5 × 10<sup>−11</sup>. Only
<em>cis</em> eQTLs for the same gene as the <em>cis</em>
pQTL protein were retained. We tested whether <em>cis</em> pQTLs were
significantly enriched for eQTLs for the corresponding gene compared to null
sets of variants appropriately matched for MAF and distance to nearest TSS. For
this analysis, we restricted eQTL data to GTEx project v6, since this project
provided complete summary statistics across a wide range of tissues and
cell-types, in contrast to many other studies which only report
<em>P</em> values below some significance level. GTEx results were
filtered to contain only variants lying in <em>cis</em> (that is, within
1 Mb) of genes that encode proteins analysed in our study and only variants in
both data sets were used.</p><p id="P57">For the enrichment analysis, the <em>cis</em> pQTL sentinel
variants were first LD-pruned (<em>r</em><sup>2</sup> &lt; 0.1) and
the proportion of sentinel <em>cis</em> pQTL variants that are also
eQTLs at our pQTL significance threshold (<em>P</em> &lt; 1.5
× 10<sup>−11</sup>), conventional genome-wide significance
(<em>P</em> &lt; 5 × 10<sup>−8</sup>) or a nominal
<em>P</em> value threshold (<em>P</em> &lt; 1 ×
10<sup>−5</sup>) for the same protein or gene was compared to a
permuted set of variants that were not pQTLs (<em>P</em> &gt; 0.0001
for all proteins). We generated 10,000 permuted sets of null variants for each
significance threshold matched for MAF, distance to TSS and LD (as described for
functional annotation enrichment in ‘Functional annotation of
pQTLs’). An empirical <em>P</em> value was calculated as the
proportion of permuted variant sets where the proportion that are also
<em>cis</em> eQTLs exceeded that of the test set of sentinel
<em>cis</em> pQTL variants.</p><p id="P58" class="p p-last">At a stringent eQTL significance threshold (<em>P</em> &lt;
1.5 × 10−11), we found significant enrichment of
<em>cis</em> pQTLs for eQTLs (<em>P</em> &lt; 0.0001)
(<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table
11</a>) with 19.5% overlap observed compared to a mean overlap of 1.8% in
the null sets. Results were similar in sensitivity analyses using the standard
genome-wide or nominal significance thresholds as well as when using only the
sentinel variants at <em>cis</em> pQTLs that were robust to adjusting
for PAVs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table
7</a>), suggesting our results are robust to the choice of threshold and
potential differential binding effects.</p></div><div id="S26" class="sec"><h3 id="S26title">Colocalization analysis</h3><p id="P59" class="p p-first-last">Colocalization testing was performed using the coloc package<sup><a href="#R62" rid="R62" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750409" role="button" aria-expanded="false" aria-haspopup="true">62</a></sup>. For testing colocalization of
pQTLs and disease associations, colocalization testing was necessarily limited
to disease traits for which full GWAS summary statistics had been made
available. We obtained GWAS summary statistics through PhenoScanner. For testing
colocalization of pQTLs with eQTLs, we used publically available summary
statistics for expression traits from GTEx<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750458" role="button" aria-expanded="false" aria-haspopup="true">28</a></sup>. We used the default priors. Regions for testing were
determined by dividing the genome into 0.1-cM chunks using recombination data.
Evidence for colocalization was assessed using the posterior probability (PP)
for hypothesis 4 (that there is an association for both traits and they are
driven by the same causal variant(s)). Associations with PP4 &gt; 0.5 were
deemed likely to colocalize as this gives hypothesis 4 the highest likelihood of
being correct, while PP4 &gt; 0.8 was deemed to be ‘highly likely to
colocalize’.</p></div><div id="S27" class="sec"><h3 id="S27title">Selection of genetic instruments for Mendelian randomization</h3><p id="P60" class="p p-first-last">In MR, genetic variants are used as ‘instrumental
variables’ (IVs) for assessing the causal effect of the exposure (here a
plasma protein) on the outcome (here a disease)<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750449" role="button" aria-expanded="false" aria-haspopup="true">11</a>,<a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750460" role="button" aria-expanded="false" aria-haspopup="true">63</a></sup>
(<a href="/pmc/articles/PMC6697541/figure/F14/" target="figure" class="fig-table-link figpopup" rid-figpopup="F14" rid-ob="ob-F14" co-legend-rid="lgnd_F14"><span>Extended Data Fig. 9</span></a>).</p></div><div id="S28" class="sec"><h3 id="S28title">Proteins in the <em>IL1RL1–IL18R1</em> locus and atopic
dermatitis</h3><p id="P61" class="p p-first-last">To identify the likely causal proteins that underpin the previous
genetic association of the <em>IL1RL1–IL18R1</em> locus
(chr11:102.5–103.5Mb) with atopic dermatitis (AD)<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750379" role="button" aria-expanded="false" aria-haspopup="true">31</a></sup>, we used the following
approach. For each protein encoded by a gene in the
<em>IL1RL1–IL18R1</em> locus, we took genetic variants that
had a <em>cis</em> association at <em>P</em> &lt; 1 ×
10<sup>−4</sup> and ‘LD-pruned’ them at
<em>r</em><sup>2</sup> &lt; 0.1 to leave largely independent
variants. We then used these genetic variants to construct a genetic score for
each protein. Formally, we used these variants as instrumental variables for
their respective proteins in univariable MR. For multivariable MR, association
estimates for all proteins in the locus were extracted for all instruments. We
used PhenoScanner to obtain association statistics for the selected variants in
the European-ancestry population of a recent large-scale GWAS meta-analysis of
AD<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750412" role="button" aria-expanded="false" aria-haspopup="true">31</a></sup>. Where the
relevant variant was not available, the strongest proxy with
<em>r</em><sup>2</sup> ≥ 0.8 was used.</p></div><div id="S29" class="sec"><h3 id="S29title">MMP-12 and coronary heart disease (CHD)</h3><p id="P62" class="p p-first-last">To test whether plasma MMP-12 levels have a causal effect on risk of
CHD, we selected genetic variants in the <em>MMP12</em> gene region to
use as instrumental variables. We constructed a genetic score comprising 17
variants that had a <em>cis</em> association with MMP-12 levels at
<em>P</em> &lt; 5 × 10<sup>−8</sup> and that were
not highly correlated with one another (<em>r</em><sup>2</sup> &lt;
0.2). To perform multivariable MR, we used association estimates for these
variants with other MMP proteins in the locus (MMP-1, MMP-7, MMP-8, MMP-10,
MMP-13). Summary associations for variants in the score with CHD were obtained
through PhenoScanner from a recent large-scale GWAS meta-analysis which
consisted mostly (77%) of individuals of European ancestry<sup><a href="#R64" rid="R64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750410" role="button" aria-expanded="false" aria-haspopup="true">64</a></sup>.</p></div><div id="S30" class="sec"><h3 id="S30title">MR analysis</h3><p id="P63" class="p p-first">Two-sample univariable MR was performed for each protein separately
using summary statistics in the inverse-variance weighted method adapted to
account for correlated variants<sup><a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750433" role="button" aria-expanded="false" aria-haspopup="true">65</a>,<a href="#R66" rid="R66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750392" role="button" aria-expanded="false" aria-haspopup="true">66</a></sup>. For each of
<em>G</em> genetic variants (<em>g</em>=1, …,
<em>G</em>) having per-allele estimate of the association with the
protein <em>β<sub>Xg</sub></em> and standard error
<em>σ<sub>Xg</sub></em>, and per-allele estimate of the
association with the outcome (here, AD or CHD)
<em>β<sub>Yg</sub></em> and standard error
<em>σ<sub>Yg</sub></em>, the IV estimate
<span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-2-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M2" overflow="scroll" role="math" style="width: 3.828em; display: inline-block;"><span style="display: inline-block; position: relative; width: 3.151em; height: 0px; font-size: 121%;"><span style="position: absolute; clip: rect(0.966em, 1003.06em, 2.626em, -1000em); top: -2.221em; left: 0em;"><span class="mrow" id="MathJax-Span-10"><span class="mrow" id="MathJax-Span-11"><span class="mo" id="MathJax-Span-12" style="font-family: MathJax_Main;">(</span><span class="msub" id="MathJax-Span-13"><span style="display: inline-block; position: relative; width: 2.3em; height: 0px;"><span style="position: absolute; clip: rect(2.722em, 1001.03em, 4.142em, -1000em); top: -3.977em; left: 0em;"><span class="mover" id="MathJax-Span-14"><span style="display: inline-block; position: relative; width: 1.083em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.46em, 4.142em, -1000em); top: -3.977em; left: 0.266em;"><span class="mi" id="MathJax-Span-15" style="font-family: MathJax_Math; font-style: italic;">θ</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.434em, 1000.94em, 4.01em, -1000em); top: -4.689em; left: 0.083em;"><span class="mo" id="MathJax-Span-16" style="font-family: MathJax_Main;">⌢</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 1.083em;"><span class="mrow" id="MathJax-Span-17"><span class="mi" id="MathJax-Span-18" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span class="mi" id="MathJax-Span-19" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic; text-rendering: optimizelegibility;">Y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.129em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-20" style="font-family: MathJax_Main;">)</span></span></span><span style="display: inline-block; width: 0px; height: 2.221em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.365em; border-left: 0px solid; width: 0px; height: 1.759em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M2" overflow="scroll"><mrow><mo stretchy="false">(</mo><msub><mover accent="true"><mi>θ</mi><mo>⌢</mo></mover><mrow><mi>X</mi><mi>Y</mi></mrow></msub><mo stretchy="false">)</mo></mrow></math></script> is obtained from generalized weighted linear
regression of the genetic associations with the outcome
(<em>β<sub>Y</sub></em>) on the genetic associations with
the protein (<em>β<sub>X</sub></em>) weighting for the precisions
of the genetic associations with the outcome and accounting for correlations
between the variants according to the regression model: </p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-3-Frame" tabindex="0" style=""><nobr><span class="math" id="M3" overflow="scroll" role="math" style="width: 15.203em; display: inline-block;"><span style="display: inline-block; position: relative; width: 12.552em; height: 0px; font-size: 121%;"><span style="position: absolute; clip: rect(1.368em, 1012.46em, 2.678em, -1000em); top: -2.273em; left: 0em;"><span class="mrow" id="MathJax-Span-22"><span class="mrow" id="MathJax-Span-23"><span class="msub" id="MathJax-Span-24"><span style="display: inline-block; position: relative; width: 1.181em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-25" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.566em;"><span class="mi" id="MathJax-Span-26" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic; text-rendering: optimizelegibility;">Y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.129em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-27" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="msub" id="MathJax-Span-28" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 1.686em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.46em, 4.142em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-29" style="font-family: MathJax_Math; font-style: italic;">θ</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.469em;"><span class="mrow" id="MathJax-Span-30"><span class="mi" id="MathJax-Span-31" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span class="mi" id="MathJax-Span-32" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic; text-rendering: optimizelegibility;">Y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.129em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mspace" id="MathJax-Span-33" style="height: 0em; vertical-align: 0em; width: 0.3em; display: inline-block; overflow: hidden;"></span><span class="msub" id="MathJax-Span-34"><span style="display: inline-block; position: relative; width: 1.243em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-35" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.566em;"><span class="mi" id="MathJax-Span-36" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-37" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="mi" id="MathJax-Span-38" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">ε</span><span class="mo" id="MathJax-Span-39" style="font-family: MathJax_Main;">,</span><span class="mspace" id="MathJax-Span-40" style="height: 0em; vertical-align: 0em; width: 0.3em; display: inline-block; overflow: hidden;"></span><span class="mi" id="MathJax-Span-41" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">ε</span><span class="mo" id="MathJax-Span-42" style="font-family: MathJax_Main;">~</span><span class="mi" id="MathJax-Span-43" style="font-family: MathJax_Math; font-style: italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span><span class="mo" id="MathJax-Span-44" style="font-family: MathJax_Main;">(</span><span class="mn" id="MathJax-Span-45" style="font-family: MathJax_Main;">0</span><span class="mo" id="MathJax-Span-46" style="font-family: MathJax_Main;">,</span><span class="mo" id="MathJax-Span-47" style="font-family: MathJax_Main; padding-left: 0.167em;">Ω</span><span class="mo" id="MathJax-Span-48" style="font-family: MathJax_Main;">)</span></span></span><span style="display: inline-block; width: 0px; height: 2.273em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.365em; border-left: 0px solid; width: 0px; height: 1.335em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-3"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M3" overflow="scroll"><mrow><msub><mi>β</mi><mi>Y</mi></msub><mo>=</mo><msub><mi>θ</mi><mrow><mi>X</mi><mi>Y</mi></mrow></msub><mspace width="0.3em"></mspace><msub><mi>β</mi><mi>X</mi></msub><mo>+</mo><mi>ε</mi><mo>,</mo><mspace width="0.3em"></mspace><mi>ε</mi><mo>~</mo><mi>N</mi><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><mo>Ω</mo><mo stretchy="false">)</mo></mrow></math></script></div></div><p> where <em>β<sub>y</sub></em> and
<em>β<sub>x</sub></em> are vectors of the univariable
(marginal) genetic associations, and the weighting matrix Ω has terms
Ω<sub><em>g<sub>1</sub>g<sub>2</sub></em></sub> =
<em>σ</em><sub><em>Yg<sub>1</sub></em></sub>
<em>σ</em><sub><em>Yg<sub>2</sub></em></sub>
<em>ρ</em><sub><em>g<sub>1</sub>g<sub>2</sub></em></sub>,
and
<em>ρ</em><sub><em>g<sub>1</sub>g<sub>2</sub></em></sub>
is the correlation between the <em>g<sub>1</sub></em>th and
<em>g<sub>2</sub></em>th variants.</p><p id="P64">The IV estimate from this method is: </p><div class="disp-formula" id="FD2"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-4-Frame" tabindex="0" style=""><nobr><span class="math" id="M4" overflow="scroll" role="math" style="width: 16.016em; display: inline-block;"><span style="display: inline-block; position: relative; width: 13.223em; height: 0px; font-size: 121%;"><span style="position: absolute; clip: rect(0.984em, 1013.22em, 2.695em, -1000em); top: -2.221em; left: 0em;"><span class="mrow" id="MathJax-Span-50"><span class="mrow" id="MathJax-Span-51"><span class="mrow" id="MathJax-Span-52"><span class="msub" id="MathJax-Span-53"><span style="display: inline-block; position: relative; width: 1.8em; height: 0px;"><span style="position: absolute; clip: rect(2.825em, 1000.48em, 4.142em, -1000em); top: -3.977em; left: 0em;"><span class="mover" id="MathJax-Span-54"><span style="display: inline-block; position: relative; width: 0.583em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.46em, 4.142em, -1000em); top: -3.977em; left: 0.016em;"><span class="mi" id="MathJax-Span-55" style="font-family: MathJax_Math; font-style: italic;">θ</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.128em, 1000.39em, 3.601em, -1000em); top: -4.28em; left: 0.083em;"><span class="mo" id="MathJax-Span-56" style=""><span style="font-family: MathJax_Main;">ˆ</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.583em;"><span class="mrow" id="MathJax-Span-57"><span class="mi" id="MathJax-Span-58" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span class="mi" id="MathJax-Span-59" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic; text-rendering: optimizelegibility;">Y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.129em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-60" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="msup" id="MathJax-Span-61" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 5.944em; height: 0px;"><span style="position: absolute; clip: rect(2.938em, 1004.87em, 4.451em, -1000em); top: -3.977em; left: 0em;"><span class="mrow" id="MathJax-Span-62"><span class="mo" id="MathJax-Span-63" style="font-family: MathJax_Main;">(</span><span class="msubsup" id="MathJax-Span-64"><span style="display: inline-block; position: relative; width: 1.243em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-65" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.344em, 1000.57em, 4.132em, -1000em); top: -4.322em; left: 0.625em;"><span class="mi" id="MathJax-Span-66" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.339em, 1000.68em, 4.132em, -1000em); top: -3.659em; left: 0.566em;"><span class="mi" id="MathJax-Span-67" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="msup" id="MathJax-Span-68"><span style="display: inline-block; position: relative; width: 1.701em; height: 0px;"><span style="position: absolute; clip: rect(3.118em, 1000.68em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mo" id="MathJax-Span-69" style="font-family: MathJax_Main;">Ω</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -4.39em; left: 0.722em;"><span class="mrow" id="MathJax-Span-70"><span class="mo" id="MathJax-Span-71" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-72" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="msub" id="MathJax-Span-73"><span style="display: inline-block; position: relative; width: 1.243em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-74" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.566em;"><span class="mi" id="MathJax-Span-75" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-76" style="font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -4.588em; left: 4.966em;"><span class="mrow" id="MathJax-Span-77"><span class="mo" id="MathJax-Span-78" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-79" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-80"><span style="display: inline-block; position: relative; width: 1.243em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-81" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.344em, 1000.57em, 4.132em, -1000em); top: -4.322em; left: 0.625em;"><span class="mi" id="MathJax-Span-82" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.339em, 1000.68em, 4.132em, -1000em); top: -3.659em; left: 0.566em;"><span class="mi" id="MathJax-Span-83" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="msup" id="MathJax-Span-84"><span style="display: inline-block; position: relative; width: 1.701em; height: 0px;"><span style="position: absolute; clip: rect(3.118em, 1000.68em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mo" id="MathJax-Span-85" style="font-family: MathJax_Main;">Ω</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -4.39em; left: 0.722em;"><span class="mrow" id="MathJax-Span-86"><span class="mo" id="MathJax-Span-87" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-88" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="msub" id="MathJax-Span-89"><span style="display: inline-block; position: relative; width: 1.181em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-90" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.566em;"><span class="mi" id="MathJax-Span-91" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic; text-rendering: optimizelegibility;">Y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.129em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.221em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.448em; border-left: 0px solid; width: 0px; height: 1.819em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-4"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M4" overflow="scroll"><mrow><mrow><msub><mover accent="true"><mi>θ</mi><mo>^</mo></mover><mrow><mi>X</mi><mi>Y</mi></mrow></msub><mo>=</mo><msup><mrow><mo stretchy="false">(</mo><msubsup><mi>β</mi><mi>X</mi><mi>T</mi></msubsup><msup><mo>Ω</mo><mrow><mo>−</mo><mn>1</mn></mrow></msup><msub><mi>β</mi><mi>X</mi></msub><mo stretchy="false">)</mo></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup><msubsup><mi>β</mi><mi>X</mi><mi>T</mi></msubsup><msup><mo>Ω</mo><mrow><mo>−</mo><mn>1</mn></mrow></msup><msub><mi>β</mi><mi>Y</mi></msub></mrow></mrow></math></script></div></div><p> and the standard error is: </p><div class="disp-formula" id="FD3"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-5-Frame" tabindex="0" style=""><nobr><span class="math" id="M5" overflow="scroll" role="math" style="width: 14.328em; display: inline-block;"><span style="display: inline-block; position: relative; width: 11.829em; height: 0px; font-size: 121%;"><span style="position: absolute; clip: rect(0.808em, 1011.83em, 2.918em, -1000em); top: -2.221em; left: 0em;"><span class="mrow" id="MathJax-Span-93"><span class="mrow" id="MathJax-Span-94"><span class="mtext" id="MathJax-Span-95" style="font-family: MathJax_Main;">se</span><span class="mo" id="MathJax-Span-96" style="font-family: MathJax_Main;">(</span><span class="msub" id="MathJax-Span-97"><span style="display: inline-block; position: relative; width: 1.8em; height: 0px;"><span style="position: absolute; clip: rect(2.825em, 1000.48em, 4.142em, -1000em); top: -3.977em; left: 0em;"><span class="mover" id="MathJax-Span-98"><span style="display: inline-block; position: relative; width: 0.583em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.46em, 4.142em, -1000em); top: -3.977em; left: 0.016em;"><span class="mi" id="MathJax-Span-99" style="font-family: MathJax_Math; font-style: italic;">θ</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.128em, 1000.39em, 3.601em, -1000em); top: -4.28em; left: 0.083em;"><span class="mo" id="MathJax-Span-100" style=""><span style="font-family: MathJax_Main;">ˆ</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.583em;"><span class="mrow" id="MathJax-Span-101"><span class="mi" id="MathJax-Span-102" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span class="mi" id="MathJax-Span-103" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic; text-rendering: optimizelegibility;">Y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.129em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-104" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-105" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="msqrt" id="MathJax-Span-106" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 6.969em; height: 0px;"><span style="position: absolute; clip: rect(2.741em, 1005.94em, 4.451em, -1000em); top: -3.977em; left: 1em;"><span class="mrow" id="MathJax-Span-107"><span class="mrow" id="MathJax-Span-108"><span class="msup" id="MathJax-Span-109"><span style="display: inline-block; position: relative; width: 5.944em; height: 0px;"><span style="position: absolute; clip: rect(2.938em, 1004.87em, 4.451em, -1000em); top: -3.977em; left: 0em;"><span class="mrow" id="MathJax-Span-110"><span class="mo" id="MathJax-Span-111" style="font-family: MathJax_Main;">(</span><span class="msubsup" id="MathJax-Span-112"><span style="display: inline-block; position: relative; width: 1.243em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-113" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.344em, 1000.57em, 4.132em, -1000em); top: -4.322em; left: 0.625em;"><span class="mi" id="MathJax-Span-114" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.339em, 1000.68em, 4.132em, -1000em); top: -3.659em; left: 0.566em;"><span class="mi" id="MathJax-Span-115" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="msup" id="MathJax-Span-116"><span style="display: inline-block; position: relative; width: 1.701em; height: 0px;"><span style="position: absolute; clip: rect(3.118em, 1000.68em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mo" id="MathJax-Span-117" style="font-family: MathJax_Main;">Ω</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -4.39em; left: 0.722em;"><span class="mrow" id="MathJax-Span-118"><span class="mo" id="MathJax-Span-119" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-120" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="msub" id="MathJax-Span-121"><span style="display: inline-block; position: relative; width: 1.243em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-122" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.566em;"><span class="mi" id="MathJax-Span-123" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-124" style="font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -4.588em; left: 4.966em;"><span class="mrow" id="MathJax-Span-125"><span class="mo" id="MathJax-Span-126" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-127" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.552em, 1005.97em, 3.902em, -1000em); top: -4.965em; left: 1em;"><span style="display: inline-block; position: relative; width: 5.969em; height: 0px;"><span style="position: absolute; font-family: MathJax_Main; top: -3.977em; left: -0.084em;">−<span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; font-family: MathJax_Main; top: -3.977em; left: 5.275em;">−<span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="font-family: MathJax_Main; position: absolute; top: -3.977em; left: 0.429em;">−<span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="font-family: MathJax_Main; position: absolute; top: -3.977em; left: 0.968em;">−<span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="font-family: MathJax_Main; position: absolute; top: -3.977em; left: 1.506em;">−<span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="font-family: MathJax_Main; position: absolute; top: -3.977em; left: 2.045em;">−<span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="font-family: MathJax_Main; position: absolute; top: -3.977em; left: 2.583em;">−<span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="font-family: MathJax_Main; position: absolute; top: -3.977em; left: 3.122em;">−<span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="font-family: MathJax_Main; position: absolute; top: -3.977em; left: 3.66em;">−<span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="font-family: MathJax_Main; position: absolute; top: -3.977em; left: 4.198em;">−<span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="font-family: MathJax_Main; position: absolute; top: -3.977em; left: 4.737em;">−<span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(2.672em, 1001.02em, 4.782em, -1000em); top: -4.085em; left: 0em;"><span style="font-family: MathJax_Size2;">√</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.221em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.719em; border-left: 0px solid; width: 0px; height: 2.303em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-5"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M5" overflow="scroll"><mrow><mtext>se</mtext><mo stretchy="false">(</mo><msub><mover accent="true"><mi>θ</mi><mo>^</mo></mover><mrow><mi>X</mi><mi>Y</mi></mrow></msub><mo stretchy="false">)</mo><mo>=</mo><msqrt><mrow><msup><mrow><mo stretchy="false">(</mo><msubsup><mi>β</mi><mi>X</mi><mi>T</mi></msubsup><msup><mo>Ω</mo><mrow><mo>−</mo><mn>1</mn></mrow></msup><msub><mi>β</mi><mi>X</mi></msub><mo stretchy="false">)</mo></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></msqrt></mrow></math></script></div></div><p> where <sup><em>T</em></sup> is a matrix
transpose. This is the estimate and standard error from the regression model
fixing the residual standard error to 1 (equivalent to a fixed-effects model in
a meta-analysis).</p><p id="P65">Genetic variants in univariable MR need to satisfy three key assumptions
to be valid instruments: (1) the variant is associated with the risk factor of
interest (that is, the protein level), (2) the variant is not associated with
any confounder of the risk factor-outcome association, and (3) the variant is
conditionally independent of the outcome given the risk factor and
confounders.</p><p id="P66">To account for potential effects of functional pleiotropy<sup><a href="#R67" rid="R67" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750445" role="button" aria-expanded="false" aria-haspopup="true">67</a></sup>, we performed multivariable MR
using the weighted regression-based method proposed by Burgess et al.<sup><a href="#R68" rid="R68" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750464" role="button" aria-expanded="false" aria-haspopup="true">68</a></sup>. For each of
<em>K</em> risk factors in the model (<em>k</em> =
1,…,<em>K</em>), the weighted regression-based method is
performed by multivariable generalized weighted linear regression of the
association estimates <em>β<sub>Y</sub></em> on each of the
association estimates with each risk factor
<em>β<sub>Xk</sub></em> in a single regression model:
</p><div class="disp-formula" id="FD4"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-6-Frame" tabindex="0" style=""><nobr><span class="math" id="M6" overflow="scroll" role="math" style="width: 30.516em; display: inline-block;"><span style="display: inline-block; position: relative; width: 25.207em; height: 0px; font-size: 121%;"><span style="position: absolute; clip: rect(1.368em, 1025.11em, 2.678em, -1000em); top: -2.273em; left: 0em;"><span class="mrow" id="MathJax-Span-129"><span class="mrow" id="MathJax-Span-130"><span class="msub" id="MathJax-Span-131"><span style="display: inline-block; position: relative; width: 1.181em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-132" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.566em;"><span class="mi" id="MathJax-Span-133" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic; text-rendering: optimizelegibility;">Y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.129em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-134" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="msub" id="MathJax-Span-135" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 2.04em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.46em, 4.142em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-136" style="font-family: MathJax_Math; font-style: italic;">θ</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.469em;"><span class="mrow" id="MathJax-Span-137"><span class="mi" id="MathJax-Span-138" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span class="mi" id="MathJax-Span-139" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic; text-rendering: optimizelegibility;">Y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.129em;"></span></span><span class="mn" id="MathJax-Span-140" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="msub" id="MathJax-Span-141"><span style="display: inline-block; position: relative; width: 1.597em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-142" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.566em;"><span class="mrow" id="MathJax-Span-143"><span class="mi" id="MathJax-Span-144" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span class="mn" id="MathJax-Span-145" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-146" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-147" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 2.04em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.46em, 4.142em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-148" style="font-family: MathJax_Math; font-style: italic;">θ</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.469em;"><span class="mrow" id="MathJax-Span-149"><span class="mi" id="MathJax-Span-150" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span class="mi" id="MathJax-Span-151" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic; text-rendering: optimizelegibility;">Y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.129em;"></span></span><span class="mn" id="MathJax-Span-152" style="font-size: 70.7%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="msub" id="MathJax-Span-153"><span style="display: inline-block; position: relative; width: 1.597em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-154" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.566em;"><span class="mrow" id="MathJax-Span-155"><span class="mi" id="MathJax-Span-156" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span class="mn" id="MathJax-Span-157" style="font-size: 70.7%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-158" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="mn" id="MathJax-Span-159" style="font-family: MathJax_Main; padding-left: 0.222em;">...</span><span class="mo" id="MathJax-Span-160" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-161" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 2.315em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.46em, 4.142em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-162" style="font-family: MathJax_Math; font-style: italic;">θ</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.469em;"><span class="mrow" id="MathJax-Span-163"><span class="mi" id="MathJax-Span-164" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span class="mi" id="MathJax-Span-165" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic; text-rendering: optimizelegibility;">Y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.129em;"></span></span><span class="mi" id="MathJax-Span-166" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">K<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.028em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="msub" id="MathJax-Span-167"><span style="display: inline-block; position: relative; width: 1.872em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-168" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.566em;"><span class="mrow" id="MathJax-Span-169"><span class="mi" id="MathJax-Span-170" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.017em;"></span></span><span class="mi" id="MathJax-Span-171" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">K<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.028em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-172" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="mi" id="MathJax-Span-173" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">ε</span><span class="mo" id="MathJax-Span-174" style="font-family: MathJax_Main;">,</span><span class="mspace" id="MathJax-Span-175" style="height: 0em; vertical-align: 0em; width: 0.3em; display: inline-block; overflow: hidden;"></span><span class="mi" id="MathJax-Span-176" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">ε</span><span class="mo" id="MathJax-Span-177" style="font-family: MathJax_Main;">~</span><span class="mi" id="MathJax-Span-178" style="font-family: MathJax_Math; font-style: italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span><span class="mo" id="MathJax-Span-179" style="font-family: MathJax_Main;">(</span><span class="mn" id="MathJax-Span-180" style="font-family: MathJax_Main;">0</span><span class="mo" id="MathJax-Span-181" style="font-family: MathJax_Main;">,</span><span class="mo" id="MathJax-Span-182" style="font-family: MathJax_Main; padding-left: 0.167em;">Ω</span><span class="mo" id="MathJax-Span-183" style="font-family: MathJax_Main;">)</span></span></span><span style="display: inline-block; width: 0px; height: 2.273em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.365em; border-left: 0px solid; width: 0px; height: 1.335em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-6"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M6" overflow="scroll"><mrow><msub><mi>β</mi><mi>Y</mi></msub><mo>=</mo><msub><mi>θ</mi><mrow><mi>X</mi><mi>Y</mi><mn>1</mn></mrow></msub><msub><mi>β</mi><mrow><mi>X</mi><mn>1</mn></mrow></msub><mo>+</mo><msub><mi>θ</mi><mrow><mi>X</mi><mi>Y</mi><mn>2</mn></mrow></msub><msub><mi>β</mi><mrow><mi>X</mi><mn>2</mn></mrow></msub><mo>+</mo><mn>...</mn><mo>+</mo><msub><mi>θ</mi><mrow><mi>X</mi><mi>Y</mi><mi>K</mi></mrow></msub><msub><mi>β</mi><mrow><mi>X</mi><mi>K</mi></mrow></msub><mo>+</mo><mi>ε</mi><mo>,</mo><mspace width="0.3em"></mspace><mi>ε</mi><mo>~</mo><mi>N</mi><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><mo>Ω</mo><mo stretchy="false">)</mo></mrow></math></script></div></div><p> where
<em>β<sub>X</sub></em><sub>1</sub> is the vectors of the
univariable genetic associations with risk factor 1, and so on. This regression
model is implemented by first pre-multiplying the association vectors by the
Cholesky decomposition of the weighting matrix, and then applying standard
linear regression to the transformed vectors. Estimates and standard errors are
obtained fixing the residual standard error to be 1 as above.</p><p id="P67" class="p p-last">The multivariable MR analysis allows the estimation of the causal effect
of a protein on disease outcome accounting for the fact that genetic variants
may be associated with multiple proteins in the region. Causal estimates from
multivariable MR represent direct causal effects, representing the effect of
intervening on one risk factor in the model while keeping others constant.</p></div><div id="S31" class="sec"><h3 id="S31title">MMP-12 genetic score sensitivity analyses</h3><p id="P68" class="p p-first-last">We performed two sensitivity analyses to determine the robustness of the
MR findings. First, we measured plasma MMP-12 levels using a different method
(proximity extension assay; Olink Bioscience, Uppsala, Sweden<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750428" role="button" aria-expanded="false" aria-haspopup="true">13</a></sup>) in 4,998 individuals, and
used this to derive genotype-MMP12 effect estimates for the 17 variants in our
genetic score. Second, we obtained effect estimates from a pQTL study based on
SOMAscan assay measurements in an independent sample of ~1,000
individuals<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_793750424" role="button" aria-expanded="false" aria-haspopup="true">3</a></sup>. In both
cases the genetic score reflecting higher plasma MMP-12 was associated with
lower risk of CHD.</p></div><div id="S32" class="sec sec-last"><h3 id="S32title">Overlap of pQTLs with drug targets</h3><p id="P69" class="p p-first">We used the Informa Pharmaprojects data-base from Citeline to obtain
information on drugs that target proteins assayed on the SOMAscan platform. This
is a manually curated database that maintains profiles for &gt;60,000 drugs.
For our analysis, we focused on the following information for each drug: protein
target, indications, and development status. We included drugs across the
development pipeline, including those in pre-clinical studies or with no
development reported, drugs in clinical trials (all phases), and
launched/registered drugs. For each protein assayed, we identified all drugs in
the Informa Pharmaprojects with a matching protein target based on UniProt ID.
When multiple drugs targeted the same protein, we selected the drug with the
latest stage of development.</p><p id="P70" class="p p-last">For drug targets with significant pQTLs, we identified the subset where
the sentinel variant or proxy variants in LD (<em>r</em><sup>2</sup>
&gt; 0.8) are also associated with disease risk through PhenoScanner. We
used an internal Merck auto-encoding method to map GWAS traits and drug
indications to a common set of terms from the Medical Dictionary for Regulatory
Activities (MedDRA). MedDRA terms are organized into a hierarchy with five
levels. We mapped each GWAS trait and indication onto the ‘lowest level
terms’ (that is, the most specific terms available). All matching terms
were recorded for each trait or indication. We matched GWAS traits to drug
indications on the basis of the highest level of the hierarchy, called
‘system organ class’ (SOC). We designated a protein as
‘matching’ if at least one GWAS trait term matched with at least
one indication term for at least one drug.</p></div></div><div id="S33" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S33title">Extended Data</h2><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F6"><h3>Extended Data Fig. 1</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142931054032" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f006.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f006.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f006.jpg"></a></div><div id="largeobj_idm140142931054032" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6697541/figure/F6/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Flowchart of sample processing and quality control stages for proteomic
and genetic measurements before genetic analyses.</strong></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F7"><h3>Extended Data Fig. 2</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142931050000" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f007.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f007.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f007.jpg"></a></div><div id="largeobj_idm140142931050000" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6697541/figure/F7/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Examples of protein targets for which the SOMAmer is highly
specific.</strong><p id="P71">SDS–PAGE with Alexa-647-labelled proteins captured by the
IL1RL2 SOMAmer (<strong>a</strong>) or GP1BA SOMAmer (<strong>b</strong>). For each
protein target, the protein captured by the SOMAmer is compared to the
standard. The cognate targets are the only ones with protein visible in the
capture lanes, whereas the proteins homologous to the target proteins show
no evidence of binding. These experiments were performed once. MW markers,
molecular weight markers.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F8"><h3>Extended Data Fig. 3</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142931044496" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f008.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f008.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f008.jpg"></a></div><div id="largeobj_idm140142931044496" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6697541/figure/F8/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Evidence for the reliability of protein measurements made using the
SOMAscan assay.</strong><p id="P72"><strong>a</strong>, Distribution of coefficients of variation of all
proteins on the SOMAscan assay in each subcohort. <strong>b</strong>,
Spearman’s correlations for all proteins passing QC derived from
contemporaneous assay of baseline and two-year samples from 60 participants.
<strong>c</strong>, Scatterplot of pQTL effect size estimates from SOMAscan
versus Olink showing all 163 pQTLs tested (top) and the 106 that replicated
(bottom). <em>r</em> is Pearson’s correlation coefficient.
<strong>d</strong>, Distribution of inflation factors across proteins that
underwent genome-wide association testing, stratified by subcohort and
allele frequency (MAF ≥ 5%, MAF &lt; 5%).</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F9"><h3>Extended Data Fig. 4</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142947650384" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f009.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f009.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f009.jpg"></a></div><div id="largeobj_idm140142947650384" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6697541/figure/F9/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>The <em>WFIKKN2</em> region is a <em>trans</em> pQTL for
GDF11/8 plasma levels.</strong><p id="P73"><strong>a</strong>, Regional association plots of the
<em>trans</em> pQTL (sentinel variant rs11079936) for GDF11/8
before and after adjusting for levels of WFIKKN2 (upper panels), and the
WFIKKN2 <em>cis</em> pQTL after adjusting for GDF11/8 levels (bottom
panel). A similar pattern of association for WFIKKN2 was seen before GDF11/8
adjustment (not shown). <strong>b</strong>, Attenuation of the GDF11/8
<em>trans</em> pQTL upon adjustment for plasma levels of the
<em>cis</em> protein WFIKKN2.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F10"><h3>Extended Data Fig. 5</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142929487312" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f010.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f010.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f010.jpg"></a></div><div id="largeobj_idm140142929487312" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6697541/figure/F10/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Genetic architecture of the pQTLs.</strong><p id="P74">pQTL mapping in <em>n</em> = 3,301 individuals.
<strong>a</strong>, Distribution of the predicted consequences of the
sentinel pQTL variants compared to matched permuted null sets of variants,
stratified by <em>cis</em> and <em>trans</em>. Asterisks
indicate empirical enrichment using a permutation test (10,000 permuted sets
of non-associated variants) at a Bonferroni-corrected significance value
(<em>P</em> &lt; 0.005). Bar height represents the mean
proportion of variants within each class and error bars reflect one standard
deviation from the mean. <strong>b</strong>, Number of proteins associated
(<em>P</em> &lt; 1.5 × 10<sup>−11</sup>) with
each sentinel variant across the genome.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F11"><h3>Extended Data Fig. 6</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142929482064" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f011.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f011.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f011.jpg"></a></div><div id="largeobj_idm140142929482064" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6697541/figure/F11/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Enrichment of pQTLs at DNase I hypersensitive sites by tissue or cell
type.</strong><p id="P75">Circle shows enrichment for DNase I hypersensitive sites
(‘hotspots’) for each of 55 tissues (183 cell types) available
from the ENCODE and Roadmap Epigenomics projects, with tissues or cell types
clustered and coloured by anatomical grouping. Some tissues have multiple
values due to availability of multiple cell types or multiple tests per cell
type. Radial lines show fold-enrichment, while dots around the inside edge
of the circle denote statistically significant enrichment at a
Bonferroni-corrected significant threshold <em>P</em> &lt; 5
× 10<sup>−5</sup>. Enrichment testing performed using GARFIELD
(which tests enrichment against permuted sets of variants matched for MAF,
distance to TSS and LD). pQTL data from <em>n</em> = 3,301
individuals.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F12"><h3>Extended Data Fig. 7</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142983193264" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f012.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f012.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f012.jpg"></a></div><div id="largeobj_idm140142983193264" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6697541/figure/F12/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Scheme outlining the combined ‘bottom-up’ and
‘top-down’ process used for candidate gene annotation of
<em>trans</em> pQTL regions.</strong><p id="P76">See <a href="#S8" rid="S8" class=" sec">Methods</a>. GbA,
guilt-by-association; KEGG, Kyoto Encyclopedia of Genes and Genomes; OMIM,
Online Mendelian Inheritance in Man; STRINGdb, STRING database.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F13"><h3>Extended Data Fig. 8</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142983188240" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f013.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f013.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f013.jpg"></a></div><div id="largeobj_idm140142983188240" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6697541/figure/F13/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Follow-up of PR3 SOMAmers.</strong><p id="P77">These experiments were repeated three times independently with
similar results. <strong>a</strong>, SOMAmer pulldowns with purified PR3, A1AT,
and PR3–A1AT complex. SOMAmer PRTN3.3514.49.2 enriched the
PR3–A1AT complex to a much greater degree than free PR3. Conversely,
SOMAmer PRTN3.13720.95.3 enriched free PR3 to a greater degree than the
PR3–A1AT complex. <strong>b</strong>, Solution affinity of
PRTN3.3514.49.2 and PRTN3.13720.95.3 for PR3, A1AT, and the PR3–A1AT
complex. SOMAmer PRTN3.3514.49.2 has a higher affinity for the
PR3–A1AT complex than for free PR3. SOMAmer PRTN3.13720.95.3, on the
other hand, has a higher affinity for free PR3 than SOMAmer PRTN3.3514.49.2.
<strong>c</strong>, Competitive binding of SOMAmers PRTN3.13720.95.3 and
PRTN3.3514.49.2 to PR3. A limiting amount of radiolabelled PRTN3.13720.95.3
was incubated with 1 nM proteinase-3 and a titration of either cold
PRTN3.13720.95.3 or cold PRTN3.3514.49.2.</p></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F14"><h3>Extended Data Fig. 9</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142983185728" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f014.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f014.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f014.jpg"></a></div><div id="largeobj_idm140142983185728" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6697541/figure/F14/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Comparison between a randomized controlled trial and Mendelian
randomization to assess the causal effect of changes in protein biomarker
levels on disease risk.</strong></div></div><!--fig ft0--><!--fig @position="anchor" mode=article f4--><!--fig mode="anchored" f5--><div class="fig  anchored whole_rhythm" id="F15"><h3>Extended Data Fig. 10</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140142933347648" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6697541_EMS83968-f015.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is EMS83968-f015.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6697541/bin/EMS83968-f015.jpg"></a></div><div id="largeobj_idm140142933347648" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6697541/figure/F15/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Characterization of protein targets measured using the SOMAscan
assay.</strong><p id="P78"><strong>a</strong>, Compartment distribution with annotations of all
proteins in the Human Protein Atlas for comparison. <strong>b</strong>, GO
molecular functions.</p></div></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><p id="P79" class="p p-first"><strong>Supplementary information</strong> is available for this paper at
<a href="https://doi.org/10.1038/s41586-018-0175-2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://doi.org/10.1038/s41586-018-0175-2</a>.</p><div class="sec suppmat" id="SD1"><h4>Supplementary tables</h4><div class="sup-box half_rhythm" id="d36e2840"><a href="/pmc/articles/PMC6697541/bin/EMS83968-supplement-Supplementary_tables.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.1M, xlsx)</sup></div></div><div class="sec suppmat" id="SD2"><h4>Supplementary figure 1</h4><div class="sup-box half_rhythm" id="d36e2844"><a href="/pmc/articles/PMC6697541/bin/EMS83968-supplement-Supplementary_figure_1.html" data-ga-action="click_feat_suppl">Click here to view.</a></div></div><div class="sec suppmat" id="SD3"><h4>Supplementary file</h4><div class="sup-box half_rhythm" id="d36e2848"><a href="/pmc/articles/PMC6697541/bin/EMS83968-supplement-Supplementary_file.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(417K, pdf)</sup></div></div></div><div id="S34" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S34title">Acknowledgements</h2><div class="sec"><p id="__p1">A. Day-Williams, J. McElwee, D. Diogo, W. Astle, E. Di Angelantonio, E. Birney, A.
Richard, J. Mason and M. Inouye commented on the manuscript, and M. Sharp helped
with mapping drug indications to GWAS traits. We thank INTERVAL study participants;
staff at recruiting NHSBT blood donation centres; and the INTERVAL Study
Co-ordination team, Operations Team (led by R. Houghton and C. Moore) and Data
Management Team (led by M. Walker). Funding sources are listed in the <a href="#S3" rid="S3" class=" sec">Supplementary Information</a>.</p></div></div><div id="idm140142947924144" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140142947924144title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN4"></p><p id="P80" class="p p-first-last"><strong>Reporting summary.</strong> Further information on experimental
design is available in the Nature Research Reporting Summary linked to this
paper.</p><p></p><p class="fn sec" id="FN5"></p><p id="P81" class="p p-first-last"><strong>Data availability.</strong> Participant-level genotype and protein
data, and full summary association results from the genetic analysis, are
available through the European Genotype Archive (accession number
EGAS00001002555). Summary association results are also publically available at
<a href="http://www.phpc.cam.ac.uk/ceu/proteins/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.phpc.cam.ac.uk/ceu/proteins/</a>, through PhenoScanner
(<a href="http://www.phenoscanner.medschl.cam.ac.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.phenoscanner.medschl.cam.ac.uk</a>) and from the
NHGRI-EBI GWAS Catalog (<a href="https://www.ebi.ac.uk/gwas/downloads/summary-statistics" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ebi.ac.uk/gwas/downloads/summary-statistics</a>).</p><p></p><p class="fn sec" id="FN6"></p><p id="P82" class="p p-first-last"><strong>Reviewer information</strong>
<em>Nature</em> thanks T. Lappalainen, M. McCarthy and the other
anonymous reviewer(s) for their contribution to the peer review of this
work.</p><p></p><div>Contributed by <p id="P83" class="p p-first-last"><strong>Author contributions</strong> Conceptualization and experimental
design: J.D., A.S.B., B.B.S., H.R., R.M.P.; methodology: B.B.S., A.S.B., J.C.M.,
J.E.P., H.R., S.B.; conducted experimental work: N.J., S.K.W., E.S.Z.; analysis:
B.B.S., J.C.M., J.E.P., D.S., J.B., J.R.S., T.J., E.P., P.S., C.O.-W., M.A.K.,
S.K.W., A.C., N.B., S.L.S.; contributed reagents, materials, protocols or
analysis tools: N.J., S.K.W., E.S.Z., J.B., M.A.K., J.R.S., B.P.P.; supervision:
A.S.B., H.R., J.D., R.M.P., C.S.F., D.S.P., A.M.W.; writing: A.S.B., J.E.P.,
B.B.S., J.C.M., H.R., J.D., J.A.T., N.S., K.S.; creation of the INTERVAL
BioResource: J.R.B., D.J.R.,W.H.O., N.W.M., J.D.; funding: N.W.M., J.R.B.,
D.J.R., W.H.O., H.R., R.M.P., J.D.; all authors critically reviewed the
manuscript.</p></div><p class="fn sec" id="FN8"></p><p id="P84" class="p p-first-last"><strong>Competing interests</strong> The authors declare the following
competing interests: A.C., CSF-Merck employee; N.J., S.K.W., SomaLogic Inc
employees and stakeholders; E.S.Z., SomaLogic Inc employee; J.C.M., R.M.P.,
Merck employees during this study, now Celgene employees; H.R., Merck employee
during this study; J.E.P., travel and accommodation expenses and hospitality
from Olink to speak at Olink-sponsored academic meetings; A.S.B., grants from
Merck, Pfizer, Novartis, Biogen and Bioverativ and personal fees from Novartis;
J.D., sits on the Novartis Cardiovascular and Metabolic Advisory Board, had
grant support from Novartis.</p><p></p><p class="fn sec" id="FN9"></p><p id="P85" class="p p-first"><strong>Extended data</strong> is available for this paper at <a href="https://doi.org/10.1038/s41586-018-0175-2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://doi.org/10.1038/s41586-018-0175-2</a>.</p><p id="P86" class="p"><strong>Reprints and permissions information</strong> is available at
<a href="http://www.nature.com/reprints" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/reprints</a>.</p><p id="P87" class="p p-last"><strong>Publisher’s note:</strong> Springer Nature remains neutral
with regard to jurisdictional claims in published maps and institutional
affiliations.</p><p></p></div></div><div id="idm140142935372128" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140142935372128title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Albert FW, Kruglyak L. The role of regulatory variation in complex traits and
disease. <span><span class="ref-journal">Nat Rev Genet. </span>2015;<span class="ref-vol">16</span>:197–212.</span> [<a href="/pubmed/25707927" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=The+role+of+regulatory+variation+in+complex+traits+and
disease&amp;author=FW+Albert&amp;author=L+Kruglyak&amp;volume=16&amp;publication_year=2015&amp;pages=197-212&amp;pmid=25707927&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Liu Y, et al.  Quantitative variability of 342 plasma proteins in a human twin
population. <span><span class="ref-journal">Mol Syst Biol. </span>2015;<span class="ref-vol">11</span>:786.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4358658/">PMC free article</a>]</span> [<a href="/pubmed/25652787" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Syst+Biol&amp;title=Quantitative+variability+of+342+plasma+proteins+in+a+human+twin
population&amp;author=Y+Liu&amp;volume=11&amp;publication_year=2015&amp;pages=786&amp;pmid=25652787&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Suhre K, et al.  Connecting genetic risk to disease end points through the human
blood plasma proteome. <span><span class="ref-journal">Nat Commun. </span>2017;<span class="ref-vol">8</span></span> 14357. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5394265/">PMC free article</a>]</span> [<a href="/pubmed/28397792" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Connecting+genetic+risk+to+disease+end+points+through+the+human
blood+plasma+proteome&amp;author=K+Suhre&amp;volume=8&amp;publication_year=2017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Yao C, et al.  Genome-wide association study of plasma proteins identifies
putatively causal genes, proteins, and pathways for cardiovascular
disease. <span>2017</span> Preprint at  <a href="https://www.biorxiv.org/content/early/2017/05/12/136523" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://www.biorxiv.org/content/early/2017/05/12/136523</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">de Vries PS, et al.  Whole-genome sequencing study of serum peptide levels: the
Atherosclerosis Risk in Communities study. <span><span class="ref-journal">Hum Mol Genet. </span>2017;<span class="ref-vol">26</span>:3442–3450.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5886054/">PMC free article</a>]</span> [<a href="/pubmed/28854705" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Whole-genome+sequencing+study+of+serum+peptide+levels:+the
Atherosclerosis+Risk+in+Communities+study&amp;author=PS+de+Vries&amp;volume=26&amp;publication_year=2017&amp;pages=3442-3450&amp;pmid=28854705&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Wu L, et al.  Variation and genetic control of protein abundance in
humans. <span><span class="ref-journal">Nature. </span>2013;<span class="ref-vol">499</span>:79–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3789121/">PMC free article</a>]</span> [<a href="/pubmed/23676674" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Variation+and+genetic+control+of+protein+abundance+in
humans&amp;author=L+Wu&amp;volume=499&amp;publication_year=2013&amp;pages=79-82&amp;pmid=23676674&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Battle A, et al.  Impact of regulatory variation from RNA to
protein. <span><span class="ref-journal">Science. </span>2015;<span class="ref-vol">347</span>:664–667.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4507520/">PMC free article</a>]</span> [<a href="/pubmed/25657249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Impact+of+regulatory+variation+from+RNA+to
protein&amp;author=A+Battle&amp;volume=347&amp;publication_year=2015&amp;pages=664-667&amp;pmid=25657249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Rohloff JC, et al.  Nucleic acid ligands with protein-like side chains: modified
aptamers and their use as diagnostic and therapeutic agents. <span><span class="ref-journal">Mol Ther Nucleic Acids. </span>2014;<span class="ref-vol">3</span>:e201.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4217074/">PMC free article</a>]</span> [<a href="/pubmed/25291143" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Ther+Nucleic+Acids&amp;title=Nucleic+acid+ligands+with+protein-like+side+chains:+modified
aptamers+and+their+use+as+diagnostic+and+therapeutic+agents&amp;author=JC+Rohloff&amp;volume=3&amp;publication_year=2014&amp;pages=e201&amp;pmid=25291143&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Di Angelantonio E, et al.  Efficiency and safety of varying the frequency of whole blood
donation (INTERVAL): a randomised trial of 45 000 donors. <span><span class="ref-journal">Lancet. </span>2017;<span class="ref-vol">390</span>:2360–2371.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5714430/">PMC free article</a>]</span> [<a href="/pubmed/28941948" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Efficiency+and+safety+of+varying+the+frequency+of+whole+blood
donation+(INTERVAL):+a+randomised+trial+of+45+000+donors&amp;author=E+Di+Angelantonio&amp;volume=390&amp;publication_year=2017&amp;pages=2360-2371&amp;pmid=28941948&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Astle WJ, et al.  The allelic landscape of human blood cell trait variation and
links to common complex disease. <span><span class="ref-journal">Cell. </span>2016;<span class="ref-vol">167</span>:1415–1429.e19.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5300907/">PMC free article</a>]</span> [<a href="/pubmed/27863252" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The+allelic+landscape+of+human+blood+cell+trait+variation+and
links+to+common+complex+disease&amp;author=WJ+Astle&amp;volume=167&amp;publication_year=2016&amp;pages=1415-1429.e19&amp;pmid=27863252&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Burgess S, Scott RA, Timpson NJ, Davey Smith G, Thompson SG. Using published data in Mendelian randomization: a blueprint for
efficient identification of causal risk factors. <span><span class="ref-journal">Eur J Epidemiol. </span>2015;<span class="ref-vol">30</span>:543–552.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4516908/">PMC free article</a>]</span> [<a href="/pubmed/25773750" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Epidemiol&amp;title=Using+published+data+in+Mendelian+randomization:+a+blueprint+for
efficient+identification+of+causal+risk+factors&amp;author=S+Burgess&amp;author=RA+Scott&amp;author=NJ+Timpson&amp;author=G+Davey+Smith&amp;author=SG+Thompson&amp;volume=30&amp;publication_year=2015&amp;pages=543-552&amp;pmid=25773750&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Stranger BE, et al.  Patterns of <em>cis</em> regulatory variation in diverse
human populations. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002639.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3330104/">PMC free article</a>]</span> [<a href="/pubmed/22532805" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Patterns+of+cis+regulatory+variation+in+diverse
human+populations&amp;author=BE+Stranger&amp;volume=8&amp;publication_year=2012&amp;pages=e1002639&amp;pmid=22532805&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Lundberg M, Eriksson A, Tran B, Assarsson E, Fredriksson S. Homogeneous antibody-based proximity extension assays provide
sensitive and specific detection of low-abundant proteins in human
blood. <span><span class="ref-journal">Nucleic Acids Res. </span>2011;<span class="ref-vol">39</span>:e102.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3159481/">PMC free article</a>]</span> [<a href="/pubmed/21646338" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Homogeneous+antibody-based+proximity+extension+assays+provide
sensitive+and+specific+detection+of+low-abundant+proteins+in+human
blood&amp;author=M+Lundberg&amp;author=A+Eriksson&amp;author=B+Tran&amp;author=E+Assarsson&amp;author=S+Fredriksson&amp;volume=39&amp;publication_year=2011&amp;pages=e102&amp;pmid=21646338&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Walter K, et al.  The UK10K project identifies rare variants in health and
disease. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">526</span>:82–90.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4773891/">PMC free article</a>]</span> [<a href="/pubmed/26367797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+UK10K+project+identifies+rare+variants+in+health+and
disease&amp;author=K+Walter&amp;volume=526&amp;publication_year=2015&amp;pages=82-90&amp;pmid=26367797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Westra H-J, et al.  Systematic identification of trans eQTLs as putative drivers of
known disease associations. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:1238–1243.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3991562/">PMC free article</a>]</span> [<a href="/pubmed/24013639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Systematic+identification+of+trans+eQTLs+as+putative+drivers+of
known+disease+associations&amp;author=H-J+Westra&amp;volume=45&amp;publication_year=2013&amp;pages=1238-1243&amp;pmid=24013639&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Lappalainen T, et al.  Transcriptome and genome sequencing uncovers functional variation
in humans. <span><span class="ref-journal">Nature. </span>2013;<span class="ref-vol">501</span>:506–511.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3918453/">PMC free article</a>]</span> [<a href="/pubmed/24037378" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Transcriptome+and+genome+sequencing+uncovers+functional+variation
in+humans&amp;author=T+Lappalainen&amp;volume=501&amp;publication_year=2013&amp;pages=506-511&amp;pmid=24037378&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Schadt EE, et al.  Mapping the genetic architecture of gene expression in human
liver. <span><span class="ref-journal">PLoS Biol. </span>2008;<span class="ref-vol">6</span>:e107.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2365981/">PMC free article</a>]</span> [<a href="/pubmed/18462017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Mapping+the+genetic+architecture+of+gene+expression+in+human
liver&amp;author=EE+Schadt&amp;volume=6&amp;publication_year=2008&amp;pages=e107&amp;pmid=18462017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Zeller T, et al.  Genetics and beyond—the transcriptome of human monocytes
and disease susceptibility. <span><span class="ref-journal">PLoS ONE. </span>2010;<span class="ref-vol">5</span>:e10693.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2872668/">PMC free article</a>]</span> [<a href="/pubmed/20502693" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Genetics+and+beyond—the+transcriptome+of+human+monocytes
and+disease+susceptibility&amp;author=T+Zeller&amp;volume=5&amp;publication_year=2010&amp;pages=e10693&amp;pmid=20502693&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Nai A, et al.  TMPRSS6 rs855791 modulates hepcidin transcription in vitro and
serum hepcidin levels in normal individuals. <span><span class="ref-journal">Blood. </span>2011;<span class="ref-vol">118</span>:4459–4462.</span> [<a href="/pubmed/21873547" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=TMPRSS6+rs855791+modulates+hepcidin+transcription+in+vitro+and
serum+hepcidin+levels+in+normal+individuals&amp;author=A+Nai&amp;volume=118&amp;publication_year=2011&amp;pages=4459-4462&amp;pmid=21873547&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Carrasquillo MM, et al.  Genome-wide screen identifies rs646776 near sortilin as a
regulator of progranulin levels in human plasma. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">87</span>:890–897.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2997361/">PMC free article</a>]</span> [<a href="/pubmed/21087763" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genome-wide+screen+identifies+rs646776+near+sortilin+as+a
regulator+of+progranulin+levels+in+human+plasma&amp;author=MM+Carrasquillo&amp;volume=87&amp;publication_year=2010&amp;pages=890-897&amp;pmid=21087763&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Gooptu B, Dickens JA, Lomas DA. The molecular and cellular pathology of α1-antitrypsin
deficiency. <span><span class="ref-journal">Trends Mol Med. </span>2014;<span class="ref-vol">20</span>:116–127.</span> [<a href="/pubmed/24374162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Mol+Med&amp;title=The+molecular+and+cellular+pathology+of+α1-antitrypsin
deficiency&amp;author=B+Gooptu&amp;author=JA+Dickens&amp;author=DA+Lomas&amp;volume=20&amp;publication_year=2014&amp;pages=116-127&amp;pmid=24374162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Stacey D, et al.  <span><span class="ref-journal">ProGeM: A framework for the prioritisation of candidate causal genes at
molecular quantitative trait loci. </span>2017 doi: 10.1101/230094.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6326795/">PMC free article</a>]</span> [<a href="/pubmed/30239796" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1101%2F230094" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ProGeM:+A+framework+for+the+prioritisation+of+candidate+causal+genes+at
molecular+quantitative+trait+loci&amp;author=D+Stacey&amp;publication_year=2017&amp;doi=10.1101/230094&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Liu JZ, et al.  Association analyses identify 38 susceptibility loci for
inflammatory bowel disease and highlight shared genetic risk across
populations. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:979–986.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4881818/">PMC free article</a>]</span> [<a href="/pubmed/26192919" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Association+analyses+identify+38+susceptibility+loci+for
inflammatory+bowel+disease+and+highlight+shared+genetic+risk+across
populations&amp;author=JZ+Liu&amp;volume=47&amp;publication_year=2015&amp;pages=979-986&amp;pmid=26192919&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Di Narzo AF, et al.  High-throughput characterization of blood serum proteomics of IBD
patients with respect to aging and genetic factors. <span><span class="ref-journal">PLoS Genet. </span>2017;<span class="ref-vol">13</span>:e1006565.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5271178/">PMC free article</a>]</span> [<a href="/pubmed/28129359" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=High-throughput+characterization+of+blood+serum+proteomics+of+IBD
patients+with+respect+to+aging+and+genetic+factors&amp;author=AF+Di+Narzo&amp;volume=13&amp;publication_year=2017&amp;pages=e1006565&amp;pmid=28129359&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Huang H, et al.  Fine-mapping inflammatory bowel disease loci to single-variant
resolution. <span><span class="ref-journal">Nature. </span>2017;<span class="ref-vol">547</span>:173–178.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5511510/">PMC free article</a>]</span> [<a href="/pubmed/28658209" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Fine-mapping+inflammatory+bowel+disease+loci+to+single-variant
resolution&amp;author=H+Huang&amp;volume=547&amp;publication_year=2017&amp;pages=173-178&amp;pmid=28658209&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Lyons PA, et al.  Genetically distinct subsets within ANCA-associated
vasculitis. <span><span class="ref-journal">N Engl J Med. </span>2012;<span class="ref-vol">367</span>:214–223.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3773907/">PMC free article</a>]</span> [<a href="/pubmed/22808956" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Genetically+distinct+subsets+within+ANCA-associated
vasculitis&amp;author=PA+Lyons&amp;volume=367&amp;publication_year=2012&amp;pages=214-223&amp;pmid=22808956&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Merkel PA, et al.  Identification of functional and expression polymorphisms
associated with risk for anti-neutrophil cytoplasmic autoantibody-associated
vasculitis. <span><span class="ref-journal">Arthritis Rheumatol. </span>2017;<span class="ref-vol">69</span>:1054–1066.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5434905/">PMC free article</a>]</span> [<a href="/pubmed/28029757" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheumatol&amp;title=Identification+of+functional+and+expression+polymorphisms
associated+with+risk+for+anti-neutrophil+cytoplasmic+autoantibody-associated
vasculitis&amp;author=PA+Merkel&amp;volume=69&amp;publication_year=2017&amp;pages=1054-1066&amp;pmid=28029757&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Battle A, Brown CD, Engelhardt BE, Montgomery SB. Genetic effects on gene expression across human
tissues. <span><span class="ref-journal">Nature. </span>2017;<span class="ref-vol">550</span>:204–213.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5776756/">PMC free article</a>]</span> [<a href="/pubmed/29022597" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+effects+on+gene+expression+across+human
tissues&amp;author=A+Battle&amp;author=CD+Brown&amp;author=BE+Engelhardt&amp;author=SB+Montgomery&amp;volume=550&amp;publication_year=2017&amp;pages=204-213&amp;pmid=29022597&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Grönberg H, et al.  Prostate cancer screening in men aged 50-69 years (STHLM3): a
prospective population-based diagnostic study. <span><span class="ref-journal">Lancet Oncol. </span>2015;<span class="ref-vol">16</span>:1667–1676.</span> [<a href="/pubmed/26563502" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Oncol&amp;title=Prostate+cancer+screening+in+men+aged+50-69+years+(STHLM3):+a
prospective+population-based+diagnostic+study&amp;author=H+Grönberg&amp;volume=16&amp;publication_year=2015&amp;pages=1667-1676&amp;pmid=26563502&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Eeles RA, et al.  Multiple newly identified loci associated with prostate cancer
susceptibility. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:316–321.</span> [<a href="/pubmed/18264097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Multiple+newly+identified+loci+associated+with+prostate+cancer
susceptibility&amp;author=RA+Eeles&amp;volume=40&amp;publication_year=2008&amp;pages=316-321&amp;pmid=18264097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Paternoster L, et al.  Multi-ancestry genome-wide association study of 21,000 cases and
95,000 controls identifies new risk loci for atopic
dermatitis. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:1449–1456.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4753676/">PMC free article</a>]</span> [<a href="/pubmed/26482879" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Multi-ancestry+genome-wide+association+study+of+21,000+cases+and
95,000+controls+identifies+new+risk+loci+for+atopic
dermatitis&amp;author=L+Paternoster&amp;volume=47&amp;publication_year=2015&amp;pages=1449-1456&amp;pmid=26482879&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Dahl R, et al.  Effects of an oral MMP-9 and -12 inhibitor, AZD1236, on
biomarkers in moderate/severe COPD: a randomised controlled
trial. <span><span class="ref-journal">Pulm Pharmacol Ther. </span>2012;<span class="ref-vol">25</span>:169–177.</span> [<a href="/pubmed/22306193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pulm+Pharmacol+Ther&amp;title=Effects+of+an+oral+MMP-9+and+-12+inhibitor,+AZD1236,+on
biomarkers+in+moderate/severe+COPD:+a+randomised+controlled
trial&amp;author=R+Dahl&amp;volume=25&amp;publication_year=2012&amp;pages=169-177&amp;pmid=22306193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Ganz P, et al.  Development and validation of a protein-based risk score for
cardiovascular outcomes among patients with stable coronary heart
disease. <span><span class="ref-journal">J Am Med Assoc. </span>2016;<span class="ref-vol">315</span>:2532–2541.</span> [<a href="/pubmed/27327800" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Med+Assoc&amp;title=Development+and+validation+of+a+protein-based+risk+score+for
cardiovascular+outcomes+among+patients+with+stable+coronary+heart
disease&amp;author=P+Ganz&amp;volume=315&amp;publication_year=2016&amp;pages=2532-2541&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Traylor M, et al.  A novel MMP12 locus is associated with large artery
atherosclerotic stroke using a genome-wide age-at-onset informed
approach. <span><span class="ref-journal">PLoS Genet. </span>2014;<span class="ref-vol">10</span>:e1004469.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4117446/">PMC free article</a>]</span> [<a href="/pubmed/25078452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+novel+MMP12+locus+is+associated+with+large+artery
atherosclerotic+stroke+using+a+genome-wide+age-at-onset+informed
approach&amp;author=M+Traylor&amp;volume=10&amp;publication_year=2014&amp;pages=e1004469&amp;pmid=25078452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Nelson MR, et al.  The support of human genetic evidence for approved drug
indications. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:856–860.</span> [<a href="/pubmed/26121088" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+support+of+human+genetic+evidence+for+approved+drug
indications&amp;author=MR+Nelson&amp;volume=47&amp;publication_year=2015&amp;pages=856-860&amp;pmid=26121088&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Albagha OME, et al.  Genome-wide association study identifies variants at CSF1, OPTN
and TNFRSF11A as genetic risk factors for Paget’s disease of
bone. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:520–524.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3217192/">PMC free article</a>]</span> [<a href="/pubmed/20436471" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+variants+at+CSF1,+OPTN
and+TNFRSF11A+as+genetic+risk+factors+for+Paget’s+disease+of
bone&amp;author=OME+Albagha&amp;volume=42&amp;publication_year=2010&amp;pages=520-524&amp;pmid=20436471&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Schwarz P, Rasmussen AQ, Kvist TM, Andersen UB, Jørgensen NR. Paget’s disease of the bone after treatment with
Denosumab: a case report. <span><span class="ref-journal">Bone. </span>2012;<span class="ref-vol">50</span>:1023–1025.</span> [<a href="/pubmed/22586699" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone&amp;title=Paget’s+disease+of+the+bone+after+treatment+with
Denosumab:+a+case+report&amp;author=P+Schwarz&amp;author=AQ+Rasmussen&amp;author=TM+Kvist&amp;author=UB+Andersen&amp;author=NR+Jørgensen&amp;volume=50&amp;publication_year=2012&amp;pages=1023-1025&amp;pmid=22586699&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Moore C, et al.  The INTERVAL trial to determine whether intervals between blood
donations can be safely and acceptably decreased to optimise blood supply:
study protocol for a randomised controlled trial. <span><span class="ref-journal">Trials. </span>2014;<span class="ref-vol">15</span>:363.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4177700/">PMC free article</a>]</span> [<a href="/pubmed/25230735" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trials&amp;title=The+INTERVAL+trial+to+determine+whether+intervals+between+blood
donations+can+be+safely+and+acceptably+decreased+to+optimise+blood+supply:
study+protocol+for+a+randomised+controlled+trial&amp;author=C+Moore&amp;volume=15&amp;publication_year=2014&amp;pages=363&amp;pmid=25230735&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Gold L, et al.  Aptamer-based multiplexed proteomic technology for biomarker
discovery. <span><span class="ref-journal">PLoS ONE. </span>2010;<span class="ref-vol">5</span>:e15004.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3000457/">PMC free article</a>]</span> [<a href="/pubmed/21165148" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Aptamer-based+multiplexed+proteomic+technology+for+biomarker
discovery&amp;author=L+Gold&amp;volume=5&amp;publication_year=2010&amp;pages=e15004&amp;pmid=21165148&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Sattlecker M, et al.  Alzheimer’s disease biomarker discovery using SOMAscan
multiplexed protein technology. <span><span class="ref-journal">Alzheimers Dement. </span>2014;<span class="ref-vol">10</span>:724–734.</span> [<a href="/pubmed/24768341" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Alzheimers+Dement&amp;title=Alzheimer’s+disease+biomarker+discovery+using+SOMAscan
multiplexed+protein+technology&amp;author=M+Sattlecker&amp;volume=10&amp;publication_year=2014&amp;pages=724-734&amp;pmid=24768341&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">McLaren W, et al.  Deriving the consequences of genomic variants with the Ensembl
API and SNP Effect Predictor. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2069–2070.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2916720/">PMC free article</a>]</span> [<a href="/pubmed/20562413" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Deriving+the+consequences+of+genomic+variants+with+the+Ensembl
API+and+SNP+Effect+Predictor&amp;author=W+McLaren&amp;volume=26&amp;publication_year=2010&amp;pages=2069-2070&amp;pmid=20562413&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">UniProt Consortium. UniProt: a hub for protein information. <span><span class="ref-journal">Nucleic Acids Res. </span>2015;<span class="ref-vol">43</span>:D204–D212.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4384041/">PMC free article</a>]</span> [<a href="/pubmed/25348405" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=UniProt:+a+hub+for+protein+information&amp;volume=43&amp;publication_year=2015&amp;pages=D204-D212&amp;pmid=25348405&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Ashburner M, et al. The Gene Ontology Consortium  Gene ontology: tool for the unification of
biology. <span><span class="ref-journal">Nat Genet. </span>2000;<span class="ref-vol">25</span>:25–29.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3037419/">PMC free article</a>]</span> [<a href="/pubmed/10802651" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Gene+ontology:+tool+for+the+unification+of
biology&amp;author=M+Ashburner&amp;volume=25&amp;publication_year=2000&amp;pages=25-29&amp;pmid=10802651&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Menni C, et al.  Circulating proteomic signatures of chronological
age. <span><span class="ref-journal">J Gerontol A Biol Sci Med Sci. </span>2015;<span class="ref-vol">70</span>:809–816.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4469006/">PMC free article</a>]</span> [<a href="/pubmed/25123647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Gerontol+A+Biol+Sci+Med+Sci&amp;title=Circulating+proteomic+signatures+of+chronological
age&amp;author=C+Menni&amp;volume=70&amp;publication_year=2015&amp;pages=809-816&amp;pmid=25123647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Ngo D, et al.  Aptamer-based proteomic profiling reveals novel candidate
biomarkers and pathways in cardiovascular disease. <span><span class="ref-journal">Circulation. </span>2016;<span class="ref-vol">134</span>:270–285.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4963294/">PMC free article</a>]</span> [<a href="/pubmed/27444932" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Aptamer-based+proteomic+profiling+reveals+novel+candidate
biomarkers+and+pathways+in+cardiovascular+disease&amp;author=D+Ngo&amp;volume=134&amp;publication_year=2016&amp;pages=270-285&amp;pmid=27444932&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Chang CC, et al.  Second-generation PLINK: rising to the challenge of larger and
richer datasets. <span><span class="ref-journal">Gigascience. </span>2015;<span class="ref-vol">4</span>:7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4342193/">PMC free article</a>]</span> [<a href="/pubmed/25722852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Second-generation+PLINK:+rising+to+the+challenge+of+larger+and
richer+datasets&amp;author=CC+Chang&amp;volume=4&amp;publication_year=2015&amp;pages=7&amp;pmid=25722852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by
imputation of genotypes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by
imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association
scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association
scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Yang J, et al.  Conditional and joint multiple-SNP analysis of GWAS summary
statistics identifies additional variants influencing complex
traits. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:369–375.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3593158/">PMC free article</a>]</span> [<a href="/pubmed/22426310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Conditional+and+joint+multiple-SNP+analysis+of+GWAS+summary
statistics+identifies+additional+variants+influencing+complex
traits&amp;author=J+Yang&amp;volume=44&amp;publication_year=2012&amp;pages=369-375&amp;pmid=22426310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Welter D, et al.  The NHGRI GWAS Catalog, a curated resource of SNP-trait
associations. <span><span class="ref-journal">Nucleic Acids Res. </span>2014;<span class="ref-vol">42</span>:D1001–D1006.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3965119/">PMC free article</a>]</span> [<a href="/pubmed/24316577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+NHGRI+GWAS+Catalog,+a+curated+resource+of+SNP-trait
associations&amp;author=D+Welter&amp;volume=42&amp;publication_year=2014&amp;pages=D1001-D1006&amp;pmid=24316577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Malone J, et al.  Modeling sample variables with an Experimental Factor
Ontology. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:1112–1118.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2853691/">PMC free article</a>]</span> [<a href="/pubmed/20200009" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Modeling+sample+variables+with+an+Experimental+Factor
Ontology&amp;author=J+Malone&amp;volume=26&amp;publication_year=2010&amp;pages=1112-1118&amp;pmid=20200009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Javierre BM, et al.  Lineage-specific genome architecture links enhancers and
non-coding disease variants to target gene promoters. <span><span class="ref-journal">Cell. </span>2016;<span class="ref-vol">167</span>:1369–1384.e19.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5123897/">PMC free article</a>]</span> [<a href="/pubmed/27863249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Lineage-specific+genome+architecture+links+enhancers+and
non-coding+disease+variants+to+target+gene+promoters&amp;author=BM+Javierre&amp;volume=167&amp;publication_year=2016&amp;pages=1369-1384.e19&amp;pmid=27863249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Lawrence M, et al.  Software for computing and annotating genomic
ranges. <span><span class="ref-journal">PLOS Comput Biol. </span>2013;<span class="ref-vol">9</span>:e1003118.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3738458/">PMC free article</a>]</span> [<a href="/pubmed/23950696" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLOS+Comput+Biol&amp;title=Software+for+computing+and+annotating+genomic
ranges&amp;author=M+Lawrence&amp;volume=9&amp;publication_year=2013&amp;pages=e1003118&amp;pmid=23950696&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Amberger JS, Bocchini CA, Schiettecatte F, Scott AF, Hamosh A. OMIM. org: Online Mendelian Inheritance in Man (OMIM®), an
online catalog of human genes and genetic disorders. <span><span class="ref-journal">Nucleic Acids Res. </span>2015;<span class="ref-vol">43</span>:D789–D798.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4383985/">PMC free article</a>]</span> [<a href="/pubmed/25428349" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=OMIM.+org:+Online+Mendelian+Inheritance+in+Man+(OMIM®),+an
online+catalog+of+human+genes+and+genetic+disorders&amp;author=JS+Amberger&amp;author=CA+Bocchini&amp;author=F+Schiettecatte&amp;author=AF+Scott&amp;author=A+Hamosh&amp;volume=43&amp;publication_year=2015&amp;pages=D789-D798&amp;pmid=25428349&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Kanehisa M, Sato Y, Kawashima M, Furumichi M, Tanabe M. KEGG as a reference resource for gene and protein
annotation. <span><span class="ref-journal">Nucleic Acids Res. </span>2016;<span class="ref-vol">44</span>:D457–D462.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4702792/">PMC free article</a>]</span> [<a href="/pubmed/26476454" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=KEGG+as+a+reference+resource+for+gene+and+protein
annotation&amp;author=M+Kanehisa&amp;author=Y+Sato&amp;author=M+Kawashima&amp;author=M+Furumichi&amp;author=M+Tanabe&amp;volume=44&amp;publication_year=2016&amp;pages=D457-D462&amp;pmid=26476454&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">Szklarczyk D, et al.  STRINGv10: protein-protein interaction networks, integrated over
the tree of life. <span><span class="ref-journal">Nucleic Acids Res. </span>2015;<span class="ref-vol">43</span>:D447–D452.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4383874/">PMC free article</a>]</span> [<a href="/pubmed/25352553" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=STRINGv10:+protein-protein+interaction+networks,+integrated+over
the+tree+of+life&amp;author=D+Szklarczyk&amp;volume=43&amp;publication_year=2015&amp;pages=D447-D452&amp;pmid=25352553&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="element-citation">Smith RN, et al.  InterMine: a flexible data warehouse system for the integration
and analysis of heterogeneous biological data. <span><span class="ref-journal">Bioinformatics. </span>2012;<span class="ref-vol">28</span>:3163–3165.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3516146/">PMC free article</a>]</span> [<a href="/pubmed/23023984" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=InterMine:+a+flexible+data+warehouse+system+for+the+integration
and+analysis+of+heterogeneous+biological+data&amp;author=RN+Smith&amp;volume=28&amp;publication_year=2012&amp;pages=3163-3165&amp;pmid=23023984&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="element-citation">Franceschini A, et al.  STRINGv9.1: protein–protein interaction networks, with
increased coverage and integration. <span><span class="ref-journal">Nucleic Acids Res. </span>2013;<span class="ref-vol">41</span>:D808–D815.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3531103/">PMC free article</a>]</span> [<a href="/pubmed/23203871" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=STRINGv9.1:+protein–protein+interaction+networks,+with
increased+coverage+and+integration&amp;author=A+Franceschini&amp;volume=41&amp;publication_year=2013&amp;pages=D808-D815&amp;pmid=23203871&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">59. <span class="element-citation">Lek M, et al.  Analysis of protein-coding genetic variation in 60,706
humans. <span><span class="ref-journal">Nature. </span>2016;<span class="ref-vol">536</span>:285–291.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5018207/">PMC free article</a>]</span> [<a href="/pubmed/27535533" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Analysis+of+protein-coding+genetic+variation+in+60,706
humans&amp;author=M+Lek&amp;volume=536&amp;publication_year=2016&amp;pages=285-291&amp;pmid=27535533&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">60. <span class="element-citation">Iotchkova V, et al.  GARFIELD—GWAS Analysis of Regulatory or Functional
Information Enrichment with LD correction. <span>2016</span> Preprint at  <a href="https://www.biorxiv.org/content/early/2016/11/07/085738" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://www.biorxiv.org/content/early/2016/11/07/085738</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R61">61. <span class="element-citation">Staley JR, et al.  PhenoScanner: a database of human genotype–phenotype
associations. <span><span class="ref-journal">Bioinformatics. </span>2016;<span class="ref-vol">32</span>:3207–3209.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5048068/">PMC free article</a>]</span> [<a href="/pubmed/27318201" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PhenoScanner:+a+database+of+human+genotype–phenotype
associations&amp;author=JR+Staley&amp;volume=32&amp;publication_year=2016&amp;pages=3207-3209&amp;pmid=27318201&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R62">62. <span class="element-citation">Giambartolomei C, et al.  Bayesian test for colocalisation between pairs of genetic
association studies using summary statistics. <span><span class="ref-journal">PLoS Genet. </span>2014;<span class="ref-vol">10</span>:e1004383.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4022491/">PMC free article</a>]</span> [<a href="/pubmed/24830394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Bayesian+test+for+colocalisation+between+pairs+of+genetic
association+studies+using+summary+statistics&amp;author=C+Giambartolomei&amp;volume=10&amp;publication_year=2014&amp;pages=e1004383&amp;pmid=24830394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R63">63. <span class="element-citation">Hingorani A, Humphries S. Nature’s randomised trials. <span><span class="ref-journal">Lancet. </span>2005;<span class="ref-vol">366</span>:1906–1908.</span> [<a href="/pubmed/16325682" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Nature’s+randomised+trials&amp;author=A+Hingorani&amp;author=S+Humphries&amp;volume=366&amp;publication_year=2005&amp;pages=1906-1908&amp;pmid=16325682&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R64">64. <span class="element-citation">Nikpay M, et al.  A comprehensive 1,000 Genomes-based genome-wide association
meta-analysis of coronary artery disease. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:1121–1130.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4589895/">PMC free article</a>]</span> [<a href="/pubmed/26343387" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+comprehensive+1,000+Genomes-based+genome-wide+association
meta-analysis+of+coronary+artery+disease&amp;author=M+Nikpay&amp;volume=47&amp;publication_year=2015&amp;pages=1121-1130&amp;pmid=26343387&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R65">65. <span class="element-citation">Burgess S, Butterworth A, Thompson SG. Mendelian randomization analysis with multiple genetic variants
using summarized data. <span><span class="ref-journal">Genet Epidemiol. </span>2013;<span class="ref-vol">37</span>:658–665.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4377079/">PMC free article</a>]</span> [<a href="/pubmed/24114802" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Mendelian+randomization+analysis+with+multiple+genetic+variants
using+summarized+data&amp;author=S+Burgess&amp;author=A+Butterworth&amp;author=SG+Thompson&amp;volume=37&amp;publication_year=2013&amp;pages=658-665&amp;pmid=24114802&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R66">66. <span class="element-citation">Burgess S, Dudbridge F, Thompson SG. Combining information on multiple instrumental variables in
Mendelian randomization: comparison of allele score and summarized data
methods. <span><span class="ref-journal">Stat Med. </span>2016;<span class="ref-vol">35</span>:1880–1906.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4832315/">PMC free article</a>]</span> [<a href="/pubmed/26661904" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Med&amp;title=Combining+information+on+multiple+instrumental+variables+in
Mendelian+randomization:+comparison+of+allele+score+and+summarized+data
methods&amp;author=S+Burgess&amp;author=F+Dudbridge&amp;author=SG+Thompson&amp;volume=35&amp;publication_year=2016&amp;pages=1880-1906&amp;pmid=26661904&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R67">67. <span class="element-citation">Burgess S, Thompson SG. Multivariable Mendelian randomization: the use of pleiotropic
genetic variants to estimate causal effects. <span><span class="ref-journal">Am J Epidemiol. </span>2015;<span class="ref-vol">181</span>:251–260.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4325677/">PMC free article</a>]</span> [<a href="/pubmed/25632051" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Multivariable+Mendelian+randomization:+the+use+of+pleiotropic
genetic+variants+to+estimate+causal+effects&amp;author=S+Burgess&amp;author=SG+Thompson&amp;volume=181&amp;publication_year=2015&amp;pages=251-260&amp;pmid=25632051&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R68">68. <span class="element-citation">Burgess S, Dudbridge F, Thompson SG. Re: “Multivariable Mendelian randomization: the use of
pleiotropic genetic variants to estimate causal
effects” <span><span class="ref-journal">Am J Epidemiol. </span>2015;<span class="ref-vol">181</span>:290–291.</span> [<a href="/pubmed/25660081" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Re:+“Multivariable+Mendelian+randomization:+the+use+of
pleiotropic+genetic+variants+to+estimate+causal
effects”&amp;author=S+Burgess&amp;author=F+Dudbridge&amp;author=SG+Thompson&amp;volume=181&amp;publication_year=2015&amp;pages=290-291&amp;pmid=25660081&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140142931798448title">Abstract</a></li><li><a href="#S1title">Genetic architecture of the plasma proteome</a></li><li><a href="#S2title">Overlap of eQTLs and pQTLs</a></li><li><a href="#S3title">trans pQTLs identify pathways to disease</a></li><li><a href="#S4title">Causal evaluation of candidate proteins in disease</a></li><li><a href="#S5title">Drug target prioritization</a></li><li><a href="#S6title">Discussion</a></li><li><a href="#S7title">Online content</a></li><li><a href="#S8title">Methods</a></li><li><a href="#S33title">Extended Data</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S34title">Acknowledgements</a></li><li><a href="#idm140142947924144title">Footnotes</a></li><li><a href="#idm140142935372128title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC6697541/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC6697541/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC6697541/pdf/EMS83968.pdf">PDF (5.3M)</a></li> | <li><a href="#" data-citationid="PMC6697541" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6697541%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6697541%2F&amp;text=Genomic%20atlas%20of%20the%20human%20plasma%20proteome" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6697541%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="29875488" data-aiid="6697541" data-aid="6697541" data-iid="340460" data-domainid="358" data-domain="wtpa" data-accid="PMC6697541" data-md5="968cdc0dd8ade07c396e77c7220228a3">
<div class="portlet pubmed_favoritesad">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Save items</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="pmfavad" toggles="ui-portlet_content-5" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-5" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<input name="absid" id="absid" value="6697541" type="hidden"><div id="pubmed_favoritesad" class="empty" data-db="pmc">
<span role="menubar"><div class="ui-ncbisetswitch ltd-hover" role="menuitem" aria-expanded="false" aria-haspopup="true"><a href="/myncbi/collections/" class="jig-ncbisetswitch blind collink ltd-hover ui-ncbisetswitch-hasStar" id="favList" data-jigconfig="destSelector:'#favUL',starExists:true,createCollectionUrl:'#',manageCollectionsUrl:'/myncbi/collections/'"><span class="star"></span>Add to Favorites</a><a class="ui-ncbisetswitch-button ltd-hover" title="View more options" href="#"><span class="ui-icon ui-icon-triangle-1-s ui-ncbisetswitch-down">View more options</span></a></div></span><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu ui-ncbisetswitch-popper"><ul id="favUL" class="ui-ncbisetswitch-options"></ul><ul class="ui-ncbisetswitch-actions"><li class="ui-ncbisetswitch-actions-create" style="display: block;"><a class="ui-ncbisetswitch-create-collection" href="#">Create collection...</a></li><li class="ui-ncbisetswitch-actions-manage" style="display: block;"><a class="ui-ncbisetswitch-manage-collections" href="/myncbi/collections/">Manage collections...</a></li></ul></div>
</div>
<div class="colloading">loading</div>
<span id="submenu_AddToCollections"><form method="post"><input class="button_apply" name="addtocollection" value="1" type="submit"></form></span>
</div></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" toggles="ui-portlet_content-6" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-6" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/28129359" ref="reftype=relart&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">High-Throughput Characterization of Blood Serum Proteomics of IBD Patients with Respect to Aging and Genetic Factors.</a><span class="source">[PLoS Genet. 2017]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Di Narzo AF, Telesco SE, Brodmerkel C, Argmann C, Peters LA, Li K, Kidd B, Dudley J, Cho J, Schadt EE, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">PLoS Genet. 2017 Jan; 13(1):e1006565. Epub 2017 Jan 27.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/29258991" ref="reftype=relart&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genetic Architecture of the Cardiovascular Risk Proteome.</a><span class="source">[Circulation. 2018]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Benson MD, Yang Q, Ngo D, Zhu Y, Shen D, Farrell LA, Sinha S, Keyes MJ, Vasan RS, Larson MG, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Circulation. 2018 Mar 13; 137(11):1158-1172. Epub 2017 Dec 19.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/24699359" ref="reftype=relart&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Protein quantitative trait loci identify novel candidates modulating cellular response to chemotherapy.</a><span class="source">[PLoS Genet. 2014]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Stark AL, Hause RJ Jr, Gorsic LK, Antao NN, Wong SS, Chung SH, Gill DF, Im HK, Myers JL, White KP, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">PLoS Genet. 2014 Apr; 10(4):e1004192. Epub 2014 Apr 3.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/28438806" ref="reftype=relart&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Emerging Affinity-Based Proteomic Technologies for Large-Scale Plasma Profiling in Cardiovascular Disease.</a><span class="source">[Circulation. 2017]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Smith JG, Gerszten RE. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Circulation. 2017 Apr 25; 135(17):1651-1664. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/26444824" ref="reftype=relart&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">The genetic architecture of inflammatory bowel disease: past, present and future.</a><span class="source">[Curr Opin Gastroenterol. 2015]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-11" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Cleynen I, Vermeire S. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Curr Opin Gastroenterol. 2015 Nov; 31(6):456-63. </em>
</div>
</li>
</ul>
<a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=29875488&amp;log%24=relatedreviews&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=29875488&amp;log%24=relatedarticles&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Cited by other articles in PMC</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-12" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-12" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6858581/" ref="reftype=relart&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genome-wide association analysis of venous thromboembolism identifies new risk loci and genetic overlap with arterial vascular disease</a><span class="source">[Nature genetics. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Klarin D, Busenkell E, Judy R, Lynch J, Levin M, Haessler J, Aragam K, Chaffin M, Haas M, Lindström S, Assimes TL, Huang J, Lee KM, Shao Q, Huffman JE, Kabrhel C, Huang Y, Sun YV, Vujkovic M, Saleheen D, Miller DR, Reaven P, DuVall S, Boden WE, Pyarajan S, Reiner AP, Trégouët DA, Henke P, Kooperberg C, Gaziano JM, Concato J, Rader DJ, Cho K, Chang KM, Wilson PW, Smith NL, O’Donnell CJ, Tsao PS, Kathiresan S, Obi A, Damrauer SM, Natarajan P, INVENT Consortium, VA Million Veteran Program. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nature genetics. 2019 Nov 1; 51(11): 1574-1579</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6839336/" ref="reftype=relart&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Multi-platform Affinity Proteomics Identify Proteins Linked to Metastasis and Immune Suppression in Ovarian Cancer Plasma</a><span class="source">[Frontiers in Oncology. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Graumann J, Finkernagel F, Reinartz S, Stief T, Brödje D, Renz H, Jansen JM, Wagner U, Worzfeld T, Pogge von Strandmann E, Müller R. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Frontiers in Oncology. 2019 Nov 1; 9: 1150</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6835559/" ref="reftype=relart&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Plasma Proteome Profiling to detect and avoid sample‐related biases in biomarker studies</a><span class="source">[EMBO Molecular Medicine. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-15" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Geyer PE, Voytik E, Treit PV, Doll S, Kleinhempel A, Niu L, Müller JB, Buchholtz M, Bader JM, Teupser D, Holdt LM, Mann M. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">EMBO Molecular Medicine. 2019 Sep 30; 11(11): e10427</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6826130/" ref="reftype=relart&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A serum protein signature of APOE genotypes in centenarians</a><span class="source">[Aging Cell. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Sebastiani P, Monti S, Morris M, Gurinovich A, Toshiko T, Andersen SL, Sweigart B, Ferrucci L, Jennings LL, Glass DJ, Perls TT. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Aging Cell. 2019 Aug 5; 18(6): e13023</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6817708/" ref="reftype=relart&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genome-wide association study identifies seven novel loci associating with circulating cytokines and cell adhesion molecules in Finns</a><span class="source">[Journal of Medical Genetics. 2...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-17" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Sliz E, Kalaoja M, Ahola-Olli A, Raitakari O, Perola M, Salomaa V, Lehtimäki T, Karhu T, Viinamäki H, Salmi M, Santalahti K, Jalkanen S, Jokelainen J, Keinänen-Kiukaanniemi S, Männikkö M, Herzig KH, Järvelin MR, Sebert S, Kettunen J. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Journal of Medical Genetics. 2019 Jun 19; 56(9): 607-616</em>
</div>
</li>
</ul>
<a class="seemore" href="citedby/" ref="reftype=relart&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|CitedBy&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Links</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-18" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-18" aria-hidden="false"><ul>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=medgen&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_medgen&amp;IdsFromResult=6697541" ref="reftype=MedGen&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">MedGen</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-19" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">MedGen<div class="brieflinkpopdesc">Related information in MedGen</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pubmed/29875488/" ref="reftype=PubMed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-20" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=taxonomy&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_taxonomy&amp;IdsFromResult=6697541" ref="reftype=Taxonomy&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Taxonomy</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-21" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Taxonomy<div class="brieflinkpopdesc">Related taxonomy entry</div>
</div>
</li>
</ul></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Recent Activity</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" toggles="ui-portlet_content-22" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-22" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class="">
<div class="action">
<a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a>
</div>
<ul id="activity">
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5dd5ba26dde0893a8ba75cf6">Genomic atlas of the human plasma proteome</a><div class="ralinkpop offscreen_noflow">Genomic atlas of the human plasma proteome<div class="brieflinkpopdesc">UKPMC Funders Author Manuscripts. 2018 Jun 1; 558(7708)73</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=5dd5b9fc75f66d018f97ee4b">A locus at 7p14.3 predisposes to refractory celiac disease progression from celi...</a><div class="ralinkpop offscreen_noflow">A locus at 7p14.3 predisposes to refractory celiac disease progression from celiac disease<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2018 Aug; 30(8)828</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=5dd5b9d4dd2142621d93e39a">Common and Rare Variants Genetic Association Analysis of Cigarettes per Day Amon...</a><div class="ralinkpop offscreen_noflow">Common and Rare Variants Genetic Association Analysis of Cigarettes per Day Among Ever-Smokers in Chronic Obstructive Pulmonary Disease Cases and Controls<div class="brieflinkpopdesc">Nicotine &amp;#x0026; Tobacco Research. 2019 Jun; 21(6)714</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=5dd5b9ac39a66eb03d6d3b6d">Common variants on 6q16.2, 12q24.31 and 16p13.3 are associated with major depres...</a><div class="ralinkpop offscreen_noflow">Common variants on 6q16.2, 12q24.31 and 16p13.3 are associated with major depressive disorder<div class="brieflinkpopdesc">Neuropsychopharmacology. 2018 Sep; 43(10)2146</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=5dd5b990dde0893a8b9471f2">A Genome-Wide Association Study of Diabetic Kidney Disease in Subjects With Type...</a><div class="ralinkpop offscreen_noflow">A Genome-Wide Association Study of Diabetic Kidney Disease in Subjects With Type 2 Diabetes<div class="brieflinkpopdesc">Diabetes. 2018 Jul; 67(7)1414</div>
</div>
<div class="tertiary"></div>
</li>
</ul>
<p class="HTOn">Your browsing activity is empty.</p>
<p class="HTOff">Activity recording is turned off.</p>
<p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p>
<a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
</div></div></div>
</div>
<div>
<div class="portlet brieflink pmc_para_cit" id="crb--P4" name="crb--P4" rid="P4" style="position: absolute; visibility: visible; width: 370px; top: 1170px; display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-23"><ul>
<li class="two_line" reference_id="R1">
<a href="/pubmed/25707927/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> The role of regulatory variation in complex traits and disease.</a><span class="one_line_source">[Nat Rev Genet.  2015]</span><div class="alt-note">
<div class="authors">Albert FW, Kruglyak L</div>
<div class="citation">Nat Rev Genet. 2015 Apr; 16(4):197-212.</div>
</div>
</li>
<li class="two_line" reference_id="R2">
<a href="/pubmed/25652787/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Quantitative variability of 342 plasma proteins in a human twin population.</a><span class="one_line_source">[Mol Syst Biol.  2015]</span><div class="alt-note">
<div class="authors">Liu Y, Buil A, Collins BC, Gillet LC, Blum LC, Cheng LY, Vitek O, Mouritsen J, Lachance G, Spector TD, Dermitzakis ET, Aebersold R</div>
<div class="citation">Mol Syst Biol. 2015 Feb 4; 11(1):786.</div>
</div>
</li>
<li class="two_line" reference_id="R3">
<a href="/pubmed/28397792/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Erratum: Connecting genetic risk to disease end points through the human blood plasma proteome.</a><span class="one_line_source">[Nat Commun.  2017]</span><div class="alt-note">
<div class="authors">Suhre K, Arnold M, Bhagwat AM, Cotton RJ, Engelke R, Raffler J, Sarwath H, Thareja G, Wahl A, DeLisle RK, Gold L, Pezer M, Lauc G, El-Din Selim MA, Mook-Kanamori DO, Al-Dous EK, Mohamoud YA, Malek J, Strauch K, Grallert H, Peters A, Kastenmüller G, Gieger C, Graumann J</div>
<div class="citation">Nat Commun. 2017 Apr 11; 8():15345.</div>
</div>
</li>
<li class="two_line" reference_id="R5" style="display: none;">
<a href="/pubmed/28854705/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Whole-genome sequencing study of serum peptide levels: the Atherosclerosis Risk in Communities study.</a><span class="one_line_source">[Hum Mol Genet.  2017]</span><div class="alt-note">
<div class="authors">de Vries PS, Yu B, Feofanova EV, Metcalf GA, Brown MR, Zeighami AL, Liu X, Muzny DM, Gibbs RA, Boerwinkle E, Morrison AC</div>
<div class="citation">Hum Mol Genet. 2017 Sep 1; 26(17):3442-3450.</div>
</div>
</li>
<li class="two_line" reference_id="R6" style="display: none;">
<a href="/pubmed/23676674/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Variation and genetic control of protein abundance in humans.</a><span class="one_line_source">[Nature.  2013]</span><div class="alt-note">
<div class="authors">Wu L, Candille SI, Choi Y, Xie D, Jiang L, Li-Pook-Than J, Tang H, Snyder M</div>
<div class="citation">Nature. 2013 Jul 4; 499(7456):79-82.</div>
</div>
</li>
<li class="two_line" reference_id="R7" style="display: none;">
<a href="/pubmed/25657249/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genomic variation. Impact of regulatory variation from RNA to protein.</a><span class="one_line_source">[Science.  2015]</span><div class="alt-note">
<div class="authors">Battle A, Khan Z, Wang SH, Mitrano A, Ford MJ, Pritchard JK, Gilad Y</div>
<div class="citation">Science. 2015 Feb 6; 347(6222):664-7.</div>
</div>
</li>
</ul><a href="" class="seemore">See more ...</a></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P5" name="crb--P5" rid="P5" style="position: absolute; visibility: visible; width: 370px; top: 1336px; display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-24"><ul>
<li class="two_line" reference_id="R8">
<a href="/pubmed/25291143/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Nucleic Acid Ligands With Protein-like Side Chains: Modified Aptamers and Their Use as Diagnostic and Therapeutic Agents.</a><span class="one_line_source">[Mol Ther Nucleic Acids.  2014]</span><div class="alt-note">
<div class="authors">Rohloff JC, Gelinas AD, Jarvis TC, Ochsner UA, Schneider DJ, Gold L, Janjic N</div>
<div class="citation">Mol Ther Nucleic Acids. 2014 Oct 7; 3():e201.</div>
</div>
</li>
<li class="two_line" reference_id="R9">
<a href="/pubmed/28941948/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Efficiency and safety of varying the frequency of whole blood donation (INTERVAL): a randomised trial of 45 000 donors.</a><span class="one_line_source">[Lancet.  2017]</span><div class="alt-note">
<div class="authors">Di Angelantonio E, Thompson SG, Kaptoge S, Moore C, Walker M, Armitage J, Ouwehand WH, Roberts DJ, Danesh J, INTERVAL Trial Group.</div>
<div class="citation">Lancet. 2017 Nov 25; 390(10110):2360-2371.</div>
</div>
</li>
<li class="two_line" reference_id="R10">
<a href="/pubmed/27863252/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The Allelic Landscape of Human Blood Cell Trait Variation and Links to Common Complex Disease.</a><span class="one_line_source">[Cell.  2016]</span><div class="alt-note">
<div class="authors">Astle WJ, Elding H, Jiang T, Allen D, Ruklisa D, Mann AL, Mead D, Bouman H, Riveros-Mckay F, Kostadima MA, Lambourne JJ, Sivapalaratnam S, Downes K, Kundu K, Bomba L, Berentsen K, Bradley JR, Daugherty LC, Delaneau O, Freson K, Garner SF, Grassi L, Guerrero J, Haimel M, Janssen-Megens EM, Kaan A, Kamat M, Kim B, Mandoli A, Marchini J, Martens JHA, Meacham S, Megy K, O'Connell J, Petersen R, Sharifi N, Sheard SM, Staley JR, Tuna S, van der Ent M, Walter K, Wang SY, Wheeler E, Wilder SP, Iotchkova V, Moore C, Sambrook J, Stunnenberg HG, Di Angelantonio E, Kaptoge S, Kuijpers TW, Carrillo-de-Santa-Pau E, Juan D, Rico D, Valencia A, Chen L, Ge B, Vasquez L, Kwan T, Garrido-Martín D, Watt S, Yang Y, Guigo R, Beck S, Paul DS, Pastinen T, Bujold D, Bourque G, Frontini M, Danesh J, Roberts DJ, Ouwehand WH, Butterworth AS, Soranzo N</div>
<div class="citation">Cell. 2016 Nov 17; 167(5):1415-1429.e19.</div>
</div>
</li>
<li class="two_line" reference_id="R11">
<a href="/pubmed/25773750/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Using published data in Mendelian randomization: a blueprint for efficient identification of causal risk factors.</a><span class="one_line_source">[Eur J Epidemiol.  2015]</span><div class="alt-note">
<div class="authors">Burgess S, Scott RA, Timpson NJ, Davey Smith G, Thompson SG, EPIC- InterAct Consortium.</div>
<div class="citation">Eur J Epidemiol. 2015 Jul; 30(7):543-52.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P7" name="crb--P7" rid="P7" style="position: absolute; visibility: visible; width: 370px; top: 1821px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-25"><ul><li class="two_line" reference_id="R12">
<a href="/pubmed/22532805/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Patterns of cis regulatory variation in diverse human populations.</a><span class="one_line_source">[PLoS Genet.  2012]</span><div class="alt-note">
<div class="authors">Stranger BE, Montgomery SB, Dimas AS, Parts L, Stegle O, Ingle CE, Sekowska M, Smith GD, Evans D, Gutierrez-Arcelus M, Price A, Raj T, Nisbett J, Nica AC, Beazley C, Durbin R, Deloukas P, Dermitzakis ET</div>
<div class="citation">PLoS Genet. 2012; 8(4):e1002639.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P8" name="crb--P8" rid="P8" style="position: absolute; visibility: visible; width: 370px; top: 3050px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-26"><ul><li class="two_line" reference_id="R13">
<a href="/pubmed/21646338/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Homogeneous antibody-based proximity extension assays provide sensitive and specific detection of low-abundant proteins in human blood.</a><span class="one_line_source">[Nucleic Acids Res.  2011]</span><div class="alt-note">
<div class="authors">Lundberg M, Eriksson A, Tran B, Assarsson E, Fredriksson S</div>
<div class="citation">Nucleic Acids Res. 2011 Aug; 39(15):e102.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P10" name="crb--P10" rid="P10" style="position: absolute; visibility: visible; width: 370px; top: 3337px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-27"><ul>
<li class="two_line" reference_id="R7">
<a href="/pubmed/25657249/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genomic variation. Impact of regulatory variation from RNA to protein.</a><span class="one_line_source">[Science.  2015]</span><div class="alt-note">
<div class="authors">Battle A, Khan Z, Wang SH, Mitrano A, Ford MJ, Pritchard JK, Gilad Y</div>
<div class="citation">Science. 2015 Feb 6; 347(6222):664-7.</div>
</div>
</li>
<li class="two_line" reference_id="R10">
<a href="/pubmed/27863252/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The Allelic Landscape of Human Blood Cell Trait Variation and Links to Common Complex Disease.</a><span class="one_line_source">[Cell.  2016]</span><div class="alt-note">
<div class="authors">Astle WJ, Elding H, Jiang T, Allen D, Ruklisa D, Mann AL, Mead D, Bouman H, Riveros-Mckay F, Kostadima MA, Lambourne JJ, Sivapalaratnam S, Downes K, Kundu K, Bomba L, Berentsen K, Bradley JR, Daugherty LC, Delaneau O, Freson K, Garner SF, Grassi L, Guerrero J, Haimel M, Janssen-Megens EM, Kaan A, Kamat M, Kim B, Mandoli A, Marchini J, Martens JHA, Meacham S, Megy K, O'Connell J, Petersen R, Sharifi N, Sheard SM, Staley JR, Tuna S, van der Ent M, Walter K, Wang SY, Wheeler E, Wilder SP, Iotchkova V, Moore C, Sambrook J, Stunnenberg HG, Di Angelantonio E, Kaptoge S, Kuijpers TW, Carrillo-de-Santa-Pau E, Juan D, Rico D, Valencia A, Chen L, Ge B, Vasquez L, Kwan T, Garrido-Martín D, Watt S, Yang Y, Guigo R, Beck S, Paul DS, Pastinen T, Bujold D, Bourque G, Frontini M, Danesh J, Roberts DJ, Ouwehand WH, Butterworth AS, Soranzo N</div>
<div class="citation">Cell. 2016 Nov 17; 167(5):1415-1429.e19.</div>
</div>
</li>
<li class="two_line" reference_id="R14">
<a href="/pubmed/26367797/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The UK10K project identifies rare variants in health and disease.</a><span class="one_line_source">[Nature.  2015]</span><div class="alt-note">
<div class="authors">UK10K Consortium., Walter K, Min JL, Huang J, Crooks L, Memari Y, McCarthy S, Perry JR, Xu C, Futema M, Lawson D, Iotchkova V, Schiffels S, Hendricks AE, Danecek P, Li R, Floyd J, Wain LV, Barroso I, Humphries SE, Hurles ME, Zeggini E, Barrett JC, Plagnol V, Richards JB, Greenwood CM, Timpson NJ, Durbin R, Soranzo N</div>
<div class="citation">Nature. 2015 Oct 1; 526(7571):82-90.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P12" name="crb--P12" rid="P12" style="position: absolute; visibility: visible; width: 370px; top: 3666px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-28"><ul>
<li class="two_line" reference_id="R15">
<a href="/pubmed/24013639/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Systematic identification of trans eQTLs as putative drivers of known disease associations.</a><span class="one_line_source">[Nat Genet.  2013]</span><div class="alt-note">
<div class="authors">Westra HJ, Peters MJ, Esko T, Yaghootkar H, Schurmann C, Kettunen J, Christiansen MW, Fairfax BP, Schramm K, Powell JE, Zhernakova A, Zhernakova DV, Veldink JH, Van den Berg LH, Karjalainen J, Withoff S, Uitterlinden AG, Hofman A, Rivadeneira F, Hoen PAC', Reinmaa E, Fischer K, Nelis M, Milani L, Melzer D, Ferrucci L, Singleton AB, Hernandez DG, Nalls MA, Homuth G, Nauck M, Radke D, Völker U, Perola M, Salomaa V, Brody J, Suchy-Dicey A, Gharib SA, Enquobahrie DA, Lumley T, Montgomery GW, Makino S, Prokisch H, Herder C, Roden M, Grallert H, Meitinger T, Strauch K, Li Y, Jansen RC, Visscher PM, Knight JC, Psaty BM, Ripatti S, Teumer A, Frayling TM, Metspalu A, van Meurs JBJ, Franke L</div>
<div class="citation">Nat Genet. 2013 Oct; 45(10):1238-1243.</div>
</div>
</li>
<li class="two_line" reference_id="R18">
<a href="/pubmed/20502693/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genetics and beyond--the transcriptome of human monocytes and disease susceptibility.</a><span class="one_line_source">[PLoS One.  2010]</span><div class="alt-note">
<div class="authors">Zeller T, Wild P, Szymczak S, Rotival M, Schillert A, Castagne R, Maouche S, Germain M, Lackner K, Rossmann H, Eleftheriadis M, Sinning CR, Schnabel RB, Lubos E, Mennerich D, Rust W, Perret C, Proust C, Nicaud V, Loscalzo J, Hübner N, Tregouet D, Münzel T, Ziegler A, Tiret L, Blankenberg S, Cambien F</div>
<div class="citation">PLoS One. 2010 May 18; 5(5):e10693.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P14" name="crb--P14" rid="P14" style="position: absolute; visibility: visible; width: 370px; top: 4237px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-29"><ul>
<li class="two_line" reference_id="R19">
<a href="/pubmed/21873547/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">TMPRSS6 rs855791 modulates hepcidin transcription in vitro and serum hepcidin levels in normal individuals.</a><span class="one_line_source">[Blood.  2011]</span><div class="alt-note">
<div class="authors">Nai A, Pagani A, Silvestri L, Campostrini N, Corbella M, Girelli D, Traglia M, Toniolo D, Camaschella C</div>
<div class="citation">Blood. 2011 Oct 20; 118(16):4459-62.</div>
</div>
</li>
<li class="two_line" reference_id="R20">
<a href="/pubmed/21087763/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide screen identifies rs646776 near sortilin as a regulator of progranulin levels in human plasma.</a><span class="one_line_source">[Am J Hum Genet.  2010]</span><div class="alt-note">
<div class="authors">Carrasquillo MM, Nicholson AM, Finch N, Gibbs JR, Baker M, Rutherford NJ, Hunter TA, DeJesus-Hernandez M, Bisceglio GD, Mackenzie IR, Singleton A, Cookson MR, Crook JE, Dillman A, Hernandez D, Petersen RC, Graff-Radford NR, Younkin SG, Rademakers R</div>
<div class="citation">Am J Hum Genet. 2010 Dec 10; 87(6):890-7.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P15" name="crb--P15" rid="P15" style="position: absolute; visibility: visible; width: 370px; top: 4402px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-30"><ul>
<li class="two_line" reference_id="R3">
<a href="/pubmed/28397792/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Erratum: Connecting genetic risk to disease end points through the human blood plasma proteome.</a><span class="one_line_source">[Nat Commun.  2017]</span><div class="alt-note">
<div class="authors">Suhre K, Arnold M, Bhagwat AM, Cotton RJ, Engelke R, Raffler J, Sarwath H, Thareja G, Wahl A, DeLisle RK, Gold L, Pezer M, Lauc G, El-Din Selim MA, Mook-Kanamori DO, Al-Dous EK, Mohamoud YA, Malek J, Strauch K, Grallert H, Peters A, Kastenmüller G, Gieger C, Graumann J</div>
<div class="citation">Nat Commun. 2017 Apr 11; 8():15345.</div>
</div>
</li>
<li class="two_line" reference_id="R5">
<a href="/pubmed/28854705/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Whole-genome sequencing study of serum peptide levels: the Atherosclerosis Risk in Communities study.</a><span class="one_line_source">[Hum Mol Genet.  2017]</span><div class="alt-note">
<div class="authors">de Vries PS, Yu B, Feofanova EV, Metcalf GA, Brown MR, Zeighami AL, Liu X, Muzny DM, Gibbs RA, Boerwinkle E, Morrison AC</div>
<div class="citation">Hum Mol Genet. 2017 Sep 1; 26(17):3442-3450.</div>
</div>
</li>
<li class="two_line" reference_id="R21">
<a href="/pubmed/24374162/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> The molecular and cellular pathology of α₁-antitrypsin deficiency.</a><span class="one_line_source">[Trends Mol Med.  2014]</span><div class="alt-note">
<div class="authors">Gooptu B, Dickens JA, Lomas DA</div>
<div class="citation">Trends Mol Med. 2014 Feb; 20(2):116-27.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P17" name="crb--P17" rid="P17" style="position: absolute; visibility: visible; width: 370px; top: 5754px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-31"><ul>
<li class="two_line" reference_id="R22">
<a href="/pubmed/30239796/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">ProGeM: a framework for the prioritization of candidate causal genes at molecular quantitative trait loci.</a><span class="one_line_source">[Nucleic Acids Res.  2019]</span><div class="alt-note">
<div class="authors">Stacey D, Fauman EB, Ziemek D, Sun BB, Harshfield EL, Wood AM, Butterworth AS, Suhre K, Paul DS</div>
<div class="citation">Nucleic Acids Res. 2019 Jan 10; 47(1):e3.</div>
</div>
</li>
<li class="two_line" reference_id="R23">
<a href="/pubmed/26192919/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Association analyses identify 38 susceptibility loci for inflammatory bowel disease and highlight shared genetic risk across populations.</a><span class="one_line_source">[Nat Genet.  2015]</span><div class="alt-note">
<div class="authors">Liu JZ, van Sommeren S, Huang H, Ng SC, Alberts R, Takahashi A, Ripke S, Lee JC, Jostins L, Shah T, Abedian S, Cheon JH, Cho J, Dayani NE, Franke L, Fuyuno Y, Hart A, Juyal RC, Juyal G, Kim WH, Morris AP, Poustchi H, Newman WG, Midha V, Orchard TR, Vahedi H, Sood A, Sung JY, Malekzadeh R, Westra HJ, Yamazaki K, Yang SK, International Multiple Sclerosis Genetics Consortium., International IBD Genetics Consortium., Barrett JC, Alizadeh BZ, Parkes M, Bk T, Daly MJ, Kubo M, Anderson CA, Weersma RK</div>
<div class="citation">Nat Genet. 2015 Sep; 47(9):979-986.</div>
</div>
</li>
<li class="two_line" reference_id="R24">
<a href="/pubmed/28129359/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">High-Throughput Characterization of Blood Serum Proteomics of IBD Patients with Respect to Aging and Genetic Factors.</a><span class="one_line_source">[PLoS Genet.  2017]</span><div class="alt-note">
<div class="authors">Di Narzo AF, Telesco SE, Brodmerkel C, Argmann C, Peters LA, Li K, Kidd B, Dudley J, Cho J, Schadt EE, Kasarskis A, Dobrin R, Hao K</div>
<div class="citation">PLoS Genet. 2017 Jan; 13(1):e1006565.</div>
</div>
</li>
<li class="two_line" reference_id="R25">
<a href="/pubmed/28658209/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Fine-mapping inflammatory bowel disease loci to single-variant resolution.</a><span class="one_line_source">[Nature.  2017]</span><div class="alt-note">
<div class="authors">Huang H, Fang M, Jostins L, Umićević Mirkov M, Boucher G, Anderson CA, Andersen V, Cleynen I, Cortes A, Crins F, D'Amato M, Deffontaine V, Dmitrieva J, Docampo E, Elansary M, Farh KK, Franke A, Gori AS, Goyette P, Halfvarson J, Haritunians T, Knight J, Lawrance IC, Lees CW, Louis E, Mariman R, Meuwissen T, Mni M, Momozawa Y, Parkes M, Spain SL, Théâtre E, Trynka G, Satsangi J, van Sommeren S, Vermeire S, Xavier RJ, International Inflammatory Bowel Disease Genetics Consortium., Weersma RK, Duerr RH, Mathew CG, Rioux JD, McGovern DPB, Cho JH, Georges M, Daly MJ, Barrett JC</div>
<div class="citation">Nature. 2017 Jul 13; 547(7662):173-178.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P18" name="crb--P18" rid="P18" style="position: absolute; visibility: visible; width: 370px; top: 6874px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-32"><ul>
<li class="two_line" reference_id="R26">
<a href="/pubmed/22808956/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genetically distinct subsets within ANCA-associated vasculitis.</a><span class="one_line_source">[N Engl J Med.  2012]</span><div class="alt-note">
<div class="authors">Lyons PA, Rayner TF, Trivedi S, Holle JU, Watts RA, Jayne DR, Baslund B, Brenchley P, Bruchfeld A, Chaudhry AN, Cohen Tervaert JW, Deloukas P, Feighery C, Gross WL, Guillevin L, Gunnarsson I, Harper L, Hrušková Z, Little MA, Martorana D, Neumann T, Ohlsson S, Padmanabhan S, Pusey CD, Salama AD, Sanders JS, Savage CO, Segelmark M, Stegeman CA, Tesař V, Vaglio A, Wieczorek S, Wilde B, Zwerina J, Rees AJ, Clayton DG, Smith KG</div>
<div class="citation">N Engl J Med. 2012 Jul 19; 367(3):214-23.</div>
</div>
</li>
<li class="two_line" reference_id="R27">
<a href="/pubmed/28029757/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Identification of Functional and Expression Polymorphisms Associated With Risk for Antineutrophil Cytoplasmic Autoantibody-Associated Vasculitis.</a><span class="one_line_source">[Arthritis Rheumatol.  2017]</span><div class="alt-note">
<div class="authors">Merkel PA, Xie G, Monach PA, Ji X, Ciavatta DJ, Byun J, Pinder BD, Zhao A, Zhang J, Tadesse Y, Qian D, Weirauch M, Nair R, Tsoi A, Pagnoux C, Carette S, Chung S, Cuthbertson D, Davis JC Jr, Dellaripa PF, Forbess L, Gewurz-Singer O, Hoffman GS, Khalidi N, Koening C, Langford CA, Mahr AD, McAlear C, Moreland L, Seo EP, Specks U, Spiera RF, Sreih A, St Clair EW, Stone JH, Ytterberg SR, Elder JT, Qu J, Ochi T, Hirano N, Edberg JC, Falk RJ, Amos CI, Siminovitch KA, Vasculitis Clinical Research Consortium.</div>
<div class="citation">Arthritis Rheumatol. 2017 May; 69(5):1054-1066.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P20" name="crb--P20" rid="P20" style="position: absolute; visibility: visible; width: 370px; top: 8300px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-33"><ul><li class="two_line" reference_id="R28">
<a href="/pubmed/29022597/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genetic effects on gene expression across human tissues.</a><span class="one_line_source">[Nature.  2017]</span><div class="alt-note">
<div class="authors">GTEx Consortium., Laboratory, Data Analysis &amp;Coordinating Center (LDACC)—Analysis Working Group., Statistical Methods groups—Analysis Working Group., Enhancing GTEx (eGTEx) groups., NIH Common Fund., NIH/NCI., NIH/NHGRI., NIH/NIMH., NIH/NIDA., Biospecimen Collection Source Site—NDRI., Biospecimen Collection Source Site—RPCI., Biospecimen Core Resource—VARI., Brain Bank Repository—University of Miami Brain Endowment Bank., Leidos Biomedical—Project Management., ELSI Study., Genome Browser Data Integration &amp;Visualization—EBI., Genome Browser Data Integration &amp;Visualization—UCSC Genomics Institute, University of California Santa Cruz., Lead analysts:., Laboratory, Data Analysis &amp;Coordinating Center (LDACC):., NIH program management:., Biospecimen collection:., Pathology:., eQTL manuscript working group:., Battle A, Brown CD, Engelhardt BE, Montgomery SB</div>
<div class="citation">Nature. 2017 Oct 11; 550(7675):204-213.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P21" name="crb--P21" rid="P21" style="position: absolute; visibility: visible; width: 370px; top: 8463px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-34"><ul>
<li class="two_line" reference_id="R11">
<a href="/pubmed/25773750/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Using published data in Mendelian randomization: a blueprint for efficient identification of causal risk factors.</a><span class="one_line_source">[Eur J Epidemiol.  2015]</span><div class="alt-note">
<div class="authors">Burgess S, Scott RA, Timpson NJ, Davey Smith G, Thompson SG, EPIC- InterAct Consortium.</div>
<div class="citation">Eur J Epidemiol. 2015 Jul; 30(7):543-52.</div>
</div>
</li>
<li class="two_line" reference_id="R29">
<a href="/pubmed/26563502/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Prostate cancer screening in men aged 50-69 years (STHLM3): a prospective population-based diagnostic study.</a><span class="one_line_source">[Lancet Oncol.  2015]</span><div class="alt-note">
<div class="authors">Grönberg H, Adolfsson J, Aly M, Nordström T, Wiklund P, Brandberg Y, Thompson J, Wiklund F, Lindberg J, Clements M, Egevad L, Eklund M</div>
<div class="citation">Lancet Oncol. 2015 Dec; 16(16):1667-76.</div>
</div>
</li>
<li class="two_line" reference_id="R30">
<a href="/pubmed/18264097/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Multiple newly identified loci associated with prostate cancer susceptibility.</a><span class="one_line_source">[Nat Genet.  2008]</span><div class="alt-note">
<div class="authors">Eeles RA, Kote-Jarai Z, Giles GG, Olama AA, Guy M, Jugurnauth SK, Mulholland S, Leongamornlert DA, Edwards SM, Morrison J, Field HI, Southey MC, Severi G, Donovan JL, Hamdy FC, Dearnaley DP, Muir KR, Smith C, Bagnato M, Ardern-Jones AT, Hall AL, O'Brien LT, Gehr-Swain BN, Wilkinson RA, Cox A, Lewis S, Brown PM, Jhavar SG, Tymrakiewicz M, Lophatananon A, Bryant SL, UK Genetic Prostate Cancer Study Collaborators., British Association of Urological Surgeons' Section of Oncology., UK ProtecT Study Collaborators., Horwich A, Huddart RA, Khoo VS, Parker CC, Woodhouse CJ, Thompson A, Christmas T, Ogden C, Fisher C, Jamieson C, Cooper CS, English DR, Hopper JL, Neal DE, Easton DF</div>
<div class="citation">Nat Genet. 2008 Mar; 40(3):316-21.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P22" name="crb--P22" rid="P22" style="position: absolute; visibility: visible; width: 370px; top: 8674px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-35"><ul><li class="two_line" reference_id="R31">
<a href="/pubmed/26482879/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Multi-ancestry genome-wide association study of 21,000 cases and 95,000 controls identifies new risk loci for atopic dermatitis.</a><span class="one_line_source">[Nat Genet.  2015]</span><div class="alt-note">
<div class="authors">Paternoster L, Standl M, Waage J, Baurecht H, Hotze M, Strachan DP, Curtin JA, Bønnelykke K, Tian C, Takahashi A, Esparza-Gordillo J, Alves AC, Thyssen JP, den Dekker HT, Ferreira MA, Altmaier E, Sleiman PM, Xiao FL, Gonzalez JR, Marenholz I, Kalb B, Yanes MP, Xu CJ, Carstensen L, Groen-Blokhuis MM, Venturini C, Pennell CE, Barton SJ, Levin AM, Curjuric I, Bustamante M, Kreiner-Møller E, Lockett GA, Bacelis J, Bunyavanich S, Myers RA, Matanovic A, Kumar A, Tung JY, Hirota T, Kubo M, McArdle WL, Henderson AJ, Kemp JP, Zheng J, Smith GD, Rüschendorf F, Bauerfeind A, Lee-Kirsch MA, Arnold A, Homuth G, Schmidt CO, Mangold E, Cichon S, Keil T, Rodríguez E, Peters A, Franke A, Lieb W, Novak N, Fölster-Holst R, Horikoshi M, Pekkanen J, Sebert S, Husemoen LL, Grarup N, de Jongste JC, Rivadeneira F, Hofman A, Jaddoe VW, Pasmans SG, Elbert NJ, Uitterlinden AG, Marks GB, Thompson PJ, Matheson MC, Robertson CF, Australian Asthma Genetics Consortium (AAGC)., Ried JS, Li J, Zuo XB, Zheng XD, Yin XY, Sun LD, McAleer MA, O'Regan GM, Fahy CM, Campbell LE, Macek M, Kurek M, Hu D, Eng C, Postma DS, Feenstra B, Geller F, Hottenga JJ, Middeldorp CM, Hysi P, Bataille V, Spector T, Tiesler CM, Thiering E, Pahukasahasram B, Yang JJ, Imboden M, Huntsman S, Vilor-Tejedor N, Relton CL, Myhre R, Nystad W, Custovic A, Weiss ST, Meyers DA, Söderhäll C, Melén E, Ober C, Raby BA, Simpson A, Jacobsson B, Holloway JW, Bisgaard H, Sunyer J, Hensch NMP, Williams LK, Godfrey KM, Wang CA, Boomsma DI, Melbye M, Koppelman GH, Jarvis D, McLean WI, Irvine AD, Zhang XJ, Hakonarson H, Gieger C, Burchard EG, Martin NG, Duijts L, Linneberg A, Jarvelin MR, Noethen MM, Lau S, Hübner N, Lee YA, Tamari M, Hinds DA, Glass D, Brown SJ, Heinrich J, Evans DM, Weidinger S</div>
<div class="citation">Nat Genet. 2015 Dec; 47(12):1449-1456.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P23" name="crb--P23" rid="P23" style="position: absolute; visibility: visible; width: 370px; top: 9772px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-36"><ul>
<li class="two_line" reference_id="R32">
<a href="/pubmed/22306193/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Effects of an oral MMP-9 and -12 inhibitor, AZD1236, on biomarkers in moderate/severe COPD: a randomised controlled trial.</a><span class="one_line_source">[Pulm Pharmacol Ther.  2012]</span><div class="alt-note">
<div class="authors">Dahl R, Titlestad I, Lindqvist A, Wielders P, Wray H, Wang M, Samuelsson V, Mo J, Holt A</div>
<div class="citation">Pulm Pharmacol Ther. 2012 Apr; 25(2):169-77.</div>
</div>
</li>
<li class="two_line" reference_id="R33">
<a href="/pubmed/27327800/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Development and Validation of a Protein-Based Risk Score for Cardiovascular Outcomes Among Patients With Stable Coronary Heart Disease.</a><span class="one_line_source">[JAMA.  2016]</span><div class="alt-note">
<div class="authors">Ganz P, Heidecker B, Hveem K, Jonasson C, Kato S, Segal MR, Sterling DG, Williams SA</div>
<div class="citation">JAMA. 2016 Jun 21; 315(23):2532-41.</div>
</div>
</li>
<li class="two_line" reference_id="R34">
<a href="/pubmed/25078452/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A novel MMP12 locus is associated with large artery atherosclerotic stroke using a genome-wide age-at-onset informed approach.</a><span class="one_line_source">[PLoS Genet.  2014]</span><div class="alt-note">
<div class="authors">Traylor M, Mäkelä KM, Kilarski LL, Holliday EG, Devan WJ, Nalls MA, Wiggins KL, Zhao W, Cheng YC, Achterberg S, Malik R, Sudlow C, Bevan S, Raitoharju E, METASTROKE, International Stroke Genetics Consortium, Wellcome Trust Case Consortium 2 (WTCCC2)., Oksala N, Thijs V, Lemmens R, Lindgren A, Slowik A, Maguire JM, Walters M, Algra A, Sharma P, Attia JR, Boncoraglio GB, Rothwell PM, de Bakker PI, Bis JC, Saleheen D, Kittner SJ, Mitchell BD, Rosand J, Meschia JF, Levi C, Dichgans M, Lehtimäki T, Lewis CM, Markus HS</div>
<div class="citation">PLoS Genet. 2014 Jul; 10(7):e1004469.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P24" name="crb--P24" rid="P24" style="position: absolute; visibility: visible; width: 370px; top: 10003px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-37"><ul><li class="two_line" reference_id="R35">
<a href="/pubmed/26121088/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The support of human genetic evidence for approved drug indications.</a><span class="one_line_source">[Nat Genet.  2015]</span><div class="alt-note">
<div class="authors">Nelson MR, Tipney H, Painter JL, Shen J, Nicoletti P, Shen Y, Floratos A, Sham PC, Li MJ, Wang J, Cardon LR, Whittaker JC, Sanseau P</div>
<div class="citation">Nat Genet. 2015 Aug; 47(8):856-60.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P25" name="crb--P25" rid="P25" style="position: absolute; visibility: visible; width: 370px; top: 10147px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-38"><ul>
<li class="two_line" reference_id="R36">
<a href="/pubmed/20436471/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide association study identifies variants at CSF1, OPTN and TNFRSF11A as genetic risk factors for Paget's disease of bone.</a><span class="one_line_source">[Nat Genet.  2010]</span><div class="alt-note">
<div class="authors">Albagha OM, Visconti MR, Alonso N, Langston AL, Cundy T, Dargie R, Dunlop MG, Fraser WD, Hooper MJ, Isaia G, Nicholson GC, del Pino Montes J, Gonzalez-Sarmiento R, di Stefano M, Tenesa A, Walsh JP, Ralston SH</div>
<div class="citation">Nat Genet. 2010 Jun; 42(6):520-4.</div>
</div>
</li>
<li class="two_line" reference_id="R37">
<a href="/pubmed/22586699/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Paget's disease of the bone after treatment with Denosumab: a case report.</a><span class="one_line_source">[Bone.  2012]</span><div class="alt-note">
<div class="authors">Schwarz P, Rasmussen AQ, Kvist TM, Andersen UB, Jørgensen NR</div>
<div class="citation">Bone. 2012 May; 50(5):1023-5.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P26" name="crb--P26" rid="P26" style="position: absolute; visibility: visible; width: 370px; top: 10357px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-39"><ul><li class="two_line" reference_id="R10">
<a href="/pubmed/27863252/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The Allelic Landscape of Human Blood Cell Trait Variation and Links to Common Complex Disease.</a><span class="one_line_source">[Cell.  2016]</span><div class="alt-note">
<div class="authors">Astle WJ, Elding H, Jiang T, Allen D, Ruklisa D, Mann AL, Mead D, Bouman H, Riveros-Mckay F, Kostadima MA, Lambourne JJ, Sivapalaratnam S, Downes K, Kundu K, Bomba L, Berentsen K, Bradley JR, Daugherty LC, Delaneau O, Freson K, Garner SF, Grassi L, Guerrero J, Haimel M, Janssen-Megens EM, Kaan A, Kamat M, Kim B, Mandoli A, Marchini J, Martens JHA, Meacham S, Megy K, O'Connell J, Petersen R, Sharifi N, Sheard SM, Staley JR, Tuna S, van der Ent M, Walter K, Wang SY, Wheeler E, Wilder SP, Iotchkova V, Moore C, Sambrook J, Stunnenberg HG, Di Angelantonio E, Kaptoge S, Kuijpers TW, Carrillo-de-Santa-Pau E, Juan D, Rico D, Valencia A, Chen L, Ge B, Vasquez L, Kwan T, Garrido-Martín D, Watt S, Yang Y, Guigo R, Beck S, Paul DS, Pastinen T, Bujold D, Bourque G, Frontini M, Danesh J, Roberts DJ, Ouwehand WH, Butterworth AS, Soranzo N</div>
<div class="citation">Cell. 2016 Nov 17; 167(5):1415-1429.e19.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P29" name="crb--P29" rid="P29" style="position: absolute; visibility: visible; width: 370px; top: 11017px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-40"><ul><li class="two_line" reference_id="R9">
<a href="/pubmed/28941948/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Efficiency and safety of varying the frequency of whole blood donation (INTERVAL): a randomised trial of 45 000 donors.</a><span class="one_line_source">[Lancet.  2017]</span><div class="alt-note">
<div class="authors">Di Angelantonio E, Thompson SG, Kaptoge S, Moore C, Walker M, Armitage J, Ouwehand WH, Roberts DJ, Danesh J, INTERVAL Trial Group.</div>
<div class="citation">Lancet. 2017 Nov 25; 390(10110):2360-2371.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P30" name="crb--P30" rid="P30" style="position: absolute; visibility: visible; width: 370px; top: 11355px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-41"><ul><li class="two_line" reference_id="R38">
<a href="/pubmed/25230735/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The INTERVAL trial to determine whether intervals between blood donations can be safely and acceptably decreased to optimise blood supply: study protocol for a randomised controlled trial.</a><span class="one_line_source">[Trials.  2014]</span><div class="alt-note">
<div class="authors">Moore C, Sambrook J, Walker M, Tolkien Z, Kaptoge S, Allen D, Mehenny S, Mant J, Di Angelantonio E, Thompson SG, Ouwehand W, Roberts DJ, Danesh J</div>
<div class="citation">Trials. 2014 Sep 17; 15():363.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P31" name="crb--P31" rid="P31" style="position: absolute; visibility: visible; width: 370px; top: 11518px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-42"><ul>
<li class="two_line" reference_id="R8">
<a href="/pubmed/25291143/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Nucleic Acid Ligands With Protein-like Side Chains: Modified Aptamers and Their Use as Diagnostic and Therapeutic Agents.</a><span class="one_line_source">[Mol Ther Nucleic Acids.  2014]</span><div class="alt-note">
<div class="authors">Rohloff JC, Gelinas AD, Jarvis TC, Ochsner UA, Schneider DJ, Gold L, Janjic N</div>
<div class="citation">Mol Ther Nucleic Acids. 2014 Oct 7; 3():e201.</div>
</div>
</li>
<li class="two_line" reference_id="R39">
<a href="/pubmed/21165148/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Aptamer-based multiplexed proteomic technology for biomarker discovery.</a><span class="one_line_source">[PLoS One.  2010]</span><div class="alt-note">
<div class="authors">Gold L, Ayers D, Bertino J, Bock C, Bock A, Brody EN, Carter J, Dalby AB, Eaton BE, Fitzwater T, Flather D, Forbes A, Foreman T, Fowler C, Gawande B, Goss M, Gunn M, Gupta S, Halladay D, Heil J, Heilig J, Hicke B, Husar G, Janjic N, Jarvis T, Jennings S, Katilius E, Keeney TR, Kim N, Koch TH, Kraemer S, Kroiss L, Le N, Levine D, Lindsey W, Lollo B, Mayfield W, Mehan M, Mehler R, Nelson SK, Nelson M, Nieuwlandt D, Nikrad M, Ochsner U, Ostroff RM, Otis M, Parker T, Pietrasiewicz S, Resnicow DI, Rohloff J, Sanders G, Sattin S, Schneider D, Singer B, Stanton M, Sterkel A, Stewart A, Stratford S, Vaught JD, Vrkljan M, Walker JJ, Watrobka M, Waugh S, Weiss A, Wilcox SK, Wolfson A, Wolk SK, Zhang C, Zichi D</div>
<div class="citation">PLoS One. 2010 Dec 7; 5(12):e15004.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P32" name="crb--P32" rid="P32" style="position: absolute; visibility: visible; width: 370px; top: 11727px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-43"><ul>
<li class="two_line" reference_id="R39">
<a href="/pubmed/21165148/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Aptamer-based multiplexed proteomic technology for biomarker discovery.</a><span class="one_line_source">[PLoS One.  2010]</span><div class="alt-note">
<div class="authors">Gold L, Ayers D, Bertino J, Bock C, Bock A, Brody EN, Carter J, Dalby AB, Eaton BE, Fitzwater T, Flather D, Forbes A, Foreman T, Fowler C, Gawande B, Goss M, Gunn M, Gupta S, Halladay D, Heil J, Heilig J, Hicke B, Husar G, Janjic N, Jarvis T, Jennings S, Katilius E, Keeney TR, Kim N, Koch TH, Kraemer S, Kroiss L, Le N, Levine D, Lindsey W, Lollo B, Mayfield W, Mehan M, Mehler R, Nelson SK, Nelson M, Nieuwlandt D, Nikrad M, Ochsner U, Ostroff RM, Otis M, Parker T, Pietrasiewicz S, Resnicow DI, Rohloff J, Sanders G, Sattin S, Schneider D, Singer B, Stanton M, Sterkel A, Stewart A, Stratford S, Vaught JD, Vrkljan M, Walker JJ, Watrobka M, Waugh S, Weiss A, Wilcox SK, Wolfson A, Wolk SK, Zhang C, Zichi D</div>
<div class="citation">PLoS One. 2010 Dec 7; 5(12):e15004.</div>
</div>
</li>
<li class="two_line" reference_id="R40">
<a href="/pubmed/24768341/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Alzheimer's disease biomarker discovery using SOMAscan multiplexed protein technology.</a><span class="one_line_source">[Alzheimers Dement.  2014]</span><div class="alt-note">
<div class="authors">Sattlecker M, Kiddle SJ, Newhouse S, Proitsi P, Nelson S, Williams S, Johnston C, Killick R, Simmons A, Westman E, Hodges A, Soininen H, Kłoszewska I, Mecocci P, Tsolaki M, Vellas B, Lovestone S, AddNeuroMed Consortium., Dobson RJ</div>
<div class="citation">Alzheimers Dement. 2014 Nov; 10(6):724-34.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P35" name="crb--P35" rid="P35" style="position: absolute; visibility: visible; width: 370px; top: 12311px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-44"><ul>
<li class="two_line" reference_id="R41">
<a href="/pubmed/20562413/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Deriving the consequences of genomic variants with the Ensembl API and SNP Effect Predictor.</a><span class="one_line_source">[Bioinformatics.  2010]</span><div class="alt-note">
<div class="authors">McLaren W, Pritchard B, Rios D, Chen Y, Flicek P, Cunningham F</div>
<div class="citation">Bioinformatics. 2010 Aug 15; 26(16):2069-70.</div>
</div>
</li>
<li class="two_line" reference_id="R42">
<a href="/pubmed/25348405/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">UniProt: a hub for protein information.</a><span class="one_line_source">[Nucleic Acids Res.  2015]</span><div class="alt-note">
<div class="authors">UniProt Consortium.</div>
<div class="citation">Nucleic Acids Res. 2015 Jan; 43(Database issue):D204-12.</div>
</div>
</li>
<li class="two_line" reference_id="R43">
<a href="/pubmed/10802651/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.</a><span class="one_line_source">[Nat Genet.  2000]</span><div class="alt-note">
<div class="authors">Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, Davis AP, Dolinski K, Dwight SS, Eppig JT, Harris MA, Hill DP, Issel-Tarver L, Kasarskis A, Lewis S, Matese JC, Richardson JE, Ringwald M, Rubin GM, Sherlock G</div>
<div class="citation">Nat Genet. 2000 May; 25(1):25-9.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P36" name="crb--P36" rid="P36" style="position: absolute; visibility: visible; width: 370px; top: 12541px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-45"><ul>
<li class="two_line" reference_id="R44">
<a href="/pubmed/25123647/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Circulating Proteomic Signatures of Chronological Age.</a><span class="one_line_source">[J Gerontol A Biol Sci Med Sci.  2015]</span><div class="alt-note">
<div class="authors">Menni C, Kiddle SJ, Mangino M, Viñuela A, Psatha M, Steves C, Sattlecker M, Buil A, Newhouse S, Nelson S, Williams S, Voyle N, Soininen H, Kloszewska I, Mecocci P, Tsolaki M, Vellas B, Lovestone S, Spector TD, Dobson R, Valdes AM</div>
<div class="citation">J Gerontol A Biol Sci Med Sci. 2015 Jul; 70(7):809-16.</div>
</div>
</li>
<li class="two_line" reference_id="R45">
<a href="/pubmed/27444932/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Aptamer-Based Proteomic Profiling Reveals Novel Candidate Biomarkers and Pathways in Cardiovascular Disease.</a><span class="one_line_source">[Circulation.  2016]</span><div class="alt-note">
<div class="authors">Ngo D, Sinha S, Shen D, Kuhn EW, Keyes MJ, Shi X, Benson MD, O'Sullivan JF, Keshishian H, Farrell LA, Fifer MA, Vasan RS, Sabatine MS, Larson MG, Carr SA, Wang TJ, Gerszten RE</div>
<div class="citation">Circulation. 2016 Jul 26; 134(4):270-85.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P37" name="crb--P37" rid="P37" style="position: absolute; visibility: visible; width: 370px; top: 12725px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-46"><ul>
<li class="two_line" reference_id="R10">
<a href="/pubmed/27863252/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The Allelic Landscape of Human Blood Cell Trait Variation and Links to Common Complex Disease.</a><span class="one_line_source">[Cell.  2016]</span><div class="alt-note">
<div class="authors">Astle WJ, Elding H, Jiang T, Allen D, Ruklisa D, Mann AL, Mead D, Bouman H, Riveros-Mckay F, Kostadima MA, Lambourne JJ, Sivapalaratnam S, Downes K, Kundu K, Bomba L, Berentsen K, Bradley JR, Daugherty LC, Delaneau O, Freson K, Garner SF, Grassi L, Guerrero J, Haimel M, Janssen-Megens EM, Kaan A, Kamat M, Kim B, Mandoli A, Marchini J, Martens JHA, Meacham S, Megy K, O'Connell J, Petersen R, Sharifi N, Sheard SM, Staley JR, Tuna S, van der Ent M, Walter K, Wang SY, Wheeler E, Wilder SP, Iotchkova V, Moore C, Sambrook J, Stunnenberg HG, Di Angelantonio E, Kaptoge S, Kuijpers TW, Carrillo-de-Santa-Pau E, Juan D, Rico D, Valencia A, Chen L, Ge B, Vasquez L, Kwan T, Garrido-Martín D, Watt S, Yang Y, Guigo R, Beck S, Paul DS, Pastinen T, Bujold D, Bourque G, Frontini M, Danesh J, Roberts DJ, Ouwehand WH, Butterworth AS, Soranzo N</div>
<div class="citation">Cell. 2016 Nov 17; 167(5):1415-1429.e19.</div>
</div>
</li>
<li class="two_line" reference_id="R46">
<a href="/pubmed/25722852/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Second-generation PLINK: rising to the challenge of larger and richer datasets.</a><span class="one_line_source">[Gigascience.  2015]</span><div class="alt-note">
<div class="authors">Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ</div>
<div class="citation">Gigascience. 2015; 4():7.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P40" name="crb--P40" rid="P40" style="position: absolute; visibility: visible; width: 370px; top: 13308px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-47"><ul><li class="two_line" reference_id="R47">
<a href="/pubmed/17572673/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A new multipoint method for genome-wide association studies by imputation of genotypes.</a><span class="one_line_source">[Nat Genet.  2007]</span><div class="alt-note">
<div class="authors">Marchini J, Howie B, Myers S, McVean G, Donnelly P</div>
<div class="citation">Nat Genet. 2007 Jul; 39(7):906-13.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P41" name="crb--P41" rid="P41" style="position: absolute; visibility: visible; width: 370px; top: 13514px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-48"><ul><li class="two_line" reference_id="R48">
<a href="/pubmed/20616382/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">METAL: fast and efficient meta-analysis of genomewide association scans.</a><span class="one_line_source">[Bioinformatics.  2010]</span><div class="alt-note">
<div class="authors">Willer CJ, Li Y, Abecasis GR</div>
<div class="citation">Bioinformatics. 2010 Sep 1; 26(17):2190-1.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P43" name="crb--P43" rid="P43" style="position: absolute; visibility: visible; width: 370px; top: 13949px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-49"><ul><li class="two_line" reference_id="R49">
<a href="/pubmed/22426310/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits.</a><span class="one_line_source">[Nat Genet.  2012]</span><div class="alt-note">
<div class="authors">Yang J, Ferreira T, Morris AP, Medland SE, Genetic Investigation of ANthropometric Traits (GIANT) Consortium., DIAbetes Genetics Replication And Meta-analysis (DIAGRAM) Consortium., Madden PA, Heath AC, Martin NG, Montgomery GW, Weedon MN, Loos RJ, Frayling TM, McCarthy MI, Hirschhorn JN, Goddard ME, Visscher PM</div>
<div class="citation">Nat Genet. 2012 Mar 18; 44(4):369-75, S1-3.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P44" name="crb--P44" rid="P44" style="position: absolute; visibility: visible; width: 370px; top: 14288px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-50"><ul>
<li class="two_line" reference_id="R50">
<a href="/pubmed/24316577/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The NHGRI GWAS Catalog, a curated resource of SNP-trait associations.</a><span class="one_line_source">[Nucleic Acids Res.  2014]</span><div class="alt-note">
<div class="authors">Welter D, MacArthur J, Morales J, Burdett T, Hall P, Junkins H, Klemm A, Flicek P, Manolio T, Hindorff L, Parkinson H</div>
<div class="citation">Nucleic Acids Res. 2014 Jan; 42(Database issue):D1001-6.</div>
</div>
</li>
<li class="two_line" reference_id="R51">
<a href="/pubmed/20200009/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Modeling sample variables with an Experimental Factor Ontology.</a><span class="one_line_source">[Bioinformatics.  2010]</span><div class="alt-note">
<div class="authors">Malone J, Holloway E, Adamusiak T, Kapushesky M, Zheng J, Kolesnikov N, Zhukova A, Brazma A, Parkinson H</div>
<div class="citation">Bioinformatics. 2010 Apr 15; 26(8):1112-8.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P46" name="crb--P46" rid="P46" style="position: absolute; visibility: visible; width: 370px; top: 14815px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-51"><ul><li class="two_line" reference_id="R13">
<a href="/pubmed/21646338/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Homogeneous antibody-based proximity extension assays provide sensitive and specific detection of low-abundant proteins in human blood.</a><span class="one_line_source">[Nucleic Acids Res.  2011]</span><div class="alt-note">
<div class="authors">Lundberg M, Eriksson A, Tran B, Assarsson E, Fredriksson S</div>
<div class="citation">Nucleic Acids Res. 2011 Aug; 39(15):e102.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P48" name="crb--P48" rid="P48" style="position: absolute; visibility: visible; width: 370px; top: 15253px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-52"><ul>
<li class="two_line" reference_id="R22">
<a href="/pubmed/30239796/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">ProGeM: a framework for the prioritization of candidate causal genes at molecular quantitative trait loci.</a><span class="one_line_source">[Nucleic Acids Res.  2019]</span><div class="alt-note">
<div class="authors">Stacey D, Fauman EB, Ziemek D, Sun BB, Harshfield EL, Wood AM, Butterworth AS, Suhre K, Paul DS</div>
<div class="citation">Nucleic Acids Res. 2019 Jan 10; 47(1):e3.</div>
</div>
</li>
<li class="two_line" reference_id="R28">
<a href="/pubmed/29022597/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genetic effects on gene expression across human tissues.</a><span class="one_line_source">[Nature.  2017]</span><div class="alt-note">
<div class="authors">GTEx Consortium., Laboratory, Data Analysis &amp;Coordinating Center (LDACC)—Analysis Working Group., Statistical Methods groups—Analysis Working Group., Enhancing GTEx (eGTEx) groups., NIH Common Fund., NIH/NCI., NIH/NHGRI., NIH/NIMH., NIH/NIDA., Biospecimen Collection Source Site—NDRI., Biospecimen Collection Source Site—RPCI., Biospecimen Core Resource—VARI., Brain Bank Repository—University of Miami Brain Endowment Bank., Leidos Biomedical—Project Management., ELSI Study., Genome Browser Data Integration &amp;Visualization—EBI., Genome Browser Data Integration &amp;Visualization—UCSC Genomics Institute, University of California Santa Cruz., Lead analysts:., Laboratory, Data Analysis &amp;Coordinating Center (LDACC):., NIH program management:., Biospecimen collection:., Pathology:., eQTL manuscript working group:., Battle A, Brown CD, Engelhardt BE, Montgomery SB</div>
<div class="citation">Nature. 2017 Oct 11; 550(7675):204-213.</div>
</div>
</li>
<li class="two_line" reference_id="R52">
<a href="/pubmed/27863249/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Lineage-Specific Genome Architecture Links Enhancers and Non-coding Disease Variants to Target Gene Promoters.</a><span class="one_line_source">[Cell.  2016]</span><div class="alt-note">
<div class="authors">Javierre BM, Burren OS, Wilder SP, Kreuzhuber R, Hill SM, Sewitz S, Cairns J, Wingett SW, Várnai C, Thiecke MJ, Burden F, Farrow S, Cutler AJ, Rehnström K, Downes K, Grassi L, Kostadima M, Freire-Pritchett P, Wang F, BLUEPRINT Consortium., Stunnenberg HG, Todd JA, Zerbino DR, Stegle O, Ouwehand WH, Frontini M, Wallace C, Spivakov M, Fraser P</div>
<div class="citation">Cell. 2016 Nov 17; 167(5):1369-1384.e19.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P49" name="crb--P49" rid="P49" style="position: absolute; visibility: visible; width: 370px; top: 15774px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-53"><ul>
<li class="two_line" reference_id="R53">
<a href="/pubmed/23950696/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Software for computing and annotating genomic ranges.</a><span class="one_line_source">[PLoS Comput Biol.  2013]</span><div class="alt-note">
<div class="authors">Lawrence M, Huber W, Pagès H, Aboyoun P, Carlson M, Gentleman R, Morgan MT, Carey VJ</div>
<div class="citation">PLoS Comput Biol. 2013; 9(8):e1003118.</div>
</div>
</li>
<li class="two_line" reference_id="R54">
<a href="/pubmed/25428349/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">OMIM.org: Online Mendelian Inheritance in Man (OMIM®), an online catalog of human genes and genetic disorders.</a><span class="one_line_source">[Nucleic Acids Res.  2015]</span><div class="alt-note">
<div class="authors">Amberger JS, Bocchini CA, Schiettecatte F, Scott AF, Hamosh A</div>
<div class="citation">Nucleic Acids Res. 2015 Jan; 43(Database issue):D789-98.</div>
</div>
</li>
<li class="two_line" reference_id="R55">
<a href="/pubmed/26476454/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">KEGG as a reference resource for gene and protein annotation.</a><span class="one_line_source">[Nucleic Acids Res.  2016]</span><div class="alt-note">
<div class="authors">Kanehisa M, Sato Y, Kawashima M, Furumichi M, Tanabe M</div>
<div class="citation">Nucleic Acids Res. 2016 Jan 4; 44(D1):D457-62.</div>
</div>
</li>
<li class="two_line" reference_id="R56">
<a href="/pubmed/25352553/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">STRING v10: protein-protein interaction networks, integrated over the tree of life.</a><span class="one_line_source">[Nucleic Acids Res.  2015]</span><div class="alt-note">
<div class="authors">Szklarczyk D, Franceschini A, Wyder S, Forslund K, Heller D, Huerta-Cepas J, Simonovic M, Roth A, Santos A, Tsafou KP, Kuhn M, Bork P, Jensen LJ, von Mering C</div>
<div class="citation">Nucleic Acids Res. 2015 Jan; 43(Database issue):D447-52.</div>
</div>
</li>
<li class="two_line" reference_id="R57">
<a href="/pubmed/23023984/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">InterMine: a flexible data warehouse system for the integration and analysis of heterogeneous biological data.</a><span class="one_line_source">[Bioinformatics.  2012]</span><div class="alt-note">
<div class="authors">Smith RN, Aleksic J, Butano D, Carr A, Contrino S, Hu F, Lyne M, Lyne R, Kalderimis A, Rutherford K, Stepan R, Sullivan J, Wakeling M, Watkins X, Micklem G</div>
<div class="citation">Bioinformatics. 2012 Dec 1; 28(23):3163-5.</div>
</div>
</li>
<li class="two_line" reference_id="R58">
<a href="/pubmed/23203871/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">STRING v9.1: protein-protein interaction networks, with increased coverage and integration.</a><span class="one_line_source">[Nucleic Acids Res.  2013]</span><div class="alt-note">
<div class="authors">Franceschini A, Szklarczyk D, Frankild S, Kuhn M, Simonovic M, Roth A, Lin J, Minguez P, Bork P, von Mering C, Jensen LJ</div>
<div class="citation">Nucleic Acids Res. 2013 Jan; 41(Database issue):D808-15.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P52" name="crb--P52" rid="P52" style="position: absolute; visibility: visible; width: 370px; top: 16750px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-54"><ul><li class="two_line" reference_id="R59">
<a href="/pubmed/27535533/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Analysis of protein-coding genetic variation in 60,706 humans.</a><span class="one_line_source">[Nature.  2016]</span><div class="alt-note">
<div class="authors">Lek M, Karczewski KJ, Minikel EV, Samocha KE, Banks E, Fennell T, O'Donnell-Luria AH, Ware JS, Hill AJ, Cummings BB, Tukiainen T, Birnbaum DP, Kosmicki JA, Duncan LE, Estrada K, Zhao F, Zou J, Pierce-Hoffman E, Berghout J, Cooper DN, Deflaux N, DePristo M, Do R, Flannick J, Fromer M, Gauthier L, Goldstein J, Gupta N, Howrigan D, Kiezun A, Kurki MI, Moonshine AL, Natarajan P, Orozco L, Peloso GM, Poplin R, Rivas MA, Ruano-Rubio V, Rose SA, Ruderfer DM, Shakir K, Stenson PD, Stevens C, Thomas BP, Tiao G, Tusie-Luna MT, Weisburd B, Won HH, Yu D, Altshuler DM, Ardissino D, Boehnke M, Danesh J, Donnelly S, Elosua R, Florez JC, Gabriel SB, Getz G, Glatt SJ, Hultman CM, Kathiresan S, Laakso M, McCarroll S, McCarthy MI, McGovern D, McPherson R, Neale BM, Palotie A, Purcell SM, Saleheen D, Scharf JM, Sklar P, Sullivan PF, Tuomilehto J, Tsuang MT, Watkins HC, Wilson JG, Daly MJ, MacArthur DG, Exome Aggregation Consortium.</div>
<div class="citation">Nature. 2016 Aug 18; 536(7616):285-91.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P55" name="crb--P55" rid="P55" style="position: absolute; visibility: visible; width: 370px; top: 17441px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-55"><ul><li class="two_line" reference_id="R61">
<a href="/pubmed/27318201/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PhenoScanner: a database of human genotype-phenotype associations.</a><span class="one_line_source">[Bioinformatics.  2016]</span><div class="alt-note">
<div class="authors">Staley JR, Blackshaw J, Kamat MA, Ellis S, Surendran P, Sun BB, Paul DS, Freitag D, Burgess S, Danesh J, Young R, Butterworth AS</div>
<div class="citation">Bioinformatics. 2016 Oct 15; 32(20):3207-3209.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P59" name="crb--P59" rid="P59" style="position: absolute; visibility: visible; width: 370px; top: 18209px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-56"><ul>
<li class="two_line" reference_id="R62">
<a href="/pubmed/24830394/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Bayesian test for colocalisation between pairs of genetic association studies using summary statistics.</a><span class="one_line_source">[PLoS Genet.  2014]</span><div class="alt-note">
<div class="authors">Giambartolomei C, Vukcevic D, Schadt EE, Franke L, Hingorani AD, Wallace C, Plagnol V</div>
<div class="citation">PLoS Genet. 2014 May; 10(5):e1004383.</div>
</div>
</li>
<li class="two_line" reference_id="R28">
<a href="/pubmed/29022597/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genetic effects on gene expression across human tissues.</a><span class="one_line_source">[Nature.  2017]</span><div class="alt-note">
<div class="authors">GTEx Consortium., Laboratory, Data Analysis &amp;Coordinating Center (LDACC)—Analysis Working Group., Statistical Methods groups—Analysis Working Group., Enhancing GTEx (eGTEx) groups., NIH Common Fund., NIH/NCI., NIH/NHGRI., NIH/NIMH., NIH/NIDA., Biospecimen Collection Source Site—NDRI., Biospecimen Collection Source Site—RPCI., Biospecimen Core Resource—VARI., Brain Bank Repository—University of Miami Brain Endowment Bank., Leidos Biomedical—Project Management., ELSI Study., Genome Browser Data Integration &amp;Visualization—EBI., Genome Browser Data Integration &amp;Visualization—UCSC Genomics Institute, University of California Santa Cruz., Lead analysts:., Laboratory, Data Analysis &amp;Coordinating Center (LDACC):., NIH program management:., Biospecimen collection:., Pathology:., eQTL manuscript working group:., Battle A, Brown CD, Engelhardt BE, Montgomery SB</div>
<div class="citation">Nature. 2017 Oct 11; 550(7675):204-213.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P60" name="crb--P60" rid="P60" style="position: absolute; visibility: visible; width: 370px; top: 18482px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-57"><ul>
<li class="two_line" reference_id="R11" style="display: none;">
<a href="/pubmed/25773750/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Using published data in Mendelian randomization: a blueprint for efficient identification of causal risk factors.</a><span class="one_line_source">[Eur J Epidemiol.  2015]</span><div class="alt-note">
<div class="authors">Burgess S, Scott RA, Timpson NJ, Davey Smith G, Thompson SG, EPIC- InterAct Consortium.</div>
<div class="citation">Eur J Epidemiol. 2015 Jul; 30(7):543-52.</div>
</div>
</li>
<li class="two_line" reference_id="R63" style="display: none;">
<a href="/pubmed/16325682/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Nature's randomised trials.</a><span class="one_line_source">[Lancet.  2005]</span><div class="alt-note">
<div class="authors">Hingorani A, Humphries S</div>
<div class="citation">Lancet. 2005 Dec 3; 366(9501):1906-8.</div>
</div>
</li>
</ul><a href="" class="seemore">See links ...</a></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P61" name="crb--P61" rid="P61" style="position: absolute; visibility: visible; width: 370px; top: 18578px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-58"><ul><li class="two_line" reference_id="R31">
<a href="/pubmed/26482879/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Multi-ancestry genome-wide association study of 21,000 cases and 95,000 controls identifies new risk loci for atopic dermatitis.</a><span class="one_line_source">[Nat Genet.  2015]</span><div class="alt-note">
<div class="authors">Paternoster L, Standl M, Waage J, Baurecht H, Hotze M, Strachan DP, Curtin JA, Bønnelykke K, Tian C, Takahashi A, Esparza-Gordillo J, Alves AC, Thyssen JP, den Dekker HT, Ferreira MA, Altmaier E, Sleiman PM, Xiao FL, Gonzalez JR, Marenholz I, Kalb B, Yanes MP, Xu CJ, Carstensen L, Groen-Blokhuis MM, Venturini C, Pennell CE, Barton SJ, Levin AM, Curjuric I, Bustamante M, Kreiner-Møller E, Lockett GA, Bacelis J, Bunyavanich S, Myers RA, Matanovic A, Kumar A, Tung JY, Hirota T, Kubo M, McArdle WL, Henderson AJ, Kemp JP, Zheng J, Smith GD, Rüschendorf F, Bauerfeind A, Lee-Kirsch MA, Arnold A, Homuth G, Schmidt CO, Mangold E, Cichon S, Keil T, Rodríguez E, Peters A, Franke A, Lieb W, Novak N, Fölster-Holst R, Horikoshi M, Pekkanen J, Sebert S, Husemoen LL, Grarup N, de Jongste JC, Rivadeneira F, Hofman A, Jaddoe VW, Pasmans SG, Elbert NJ, Uitterlinden AG, Marks GB, Thompson PJ, Matheson MC, Robertson CF, Australian Asthma Genetics Consortium (AAGC)., Ried JS, Li J, Zuo XB, Zheng XD, Yin XY, Sun LD, McAleer MA, O'Regan GM, Fahy CM, Campbell LE, Macek M, Kurek M, Hu D, Eng C, Postma DS, Feenstra B, Geller F, Hottenga JJ, Middeldorp CM, Hysi P, Bataille V, Spector T, Tiesler CM, Thiering E, Pahukasahasram B, Yang JJ, Imboden M, Huntsman S, Vilor-Tejedor N, Relton CL, Myhre R, Nystad W, Custovic A, Weiss ST, Meyers DA, Söderhäll C, Melén E, Ober C, Raby BA, Simpson A, Jacobsson B, Holloway JW, Bisgaard H, Sunyer J, Hensch NMP, Williams LK, Godfrey KM, Wang CA, Boomsma DI, Melbye M, Koppelman GH, Jarvis D, McLean WI, Irvine AD, Zhang XJ, Hakonarson H, Gieger C, Burchard EG, Martin NG, Duijts L, Linneberg A, Jarvelin MR, Noethen MM, Lau S, Hübner N, Lee YA, Tamari M, Hinds DA, Glass D, Brown SJ, Heinrich J, Evans DM, Weidinger S</div>
<div class="citation">Nat Genet. 2015 Dec; 47(12):1449-1456.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P62" name="crb--P62" rid="P62" style="position: absolute; visibility: visible; width: 370px; top: 18829px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-59"><ul><li class="two_line" reference_id="R64">
<a href="/pubmed/26343387/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A comprehensive 1,000 Genomes-based genome-wide association meta-analysis of coronary artery disease.</a><span class="one_line_source">[Nat Genet.  2015]</span><div class="alt-note">
<div class="authors">Nikpay M, Goel A, Won HH, Hall LM, Willenborg C, Kanoni S, Saleheen D, Kyriakou T, Nelson CP, Hopewell JC, Webb TR, Zeng L, Dehghan A, Alver M, Armasu SM, Auro K, Bjonnes A, Chasman DI, Chen S, Ford I, Franceschini N, Gieger C, Grace C, Gustafsson S, Huang J, Hwang SJ, Kim YK, Kleber ME, Lau KW, Lu X, Lu Y, Lyytikäinen LP, Mihailov E, Morrison AC, Pervjakova N, Qu L, Rose LM, Salfati E, Saxena R, Scholz M, Smith AV, Tikkanen E, Uitterlinden A, Yang X, Zhang W, Zhao W, de Andrade M, de Vries PS, van Zuydam NR, Anand SS, Bertram L, Beutner F, Dedoussis G, Frossard P, Gauguier D, Goodall AH, Gottesman O, Haber M, Han BG, Huang J, Jalilzadeh S, Kessler T, König IR, Lannfelt L, Lieb W, Lind L, Lindgren CM, Lokki ML, Magnusson PK, Mallick NH, Mehra N, Meitinger T, Memon FU, Morris AP, Nieminen MS, Pedersen NL, Peters A, Rallidis LS, Rasheed A, Samuel M, Shah SH, Sinisalo J, Stirrups KE, Trompet S, Wang L, Zaman KS, Ardissino D, Boerwinkle E, Borecki IB, Bottinger EP, Buring JE, Chambers JC, Collins R, Cupples LA, Danesh J, Demuth I, Elosua R, Epstein SE, Esko T, Feitosa MF, Franco OH, Franzosi MG, Granger CB, Gu D, Gudnason V, Hall AS, Hamsten A, Harris TB, Hazen SL, Hengstenberg C, Hofman A, Ingelsson E, Iribarren C, Jukema JW, Karhunen PJ, Kim BJ, Kooner JS, Kullo IJ, Lehtimäki T, Loos RJF, Melander O, Metspalu A, März W, Palmer CN, Perola M, Quertermous T, Rader DJ, Ridker PM, Ripatti S, Roberts R, Salomaa V, Sanghera DK, Schwartz SM, Seedorf U, Stewart AF, Stott DJ, Thiery J, Zalloua PA, O'Donnell CJ, Reilly MP, Assimes TL, Thompson JR, Erdmann J, Clarke R, Watkins H, Kathiresan S, McPherson R, Deloukas P, Schunkert H, Samani NJ, Farrall M</div>
<div class="citation">Nat Genet. 2015 Oct; 47(10):1121-1130.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P63" name="crb--P63" rid="P63" style="position: absolute; visibility: visible; width: 370px; top: 19037px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-60"><ul>
<li class="two_line" reference_id="R65">
<a href="/pubmed/24114802/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Mendelian randomization analysis with multiple genetic variants using summarized data.</a><span class="one_line_source">[Genet Epidemiol.  2013]</span><div class="alt-note">
<div class="authors">Burgess S, Butterworth A, Thompson SG</div>
<div class="citation">Genet Epidemiol. 2013 Nov; 37(7):658-65.</div>
</div>
</li>
<li class="two_line" reference_id="R66">
<a href="/pubmed/26661904/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Combining information on multiple instrumental variables in Mendelian randomization: comparison of allele score and summarized data methods.</a><span class="one_line_source">[Stat Med.  2016]</span><div class="alt-note">
<div class="authors">Burgess S, Dudbridge F, Thompson SG</div>
<div class="citation">Stat Med. 2016 May 20; 35(11):1880-906.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P66" name="crb--P66" rid="P66" style="position: absolute; visibility: visible; width: 370px; top: 19788px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-61"><ul>
<li class="two_line" reference_id="R67">
<a href="/pubmed/25632051/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Multivariable Mendelian randomization: the use of pleiotropic genetic variants to estimate causal effects.</a><span class="one_line_source">[Am J Epidemiol.  2015]</span><div class="alt-note">
<div class="authors">Burgess S, Thompson SG</div>
<div class="citation">Am J Epidemiol. 2015 Feb 15; 181(4):251-60.</div>
</div>
</li>
<li class="two_line" reference_id="R68">
<a href="/pubmed/25660081/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Re: "Multivariable Mendelian randomization: the use of pleiotropic genetic variants to estimate causal effects".</a><span class="one_line_source">[Am J Epidemiol.  2015]</span><div class="alt-note">
<div class="authors">Burgess S, Dudbridge F, Thompson SG</div>
<div class="citation">Am J Epidemiol. 2015 Feb 15; 181(4):290-1.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P68" name="crb--P68" rid="P68" style="position: absolute; visibility: visible; width: 370px; top: 20236px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-62"><ul>
<li class="two_line" reference_id="R13">
<a href="/pubmed/21646338/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Homogeneous antibody-based proximity extension assays provide sensitive and specific detection of low-abundant proteins in human blood.</a><span class="one_line_source">[Nucleic Acids Res.  2011]</span><div class="alt-note">
<div class="authors">Lundberg M, Eriksson A, Tran B, Assarsson E, Fredriksson S</div>
<div class="citation">Nucleic Acids Res. 2011 Aug; 39(15):e102.</div>
</div>
</li>
<li class="two_line" reference_id="R3">
<a href="/pubmed/28397792/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Erratum: Connecting genetic risk to disease end points through the human blood plasma proteome.</a><span class="one_line_source">[Nat Commun.  2017]</span><div class="alt-note">
<div class="authors">Suhre K, Arnold M, Bhagwat AM, Cotton RJ, Engelke R, Raffler J, Sarwath H, Thareja G, Wahl A, DeLisle RK, Gold L, Pezer M, Lauc G, El-Din Selim MA, Mook-Kanamori DO, Al-Dous EK, Mohamoud YA, Malek J, Strauch K, Grallert H, Peters A, Kastenmüller G, Gieger C, Graumann J</div>
<div class="citation">Nat Commun. 2017 Apr 11; 8():15345.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p4" name="crb--__p4" rid="__p4" style="position: absolute; visibility: visible; width: 370px; top: 6686px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-63"><ul><li class="two_line" reference_id="R23">
<a href="/pubmed/26192919/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Association analyses identify 38 susceptibility loci for inflammatory bowel disease and highlight shared genetic risk across populations.</a><span class="one_line_source">[Nat Genet.  2015]</span><div class="alt-note">
<div class="authors">Liu JZ, van Sommeren S, Huang H, Ng SC, Alberts R, Takahashi A, Ripke S, Lee JC, Jostins L, Shah T, Abedian S, Cheon JH, Cho J, Dayani NE, Franke L, Fuyuno Y, Hart A, Juyal RC, Juyal G, Kim WH, Morris AP, Poustchi H, Newman WG, Midha V, Orchard TR, Vahedi H, Sood A, Sung JY, Malekzadeh R, Westra HJ, Yamazaki K, Yang SK, International Multiple Sclerosis Genetics Consortium., International IBD Genetics Consortium., Barrett JC, Alizadeh BZ, Parkes M, Bk T, Daly MJ, Kubo M, Anderson CA, Weersma RK</div>
<div class="citation">Nat Genet. 2015 Sep; 47(9):979-986.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p6" name="crb--__p6" rid="__p6" style="position: absolute; visibility: visible; width: 370px; top: 7815px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-64"><ul>
<li class="two_line" reference_id="R26">
<a href="/pubmed/22808956/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genetically distinct subsets within ANCA-associated vasculitis.</a><span class="one_line_source">[N Engl J Med.  2012]</span><div class="alt-note">
<div class="authors">Lyons PA, Rayner TF, Trivedi S, Holle JU, Watts RA, Jayne DR, Baslund B, Brenchley P, Bruchfeld A, Chaudhry AN, Cohen Tervaert JW, Deloukas P, Feighery C, Gross WL, Guillevin L, Gunnarsson I, Harper L, Hrušková Z, Little MA, Martorana D, Neumann T, Ohlsson S, Padmanabhan S, Pusey CD, Salama AD, Sanders JS, Savage CO, Segelmark M, Stegeman CA, Tesař V, Vaglio A, Wieczorek S, Wilde B, Zwerina J, Rees AJ, Clayton DG, Smith KG</div>
<div class="citation">N Engl J Med. 2012 Jul 19; 367(3):214-23.</div>
</div>
</li>
<li class="two_line" reference_id="R27">
<a href="/pubmed/28029757/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Identification of Functional and Expression Polymorphisms Associated With Risk for Antineutrophil Cytoplasmic Autoantibody-Associated Vasculitis.</a><span class="one_line_source">[Arthritis Rheumatol.  2017]</span><div class="alt-note">
<div class="authors">Merkel PA, Xie G, Monach PA, Ji X, Ciavatta DJ, Byun J, Pinder BD, Zhao A, Zhang J, Tadesse Y, Qian D, Weirauch M, Nair R, Tsoi A, Pagnoux C, Carette S, Chung S, Cuthbertson D, Davis JC Jr, Dellaripa PF, Forbess L, Gewurz-Singer O, Hoffman GS, Khalidi N, Koening C, Langford CA, Mahr AD, McAlear C, Moreland L, Seo EP, Specks U, Spiera RF, Sreih A, St Clair EW, Stone JH, Ytterberg SR, Elder JT, Qu J, Ochi T, Hirano N, Edberg JC, Falk RJ, Amos CI, Siminovitch KA, Vasculitis Clinical Research Consortium.</div>
<div class="citation">Arthritis Rheumatol. 2017 May; 69(5):1054-1066.</div>
</div>
</li>
</ul></div>
</div>
</div>   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"><div id="body-link-popper-R1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="invert">Review</span> The role of regulatory variation in complex traits and disease.<div class="authors">Albert FW, Kruglyak L</div><div class="citation">Nat Rev Genet. 2015 Apr; 16(4):197-212.</div><p>[<a href="/pubmed/25707927/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R1">Ref list</a>]</p></div><div id="body-link-popper-R2" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Quantitative variability of 342 plasma proteins in a human twin population.<div class="authors">Liu Y, Buil A, Collins BC, Gillet LC, Blum LC, Cheng LY, Vitek O, Mouritsen J, Lachance G, Spector TD, Dermitzakis ET, Aebersold R</div><div class="citation">Mol Syst Biol. 2015 Feb 4; 11(1):786.</div><p>[<a href="/pubmed/25652787/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R2">Ref list</a>]</p></div><div id="body-link-popper-R3" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Erratum: Connecting genetic risk to disease end points through the human blood plasma proteome.<div class="authors">Suhre K, Arnold M, Bhagwat AM, Cotton RJ, Engelke R, Raffler J, Sarwath H, Thareja G, Wahl A, DeLisle RK, Gold L, Pezer M, Lauc G, El-Din Selim MA, Mook-Kanamori DO, Al-Dous EK, Mohamoud YA, Malek J, Strauch K, Grallert H, Peters A, Kastenmüller G, Gieger C, Graumann J</div><div class="citation">Nat Commun. 2017 Apr 11; 8():15345.</div><p>[<a href="/pubmed/28397792/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R3">Ref list</a>]</p></div><div id="body-link-popper-R5" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Whole-genome sequencing study of serum peptide levels: the Atherosclerosis Risk in Communities study.<div class="authors">de Vries PS, Yu B, Feofanova EV, Metcalf GA, Brown MR, Zeighami AL, Liu X, Muzny DM, Gibbs RA, Boerwinkle E, Morrison AC</div><div class="citation">Hum Mol Genet. 2017 Sep 1; 26(17):3442-3450.</div><p>[<a href="/pubmed/28854705/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R5">Ref list</a>]</p></div><div id="body-link-popper-R6" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Variation and genetic control of protein abundance in humans.<div class="authors">Wu L, Candille SI, Choi Y, Xie D, Jiang L, Li-Pook-Than J, Tang H, Snyder M</div><div class="citation">Nature. 2013 Jul 4; 499(7456):79-82.</div><p>[<a href="/pubmed/23676674/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R6">Ref list</a>]</p></div><div id="body-link-popper-R7" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Genomic variation. Impact of regulatory variation from RNA to protein.<div class="authors">Battle A, Khan Z, Wang SH, Mitrano A, Ford MJ, Pritchard JK, Gilad Y</div><div class="citation">Science. 2015 Feb 6; 347(6222):664-7.</div><p>[<a href="/pubmed/25657249/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R7">Ref list</a>]</p></div><div id="body-link-popper-R8" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="invert">Review</span> Nucleic Acid Ligands With Protein-like Side Chains: Modified Aptamers and Their Use as Diagnostic and Therapeutic Agents.<div class="authors">Rohloff JC, Gelinas AD, Jarvis TC, Ochsner UA, Schneider DJ, Gold L, Janjic N</div><div class="citation">Mol Ther Nucleic Acids. 2014 Oct 7; 3():e201.</div><p>[<a href="/pubmed/25291143/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R8">Ref list</a>]</p></div><div id="body-link-popper-R9" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Efficiency and safety of varying the frequency of whole blood donation (INTERVAL): a randomised trial of 45 000 donors.<div class="authors">Di Angelantonio E, Thompson SG, Kaptoge S, Moore C, Walker M, Armitage J, Ouwehand WH, Roberts DJ, Danesh J, INTERVAL Trial Group.</div><div class="citation">Lancet. 2017 Nov 25; 390(10110):2360-2371.</div><p>[<a href="/pubmed/28941948/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R9">Ref list</a>]</p></div><div id="body-link-popper-R10" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">The Allelic Landscape of Human Blood Cell Trait Variation and Links to Common Complex Disease.<div class="authors">Astle WJ, Elding H, Jiang T, Allen D, Ruklisa D, Mann AL, Mead D, Bouman H, Riveros-Mckay F, Kostadima MA, Lambourne JJ, Sivapalaratnam S, Downes K, Kundu K, Bomba L, Berentsen K, Bradley JR, Daugherty LC, Delaneau O, Freson K, Garner SF, Grassi L, Guerrero J, Haimel M, Janssen-Megens EM, Kaan A, Kamat M, Kim B, Mandoli A, Marchini J, Martens JHA, Meacham S, Megy K, O'Connell J, Petersen R, Sharifi N, Sheard SM, Staley JR, Tuna S, van der Ent M, Walter K, Wang SY, Wheeler E, Wilder SP, Iotchkova V, Moore C, Sambrook J, Stunnenberg HG, Di Angelantonio E, Kaptoge S, Kuijpers TW, Carrillo-de-Santa-Pau E, Juan D, Rico D, Valencia A, Chen L, Ge B, Vasquez L, Kwan T, Garrido-Martín D, Watt S, Yang Y, Guigo R, Beck S, Paul DS, Pastinen T, Bujold D, Bourque G, Frontini M, Danesh J, Roberts DJ, Ouwehand WH, Butterworth AS, Soranzo N</div><div class="citation">Cell. 2016 Nov 17; 167(5):1415-1429.e19.</div><p>[<a href="/pubmed/27863252/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R10">Ref list</a>]</p></div><div id="body-link-popper-R11" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Using published data in Mendelian randomization: a blueprint for efficient identification of causal risk factors.<div class="authors">Burgess S, Scott RA, Timpson NJ, Davey Smith G, Thompson SG, EPIC- InterAct Consortium.</div><div class="citation">Eur J Epidemiol. 2015 Jul; 30(7):543-52.</div><p>[<a href="/pubmed/25773750/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R11">Ref list</a>]</p></div><div id="body-link-popper-R12" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Patterns of cis regulatory variation in diverse human populations.<div class="authors">Stranger BE, Montgomery SB, Dimas AS, Parts L, Stegle O, Ingle CE, Sekowska M, Smith GD, Evans D, Gutierrez-Arcelus M, Price A, Raj T, Nisbett J, Nica AC, Beazley C, Durbin R, Deloukas P, Dermitzakis ET</div><div class="citation">PLoS Genet. 2012; 8(4):e1002639.</div><p>[<a href="/pubmed/22532805/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R12">Ref list</a>]</p></div><div id="body-link-popper-R13" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Homogeneous antibody-based proximity extension assays provide sensitive and specific detection of low-abundant proteins in human blood.<div class="authors">Lundberg M, Eriksson A, Tran B, Assarsson E, Fredriksson S</div><div class="citation">Nucleic Acids Res. 2011 Aug; 39(15):e102.</div><p>[<a href="/pubmed/21646338/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R13">Ref list</a>]</p></div><div id="body-link-popper-R14" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">The UK10K project identifies rare variants in health and disease.<div class="authors">UK10K Consortium., Walter K, Min JL, Huang J, Crooks L, Memari Y, McCarthy S, Perry JR, Xu C, Futema M, Lawson D, Iotchkova V, Schiffels S, Hendricks AE, Danecek P, Li R, Floyd J, Wain LV, Barroso I, Humphries SE, Hurles ME, Zeggini E, Barrett JC, Plagnol V, Richards JB, Greenwood CM, Timpson NJ, Durbin R, Soranzo N</div><div class="citation">Nature. 2015 Oct 1; 526(7571):82-90.</div><p>[<a href="/pubmed/26367797/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R14">Ref list</a>]</p></div><div id="body-link-popper-R15" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Systematic identification of trans eQTLs as putative drivers of known disease associations.<div class="authors">Westra HJ, Peters MJ, Esko T, Yaghootkar H, Schurmann C, Kettunen J, Christiansen MW, Fairfax BP, Schramm K, Powell JE, Zhernakova A, Zhernakova DV, Veldink JH, Van den Berg LH, Karjalainen J, Withoff S, Uitterlinden AG, Hofman A, Rivadeneira F, Hoen PAC', Reinmaa E, Fischer K, Nelis M, Milani L, Melzer D, Ferrucci L, Singleton AB, Hernandez DG, Nalls MA, Homuth G, Nauck M, Radke D, Völker U, Perola M, Salomaa V, Brody J, Suchy-Dicey A, Gharib SA, Enquobahrie DA, Lumley T, Montgomery GW, Makino S, Prokisch H, Herder C, Roden M, Grallert H, Meitinger T, Strauch K, Li Y, Jansen RC, Visscher PM, Knight JC, Psaty BM, Ripatti S, Teumer A, Frayling TM, Metspalu A, van Meurs JBJ, Franke L</div><div class="citation">Nat Genet. 2013 Oct; 45(10):1238-1243.</div><p>[<a href="/pubmed/24013639/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R15">Ref list</a>]</p></div><div id="body-link-popper-R18" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Genetics and beyond--the transcriptome of human monocytes and disease susceptibility.<div class="authors">Zeller T, Wild P, Szymczak S, Rotival M, Schillert A, Castagne R, Maouche S, Germain M, Lackner K, Rossmann H, Eleftheriadis M, Sinning CR, Schnabel RB, Lubos E, Mennerich D, Rust W, Perret C, Proust C, Nicaud V, Loscalzo J, Hübner N, Tregouet D, Münzel T, Ziegler A, Tiret L, Blankenberg S, Cambien F</div><div class="citation">PLoS One. 2010 May 18; 5(5):e10693.</div><p>[<a href="/pubmed/20502693/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R18">Ref list</a>]</p></div><div id="body-link-popper-R19" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">TMPRSS6 rs855791 modulates hepcidin transcription in vitro and serum hepcidin levels in normal individuals.<div class="authors">Nai A, Pagani A, Silvestri L, Campostrini N, Corbella M, Girelli D, Traglia M, Toniolo D, Camaschella C</div><div class="citation">Blood. 2011 Oct 20; 118(16):4459-62.</div><p>[<a href="/pubmed/21873547/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R19">Ref list</a>]</p></div><div id="body-link-popper-R20" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Genome-wide screen identifies rs646776 near sortilin as a regulator of progranulin levels in human plasma.<div class="authors">Carrasquillo MM, Nicholson AM, Finch N, Gibbs JR, Baker M, Rutherford NJ, Hunter TA, DeJesus-Hernandez M, Bisceglio GD, Mackenzie IR, Singleton A, Cookson MR, Crook JE, Dillman A, Hernandez D, Petersen RC, Graff-Radford NR, Younkin SG, Rademakers R</div><div class="citation">Am J Hum Genet. 2010 Dec 10; 87(6):890-7.</div><p>[<a href="/pubmed/21087763/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R20">Ref list</a>]</p></div><div id="body-link-popper-R21" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="invert">Review</span> The molecular and cellular pathology of α₁-antitrypsin deficiency.<div class="authors">Gooptu B, Dickens JA, Lomas DA</div><div class="citation">Trends Mol Med. 2014 Feb; 20(2):116-27.</div><p>[<a href="/pubmed/24374162/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R21">Ref list</a>]</p></div><div id="body-link-popper-R22" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">ProGeM: a framework for the prioritization of candidate causal genes at molecular quantitative trait loci.<div class="authors">Stacey D, Fauman EB, Ziemek D, Sun BB, Harshfield EL, Wood AM, Butterworth AS, Suhre K, Paul DS</div><div class="citation">Nucleic Acids Res. 2019 Jan 10; 47(1):e3.</div><p>[<a href="/pubmed/30239796/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R22">Ref list</a>]</p></div><div id="body-link-popper-R23" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Association analyses identify 38 susceptibility loci for inflammatory bowel disease and highlight shared genetic risk across populations.<div class="authors">Liu JZ, van Sommeren S, Huang H, Ng SC, Alberts R, Takahashi A, Ripke S, Lee JC, Jostins L, Shah T, Abedian S, Cheon JH, Cho J, Dayani NE, Franke L, Fuyuno Y, Hart A, Juyal RC, Juyal G, Kim WH, Morris AP, Poustchi H, Newman WG, Midha V, Orchard TR, Vahedi H, Sood A, Sung JY, Malekzadeh R, Westra HJ, Yamazaki K, Yang SK, International Multiple Sclerosis Genetics Consortium., International IBD Genetics Consortium., Barrett JC, Alizadeh BZ, Parkes M, Bk T, Daly MJ, Kubo M, Anderson CA, Weersma RK</div><div class="citation">Nat Genet. 2015 Sep; 47(9):979-986.</div><p>[<a href="/pubmed/26192919/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R23">Ref list</a>]</p></div><div id="body-link-popper-R24" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">High-Throughput Characterization of Blood Serum Proteomics of IBD Patients with Respect to Aging and Genetic Factors.<div class="authors">Di Narzo AF, Telesco SE, Brodmerkel C, Argmann C, Peters LA, Li K, Kidd B, Dudley J, Cho J, Schadt EE, Kasarskis A, Dobrin R, Hao K</div><div class="citation">PLoS Genet. 2017 Jan; 13(1):e1006565.</div><p>[<a href="/pubmed/28129359/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R24">Ref list</a>]</p></div><div id="body-link-popper-R25" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Fine-mapping inflammatory bowel disease loci to single-variant resolution.<div class="authors">Huang H, Fang M, Jostins L, Umićević Mirkov M, Boucher G, Anderson CA, Andersen V, Cleynen I, Cortes A, Crins F, D'Amato M, Deffontaine V, Dmitrieva J, Docampo E, Elansary M, Farh KK, Franke A, Gori AS, Goyette P, Halfvarson J, Haritunians T, Knight J, Lawrance IC, Lees CW, Louis E, Mariman R, Meuwissen T, Mni M, Momozawa Y, Parkes M, Spain SL, Théâtre E, Trynka G, Satsangi J, van Sommeren S, Vermeire S, Xavier RJ, International Inflammatory Bowel Disease Genetics Consortium., Weersma RK, Duerr RH, Mathew CG, Rioux JD, McGovern DPB, Cho JH, Georges M, Daly MJ, Barrett JC</div><div class="citation">Nature. 2017 Jul 13; 547(7662):173-178.</div><p>[<a href="/pubmed/28658209/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R25">Ref list</a>]</p></div><div id="body-link-popper-R26" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Genetically distinct subsets within ANCA-associated vasculitis.<div class="authors">Lyons PA, Rayner TF, Trivedi S, Holle JU, Watts RA, Jayne DR, Baslund B, Brenchley P, Bruchfeld A, Chaudhry AN, Cohen Tervaert JW, Deloukas P, Feighery C, Gross WL, Guillevin L, Gunnarsson I, Harper L, Hrušková Z, Little MA, Martorana D, Neumann T, Ohlsson S, Padmanabhan S, Pusey CD, Salama AD, Sanders JS, Savage CO, Segelmark M, Stegeman CA, Tesař V, Vaglio A, Wieczorek S, Wilde B, Zwerina J, Rees AJ, Clayton DG, Smith KG</div><div class="citation">N Engl J Med. 2012 Jul 19; 367(3):214-23.</div><p>[<a href="/pubmed/22808956/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R26">Ref list</a>]</p></div><div id="body-link-popper-R27" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Identification of Functional and Expression Polymorphisms Associated With Risk for Antineutrophil Cytoplasmic Autoantibody-Associated Vasculitis.<div class="authors">Merkel PA, Xie G, Monach PA, Ji X, Ciavatta DJ, Byun J, Pinder BD, Zhao A, Zhang J, Tadesse Y, Qian D, Weirauch M, Nair R, Tsoi A, Pagnoux C, Carette S, Chung S, Cuthbertson D, Davis JC Jr, Dellaripa PF, Forbess L, Gewurz-Singer O, Hoffman GS, Khalidi N, Koening C, Langford CA, Mahr AD, McAlear C, Moreland L, Seo EP, Specks U, Spiera RF, Sreih A, St Clair EW, Stone JH, Ytterberg SR, Elder JT, Qu J, Ochi T, Hirano N, Edberg JC, Falk RJ, Amos CI, Siminovitch KA, Vasculitis Clinical Research Consortium.</div><div class="citation">Arthritis Rheumatol. 2017 May; 69(5):1054-1066.</div><p>[<a href="/pubmed/28029757/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R27">Ref list</a>]</p></div><div id="body-link-popper-R28" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Genetic effects on gene expression across human tissues.<div class="authors">GTEx Consortium., Laboratory, Data Analysis &amp;Coordinating Center (LDACC)—Analysis Working Group., Statistical Methods groups—Analysis Working Group., Enhancing GTEx (eGTEx) groups., NIH Common Fund., NIH/NCI., NIH/NHGRI., NIH/NIMH., NIH/NIDA., Biospecimen Collection Source Site—NDRI., Biospecimen Collection Source Site—RPCI., Biospecimen Core Resource—VARI., Brain Bank Repository—University of Miami Brain Endowment Bank., Leidos Biomedical—Project Management., ELSI Study., Genome Browser Data Integration &amp;Visualization—EBI., Genome Browser Data Integration &amp;Visualization—UCSC Genomics Institute, University of California Santa Cruz., Lead analysts:., Laboratory, Data Analysis &amp;Coordinating Center (LDACC):., NIH program management:., Biospecimen collection:., Pathology:., eQTL manuscript working group:., Battle A, Brown CD, Engelhardt BE, Montgomery SB</div><div class="citation">Nature. 2017 Oct 11; 550(7675):204-213.</div><p>[<a href="/pubmed/29022597/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R28">Ref list</a>]</p></div><div id="body-link-popper-R29" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Prostate cancer screening in men aged 50-69 years (STHLM3): a prospective population-based diagnostic study.<div class="authors">Grönberg H, Adolfsson J, Aly M, Nordström T, Wiklund P, Brandberg Y, Thompson J, Wiklund F, Lindberg J, Clements M, Egevad L, Eklund M</div><div class="citation">Lancet Oncol. 2015 Dec; 16(16):1667-76.</div><p>[<a href="/pubmed/26563502/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R29">Ref list</a>]</p></div><div id="body-link-popper-R30" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Multiple newly identified loci associated with prostate cancer susceptibility.<div class="authors">Eeles RA, Kote-Jarai Z, Giles GG, Olama AA, Guy M, Jugurnauth SK, Mulholland S, Leongamornlert DA, Edwards SM, Morrison J, Field HI, Southey MC, Severi G, Donovan JL, Hamdy FC, Dearnaley DP, Muir KR, Smith C, Bagnato M, Ardern-Jones AT, Hall AL, O'Brien LT, Gehr-Swain BN, Wilkinson RA, Cox A, Lewis S, Brown PM, Jhavar SG, Tymrakiewicz M, Lophatananon A, Bryant SL, UK Genetic Prostate Cancer Study Collaborators., British Association of Urological Surgeons' Section of Oncology., UK ProtecT Study Collaborators., Horwich A, Huddart RA, Khoo VS, Parker CC, Woodhouse CJ, Thompson A, Christmas T, Ogden C, Fisher C, Jamieson C, Cooper CS, English DR, Hopper JL, Neal DE, Easton DF</div><div class="citation">Nat Genet. 2008 Mar; 40(3):316-21.</div><p>[<a href="/pubmed/18264097/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R30">Ref list</a>]</p></div><div id="body-link-popper-R31" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Multi-ancestry genome-wide association study of 21,000 cases and 95,000 controls identifies new risk loci for atopic dermatitis.<div class="authors">Paternoster L, Standl M, Waage J, Baurecht H, Hotze M, Strachan DP, Curtin JA, Bønnelykke K, Tian C, Takahashi A, Esparza-Gordillo J, Alves AC, Thyssen JP, den Dekker HT, Ferreira MA, Altmaier E, Sleiman PM, Xiao FL, Gonzalez JR, Marenholz I, Kalb B, Yanes MP, Xu CJ, Carstensen L, Groen-Blokhuis MM, Venturini C, Pennell CE, Barton SJ, Levin AM, Curjuric I, Bustamante M, Kreiner-Møller E, Lockett GA, Bacelis J, Bunyavanich S, Myers RA, Matanovic A, Kumar A, Tung JY, Hirota T, Kubo M, McArdle WL, Henderson AJ, Kemp JP, Zheng J, Smith GD, Rüschendorf F, Bauerfeind A, Lee-Kirsch MA, Arnold A, Homuth G, Schmidt CO, Mangold E, Cichon S, Keil T, Rodríguez E, Peters A, Franke A, Lieb W, Novak N, Fölster-Holst R, Horikoshi M, Pekkanen J, Sebert S, Husemoen LL, Grarup N, de Jongste JC, Rivadeneira F, Hofman A, Jaddoe VW, Pasmans SG, Elbert NJ, Uitterlinden AG, Marks GB, Thompson PJ, Matheson MC, Robertson CF, Australian Asthma Genetics Consortium (AAGC)., Ried JS, Li J, Zuo XB, Zheng XD, Yin XY, Sun LD, McAleer MA, O'Regan GM, Fahy CM, Campbell LE, Macek M, Kurek M, Hu D, Eng C, Postma DS, Feenstra B, Geller F, Hottenga JJ, Middeldorp CM, Hysi P, Bataille V, Spector T, Tiesler CM, Thiering E, Pahukasahasram B, Yang JJ, Imboden M, Huntsman S, Vilor-Tejedor N, Relton CL, Myhre R, Nystad W, Custovic A, Weiss ST, Meyers DA, Söderhäll C, Melén E, Ober C, Raby BA, Simpson A, Jacobsson B, Holloway JW, Bisgaard H, Sunyer J, Hensch NMP, Williams LK, Godfrey KM, Wang CA, Boomsma DI, Melbye M, Koppelman GH, Jarvis D, McLean WI, Irvine AD, Zhang XJ, Hakonarson H, Gieger C, Burchard EG, Martin NG, Duijts L, Linneberg A, Jarvelin MR, Noethen MM, Lau S, Hübner N, Lee YA, Tamari M, Hinds DA, Glass D, Brown SJ, Heinrich J, Evans DM, Weidinger S</div><div class="citation">Nat Genet. 2015 Dec; 47(12):1449-1456.</div><p>[<a href="/pubmed/26482879/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R31">Ref list</a>]</p></div><div id="body-link-popper-R32" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Effects of an oral MMP-9 and -12 inhibitor, AZD1236, on biomarkers in moderate/severe COPD: a randomised controlled trial.<div class="authors">Dahl R, Titlestad I, Lindqvist A, Wielders P, Wray H, Wang M, Samuelsson V, Mo J, Holt A</div><div class="citation">Pulm Pharmacol Ther. 2012 Apr; 25(2):169-77.</div><p>[<a href="/pubmed/22306193/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R32">Ref list</a>]</p></div><div id="body-link-popper-R33" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Development and Validation of a Protein-Based Risk Score for Cardiovascular Outcomes Among Patients With Stable Coronary Heart Disease.<div class="authors">Ganz P, Heidecker B, Hveem K, Jonasson C, Kato S, Segal MR, Sterling DG, Williams SA</div><div class="citation">JAMA. 2016 Jun 21; 315(23):2532-41.</div><p>[<a href="/pubmed/27327800/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R33">Ref list</a>]</p></div><div id="body-link-popper-R34" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">A novel MMP12 locus is associated with large artery atherosclerotic stroke using a genome-wide age-at-onset informed approach.<div class="authors">Traylor M, Mäkelä KM, Kilarski LL, Holliday EG, Devan WJ, Nalls MA, Wiggins KL, Zhao W, Cheng YC, Achterberg S, Malik R, Sudlow C, Bevan S, Raitoharju E, METASTROKE, International Stroke Genetics Consortium, Wellcome Trust Case Consortium 2 (WTCCC2)., Oksala N, Thijs V, Lemmens R, Lindgren A, Slowik A, Maguire JM, Walters M, Algra A, Sharma P, Attia JR, Boncoraglio GB, Rothwell PM, de Bakker PI, Bis JC, Saleheen D, Kittner SJ, Mitchell BD, Rosand J, Meschia JF, Levi C, Dichgans M, Lehtimäki T, Lewis CM, Markus HS</div><div class="citation">PLoS Genet. 2014 Jul; 10(7):e1004469.</div><p>[<a href="/pubmed/25078452/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R34">Ref list</a>]</p></div><div id="body-link-popper-R35" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">The support of human genetic evidence for approved drug indications.<div class="authors">Nelson MR, Tipney H, Painter JL, Shen J, Nicoletti P, Shen Y, Floratos A, Sham PC, Li MJ, Wang J, Cardon LR, Whittaker JC, Sanseau P</div><div class="citation">Nat Genet. 2015 Aug; 47(8):856-60.</div><p>[<a href="/pubmed/26121088/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R35">Ref list</a>]</p></div><div id="body-link-popper-R36" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Genome-wide association study identifies variants at CSF1, OPTN and TNFRSF11A as genetic risk factors for Paget's disease of bone.<div class="authors">Albagha OM, Visconti MR, Alonso N, Langston AL, Cundy T, Dargie R, Dunlop MG, Fraser WD, Hooper MJ, Isaia G, Nicholson GC, del Pino Montes J, Gonzalez-Sarmiento R, di Stefano M, Tenesa A, Walsh JP, Ralston SH</div><div class="citation">Nat Genet. 2010 Jun; 42(6):520-4.</div><p>[<a href="/pubmed/20436471/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R36">Ref list</a>]</p></div><div id="body-link-popper-R37" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Paget's disease of the bone after treatment with Denosumab: a case report.<div class="authors">Schwarz P, Rasmussen AQ, Kvist TM, Andersen UB, Jørgensen NR</div><div class="citation">Bone. 2012 May; 50(5):1023-5.</div><p>[<a href="/pubmed/22586699/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R37">Ref list</a>]</p></div><div id="body-link-popper-R38" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">The INTERVAL trial to determine whether intervals between blood donations can be safely and acceptably decreased to optimise blood supply: study protocol for a randomised controlled trial.<div class="authors">Moore C, Sambrook J, Walker M, Tolkien Z, Kaptoge S, Allen D, Mehenny S, Mant J, Di Angelantonio E, Thompson SG, Ouwehand W, Roberts DJ, Danesh J</div><div class="citation">Trials. 2014 Sep 17; 15():363.</div><p>[<a href="/pubmed/25230735/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R38">Ref list</a>]</p></div><div id="body-link-popper-R39" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Aptamer-based multiplexed proteomic technology for biomarker discovery.<div class="authors">Gold L, Ayers D, Bertino J, Bock C, Bock A, Brody EN, Carter J, Dalby AB, Eaton BE, Fitzwater T, Flather D, Forbes A, Foreman T, Fowler C, Gawande B, Goss M, Gunn M, Gupta S, Halladay D, Heil J, Heilig J, Hicke B, Husar G, Janjic N, Jarvis T, Jennings S, Katilius E, Keeney TR, Kim N, Koch TH, Kraemer S, Kroiss L, Le N, Levine D, Lindsey W, Lollo B, Mayfield W, Mehan M, Mehler R, Nelson SK, Nelson M, Nieuwlandt D, Nikrad M, Ochsner U, Ostroff RM, Otis M, Parker T, Pietrasiewicz S, Resnicow DI, Rohloff J, Sanders G, Sattin S, Schneider D, Singer B, Stanton M, Sterkel A, Stewart A, Stratford S, Vaught JD, Vrkljan M, Walker JJ, Watrobka M, Waugh S, Weiss A, Wilcox SK, Wolfson A, Wolk SK, Zhang C, Zichi D</div><div class="citation">PLoS One. 2010 Dec 7; 5(12):e15004.</div><p>[<a href="/pubmed/21165148/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R39">Ref list</a>]</p></div><div id="body-link-popper-R40" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Alzheimer's disease biomarker discovery using SOMAscan multiplexed protein technology.<div class="authors">Sattlecker M, Kiddle SJ, Newhouse S, Proitsi P, Nelson S, Williams S, Johnston C, Killick R, Simmons A, Westman E, Hodges A, Soininen H, Kłoszewska I, Mecocci P, Tsolaki M, Vellas B, Lovestone S, AddNeuroMed Consortium., Dobson RJ</div><div class="citation">Alzheimers Dement. 2014 Nov; 10(6):724-34.</div><p>[<a href="/pubmed/24768341/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R40">Ref list</a>]</p></div><div id="body-link-popper-R41" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Deriving the consequences of genomic variants with the Ensembl API and SNP Effect Predictor.<div class="authors">McLaren W, Pritchard B, Rios D, Chen Y, Flicek P, Cunningham F</div><div class="citation">Bioinformatics. 2010 Aug 15; 26(16):2069-70.</div><p>[<a href="/pubmed/20562413/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R41">Ref list</a>]</p></div><div id="body-link-popper-R42" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">UniProt: a hub for protein information.<div class="authors">UniProt Consortium.</div><div class="citation">Nucleic Acids Res. 2015 Jan; 43(Database issue):D204-12.</div><p>[<a href="/pubmed/25348405/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R42">Ref list</a>]</p></div><div id="body-link-popper-R43" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.<div class="authors">Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, Davis AP, Dolinski K, Dwight SS, Eppig JT, Harris MA, Hill DP, Issel-Tarver L, Kasarskis A, Lewis S, Matese JC, Richardson JE, Ringwald M, Rubin GM, Sherlock G</div><div class="citation">Nat Genet. 2000 May; 25(1):25-9.</div><p>[<a href="/pubmed/10802651/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R43">Ref list</a>]</p></div><div id="body-link-popper-R44" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Circulating Proteomic Signatures of Chronological Age.<div class="authors">Menni C, Kiddle SJ, Mangino M, Viñuela A, Psatha M, Steves C, Sattlecker M, Buil A, Newhouse S, Nelson S, Williams S, Voyle N, Soininen H, Kloszewska I, Mecocci P, Tsolaki M, Vellas B, Lovestone S, Spector TD, Dobson R, Valdes AM</div><div class="citation">J Gerontol A Biol Sci Med Sci. 2015 Jul; 70(7):809-16.</div><p>[<a href="/pubmed/25123647/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R44">Ref list</a>]</p></div><div id="body-link-popper-R45" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Aptamer-Based Proteomic Profiling Reveals Novel Candidate Biomarkers and Pathways in Cardiovascular Disease.<div class="authors">Ngo D, Sinha S, Shen D, Kuhn EW, Keyes MJ, Shi X, Benson MD, O'Sullivan JF, Keshishian H, Farrell LA, Fifer MA, Vasan RS, Sabatine MS, Larson MG, Carr SA, Wang TJ, Gerszten RE</div><div class="citation">Circulation. 2016 Jul 26; 134(4):270-85.</div><p>[<a href="/pubmed/27444932/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R45">Ref list</a>]</p></div><div id="body-link-popper-R46" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Second-generation PLINK: rising to the challenge of larger and richer datasets.<div class="authors">Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ</div><div class="citation">Gigascience. 2015; 4():7.</div><p>[<a href="/pubmed/25722852/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R46">Ref list</a>]</p></div><div id="body-link-popper-R47" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">A new multipoint method for genome-wide association studies by imputation of genotypes.<div class="authors">Marchini J, Howie B, Myers S, McVean G, Donnelly P</div><div class="citation">Nat Genet. 2007 Jul; 39(7):906-13.</div><p>[<a href="/pubmed/17572673/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R47">Ref list</a>]</p></div><div id="body-link-popper-R48" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">METAL: fast and efficient meta-analysis of genomewide association scans.<div class="authors">Willer CJ, Li Y, Abecasis GR</div><div class="citation">Bioinformatics. 2010 Sep 1; 26(17):2190-1.</div><p>[<a href="/pubmed/20616382/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R48">Ref list</a>]</p></div><div id="body-link-popper-R49" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits.<div class="authors">Yang J, Ferreira T, Morris AP, Medland SE, Genetic Investigation of ANthropometric Traits (GIANT) Consortium., DIAbetes Genetics Replication And Meta-analysis (DIAGRAM) Consortium., Madden PA, Heath AC, Martin NG, Montgomery GW, Weedon MN, Loos RJ, Frayling TM, McCarthy MI, Hirschhorn JN, Goddard ME, Visscher PM</div><div class="citation">Nat Genet. 2012 Mar 18; 44(4):369-75, S1-3.</div><p>[<a href="/pubmed/22426310/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R49">Ref list</a>]</p></div><div id="body-link-popper-R50" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">The NHGRI GWAS Catalog, a curated resource of SNP-trait associations.<div class="authors">Welter D, MacArthur J, Morales J, Burdett T, Hall P, Junkins H, Klemm A, Flicek P, Manolio T, Hindorff L, Parkinson H</div><div class="citation">Nucleic Acids Res. 2014 Jan; 42(Database issue):D1001-6.</div><p>[<a href="/pubmed/24316577/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R50">Ref list</a>]</p></div><div id="body-link-popper-R51" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Modeling sample variables with an Experimental Factor Ontology.<div class="authors">Malone J, Holloway E, Adamusiak T, Kapushesky M, Zheng J, Kolesnikov N, Zhukova A, Brazma A, Parkinson H</div><div class="citation">Bioinformatics. 2010 Apr 15; 26(8):1112-8.</div><p>[<a href="/pubmed/20200009/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R51">Ref list</a>]</p></div><div id="body-link-popper-R52" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Lineage-Specific Genome Architecture Links Enhancers and Non-coding Disease Variants to Target Gene Promoters.<div class="authors">Javierre BM, Burren OS, Wilder SP, Kreuzhuber R, Hill SM, Sewitz S, Cairns J, Wingett SW, Várnai C, Thiecke MJ, Burden F, Farrow S, Cutler AJ, Rehnström K, Downes K, Grassi L, Kostadima M, Freire-Pritchett P, Wang F, BLUEPRINT Consortium., Stunnenberg HG, Todd JA, Zerbino DR, Stegle O, Ouwehand WH, Frontini M, Wallace C, Spivakov M, Fraser P</div><div class="citation">Cell. 2016 Nov 17; 167(5):1369-1384.e19.</div><p>[<a href="/pubmed/27863249/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R52">Ref list</a>]</p></div><div id="body-link-popper-R53" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Software for computing and annotating genomic ranges.<div class="authors">Lawrence M, Huber W, Pagès H, Aboyoun P, Carlson M, Gentleman R, Morgan MT, Carey VJ</div><div class="citation">PLoS Comput Biol. 2013; 9(8):e1003118.</div><p>[<a href="/pubmed/23950696/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R53">Ref list</a>]</p></div><div id="body-link-popper-R54" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">OMIM.org: Online Mendelian Inheritance in Man (OMIM®), an online catalog of human genes and genetic disorders.<div class="authors">Amberger JS, Bocchini CA, Schiettecatte F, Scott AF, Hamosh A</div><div class="citation">Nucleic Acids Res. 2015 Jan; 43(Database issue):D789-98.</div><p>[<a href="/pubmed/25428349/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R54">Ref list</a>]</p></div><div id="body-link-popper-R55" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">KEGG as a reference resource for gene and protein annotation.<div class="authors">Kanehisa M, Sato Y, Kawashima M, Furumichi M, Tanabe M</div><div class="citation">Nucleic Acids Res. 2016 Jan 4; 44(D1):D457-62.</div><p>[<a href="/pubmed/26476454/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R55">Ref list</a>]</p></div><div id="body-link-popper-R56" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">STRING v10: protein-protein interaction networks, integrated over the tree of life.<div class="authors">Szklarczyk D, Franceschini A, Wyder S, Forslund K, Heller D, Huerta-Cepas J, Simonovic M, Roth A, Santos A, Tsafou KP, Kuhn M, Bork P, Jensen LJ, von Mering C</div><div class="citation">Nucleic Acids Res. 2015 Jan; 43(Database issue):D447-52.</div><p>[<a href="/pubmed/25352553/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R56">Ref list</a>]</p></div><div id="body-link-popper-R57" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">InterMine: a flexible data warehouse system for the integration and analysis of heterogeneous biological data.<div class="authors">Smith RN, Aleksic J, Butano D, Carr A, Contrino S, Hu F, Lyne M, Lyne R, Kalderimis A, Rutherford K, Stepan R, Sullivan J, Wakeling M, Watkins X, Micklem G</div><div class="citation">Bioinformatics. 2012 Dec 1; 28(23):3163-5.</div><p>[<a href="/pubmed/23023984/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R57">Ref list</a>]</p></div><div id="body-link-popper-R58" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">STRING v9.1: protein-protein interaction networks, with increased coverage and integration.<div class="authors">Franceschini A, Szklarczyk D, Frankild S, Kuhn M, Simonovic M, Roth A, Lin J, Minguez P, Bork P, von Mering C, Jensen LJ</div><div class="citation">Nucleic Acids Res. 2013 Jan; 41(Database issue):D808-15.</div><p>[<a href="/pubmed/23203871/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R58">Ref list</a>]</p></div><div id="body-link-popper-R59" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Analysis of protein-coding genetic variation in 60,706 humans.<div class="authors">Lek M, Karczewski KJ, Minikel EV, Samocha KE, Banks E, Fennell T, O'Donnell-Luria AH, Ware JS, Hill AJ, Cummings BB, Tukiainen T, Birnbaum DP, Kosmicki JA, Duncan LE, Estrada K, Zhao F, Zou J, Pierce-Hoffman E, Berghout J, Cooper DN, Deflaux N, DePristo M, Do R, Flannick J, Fromer M, Gauthier L, Goldstein J, Gupta N, Howrigan D, Kiezun A, Kurki MI, Moonshine AL, Natarajan P, Orozco L, Peloso GM, Poplin R, Rivas MA, Ruano-Rubio V, Rose SA, Ruderfer DM, Shakir K, Stenson PD, Stevens C, Thomas BP, Tiao G, Tusie-Luna MT, Weisburd B, Won HH, Yu D, Altshuler DM, Ardissino D, Boehnke M, Danesh J, Donnelly S, Elosua R, Florez JC, Gabriel SB, Getz G, Glatt SJ, Hultman CM, Kathiresan S, Laakso M, McCarroll S, McCarthy MI, McGovern D, McPherson R, Neale BM, Palotie A, Purcell SM, Saleheen D, Scharf JM, Sklar P, Sullivan PF, Tuomilehto J, Tsuang MT, Watkins HC, Wilson JG, Daly MJ, MacArthur DG, Exome Aggregation Consortium.</div><div class="citation">Nature. 2016 Aug 18; 536(7616):285-91.</div><p>[<a href="/pubmed/27535533/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R59">Ref list</a>]</p></div><div id="body-link-popper-R60" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">60. <span class="element-citation">Iotchkova V, et al.  GARFIELD—GWAS Analysis of Regulatory or Functional
Information Enrichment with LD correction. <span>2016</span> Preprint at  <a href="https://www.biorxiv.org/content/early/2016/11/07/085738" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://www.biorxiv.org/content/early/2016/11/07/085738</a>.</span> [<a href="#R60">Ref list</a>]</div><div id="body-link-popper-R61" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">PhenoScanner: a database of human genotype-phenotype associations.<div class="authors">Staley JR, Blackshaw J, Kamat MA, Ellis S, Surendran P, Sun BB, Paul DS, Freitag D, Burgess S, Danesh J, Young R, Butterworth AS</div><div class="citation">Bioinformatics. 2016 Oct 15; 32(20):3207-3209.</div><p>[<a href="/pubmed/27318201/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R61">Ref list</a>]</p></div><div id="body-link-popper-R62" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Bayesian test for colocalisation between pairs of genetic association studies using summary statistics.<div class="authors">Giambartolomei C, Vukcevic D, Schadt EE, Franke L, Hingorani AD, Wallace C, Plagnol V</div><div class="citation">PLoS Genet. 2014 May; 10(5):e1004383.</div><p>[<a href="/pubmed/24830394/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R62">Ref list</a>]</p></div><div id="body-link-popper-R63" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Nature's randomised trials.<div class="authors">Hingorani A, Humphries S</div><div class="citation">Lancet. 2005 Dec 3; 366(9501):1906-8.</div><p>[<a href="/pubmed/16325682/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R63">Ref list</a>]</p></div><div id="body-link-popper-R64" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">A comprehensive 1,000 Genomes-based genome-wide association meta-analysis of coronary artery disease.<div class="authors">Nikpay M, Goel A, Won HH, Hall LM, Willenborg C, Kanoni S, Saleheen D, Kyriakou T, Nelson CP, Hopewell JC, Webb TR, Zeng L, Dehghan A, Alver M, Armasu SM, Auro K, Bjonnes A, Chasman DI, Chen S, Ford I, Franceschini N, Gieger C, Grace C, Gustafsson S, Huang J, Hwang SJ, Kim YK, Kleber ME, Lau KW, Lu X, Lu Y, Lyytikäinen LP, Mihailov E, Morrison AC, Pervjakova N, Qu L, Rose LM, Salfati E, Saxena R, Scholz M, Smith AV, Tikkanen E, Uitterlinden A, Yang X, Zhang W, Zhao W, de Andrade M, de Vries PS, van Zuydam NR, Anand SS, Bertram L, Beutner F, Dedoussis G, Frossard P, Gauguier D, Goodall AH, Gottesman O, Haber M, Han BG, Huang J, Jalilzadeh S, Kessler T, König IR, Lannfelt L, Lieb W, Lind L, Lindgren CM, Lokki ML, Magnusson PK, Mallick NH, Mehra N, Meitinger T, Memon FU, Morris AP, Nieminen MS, Pedersen NL, Peters A, Rallidis LS, Rasheed A, Samuel M, Shah SH, Sinisalo J, Stirrups KE, Trompet S, Wang L, Zaman KS, Ardissino D, Boerwinkle E, Borecki IB, Bottinger EP, Buring JE, Chambers JC, Collins R, Cupples LA, Danesh J, Demuth I, Elosua R, Epstein SE, Esko T, Feitosa MF, Franco OH, Franzosi MG, Granger CB, Gu D, Gudnason V, Hall AS, Hamsten A, Harris TB, Hazen SL, Hengstenberg C, Hofman A, Ingelsson E, Iribarren C, Jukema JW, Karhunen PJ, Kim BJ, Kooner JS, Kullo IJ, Lehtimäki T, Loos RJF, Melander O, Metspalu A, März W, Palmer CN, Perola M, Quertermous T, Rader DJ, Ridker PM, Ripatti S, Roberts R, Salomaa V, Sanghera DK, Schwartz SM, Seedorf U, Stewart AF, Stott DJ, Thiery J, Zalloua PA, O'Donnell CJ, Reilly MP, Assimes TL, Thompson JR, Erdmann J, Clarke R, Watkins H, Kathiresan S, McPherson R, Deloukas P, Schunkert H, Samani NJ, Farrall M</div><div class="citation">Nat Genet. 2015 Oct; 47(10):1121-1130.</div><p>[<a href="/pubmed/26343387/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R64">Ref list</a>]</p></div><div id="body-link-popper-R65" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Mendelian randomization analysis with multiple genetic variants using summarized data.<div class="authors">Burgess S, Butterworth A, Thompson SG</div><div class="citation">Genet Epidemiol. 2013 Nov; 37(7):658-65.</div><p>[<a href="/pubmed/24114802/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R65">Ref list</a>]</p></div><div id="body-link-popper-R66" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Combining information on multiple instrumental variables in Mendelian randomization: comparison of allele score and summarized data methods.<div class="authors">Burgess S, Dudbridge F, Thompson SG</div><div class="citation">Stat Med. 2016 May 20; 35(11):1880-906.</div><p>[<a href="/pubmed/26661904/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R66">Ref list</a>]</p></div><div id="body-link-popper-R67" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Multivariable Mendelian randomization: the use of pleiotropic genetic variants to estimate causal effects.<div class="authors">Burgess S, Thompson SG</div><div class="citation">Am J Epidemiol. 2015 Feb 15; 181(4):251-60.</div><p>[<a href="/pubmed/25632051/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R67">Ref list</a>]</p></div><div id="body-link-popper-R68" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Re: "Multivariable Mendelian randomization: the use of pleiotropic genetic variants to estimate causal effects".<div class="authors">Burgess S, Dudbridge F, Thompson SG</div><div class="citation">Am J Epidemiol. 2015 Feb 15; 181(4):290-1.</div><p>[<a href="/pubmed/25660081/" ref="reftype=pubmed&amp;article-id=6697541&amp;issue-id=340460&amp;journal-id=358&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="#R68">Ref list</a>]</p></div></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T17:11:50-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE8878C9DD5B8D110000000000C5006D&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6697541%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC6697541/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size2, sans-serif;"></div></div></body></html>