<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC6078592/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies WNT7B as a novel locus for central corneal thickness in Latinos</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6078592/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association study identifies WNT7B as a novel locus for central corneal thickness in Latinos"><meta name="citation_authors" content="Xiaoyi Gao, Drew R. Nannini, Kristen Corrao, Mina Torres, Yii-Der I. Chen, Bao J. Fan, Janey L. Wiggs, International Glaucoma Genetics Consortium, Kent D. Taylor, W. James Gauderman, Jerome I. Rotter, Rohit Varma"><meta name="citation_date" content="15 November 2016"><meta name="citation_issue" content="22"><meta name="citation_volume" content="25"><meta name="citation_firstpage" content="5035"><meta name="citation_doi" content="10.1093/hmg/ddw319"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC6078592/?report=abstract"><meta name="citation_pmid" content="28171582"><meta name="DC.Title" content="Genome-wide association study identifies WNT7B as a novel locus for central corneal thickness in Latinos"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Xiaoyi Gao"><meta name="DC.Contributor" content="Drew R. Nannini"><meta name="DC.Contributor" content="Kristen Corrao"><meta name="DC.Contributor" content="Mina Torres"><meta name="DC.Contributor" content="Yii-Der I. Chen"><meta name="DC.Contributor" content="Bao J. Fan"><meta name="DC.Contributor" content="Janey L. Wiggs"><meta name="DC.Contributor" content="International Glaucoma Genetics Consortium"><meta name="DC.Contributor" content="Kent D. Taylor"><meta name="DC.Contributor" content="W. James Gauderman"><meta name="DC.Contributor" content="Jerome I. Rotter"><meta name="DC.Contributor" content="Rohit Varma"><meta name="DC.Date" content="2016 Nov 15"><meta name="DC.Identifier" content="10.1093/hmg/ddw319"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies WNT7B as a novel locus for central corneal thickness in Latinos"><meta property="og:type" content="article"><meta property="og:description" content="The cornea is the outermost layer of the eye and is a vital component of focusing incoming light on the retina. Central corneal thickness (CCT) is now recognized to have a significant role in ocular health and is a risk factor for various ocular diseases, ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6078592/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC6078592/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC6078592/pdf/ddw319.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A8CB6DD598CE100000000083304D2.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6078592%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC6078592</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2016 Nov 15; 25(22): 5035–5045. </span></div><div><span class="fm-vol-iss-date">Published online 2016 Sep 20. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddw319" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddw319</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC6078592</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/28171582">28171582</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies <em>WNT7B</em> as a novel locus for central corneal thickness in Latinos</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Gao%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582" class="affpopup" co-rid="_co_idm140380848414416" co-class="co-affbox">Xiaoyi Gao</a>,<sup>1</sup><sup></sup> <a href="/pubmed/?term=Nannini%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582" class="affpopup" co-rid="_co_idm140380848399424" co-class="co-affbox">Drew R. Nannini</a>,<sup>1</sup> <a href="/pubmed/?term=Corrao%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582" class="affpopup" co-rid="_co_idm140380848397664" co-class="co-affbox">Kristen Corrao</a>,<sup>1</sup> <a href="/pubmed/?term=Torres%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582" class="affpopup" co-rid="_co_idm140380848395904" co-class="co-affbox">Mina Torres</a>,<sup>2</sup> <a href="/pubmed/?term=Chen%20YD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582" class="affpopup" co-rid="_co_idm140380848394080" co-class="co-affbox">Yii-Der I. Chen</a>,<sup>3</sup> <a href="/pubmed/?term=Fan%20BJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582" class="affpopup" co-rid="_co_idm140380848392256" co-class="co-affbox">Bao J. Fan</a>,<sup>4</sup> <a href="/pubmed/?term=Wiggs%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582" class="affpopup" co-rid="_co_idm140380846849296" co-class="co-affbox">Janey L. Wiggs</a>,<sup>4</sup> International Glaucoma Genetics Consortium,<sup></sup> <a href="/pubmed/?term=Taylor%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582" class="affpopup" co-rid="_co_idm140380846846304" co-class="co-affbox">Kent D. Taylor</a>,<sup>3</sup> <a href="/pubmed/?term=Gauderman%20WJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582" class="affpopup" co-rid="_co_idm140380846844320" co-class="co-affbox">W. James Gauderman</a>,<sup>5</sup> <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582" class="affpopup" co-rid="_co_idm140380846842336" co-class="co-affbox">Jerome I. Rotter</a>,<sup>3</sup> and  <a href="/pubmed/?term=Varma%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582" class="affpopup" co-rid="_co_idm140380846840352" co-class="co-affbox">Rohit Varma</a><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140380848414416"><h3 class="no_margin">Xiaoyi Gao</h3><p>  <sup>1</sup>Department of Ophthalmology and Visual Sciences, University of Illinois at Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Gao%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582">Xiaoyi Gao</a></div></div><div id="_co_idm140380848399424"><h3 class="no_margin">Drew R. Nannini</h3><p>  <sup>1</sup>Department of Ophthalmology and Visual Sciences, University of Illinois at Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Nannini%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582">Drew R. Nannini</a></div></div><div id="_co_idm140380848397664"><h3 class="no_margin">Kristen Corrao</h3><p>  <sup>1</sup>Department of Ophthalmology and Visual Sciences, University of Illinois at Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Corrao%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582">Kristen Corrao</a></div></div><div id="_co_idm140380848395904"><h3 class="no_margin">Mina Torres</h3><p>  <sup>2</sup>USC Roski Eye Institute, Department of Ophthalmology, University of Southern California, Los Angeles, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Torres%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582">Mina Torres</a></div></div><div id="_co_idm140380848394080"><h3 class="no_margin">Yii-Der I. Chen</h3><p>  <sup>3</sup>Institute for Translational Genomics and Population Sciences, Los Angeles Biomedical Research Institute and Department of Pediatrics and Medicine at Harbor-UCLA, Torrance, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Chen%20YD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582">Yii-Der I. Chen</a></div></div><div id="_co_idm140380848392256"><h3 class="no_margin">Bao J. Fan</h3><p>  <sup>4</sup>Department of Ophthalmology, Harvard Medical School, Massachusetts Eye and Ear Infirmary, Boston, MA 02114, USA</p><div>Find articles by <a href="/pubmed/?term=Fan%20BJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582">Bao J. Fan</a></div></div><div id="_co_idm140380846849296"><h3 class="no_margin">Janey L. Wiggs</h3><p>  <sup>4</sup>Department of Ophthalmology, Harvard Medical School, Massachusetts Eye and Ear Infirmary, Boston, MA 02114, USA</p><div>Find articles by <a href="/pubmed/?term=Wiggs%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582">Janey L. Wiggs</a></div></div><div id="_co_idm140380846846304"><h3 class="no_margin">Kent D. Taylor</h3><p>  <sup>3</sup>Institute for Translational Genomics and Population Sciences, Los Angeles Biomedical Research Institute and Department of Pediatrics and Medicine at Harbor-UCLA, Torrance, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Taylor%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582">Kent D. Taylor</a></div></div><div id="_co_idm140380846844320"><h3 class="no_margin">W. James Gauderman</h3><p>  <sup>5</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California, Los Angeles, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Gauderman%20WJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582">W. James Gauderman</a></div></div><div id="_co_idm140380846842336"><h3 class="no_margin">Jerome I. Rotter</h3><p>  <sup>3</sup>Institute for Translational Genomics and Population Sciences, Los Angeles Biomedical Research Institute and Department of Pediatrics and Medicine at Harbor-UCLA, Torrance, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582">Jerome I. Rotter</a></div></div><div id="_co_idm140380846840352"><h3 class="no_margin">Rohit Varma</h3><p>  <sup>2</sup>USC Roski Eye Institute, Department of Ophthalmology, University of Southern California, Los Angeles, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Varma%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28171582">Rohit Varma</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140380851101664_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140380851101664_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140380851101664_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140380851101664_ai" style="display:none"><div class="fm-affl" id="ddw319-aff1" lang="en">  <sup>1</sup>Department of Ophthalmology and Visual Sciences, University of Illinois at Chicago, Chicago, IL, USA</div><div class="fm-affl" id="ddw319-aff2" lang="en">  <sup>2</sup>USC Roski Eye Institute, Department of Ophthalmology, University of Southern California, Los Angeles, CA, USA</div><div class="fm-affl" id="ddw319-aff3" lang="en">  <sup>3</sup>Institute for Translational Genomics and Population Sciences, Los Angeles Biomedical Research Institute and Department of Pediatrics and Medicine at Harbor-UCLA, Torrance, CA, USA</div><div class="fm-affl" id="ddw319-aff4" lang="en">  <sup>4</sup>Department of Ophthalmology, Harvard Medical School, Massachusetts Eye and Ear Infirmary, Boston, MA 02114, USA</div><div class="fm-affl" id="ddw319-aff5" lang="en">  <sup>5</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California, Los Angeles, CA, USA</div><div id="ddw319-cor1"><sup>*</sup>To whom correspondence should be addressed at: Xiaoyi Gao, PhD, Department of Ophthalmology and Visual Sciences, University of Illinois at Chicago, 1905 W Taylor Street, Room 235, Chicago, IL 60612, USA. Tel: (312) 996-5825; Fax: (312) 996-0759; Email: <a href="mailto:dev@null" data-email="ude.ciu@oagr" class="oemail">ude.ciu@oagr</a></div><div id="ddw319-FN1">
<sup>†</sup>See Appendix for the list of International Glaucoma Genetics Consortium members.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140380851101664_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2016 Mar 28; Revised 2016 Aug 18; Accepted 2016 Sep 12.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140380851101664_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2016. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC6078592/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Data.</strong></div><div><a href="/pmc/articles/PMC6078592/bin/ddw319_supp.zip" data-ga-action="click_feat_suppl">ddw319_supp.zip</a><span style="color:gray"> (55M)</span></div><div class="small guid">GUID:&nbsp;323B7570-D820-45B3-ADC9-0E6BF8B54AF5</div></div></dd></dl></div><div id="idm140380846808624" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140380846808624title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">The cornea is the outermost layer of the eye and is a vital component of focusing incoming light on the retina. Central corneal thickness (CCT) is now recognized to have a significant role in ocular health and is a risk factor for various ocular diseases, such as keratoconus and primary open angle glaucoma. Most previous genetic studies utilized European and Asian subjects to identify genetic loci associated with CCT. Minority populations, such as Latinos, may aid in identifying additional loci and improve our understanding of the genetic architecture of CCT. In this study, we conducted a genome-wide association study (GWAS) in Latinos, a traditionally understudied population in genetic research, to further identify loci contributing to CCT. Study participants were genotyped using either the Illumina OmniExpress BeadChip (∼730K markers) or the Illumina Hispanic/SOL BeadChip (∼2.5 million markers). All study participants were 40 years of age and older. We assessed the association between individual single nucleotide polymorphisms (SNPs) and CCT using linear regression, adjusting for age, gender and principal components of genetic ancestry. To expand genomic coverage and to interrogate additional SNPs, we imputed SNPs from the 1000 Genomes Project reference panels. We identified a novel SNP, rs10453441 (<em>P</em> = 6.01E-09), in an intron of <em>WNT7B</em> that is associated with CCT. Furthermore, <em>WNT7B</em> is expressed in the human cornea. We also replicated 11 previously reported loci, including <em>IBTK, RXRA-COL5A1</em>, <em>COL5A1</em>, <em>FOXO1</em>, <em>LRRK1</em> and <em>ZNF469</em> (<em>P</em> &lt; 1.25E-3). These findings provide further insight into the genetic architecture of CCT and illustrate that the use of minority groups in GWAS will help identify additional loci.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p3" class="p p-first">The cornea is the clear, outermost layer of the eye that aids in refracting incoming light onto the lens, which is then further refocused onto the retina. Compared to the lens, the cornea has greater refractive power and thus, is a critical ocular structure to focus light on the retina. Due to the large refractive power, any structural variations of the cornea (i.e. smoothness, curvature, or thickness) may result in adverse effects on vision. Mounting evidence suggests central corneal thickness (CCT) plays a significant factor in ocular health. Thin CCT has been observed in several visual disorders, including brittle cornea syndrome, keratoconus and osteogenesis imperfecta (<a href="#ddw319-B1" rid="ddw319-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593836">1–3</a>). CCT is also an important endophenotype for primary open angle glaucoma (POAG). Most notably, thin CCT has been found to be associated with POAG in multiple epidemiologic studies (<a href="#ddw319-B4" rid="ddw319-B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593786">4–10</a>). Therefore, understanding the factors that influence CCT may help to better predict CCT and potentially, POAG.</p><p id="__p4">Ethnic differences in CCT have been observed in several studies. Generally, Hispanics have slightly higher CCT measurements than Caucasians, while individuals of African descent exhibit the lowest CCT, and CCT from Asian populations representing a wide range of values (<a href="#ddw319-B7" rid="ddw319-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593857">7</a>,<a href="#ddw319-B11" rid="ddw319-B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593847">11–13</a>). While environmental factors may contribute to the observed racial variation, genetic variants specific to certain ethnic populations are more likely to have a major role in CCT determination (<a href="#ddw319-B14" rid="ddw319-B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593810">14</a>). Moreover, CCT is a highly heritable trait, with heritability ranging from 0.60 to 0.95 (<a href="#ddw319-B15" rid="ddw319-B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593838">15</a>,<a href="#ddw319-B16" rid="ddw319-B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593811">16</a>). Taken together, these findings suggest that the primary factors determining CCT are genetic rather than environmental.</p><p id="__p5">Recently, genome-wide association studies (GWAS) have identified numerous genetic loci that are associated with CCT in various ethnic populations. These GWAS, conducted in European, Asian and Latino populations, have identified single nucleotide polymorphisms (SNPs) in or nearby <em>AKAP13</em> (<a href="#ddw319-B17" rid="ddw319-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593848">17</a>,<a href="#ddw319-B18" rid="ddw319-B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593764">18</a>), <em>COL5A1</em> (<a href="#ddw319-B18" rid="ddw319-B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593781">18</a>,<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593785">19</a>), <em>COL8A2</em> (<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593768">19</a>), <em>FAM53B</em> (<a href="#ddw319-B20" rid="ddw319-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593791">20</a>), <em>FOXO1</em> (<a href="#ddw319-B20" rid="ddw319-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593822">20</a>), <em>IBTK</em> (<a href="#ddw319-B17" rid="ddw319-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593738">17</a>), <em>LRRK1</em> (<a href="#ddw319-B17" rid="ddw319-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593749">17</a>), <em>RXRA-COL5A1</em> (<a href="#ddw319-B18" rid="ddw319-B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593745">18</a>,<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593826">19</a>,<a href="#ddw319-B21" rid="ddw319-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593769">21</a>,<a href="#ddw319-B22" rid="ddw319-B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593854">22</a>) and <em>ZNF469</em> (<a href="#ddw319-B18" rid="ddw319-B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593844">18–</a><a href="#ddw319-B21" rid="ddw319-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593797">21</a>) to be associated with CCT. A meta-analysis consisting of a large number of European and Asian individuals identified 16 additional loci associated with CCT, including <em>LPAR1</em> and <em>ARID5B</em> (<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593828">23</a>). Furthermore, we reported the first GWAS on CCT in Latinos and replicated previously reported loci, as well as identified novel SNPs (<a href="#ddw319-B22" rid="ddw319-B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593805">22</a>).</p><p id="__p6" class="p p-last">While genetic loci associated with CCT have consistently been identified, these loci have primarily been found in populations of European and Asian descent. We hypothesize performing GWAS in minority populations may identify new loci associated with CCT, as well as elucidate the racial differences of CCT. Latinos, the largest minority group in the United States, are traditionally an understudied population among genetic studies. Furthermore, Latinos exhibit thicker CCT compared to other ethnic groups. Taken together, identifying novel genetic variants associated with CCT in Latinos may elucidate racial differences and further our understanding of the genetic architecture of CCT. Therefore, the purpose of this study is to conduct a larger GWAS using Latino subjects to identify novel genetic loci for CCT. To our knowledge, this is the largest GWAS for CCT conducted in a Latino population.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Results</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Study sample</h3><p id="__p7" class="p p-first-last">The descriptive statistics of the study sample are presented in <a href="/pmc/articles/PMC6078592/table/ddw319-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw319-T1" rid-ob="ob-ddw319-T1" co-legend-rid=""><span>Table 1</span></a>. Overall, a total of 4,515 individuals were used to assess the associations between SNPs and CCT, of which 3,584 unrelated individuals were used as a discovery set and 931 related individuals were used in a replication set. The overall mean age (standard deviation) is 54.8 (10.5), with the discovery set and replication set having mean ages of 54.2 (9.9) and 56.9 (12.5), respectively. The proportion of females in the study sample is 58.8%, with the discovery set composed of 56.5% women and the replication set composed of 68.0% women. Combined, the CCT values are normally distributed with an overall mean of 550.0 (33.4) μm, with the average of the discovery and replicate sets as 550.4 (33.2) μm (range: 443–668) and 548.7 (34.0) μm (range: 451–661), respectively.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw319-T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p8">Descriptive statistics of the study sample</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140380846749104" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Study</th><th rowspan="1" colspan="1">Sample Size</th><th rowspan="1" colspan="1">Females, %</th><th rowspan="1" colspan="1">Age (years), Mean (SD)</th><th rowspan="1" colspan="1">CCT (μm), Mean (SD)</th><th rowspan="1" colspan="1">CCT (μm) Range</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Discovery Set (stage 1)</td><td rowspan="1" colspan="1">3,584</td><td rowspan="1" colspan="1">56.5</td><td rowspan="1" colspan="1">54.2 (9.9)</td><td rowspan="1" colspan="1">550.4 (33.2)</td><td rowspan="1" colspan="1">443–668</td></tr><tr><td rowspan="1" colspan="1">Replication Set (stage 2)</td><td rowspan="1" colspan="1">931</td><td rowspan="1" colspan="1">68.0</td><td rowspan="1" colspan="1">56.9 (12.5)</td><td rowspan="1" colspan="1">548.7 (34.0)</td><td rowspan="1" colspan="1">451–661</td></tr><tr><td rowspan="1" colspan="1">Total</td><td rowspan="1" colspan="1">4,515</td><td rowspan="1" colspan="1">58.8</td><td rowspan="1" colspan="1">54.8 (10.5)</td><td rowspan="1" colspan="1">550.0 (33.4)</td><td rowspan="1" colspan="1">443–668</td></tr></tbody></table></div><div id="largeobj_idm140380846749104" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6078592/table/ddw319-T1/?report=objectonly">Open in a separate window</a></div></div></div><div id="__sec4" class="sec"><h3 id="__sec4title">Genome-wide association results</h3><p id="__p9" class="p p-first">The genomic control (GC) inflation factor, λ, was 1.05, after adjusting for age, gender, and principal components of genetic ancestry. A Q-Q plot was generated and is presented in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material</a>, Figure S1. Upon visual inspection of the plot, the observed –log<sub>10</sub>(<em>P</em> values) do not deviate from the null, except at the extreme tail. This plot suggests proper control of population stratification in this Latino sample.
</p><p id="__p10">
<a href="/pmc/articles/PMC6078592/figure/ddw319-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw319-F1" rid-ob="ob-ddw319-F1" co-legend-rid="lgnd_ddw319-F1"><span>Figure 1</span></a> presents a Manhattan plot displaying the discovery set SNP association <em>P</em> values for the 576, 798 SNPs included in this analysis. <a href="/pmc/articles/PMC6078592/table/ddw319-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw319-T2" rid-ob="ob-ddw319-T2" co-legend-rid=""><span>Table 2</span></a> summarizes the SNPs, rs3118515, rs12447690, and rs10453441, that reached genome-wide significance (<em>P</em> &lt; 5 × 10<sup>−</sup><sup>8</sup>) during the discovery set analysis. Among these three SNPs, all have previously been associated with CCT except for rs10453441, which is a novel finding in this study. rs10453441 (<em>P </em>= 3.14 × 10<sup>−</sup><sup>8</sup>, GRCh37/hg19 position 46,363,739, chromosome 22) is located in an intron of the <em>WNT7B</em> gene (wingless-type MMTV integration site family, member 7B). The minor allele G (MAF = 0.455) is associated with a reduction in CCT (β = −4.51). rs3118515 (<em>P </em>= 1.16 × 10<sup>−</sup><sup>8</sup>, GRCh37/hg19 position 137,436,314) resides in the uncharacterized gene <em>LOC100506532</em>, which is situated 86.5 kb 3’ of the <em>RXRA</em> gene (retinoid X receptor, alpha) and 96.5 kb 5’ of the <em>COL5A1</em> gene (collagen, type V, alpha 1) on chromosome 9. The minor allele of rs3118515, A (MAF = 0.268), is associated with a reduction in CCT (β = −5.15). rs12447690 (<em>P</em> = 9.16 × 10<sup>−</sup><sup>9</sup>, GRCh37/hg19 position 88,298,124) is located 195 kb upstream of the <em>ZNF469</em> gene (zinc finger protein 469) and 187 kb downstream of the <em>BANP</em> gene (BTG3 associated nuclear protein) on chromosome 16. The minor allele of rs12447690 C (MAF = 0.360) is associated with a reduction in CCT per allele copy (β = −4.86).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddw319-F1" co-legend-rid="lgnd_ddw319-F1"><a href="/pmc/articles/PMC6078592/figure/ddw319-F1/" target="figure" rid-figpopup="ddw319-F1" rid-ob="ob-ddw319-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140380846708240" class="figure"><a href="/pmc/articles/PMC6078592/figure/ddw319-F1/" target="figure" rid-figpopup="ddw319-F1" rid-ob="ob-ddw319-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6078592_ddw319f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddw319f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6078592/bin/ddw319f1.jpg"></a></div><div id="largeobj_idm140380846708240" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6078592/figure/ddw319-F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddw319-F1"><div><a class="figpopup" href="/pmc/articles/PMC6078592/figure/ddw319-F1/" target="figure" rid-figpopup="ddw319-F1" rid-ob="ob-ddw319-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p11">A Manhattan plot presenting the –log<sub>10</sub>(<em>P</em> values) for the 576,798 SNPs included in the discovery set (stage 1) plotted by genomic position The red horizontal dotted line indicates the genome-wide significance level of <em>P</em> = 5.00 × 10<sup>−8</sup>. Loci previously associated with CCT are shown in grey (<em>LOC100506532</em> and <em>ZNF469</em>) and the novel locus is shown in black (<em>WNT7B</em>).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw319-T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p12">Single nucleotide polymorphisms significantly associated (<em>P</em> &lt; 5 × 10<sup>−</sup><sup>8</sup>) with CCT in Latinos</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140380846704400" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="2" rowspan="1">Discovery<hr></th><th colspan="2" rowspan="1">Replication<hr></th><th rowspan="1" colspan="1">Meta-Analysis<hr></th></tr><tr><th rowspan="1" colspan="1">SNP</th><th rowspan="1" colspan="1">Chr</th><th rowspan="1" colspan="1">Position</th><th rowspan="1" colspan="1">Gene</th><th rowspan="1" colspan="1">A1/A2</th><th rowspan="1" colspan="1">AF1</th><th rowspan="1" colspan="1">β</th><th rowspan="1" colspan="1"><em>P</em></th><th rowspan="1" colspan="1">β</th><th rowspan="1" colspan="1"><em>P</em>*</th><th rowspan="1" colspan="1"><em>P</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs3118515</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">137,436,314</td><td rowspan="1" colspan="1"><strong><em>LOC100506532</em></strong><em>, RXRA, COL5A1</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.268</td><td rowspan="1" colspan="1">−5.15</td><td rowspan="1" colspan="1">1.16E-08</td><td rowspan="1" colspan="1">−6.41</td><td rowspan="1" colspan="1">3.03E-04</td><td rowspan="1" colspan="1">9.29E-11</td></tr><tr><td rowspan="1" colspan="1">rs12447690</td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">88,298,124</td><td rowspan="1" colspan="1"><em>BANP, ZNF469</em></td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.360</td><td rowspan="1" colspan="1">−4.86</td><td rowspan="1" colspan="1">9.16E-09</td><td rowspan="1" colspan="1">−1.03</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">1.81E-08</td></tr><tr><td rowspan="1" colspan="1">rs10453441</td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">46,363,739</td><td rowspan="1" colspan="1"><strong><em>WNT7B</em></strong></td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.455</td><td rowspan="1" colspan="1">−4.51</td><td rowspan="1" colspan="1">3.14E-08</td><td rowspan="1" colspan="1">−2.95</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">6.01E-09</td></tr></tbody></table></div><div id="largeobj_idm140380846704400" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6078592/table/ddw319-T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="ddw319-TF1"><p id="__p13" class="p p-first-last">Abbreviations: Chr, chromosome; A1/A2, allele 1/allele 2; AF1, allele 1 frequency. β models the expected change in mean CCT per increase of one A1 allele. Gene name is in boldface if the SNP is inside the gene. SNP positions are according to GRCh37/hg19.</p></div><div id="ddw319-TF2"><sup>*</sup>One-sided <em>P</em> values computed with regard to the direction of effect found in the discovery set.</div></div></div><p id="__p15" class="p p-last">For the replication set, we analysed the top SNPs using a set of related individuals and adjusted for the relatedness among individuals using linear mixed-effects models. All three of the SNPs showed similar directionality of the effect estimates for the minor alleles, compared to the discovery set (<a href="/pmc/articles/PMC6078592/table/ddw319-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw319-T2" rid-ob="ob-ddw319-T2" co-legend-rid=""><span>Table 2</span></a>). rs10453441 was significant (<em>P</em> = 0.03) in our replicate set. The meta-analysis of the discovery and replication sets strengthened the association for rs10453441, <em>P</em> = 6.01 × 10<sup>−</sup><sup>9</sup>. Therefore, rs10453441 represents a novel genetic locus associated with CCT in Latinos. Furthermore, rs10453441 was associated with CCT in our Indian consanguineous pedigree dataset (240 individuals from 16 pedigrees, <em>P</em> = 5.85 × 10<sup>−</sup><sup>4</sup>) (<a href="#ddw319-B24" rid="ddw319-B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716916823">24</a>), providing another replication for the novel discovery. Similarly, the association result for SNP rs3118515 was also strengthened after the meta-analysis, <em>P</em> = 9.29 × 10<sup>−</sup><sup>11</sup>.</p></div><div id="__sec5" class="sec"><h3 id="__sec5title">Results from imputed SNPs</h3><p id="__p16" class="p p-first-last">To expand genomic coverage and further identify additional genetic loci associated with CCT, we used the maximum number of unrelated individuals (<em>n =</em> 4,038), including unrelated individuals from the replication set, to analyse imputed SNPs. The GC λ for the imputed dataset was 1.05. Analysis of imputed SNPs did not identify any additional GWAS loci for CCT stronger than directly genotyped SNPs. The SNP rs10453441 was the only significant SNP associated with CCT on chromosome 22 for genotyped and imputed SNPs (Rsq ≥ 0.80). <a href="/pmc/articles/PMC6078592/figure/ddw319-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw319-F2" rid-ob="ob-ddw319-F2" co-legend-rid="lgnd_ddw319-F2"><span>Figure 2</span></a> presents the regional SNP association plots for the <em>WNT7B</em> region. Genotyped and imputed SNPs are plotted as squares and circles, respectively.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddw319-F2" co-legend-rid="lgnd_ddw319-F2"><a href="/pmc/articles/PMC6078592/figure/ddw319-F2/" target="figure" rid-figpopup="ddw319-F2" rid-ob="ob-ddw319-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140380846642128" class="figure"><a href="/pmc/articles/PMC6078592/figure/ddw319-F2/" target="figure" rid-figpopup="ddw319-F2" rid-ob="ob-ddw319-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6078592_ddw319f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddw319f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6078592/bin/ddw319f2.jpg"></a></div><div id="largeobj_idm140380846642128" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6078592/figure/ddw319-F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddw319-F2"><div><a class="figpopup" href="/pmc/articles/PMC6078592/figure/ddw319-F2/" target="figure" rid-figpopup="ddw319-F2" rid-ob="ob-ddw319-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><p id="__p17">Regional SNP association plot for rs10453441 The most significant SNP, using the maximum number of unrelated individuals, is plotted in purple. The lead SNP, rs10453441 (<em>P</em> = 7.15 × 10<sup>−9</sup>), remained the most significantly associated in the <em>WNT7B</em> gene. Genotyped and imputed SNPs are plotted as squares and circles, respectively. Color-coding of SNPs represents the level of linkage disequilibrium with the most significant SNP, rs10453441. Genes in the region are shown below the SNPs with arrows indicating the strand orientation.</p></div></div></div></div><div id="__sec6" class="sec"><h3 id="__sec6title">Analysis of previously reported loci for CCT</h3><p id="__p18" class="p p-first-last">We evaluated 52 previously reported SNPs that reached genome-wide significance (<em>P </em>&lt; 5 × 10<sup>−</sup><sup>8</sup>) in European and Asian populations to determine whether these associations were consistent in a Latino population. The results are summarized in <a href="/pmc/articles/PMC6078592/table/ddw319-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw319-T3" rid-ob="ob-ddw319-T3" co-legend-rid=""><span>Table 3</span></a>. Of the 52 previously reported SNPs, 51 SNPs were either directly genotyped or imputed well and among these SNPs, a majority of SNPs had a <em>P </em>&lt; 0.05 (38/51 = 75%). Furthermore, in our Latino population, all except one SNP exhibited consistent directionality of the effect with these previously reported SNPs. To account for multiple testing among correlated SNPs, we employed <em>simpleM</em> (<a href="#ddw319-B25" rid="ddw319-B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593746">25</a>,<a href="#ddw319-B26" rid="ddw319-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593771">26</a>) to estimate the number of independent tests for the 51 SNPs. This method identified 40 independent tests, resulting in a Bonferroni corrected <em>P</em> value of 0.00125. Using this corrected <em>P</em> value, we replicated 28 SNPs in the following 11 loci, <em>NR3C2</em>, <em>IBTK</em>, <em>C7orf42</em>, <em>LPAR1</em>, <em>RXRA-COL5A1</em>, <em>COL5A1</em>, <em>LCN12-PTGDS</em>, <em>ARHGAP20-POU2AF1</em>, <em>FOXO1</em>, <em>LRRK1</em>, and <em>ZNF469</em> regions (boldface <em>P</em> values in <a href="/pmc/articles/PMC6078592/table/ddw319-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw319-T3" rid-ob="ob-ddw319-T3" co-legend-rid=""><span>Table 3</span></a>). We were, however, unable to replicate previous index SNPs in other regions, including <em>COL8A2</em>, <em>COL4A3</em>, <em>FAM53B</em>, <em>SMAD3</em>, and <em>AKAP13</em>. Intriguingly, a study conducted in individuals of European ancestry was unable to replicate several of these associations (<a href="#ddw319-B27" rid="ddw319-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593743">27</a>); however, this later study was restricted to NEIGHBOR and GLAUGEN POAG cases and controls, suggesting that not all CCT loci are POAG related.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw319-T3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p19">Comparison with previously reported SNPs associated with CCT</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140380846617840" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="3" rowspan="1">Previously Reported<hr></th><th rowspan="1" colspan="1"></th><th colspan="5" rowspan="1">Latinos<hr></th><th rowspan="1" colspan="1"></th></tr><tr><th rowspan="2" colspan="1">SNP ID</th><th rowspan="2" colspan="1">Chr</th><th rowspan="2" colspan="1">Position</th><th rowspan="2" colspan="1">Genes Nearby</th><th rowspan="1" colspan="1">Effect</th><th rowspan="2" colspan="1">Freq</th><th rowspan="2" colspan="1">β</th><th rowspan="2" colspan="1">Reference(s)</th><th rowspan="2" colspan="1">A1/A2</th><th rowspan="2" colspan="1">AF1</th><th rowspan="2" colspan="1">β</th><th rowspan="2" colspan="1"><em>P</em></th><th rowspan="2" colspan="1">Imputed</th><th rowspan="2" colspan="1">Consistency of Direction</th></tr><tr><th rowspan="1" colspan="1">Allele</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs3767703</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">36555758</td><td rowspan="1" colspan="1" align="center"><em>COL8A2</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">−4.43</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593792">19</a>)</td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">−0.03</td><td rowspan="1" colspan="1" align="center">9.81E-01</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs7550047</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">36567343</td><td rowspan="1" colspan="1" align="center"><em>COL8A2</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">−4.42</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593818">19</a>)</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">0.88</td><td rowspan="1" colspan="1" align="center">5.00E-01</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs96067</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">36571920</td><td rowspan="1" colspan="1" align="center"><em>COL8A2</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">−4.80</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593855">19</a>)</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.38</td><td rowspan="1" colspan="1" align="center">−1.15</td><td rowspan="1" colspan="1" align="center">1.29E-01</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs10189064</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">219327500</td><td rowspan="1" colspan="1" align="center"><em>USP37</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.04</td><td rowspan="1" colspan="1" align="center">−0.19</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593834">23</a>)</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">-</td><td rowspan="1" colspan="1" align="center">-</td><td rowspan="1" colspan="1" align="center">-</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs7606754</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">228135180</td><td rowspan="1" colspan="1" align="center"><em>COL4A3</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.35</td><td rowspan="1" colspan="1" align="center">−0.07</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593772">23</a>)</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.41</td><td rowspan="1" colspan="1" align="center">−0.79</td><td rowspan="1" colspan="1" align="center">2.91E-01</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs3749260</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">98250862</td><td rowspan="1" colspan="1" align="center"><em>GPR15</em></td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.13</td><td rowspan="1" colspan="1" align="center">−0.12</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593759">23</a>)</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center">1.76</td><td rowspan="1" colspan="1" align="center">1.12E-01</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">N</td></tr><tr><td rowspan="1" colspan="1">rs9822953</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">156472071</td><td rowspan="1" colspan="1" align="center"><em>TIPARP</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.67</td><td rowspan="1" colspan="1" align="center">0.08</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593779">23</a>)</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.74</td><td rowspan="1" colspan="1" align="center">1.81</td><td rowspan="1" colspan="1" align="center">3.51E-02</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs4894535</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">171995605</td><td rowspan="1" colspan="1" align="center"><em>FNDC3B</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="center">−0.10</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593776">23</a>)</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.25</td><td rowspan="1" colspan="1" align="center">−2.16</td><td rowspan="1" colspan="1" align="center">1.09E-02</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs7620503</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">177304298</td><td rowspan="1" colspan="1" align="center"><em>TBL1XR1-KCNMB2</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.39</td><td rowspan="1" colspan="1" align="center">−0.06</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593842">23</a>)</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.44</td><td rowspan="1" colspan="1" align="center">−1.18</td><td rowspan="1" colspan="1" align="center">1.16E-01</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs3931397</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">149079497</td><td rowspan="1" colspan="1" align="center"><em>NR3C2</em></td><td rowspan="1" colspan="1" align="center">T/G</td><td rowspan="1" colspan="1" align="center">0.07</td><td rowspan="1" colspan="1" align="center">−0.12</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593819">23</a>)</td><td rowspan="1" colspan="1" align="center">T/G</td><td rowspan="1" colspan="1" align="center">0.06</td><td rowspan="1" colspan="1" align="center">−6.77</td><td rowspan="1" colspan="1" align="center"><strong>1.28E-05</strong></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs1117707</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">64389665</td><td rowspan="1" colspan="1" align="center"><em>CWC27-ADAMTS6</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.7</td><td rowspan="1" colspan="1" align="center">−0.11</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593807">23</a>)</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.55</td><td rowspan="1" colspan="1" align="center">−0.59</td><td rowspan="1" colspan="1" align="center">4.26E-01</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs1538138</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">82794594</td><td rowspan="1" colspan="1" align="center"><em>IBTK</em></td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">0.18–0.30</td><td rowspan="1" colspan="1" align="center">−4.23</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B17" rid="ddw319-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593803">17</a>)</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center">−3.53</td><td rowspan="1" colspan="1" align="center"><strong>7.16E-04</strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs11763147</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">65326821</td><td rowspan="1" colspan="1" align="center"><em>VKORC1L1</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.45</td><td rowspan="1" colspan="1" align="center">0.07</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593757">23</a>)</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.45</td><td rowspan="1" colspan="1" align="center">1.62</td><td rowspan="1" colspan="1" align="center">2.85E-02</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs4718428</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">66421446</td><td rowspan="1" colspan="1" align="center"><em>C7orf42</em></td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.46–0.74</td><td rowspan="1" colspan="1" align="center">−3.18</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B17" rid="ddw319-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593750">17</a>)</td><td rowspan="1" colspan="1" align="center">G/T</td><td rowspan="1" colspan="1" align="center">0.45</td><td rowspan="1" colspan="1" align="center">−2.89</td><td rowspan="1" colspan="1" align="center"><strong>1.89E-04</strong></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs1324183</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">13557491</td><td rowspan="1" colspan="1" align="center"><em>9p23</em></td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.21–0.27</td><td rowspan="1" colspan="1" align="center">−3.37</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B17" rid="ddw319-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593800">17</a>)</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center">−2.86</td><td rowspan="1" colspan="1" align="center">1.15E-02</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs1007000</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">113662681</td><td rowspan="1" colspan="1" align="center"><em>LPAR1</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.22</td><td rowspan="1" colspan="1" align="center">0.07</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593782">23</a>)</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.30</td><td rowspan="1" colspan="1" align="center">3.43</td><td rowspan="1" colspan="1" align="center"><strong>1.62E-05</strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs1409832</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">137428425</td><td rowspan="1" colspan="1" align="center"><em>RXRA-COL5A1</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">−3.95</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593752">19</a>)</td><td rowspan="1" colspan="1" align="center">T/G</td><td rowspan="1" colspan="1" align="center">0.76</td><td rowspan="1" colspan="1" align="center">4.32</td><td rowspan="1" colspan="1" align="center"><strong>7.11E-07</strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs4842044</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">137431904</td><td rowspan="1" colspan="1" align="center"><em>RXRA-COL5A1</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">−4.67</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593744">19</a>)</td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.47</td><td rowspan="1" colspan="1" align="center">−4.14</td><td rowspan="1" colspan="1" align="center"><strong>2.25E-08</strong></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs1536478</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">137432248</td><td rowspan="1" colspan="1" align="center"><em>RXRA-COL5A1</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">−4.63</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593840">19</a>)</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.47</td><td rowspan="1" colspan="1" align="center">−4.17</td><td rowspan="1" colspan="1" align="center"><strong>2.03E-08</strong></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs3118516</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">137439792</td><td rowspan="1" colspan="1" align="center"><em>RXRA-COL5A1</em></td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.34</td><td rowspan="1" colspan="1" align="center">−0.15</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B21" rid="ddw319-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593780">21</a>)</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.26</td><td rowspan="1" colspan="1" align="center">−4.91</td><td rowspan="1" colspan="1" align="center"><strong>5.89E-07</strong></td><td rowspan="1" colspan="1" align="center">Y*</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs3132306</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">137440212</td><td rowspan="1" colspan="1" align="center"><em>RXRA-COL5A1</em></td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">0.66</td><td rowspan="1" colspan="1" align="center">0.15</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B21" rid="ddw319-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593761">21</a>)</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.72</td><td rowspan="1" colspan="1" align="center">4.83</td><td rowspan="1" colspan="1" align="center"><strong>7.63E-07</strong></td><td rowspan="1" colspan="1" align="center">Y*</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="2" colspan="1">rs1536482</td><td rowspan="2" colspan="1" align="center">9</td><td rowspan="2" colspan="1" align="center">137440528</td><td rowspan="2" colspan="1" align="center"><em>RXRA-COL5A1</em></td><td rowspan="2" colspan="1" align="center">G</td><td rowspan="2" colspan="1" align="center">0.34</td><td rowspan="2" colspan="1" align="center">0.22</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B18" rid="ddw319-B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593849">18</a>)<sup>,</sup>(<a href="#ddw319-B21" rid="ddw319-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593742">21</a>) (A, Freq</td><td rowspan="2" colspan="1" align="center">G/A</td><td rowspan="2" colspan="1" align="center">0.71</td><td rowspan="2" colspan="1" align="center">5.21</td><td rowspan="2" colspan="1" align="center"><strong>5.47E-08</strong></td><td rowspan="2" colspan="1" align="center">Y*</td><td rowspan="2" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">= 0.33, β = −0.15)</td></tr><tr><td rowspan="1" colspan="1">rs7044529</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">137568051</td><td rowspan="1" colspan="1" align="center"><em>COL5A1</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593748">19</a>)</td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.78</td><td rowspan="1" colspan="1" align="center">3.23</td><td rowspan="1" colspan="1" align="center"><strong>2.58E-04</strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs11145951</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">139860264</td><td rowspan="1" colspan="1" align="center"><em>LCN12-PTGDS</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.49</td><td rowspan="1" colspan="1" align="center">0.09</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593758">23</a>)</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.44</td><td rowspan="1" colspan="1" align="center">3.26</td><td rowspan="1" colspan="1" align="center"><strong>1.15E-05</strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs7090871</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">63830286</td><td rowspan="1" colspan="1" align="center"><em>ARID5B</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.59</td><td rowspan="1" colspan="1" align="center">0.06</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593815">23</a>)</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.71</td><td rowspan="1" colspan="1" align="center">1.40</td><td rowspan="1" colspan="1" align="center">8.75E-02</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs1006368</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">126346603</td><td rowspan="1" colspan="1" align="center"><em>FAM53B</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B20" rid="ddw319-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593784">20</a>)</td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.80</td><td rowspan="1" colspan="1" align="center">−2.01</td><td rowspan="1" colspan="1" align="center">2.59E-02</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs11245330</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">126380338</td><td rowspan="1" colspan="1" align="center"><em>FAM53B</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B20" rid="ddw319-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593741">20</a>)</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.20</td><td rowspan="1" colspan="1" align="center">2.10</td><td rowspan="1" colspan="1" align="center">1.96E-02</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs4938174</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">110913240</td><td rowspan="1" colspan="1" align="center"><em>ARHGAP20-POU2AF1</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.31</td><td rowspan="1" colspan="1" align="center">0.06</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593736">23</a>)</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.20</td><td rowspan="1" colspan="1" align="center">3.34</td><td rowspan="1" colspan="1" align="center"><strong>2.97E-04</strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs1564892</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">104445742</td><td rowspan="1" colspan="1" align="center"><em>Near GLT8D2 (5')</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.76</td><td rowspan="1" colspan="1" align="center">−0.08</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593793">23</a>)</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.74</td><td rowspan="1" colspan="1" align="center">−0.37</td><td rowspan="1" colspan="1" align="center">6.66E-01</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs1034200</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="center">23228691</td><td rowspan="1" colspan="1" align="center"><em>FGF9-FTHL7</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B18" rid="ddw319-B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593774">18</a>)</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.75</td><td rowspan="1" colspan="1" align="center">−1.69</td><td rowspan="1" colspan="1" align="center">5.06E-02</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs2755237</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="center">41109429</td><td rowspan="1" colspan="1" align="center"><em>FOXO1</em></td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B20" rid="ddw319-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593767">20</a>)</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.86</td><td rowspan="1" colspan="1" align="center">3.62</td><td rowspan="1" colspan="1" align="center"><strong>6.45E-04</strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs2721051</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="center">41110884</td><td rowspan="1" colspan="1" align="center"><em>FOXO1</em></td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B20" rid="ddw319-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593808">20</a>)</td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">4.24</td><td rowspan="1" colspan="1" align="center">1.89E-03</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs785422</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">30173885</td><td rowspan="1" colspan="1" align="center"><em>Near TJP1 (5')</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.11</td><td rowspan="1" colspan="1" align="center">−0.14</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593809">23</a>)</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.07</td><td rowspan="1" colspan="1" align="center">−3.24</td><td rowspan="1" colspan="1" align="center">2.66E-02</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs12913547</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">67467507</td><td rowspan="1" colspan="1" align="center"><em>SMAD3</em></td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.77</td><td rowspan="1" colspan="1" align="center">−0.08</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593846">23</a>)</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.71</td><td rowspan="1" colspan="1" align="center">−0.44</td><td rowspan="1" colspan="1" align="center">5.87E-01</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs6496932</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">85825567</td><td rowspan="1" colspan="1" align="center"><em>PDE8A-AKAP13</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.13</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B18" rid="ddw319-B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593787">18</a>)</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.77</td><td rowspan="1" colspan="1" align="center">1.53</td><td rowspan="1" colspan="1" align="center">8.32E-02</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs1828481</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">85840912</td><td rowspan="1" colspan="1" align="center"><em>AKAP13</em></td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">0.45–0.56</td><td rowspan="1" colspan="1" align="center">3.12</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B17" rid="ddw319-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593835">17</a>)</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.49</td><td rowspan="1" colspan="1" align="center">1.88</td><td rowspan="1" colspan="1" align="center">1.21E-02</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs7172789</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">85843517</td><td rowspan="1" colspan="1" align="center"><em>AKAP13</em></td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">0.45–0.56</td><td rowspan="1" colspan="1" align="center">3.14</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B17" rid="ddw319-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593770">17</a>)</td><td rowspan="1" colspan="1" align="center">C/T</td><td rowspan="1" colspan="1" align="center">0.48</td><td rowspan="1" colspan="1" align="center">1.72</td><td rowspan="1" colspan="1" align="center">2.14E-02</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs930847</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">101558562</td><td rowspan="1" colspan="1" align="center"><em>LRRK1</em></td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.17–0.39</td><td rowspan="1" colspan="1" align="center">3.72</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B17" rid="ddw319-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593751">17</a>)</td><td rowspan="1" colspan="1" align="center">G/T</td><td rowspan="1" colspan="1" align="center">0.19</td><td rowspan="1" colspan="1" align="center">3.97</td><td rowspan="1" colspan="1" align="center"><strong>2.79E-05</strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs4965359</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">101585336</td><td rowspan="1" colspan="1" align="center"><em>LRRK1</em></td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.40–0.67</td><td rowspan="1" colspan="1" align="center">−3.50</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B17" rid="ddw319-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593799">17</a>)</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.54</td><td rowspan="1" colspan="1" align="center">−3.77</td><td rowspan="1" colspan="1" align="center"><strong>4.32E-07</strong></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="2" colspan="1">rs12447690</td><td rowspan="2" colspan="1" align="center">16</td><td rowspan="2" colspan="1" align="center">88298124</td><td rowspan="2" colspan="1" align="center"><em>ZNF469</em></td><td rowspan="2" colspan="1" align="center">T/C</td><td rowspan="2" colspan="1" align="center"></td><td rowspan="2" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B20" rid="ddw319-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593778">20</a>)<sup>,</sup>(<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593825">19</a>)<sup>,</sup>(<a href="#ddw319-B18" rid="ddw319-B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593843">18</a>) (G,</td><td rowspan="2" colspan="1" align="center">T/C</td><td rowspan="2" colspan="1" align="center">0.64</td><td rowspan="2" colspan="1" align="center">4.14</td><td rowspan="2" colspan="1" align="center"><strong>1.18E-07</strong></td><td rowspan="2" colspan="1" align="center"></td><td rowspan="2" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">β = −0.18), (<a href="#ddw319-B21" rid="ddw319-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593802">21</a>) (T, Freq = 0.64, β = 0.16)</td></tr><tr><td rowspan="1" colspan="1">rs7500824</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">88299491</td><td rowspan="1" colspan="1" align="center"><em>ZNF469</em></td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.36</td><td rowspan="1" colspan="1" align="center">−0.16</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B21" rid="ddw319-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593801">21</a>)</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.24</td><td rowspan="1" colspan="1" align="center">−4.70</td><td rowspan="1" colspan="1" align="center"><strong>6.35E-07</strong></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs7405095</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">88307825</td><td rowspan="1" colspan="1" align="center"><em>ZNF469</em></td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.36</td><td rowspan="1" colspan="1" align="center">−0.16</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B21" rid="ddw319-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593795">21</a>)</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.20</td><td rowspan="1" colspan="1" align="center">−6.02</td><td rowspan="1" colspan="1" align="center"><strong>1.87E-07</strong></td><td rowspan="1" colspan="1" align="center">Y*</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs7501109</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">88320862</td><td rowspan="1" colspan="1" align="center"><em>ZNF469</em></td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">0.64</td><td rowspan="1" colspan="1" align="center">0.16</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B21" rid="ddw319-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593829">21</a>)</td><td rowspan="1" colspan="1" align="center">C/G</td><td rowspan="1" colspan="1" align="center">0.80</td><td rowspan="1" colspan="1" align="center">4.95</td><td rowspan="1" colspan="1" align="center"><strong>2.15E-06</strong></td><td rowspan="1" colspan="1" align="center">Y*</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs7501402</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">88320911</td><td rowspan="1" colspan="1" align="center"><em>ZNF469</em></td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.36</td><td rowspan="1" colspan="1" align="center">−0.16</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B21" rid="ddw319-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593850">21</a>)</td><td rowspan="1" colspan="1" align="center">A/T</td><td rowspan="1" colspan="1" align="center">0.36</td><td rowspan="1" colspan="1" align="center">−4.38</td><td rowspan="1" colspan="1" align="center"><strong>2.94E-07</strong></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs6540223</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">88321436</td><td rowspan="1" colspan="1" align="center"><em>ZNF469</em></td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">0.64</td><td rowspan="1" colspan="1" align="center">0.16</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B21" rid="ddw319-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593841">21</a>)</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.80</td><td rowspan="1" colspan="1" align="center">4.96</td><td rowspan="1" colspan="1" align="center"><strong>2.22E-06</strong></td><td rowspan="1" colspan="1" align="center">Y*</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs12448211</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">88330513</td><td rowspan="1" colspan="1" align="center"><em>ZNF469</em></td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.62</td><td rowspan="1" colspan="1" align="center">0.16</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B21" rid="ddw319-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593851">21</a>)</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.65</td><td rowspan="1" colspan="1" align="center">4.51</td><td rowspan="1" colspan="1" align="center"><strong>3.89E-08</strong></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="2" colspan="1">rs9938149</td><td rowspan="2" colspan="1" align="center">16</td><td rowspan="2" colspan="1" align="center">88331640</td><td rowspan="2" colspan="1" align="center"><em>ZNF469</em></td><td rowspan="2" colspan="1" align="center">A/C</td><td rowspan="2" colspan="1" align="center"></td><td rowspan="2" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B20" rid="ddw319-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593853">20</a>)<sup>,</sup>(<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593756">19</a>)<sup>,</sup>(<a href="#ddw319-B21" rid="ddw319-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593830">21</a>) (A,</td><td rowspan="2" colspan="1" align="center">A/C</td><td rowspan="2" colspan="1" align="center">0.79</td><td rowspan="2" colspan="1" align="center">4.56</td><td rowspan="2" colspan="1" align="center"><strong>3.62E-07</strong></td><td rowspan="2" colspan="1" align="center"></td><td rowspan="2" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">Freq = 0.62, β = 0.16)</td></tr><tr><td rowspan="1" colspan="1">rs9922572</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">88334112</td><td rowspan="1" colspan="1" align="center"><em>ZNF469</em></td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.34</td><td rowspan="1" colspan="1" align="center">−0.14</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B21" rid="ddw319-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593839">21</a>)</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.20</td><td rowspan="1" colspan="1" align="center">−4.82</td><td rowspan="1" colspan="1" align="center"><strong>2.01E-06</strong></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr><tr><td rowspan="1" colspan="1">rs9925231</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">88338107</td><td rowspan="1" colspan="1" align="center"><em>ZNF469</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">−4.79</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593796">19</a>)</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.33</td><td rowspan="1" colspan="1" align="center">−4.37</td><td rowspan="1" colspan="1" align="center"><strong>4.33E-07</strong></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs7204132</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">88344517</td><td rowspan="1" colspan="1" align="center"><em>ZNF469</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">−4.95</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593821">19</a>)</td><td rowspan="1" colspan="1" align="center">T/G</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center">−4.32</td><td rowspan="1" colspan="1" align="center"><strong>4.35E-05</strong></td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs9927272</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">88346709</td><td rowspan="1" colspan="1" align="center"><em>ZNF469</em></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">−3.95</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B19" rid="ddw319-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593754">19</a>)</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.34</td><td rowspan="1" colspan="1" align="center">−2.72</td><td rowspan="1" colspan="1" align="center"><strong>5.68E-04</strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1">rs2323457</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="center">14554190</td><td rowspan="1" colspan="1" align="center"><em>HS3ST3B1-PMP22</em></td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.29</td><td rowspan="1" colspan="1" align="center">−0.07</td><td rowspan="1" colspan="1" align="center">(<a href="#ddw319-B23" rid="ddw319-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593777">23</a>)</td><td rowspan="1" colspan="1" align="center">A/C</td><td rowspan="1" colspan="1" align="center">0.40</td><td rowspan="1" colspan="1" align="center">−0.89</td><td rowspan="1" colspan="1" align="center">2.32E-01</td><td rowspan="1" colspan="1" align="center">Y</td><td rowspan="1" colspan="1" align="center">Y</td></tr></tbody></table></div><div id="largeobj_idm140380846617840" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6078592/table/ddw319-T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="ddw319-TF3"><p id="__p20" class="p p-first-last">Abbreviations: Chr, chromosome; Freq, frequency; A1/A2, allele 1/allele 2. For previously reported SNPs with multiple references, additional information on the allele, frequency, and effect size is given in parentheses. For this Latino population, the frequency for allele 1 is given and is modeled as the effect allele. The program <em>simpleM</em> was used as a multiple testing correction method for correlated SNPs, identifying 40 independent tests and thus, giving a Bonferroni correction <em>P</em> value of 0.05/40 = 0.00125. <em>P</em> values meeting this threshold are shown in bold. The direction of effect was consistent for most SNPs, except rs3749260. SNP positions are according to GRCh37/hg19.</p></div><div id="ddw319-TF4"><p id="__p21" class="p p-first-last">*SNPs with suboptimal Rsq (0.62 &lt; Rsq &lt; 0.78).</p></div></div></div></div><div id="__sec7" class="sec"><h3 id="__sec7title">Biological evidence</h3><p id="__p22" class="p p-first-last">The <em>WNT7B</em> gene has been reported to be expressed in the human cornea (<a href="#ddw319-B28" rid="ddw319-B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593812">28</a>) and the mouse cornea (<a href="#ddw319-B29" rid="ddw319-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593747">29</a>). <em>WNT7B</em> was upregulated in the human central cornea and is specific to the mature corneal epithelium (<a href="#ddw319-B28" rid="ddw319-B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593856">28</a>) and was mainly expressed in the endothelial cell layer in the mouse cornea (<a href="#ddw319-B29" rid="ddw319-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593790">29</a>). Genevestigator (see Web Resources), an online database to verify gene expression in different tissue types, also shows that <em>WNT7B</em> is highly expressed in normal human corneas (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material</a>, <a href="/pmc/articles/PMC6078592/figure/ddw319-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw319-F2" rid-ob="ob-ddw319-F2" co-legend-rid="lgnd_ddw319-F2"><span>Fig. S2</span></a>).</p></div><div id="__sec8" class="sec sec-last"><h3 id="__sec8title">Pathway analysis</h3><p id="__p23" class="p p-first-last">
<a href="/pmc/articles/PMC6078592/table/ddw319-T4/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw319-T4" rid-ob="ob-ddw319-T4" co-legend-rid=""><span>Table 4</span></a> summarizes the top five canonical pathways from IPA and KEGG pathways from WebGestalt through gene-set association testing of directly genotyped SNPs. The most significantly enriched canonical pathway is 14-3-3-mediated signalling (<em>P</em> = 2.9E-03), followed by the IGF-1 signalling (<em>P</em> = 2.9E-03), myc mediated apoptosis signalling (<em>P</em> = 6.45E-03), neuregulin signalling (<em>P</em> = 6.45E-03), and NGF signalling (<em>P</em> = 3.8E-02) pathways, respectively. The top pathway, 14-3-3-mediated signalling, is a family of proteins that are involved in cell cycle, apoptosis, and protein trafficking and the presence of 14-3-3 proteins have been reported in the human cornea (<a href="#ddw319-B30" rid="ddw319-B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593794">30</a>). Similarly, the Myc mediated apoptosis signalling pathway is related to the cell cycle and may have a role in maintaining the homeostasis of the corneal epithelium. The IGF-1 signalling, neuregulin signalling, and NGF signalling pathways involve growth factors and IGF-1 (<a href="#ddw319-B31" rid="ddw319-B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593820">31</a>), neuregulin 1 (<a href="#ddw319-B32" rid="ddw319-B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593813">32</a>), and NGF (<a href="#ddw319-B33" rid="ddw319-B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593833">33</a>) have been reported to have a biological effect in human corneas. In addition, the top KEGG pathways are metabolic pathways (<em>P</em> = 7.27E-06), ErbB signalling pathway (<em>P</em> = 2.95E-05), tight junction (<em>P</em> = 1.0E-04), pathways in cancer (<em>P</em> = 1.0E-04), and T cell receptor signalling pathway (<em>P</em> = 8.0E-04). The ErbB family consists of receptor tyrosine kinases that regulates various biological responses, including cell proliferation, differentiation, migration, and carcinogenesis, and this family has been shown to play a role in corneal epithelial wound healing (<a href="#ddw319-B34" rid="ddw319-B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593804">34</a>). Tight junctions are composed of proteins that mediate cell adhesion and serve as diffusion barriers with the tight junction-related proteins occludin, claudin, and ZO-1 expressed in human corneal epithelium (<a href="#ddw319-B35" rid="ddw319-B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593753">35</a>). And lastly, genes that have been studied in tumour metastasis models were also found to be differentially expressed in the cornea (<a href="#ddw319-B36" rid="ddw319-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593845">36</a>). Interestingly, during both wound healing and cancer development, related cells exhibit rapid proliferation and tissue remodelling (<a href="#ddw319-B36" rid="ddw319-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593773">36</a>). Taken together, our pathway analysis results from IPA and KEGG yielded comparable pathways with regard to the cornea in that the programs identified pathways relevant to cellular proliferation, migration, and apoptosis, suggesting such pathways may have a role in the integrity and homeostasis of the human cornea.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw319-T4"><h3>Table 4.</h3><!--caption a7--><div class="caption"><p id="__p24">Top pathways associated with CCT in Latinos</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140380845754704" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1">Pathway</th><th rowspan="1" colspan="1">Observed/Total Genes</th><th rowspan="1" colspan="1"><em>P</em></th><th rowspan="1" colspan="1">Adjusted <em>P</em><sup>‡</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">IPA</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">14-3-3-mediated signalling</td><td rowspan="1" colspan="1">19/117</td><td rowspan="1" colspan="1">1.30E-05</td><td rowspan="1" colspan="1">2.9E-03</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">IGF-1 signalling</td><td rowspan="1" colspan="1">17/97</td><td rowspan="1" colspan="1">1.31E-05</td><td rowspan="1" colspan="1">2.9E-03</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">Myc mediated apoptosis signalling</td><td rowspan="1" colspan="1">12/58</td><td rowspan="1" colspan="1">4.47E-05</td><td rowspan="1" colspan="1">6.45E-03</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">Neuregulin signalling</td><td rowspan="1" colspan="1">15/88</td><td rowspan="1" colspan="1">5.81E-05</td><td rowspan="1" colspan="1">6.45E-03</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">NGF signalling</td><td rowspan="1" colspan="1">15/107</td><td rowspan="1" colspan="1">5.38E-04</td><td rowspan="1" colspan="1">3.8E-02</td></tr><tr><td rowspan="1" colspan="1">WebGestalt</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">Metabolic pathways</td><td rowspan="1" colspan="1">69/1130</td><td rowspan="1" colspan="1">4.49E-08</td><td rowspan="1" colspan="1">7.27E-06</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">ErbB signalling pathway</td><td rowspan="1" colspan="1">14/87</td><td rowspan="1" colspan="1">3.64E-07</td><td rowspan="1" colspan="1">2.95E-05</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">Tight junction</td><td rowspan="1" colspan="1">16/132</td><td rowspan="1" colspan="1">2.77E-06</td><td rowspan="1" colspan="1">1.0E-04</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">Pathways in cancer</td><td rowspan="1" colspan="1">27/326</td><td rowspan="1" colspan="1">3.06E-06</td><td rowspan="1" colspan="1">1.0E-04</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">T cell receptor signalling pathway</td><td rowspan="1" colspan="1">13/108</td><td rowspan="1" colspan="1">2.51E-05</td><td rowspan="1" colspan="1">8.0E-04</td></tr></tbody></table></div><div id="largeobj_idm140380845754704" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6078592/table/ddw319-T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="ddw319-TF5"><p id="__p25" class="p p-first-last"><sup>‡</sup><em>P</em>-values are Benjamini-Hochberg adjusted. The top five canonical pathways from IPA and KEGG pathways from WebGestalt are displayed.</p></div></div></div></div></div><div id="__sec9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec9title">Discussion</h2><p id="__p26" class="p p-first">This study is the largest GWAS conducted in Latinos for CCT. We identified rs10453441 in the <em>WNT7B</em> gene on chromosome 22 as a novel SNP for CCT. Furthermore, one previously reported SNP on chromosome 9, rs3118515, and one previously reported SNP on chromosome 16, rs12447690, also reached genome-wide significance. These SNPs were replicated at the 0.05 significance level (one-sided <em>P</em> values being computed with regard to the direction of the effect found in the discovery set) in our Latino replication set, except for rs12447690. The association of rs10453441 with CCT was also replicated in an Indian dataset (<em>P</em> = 5.85 × 10<sup>−</sup><sup>4</sup>) (<a href="#ddw319-B24" rid="ddw319-B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716916822">24</a>). We were also able to replicate 11 previous GWAS loci, such as variants in the <em>RXRA-COL5A1</em>, <em>COL5A1</em>, <em>IBTK</em>, <em>FOXO1</em>, <em>LRRK1</em>, and <em>ZNF469</em> regions. Though genotype imputation using the 1KGP increased the number of SNPs in our dataset, we did not identify additional GWAS loci for CCT in the imputed dataset. As further support, the <em>WNT7B</em> gene is expressed in the human cornea, which provides biological evidence for its association with CCT.</p><p id="__p27">The novel SNP identified here associated with CCT, rs10453441 is located in the genomic region that includes <em>WNT7B</em>, a member of the WNT gene family, a group of structurally related genes that encode secreted signalling proteins. This SNP has previously been positively associated with axial length and corneal curvature, ocular parameters related to myopia, in a GWAS among Asian and Caucasian populations (<a href="#ddw319-B29" rid="ddw319-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593832">29</a>). These investigators further identified a positive association between this SNP and extreme myopia, as well as localized <em>WNT7B</em> expression in the endothelial cell layer of mouse cornea and the downregulation of this gene in extreme myopia. Compared to Miyake et al. (2014), where rs10453441 was associated with an increase in both axial length and corneal curvature, our results identified its association with CCT. CCT is not highly correlated with either axial length or corneal curvature (<a href="#ddw319-B37" rid="ddw319-B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716916820">37</a>), suggesting rs10453441 may influence different ocular phenotypes in different ethnic groups.</p><p id="__p28">As a member of Wnt signalling and one of 19 Wnt proteins, <em>WNT7B</em> was found to be upregulated in the central cornea of humans and the Wnt signalling pathway may have a role in primary limbal stem cell proliferation (<a href="#ddw319-B28" rid="ddw319-B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593763">28</a>). Similarly, various other components of Wnt signalling have previously been implicated in the cornea. Wnt4 has been detected in the basal and immediate parabasal limbal epithelial cells in both normal foetal and adult human corneas (<a href="#ddw319-B38" rid="ddw319-B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593755">38</a>). Additionally, during corneal wound healing, Wnt7a was found to promote the proliferation of corneal epithelial cells (<a href="#ddw319-B39" rid="ddw319-B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593760">39</a>). The results from our study suggest rs10453441 may play an important role in CCT determination, independent of axial length and corneal curvature, and similar to other members of Wnt signalling, <em>WNT7B</em> may have an active biological role in regulating CCT.</p><p id="__p29">Our study has many strengths, including the use of the largest Latino dataset containing both ophthalmic and genotyped data thus far. We conducted a two stage GWAS, using both population- and family-based study designs to replicate associated loci. Our sample size was able to achieve 80% power to detect variants with MAF 0.455 explaining 1% (like rs10453441) of trait variation (two sided test, <em>α<sub>p</sub></em>= 5 × 10<sup>−</sup><sup>8</sup>, calculated by Quanto (<a href="#ddw319-B40" rid="ddw319-B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593798">40</a>,<a href="#ddw319-B41" rid="ddw319-B41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593852">41</a>)). We imputed and analysed SNPs from the 1KGP to extend genomic coverage to further identify common variants not directly genotyped, using a specialized imputation protocol for Latino populations (<a href="#ddw319-B42" rid="ddw319-B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593814">42</a>). Both the imputed and genotyped GWAS results are presented in the <a href="#sup1" rid="sup1" class=" supplementary-material">supplementary data</a>. We also carried out imputation using our own Hispanic/SOL data as the reference panel and analysed these SNPs using linear mixed models, though we did not identify any additional new loci for CCT (data not shown). Gene expression analysis provided biological support for the expression of the <em>WNT7B</em> gene in the cornea. This study, however, also has some limitations. Latinos are historically understudied in genetic research. Despite the largest Latino data for studying ophthalmic phenotypes, our sample size is limited compared to meta-analyses using European and Asian subjects. We emphasize the need to replicate our findings in an independent Latino cohort. Variants of smaller effects may also not be detected. The effect size estimates of GWAS hits may suffer from the winner’s curse (<a href="#ddw319-B43" rid="ddw319-B43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593837">43</a>,<a href="#ddw319-B44" rid="ddw319-B44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593831">44</a>). For example, the bias-corrected beta for rs12447690 in the discovery set was −4.14 with a confidence interval [−2.06, −8.34] calculated using BR-squared software (<a href="#ddw319-B45" rid="ddw319-B45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593762">45</a>) via 1,000 bootstrap re-sampling. Due to the three-way admixture (the ancestral composition of this sample is shown in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Fig. S3</a>), it is generally more challenging to impute in Latinos (<a href="#ddw319-B42" rid="ddw319-B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593823">42</a>). The imputation quality of the <em>WNT7B</em> region on chromosome 22 was low, which can mask other possible interesting SNPs, e.g. rs9330813 (<em>P</em> = 5.14 × 10<sup>−</sup><sup>8</sup>) and rs10453458 (<em>P</em> = 1.29 × 10<sup>−</sup><sup>8</sup>) only had imputation scores of Rsq = 0.74 and 0.56, respectively. Chromosome 22 has previously been shown to be difficult to impute for Latinos (<a href="#ddw319-B42" rid="ddw319-B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593806">42</a>). As reference panels and imputation technologies continue to improve for Latinos, the ability to impute and consequently, identify associations with non-directly genotyped SNPs, may improve as well. We investigated association for the top SNP (rs10453441) in 5 independent European CCT studies, but did not observe a consistent association with this SNP and CCT in these populations (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material</a>, <a href="/pmc/articles/PMC6078592/table/ddw319-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw319-T1" rid-ob="ob-ddw319-T1" co-legend-rid=""><span>Table S1</span></a>). This suggests that rs10453441 may not be generalized to the European populations. But it is also possible that the association in the European populations was mitigated by the suboptimal imputation quality. Furthermore, we are genotyping our entire cohort using the Illumina HumanExome BeadChip through MAGGS. As our genotyping progresses, we will have more power to uncover additional loci.</p><p id="__p30" class="p p-last">In summary, we identified a novel genome-wide significant association between rs10453441 in <em>WNT7B</em> and CCT in the largest GWAS of CCT in Latinos. We also replicated previously reported SNPs identified in other ethnic groups. Furthermore, our pathway analysis identified relevant pathways associated with apoptosis and cell development for CCT. Our results suggest GWAS conducted in minority populations can identify additional loci associated with CCT not previously identified in GWAS or large-scale meta-analyses using European or Asian populations and may aid in furthering our understanding of ethnic differences of CCT, as well as elucidating the genetic architecture of CCT.</p></div><div id="__sec10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec10title">Materials and Methods</h2><div id="__sec11" class="sec sec-first"><h3 id="__sec11title">Ethics statement</h3><p id="__p31" class="p p-first-last">The following research was approved by the University of Illinois at Chicago, the University of Southern California Health Sciences Campus, and the Los Angeles Biomedical Research Institute at Harbor-University of California, Los Angeles, (UCLA) institutional review boards. All clinical investigation was conducted according to the principles expressed in the Declaration of Helsinki.</p></div><div id="__sec12" class="sec"><h3 id="__sec12title">Study sample</h3><p id="__p32" class="p p-first-last">We conducted this research using data collected from the Los Angeles Latino Eye Study (LALES), the largest population-based study of visual impairment and ophthalmic diseases conducted among 6,357 Latinos living in six census tracts in the city of La Puente, Los Angeles County, California. All subjects included in this study were 40 years of age and older and written, informed consent was obtained from each study participant.</p></div><div id="__sec13" class="sec"><h3 id="__sec13title">Measurement of CCT</h3><p id="__p33" class="p p-first-last">All study participants underwent a detailed ophthalmologic examination. Three CCT measurements were obtained in each eye using a commercial ultrasonic computation module (A-scan/pachymeter DGH 4000B SBH IOL Computation Module; DGH Tech, Inc., Exton, PA) during examination. The three CCT measurements were averaged to obtain a single value in each eye. The average CCT measurement for the left and right eyes was taken to yield the final CCT value. If CCT measurements were available for only one eye, this value was used as a surrogate for the final CCT measurement.</p></div><div id="__sec14" class="sec"><h3 id="__sec14title">Genotyping and quality control</h3><p id="__p34" class="p p-first-last">We genotyped a total of 4,996 Latinos through LALES and the Mexican American Glaucoma Genetic Study (MAGGS) using either the Illumina OmniExpress BeadChip Kit (730, 525 markers; Illumina, Inc., San Diego, CA; <em>n =</em> 4,278) or the Illumina Hispanic/SOL BeadChip (∼2.5 million markers; Illumina, Inc., San Diego, CA; <em>n =</em> 718). The genotyping for this study was performed at the Genotyping Laboratory of the Institute for Translational Genomics and Population Sciences at the Los Angeles Biomedical Research Institute at Harbor-UCLA. SNPs were called using the software Illumina GenomeStudio (v2011.1; Illumina, Inc.). Study participants whose genotyping call rate was less than 97% were excluded from further analysis. Additionally, study participants with gender inconsistency between reported and genetically inferred gender, missing CCT measurements, CCT outliers, and duplicates were removed. After removal of these study subjects, 4,515 subjects remained for downstream analysis, of which 3,584 unrelated individuals were used as a discovery set (stage 1) and 931 first-degree relatives (416 families) were used as a replication set (stage 2). The program PLINK (v1.90) was used to perform quality control on the genotype data (<a href="#ddw319-B46" rid="ddw319-B46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593783">46</a>). SNPs that overlapped between the genotyping chips were retained for analysis and were further excluded if the minor allele frequency (MAF) &lt; 1%, the call rate &lt; 95%, or the Hardy-Weinberg equilibrium <em>P</em> values &lt; 10<sup>−</sup><sup>6</sup>. After the above quality control parameters, 576,798 SNPs remained for further analysis. SNPs were coded on the forward strand to facilitate the imputation process.</p></div><div id="__sec15" class="sec"><h3 id="__sec15title">Genotype imputation</h3><p id="__p35" class="p p-first">To increase genomic coverage, we used Shapeit2 (<a href="#ddw319-B47" rid="ddw319-B47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593740">47</a>), Minimac3 (see Web Resources), and the 1000 Genomes Project (1KGP) reference panels to perform genotype imputation to integrate additional SNPs that were not directly genotyped. Both Minimac3 and the 1KGP Phase 1v3 haplotype reference panels were downloaded from the Center for Statistical Genetics at the University of Michigan (see Web Resources). The 1KGP reference panels used for imputation contain a large number of variants, 39.7 million variants, increasing the ability to interrogate non-genotyped SNPs through the use of linkage disequilibrium (LD). Due to the unique ancestral populations of Latinos, we used the AMR + CEU + YRI reference panels (a combination of Colombian, Mexican, Puerto Rican, CEPH, and Yoruba haplotypes), which we have previously shown to have the highest genotype imputation accuracy for Latinos (<a href="#ddw319-B42" rid="ddw319-B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593766">42</a>).</p><p id="__p36" class="p p-last">We phased our genotypes using Shapeit2 and carried out genotype imputation using Minimac3 on the phased data. Genotypes that were imputed were coded as allelic dosages (estimated counts ranging from 0 to 2). Quality control parameters were applied to filter out low quality imputed SNPs, i.e. Rsq &lt; 0.80 and MAF &lt; 1%. After removal of low quality imputed SNPs, 6,844,888 imputed SNPs remained for further analysis.</p></div><div id="__sec16" class="sec"><h3 id="__sec16title">Statistical analysis</h3><p id="__p37" class="p p-first-last">The program EIGENSOFT was used to infer principal components of genetic ancestry (<a href="#ddw319-B48" rid="ddw319-B48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593789">48</a>). To make inferences of known ancestral populations, we included reference panels of unrelated Northern Europeans (CEU, <em>n</em> = 87) and West Africans (YRI, <em>n</em> = 88) from the 1000 Genomes Project (<a href="#ddw319-B49" rid="ddw319-B49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593739">49</a>), and Native Americans (<em>n =</em> 105) (<a href="#ddw319-B50" rid="ddw319-B50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716916821">50</a>). The first four principal components were retained and were included as covariates during linear regression. The genomic control inflation factor (<a href="#ddw319-B51" rid="ddw319-B51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593765">51</a>) was calculated and applied and a quantile-quantile (Q-Q) plot was generated to visually inspect the distribution of test statistics. For the discovery set, we used PLINK to test the association between individual SNPs and CCT using linear regression, adjusting for age, gender, and principal components, assuming an additive genetic effects model (<a href="#ddw319-B46" rid="ddw319-B46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593788">46</a>). For the replication set, the association between SNPs and CCT was assessed using a linear mixed-effects model (Proc Mixed procedure of SAS v9.4; SAS Institute, Cary, NC), adjusting for age, gender, and principal components of genetic ancestry. The compound symmetry covariance structure and the empirical “sandwich” estimator were used. Fixed-effects meta-analysis for the discovery and replication sets was conducted in METAL using inverse-variance weighting (<a href="#ddw319-B52" rid="ddw319-B52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593817">52</a>). Accounting for genotype uncertainty, allelic dosages of imputed SNPs were analysed using the software mach2qtl (see Web Resources). The conventional SNP <em>P</em> value significance threshold of 5 ×10<sup>−</sup><sup>8</sup> was used to declare genome-wide significant SNPs. We applied the <em>simpleM</em> (<a href="#ddw319-B25" rid="ddw319-B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593816">25</a>,<a href="#ddw319-B26" rid="ddw319-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593775">26</a>,<a href="#ddw319-B53" rid="ddw319-B53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593737">53</a>) method for multiple testing correction to evaluate previously reported loci. The software R (<a href="#ddw319-B54" rid="ddw319-B54" class=" bibr popnode">54</a>) and LocusZoom (hg19/1KGP AMR) (<a href="#ddw319-B55" rid="ddw319-B55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593824">55</a>) were used for graphing.</p></div><div id="__sec17" class="sec sec-last"><h3 id="__sec17title">Pathway analysis</h3><p id="__p38" class="p p-first-last">To investigate the contribution of combined genetic effects on the underlying biological processes influencing CCT, we performed gene-based and pathway analysis on the maximum number of unrelated individuals using SKAT-O (<a href="#ddw319-B56" rid="ddw319-B56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716916824">56</a>), QIAGEN’s Ingenuity Pathway Analysis (IPA) (see Web Resources), and WEB-based GEne SeT AnaLysis Toolkit (WebGestalt) (<a href="#ddw319-B57" rid="ddw319-B57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716593827">57</a>). Using all directly genotyped SNPs, we assigned these SNPs to genes according to genomic position based on the hg19 assembly. Moreover, in order to capture proximal regulatory and other functional elements occurring outside of a gene, we extended the gene boundaries to include SNPs ± 10kb of a gene. SNPs occurring in multiple gene boundaries were included in these boundaries separately. Similar to the GWAS model, the gene-set associations were further adjusted for age, gender, and principal components of genetic ancestry. Enrichment of canonical pathways and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways with genes associated with CCT was analysed using IPA and WebGestalt. Canonical pathways and KEGG pathways with a <em>P</em> value &lt; 0.05 after Benjamini-Hochberg multiple testing adjustment (<a href="#ddw319-B58" rid="ddw319-B58" class=" bibr popnode">58</a>) were declared significantly enriched. The top five most enriched canonical pathways from IPA and KEGG pathways from WebGestalt are reported.</p></div></div><div id="__sec18" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec18title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="sup1"><h4>Supplementary Data</h4><div class="sup-box half_rhythm" id="idm140380845597248"><a href="/pmc/articles/PMC6078592/bin/ddw319_supp.zip" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(55M, zip)</sup></div></div></div><div id="idm140380845595344" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140380845595344title">Acknowledgements</h2><div class="sec"><p id="__p40">The authors would like to thank the study participants from LALES and the staff who aided in data collection and processing.</p><p id="__p41">
<em>Conflict of Interest statement</em>. None declared. </p><div id="__sec19" class="sec"><h3 id="__sec19title">Funding </h3><p id="__p42" class="p p-first">This work was supported in part by National Institutes of Health (NIH; Bethesda, MD, USA) grants R01EY022651 (to XG), U10EY011753 (to RV), P30EY001792 (departmental core grant), and a grant from Midwest Eye-Banks (to XG). The provision of genotyping data was supported in part by the National Center for Advancing Translational Sciences (CTSI) grant UL1TR000124, NIDDK Diabetes Research Center (DRC) Grant DK063491 to the Southern California Diabetes Research Center. The content is solely the responsibility of the authors and does not necessarily represent the official views of the NIH.</p><p id="__p43">Web Resources</p><p id="__p44">The URLs for downloaded data and programs (date last accessed February 2016):</p><p id="__p45">1000 Genomes Project Phase 1v3 haplotypes, <a href="http://csg.sph.umich.edu/abecasis/MACH/download/1000G.2012-03-14.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://csg.sph.umich.edu/abecasis/MACH/download/1000G.2012-03-14.html</a></p><p id="__p46">EIGENSOFT, <a href="https://genetics.med.harvard.edu/reich/Reich_Lab/Software.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://genetics.med.harvard.edu/reich/Reich_Lab/Software.html</a></p><p id="__p47">Genevestigator, <a href="https://genevestigator.com/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://genevestigator.com/</a></p><p id="__p48">mach2qtl, <a href="http://csg.sph.umich.edu/abecasis/MACH/download/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://csg.sph.umich.edu/abecasis/MACH/download/</a></p><p id="__p49">METAL <a href="http://csg.sph.umich.edu/abecasis/Metal/download/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://csg.sph.umich.edu/abecasis/Metal/download/</a></p><p id="__p50">Minimac3, <a href="http://genome.sph.umich.edu/wiki/Minimac3#Download" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genome.sph.umich.edu/wiki/Minimac3#Download</a></p><p id="__p51">PLINK, <a href="https://www.cog-genomics.org/plink2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.cog-genomics.org/plink2</a></p><p id="__p52">QIAGEN, <a href="http://www.ingenuity.com/products/ipa/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ingenuity.com/products/ipa/</a></p><p id="__p53">
<em>simpleM</em>, <a href="http://simplem.sourceforge.net" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://simplem.sourceforge.net</a></p><p id="__p54" class="p p-last">WebGestalt, <a href="http://bioinfo.vanderbilt.edu/webgestalt/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://bioinfo.vanderbilt.edu/webgestalt/</a></p></div></div></div><div id="app1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app1title">Appendix</h2><!--/article/back/app-group/app/--><p id="__p56">Tin Aung<sup>1,2</sup>, Kathryn P. Burdon<sup>3</sup>, Ching-Yu Cheng<sup>1,2</sup>, Jamie E. Craig<sup>4</sup>, Angela J Cree<sup>5</sup>, Puya Gharahkhani<sup>6</sup>, Christopher J. Hammond<sup>7</sup>, Alex W. Hewitt<sup>8</sup>, René Höhn<sup>9</sup>, Pirro Hysi<sup>7</sup>, Adriana I Iglesias Gonzalez<sup>10</sup>, Jost Jonas<sup>11</sup>, Anthony Khawaja<sup>12,13</sup>, Chiea-Cheun Khor<sup>14</sup>, Caroline CW Klaver<sup>15</sup>, Francesca Pasutto<sup>16</sup>, Stuart MacGregor<sup>6</sup>, David Mackey<sup>17</sup>, Paul Mitchell<sup>18</sup>, Aniket Mishra<sup>6</sup>, Calvin Pang<sup>19</sup>, Louis R Pasquale<sup>20</sup>, Henriette Springelkamp<sup>15</sup>, Gudmar Thorleifsson<sup>21</sup>, Unnur Thorsteinsdottir<sup>21</sup>, Cornelia M van Duijn<sup>22</sup>, Ananth Viswanathan<sup>23</sup>, Veronique Vitart<sup>24</sup>, Janey L Wiggs<sup>20</sup>, Robert Wojciechowski<sup>25,26,27</sup>, Tien Wong<sup>1,2</sup>, Terrri L Young<sup>28</sup>, Tanja Zeller<sup>29</sup></p><p id="__p57">
<sup>1</sup>Singapore Eye Research Institute, Singapore National Eye Centre, Singapore; <sup>2</sup>Department of Ophthalmology, National University of Singapore and National University Health System, Singapore; <sup>3</sup>Menzies Institute for Medical Research, University of Tasmania, Hobart, Tasmania, Australia; <sup>4</sup>Department of Ophthalmology, Flinders University, Adelaide, Australia; <sup>5</sup>Clinical &amp; Experimental Sciences, Faculty of Medicine, University of Southampton, Southampton, UK; <sup>6</sup>Statistical Genetics, Queensland Institute of Medical Research, Brisbane 4029, Australia; <sup>7</sup>Department of Twin Research and Genetic Epidemiology, King's College London, St. Thomas’ Hospital, London, UK; <sup>8</sup>Centre for Eye Research Australia, University of Melbourne, Royal Victorian Eye and Ear Hospital, Melbourne, Australia; <sup>9</sup>Department of Ophthalmology, University Medical Center Mainz, Mainz, Germany; <sup>10</sup>Department of Epidemiology, Erasmus University Medical Center, Rotterdam, The Netherlands; <sup>11</sup>Department of Ophthalmology, Medical Faculty Mannheim of the Ruprecht-Karls-University of Heidelberg, Mannheim, Germany; <sup>12</sup>Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK; <sup>13</sup>NIHR Biomedical Research Centre for Ophthalmology, Moorfields Eye Hospital and UCL Institute of Ophthalmology, London, UK; <sup>14</sup>Department of Human Genetics, Genome Institute of Singapore, Singapore; <sup>15</sup>Department of Epidemiology and Department of Ophthalmology, Erasmus Medical Center, Rotterdam, The Netherlands; <sup>16</sup>Institute of Human Genetics, Friedrich-Alexander-Universität Erlangen-Nürnberg (FAU), Erlangen, Germany; <sup>17</sup>Centre for Ophthalmology and Visual Science, Lions Eye Institute, University of Western Australia, Perth, Australia; <sup>18</sup>Centre for Vision Research, Department of Ophthalmology and Westmead Millennium Institute, University of Sydney, Sydney, Australia; <sup>19</sup>Department of Ophthalmology and Visual Sciences, Chinese University of Hong Kong, Hong Kong, China; <sup>20</sup>Department of Ophthalmology, Harvard Medical School, Massachusetts Eye and Ear, Boston, MA; <sup>21</sup>deCODE Genetics/Amgen, 101 Reykjavik, Iceland; <sup>22</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam, The Netherlands; <sup>23</sup>NIHR Biomedical Research Centre, Moorfields Eye Hospital NHS Foundation Trust, University College London, Institute of Ophthalmology, London, UK; <sup>24</sup>MRC Human Genetics Unit, IGMM, University of Edinburgh, Edinburgh EH4 2XU, UK; <sup>25</sup>Department of Epidemiology, Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USA; <sup>26</sup>Wilmer Eye Institute, Johns Hopkins School of Medicine, Baltimore, MD, USA; <sup>27</sup>National Human Genome Research Institute (NIH), Baltimore, MD, USA; <sup>28</sup>Department of Ophthalmology and Visual Sciences, School of Medicine and Public Health, University of Wisconsin, Madison, Wisconsin 53705, USA; <sup>29</sup>Clinic for General and Interventional Cardiology, University Heart Center Hamburg, Hamburg, Germany.</p><p id="__p58">
<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure 1</a>. Q<strong>uantile-Quantile (Q-Q) plot of the genome-wide <em>P</em> values</strong></p><p id="__p59">For the 576, 798 SNPs that passed quality control filters in the discovery set, –log<sub>10</sub>(<em>P</em> values) is plotted against the corresponding expected –log<sub>10</sub>(<em>P</em> values). The genomic control inflation factor λ = 1.05.</p><p id="__p60">
<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure 2</a>. <strong>Expression of <em>WNT7B</em> in various tissues from Genevestigator</strong></p><p id="__p61">The boxplot-list displays the <em>WNT7B</em> gene expression in 62 postmortem healthy human anatomical parts, ranked by the level of expression. <em>WNT7B</em> is highly expressed in the human cornea (<em>n =</em> 6).</p><p id="__p62">
<strong><a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure 3</a></strong>. Ancestral composition of our Latino sample.</p><p id="__p63">The first two dimensions from principal component analysis of our Latino subjects are shown along with three ancestral populations. Abbreviations: NA, Native Americans; CEU, CEPH in Utah residents; YRI, Yoruba in Ibadan, Nigeria.</p></div><div id="__sec20" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec20title">Supplementary Material</h2><!--/article/back/sec/--><p id="__p55" class="p p-first-last">
<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material</a> is available at <em>HMG</em> online.</p></div><div id="idm140380845569952" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140380845569952title">References </h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="ddw319-B1">1. <span class="mixed-citation">
Burkitt Wright E.M., Porter L.F., Spencer H.L., Clayton-Smith J., Au L., Munier F.L., Smithson S., Suri M., Rohrbach M., Manson F.D., et al. (2013) 
<span class="ref-title">Brittle cornea syndrome: recognition, molecular diagnosis and management</span>. <span class="ref-journal">Orphanet. J. Rare Dis</span>., <span class="ref-vol">8</span>, 68.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3659006/">PMC free article</a>]</span> [<a href="/pubmed/23642083" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Orphanet.+J.+Rare+Dis&amp;title=Brittle+cornea+syndrome:+recognition,+molecular+diagnosis+and+management&amp;author=E.M.+Burkitt+Wright&amp;author=L.F.+Porter&amp;author=H.L.+Spencer&amp;author=J.+Clayton-Smith&amp;author=L.+Au&amp;volume=8&amp;publication_year=2013&amp;pages=68.&amp;pmid=23642083&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B2">2. <span class="mixed-citation">
Romero-Jimenez M., Santodomingo-Rubido J., Wolffsohn J.S. (2010) 
<span class="ref-title">Keratoconus: a review</span>. <span class="ref-journal">Cont. Lens Anterior Eye</span>, <span class="ref-vol">33</span>, 157–166. quiz 205. [<a href="/pubmed/20537579" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cont.+Lens+Anterior+Eye&amp;title=Keratoconus:+a+review&amp;author=M.+Romero-Jimenez&amp;author=J.+Santodomingo-Rubido&amp;author=J.S.+Wolffsohn&amp;volume=33&amp;publication_year=2010&amp;pages=157-166&amp;pmid=20537579&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B3">3. <span class="mixed-citation">
Dimasi D.P., Chen J.Y., Hewitt A.W., Klebe S., Davey R., Stirling J., Thompson E., Forbes R., Tan T.Y., Savarirayan R., et al. (2010) 
<span class="ref-title">Novel quantitative trait loci for central corneal thickness identified by candidate gene analysis of osteogenesis imperfecta genes</span>. <span class="ref-journal">Hum. Genet</span>., <span class="ref-vol">127</span>, 33–44. [<a href="/pubmed/19714363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet&amp;title=Novel+quantitative+trait+loci+for+central+corneal+thickness+identified+by+candidate+gene+analysis+of+osteogenesis+imperfecta+genes&amp;author=D.P.+Dimasi&amp;author=J.Y.+Chen&amp;author=A.W.+Hewitt&amp;author=S.+Klebe&amp;author=R.+Davey&amp;volume=127&amp;publication_year=2010&amp;pages=33-44&amp;pmid=19714363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B4">4. <span class="mixed-citation">
Leske M.C., Wu S.Y., Hennis A., Honkanen R., Nemesure B., Group B.E.S. (2008) 
<span class="ref-title">Risk factors for incident open-angle glaucoma: the Barbados Eye Studies</span>. <span class="ref-journal">Ophthalmology</span>, <span class="ref-vol">115</span>, 85–93. [<a href="/pubmed/17629563" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=Risk+factors+for+incident+open-angle+glaucoma:+the+Barbados+Eye+Studies&amp;author=M.C.+Leske&amp;author=S.Y.+Wu&amp;author=A.+Hennis&amp;author=R.+Honkanen&amp;author=B.+Nemesure&amp;volume=115&amp;publication_year=2008&amp;pages=85-93&amp;pmid=17629563&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B5">5. <span class="mixed-citation">
Jiang X., Varma R., Wu S., Torres M., Azen S.P., Francis B.A., Chopra V., Nguyen B.B., Los Angeles Latino Eye Study G. (2012) 
<span class="ref-title">Baseline risk factors that predict the development of open-angle glaucoma in a population: the Los Angeles Latino Eye Study</span>. <span class="ref-journal">Ophthalmology</span>, <span class="ref-vol">119</span>, 2245–2253. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3474872/">PMC free article</a>]</span> [<a href="/pubmed/22796305" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=Baseline+risk+factors+that+predict+the+development+of+open-angle+glaucoma+in+a+population:+the+Los+Angeles+Latino+Eye+Study&amp;author=X.+Jiang&amp;author=R.+Varma&amp;author=S.+Wu&amp;author=M.+Torres&amp;author=S.P.+Azen&amp;volume=119&amp;publication_year=2012&amp;pages=2245-2253&amp;pmid=22796305&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B6">6. <span class="mixed-citation">
European Glaucoma Prevention Study G., Miglior S., Pfeiffer N., Torri V., Zeyen T., Cunha-Vaz J., Adamsons I. (2007) 
<span class="ref-title">Predictive factors for open-angle glaucoma among patients with ocular hypertension in the European Glaucoma Prevention Study</span>. <span class="ref-journal">Ophthalmology</span>, <span class="ref-vol">114</span>, 3–9. [<a href="/pubmed/17070596" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=Predictive+factors+for+open-angle+glaucoma+among+patients+with+ocular+hypertension+in+the+European+Glaucoma+Prevention+Study&amp;author=G.+European+Glaucoma+Prevention+Study&amp;author=S.+Miglior&amp;author=N.+Pfeiffer&amp;author=V.+Torri&amp;author=T.+Zeyen&amp;volume=114&amp;publication_year=2007&amp;pages=3-9&amp;pmid=17070596&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B7">7. <span class="mixed-citation">
Aghaian E., Choe J.E., Lin S., Stamper R.L. (2004) 
<span class="ref-title">Central corneal thickness of Caucasians, Chinese, Hispanics, Filipinos, African Americans, and Japanese in a glaucoma clinic</span>. <span class="ref-journal">Ophthalmology</span>, <span class="ref-vol">111</span>, 2211–2219. [<a href="/pubmed/15582076" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=Central+corneal+thickness+of+Caucasians,+Chinese,+Hispanics,+Filipinos,+African+Americans,+and+Japanese+in+a+glaucoma+clinic&amp;author=E.+Aghaian&amp;author=J.E.+Choe&amp;author=S.+Lin&amp;author=R.L.+Stamper&amp;volume=111&amp;publication_year=2004&amp;pages=2211-2219&amp;pmid=15582076&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B8">8. <span class="mixed-citation">
Gordon M.O., Beiser J.A., Brandt J.D., Heuer D.K., Higginbotham E.J., Johnson C.A., Keltner J.L., Miller J.P., Parrish R.K., 2nd, Wilson M.R., et al. (2002) 
<span class="ref-title">The Ocular Hypertension Treatment Study: baseline factors that predict the onset of primary open-angle glaucoma</span>. <span class="ref-journal">Arch. Ophthalmol</span>., <span class="ref-vol">120</span>, 714–720. discussion 829-730. [<a href="/pubmed/12049575" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Ophthalmol&amp;title=The+Ocular+Hypertension+Treatment+Study:+baseline+factors+that+predict+the+onset+of+primary+open-angle+glaucoma&amp;author=M.O.+Gordon&amp;author=J.A.+Beiser&amp;author=J.D.+Brandt&amp;author=D.K.+Heuer&amp;author=E.J.+Higginbotham&amp;volume=120&amp;publication_year=2002&amp;pages=714-720&amp;pmid=12049575&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B9">9. <span class="mixed-citation">
Varma R., Ying-Lai M., Francis B.A., Nguyen B.B., Deneen J., Wilson M.R., Azen S.P., Los Angeles Latino Eye Study G. (2004) 
<span class="ref-title">Prevalence of open-angle glaucoma and ocular hypertension in Latinos: the Los Angeles Latino Eye Study</span>. <span class="ref-journal">Ophthalmology</span>, <span class="ref-vol">111</span>, 1439–1448. [<a href="/pubmed/15288969" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=Prevalence+of+open-angle+glaucoma+and+ocular+hypertension+in+Latinos:+the+Los+Angeles+Latino+Eye+Study&amp;author=R.+Varma&amp;author=M.+Ying-Lai&amp;author=B.A.+Francis&amp;author=B.B.+Nguyen&amp;author=J.+Deneen&amp;volume=111&amp;publication_year=2004&amp;pages=1439-1448&amp;pmid=15288969&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B10">10. <span class="mixed-citation">
Leske M.C., Heijl A., Hyman L., Bengtsson B., Dong L., Yang Z., Group E. (2007) 
<span class="ref-title">Predictors of long-term progression in the early manifest glaucoma trial</span>. <span class="ref-journal">Ophthalmology</span>, <span class="ref-vol">114</span>, 1965–1972. [<a href="/pubmed/17628686" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=Predictors+of+long-term+progression+in+the+early+manifest+glaucoma+trial&amp;author=M.C.+Leske&amp;author=A.+Heijl&amp;author=L.+Hyman&amp;author=B.+Bengtsson&amp;author=L.+Dong&amp;volume=114&amp;publication_year=2007&amp;pages=1965-1972&amp;pmid=17628686&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B11">11. <span class="mixed-citation">
Nemesure B., Wu S.Y., Hennis A., Leske M.C., Barbados Eye Study G. (2003) 
<span class="ref-title">Corneal thickness and intraocular pressure in the Barbados eye studies</span>. <span class="ref-journal">Arch. Ophthalmol</span>., <span class="ref-vol">121</span>, 240–244. [<a href="/pubmed/12583791" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Ophthalmol&amp;title=Corneal+thickness+and+intraocular+pressure+in+the+Barbados+eye+studies&amp;author=B.+Nemesure&amp;author=S.Y.+Wu&amp;author=A.+Hennis&amp;author=M.C.+Leske&amp;author=G.+Barbados+Eye+Study&amp;volume=121&amp;publication_year=2003&amp;pages=240-244&amp;pmid=12583791&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B12">12. <span class="mixed-citation">
Hahn S., Azen S., Ying-Lai M., Varma R., Los Angeles Latino Eye Study G. (2003) 
<span class="ref-title">Central corneal thickness in Latinos</span>. <span class="ref-journal">Invest. Ophthalmol. Vis. Sci</span>., <span class="ref-vol">44</span>, 1508–1512. [<a href="/pubmed/12657586" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci&amp;title=Central+corneal+thickness+in+Latinos&amp;author=S.+Hahn&amp;author=S.+Azen&amp;author=M.+Ying-Lai&amp;author=R.+Varma&amp;author=G.+Los+Angeles+Latino+Eye+Study&amp;volume=44&amp;publication_year=2003&amp;pages=1508-1512&amp;pmid=12657586&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B13">13. <span class="mixed-citation">
Wolfs R.C., Klaver C.C., Vingerling J.R., Grobbee D.E., Hofman A., de Jong P.T. (1997) 
<span class="ref-title">Distribution of central corneal thickness and its association with intraocular pressure: The Rotterdam Study</span>. <span class="ref-journal">Am. J. Ophthalmol</span>., <span class="ref-vol">123</span>, 767–772. [<a href="/pubmed/9535620" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Ophthalmol&amp;title=Distribution+of+central+corneal+thickness+and+its+association+with+intraocular+pressure:+The+Rotterdam+Study&amp;author=R.C.+Wolfs&amp;author=C.C.+Klaver&amp;author=J.R.+Vingerling&amp;author=D.E.+Grobbee&amp;author=A.+Hofman&amp;volume=123&amp;publication_year=1997&amp;pages=767-772&amp;pmid=9535620&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B14">14. <span class="mixed-citation">
Dimasi D.P., Burdon K.P., Craig J.E. (2010) 
<span class="ref-title">The genetics of central corneal thickness</span>. <span class="ref-journal">Br. J. Ophthalmol</span>., <span class="ref-vol">94</span>, 971–976. [<a href="/pubmed/19556215" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Ophthalmol&amp;title=The+genetics+of+central+corneal+thickness&amp;author=D.P.+Dimasi&amp;author=K.P.+Burdon&amp;author=J.E.+Craig&amp;volume=94&amp;publication_year=2010&amp;pages=971-976&amp;pmid=19556215&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B15">15. <span class="mixed-citation">
Alsbirk P.H. (1978) 
<span class="ref-title">Corneal thickness. II. Environmental and genetic factors</span>. <span class="ref-journal">Acta. Ophthalmol. (Copenh.)</span>, <span class="ref-vol">56</span>, 105–113. [<a href="/pubmed/580329" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta.+Ophthalmol.+(Copenh.)&amp;title=Corneal+thickness.+II.+Environmental+and+genetic+factors&amp;author=P.H.+Alsbirk&amp;volume=56&amp;publication_year=1978&amp;pages=105-113&amp;pmid=580329&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B16">16. <span class="mixed-citation">
Toh T., Liew S.H., MacKinnon J.R., Hewitt A.W., Poulsen J.L., Spector T.D., Gilbert C.E., Craig J.E., Hammond C.J., Mackey D.A. (2005) 
<span class="ref-title">Central corneal thickness is highly heritable: the twin eye studies</span>. <span class="ref-journal">Invest. Ophthalmol. Vis. Sci</span>., <span class="ref-vol">46</span>, 3718–3722. [<a href="/pubmed/16186354" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci&amp;title=Central+corneal+thickness+is+highly+heritable:+the+twin+eye+studies&amp;author=T.+Toh&amp;author=S.H.+Liew&amp;author=J.R.+MacKinnon&amp;author=A.W.+Hewitt&amp;author=J.L.+Poulsen&amp;volume=46&amp;publication_year=2005&amp;pages=3718-3722&amp;pmid=16186354&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B17">17. <span class="mixed-citation">
Cornes B.K., Khor C.C., Nongpiur M.E., Xu L., Tay W.T., Zheng Y., Lavanya R., Li Y., Wu R., Sim X., et al. (2012) 
<span class="ref-title">Identification of four novel variants that influence central corneal thickness in multi-ethnic Asian populations</span>. <span class="ref-journal">Hum. Mol. Genet</span>., <span class="ref-vol">21</span>, 437–445. [<a href="/pubmed/21984434" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Identification+of+four+novel+variants+that+influence+central+corneal+thickness+in+multi-ethnic+Asian+populations&amp;author=B.K.+Cornes&amp;author=C.C.+Khor&amp;author=M.E.+Nongpiur&amp;author=L.+Xu&amp;author=W.T.+Tay&amp;volume=21&amp;publication_year=2012&amp;pages=437-445&amp;pmid=21984434&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B18">18. <span class="mixed-citation">
Vitart V., Bencic G., Hayward C., Skunca Herman J., Huffman J., Campbell S., Bucan K., Navarro P., Gunjaca G., Marin J., et al. (2010) 
<span class="ref-title">New loci associated with central cornea thickness include COL5A1, AKAP13 and AVGR8</span>. <span class="ref-journal">Hum. Mol. Genet</span>., <span class="ref-vol">19</span>, 4304–4311. [<a href="/pubmed/20719862" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=New+loci+associated+with+central+cornea+thickness+include+COL5A1,+AKAP13+and+AVGR8&amp;author=V.+Vitart&amp;author=G.+Bencic&amp;author=C.+Hayward&amp;author=J.+Skunca+Herman&amp;author=J.+Huffman&amp;volume=19&amp;publication_year=2010&amp;pages=4304-4311&amp;pmid=20719862&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B19">19. <span class="mixed-citation">
Vithana E.N., Aung T., Khor C.C., Cornes B.K., Tay W.T., Sim X., Lavanya R., Wu R., Zheng Y., Hibberd M.L., et al. (2011) 
<span class="ref-title">Collagen-related genes influence the glaucoma risk factor, central corneal thickness</span>. <span class="ref-journal">Hum. Mol. Genet</span>., <span class="ref-vol">20</span>, 649–658. [<a href="/pubmed/21098505" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Collagen-related+genes+influence+the+glaucoma+risk+factor,+central+corneal+thickness&amp;author=E.N.+Vithana&amp;author=T.+Aung&amp;author=C.C.+Khor&amp;author=B.K.+Cornes&amp;author=W.T.+Tay&amp;volume=20&amp;publication_year=2011&amp;pages=649-658&amp;pmid=21098505&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B20">20. <span class="mixed-citation">
Lu Y., Dimasi D.P., Hysi P.G., Hewitt A.W., Burdon K.P., Toh T., Ruddle J.B., Li Y.J., Mitchell P., Healey P.R., et al. (2010) 
<span class="ref-title">Common genetic variants near the Brittle Cornea Syndrome locus ZNF469 influence the blinding disease risk factor central corneal thickness</span>. <span class="ref-journal">PLoS. Genet</span>., <span class="ref-vol">6</span>, e1000947.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2869325/">PMC free article</a>]</span> [<a href="/pubmed/20485516" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS.+Genet&amp;title=Common+genetic+variants+near+the+Brittle+Cornea+Syndrome+locus+ZNF469+influence+the+blinding+disease+risk+factor+central+corneal+thickness&amp;author=Y.+Lu&amp;author=D.P.+Dimasi&amp;author=P.G.+Hysi&amp;author=A.W.+Hewitt&amp;author=K.P.+Burdon&amp;volume=6&amp;publication_year=2010&amp;pages=e1000947.&amp;pmid=20485516&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B21">21. <span class="mixed-citation">
Hoehn R., Zeller T., Verhoeven V.J., Grus F., Adler M., Wolfs R.C., Uitterlinden A.G., Castagne R., Schillert A., Klaver C.C., et al. (2012) 
<span class="ref-title">Population-based meta-analysis in Caucasians confirms association with COL5A1 and ZNF469 but not COL8A2 with central corneal thickness</span>. <span class="ref-journal">Hum. Genet</span>., <span class="ref-vol">131</span>, 1783–1793. [<a href="/pubmed/22814818" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet&amp;title=Population-based+meta-analysis+in+Caucasians+confirms+association+with+COL5A1+and+ZNF469+but+not+COL8A2+with+central+corneal+thickness&amp;author=R.+Hoehn&amp;author=T.+Zeller&amp;author=V.J.+Verhoeven&amp;author=F.+Grus&amp;author=M.+Adler&amp;volume=131&amp;publication_year=2012&amp;pages=1783-1793&amp;pmid=22814818&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B22">22. <span class="mixed-citation">
Gao X., Gauderman W.J., Liu Y., Marjoram P., Torres M., Haritunians T., Kuo J.Z., Chen Y.D., Allingham R.R., Hauser M.A., et al. (2013) 
<span class="ref-title">A genome-wide association study of central corneal thickness in Latinos</span>. <span class="ref-journal">Invest. Ophthalmol. Vis. Sci</span>., <span class="ref-vol">54</span>, 2435–2443. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3621577/">PMC free article</a>]</span> [<a href="/pubmed/23493294" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci&amp;title=A+genome-wide+association+study+of+central+corneal+thickness+in+Latinos&amp;author=X.+Gao&amp;author=W.J.+Gauderman&amp;author=Y.+Liu&amp;author=P.+Marjoram&amp;author=M.+Torres&amp;volume=54&amp;publication_year=2013&amp;pages=2435-2443&amp;pmid=23493294&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B23">23. <span class="mixed-citation">
Lu Y., Vitart V., Burdon K.P., Khor C.C., Bykhovskaya Y., Mirshahi A., Hewitt A.W., Koehn D., Hysi P.G., Ramdas W.D., et al. (2013) 
<span class="ref-title">Genome-wide association analyses identify multiple loci associated with central corneal thickness and keratoconus</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">45</span>, 155–163. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3720123/">PMC free article</a>]</span> [<a href="/pubmed/23291589" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+analyses+identify+multiple+loci+associated+with+central+corneal+thickness+and+keratoconus&amp;author=Y.+Lu&amp;author=V.+Vitart&amp;author=K.P.+Burdon&amp;author=C.C.+Khor&amp;author=Y.+Bykhovskaya&amp;volume=45&amp;publication_year=2013&amp;pages=155-163&amp;pmid=23291589&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B24">24. <span class="mixed-citation">
Fan B.J., Chen X., Sondhi N., Sharmila PFm., Soumittra N., Sripriya S., Sacikala S., Asokan R., Friedman D.S., Pasquale L.R., et al.. Family-based Genome-wide Association Study of South Indian Pedigrees Supports WNT7B as a Central Corneal Thickness Locus (Submitted). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5961220/">PMC free article</a>]</span> [<a href="/pubmed/29847655" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B25">25. <span class="mixed-citation">
Gao X., Starmer J., Martin E.R. (2008) 
<span class="ref-title">A multiple testing correction method for genetic association studies using correlated single nucleotide polymorphisms</span>. <span class="ref-journal">Genet. Epidemiol</span>., <span class="ref-vol">32</span>, 361–369. [<a href="/pubmed/18271029" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=A+multiple+testing+correction+method+for+genetic+association+studies+using+correlated+single+nucleotide+polymorphisms&amp;author=X.+Gao&amp;author=J.+Starmer&amp;author=E.R.+Martin&amp;volume=32&amp;publication_year=2008&amp;pages=361-369&amp;pmid=18271029&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B26">26. <span class="mixed-citation">
Gao X. (2011) 
<span class="ref-title">Multiple testing corrections for imputed SNPs</span>. <span class="ref-journal">Genet. Epidemiol</span>., <span class="ref-vol">35</span>, 154–158. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3055936/">PMC free article</a>]</span> [<a href="/pubmed/21254223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Multiple+testing+corrections+for+imputed+SNPs&amp;author=X.+Gao&amp;volume=35&amp;publication_year=2011&amp;pages=154-158&amp;pmid=21254223&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B27">27. <span class="mixed-citation">
Ulmer M., Li J., Yaspan B.L., Ozel A.B., Richards J.E., Moroi S.E., Hawthorne F., Budenz D.L., Friedman D.S., Gaasterland D., et al. (2012) 
<span class="ref-title">Genome-wide analysis of central corneal thickness in primary open-angle glaucoma cases in the NEIGHBOR and GLAUGEN consortia</span>. <span class="ref-journal">Invest. Ophthalmol. Vis. Sci</span>., <span class="ref-vol">53</span>, 4468–4474. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3394688/">PMC free article</a>]</span> [<a href="/pubmed/22661486" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci&amp;title=Genome-wide+analysis+of+central+corneal+thickness+in+primary+open-angle+glaucoma+cases+in+the+NEIGHBOR+and+GLAUGEN+consortia&amp;author=M.+Ulmer&amp;author=J.+Li&amp;author=B.L.+Yaspan&amp;author=A.B.+Ozel&amp;author=J.E.+Richards&amp;volume=53&amp;publication_year=2012&amp;pages=4468-4474&amp;pmid=22661486&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B28">28. <span class="mixed-citation">
Nakatsu M.N., Ding Z., Ng M.Y., Truong T.T., Yu F., Deng S.X. (2011) 
<span class="ref-title">Wnt/beta-catenin signaling regulates proliferation of human cornea epithelial stem/progenitor cells</span>. <span class="ref-journal">Invest. Ophthalmol. Vis. Sci</span>., <span class="ref-vol">52</span>, 4734–4741. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175950/">PMC free article</a>]</span> [<a href="/pubmed/21357396" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci&amp;title=Wnt/beta-catenin+signaling+regulates+proliferation+of+human+cornea+epithelial+stem/progenitor+cells&amp;author=M.N.+Nakatsu&amp;author=Z.+Ding&amp;author=M.Y.+Ng&amp;author=T.T.+Truong&amp;author=F.+Yu&amp;volume=52&amp;publication_year=2011&amp;pages=4734-4741&amp;pmid=21357396&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B29">29. <span class="mixed-citation">
Miyake M., Yamashiro K., Tabara Y., Suda K., Morooka S., Nakanishi H., Khor C.C., Chen P., Qiao F., Nakata I., et al. (2015) 
<span class="ref-title">Identification of myopia-associated WNT7B polymorphisms provides insights into the mechanism underlying the development of myopia</span>. <span class="ref-journal">Nat. Commun</span>., <span class="ref-vol">6</span>, 6689.. [<a href="/pubmed/25823570" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun&amp;title=Identification+of+myopia-associated+WNT7B+polymorphisms+provides+insights+into+the+mechanism+underlying+the+development+of+myopia&amp;author=M.+Miyake&amp;author=K.+Yamashiro&amp;author=Y.+Tabara&amp;author=K.+Suda&amp;author=S.+Morooka&amp;volume=6&amp;publication_year=2015&amp;pages=6689.&amp;pmid=25823570&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B30">30. <span class="mixed-citation">
Shankardas J., Senchyna M., Dimitrijevich S.D. (2008) 
<span class="ref-title">Presence and distribution of 14-3-3 proteins in human ocular surface tissues</span>. <span class="ref-journal">Mol. Vis</span>., <span class="ref-vol">14</span>, 2604–2615. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2613072/">PMC free article</a>]</span> [<a href="/pubmed/19122826" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Vis&amp;title=Presence+and+distribution+of+14-3-3+proteins+in+human+ocular+surface+tissues&amp;author=J.+Shankardas&amp;author=M.+Senchyna&amp;author=S.D.+Dimitrijevich&amp;volume=14&amp;publication_year=2008&amp;pages=2604-2615&amp;pmid=19122826&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B31">31. <span class="mixed-citation">
Lee H.K., Lee J.H., Kim M., Kariya Y., Miyazaki K., Kim E.K. (2006) 
<span class="ref-title">Insulin-like growth factor-1 induces migration and expression of laminin-5 in cultured human corneal epithelial cells</span>. <span class="ref-journal">Invest. Ophthalmol. Vis. Sci</span>., <span class="ref-vol">47</span>, 873–882. [<a href="/pubmed/16505019" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci&amp;title=Insulin-like+growth+factor-1+induces+migration+and+expression+of+laminin-5+in+cultured+human+corneal+epithelial+cells&amp;author=H.K.+Lee&amp;author=J.H.+Lee&amp;author=M.+Kim&amp;author=Y.+Kariya&amp;author=K.+Miyazaki&amp;volume=47&amp;publication_year=2006&amp;pages=873-882&amp;pmid=16505019&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B32">32. <span class="mixed-citation">
Brown D.J., Lin B., Holguin B. (2004) 
<span class="ref-title">Expression of neuregulin 1, a member of the epidermal growth factor family, is expressed as multiple splice variants in the adult human cornea</span>. <span class="ref-journal">Invest. Ophthalmol. Vis. Sci</span>., <span class="ref-vol">45</span>, 3021–3029. [<a href="/pubmed/15326116" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci&amp;title=Expression+of+neuregulin+1,+a+member+of+the+epidermal+growth+factor+family,+is+expressed+as+multiple+splice+variants+in+the+adult+human+cornea&amp;author=D.J.+Brown&amp;author=B.+Lin&amp;author=B.+Holguin&amp;volume=45&amp;publication_year=2004&amp;pages=3021-3029&amp;pmid=15326116&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B33">33. <span class="mixed-citation">
Lambiase A., Manni L., Bonini S., Rama P., Micera A., Aloe L. (2000) 
<span class="ref-title">Nerve growth factor promotes corneal healing: structural, biochemical, and molecular analyses of rat and human corneas</span>. <span class="ref-journal">Invest. Ophthalmol. Vis. Sci</span>., <span class="ref-vol">41</span>, 1063–1069. [<a href="/pubmed/10752942" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci&amp;title=Nerve+growth+factor+promotes+corneal+healing:+structural,+biochemical,+and+molecular+analyses+of+rat+and+human+corneas&amp;author=A.+Lambiase&amp;author=L.+Manni&amp;author=S.+Bonini&amp;author=P.+Rama&amp;author=A.+Micera&amp;volume=41&amp;publication_year=2000&amp;pages=1063-1069&amp;pmid=10752942&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B34">34. <span class="mixed-citation">
Xu K.P., Riggs A., Ding Y., Yu F.S. (2004) 
<span class="ref-title">Role of ErbB2 in Corneal Epithelial Wound Healing</span>. <span class="ref-journal">Invest. Ophthalmol. Vis. Sci</span>., <span class="ref-vol">45</span>, 4277–4283. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2666385/">PMC free article</a>]</span> [<a href="/pubmed/15557433" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci&amp;title=Role+of+ErbB2+in+Corneal+Epithelial+Wound+Healing&amp;author=K.P.+Xu&amp;author=A.+Riggs&amp;author=Y.+Ding&amp;author=F.S.+Yu&amp;volume=45&amp;publication_year=2004&amp;pages=4277-4283&amp;pmid=15557433&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B35">35. <span class="mixed-citation">
Ban Y., Dota A., Cooper L.J., Fullwood N.J., Nakamura T., Tsuzuki M., Mochida C., Kinoshita S. (2003) 
<span class="ref-title">Tight junction-related protein expression and distribution in human corneal epithelium</span>. <span class="ref-journal">Exp. Eye Res</span>., <span class="ref-vol">76</span>, 663–669. [<a href="/pubmed/12742348" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.+Eye+Res&amp;title=Tight+junction-related+protein+expression+and+distribution+in+human+corneal+epithelium&amp;author=Y.+Ban&amp;author=A.+Dota&amp;author=L.J.+Cooper&amp;author=N.J.+Fullwood&amp;author=T.+Nakamura&amp;volume=76&amp;publication_year=2003&amp;pages=663-669&amp;pmid=12742348&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B36">36. <span class="mixed-citation">
Diehn J.J., Diehn M., Marmor M.F., Brown P.O. (2005) 
<span class="ref-title">Differential gene expression in anatomical compartments of the human eye</span>. <span class="ref-journal">Genome Biol</span>., <span class="ref-vol">6</span>, R74.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1242209/">PMC free article</a>]</span> [<a href="/pubmed/16168081" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Differential+gene+expression+in+anatomical+compartments+of+the+human+eye&amp;author=J.J.+Diehn&amp;author=M.+Diehn&amp;author=M.F.+Marmor&amp;author=P.O.+Brown&amp;volume=6&amp;publication_year=2005&amp;pages=R74.&amp;pmid=16168081&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B37">37. <span class="mixed-citation">
Chen M.J., Liu Y.T., Tsai C.C., Chen Y.C., Chou C.K., Lee S.M. (2009) 
<span class="ref-title">Relationship between central corneal thickness, refractive error, corneal curvature, anterior chamber depth and axial length. <em>J</em></span>. <span class="ref-journal">Chin. Med. Assoc</span>., <span class="ref-vol">72</span>, 133–137. [<a href="/pubmed/19299220" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chin.+Med.+Assoc&amp;title=Relationship+between+central+corneal+thickness,+refractive+error,+corneal+curvature,+anterior+chamber+depth+and+axial+length.+J&amp;author=M.J.+Chen&amp;author=Y.T.+Liu&amp;author=C.C.+Tsai&amp;author=Y.C.+Chen&amp;author=C.K.+Chou&amp;volume=72&amp;publication_year=2009&amp;pages=133-137&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B38">38. <span class="mixed-citation">
Figueira E.C., Di Girolamo N., Coroneo M.T., Wakefield D. (2007) 
<span class="ref-title">The phenotype of limbal epithelial stem cells</span>. <span class="ref-journal">Invest. Ophthalmol. Vis. Sci</span>., <span class="ref-vol">48</span>, 144–156. [<a href="/pubmed/17197527" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci&amp;title=The+phenotype+of+limbal+epithelial+stem+cells&amp;author=E.C.+Figueira&amp;author=N.+Di+Girolamo&amp;author=M.T.+Coroneo&amp;author=D.+Wakefield&amp;volume=48&amp;publication_year=2007&amp;pages=144-156&amp;pmid=17197527&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B39">39. <span class="mixed-citation">
Lyu J., Joo C.K. (2005) 
<span class="ref-title">Wnt-7a up-regulates matrix metalloproteinase-12 expression and promotes cell proliferation in corneal epithelial cells during wound healing</span>. <span class="ref-journal">J. Biol. Chem</span>., <span class="ref-vol">280</span>, 21653–21660. [<a href="/pubmed/15802269" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Wnt-7a+up-regulates+matrix+metalloproteinase-12+expression+and+promotes+cell+proliferation+in+corneal+epithelial+cells+during+wound+healing&amp;author=J.+Lyu&amp;author=C.K.+Joo&amp;volume=280&amp;publication_year=2005&amp;pages=21653-21660&amp;pmid=15802269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B40">40. <span class="mixed-citation">
Gauderman W.J. (2002) 
<span class="ref-title">Sample size requirements for matched case-control studies of gene-environment interaction</span>. <span class="ref-journal">Stat. Med</span>., <span class="ref-vol">21</span>, 35–50. [<a href="/pubmed/11782049" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat.+Med&amp;title=Sample+size+requirements+for+matched+case-control+studies+of+gene-environment+interaction&amp;author=W.J.+Gauderman&amp;volume=21&amp;publication_year=2002&amp;pages=35-50&amp;pmid=11782049&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B41">41. <span class="mixed-citation">
Gauderman W.J. (2002) 
<span class="ref-title">Sample size requirements for association studies of gene-gene interaction</span>. <span class="ref-journal">Am. J. Epidemiol</span>., <span class="ref-vol">155</span>, 478–484. [<a href="/pubmed/11867360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol&amp;title=Sample+size+requirements+for+association+studies+of+gene-gene+interaction&amp;author=W.J.+Gauderman&amp;volume=155&amp;publication_year=2002&amp;pages=478-484&amp;pmid=11867360&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B42">42. <span class="mixed-citation">
Gao X., Haritunians T., Marjoram P., McKean-Cowdin R., Torres M., Taylor K.D., Rotter J.I., Gauderman W.J., Varma R. (2012) 
<span class="ref-title">Genotype Imputation for Latinos Using the HapMap and 1000 Genomes Project Reference Panels</span>. <span class="ref-journal">Front. Genet</span>., <span class="ref-vol">3</span>, 117.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3384355/">PMC free article</a>]</span> [<a href="/pubmed/22754564" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Genet&amp;title=Genotype+Imputation+for+Latinos+Using+the+HapMap+and+1000+Genomes+Project+Reference+Panels&amp;author=X.+Gao&amp;author=T.+Haritunians&amp;author=P.+Marjoram&amp;author=R.+McKean-Cowdin&amp;author=M.+Torres&amp;volume=3&amp;publication_year=2012&amp;pages=117.&amp;pmid=22754564&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B43">43. <span class="mixed-citation">
Xiao R., Boehnke M. (2009) 
<span class="ref-title">Quantifying and correcting for the winner's curse in genetic association studies</span>. <span class="ref-journal">Genet. Epidemiol</span>., <span class="ref-vol">33</span>, 453–462. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2706290/">PMC free article</a>]</span> [<a href="/pubmed/19140131" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Quantifying+and+correcting+for+the+winner's+curse+in+genetic+association+studies&amp;author=R.+Xiao&amp;author=M.+Boehnke&amp;volume=33&amp;publication_year=2009&amp;pages=453-462&amp;pmid=19140131&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B44">44. <span class="mixed-citation">
Xiao R., Boehnke M. (2011) 
<span class="ref-title">Quantifying and correcting for the winner's curse in quantitative-trait association studies</span>. <span class="ref-journal">Genet. Epidemiol</span>., <span class="ref-vol">35</span>, 133–138. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3500533/">PMC free article</a>]</span> [<a href="/pubmed/21284035" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Quantifying+and+correcting+for+the+winner's+curse+in+quantitative-trait+association+studies&amp;author=R.+Xiao&amp;author=M.+Boehnke&amp;volume=35&amp;publication_year=2011&amp;pages=133-138&amp;pmid=21284035&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B45">45. <span class="mixed-citation">
Sun L., Dimitromanolakis A., Faye L.L., Paterson A.D., Waggott D., Group D.E.R., Bull S.B. (2011) 
<span class="ref-title">BR-squared: a practical solution to the winner's curse in genome-wide scans</span>. <span class="ref-journal">Hum. Genet</span>., <span class="ref-vol">129</span>, 545–552. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3074069/">PMC free article</a>]</span> [<a href="/pubmed/21246217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet&amp;title=BR-squared:+a+practical+solution+to+the+winner's+curse+in+genome-wide+scans&amp;author=L.+Sun&amp;author=A.+Dimitromanolakis&amp;author=L.L.+Faye&amp;author=A.D.+Paterson&amp;author=D.+Waggott&amp;volume=129&amp;publication_year=2011&amp;pages=545-552&amp;pmid=21246217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B46">46. <span class="mixed-citation">
Chang C.C., Chow C.C., Tellier L.C., Vattikuti S., Purcell S.M., Lee J.J. (2015) 
<span class="ref-title">Second-generation PLINK: rising to the challenge of larger and richer datasets</span>. <span class="ref-journal">Gigascience</span>, <span class="ref-vol">4</span>, 7.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4342193/">PMC free article</a>]</span> [<a href="/pubmed/25722852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Second-generation+PLINK:+rising+to+the+challenge+of+larger+and+richer+datasets&amp;author=C.C.+Chang&amp;author=C.C.+Chow&amp;author=L.C.+Tellier&amp;author=S.+Vattikuti&amp;author=S.M.+Purcell&amp;volume=4&amp;publication_year=2015&amp;pages=7.&amp;pmid=25722852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B47">47. <span class="mixed-citation">
Delaneau O., Marchini J., Genomes Project C., Genomes Project C. (2014) 
<span class="ref-title">Integrating sequence and array data to create an improved 1000 Genomes Project haplotype reference panel</span>. <span class="ref-journal">Nat. Commun</span>., <span class="ref-vol">5</span>, 3934.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4338501/">PMC free article</a>]</span> [<a href="/pubmed/25653097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun&amp;title=Integrating+sequence+and+array+data+to+create+an+improved+1000+Genomes+Project+haplotype+reference+panel&amp;author=O.+Delaneau&amp;author=J.+Marchini&amp;author=C.+Genomes+Project&amp;author=C.+Genomes+Project&amp;volume=5&amp;publication_year=2014&amp;pages=3934.&amp;pmid=25653097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B48">48. <span class="mixed-citation">
Patterson N., Price A.L., Reich D. (2006) 
<span class="ref-title">Population structure and eigenanalysis</span>. <span class="ref-journal">PLoS. Genet</span>., <span class="ref-vol">2</span>, e190.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS.+Genet&amp;title=Population+structure+and+eigenanalysis&amp;author=N.+Patterson&amp;author=A.L.+Price&amp;author=D.+Reich&amp;volume=2&amp;publication_year=2006&amp;pages=e190.&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B49">49. <span class="mixed-citation">
Genomes Project C., Abecasis G.R., Auton A., Brooks L.D., DePristo M.A., Durbin R.M., Handsaker R.E., Kang H.M., Marth G.T., McVean G.A. (2012) 
<span class="ref-title">An integrated map of genetic variation from 1,092 human genomes</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">491</span>, 56–65. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1,092+human+genomes&amp;author=C.+Genomes+Project&amp;author=G.R.+Abecasis&amp;author=A.+Auton&amp;author=L.D.+Brooks&amp;author=M.A.+DePristo&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B50">50. <span class="mixed-citation">
Mao X., Bigham A.W., Mei R., Gutierrez G., Weiss K.M., Brutsaert T.D., Leon-Velarde F., Moore L.G., Vargas E., McKeigue P.M., et al. (2007) 
<span class="ref-title">A genomewide admixture mapping panel for Hispanic/Latino populations. <em>Am</em></span>. <span class="ref-journal">J. Hum. Genet</span>., <span class="ref-vol">80</span>, 1171–1178. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1867104/">PMC free article</a>]</span> [<a href="/pubmed/17503334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hum.+Genet&amp;title=A+genomewide+admixture+mapping+panel+for+Hispanic/Latino+populations.+Am&amp;author=X.+Mao&amp;author=A.W.+Bigham&amp;author=R.+Mei&amp;author=G.+Gutierrez&amp;author=K.M.+Weiss&amp;volume=80&amp;publication_year=2007&amp;pages=1171-1178&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B51">51. <span class="mixed-citation">
Devlin B., Roeder K. (1999) 
<span class="ref-title">Genomic control for association studies</span>. <span class="ref-journal">Biometrics</span>, <span class="ref-vol">55</span>, 997–1004. [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B.+Devlin&amp;author=K.+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B52">52. <span class="mixed-citation">
Willer C.J., Li Y., Abecasis G.R. (2010) 
<span class="ref-title">METAL: fast and efficient meta-analysis of genomewide association scans</span>. <span class="ref-journal">Bioinformatics</span>, <span class="ref-vol">26</span>, 2190–2191. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B53">53. <span class="mixed-citation">
Gao X., Becker L.C., Becker D.M., Starmer J.D., Province M.A. (2010) 
<span class="ref-title">Avoiding the high Bonferroni penalty in genome-wide association studies</span>. <span class="ref-journal">Genet. Epidemiol</span>., <span class="ref-vol">34</span>, 100–105. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2796708/">PMC free article</a>]</span> [<a href="/pubmed/19434714" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Avoiding+the+high+Bonferroni+penalty+in+genome-wide+association+studies&amp;author=X.+Gao&amp;author=L.C.+Becker&amp;author=D.M.+Becker&amp;author=J.D.+Starmer&amp;author=M.A.+Province&amp;volume=34&amp;publication_year=2010&amp;pages=100-105&amp;pmid=19434714&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B54">54. <span class="mixed-citation">
R Development Core Team. (2009), In <em>R Foundation for Statistical Computing</em>, Vienna, Austria.</span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B55">55. <span class="mixed-citation">
Pruim R.J., Welch R.P., Sanna S., Teslovich T.M., Chines P.S., Gliedt T.P., Boehnke M., Abecasis G.R., Willer C.J. (2010) 
<span class="ref-title">LocusZoom: regional visualization of genome-wide association scan results</span>. <span class="ref-journal">Bioinformatics</span>, <span class="ref-vol">26</span>, 2336–2337. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=R.J.+Pruim&amp;author=R.P.+Welch&amp;author=S.+Sanna&amp;author=T.M.+Teslovich&amp;author=P.S.+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B56">56. <span class="mixed-citation">
Lee S., Emond M.J., Bamshad M.J., Barnes K.C., Rieder M.J., Nickerson D.A., Team N.G.E.S.P.E.L.P., Christiani D.C., Wurfel M.M., Lin X. (2012) 
<span class="ref-title">Optimal unified approach for rare-variant association testing with application to small-sample case-control whole-exome sequencing studies. <em>Am</em></span>. <span class="ref-journal">J. Hum. Genet</span>., <span class="ref-vol">91</span>, 224–237. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3415556/">PMC free article</a>]</span> [<a href="/pubmed/22863193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hum.+Genet&amp;title=Optimal+unified+approach+for+rare-variant+association+testing+with+application+to+small-sample+case-control+whole-exome+sequencing+studies.+Am&amp;author=S.+Lee&amp;author=M.J.+Emond&amp;author=M.J.+Bamshad&amp;author=K.C.+Barnes&amp;author=M.J.+Rieder&amp;volume=91&amp;publication_year=2012&amp;pages=224-237&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B57">57. <span class="mixed-citation">
Wang J., Duncan D., Shi Z., Zhang B. (2013) 
<span class="ref-title">WEB-based GEne SeT AnaLysis Toolkit (WebGestalt): update 2013</span>. <span class="ref-journal">Nucleic Acids Res</span>., <span class="ref-vol">41</span>, W77–W83. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3692109/">PMC free article</a>]</span> [<a href="/pubmed/23703215" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=WEB-based+GEne+SeT+AnaLysis+Toolkit+(WebGestalt):+update+2013&amp;author=J.+Wang&amp;author=D.+Duncan&amp;author=Z.+Shi&amp;author=B.+Zhang&amp;volume=41&amp;publication_year=2013&amp;pages=W77-W83&amp;pmid=23703215&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw319-B58">58. <span class="mixed-citation">
Benjamini Y., Hochberg Y. (1995) 
<span class="ref-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing</span>. <span class="ref-journal">J. R. Stat. Soc. Series B</span>, <span class="ref-vol">57</span>, 289–300. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+R.+Stat.+Soc.+Series+B&amp;title=Controlling+the+false+discovery+rate:+a+practical+and+powerful+approach+to+multiple+testing&amp;author=Y.+Benjamini&amp;author=Y.+Hochberg&amp;volume=57&amp;publication_year=1995&amp;pages=289-300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078592&amp;issue-id=292036&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140380846808624title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Results</a></li><li><a href="#__sec9title">Discussion</a></li><li><a href="#__sec10title">Materials and Methods</a></li><li><a href="#__sec18title">Supplementary Material</a></li><li><a href="#idm140380845595344title">Acknowledgements</a></li><li><a href="#app1title">Appendix</a></li><li><a href="#__sec20title">Supplementary Material</a></li><li><a href="#idm140380845569952title">References </a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC6078592/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC6078592/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC6078592/pdf/ddw319.pdf">PDF (416K)</a></li> | <li><a href="#" data-citationid="PMC6078592" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6078592%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6078592%2F&amp;text=Genome-wide%20association%20study%20identifies%20WNT7B%20as%20a%20novel%20locus%20for%20central%20corneal%20thickness%20in%20Latinos" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6078592%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="28171582" data-aiid="6078592" data-aid="6078592" data-iid="292036" data-domainid="740" data-domain="hmg" data-accid="PMC6078592" data-md5="1456585669a00035e6f8a5ef5cf59876"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T15:45:09-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A8CB6DD598CE100000000083304D2&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6078592%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC6078592/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>