<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC6059197/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide analysis of disease progression in age-related macular degeneration</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6059197/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide analysis of disease progression in age-related macular degeneration"><meta name="citation_authors" content="Qi Yan, Ying Ding, Yi Liu, Tao Sun, Lars G Fritsche, Traci Clemons, Rinki Ratnapriya, Michael L Klein, Richard J Cook, Yu Liu, Ruzong Fan, Lai Wei, Gonçalo R Abecasis, Anand Swaroop, Emily Y Chew, AREDS2 Research Group, Daniel E Weeks, Wei Chen"><meta name="citation_date" content="1 March 2018"><meta name="citation_issue" content="5"><meta name="citation_volume" content="27"><meta name="citation_firstpage" content="929"><meta name="citation_doi" content="10.1093/hmg/ddy002"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC6059197/?report=abstract"><meta name="citation_pmid" content="29346644"><meta name="DC.Title" content="Genome-wide analysis of disease progression in age-related macular degeneration"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Qi Yan"><meta name="DC.Contributor" content="Ying Ding"><meta name="DC.Contributor" content="Yi Liu"><meta name="DC.Contributor" content="Tao Sun"><meta name="DC.Contributor" content="Lars G Fritsche"><meta name="DC.Contributor" content="Traci Clemons"><meta name="DC.Contributor" content="Rinki Ratnapriya"><meta name="DC.Contributor" content="Michael L Klein"><meta name="DC.Contributor" content="Richard J Cook"><meta name="DC.Contributor" content="Yu Liu"><meta name="DC.Contributor" content="Ruzong Fan"><meta name="DC.Contributor" content="Lai Wei"><meta name="DC.Contributor" content="Gonçalo R Abecasis"><meta name="DC.Contributor" content="Anand Swaroop"><meta name="DC.Contributor" content="Emily Y Chew"><meta name="DC.Contributor" content="AREDS2 Research Group"><meta name="DC.Contributor" content="Daniel E Weeks"><meta name="DC.Contributor" content="Wei Chen"><meta name="DC.Date" content="2018 Mar 1"><meta name="DC.Identifier" content="10.1093/hmg/ddy002"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide analysis of disease progression in age-related macular degeneration"><meta property="og:type" content="article"><meta property="og:description" content="Family- and population-based genetic studies have successfully identified multiple disease-susceptibility loci for Age-related macular degeneration (AMD), one of the first batch and most successful examples of genome-wide association study. However, most ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6059197/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC6059197/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC6059197/pdf/ddy002.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BE73ADD5B4D010000000000CF0081.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6059197%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC6059197</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2018 Mar 1; 27(5): 929–940. </span></div><div><span class="fm-vol-iss-date">Published online 2018 Jan 15. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddy002" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddy002</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC6059197</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/29346644">29346644</a></div></div></div></div><h1 class="content-title">Genome-wide analysis of disease progression in age-related macular degeneration</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Yan%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999860091360" co-class="co-affbox">Qi Yan</a>,<sup>1</sup><sup></sup> <a href="/pubmed/?term=Ding%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999901001648" co-class="co-affbox">Ying Ding</a>,<sup>2</sup><sup></sup> <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999860076048" co-class="co-affbox">Yi Liu</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Sun%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999860073968" co-class="co-affbox">Tao Sun</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Fritsche%20LG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999860071664" co-class="co-affbox">Lars G Fritsche</a>,<sup>3</sup> <a href="/pubmed/?term=Clemons%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999860069840" co-class="co-affbox">Traci Clemons</a>,<sup>4</sup> <a href="/pubmed/?term=Ratnapriya%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999860068016" co-class="co-affbox">Rinki Ratnapriya</a>,<sup>5</sup> <a href="/pubmed/?term=Klein%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999860065856" co-class="co-affbox">Michael L Klein</a>,<sup>6</sup> <a href="/pubmed/?term=Cook%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999860063584" co-class="co-affbox">Richard J Cook</a>,<sup>7</sup> <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999860061312" co-class="co-affbox">Yu Liu</a>,<sup>8</sup> <a href="/pubmed/?term=Fan%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999860059104" co-class="co-affbox">Ruzong Fan</a>,<sup>9</sup> <a href="/pubmed/?term=Wei%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999862151872" co-class="co-affbox">Lai Wei</a>,<sup>8</sup> <a href="/pubmed/?term=Abecasis%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999862149728" co-class="co-affbox">Gonçalo R Abecasis</a>,<sup>3</sup> <a href="/pubmed/?term=Swaroop%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999862147520" co-class="co-affbox">Anand Swaroop</a>,<sup>5</sup> <a href="/pubmed/?term=Chew%20EY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999862145312" co-class="co-affbox">Emily Y Chew</a>,<sup>10</sup> AREDS2 Research Group,<sup>10</sup> <a href="/pubmed/?term=Weeks%20DE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999862141520" co-class="co-affbox">Daniel E Weeks</a>,<sup>2,</sup><sup>11</sup><sup></sup><sup></sup> and  <a href="/pubmed/?term=Chen%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644" class="affpopup" co-rid="_co_idm139999862136688" co-class="co-affbox">Wei Chen</a><sup>1,</sup><sup>2,</sup><sup>11</sup><sup></sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139999860091360"><h3 class="no_margin">Qi Yan</h3><p><sup>1</sup>Division of Pulmonary Medicine, Allergy and Immunology, Department of Pediatrics, Children’s Hospital of Pittsburgh of UPMC University of Pittsburgh, Pittsburgh, PA 15224, USA,</p><div>Find articles by <a href="/pubmed/?term=Yan%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Qi Yan</a></div></div><div id="_co_idm139999901001648"><h3 class="no_margin">Ying Ding</h3><p><sup>2</sup>Department of Biostatistics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, PA 15261, USA</p><div>Find articles by <a href="/pubmed/?term=Ding%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Ying Ding</a></div></div><div id="_co_idm139999860076048"><h3 class="no_margin">Yi Liu</h3><p><sup>1</sup>Division of Pulmonary Medicine, Allergy and Immunology, Department of Pediatrics, Children’s Hospital of Pittsburgh of UPMC University of Pittsburgh, Pittsburgh, PA 15224, USA,</p><p><sup>2</sup>Department of Biostatistics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, PA 15261, USA</p><div>Find articles by <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Yi Liu</a></div></div><div id="_co_idm139999860073968"><h3 class="no_margin">Tao Sun</h3><p><sup>1</sup>Division of Pulmonary Medicine, Allergy and Immunology, Department of Pediatrics, Children’s Hospital of Pittsburgh of UPMC University of Pittsburgh, Pittsburgh, PA 15224, USA,</p><p><sup>2</sup>Department of Biostatistics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, PA 15261, USA</p><div>Find articles by <a href="/pubmed/?term=Sun%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Tao Sun</a></div></div><div id="_co_idm139999860071664"><h3 class="no_margin">Lars G Fritsche</h3><p><sup>3</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Fritsche%20LG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Lars G Fritsche</a></div></div><div id="_co_idm139999860069840"><h3 class="no_margin">Traci Clemons</h3><p><sup>4</sup>The Emmes Corporation, Rockville, MD 20850, USA</p><div>Find articles by <a href="/pubmed/?term=Clemons%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Traci Clemons</a></div></div><div id="_co_idm139999860068016"><h3 class="no_margin">Rinki Ratnapriya</h3><p><sup>5</sup>Neurobiology Neurodegeneration and Repair Laboratory, National Eye Institute, National Institutes of Health, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Ratnapriya%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Rinki Ratnapriya</a></div></div><div id="_co_idm139999860065856"><h3 class="no_margin">Michael L Klein</h3><p><sup>6</sup>Casey Eye Institute, Oregon Health &amp; Science University, Portland, OR 97239, USA</p><div>Find articles by <a href="/pubmed/?term=Klein%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Michael L Klein</a></div></div><div id="_co_idm139999860063584"><h3 class="no_margin">Richard J Cook</h3><p><sup>7</sup>Department of Statistics and Actuarial Science, University of Waterloo, Waterloo, ON N2L 3G1, Canada</p><div>Find articles by <a href="/pubmed/?term=Cook%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Richard J Cook</a></div></div><div id="_co_idm139999860061312"><h3 class="no_margin">Yu Liu</h3><p><sup>8</sup>State Key Laboratory of Ophthalmology, Zhongshan Ophthalmic Center, Sun Yat-sen University, Guangzhou, Guangdong Province 510030, China</p><div>Find articles by <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Yu Liu</a></div></div><div id="_co_idm139999860059104"><h3 class="no_margin">Ruzong Fan</h3><p><sup>9</sup>Department of Biostatistics, Bioinformatics, and Biomathematics, Georgetown University Medical Center, Washington, DC 20007, USA</p><div>Find articles by <a href="/pubmed/?term=Fan%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Ruzong Fan</a></div></div><div id="_co_idm139999862151872"><h3 class="no_margin">Lai Wei</h3><p><sup>8</sup>State Key Laboratory of Ophthalmology, Zhongshan Ophthalmic Center, Sun Yat-sen University, Guangzhou, Guangdong Province 510030, China</p><div>Find articles by <a href="/pubmed/?term=Wei%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Lai Wei</a></div></div><div id="_co_idm139999862149728"><h3 class="no_margin">Gonçalo R Abecasis</h3><p><sup>3</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Abecasis%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Gonçalo R Abecasis</a></div></div><div id="_co_idm139999862147520"><h3 class="no_margin">Anand Swaroop</h3><p><sup>5</sup>Neurobiology Neurodegeneration and Repair Laboratory, National Eye Institute, National Institutes of Health, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Swaroop%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Anand Swaroop</a></div></div><div id="_co_idm139999862145312"><h3 class="no_margin">Emily Y Chew</h3><p><sup>10</sup>Division of Epidemiology and Clinical Applications, National Eye Institute, National Institutes of Health, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Chew%20EY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Emily Y Chew</a></div></div><div id="_co_idm139999862143040"><h3 class="no_margin">AREDS2 Research Group</h3><p><sup>10</sup>Division of Epidemiology and Clinical Applications, National Eye Institute, National Institutes of Health, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">AREDS2 Research Group</a></div></div><div id="_co_idm139999862141520"><h3 class="no_margin">Daniel E Weeks</h3><p><sup>2</sup>Department of Biostatistics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, PA 15261, USA</p><p><sup>11</sup>Department of Human Genetics, Graduate School of Public Health, University of Pittsburgh, PA 15261, USA</p><div>Find articles by <a href="/pubmed/?term=Weeks%20DE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Daniel E Weeks</a></div></div><div id="_co_idm139999862136688"><h3 class="no_margin">Wei Chen</h3><p><sup>1</sup>Division of Pulmonary Medicine, Allergy and Immunology, Department of Pediatrics, Children’s Hospital of Pittsburgh of UPMC University of Pittsburgh, Pittsburgh, PA 15224, USA,</p><p><sup>2</sup>Department of Biostatistics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, PA 15261, USA</p><p><sup>11</sup>Department of Human Genetics, Graduate School of Public Health, University of Pittsburgh, PA 15261, USA</p><div>Find articles by <a href="/pubmed/?term=Chen%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29346644">Wei Chen</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139999864661104_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139999864661104_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139999864661104_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139999864661104_ai" style="display:none"><div class="fm-affl" id="ddy002-aff1" lang="en"><sup>1</sup>Division of Pulmonary Medicine, Allergy and Immunology, Department of Pediatrics, Children’s Hospital of Pittsburgh of UPMC University of Pittsburgh, Pittsburgh, PA 15224, USA,</div><div class="fm-affl" id="ddy002-aff2" lang="en"><sup>2</sup>Department of Biostatistics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, PA 15261, USA</div><div class="fm-affl" id="ddy002-aff3" lang="en"><sup>3</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, MI 48109, USA</div><div class="fm-affl" id="ddy002-aff4" lang="en"><sup>4</sup>The Emmes Corporation, Rockville, MD 20850, USA</div><div class="fm-affl" id="ddy002-aff5" lang="en"><sup>5</sup>Neurobiology Neurodegeneration and Repair Laboratory, National Eye Institute, National Institutes of Health, Bethesda, MD 20892, USA</div><div class="fm-affl" id="ddy002-aff6" lang="en"><sup>6</sup>Casey Eye Institute, Oregon Health &amp; Science University, Portland, OR 97239, USA</div><div class="fm-affl" id="ddy002-aff7" lang="en"><sup>7</sup>Department of Statistics and Actuarial Science, University of Waterloo, Waterloo, ON N2L 3G1, Canada</div><div class="fm-affl" id="ddy002-aff8" lang="en"><sup>8</sup>State Key Laboratory of Ophthalmology, Zhongshan Ophthalmic Center, Sun Yat-sen University, Guangzhou, Guangdong Province 510030, China</div><div class="fm-affl" id="ddy002-aff9" lang="en"><sup>9</sup>Department of Biostatistics, Bioinformatics, and Biomathematics, Georgetown University Medical Center, Washington, DC 20007, USA</div><div class="fm-affl" id="ddy002-aff10" lang="en"><sup>10</sup>Division of Epidemiology and Clinical Applications, National Eye Institute, National Institutes of Health, Bethesda, MD 20892, USA</div><div class="fm-affl" id="ddy002-aff11" lang="en"><sup>11</sup>Department of Human Genetics, Graduate School of Public Health, University of Pittsburgh, PA 15261, USA</div><div id="ddy002-FM1">Qi Yan and Ying Ding contributed equally to this work.</div><div id="ddy002-FM2">Daniel E. Weeks and Wei Chen jointly supervised this work.</div><div id="ddy002-cor1">To whom correspondence should be addressed at: Tel: +1 4126926241; Fax: +1 4126927636; Email: <a href="mailto:dev@null" data-email="ude.phc@nehc.iew" class="oemail">ude.phc@nehc.iew</a> (W.C.); Tel: +1 4126245388; Fax: +1 4126243020; Email: <a href="mailto:dev@null" data-email="ude.ttip@skeew" class="oemail">ude.ttip@skeew</a> (D.E.W.) </div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139999864661104_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2017 Oct 11; Revised 2017 Dec 22; Accepted 2017 Dec 29.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139999864661104_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author(s) 2018. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC6059197/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Figures and Tables.</strong></div><div><a href="/pmc/articles/PMC6059197/bin/supplementtablesfigures_ddy002.pdf" data-ga-action="click_feat_suppl">supplementtablesfigures_ddy002.pdf</a><span style="color:gray"> (1.0M)</span></div><div class="small guid">GUID:&nbsp;35D22747-AA8C-4AB5-BF45-27959A35EDA2</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Appendix 1.</strong></div><div><a href="/pmc/articles/PMC6059197/bin/areds2_research_group_ddy002.pdf" data-ga-action="click_feat_suppl">areds2_research_group_ddy002.pdf</a><span style="color:gray"> (82K)</span></div><div class="small guid">GUID:&nbsp;9A4676C5-6585-491D-8A6D-699D5B1ED2E0</div></div></dd></dl><dl data-length="279" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">All the phenotype data of AREDS participants required in this report were obtained from the public available website dbGap (accession: phs000001.v3.p1). The genotype data on AREDS participants has been reported earlier (<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip">3</a>) and is available from dbGap (accession phs001039.v1.p1).</p></dd></dl></div><div id="idm139999862093808" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139999862093808title">Abstract</h2><!--article-meta--><div><p id="__p3" class="p p-first-last">Family- and population-based genetic studies have successfully identified multiple disease-susceptibility loci for Age-related macular degeneration (AMD), one of the first batch and most successful examples of genome-wide association study. However, most genetic studies to date have focused on case–control studies of late AMD (choroidal neovascularization or geographic atrophy). The genetic influences on disease progression are largely unexplored. We assembled unique resources to perform a genome-wide bivariate time-to-event analysis to test for association of time-to-late-AMD with ∼9 million variants on 2721 Caucasians from a large multi-center randomized clinical trial, the Age-Related Eye Disease Study. To our knowledge, this is the first genome-wide association study of disease progression (bivariate survival outcome) in AMD genetic studies, thus providing novel insights to AMD genetics. We used a robust Cox proportional hazards model to appropriately account for between-eye correlation when analyzing the progression time in the two eyes of each participant. We identified four previously reported susceptibility loci showing genome-wide significant association with AMD progression: <em>ARMS2-HTRA1</em> (<em>P </em>=<em> </em>8.1 × 10<sup>−43</sup>), <em>CFH</em> (<em>P</em> = 3.5 × 10<sup>−37</sup>), <em>C2-CFB-SKIV2L</em> (<em>P</em> = 8.1 × 10<sup>−10</sup>) and <em>C3</em> (<em>P</em> = 1.2 × 10<sup>−9</sup>). Furthermore, we detected association of rs58978565 near <em>TNR</em> (<em>P</em> = 2.3 × 10<sup>−8</sup>), rs28368872 near <em>ATF7IP2</em> (<em>P</em> = 2.9 × 10<sup>−8</sup>) and rs142450006 near <em>MMP9</em> (<em>P</em> = 0.0006) with progression to choroidal neovascularization but not geographic atrophy. Secondary analysis limited to 34 reported risk variants revealed that <em>LIPC</em> and <em>CTRB2-CTRB1</em> were also associated with AMD progression (<em>P</em> &lt; 0.0015). Our genome-wide analysis thus expands the genetics in both development and progression of AMD and should assist in early identification of high risk individuals.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p4" class="p p-first">Age-related macular degeneration (AMD) is a heritable neurodegenerative disease and a leading cause of blindness in the elderly population in the United States. The disease is progressive and irreversible in affecting central vision. The progress starts with appearance of drusen and advances to late AMD, which has two major forms: wet AMD [choroidal neovascularization (CNV)] and dry AMD [geographic atrophy (GA)] (<a href="#ddy002-B1" rid="ddy002-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387538">1</a>).</p><p id="__p5">Patients have different AMD progression rates. Some with early AMD maintain good vision for a long time without advancing to late AMD, while others quickly develop vision-threatening late AMD. Thus, it is critical to identify AMD progression-associated genetic variants to better understand the pathobiology of the disease. However, the large-scale genome-wide studies conducted so far have mainly focused on dichotomous affected/unaffected phenotypes but ignored progression phenotypes, which can be more crucial for understanding the pathways for AMD and for identifying potential therapies. For example, over the past few years, case–control genome-wide association studies (GWASs) have successfully detected multiple single nucleotide polymorphisms (SNPs) associated with AMD susceptibility (<a href="#ddy002-B2" rid="ddy002-B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387545">2–4</a>), in addition to the loci in two well-replicated genes, <em>CFH</em> in chromosome 1 (<a href="#ddy002-B5" rid="ddy002-B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387498">5</a>–<a href="#ddy002-B7" rid="ddy002-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387513">7</a>) and <em>ARMS2</em> in chromosome 10 (<a href="#ddy002-B8" rid="ddy002-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387533">8</a>,<a href="#ddy002-B9" rid="ddy002-B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387511">9</a>). A few of the reported AMD risk variants in <em>CFH</em> and <em>ARMS2</em> appear to have significant effect on AMD progression (<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387539">3</a>,<a href="#ddy002-B10" rid="ddy002-B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387522">10–14</a>). <em>C3</em>, <em>COL8A1</em>, <em>CFB</em> and <em>RAD51B</em> have also been reported to exhibit association with AMD progression in both uni- and multi-variable studies, whereas <em>C2</em> and <em>C9</em> are associated with AMD progression based on uni-variable studies (<a href="#ddy002-B15" rid="ddy002-B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387529">15</a>). However, these studies investigated only a small number of variants. To our knowledge, no GWAS has been performed so far for discovering risk variants for AMD progression.</p><p id="__p6" class="p p-last">Using data from a National Eye Institute-funded large multi-center randomized clinical trial of oral supplementation with micronutrients—the Age-related Eye Disease Study (AREDS) (<a href="#ddy002-B1" rid="ddy002-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387506">1</a>), which was also designed to evaluate the risk factors for the development and progression of AMD (<a href="#ddy002-B1" rid="ddy002-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387502">1</a>), we conducted a genome-wide scan to test for association between time-to-late AMD (either CNV or GA) and genetic variants. Previous studies considered individuals with both eyes free of late AMD at a baseline time and typically defined progression at the level of the subject when assessing genetic effects (<a href="#ddy002-B10" rid="ddy002-B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387537">10</a>,<a href="#ddy002-B16" rid="ddy002-B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387536">16</a>). In such studies, the subject was considered as having “progressed” as soon as one eye progressed to late AMD. As a consequence, the progression time of the second eye to progress was ignored. Instead of defining progression time at the subject level, we defined AMD progression time for each eye and therefore made more complete and efficient use of the available data. To account for the association in the progression times in the two eyes within a subject, robust variance estimates were used for the regression coefficients in the Cox proportional hazards model, treating each subject as a cluster.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Results</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Study data characteristics</h3><p id="__p7" class="p p-first-last">The demographic and clinical characteristics of the 2721 Caucasian participants from AREDS are described in <a href="/pmc/articles/PMC6059197/table/ddy002-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T1" rid-ob="ob-ddy002-T1" co-legend-rid=""><span>Table&nbsp;1</span></a>. About 26% participants were free of AMD at enrollment and the rest entered the study with certain severity level of the disease. The participants were aged between 55- and 81-years old at enrollment (mean ± SD = 68.7 ± 4.9). Women comprised 56% (<em>N</em> = 1527) of the cohort. Most participants received education higher than high school (67%). About 47% of participants were never smokers (<em>N</em> = 1272) and another 47% were former smokers (<em>N</em> = 1288). The mean follow-up time was 10.3 (SD = 1.7) years. The 5017 eyes of the 2721 participants were free of late AMD at baseline and the majority of eyes had low baseline severity score (<em>N</em> = 3125, 62% for baseline severity score between 1 and 3). The rest 425 eyes were already at late AMD at baseline and were excluded from the progression analysis. More participants were randomized to placebo (<em>N</em> = 842, 31%) or antioxidants alone (<em>N</em> = 850, 31%) than zinc alone (<em>N</em> = 507, 19%) or antioxidant plus zinc (<em>N</em> = 522, 19%).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddy002-T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p8">Baseline characteristics of the AREDS cohort</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139999859899872" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Subject-level variables</th><th rowspan="1" colspan="1"><em>N</em> = 2721 subjects</th></tr></thead><tbody><tr><td colspan="2" rowspan="1">Enrollment age, year</td></tr><tr><td rowspan="1" colspan="1"> Mean ± SD</td><td rowspan="1" colspan="1">68.7 ± 4.9</td></tr><tr><td rowspan="1" colspan="1"> Range</td><td rowspan="1" colspan="1" align="center">[55.3, 81.2]</td></tr><tr><td rowspan="1" colspan="1">Female (<em>N</em>, %)</td><td rowspan="1" colspan="1">1527 (56)</td></tr><tr><td rowspan="1" colspan="1">Follow-up time, (mean ± SD)</td><td rowspan="1" colspan="1">10.3 ± 1.7</td></tr><tr><td colspan="2" rowspan="1">Education (<em>N</em>, %)</td></tr><tr><td rowspan="1" colspan="1"> ≤High school</td><td rowspan="1" colspan="1">906 (33)</td></tr><tr><td rowspan="1" colspan="1"> &gt;High school</td><td rowspan="1" colspan="1">1814 (67)</td></tr><tr><td rowspan="1" colspan="1"> Missing</td><td rowspan="1" colspan="1">1 (0)</td></tr><tr><td colspan="2" rowspan="1">Smoking (<em>N</em>, %)</td></tr><tr><td rowspan="1" colspan="1"> Never smoked</td><td rowspan="1" colspan="1">1272 (47)</td></tr><tr><td rowspan="1" colspan="1"> Former smoker</td><td rowspan="1" colspan="1">1288 (47)</td></tr><tr><td rowspan="1" colspan="1"> Current smoker</td><td rowspan="1" colspan="1">161 (6)</td></tr><tr><td colspan="2" rowspan="1">Treatment (<em>N</em>, %)</td></tr><tr><td rowspan="1" colspan="1"> Placebo</td><td rowspan="1" colspan="1">842 (31)</td></tr><tr><td rowspan="1" colspan="1"> Antioxidants alone</td><td rowspan="1" colspan="1">850 (31)</td></tr><tr><td rowspan="1" colspan="1"> Zinc</td><td rowspan="1" colspan="1">507 (19)</td></tr><tr><td rowspan="1" colspan="1"> Antioxidants+zinc</td><td rowspan="1" colspan="1">522 (19)</td></tr><tr><td colspan="2" rowspan="1">AMD categories (<em>N</em>, %)</td></tr><tr><td rowspan="1" colspan="1"> 1</td><td rowspan="1" colspan="1">694 (26)</td></tr><tr><td rowspan="1" colspan="1"> 2</td><td rowspan="1" colspan="1">631 (23)</td></tr><tr><td rowspan="1" colspan="1"> 3</td><td rowspan="1" colspan="1">930 (34)</td></tr><tr><td rowspan="1" colspan="1"> 4</td><td rowspan="1" colspan="1">466 (17)</td></tr><tr><td colspan="2" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">Eye-level variables</td><td rowspan="1" colspan="1" align="center"><em>N</em> = 5017 eyes</td></tr><tr><td colspan="2" rowspan="1"><hr></td></tr><tr><td colspan="2" rowspan="1">Baseline AMD severity score at eye-level</td></tr><tr><td rowspan="1" colspan="1"> Mean ± SD</td><td rowspan="1" colspan="1">3.0 ± 2.3</td></tr><tr><td rowspan="1" colspan="1"> 1–3 (<em>N</em>, %)</td><td rowspan="1" colspan="1">3125 (62)</td></tr><tr><td rowspan="1" colspan="1"> 4–6 (<em>N</em>, %)</td><td rowspan="1" colspan="1">1293 (26)</td></tr><tr><td rowspan="1" colspan="1"> 7–8 (<em>N</em>, %)</td><td rowspan="1" colspan="1">599 (12)</td></tr></tbody></table></div><div id="largeobj_idm139999859899872" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6059197/table/ddy002-T1/?report=objectonly">Open in a separate window</a></div></div></div><div id="__sec4" class="sec"><h3 id="__sec4title">Baseline AMD severity and progression</h3><p id="__p9" class="p p-first-last">It has been shown in Ding <em>et al.</em> (<a href="#ddy002-B14" rid="ddy002-B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387515">14</a>) that the AMD progression rate increases as the eye’s baseline AMD severity increases. Ding <em>et al.</em> (<a href="#ddy002-B14" rid="ddy002-B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387535">14</a>) categorized the eyes that were free of late AMD at baseline into three groups based on their baseline severity score: 1–3 (low), 4–6 (middle) and 7–8 (high). In these 5017 eyes from AREDS, only 3% eyes with baseline severity 1–3 progressed to late AMD by the end of their follow-up, while 40% eyes with baseline severity 4–6 progressed and 81% eyes with baseline severity 7–8 progressed. In addition, within each of the three groups (1–3, 4–6 and 7–8), the progression rate increases as their fellow eyes’ baseline AMD severity increases. Moreover, it has been found that the genetic effect on AMD progression can decrease drastically after adjusting for baseline severity (<a href="#ddy002-B14" rid="ddy002-B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387521">14</a>), which is not surprising given the progression is determined based on this severity score. Thus, we considered two statistical models in our GWAS: (i) with baseline severity score included for controlling its effect on AMD progression, and (ii) without adjusting for baseline severity score to avoid masking certain moderate genetic effects.</p></div><div id="__sec5" class="sec"><h3 id="__sec5title">GWAS of AMD progression</h3><p id="__p10" class="p p-first">In the modeling of AMD progression, based on the univariable Cox models, baseline age, smoking status, education level and baseline severity score (treated as a continuous variable) were selected as covariates. In addition, the first two principal components were also included as covariates to account for any effect of population stratification.</p><p id="__p11">Inspection of genomic control values (λ<sub>BL</sub>=1.02; λ<sub>noBL</sub>=1.01 where BL stands for including baseline severity in the model and noBL is otherwise) suggested that the false positive rate is well-controlled (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Fig. S1</a>). As shown in <a href="/pmc/articles/PMC6059197/figure/ddy002-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F1" rid-ob="ob-ddy002-F1" co-legend-rid="lgnd_ddy002-F1"><span>Figure&nbsp;1</span></a>, rs10922109 in the <em>CFH</em> region (<em>P</em><sub>BL</sub>=1.33 × 10<sup>−8</sup>; <em>P</em><sub>noBL</sub>=3.46 × 10<sup>−37</sup>) and rs2284665 in the <em>ARMS2-HTRA1</em> region (<em>P</em><sub>BL</sub>=6.28 × 10<sup>−13</sup>; <em>P</em><sub>noBL</sub>=8.08 × 10<sup>−43</sup>) reach genome-wide significance (<em>P</em> &lt; 5 × 10<sup>−8</sup>) in both GWASs with and without controlling for baseline severity (<a href="/pmc/articles/PMC6059197/table/ddy002-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T2" rid-ob="ob-ddy002-T2" co-legend-rid=""><span>Table&nbsp;2</span></a>; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Figs S2–S5</a>). The Kaplan-Meier (KM) plots depict clear separations between different genotypes for these two SNPs either in the overall or stratified (by baseline severity) plots (<a href="/pmc/articles/PMC6059197/figure/ddy002-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F2" rid-ob="ob-ddy002-F2" co-legend-rid="lgnd_ddy002-F2"><span>Fig.&nbsp;2A</span></a> and <a href="/pmc/articles/PMC6059197/figure/ddy002-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F2" rid-ob="ob-ddy002-F2" co-legend-rid="lgnd_ddy002-F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">andB).</span></span><span>B</span></a>). In addition, as shown in <a href="/pmc/articles/PMC6059197/figure/ddy002-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F1" rid-ob="ob-ddy002-F1" co-legend-rid="lgnd_ddy002-F1"><span>Figure&nbsp;1B</span></a>, rs116503776 in the <em>C2-CFB-SKIV2L</em> region (<em>P</em><sub>noBL</sub>=8.07 × 10<sup>−10</sup>) and rs2230199 in the <em>C3</em> region (<em>P</em><sub>noBL</sub>=1.20 × 10<sup>−9</sup>) reach genome-wide significance without controlling for baseline severity (<a href="/pmc/articles/PMC6059197/table/ddy002-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T2" rid-ob="ob-ddy002-T2" co-legend-rid=""><span>Table&nbsp;2</span></a>; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Figs S6</a> and <a href="#sup1" rid="sup1" class=" supplementary-material">S7</a>), and these two SNPs show separable curves in the overall KM plots (<a href="/pmc/articles/PMC6059197/figure/ddy002-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F2" rid-ob="ob-ddy002-F2" co-legend-rid="lgnd_ddy002-F2"><span>Fig.&nbsp;2C</span></a> and <a href="/pmc/articles/PMC6059197/figure/ddy002-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F2" rid-ob="ob-ddy002-F2" co-legend-rid="lgnd_ddy002-F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">andD).</span></span><span>D</span></a>). However, in the baseline severity stratified plots, rs116503776 (<em>C2-CFB-SKIV2L</em>) (<a href="/pmc/articles/PMC6059197/figure/ddy002-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F2" rid-ob="ob-ddy002-F2" co-legend-rid="lgnd_ddy002-F2"><span>Fig.&nbsp;2C</span></a>) does not show separable KM curves with low baseline severity while rs2230199 (<em>C3</em>) (<a href="/pmc/articles/PMC6059197/figure/ddy002-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F2" rid-ob="ob-ddy002-F2" co-legend-rid="lgnd_ddy002-F2"><span>Fig.&nbsp;2D</span></a>) does not show separable KM curves with high baseline severity. Minor alleles at rs2284665 (<em>ARMS2-HTRA1</em>; HR<sub>BL</sub> = 1.48; HR<sub>noBL</sub> = 2.06) and rs2230199 (<em>C3</em>; HR<sub>BL</sub> = 1.16; HR<sub>noBL</sub> = 1.45) were associated with faster progression. On the contrary, minor alleles at rs10922109 (<em>CFH</em>; HR<sub>BL</sub> = 0.70; HR<sub>noBL</sub> = 0.43) and rs116503776 (<em>C2-CFB-SKIV2L</em>; HR<sub>BL</sub> = 0.74; HR<sub>noBL</sub> = 0.56) were associated with slower progression. These four loci (<em>CFH</em>, <em>ARMS2-HTRA1</em>, <em>C2-CFB-SKIV2L</em> and <em>C3</em>) were reported to be associated with AMD risk in the consortium case–control studies (<a href="#ddy002-B2" rid="ddy002-B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387526">2</a>,<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387544">3</a>). The hazard ratios for these four loci from our progression analysis were all consistent with the odds ratios from Fritsch <em>et al.</em> (<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387524">3</a>) (<a href="/pmc/articles/PMC6059197/table/ddy002-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T2" rid-ob="ob-ddy002-T2" co-legend-rid=""><span>Table&nbsp;2</span></a>). Moreover, in these four loci, the proportion of minor allele copies was significantly different between progressors and non-progressors (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S1</a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddy002-T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p12">List of loci associated with AMD progression identified in AREDS </p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139999898028640" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">Position</th><th rowspan="1" colspan="1" align="left">Major/ minor allele</th><th rowspan="1" colspan="1" align="left">MAF</th><th rowspan="1" colspan="1" align="left">Gene</th><th colspan="2" rowspan="1">With BL severity<hr></th><th colspan="2" rowspan="1">Without BL severity<hr></th><th rowspan="1" colspan="1" align="left">Fritsche <em>et al.</em> case–control results</th></tr><tr><th colspan="5" rowspan="1"></th><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="left">HR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">HR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td colspan="11" rowspan="1">Significant loci reported also in consortium case–control studies</td></tr><tr><td rowspan="1" colspan="1"> rs2284665</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">124&nbsp;226&nbsp;630</td><td rowspan="1" colspan="1">G/T</td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1"><em>ARMS2-HTRA1</em></td><td rowspan="1" colspan="1">1.48</td><td rowspan="1" colspan="1">6.3 × 10<sup>−13</sup></td><td rowspan="1" colspan="1">2.06</td><td rowspan="1" colspan="1">8.1 × 10<sup>−43</sup></td><td rowspan="1" colspan="1">4.0 × 10<sup>−697</sup></td></tr><tr><td rowspan="1" colspan="1"> <strong>rs10922109</strong></td><td rowspan="1" colspan="1"><strong>1</strong></td><td rowspan="1" colspan="1"><strong>196&nbsp;704&nbsp;632</strong></td><td rowspan="1" colspan="1"><strong>C/A</strong></td><td rowspan="1" colspan="1"><strong>0.33</strong></td><td rowspan="1" colspan="1"><strong><em>CFH</em></strong></td><td rowspan="1" colspan="1"><strong>0.70</strong></td><td rowspan="1" colspan="1"><strong>1.3 × 10<sup>−8</sup></strong></td><td rowspan="1" colspan="1"><strong>0.43</strong></td><td rowspan="1" colspan="1"><strong>3.5 × 10<sup>−37</sup></strong></td><td rowspan="1" colspan="1"><strong>9.6 × 10<sup>−618</sup></strong></td></tr><tr><td rowspan="1" colspan="1"> <strong>rs116503776</strong></td><td rowspan="1" colspan="1"><strong>6</strong></td><td rowspan="1" colspan="1"><strong>31&nbsp;930&nbsp;462</strong></td><td rowspan="1" colspan="1"><strong>G/A</strong></td><td rowspan="1" colspan="1"><strong>0.12</strong></td><td rowspan="1" colspan="1"><strong><em>C2-CFB-SKIV2L</em></strong></td><td rowspan="1" colspan="1"><strong>0.74</strong></td><td rowspan="1" colspan="1"><strong>0.001</strong></td><td rowspan="1" colspan="1"><strong>0.56</strong></td><td rowspan="1" colspan="1"><strong>8.1 × 10<sup>−10</sup></strong></td><td rowspan="1" colspan="1"><strong>1.2 × 10<sup>−103</sup></strong></td></tr><tr><td rowspan="1" colspan="1"> <strong>rs2230199</strong></td><td rowspan="1" colspan="1"><strong>19</strong></td><td rowspan="1" colspan="1"><strong>6&nbsp;718&nbsp;387</strong></td><td rowspan="1" colspan="1"><strong>C/G</strong></td><td rowspan="1" colspan="1"><strong>0.24</strong></td><td rowspan="1" colspan="1"><strong><em>C3</em></strong></td><td rowspan="1" colspan="1"><strong>1.16</strong></td><td rowspan="1" colspan="1"><strong>0.014</strong></td><td rowspan="1" colspan="1"><strong>1.45</strong></td><td rowspan="1" colspan="1"><strong>1.2 × 10<sup>−9</sup></strong></td><td rowspan="1" colspan="1"><strong>3.8 × 10<sup>−69</sup></strong></td></tr><tr><td colspan="11" rowspan="1">Marginally significant novel loci</td></tr><tr><td rowspan="1" colspan="1"> rs79069165</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">75&nbsp;184&nbsp;914</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1"><em>POC5/SV2C</em></td><td rowspan="1" colspan="1">1.52</td><td rowspan="1" colspan="1">7.4 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">1.19</td><td rowspan="1" colspan="1">0.062</td><td rowspan="1" colspan="1">0.724</td></tr><tr><td rowspan="1" colspan="1"> rs74320127</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">64&nbsp;075&nbsp;102</td><td rowspan="1" colspan="1">T/G</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1"><em>HERC1</em></td><td rowspan="1" colspan="1">1.71</td><td rowspan="1" colspan="1">5.3 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">1.15</td><td rowspan="1" colspan="1">0.279</td><td rowspan="1" colspan="1">0.232</td></tr><tr><td rowspan="1" colspan="1"> rs56072732</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">237&nbsp;519&nbsp;496</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1"><em>ACKR3</em></td><td rowspan="1" colspan="1">1.34</td><td rowspan="1" colspan="1">0.002</td><td rowspan="1" colspan="1">1.71</td><td rowspan="1" colspan="1">6.4 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">0.497</td></tr></tbody></table></div><div id="largeobj_idm139999898028640" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6059197/table/ddy002-T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn1"><p id="__p13" class="p p-first-last">MAF, minor allele frequency; BL, baseline severity; HR, hazard ratio relative to the minor allele (minor allele/major allele). Bolded SNPs are also the lead variants in Fritsche <em>et al. </em>(<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387532">3</a>)</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddy002-F1" co-legend-rid="lgnd_ddy002-F1"><a href="/pmc/articles/PMC6059197/figure/ddy002-F1/" target="figure" rid-figpopup="ddy002-F1" rid-ob="ob-ddy002-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139999864988512" class="figure"><a href="/pmc/articles/PMC6059197/figure/ddy002-F1/" target="figure" rid-figpopup="ddy002-F1" rid-ob="ob-ddy002-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6059197_ddy002f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddy002f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6059197/bin/ddy002f1.jpg"></a></div><div id="largeobj_idm139999864988512" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6059197/figure/ddy002-F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddy002-F1"><div><a class="figpopup" href="/pmc/articles/PMC6059197/figure/ddy002-F1/" target="figure" rid-figpopup="ddy002-F1" rid-ob="ob-ddy002-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p14">Manhattan plots of GWAS results of AMD progression. <strong>(A)</strong> GWAS results with adjustment for baseline severity score, and <strong>(B)</strong> GWAS results without adjustment for baseline severity score. Summary of genome-wide association results using Cox model with robust variance for paired eyes. The red horizontal line is the conservative significance level (<em>P</em> = 5 × 10<sup>−8</sup>) and the blue horizontal line is the suggestive significance level (<em>P</em> = 1 × 10<sup>−5</sup>).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddy002-F2" co-legend-rid="lgnd_ddy002-F2"><a href="/pmc/articles/PMC6059197/figure/ddy002-F2/" target="figure" rid-figpopup="ddy002-F2" rid-ob="ob-ddy002-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139999864979232" class="figure"><a href="/pmc/articles/PMC6059197/figure/ddy002-F2/" target="figure" rid-figpopup="ddy002-F2" rid-ob="ob-ddy002-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6059197_ddy002f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddy002f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6059197/bin/ddy002f2.jpg"></a></div><div id="largeobj_idm139999864979232" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6059197/figure/ddy002-F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddy002-F2"><div><a class="figpopup" href="/pmc/articles/PMC6059197/figure/ddy002-F2/" target="figure" rid-figpopup="ddy002-F2" rid-ob="ob-ddy002-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><p id="__p15">KM curves for SNPs from the four known genes identified by case–control studies. <strong>(A)</strong><em>ARMS2-HTRA1</em> (rs2284665), <strong>(B)</strong><em>CFH</em> (rs10922109), <strong>(C)</strong><em>C2-CFB-SKIV2L</em> (rs116503776) and <strong>(D)</strong><em>C3</em> (rs2230199). For all the KM plots (<a href="/pmc/articles/PMC6059197/figure/ddy002-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F2" rid-ob="ob-ddy002-F2" co-legend-rid="lgnd_ddy002-F2"><span>Figures 2, 3,</span></a><a href="/pmc/articles/PMC6059197/figure/ddy002-F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F5" rid-ob="ob-ddy002-F5" co-legend-rid="lgnd_ddy002-F5"><span>5–7</span></a>), the 1st column shows the KM curves of total samples. The second to fourth columns show the KM curves of samples with baseline severity 1–3, 4–6 and 7–8, respectively. The bottom left legend shows the number of samples for each genotype category and the percentage of how many these samples progressed to late AMD. The genotypes were derived from dosages: common homozygotes when dosage ≤ 0.5, heterozygotes when 0.5 &lt; dosage &lt; 1.5 and rare homozygotes when dosage ≥ 1.5.</p></div></div></div><p id="__p16">In addition, we found rs79069165 in the <em>POC5/SV2C</em> region (<em>P</em><sub>BL</sub> = 7.41 × 10<sup>−8</sup>) and rs74320127 in the <em>HERC1</em> region (<em>P</em><sub>BL</sub> = 5.29 × 10<sup>−8</sup>) marginally reach 5 × 10<sup>−8</sup> level in the GWAS with baseline severity adjustment (<a href="/pmc/articles/PMC6059197/table/ddy002-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T2" rid-ob="ob-ddy002-T2" co-legend-rid=""><span>Table&nbsp;2</span></a>; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Figs S8</a> and <a href="#sup1" rid="sup1" class=" supplementary-material">S9</a>), and rs56072732 in the <em>ACKR3</em> region (<em>P</em><sub>noBL</sub> = 6.40 × 10<sup>−8</sup>) marginally reach 5 × 10<sup>−8</sup> level in the GWAS without baseline severity adjustment (<a href="/pmc/articles/PMC6059197/table/ddy002-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T2" rid-ob="ob-ddy002-T2" co-legend-rid=""><span>Table&nbsp;2</span></a>; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Fig. S10</a>). These three SNPs are relatively rare (i.e. 0.05 &lt; MAF &lt; 0.10) and have not been identified by previous case–control studies. <a href="/pmc/articles/PMC6059197/figure/ddy002-F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F3" rid-ob="ob-ddy002-F3" co-legend-rid="lgnd_ddy002-F3"><span>Figure&nbsp;3</span></a> indicates they have more significant effects for subjects with moderate or high baseline severity than low baseline severity, ignoring the rare homozygous group with sample size &lt; 10 (in all three cases). In <em>HERC1</em> (rs74320127) and <em>ACKR3</em> (rs56072732), the proportion of minor allele copies was significantly different between progressors and non-progressors (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S2</a>). All other genes reaching the suggestive significance level (1 × 10<sup>−5</sup>) are listed in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Tables S3</a> and<a href="#sup1" rid="sup1" class=" supplementary-material"> S4</a>.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddy002-F3" co-legend-rid="lgnd_ddy002-F3"><a href="/pmc/articles/PMC6059197/figure/ddy002-F3/" target="figure" rid-figpopup="ddy002-F3" rid-ob="ob-ddy002-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139999864954128" class="figure"><a href="/pmc/articles/PMC6059197/figure/ddy002-F3/" target="figure" rid-figpopup="ddy002-F3" rid-ob="ob-ddy002-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6059197_ddy002f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddy002f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6059197/bin/ddy002f3.jpg"></a></div><div id="largeobj_idm139999864954128" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6059197/figure/ddy002-F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddy002-F3"><div><a class="figpopup" href="/pmc/articles/PMC6059197/figure/ddy002-F3/" target="figure" rid-figpopup="ddy002-F3" rid-ob="ob-ddy002-F3">Figure 3.</a></div><!--caption a7--><div class="caption"><p id="__p17">KM curves for SNPs from the three novel genes. <strong>(A)</strong><em>POC5/SV2C</em> (rs79069165), <strong>(B)</strong><em>HERC1</em> (rs74320127) and <strong>(C)</strong><em>ACKR3</em> (rs56072732).</p></div></div></div><p id="__p18">In addition to the GWAS of progression to any type of late AMD, we conducted two additional GWAS to investigate the effects of genetic variants on progression to different late AMD subtypes: (i) Progression to CNV (including 508 eyes progressed to CNV and 3912 censored eyes) and (ii) Progression to GA (including 615 eyes progressed to GA and 3912 censored eyes). The eyes that progressed to CNV + GA (<em>N</em> = 18) contributed to both subtype analyses. Excluding these 18 eyes led to very similar results (not shown here). We did not adjust for baseline severity in these two subtype GWAS, to avoid missing moderate genetic effects given the sample size is smaller in this subtype analysis. Two SNPs in the <em>TNR</em> and <em>ATF7IP2</em> regions were found to be associated with progression to CNV (rs58978565 in the <em>TNR</em> region: <em>P</em> = 2.31 × 10<sup>−8</sup>, MAF = 0.36 and HR = 1.51; rs28368872 in the <em>ATF7IP2</em> region: <em>P</em> = 2.95 × 10<sup>−8</sup>, MAF = 0.12 and HR = 1.69; <a href="/pmc/articles/PMC6059197/figure/ddy002-F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F4" rid-ob="ob-ddy002-F4" co-legend-rid="lgnd_ddy002-F4"><span>Fig.&nbsp;4A;</span></a><a href="/pmc/articles/PMC6059197/table/ddy002-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T3" rid-ob="ob-ddy002-T3" co-legend-rid=""><span>Table&nbsp;3</span></a>; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Figs S11</a> and <a href="#sup1" rid="sup1" class=" supplementary-material">S12</a>) at the level of 5 × 10<sup>−8</sup>, but not associated with progression to GA (rs58978565 in the <em>TNR</em> region: <em>P</em> = 0.98, MAF = 0.35 and HR = 1.00; rs28368872 in the <em>ATF7IP2</em> region: <em>P</em> = 0.03, MAF = 0.12 and HR = 1.26; <a href="/pmc/articles/PMC6059197/figure/ddy002-F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F4" rid-ob="ob-ddy002-F4" co-legend-rid="lgnd_ddy002-F4"><span>Fig.&nbsp;4B;</span></a><a href="/pmc/articles/PMC6059197/table/ddy002-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T3" rid-ob="ob-ddy002-T3" co-legend-rid=""><span>Table&nbsp;3</span></a>). Both loci did not reach the genome wide significance level in the previous main GWAS without baseline severity (rs58978565 in the <em>TNR</em> region: <em>P</em> = 0.0016; rs28368872 in the <em>ATF7IP2</em> region: <em>P</em> = 8.23 × 10<sup>−6</sup>). The variant rs58978565 in the <em>TNR</em> region is not associated with progression to GA even at a conservative significance level of 0.05, which indicates that <em>TNR</em> is specifically associated with progression to CNV. We refit the models by adjusting for baseline severity on <em>TNR</em> (rs58978565) and <em>ATF7IP2</em> (rs28368872) and obtained similar results (<a href="/pmc/articles/PMC6059197/table/ddy002-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T3" rid-ob="ob-ddy002-T3" co-legend-rid=""><span>Table&nbsp;3</span></a>). The KM plot (<a href="/pmc/articles/PMC6059197/figure/ddy002-F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F5" rid-ob="ob-ddy002-F5" co-legend-rid="lgnd_ddy002-F5"><span>Fig.&nbsp;5A</span></a>) shows a clear separation for rs58978565 in the <em>TNR</em> region for progression to CNV. Such separation for rs58978565 was not observed for progression to GA (<a href="/pmc/articles/PMC6059197/figure/ddy002-F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F5" rid-ob="ob-ddy002-F5" co-legend-rid="lgnd_ddy002-F5"><span>Fig.&nbsp;5B</span></a>). For rs28368872 in the <em>ATF7IP2</em> region, the plots (<a href="/pmc/articles/PMC6059197/figure/ddy002-F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F6" rid-ob="ob-ddy002-F6" co-legend-rid="lgnd_ddy002-F6"><span>Fig.&nbsp;6A</span></a>) shows clear separations for progression to CNV for eyes with baseline severity &gt;4, whereas the separations are moderate for progression to GA (<a href="/pmc/articles/PMC6059197/figure/ddy002-F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F6" rid-ob="ob-ddy002-F6" co-legend-rid="lgnd_ddy002-F6"><span>Fig.&nbsp;6B</span></a>). Moreover, in <em>TNR</em> (rs58978565), the number of minor allele copies is correlated with the proportion of CNV progression but not with the GA progression (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S5</a>). In <em>ATF7IP2</em> (rs28368872), the number of minor allele copies is correlated with the proportion of both CNV and GA progression (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S5</a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddy002-T3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p19">Results for rs58978565 in <em>TNR</em> and rs28368872 in <em>ATF7IP2</em></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139999864916304" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">Position</th><th rowspan="1" colspan="1" align="left">Major/minor allele</th><th rowspan="1" colspan="1" align="left">Gene</th><th rowspan="1" colspan="1" align="left">AMD subtypes</th><th rowspan="1" colspan="1" align="left">MAF</th><th colspan="2" rowspan="1">With BL severity<hr></th><th colspan="2" rowspan="1">Without BL severity<hr></th></tr><tr><th colspan="6" rowspan="1"></th><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1">HR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">HR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs58978565</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">175&nbsp;345&nbsp;602</td><td rowspan="1" colspan="1">C/CAGAGT</td><td rowspan="1" colspan="1"><em>TNR</em></td><td rowspan="1" colspan="1">GA</td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">0.99</td><td rowspan="1" colspan="1">0.83</td><td rowspan="1" colspan="1">1.00</td><td rowspan="1" colspan="1">0.98</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">CNV</td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">1.34</td><td rowspan="1" colspan="1">1.2 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">1.51</td><td rowspan="1" colspan="1">2.3 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">rs28368872</td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">10&nbsp;585&nbsp;350</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1"><em>ATF7IP2</em></td><td rowspan="1" colspan="1">GA</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">1.31</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">1.26</td><td rowspan="1" colspan="1">0.03</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">CNV</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">1.75</td><td rowspan="1" colspan="1">8.0 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">1.69</td><td rowspan="1" colspan="1">2.9 × 10<sup>−8</sup></td></tr></tbody></table></div><div id="largeobj_idm139999864916304" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6059197/table/ddy002-T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn2"><p id="__p20" class="p p-first-last">MAF, minor allele frequency; BL, baseline severity; HR, hazard ratio relative to the minor allele (minor allele/major allele).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddy002-F4" co-legend-rid="lgnd_ddy002-F4"><a href="/pmc/articles/PMC6059197/figure/ddy002-F4/" target="figure" rid-figpopup="ddy002-F4" rid-ob="ob-ddy002-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139999864521152" class="figure"><a href="/pmc/articles/PMC6059197/figure/ddy002-F4/" target="figure" rid-figpopup="ddy002-F4" rid-ob="ob-ddy002-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6059197_ddy002f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddy002f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6059197/bin/ddy002f4.jpg"></a></div><div id="largeobj_idm139999864521152" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6059197/figure/ddy002-F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddy002-F4"><div><a class="figpopup" href="/pmc/articles/PMC6059197/figure/ddy002-F4/" target="figure" rid-figpopup="ddy002-F4" rid-ob="ob-ddy002-F4">Figure 4.</a></div><!--caption a7--><div class="caption"><p id="__p21">Manhattan plots of GWAS results of CNV and GA progression. <strong>(A)</strong> progression to CNV and <strong>(B)</strong> progression to GA. The baseline severity was not included as a covariate. Summary of genome-wide association results using Cox model with robust variance for paired eyes. The red horizontal line is the conservative significance level (<em>P</em> = 5 × 10<sup>−8</sup>) and the blue horizontal line is the suggestive significance level (<em>P</em> = 1 × 10<sup>−5</sup>).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddy002-F5" co-legend-rid="lgnd_ddy002-F5"><a href="/pmc/articles/PMC6059197/figure/ddy002-F5/" target="figure" rid-figpopup="ddy002-F5" rid-ob="ob-ddy002-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139999864517088" class="figure"><a href="/pmc/articles/PMC6059197/figure/ddy002-F5/" target="figure" rid-figpopup="ddy002-F5" rid-ob="ob-ddy002-F5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6059197_ddy002f5.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddy002f5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6059197/bin/ddy002f5.jpg"></a></div><div id="largeobj_idm139999864517088" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6059197/figure/ddy002-F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddy002-F5"><div><a class="figpopup" href="/pmc/articles/PMC6059197/figure/ddy002-F5/" target="figure" rid-figpopup="ddy002-F5" rid-ob="ob-ddy002-F5">Figure 5.</a></div><!--caption a7--><div class="caption"><p id="__p22">KM curves for rs58978565 in <em>TNR</em>. It was associated with progression to CNV at significance level of 5 × 10<sup>−8</sup>, but not with progression to GA.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddy002-F6" co-legend-rid="lgnd_ddy002-F6"><a href="/pmc/articles/PMC6059197/figure/ddy002-F6/" target="figure" rid-figpopup="ddy002-F6" rid-ob="ob-ddy002-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139999864512992" class="figure"><a href="/pmc/articles/PMC6059197/figure/ddy002-F6/" target="figure" rid-figpopup="ddy002-F6" rid-ob="ob-ddy002-F6"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6059197_ddy002f6.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddy002f6.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6059197/bin/ddy002f6.jpg"></a></div><div id="largeobj_idm139999864512992" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6059197/figure/ddy002-F6/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddy002-F6"><div><a class="figpopup" href="/pmc/articles/PMC6059197/figure/ddy002-F6/" target="figure" rid-figpopup="ddy002-F6" rid-ob="ob-ddy002-F6">Figure 6.</a></div><!--caption a7--><div class="caption"><p id="__p23">KM curves for rs28368872 in <em>ATF7IP2</em>. It was associated with progression to CNV at significance level of 5 × 10<sup>−8</sup>, but not with progression to GA.</p></div></div></div><p id="__p24" class="p p-last">We queried pathway databases (GO and KEGG) using DAVID (<a href="#ddy002-B17" rid="ddy002-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387514">17</a>) for 36 genes (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S6</a>) in the nine narrow AMD locus regions (seven loci identified in the main GWAS in <a href="/pmc/articles/PMC6059197/table/ddy002-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T2" rid-ob="ob-ddy002-T2" co-legend-rid=""><span>Table&nbsp;2</span></a> and two additional loci identified in the CNV GWAS in <a href="/pmc/articles/PMC6059197/table/ddy002-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T3" rid-ob="ob-ddy002-T3" co-legend-rid=""><span>Table&nbsp;3</span></a>). Novel identified loci are <em>POC5/SV2C</em>, <em>HERC1/DAPK2</em>, <em>ACKR3/COPS8</em>, <em>TNR</em> and <em>ATF7IP2/EMP2</em>. We found biological pathways enriched across AMD progression associated loci, such as the complement pathway, the protein regulation pathway and the immune response pathway (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S7</a>), highlighting novel genes (<em>ACKR3</em>, <em>EMP2</em> and <em>DAPK2</em>) that connect multiple pathways.</p></div><div id="__sec6" class="sec sec-last"><h3 id="__sec6title">Connection with existing GWAS findings for AMD risk</h3><p id="__p25" class="p p-first">We examined whether the 34 AMD risk variants reported in the Fritsche <em>et al.</em> (<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387543">3</a>) case–control study were also associated with late AMD progression (<a href="/pmc/articles/PMC6059197/table/ddy002-T4/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T4" rid-ob="ob-ddy002-T4" co-legend-rid=""><span>Table&nbsp;4</span></a>). We found that rs10922109 (<em>CFH</em>), rs116503776 (<em>C2-CFB-SKIV2L</em>), rs3750846 (<em>ARMS2-HTRA1</em>), rs2043085 (<em>LIPC</em>), rs2230199 (<em>C3</em>) and rs72802342 (<em>CTRB2-CTRB1</em>) were also associated with late AMD progression at <em>P</em> &lt; 0.0015 (Bonferroni correction for 34 tests). The variant rs10922109 is the most significant SNP in the <em>CFH</em> region in both our late AMD progression study and the consortium case–control study. Most of the 34 SNPs (24 from with baseline severity analyses and 25 from without baseline severity analyses) have the same odds ratio and hazard ratio directions.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddy002-T4"><h3>Table 4.</h3><!--caption a7--><div class="caption"><p id="__p26">Results for 34 AMD risk variants reported in the latest case–control study conducted by the International AMD Genomics Consortium</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139999864487216" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">Position</th><th rowspan="1" colspan="1" align="left">Major/ minor allele</th><th rowspan="1" colspan="1" align="left">MAF</th><th rowspan="1" colspan="1" align="left">Gene</th><th colspan="2" rowspan="1">With BL severity<hr></th><th colspan="2" rowspan="1">Without BL severity<hr></th><th colspan="2" rowspan="1">Fritsche <em>et al.</em> case–control results<hr></th></tr><tr><th colspan="5" rowspan="1"></th><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="left">HR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">HR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td colspan="12" rowspan="1"><strong>Known SNPs identified in consortium case–control studies</strong></td></tr><tr><td rowspan="1" colspan="1"><strong>rs10922109</strong></td><td rowspan="1" colspan="1"><strong>1</strong></td><td rowspan="1" colspan="1"><strong>196&nbsp;704&nbsp;632</strong></td><td rowspan="1" colspan="1"><strong>C/A</strong></td><td rowspan="1" colspan="1"><strong>0.33</strong></td><td rowspan="1" colspan="1"><strong><em>CFH</em></strong></td><td rowspan="1" colspan="1"><strong>0.70</strong></td><td rowspan="1" colspan="1"><strong>1.3 × 10<sup>−8</sup></strong></td><td rowspan="1" colspan="1"><strong>0.43</strong></td><td rowspan="1" colspan="1"><strong>3.5 × 10<sup>−37</sup></strong></td><td rowspan="1" colspan="1"><strong>0.38</strong></td><td rowspan="1" colspan="1"><strong>9.6 × 10<sup>−618</sup></strong></td></tr><tr><td rowspan="1" colspan="1">rs62247658</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">64&nbsp;715&nbsp;155</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.45</td><td rowspan="1" colspan="1"><em>ADAMTS9-AS2</em></td><td rowspan="1" colspan="1">0.93</td><td rowspan="1" colspan="1">0.1798</td><td rowspan="1" colspan="1">0.97</td><td rowspan="1" colspan="1">0.5813</td><td rowspan="1" colspan="1">1.14</td><td rowspan="1" colspan="1">1.8 × 10<sup>−14</sup></td></tr><tr><td rowspan="1" colspan="1">rs140647181</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">99&nbsp;180&nbsp;668</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1"><em>COL8A1</em></td><td rowspan="1" colspan="1">1.62</td><td rowspan="1" colspan="1">0.0549</td><td rowspan="1" colspan="1">1.31</td><td rowspan="1" colspan="1">0.2420</td><td rowspan="1" colspan="1">1.59</td><td rowspan="1" colspan="1">1.4 × 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1">rs10033900</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">110&nbsp;659&nbsp;067</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.50</td><td rowspan="1" colspan="1"><em>CFI</em></td><td rowspan="1" colspan="1">0.98</td><td rowspan="1" colspan="1">0.7502</td><td rowspan="1" colspan="1">0.86</td><td rowspan="1" colspan="1">0.0045</td><td rowspan="1" colspan="1">1.15</td><td rowspan="1" colspan="1">5.4 × 10<sup>−17</sup></td></tr><tr><td rowspan="1" colspan="1">rs62358361</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">39&nbsp;327&nbsp;888</td><td rowspan="1" colspan="1">G/T</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1"><em>C9</em></td><td rowspan="1" colspan="1">1.04</td><td rowspan="1" colspan="1">0.8757</td><td rowspan="1" colspan="1">1.64</td><td rowspan="1" colspan="1">0.0170</td><td rowspan="1" colspan="1">1.8</td><td rowspan="1" colspan="1">1.3 × 10<sup>−14</sup></td></tr><tr><td rowspan="1" colspan="1"><strong>rs116503776</strong></td><td rowspan="1" colspan="1"><strong>6</strong></td><td rowspan="1" colspan="1"><strong>31&nbsp;930&nbsp;462</strong></td><td rowspan="1" colspan="1"><strong>G/A</strong></td><td rowspan="1" colspan="1"><strong>0.12</strong></td><td rowspan="1" colspan="1"><strong><em>C2-CFB-SKIV2L</em></strong></td><td rowspan="1" colspan="1"><strong>0.74</strong></td><td rowspan="1" colspan="1"><strong>0.0011</strong></td><td rowspan="1" colspan="1"><strong>0.56</strong></td><td rowspan="1" colspan="1"><strong>8.1 × 10<sup>−10</sup></strong></td><td rowspan="1" colspan="1"><strong>0.57</strong></td><td rowspan="1" colspan="1"><strong>1.2 × 10<sup>−103</sup></strong></td></tr><tr><td rowspan="1" colspan="1">rs943080</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">43&nbsp;826&nbsp;627</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.48</td><td rowspan="1" colspan="1"><em>VEGFA</em></td><td rowspan="1" colspan="1">1.06</td><td rowspan="1" colspan="1">0.1927</td><td rowspan="1" colspan="1">0.93</td><td rowspan="1" colspan="1">0.2250</td><td rowspan="1" colspan="1">0.88</td><td rowspan="1" colspan="1">1.1 × 10<sup>−14</sup></td></tr><tr><td rowspan="1" colspan="1">rs79037040</td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">23&nbsp;082&nbsp;971</td><td rowspan="1" colspan="1">T/G</td><td rowspan="1" colspan="1">0.46</td><td rowspan="1" colspan="1"><em>TNFRSF10A</em></td><td rowspan="1" colspan="1">1.04</td><td rowspan="1" colspan="1">0.3570</td><td rowspan="1" colspan="1">1.05</td><td rowspan="1" colspan="1">0.3666</td><td rowspan="1" colspan="1">0.9</td><td rowspan="1" colspan="1">4.5 × 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1">rs1626340</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">101&nbsp;923&nbsp;372</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1"><em>TGFBR1</em></td><td rowspan="1" colspan="1">0.92</td><td rowspan="1" colspan="1">0.2422</td><td rowspan="1" colspan="1">0.81</td><td rowspan="1" colspan="1">0.0024</td><td rowspan="1" colspan="1">0.88</td><td rowspan="1" colspan="1">3.8 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1"><strong>rs3750846</strong></td><td rowspan="1" colspan="1"><strong>10</strong></td><td rowspan="1" colspan="1"><strong>124&nbsp;215&nbsp;565</strong></td><td rowspan="1" colspan="1"><strong>T/C</strong></td><td rowspan="1" colspan="1"><strong>0.30</strong></td><td rowspan="1" colspan="1"><strong><em>ARMS2-HTRA1</em></strong></td><td rowspan="1" colspan="1"><strong>1.44</strong></td><td rowspan="1" colspan="1"><strong>1.8 × 10<sup>−11</sup></strong></td><td rowspan="1" colspan="1"><strong>2.04</strong></td><td rowspan="1" colspan="1"><strong>5.3 × 10<sup>−42</sup></strong></td><td rowspan="1" colspan="1"><strong>2.81</strong></td><td rowspan="1" colspan="1"><strong>6.5 × 10<sup>−735</sup></strong></td></tr><tr><td rowspan="1" colspan="1">rs9564692</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">31&nbsp;821&nbsp;240</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1"><em>B3GALTL</em></td><td rowspan="1" colspan="1">0.84</td><td rowspan="1" colspan="1">0.0023</td><td rowspan="1" colspan="1">0.87</td><td rowspan="1" colspan="1">0.0237</td><td rowspan="1" colspan="1">0.89</td><td rowspan="1" colspan="1">3.3 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1">rs61985136</td><td rowspan="1" colspan="1">14</td><td rowspan="1" colspan="1">68&nbsp;769&nbsp;199</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1"><em>RAD51B</em></td><td rowspan="1" colspan="1">0.87</td><td rowspan="1" colspan="1">0.0139</td><td rowspan="1" colspan="1">0.86</td><td rowspan="1" colspan="1">0.0079</td><td rowspan="1" colspan="1">0.9</td><td rowspan="1" colspan="1">1.6 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1"><strong>rs2043085</strong></td><td rowspan="1" colspan="1"><strong>15</strong></td><td rowspan="1" colspan="1"><strong>58&nbsp;680&nbsp;954</strong></td><td rowspan="1" colspan="1"><strong>T/C</strong></td><td rowspan="1" colspan="1"><strong>0.37</strong></td><td rowspan="1" colspan="1"><strong><em>LIPC</em></strong></td><td rowspan="1" colspan="1"><strong>0.88</strong></td><td rowspan="1" colspan="1"><strong>0.0114</strong></td><td rowspan="1" colspan="1"><strong>0.83</strong></td><td rowspan="1" colspan="1"><strong>0.0012</strong></td><td rowspan="1" colspan="1"><strong>0.87</strong></td><td rowspan="1" colspan="1"><strong>4.3 × 10<sup>−15</sup></strong></td></tr><tr><td rowspan="1" colspan="1">rs5817082</td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">56&nbsp;997&nbsp;349</td><td rowspan="1" colspan="1">C/CA</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1"><em>CETP</em></td><td rowspan="1" colspan="1">0.90</td><td rowspan="1" colspan="1">0.0745</td><td rowspan="1" colspan="1">0.89</td><td rowspan="1" colspan="1">0.0632</td><td rowspan="1" colspan="1">0.84</td><td rowspan="1" colspan="1">3.6 × 10<sup>−19</sup></td></tr><tr><td rowspan="1" colspan="1"><strong>rs2230199</strong></td><td rowspan="1" colspan="1"><strong>19</strong></td><td rowspan="1" colspan="1"><strong>6&nbsp;718&nbsp;387</strong></td><td rowspan="1" colspan="1"><strong>C/G</strong></td><td rowspan="1" colspan="1"><strong>0.26</strong></td><td rowspan="1" colspan="1"><strong><em>C3</em></strong></td><td rowspan="1" colspan="1"><strong>1.16</strong></td><td rowspan="1" colspan="1"><strong>0.0139</strong></td><td rowspan="1" colspan="1"><strong>1.45</strong></td><td rowspan="1" colspan="1"><strong>1.2 × 10<sup>−9</sup></strong></td><td rowspan="1" colspan="1"><strong>1.43</strong></td><td rowspan="1" colspan="1"><strong>3.8 × 10<sup>−69</sup></strong></td></tr><tr><td rowspan="1" colspan="1">rs429358</td><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">45&nbsp;411&nbsp;941</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1"><em>APOE</em></td><td rowspan="1" colspan="1">0.95</td><td rowspan="1" colspan="1">0.5298</td><td rowspan="1" colspan="1">0.82</td><td rowspan="1" colspan="1">0.0261</td><td rowspan="1" colspan="1">0.7</td><td rowspan="1" colspan="1">2.4 × 10<sup>−42</sup></td></tr><tr><td rowspan="1" colspan="1">rs5754227</td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">33&nbsp;105&nbsp;817</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1"><em>SYN3-TIMP3</em></td><td rowspan="1" colspan="1">0.91</td><td rowspan="1" colspan="1">0.2080</td><td rowspan="1" colspan="1">0.88</td><td rowspan="1" colspan="1">0.1203</td><td rowspan="1" colspan="1">0.77</td><td rowspan="1" colspan="1">1.1 × 10<sup>−24</sup></td></tr><tr><td rowspan="1" colspan="1">rs8135665</td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">38&nbsp;476&nbsp;276</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1"><em>SLC16A8</em></td><td rowspan="1" colspan="1">0.99</td><td rowspan="1" colspan="1">0.8491</td><td rowspan="1" colspan="1">1.08</td><td rowspan="1" colspan="1">0.2515</td><td rowspan="1" colspan="1">1.14</td><td rowspan="1" colspan="1">5.5 × 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1">rs11884770</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">228&nbsp;086&nbsp;920</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1"><em>COL4A3</em></td><td rowspan="1" colspan="1">0.98</td><td rowspan="1" colspan="1">0.6881</td><td rowspan="1" colspan="1">0.95</td><td rowspan="1" colspan="1">0.4445</td><td rowspan="1" colspan="1">0.9</td><td rowspan="1" colspan="1">2.9 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">rs114092250</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">35&nbsp;494&nbsp;448</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1"><em>PRLR-SPEF2</em></td><td rowspan="1" colspan="1">0.60</td><td rowspan="1" colspan="1">0.0122</td><td rowspan="1" colspan="1">0.59</td><td rowspan="1" colspan="1">0.0280</td><td rowspan="1" colspan="1">0.7</td><td rowspan="1" colspan="1">2.1 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">rs7803454</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">99&nbsp;991&nbsp;548</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1"><em>PILRB-PILRA</em></td><td rowspan="1" colspan="1">1.00</td><td rowspan="1" colspan="1">0.9929</td><td rowspan="1" colspan="1">1.09</td><td rowspan="1" colspan="1">0.1677</td><td rowspan="1" colspan="1">1.13</td><td rowspan="1" colspan="1">4.8 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1">rs1142</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">104&nbsp;756&nbsp;326</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1"><em>KMT2E-SRPK2</em></td><td rowspan="1" colspan="1">1.01</td><td rowspan="1" colspan="1">0.8829</td><td rowspan="1" colspan="1">1.04</td><td rowspan="1" colspan="1">0.5327</td><td rowspan="1" colspan="1">1.11</td><td rowspan="1" colspan="1">1.4 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1">rs71507014</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">73&nbsp;438&nbsp;605</td><td rowspan="1" colspan="1">GC/G</td><td rowspan="1" colspan="1">0.43</td><td rowspan="1" colspan="1"><em>TRPM3</em></td><td rowspan="1" colspan="1">1.01</td><td rowspan="1" colspan="1">0.8196</td><td rowspan="1" colspan="1">1.00</td><td rowspan="1" colspan="1">0.9360</td><td rowspan="1" colspan="1">1.1</td><td rowspan="1" colspan="1">3.0 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">rs10781182</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">76&nbsp;617&nbsp;720</td><td rowspan="1" colspan="1">G/T</td><td rowspan="1" colspan="1">0.32</td><td rowspan="1" colspan="1"><em>MIR6130-RORB</em></td><td rowspan="1" colspan="1">1.15</td><td rowspan="1" colspan="1">0.0104</td><td rowspan="1" colspan="1">1.14</td><td rowspan="1" colspan="1">0.0180</td><td rowspan="1" colspan="1">1.11</td><td rowspan="1" colspan="1">2.6 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1">rs2740488</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">107&nbsp;661&nbsp;742</td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1"><em>ABCA1</em></td><td rowspan="1" colspan="1">0.99</td><td rowspan="1" colspan="1">0.8419</td><td rowspan="1" colspan="1">0.95</td><td rowspan="1" colspan="1">0.4548</td><td rowspan="1" colspan="1">0.9</td><td rowspan="1" colspan="1">1.2 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">rs12357257</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">24&nbsp;999&nbsp;593</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1"><em>ARHGAP21</em></td><td rowspan="1" colspan="1">1.08</td><td rowspan="1" colspan="1">0.2005</td><td rowspan="1" colspan="1">1.18</td><td rowspan="1" colspan="1">0.0057</td><td rowspan="1" colspan="1">1.11</td><td rowspan="1" colspan="1">4.4 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">rs3138141</td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">56&nbsp;115&nbsp;778</td><td rowspan="1" colspan="1">C/A</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1"><em>RDH5-CD63</em></td><td rowspan="1" colspan="1">1.18</td><td rowspan="1" colspan="1">0.0416</td><td rowspan="1" colspan="1">1.19</td><td rowspan="1" colspan="1">0.0338</td><td rowspan="1" colspan="1">1.16</td><td rowspan="1" colspan="1">4.3 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1">rs61941274</td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">112&nbsp;132&nbsp;610</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1"><em>ACAD10</em></td><td rowspan="1" colspan="1">1.12</td><td rowspan="1" colspan="1">0.5593</td><td rowspan="1" colspan="1">0.94</td><td rowspan="1" colspan="1">0.7565</td><td rowspan="1" colspan="1">1.51</td><td rowspan="1" colspan="1">1.1 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1"><strong>rs72802342</strong></td><td rowspan="1" colspan="1"><strong>16</strong></td><td rowspan="1" colspan="1"><strong>75&nbsp;234&nbsp;872</strong></td><td rowspan="1" colspan="1"><strong>C/A</strong></td><td rowspan="1" colspan="1"><strong>0.07</strong></td><td rowspan="1" colspan="1"><strong><em>CTRB2-CTRB1</em></strong></td><td rowspan="1" colspan="1"><strong>0.79</strong></td><td rowspan="1" colspan="1"><strong>0.0570</strong></td><td rowspan="1" colspan="1"><strong>0.67</strong></td><td rowspan="1" colspan="1"><strong>0.0008</strong></td><td rowspan="1" colspan="1"><strong>0.79</strong></td><td rowspan="1" colspan="1"><strong>5.0 × 10<sup>−12</sup></strong></td></tr><tr><td rowspan="1" colspan="1">rs11080055</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">26&nbsp;649&nbsp;724</td><td rowspan="1" colspan="1">C/A</td><td rowspan="1" colspan="1">0.48</td><td rowspan="1" colspan="1"><em>TMEM97-VTN</em></td><td rowspan="1" colspan="1">0.89</td><td rowspan="1" colspan="1">0.0267</td><td rowspan="1" colspan="1">0.97</td><td rowspan="1" colspan="1">0.5847</td><td rowspan="1" colspan="1">0.91</td><td rowspan="1" colspan="1">1.0 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">rs6565597</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">79&nbsp;526&nbsp;821</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.38</td><td rowspan="1" colspan="1"><em>NPLOC4-TSPAN10</em></td><td rowspan="1" colspan="1">1.03</td><td rowspan="1" colspan="1">0.5769</td><td rowspan="1" colspan="1">1.06</td><td rowspan="1" colspan="1">0.2877</td><td rowspan="1" colspan="1">1.13</td><td rowspan="1" colspan="1">1.5 × 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1">rs67538026</td><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">1&nbsp;031&nbsp;438</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.45</td><td rowspan="1" colspan="1"><em>CNN2</em></td><td rowspan="1" colspan="1">0.90</td><td rowspan="1" colspan="1">0.0475</td><td rowspan="1" colspan="1">0.90</td><td rowspan="1" colspan="1">0.0684</td><td rowspan="1" colspan="1">0.9</td><td rowspan="1" colspan="1">2.6 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">rs142450006</td><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">44&nbsp;614&nbsp;991</td><td rowspan="1" colspan="1">TTTTC/T</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1"><em>MMP9</em></td><td rowspan="1" colspan="1">0.77</td><td rowspan="1" colspan="1">0.0021</td><td rowspan="1" colspan="1">0.77</td><td rowspan="1" colspan="1">0.0029</td><td rowspan="1" colspan="1">0.85</td><td rowspan="1" colspan="1">2.4 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1">rs201459901</td><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">56&nbsp;653&nbsp;724</td><td rowspan="1" colspan="1">T/TA</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1"><em>C20orf85</em></td><td rowspan="1" colspan="1">1.07</td><td rowspan="1" colspan="1">0.4688</td><td rowspan="1" colspan="1">0.96</td><td rowspan="1" colspan="1">0.7440</td><td rowspan="1" colspan="1">0.76</td><td rowspan="1" colspan="1">3.1 × 10<sup>−16</sup></td></tr></tbody></table></div><div id="largeobj_idm139999864487216" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6059197/table/ddy002-T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn3"><p id="__p27" class="p p-first-last">Bolded SNPs are those SNPs reaching the Bonferroni corrected significance level = 0.0015 (either with BL severity adjusted or not) in the GWAS of AMD progression. MAF, minor allele frequency; BL, baseline severity; OR, odds ratio; HR, hazard ratio relative to the minor allele (minor allele/major allele). Model was adjusted for baseline age, education, smoking status and first two principal components in addition to the BL severity.</p></div></div></div><p id="__p28" class="p">Of the 34 reported loci (<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387505">3</a>), rs42450006 upstream of <em>MMP9</em> was previously identified in the case–control study to be exclusively associated with CNV but not with GA. That was the first variant identified to be specific to one subtype of AMD. We also confirmed the association for this variant with AMD progression: rs42450006 is associated with CNV progression (<em>P</em> = 0.0006) but not significantly with GA progression (<em>P</em> = 0.17; <a href="/pmc/articles/PMC6059197/table/ddy002-T5/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T5" rid-ob="ob-ddy002-T5" co-legend-rid=""><span>Table&nbsp;5</span></a>; <a href="/pmc/articles/PMC6059197/figure/ddy002-F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddy002-F7" rid-ob="ob-ddy002-F7" co-legend-rid="lgnd_ddy002-F7"><span>Fig.&nbsp;7</span></a>). The <em>MMP9</em> signal for CNV is consistent with previous evidence that upregulation of <em>MMP9</em> induces neovascularization (<a href="#ddy002-B18" rid="ddy002-B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387510">18</a>) and interacts with vascular endothelial growth factor (VEGF) signaling in the retinal pigment epithelial (<a href="#ddy002-B19" rid="ddy002-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387540">19</a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddy002-T5"><h3>Table 5.</h3><!--caption a7--><div class="caption"><p id="__p29">Results for rs142450006 in <em>MMP9</em></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139999900264912" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">Position</th><th rowspan="1" colspan="1" align="left">Major/minor allele</th><th rowspan="1" colspan="1" align="left">Gene</th><th rowspan="1" colspan="1" align="left">AMD subtypes</th><th rowspan="1" colspan="1" align="left">MAF</th><th colspan="2" rowspan="1">With BL severity<hr></th><th colspan="2" rowspan="1">Without BL severity<hr></th></tr><tr><th colspan="6" rowspan="1"></th><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="left">HR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">HR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td colspan="11" rowspan="1"><strong>Significant loci reported also in consortium case–control studies</strong></td></tr><tr><td rowspan="1" colspan="1">rs142450006</td><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">44&nbsp;614&nbsp;991</td><td rowspan="1" colspan="1">TTTTC/T</td><td rowspan="1" colspan="1"><em>MMP9</em></td><td rowspan="1" colspan="1">GA</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">0.83</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.86</td><td rowspan="1" colspan="1">0.17</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">CNV</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.66</td><td rowspan="1" colspan="1">0.0008</td><td rowspan="1" colspan="1">0.66</td><td rowspan="1" colspan="1">0.0006</td></tr></tbody></table></div><div id="largeobj_idm139999900264912" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6059197/table/ddy002-T5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn4"><p id="__p30" class="p p-first-last">MAF, minor allele frequency; BL, baseline severity; HR, hazard ratio relative to the minor allele (minor allele/major allele).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddy002-F7" co-legend-rid="lgnd_ddy002-F7"><a href="/pmc/articles/PMC6059197/figure/ddy002-F7/" target="figure" rid-figpopup="ddy002-F7" rid-ob="ob-ddy002-F7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139999900226288" class="figure"><a href="/pmc/articles/PMC6059197/figure/ddy002-F7/" target="figure" rid-figpopup="ddy002-F7" rid-ob="ob-ddy002-F7"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6059197_ddy002f7.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddy002f7.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6059197/bin/ddy002f7.jpg"></a></div><div id="largeobj_idm139999900226288" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6059197/figure/ddy002-F7/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddy002-F7"><div><a class="figpopup" href="/pmc/articles/PMC6059197/figure/ddy002-F7/" target="figure" rid-figpopup="ddy002-F7" rid-ob="ob-ddy002-F7">Figure 7.</a></div><!--caption a7--><div class="caption"><p id="__p31">KM curves for rs142450006 in <em>MMP9</em>. It was identified by the case–control study to be exclusively associated with CNV.</p></div></div></div></div></div><div id="__sec7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec7title">Discussion</h2><p id="__p32" class="p p-first">We conducted the first GWAS for AMD progression by using a robust Cox proportional hazards regression model. The study of AMD progression complements earlier case–control studies by providing details about the process of AMD development and progression. Instead of using the subject-level analysis used in previous AMD progression papers (<a href="#ddy002-B10" rid="ddy002-B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387504">10</a>,<a href="#ddy002-B16" rid="ddy002-B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387516">16</a>), we performed the analysis at the eye-level to make full use of the data. Our results show that the progression of one eye is strongly dependent on its fellow eye’s progression. The subject-level analysis approach only considers the first progressed eye and ignores the correlation between two eyes. In contrast, we used the robust variance estimate in the Cox model to appropriately account for the correlation between eyes within a subject when assessing the genetic effects on AMD progression, which is consistent with the approach in Ding <em>et al.</em> (<a href="#ddy002-B14" rid="ddy002-B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387500">14</a>).</p><p id="__p33">In this study, we considered two sets of models: with and without adjustment for baseline severity. The baseline non-genetic risk factors are typically included in the GWAS so that the obtained genetic effect has been adjusted for other important risk factors. Thus, we considered the adjustment for baseline severity in addition to baseline age and smoking status. However, in our study the baseline severity is a special risk factor, given the response variable is determined by the longitudinal observations of this severity measurement. Therefore, inclusion of baseline severity can potentially mask the effects of other risk factors, including the genetic effect. Thus, we also considered the GWAS without adjustment for baseline severity.</p><p id="__p34">We identified four progression loci with genome-wide significant association (<em>P</em> &lt; 5 × 10<sup>−8</sup>) (<a href="/pmc/articles/PMC6059197/table/ddy002-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T2" rid-ob="ob-ddy002-T2" co-legend-rid=""><span>Table&nbsp;2</span></a>). All of them were reported in the Fritsche <em>et al.</em> (<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387503">3</a>) case–control study, near <em>ARMS2-HTRA1</em>, <em>CFH</em>, <em>C2-CFB-SKIV2L</em> and <em>C3</em>, and they showed consistent effect directions as in the Fritsche <em>et al.</em> case–control study (<a href="/pmc/articles/PMC6059197/table/ddy002-T4/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T4" rid-ob="ob-ddy002-T4" co-legend-rid=""><span>Table&nbsp;4</span></a>). Of the lead SNPs in the four significant loci, rs10922109 (<em>CFH</em>), rs116503776 (<em>C2-CFB-SKIV2L</em>) and rs2230199 (<em>C3</em>) are also the lead SNPs in their corresponding loci in the Fritsche <em>et al.</em> (<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387534">3</a>) case–control study. The previous AMD progression studies (<a href="#ddy002-B20" rid="ddy002-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387520">20</a>,<a href="#ddy002-B21" rid="ddy002-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387518">21</a>) also showed the association of <em>CFH</em> and <em>ARMS2</em> with AMD progression. In addition, <em>C3</em> and <em>C2</em> regions were reported to be associated with AMD progression in Seddon <em>et al.</em> (<a href="#ddy002-B15" rid="ddy002-B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387519">15</a>), which analyzed 10 SNPs from the AREDS data. Thus, our GWAS results reinforced the understanding of these four gene regions on AMD progression. Although these four susceptibility genes have been consistently identified in case–control studies over years (<a href="#ddy002-B2" rid="ddy002-B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387525">2–4</a>), their exact functional effects on AMD are still unclear. Our finding of the association between these genes and AMD progression can potentially help researchers better understand their functions. Furthermore, the four loci could be used to generate a genetic risk score (GRS) for the prediction of AMD progression. We also identified three additional loci that were marginally significant (<em>P</em> &lt; 1 × 10<sup>−7</sup>) with AMD progression in the <em>POC5/SV2C</em>, <em>HERC1</em> and <em>ACKR3</em> regions (<a href="/pmc/articles/PMC6059197/table/ddy002-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T2" rid-ob="ob-ddy002-T2" co-legend-rid=""><span>Table&nbsp;2</span></a>). These three loci were not identified by previous case–control studies or progression studies. They are relatively rare variants (i.e. 0.05 &lt; MAF &lt; 0.1). Thus, larger sample size studies are needed to verify their significance with AMD progression.</p><p id="__p35">We also identified that the locus at <em>TNR</em> was specifically associated with progression to CNV at the <em>P</em> &lt; 5 × 10<sup>−8</sup> level, but not to GA even at a significance level of 0.05 (<a href="/pmc/articles/PMC6059197/table/ddy002-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T3" rid-ob="ob-ddy002-T3" co-legend-rid=""><span>Table&nbsp;3</span></a>). The locus at <em>ATF7IP2</em> was also found to be associated with progression to CNV at the level of 5 × 10<sup>−8</sup>, but not to GA at <em>P</em> = 5 × 10<sup>−8</sup>. Both loci did not reach genome wide significance level in the GWAS of any type of AMD progression. This finding indicates that the subtypes of AMD progression may have different genetic determinants.</p><p id="__p36">Next, we examined whether the 34 loci reported in Fritsche <em>et al.</em> (<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387509">3</a>) were also associated with AMD progression (<a href="/pmc/articles/PMC6059197/table/ddy002-T4/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T4" rid-ob="ob-ddy002-T4" co-legend-rid=""><span>Table&nbsp;4</span></a>). In addition to the four loci (i.e. <em>ARMS2-HTRA1</em>, <em>CFH</em>, <em>C2-CFB-SKIV2L</em> and <em>C3</em>) associated at <em>P</em> &lt; 5 × 10<sup>−8</sup>, <em>LIPC</em> and <em>CTRB2-CTRB1</em> were also found to be associated with AMD progression (<em>P</em> &lt; 0.0015, Bonferroni corrected significance level). Of the 34 loci, rs42450006 upstream of <em>MMP9</em>, which is the first locus identified to specifically associated with one AMD subtype (i.e. CNV) in a case–control study, was also found to be specifically associated with CNV progression in our study.</p><p id="__p37">In addition to the AREDS data, AREDS2, a subsequent randomized clinical trial of additional oral supplements, was conducted for patients who were at high risk for developing late AMD. However, the AREDS2 data have quite different characteristics from the AREDS data (<a href="#ddy002-B14" rid="ddy002-B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387508">14</a>) (see <a href="/pmc/articles/PMC6059197/table/ddy002-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddy002-T1" rid-ob="ob-ddy002-T1" co-legend-rid=""><span>Table&nbsp;1</span></a> in Ding <em>et al.</em>). The average age of AREDS2 participants is 2.8 years older than in the AREDS participants. The AREDS2 eyes are much more severe than the AREDS eyes at baseline. The majority (53%) of subjects have severity score of 7–8 in AREDS2. In contrast, only 12% subjects have severity score of 7–8 in AREDS. Thus, the AREDS2 subjects could progress to late AMD more easily due to high baseline severity. The genetic effects may not play a substantial role in the progression under this circumstance. We examined whether the four loci identified in the AREDS data are also associated with AMD progression in the AREDS2 data. In <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S8</a>, the results show that only the variant at <em>ARMS2-HTRA1</em> is significant (<em>P</em> &lt; 0.0125, Bonferroni corrected significance level) with and without adjustment for baseline severity.</p><p id="__p38" class="p p-last">Our findings successfully demonstrate the benefit of GWAS of AMD progression as a complement of the case–control GWAS. The genetic study of AMD progression provides new insights for AMD risk investigation, which is helpful for researchers to better understand the AMD pathobiology. The confirmation of existing AMD risk related genes and identification of additional novel genes offer a potentially larger range of targets for AMD therapies. In our previous study (<a href="#ddy002-B14" rid="ddy002-B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387527">14</a>), we derived a GRS) based on 34 risk variants from a case–control study for AMD risk (<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387523">3</a>) to predict AMD progression. In future, it will be more ideal to use the risk variants identified in this AMD progression GWAS to derive a GRS to establish prediction models for AMD progression.</p></div><div id="__sec8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec8title">Materials and Methods</h2><div id="__sec9" class="sec sec-first"><h3 id="__sec9title">Study population and phenotype definition</h3><p id="__p39" class="p p-first">The study subjects are from the AREDS study, which was a multicenter, controlled, randomized clinical trial of treatment effects of antioxidant vitamins and/or zinc with 12 years follow up to assess the risk factors and clinical course of the progression of AMD and age-related cataracts. The AREDS data were part of a larger data set used in the previous consortium case–control studies (<a href="#ddy002-B2" rid="ddy002-B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387541">2</a>,<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387528">3</a>). In this report, we confined our analyses to the AMD data and used Caucasian participants (validated with principal component analysis) with genotype data and at least one follow-up visit.</p><p id="__p40" class="p p-last">In AREDS, the participants were phenotyped based on fundus photographs (graded by a central reading center), best corrected visual acuity and ophthalmologic evaluations. The AMD severity was measured using a validated severity score based on centralized grading of stereoscopic fundus photographs obtained at each semi-annual/annual follow-up visit (<a href="#ddy002-B22" rid="ddy002-B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387501">22</a>). The severity score ranges from 1 to 12, with 12 being the most severe stage. Late AMD is defined as the severity score is equal to 9 (non-central GA) or higher (10: central GA, 11: CNV, and 12: CNV and central GA). For each eye that is free of late AMD at baseline (i.e. baseline severity score &lt;9), the time-to-progression is defined as the time (in years) from the baseline visit to the first visit when the severity score reaches 9 or higher. If the eye’s severity score is &lt;9 by the end of follow-up, time-to-progression is treated as censored at the last visit. The covariates we have considered in the study are: gender, baseline age, education level (≤high school, &gt;high school), baseline smoking status (never, former, current), and treatment group (placebo, antioxidants alone, zinc, antioxidants plus zinc).</p></div><div id="__sec10" class="sec"><h3 id="__sec10title">Genotype data</h3><p id="__p41" class="p p-first-last">DNA samples from consenting subjects in AREDS were collected and genotyped centrally by the International AMD Genomics Consortium (<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387512">3</a>). A custom-modified HumanCoreExome array by Illumina was used to obtain the genotypes followed by imputation with the 1000 Genomes Project reference panel (Phase I). Detailed information about genotype calling and imputation was described previously in (<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387531">3</a>). In this study, we use SNPs with minor allele frequency (MAF) &gt; 0.05. Instead of using genotypes (0: no minor allele; 1: one copy of the minor allele; 2: two copies of the minor allele), we use dosages that can be any number between 0 and 2 in the analysis. The final data set includes 8 974 355 SNPs (265 096 genotyped and 8 709 259 imputed SNPs).</p></div><div id="__sec11" class="sec sec-last"><h3 id="__sec11title">Statistical analysis</h3><p id="__p42" class="p p-first">We use a Cox proportional hazards regression model to test for association between time-to-late AMD and SNPs. In order to account for the intra-subject correlation, a robust variance is used, which is calculated by grouped jackknife using the R function <em>coxph</em> with the ‘cluster’ option (<a href="#ddy002-B23" rid="ddy002-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387507">23</a>), where each subject is treated as a group/cluster (<a href="#ddy002-B24" rid="ddy002-B24" class=" bibr popnode">24</a>). This approach has also been used in Sardell <em>et al.</em> (<a href="#ddy002-B21" rid="ddy002-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387517">21</a>) and Ding <em>et al.</em> (<a href="#ddy002-B14" rid="ddy002-B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387499">14</a>) to analyze AMD progression, which we will refer to as the ‘robust Cox model’. The model formulation involves specification of a marginal Cox proportional hazards model for each eye which can be written as:
</p><div class="disp-formula" id="E1"><div class="f">λ<sub><em>i</em><em>j</em></sub>(<em>t</em>|<em>G</em><sub>i</sub>,&nbsp;<em>X</em><sub><em>i</em><em>j</em></sub>,&nbsp;<em>P</em><em>C</em><sub><em>i</em></sub>) =&nbsp;λ<sub>0</sub>(<em>t</em>)exp{<em>G</em><sub><em>i</em></sub><em>α</em> +&nbsp;<em>X</em><sub><em>i</em><em>j</em></sub><em>β</em> +&nbsp;<em>P</em><em>C</em><sub><em>i</em></sub><em>γ</em>}</div></div><p>
where <span id="IE1"><em>t</em></span> is the progression time, <span id="IE2">λ<sub><em>i</em><em>j</em></sub>(<em>t</em>)</span> is the hazard function of progression time for the <em>j</em>th eye (<em>j</em> = 1: left, 2: right) in the <em>i</em>th subject, <span id="IE3">λ<sub>0</sub>(<em>t</em>)</span> is the baseline hazard function (we assume the baseline hazard function to be the same for left and right eyes), <span id="IE4"><em>G</em><sub><em>i</em></sub></span> (ranging from 0 to 2) is the SNP dosage for the <em>i</em>th subject, representing the copies of minor alleles, <span id="IE5"><em>X</em><sub><em>i</em><em>j</em></sub></span> are the covariates for the <em>j</em>th eye of <em>i</em>th subject and <span id="IE6"><em>P</em><em>C</em><sub><em>i</em></sub></span> is the principal component(s) of ancestry for subject <em>i</em>. Note that the covariates <span id="IE7"><em>X</em><sub><em>i</em><em>j</em></sub></span> may be defined based on either subject-level or eye-level features. The variables that show significant association with AMD progression in the uni-variable robust Cox model with <em>P</em> &lt; 0.1 are considered as covariates in the GWAS analysis.</p><p id="__p43" class="p p-last">In addition, we utilize DAVID (<a href="#ddy002-B17" rid="ddy002-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387530">17</a>) to perform gene-annotation enrichment analysis to highlight the most relevant GO terms and KEGG pathways associated with a list of AMD progression associated genes. This is to test whether the number of AMD progression associated genes included in some relevant GO terms or KEGG pathways is more than expected by chance. A modified Fisher’s exact test is used for this enrichment analysis.</p></div></div><div id="__sec12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec12title">Data availability</h2><p id="__p44" class="p p-first-last">All the phenotype data of AREDS participants required in this report were obtained from the public available website dbGap (accession: phs000001.v3.p1). The genotype data on AREDS participants has been reported earlier (<a href="#ddy002-B3" rid="ddy002-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716387542">3</a>) and is available from dbGap (accession phs001039.v1.p1).</p></div><div id="__sec13" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec13title">Supplementary Material</h2><!--/article/body/sec/--><p id="__p45" class="p p-first-last">
<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material</a> is available at <em>HMG</em> online.</p></div><div id="__sec14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec14title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="sup1"><h4>Supplementary Figures and Tables</h4><div class="sup-box half_rhythm" id="idm139999900086752"><a href="/pmc/articles/PMC6059197/bin/supplementtablesfigures_ddy002.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.0M, pdf)</sup></div></div><div class="sec suppmat" id="sup2"><h4>Supplementary Appendix 1</h4><div class="sup-box half_rhythm" id="idm139999900083504"><a href="/pmc/articles/PMC6059197/bin/areds2_research_group_ddy002.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(82K, pdf)</sup></div></div></div><div id="idm139999900081648" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139999900081648title">Acknowledgements</h2><div class="sec"><p id="__p48">We would like to thank the participants in the AREDS and the AREDS2 investigators, who made this study possible and International AMD Genomics Consortium for generating the genetic data and performing quality checks.</p><p id="__p49">
<em>Conflict of Interest statement.</em> D.E.W. is a co-inventor on licensed patents held by the University of Pittsburgh for the chromosome 10q26 PLEHA1 and ARMS2 loci for AMD. A.S. W.C. and G.R.A. are listed as co-inventors on an AMD patent held by the University of Michigan. L.G.F. is a co-inventor on a licensed patent held by the University of Regensburg, Germany, for the ARMS2 indel variant for AMD.</p><div id="__sec15" class="sec"><h3 id="__sec15title">Funding</h3><p id="__p50" class="p p-first-last">National Institutes of Health (EY024226 to W.C., EY022005 to G.R.A. and EY021532 to M.L.K.); Intramural Research Program of the National Eye Institute (EY000474 and EY000546 to A.S., and AREDS contract NOI-EY-02127 to E.Y.C.); and RPB unrestricted departmental grant (to M.L.K).</p></div></div></div><div id="idm139999900077168" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139999900077168title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="ddy002-B1">1. <span class="mixed-citation">
Age-Related Eye Disease Study Research G. (1999) 
<span class="ref-title">The age-related eye disease study (AREDS): design implications. AREDS report no. 1</span>. <span class="ref-journal">Control. Clin. Trials</span>, <span class="ref-vol">20</span>, 573–600. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1473211/">PMC free article</a>]</span> [<a href="/pubmed/10588299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Control.+Clin.+Trials&amp;title=The+age-related+eye+disease+study+(AREDS):+design+implications.+AREDS+report+no.+1&amp;author=G.+Age-Related+Eye+Disease+Study+Research&amp;volume=20&amp;publication_year=1999&amp;pages=573-600&amp;pmid=10588299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B2">2. <span class="mixed-citation">
Fritsche L.G., Chen W., Schu M., Yaspan B.L., Yu Y., Thorleifsson G., Zack D.J., Arakawa S., Cipriani V., Ripke S.
et al. (2013) 
<span class="ref-title">Seven new loci associated with age-related macular degeneration</span>. <span class="ref-journal">Nat. Genet</span>, <span class="ref-vol">45</span>, 433–439. 439e431-432. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3739472/">PMC free article</a>]</span> [<a href="/pubmed/23455636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Seven+new+loci+associated+with+age-related+macular+degeneration&amp;author=L.G.+Fritsche&amp;author=W.+Chen&amp;author=M.+Schu&amp;author=B.L.+Yaspan&amp;author=Y.+Yu&amp;volume=45&amp;publication_year=2013&amp;pages=433-439&amp;pmid=23455636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B3">3. <span class="mixed-citation">
Fritsche L.G., Igl W., Bailey J.N., Grassmann F., Sengupta S., Bragg-Gresham J.L., Burdon K.P., Hebbring S.J., Wen C., Gorski M.
et al. (2016) 
<span class="ref-title">A large genome-wide association study of age-related macular degeneration highlights contributions of rare and common variants</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">48</span>, 134–143. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4745342/">PMC free article</a>]</span> [<a href="/pubmed/26691988" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+large+genome-wide+association+study+of+age-related+macular+degeneration+highlights+contributions+of+rare+and+common+variants&amp;author=L.G.+Fritsche&amp;author=W.+Igl&amp;author=J.N.+Bailey&amp;author=F.+Grassmann&amp;author=S.+Sengupta&amp;volume=48&amp;publication_year=2016&amp;pages=134-143&amp;pmid=26691988&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B4">4. <span class="mixed-citation">
Chen W., Stambolian D., Edwards A.O., Branham K.E., Othman M., Jakobsdottir J., Tosakulwong N., Pericak-Vance M.A., Campochiaro P.A., Klein M.L.
et al. (2010) 
<span class="ref-title">Genetic variants near TIMP3 and high-density lipoprotein-associated loci influence susceptibility to age-related macular degeneration</span>. <span class="ref-journal">Proc. Natl. Aacd Sci. U.S.A</span>., <span class="ref-vol">107</span>, 7401–7406. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2867722/">PMC free article</a>]</span> [<a href="/pubmed/20385819" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Aacd+Sci.+U.S.A&amp;title=Genetic+variants+near+TIMP3+and+high-density+lipoprotein-associated+loci+influence+susceptibility+to+age-related+macular+degeneration&amp;author=W.+Chen&amp;author=D.+Stambolian&amp;author=A.O.+Edwards&amp;author=K.E.+Branham&amp;author=M.+Othman&amp;volume=107&amp;publication_year=2010&amp;pages=7401-7406&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B5">5. <span class="mixed-citation">
Seddon J.M., Francis P.J., George S., Schultz D.W., Rosner B., Klein M.L. (2007) 
<span class="ref-title">Association of CFH Y402H and LOC387715 A69S with progression of age-related macular degeneration</span>. <span class="ref-journal">JAMA</span>, <span class="ref-vol">297</span>, 1793–1800. [<a href="/pubmed/17456821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Association+of+CFH+Y402H+and+LOC387715+A69S+with+progression+of+age-related+macular+degeneration&amp;author=J.M.+Seddon&amp;author=P.J.+Francis&amp;author=S.+George&amp;author=D.W.+Schultz&amp;author=B.+Rosner&amp;volume=297&amp;publication_year=2007&amp;pages=1793-1800&amp;pmid=17456821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B6">6. <span class="mixed-citation">
Li M., Atmaca-Sonmez P., Othman M., Branham K.E., Khanna R., Wade M.S., Li Y., Liang L., Zareparsi S., Swaroop A.
et al. (2006) 
<span class="ref-title">CFH haplotypes without the Y402H coding variant show strong association with susceptibility to age-related macular degeneration</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">38</span>, 1049–1054. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1941700/">PMC free article</a>]</span> [<a href="/pubmed/16936733" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=CFH+haplotypes+without+the+Y402H+coding+variant+show+strong+association+with+susceptibility+to+age-related+macular+degeneration&amp;author=M.+Li&amp;author=P.+Atmaca-Sonmez&amp;author=M.+Othman&amp;author=K.E.+Branham&amp;author=R.+Khanna&amp;volume=38&amp;publication_year=2006&amp;pages=1049-1054&amp;pmid=16936733&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B7">7. <span class="mixed-citation">
Maller J., George S., Purcell S., Fagerness J., Altshuler D., Daly M.J., Seddon J.M. (2006) 
<span class="ref-title">Common variation in three genes, including a noncoding variant in CFH, strongly influences risk of age-related macular degeneration</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">38</span>, 1055–1059. [<a href="/pubmed/16936732" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Common+variation+in+three+genes,+including+a+noncoding+variant+in+CFH,+strongly+influences+risk+of+age-related+macular+degeneration&amp;author=J.+Maller&amp;author=S.+George&amp;author=S.+Purcell&amp;author=J.+Fagerness&amp;author=D.+Altshuler&amp;volume=38&amp;publication_year=2006&amp;pages=1055-1059&amp;pmid=16936732&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B8">8. <span class="mixed-citation">
Rivera A., Fisher S.A., Fritsche L.G., Keilhauer C.N., Lichtner P., Meitinger T., Weber B.H. (2005) 
<span class="ref-title">Hypothetical LOC387715 is a second major susceptibility gene for age-related macular degeneration, contributing independently of complement factor H to disease risk</span>. <span class="ref-journal">Hum. Mol. Genet</span>., <span class="ref-vol">14</span>, 3227–3236. [<a href="/pubmed/16174643" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Hypothetical+LOC387715+is+a+second+major+susceptibility+gene+for+age-related+macular+degeneration,+contributing+independently+of+complement+factor+H+to+disease+risk&amp;author=A.+Rivera&amp;author=S.A.+Fisher&amp;author=L.G.+Fritsche&amp;author=C.N.+Keilhauer&amp;author=P.+Lichtner&amp;volume=14&amp;publication_year=2005&amp;pages=3227-3236&amp;pmid=16174643&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B9">9. <span class="mixed-citation">
Jakobsdottir J., Conley Y.P., Weeks D.E., Mah T.S., Ferrell R.E., Gorin M.B. (2005) 
<span class="ref-title">Susceptibility genes for age-related maculopathy on chromosome 10q26</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">77</span>, 389–407. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1226205/">PMC free article</a>]</span> [<a href="/pubmed/16080115" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Susceptibility+genes+for+age-related+maculopathy+on+chromosome+10q26&amp;author=J.+Jakobsdottir&amp;author=Y.P.+Conley&amp;author=D.E.+Weeks&amp;author=T.S.+Mah&amp;author=R.E.+Ferrell&amp;volume=77&amp;publication_year=2005&amp;pages=389-407&amp;pmid=16080115&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B10">10. <span class="mixed-citation">
Seddon J.M., Reynolds R., Yu Y., Daly M.J., Rosner B. (2011) 
<span class="ref-title">Risk models for progression to advanced age-related macular degeneration using demographic, environmental, genetic, and ocular factors</span>. <span class="ref-journal">Ophthalmology</span>, <span class="ref-vol">118</span>, 2203–2211. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4097877/">PMC free article</a>]</span> [<a href="/pubmed/21959373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=Risk+models+for+progression+to+advanced+age-related+macular+degeneration+using+demographic,+environmental,+genetic,+and+ocular+factors&amp;author=J.M.+Seddon&amp;author=R.+Reynolds&amp;author=Y.+Yu&amp;author=M.J.+Daly&amp;author=B.+Rosner&amp;volume=118&amp;publication_year=2011&amp;pages=2203-2211&amp;pmid=21959373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B11">11. <span class="mixed-citation">
Seddon J.M., Reynolds R., Maller J., Fagerness J.A., Daly M.J., Rosner B. (2009) 
<span class="ref-title">Prediction model for prevalence and incidence of advanced age-related macular degeneration based on genetic, demographic, and environmental variables</span>. <span class="ref-journal">Invest. Ophthalmol. Vis. Sci</span>., <span class="ref-vol">50</span>, 2044–2053. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772781/">PMC free article</a>]</span> [<a href="/pubmed/19117936" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci&amp;title=Prediction+model+for+prevalence+and+incidence+of+advanced+age-related+macular+degeneration+based+on+genetic,+demographic,+and+environmental+variables&amp;author=J.M.+Seddon&amp;author=R.+Reynolds&amp;author=J.+Maller&amp;author=J.A.+Fagerness&amp;author=M.J.+Daly&amp;volume=50&amp;publication_year=2009&amp;pages=2044-2053&amp;pmid=19117936&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B12">12. <span class="mixed-citation">
Farwick A., Wellmann J., Stoll M., Pauleikhoff D., Hense H.W. (2010) 
<span class="ref-title">Susceptibility genes and progression in age-related maculopathy: a study of single eyes</span>. <span class="ref-journal">Invest. Ophthalmol. Vis. Sci</span>., <span class="ref-vol">51</span>, 731–736. [<a href="/pubmed/19797206" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci&amp;title=Susceptibility+genes+and+progression+in+age-related+maculopathy:+a+study+of+single+eyes&amp;author=A.+Farwick&amp;author=J.+Wellmann&amp;author=M.+Stoll&amp;author=D.+Pauleikhoff&amp;author=H.W.+Hense&amp;volume=51&amp;publication_year=2010&amp;pages=731-736&amp;pmid=19797206&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B13">13. <span class="mixed-citation">
Klein M.L., Francis P.J., Ferris F.L. 3rd, Hamon S.C., Clemons T.E. (2011) 
<span class="ref-title">Risk assessment model for development of advanced age-related macular degeneration</span>. <span class="ref-journal">Arch. Ophthalmol</span>., <span class="ref-vol">129</span>, 1543–1550. [<a href="/pubmed/21825180" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Ophthalmol&amp;title=Risk+assessment+model+for+development+of+advanced+age-related+macular+degeneration&amp;author=M.L.+Klein&amp;author=P.J.+Francis&amp;author=F.L.+Ferris&amp;author=S.C.+Hamon&amp;author=T.E.+Clemons&amp;volume=129&amp;publication_year=2011&amp;pages=1543-1550&amp;pmid=21825180&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B14">14. <span class="mixed-citation">
Ding Y., Liu Y., Yan Q., Fritsche L.G., Cook R.J., Clemons T., Ratnapriya R., Klein M.L., Abecasis G.R., Swaroop A.
et al. (2017) 
<span class="ref-title">Bivariate analysis of age-related macular degeneration progression using genetic risk scores</span>. <span class="ref-journal">Genetics</span>, <span class="ref-vol">206</span>, 119–133. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5419464/">PMC free article</a>]</span> [<a href="/pubmed/28341650" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Bivariate+analysis+of+age-related+macular+degeneration+progression+using+genetic+risk+scores&amp;author=Y.+Ding&amp;author=Y.+Liu&amp;author=Q.+Yan&amp;author=L.G.+Fritsche&amp;author=R.J.+Cook&amp;volume=206&amp;publication_year=2017&amp;pages=119-133&amp;pmid=28341650&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B15">15. <span class="mixed-citation">
Seddon J.M., Silver R.E., Kwong M., Rosner B. (2015) 
<span class="ref-title">Risk prediction for progression of macular degeneration: 10 common and rare genetic variants, demographic, environmental, and macular covariates</span>. <span class="ref-journal">Invest. Ophthalmol. Vis. Sci</span>., <span class="ref-vol">56</span>, 2192–2202. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4405097/">PMC free article</a>]</span> [<a href="/pubmed/25655794" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci&amp;title=Risk+prediction+for+progression+of+macular+degeneration:+10+common+and+rare+genetic+variants,+demographic,+environmental,+and+macular+covariates&amp;author=J.M.+Seddon&amp;author=R.E.+Silver&amp;author=M.+Kwong&amp;author=B.+Rosner&amp;volume=56&amp;publication_year=2015&amp;pages=2192-2202&amp;pmid=25655794&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B16">16. <span class="mixed-citation">
Perlee L.T., Bansal A.T., Gehrs K., Heier J.S., Csaky K., Allikmets R., Oeth P., Paladino T., Farkas D.H., Rawlings P.L.
et al. (2013) 
<span class="ref-title">Inclusion of genotype with fundus phenotype improves accuracy of predicting choroidal neovascularization and geographic atrophy</span>. <span class="ref-journal">Ophthalmology</span>, <span class="ref-vol">120</span>, 1880–1892. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3695024/">PMC free article</a>]</span> [<a href="/pubmed/23523162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=Inclusion+of+genotype+with+fundus+phenotype+improves+accuracy+of+predicting+choroidal+neovascularization+and+geographic+atrophy&amp;author=L.T.+Perlee&amp;author=A.T.+Bansal&amp;author=K.+Gehrs&amp;author=J.S.+Heier&amp;author=K.+Csaky&amp;volume=120&amp;publication_year=2013&amp;pages=1880-1892&amp;pmid=23523162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B17">17. <span class="mixed-citation">
Huang da W., Sherman B.T., Lempicki R.A. (2009) 
<span class="ref-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources</span>. <span class="ref-journal">Nat. Protoc</span>., <span class="ref-vol">4</span>, 44–57. [<a href="/pubmed/19131956" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Protoc&amp;title=Systematic+and+integrative+analysis+of+large+gene+lists+using+DAVID+bioinformatics+resources&amp;author=W.+Huang+da&amp;author=B.T.+Sherman&amp;author=R.A.+Lempicki&amp;volume=4&amp;publication_year=2009&amp;pages=44-57&amp;pmid=19131956&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B18">18. <span class="mixed-citation">
Lambert C., Dubuc J.E., Montell E., Verges J., Munaut C., Noel A., Henrotin Y. (2014) 
<span class="ref-title">Gene expression pattern of cells from inflamed and normal areas of osteoarthritis synovial membrane</span>. <span class="ref-journal">Arthritis Rheum</span>., <span class="ref-vol">66</span>, 960–968. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4033569/">PMC free article</a>]</span> [<a href="/pubmed/24757147" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=Gene+expression+pattern+of+cells+from+inflamed+and+normal+areas+of+osteoarthritis+synovial+membrane&amp;author=C.+Lambert&amp;author=J.E.+Dubuc&amp;author=E.+Montell&amp;author=J.+Verges&amp;author=C.+Munaut&amp;volume=66&amp;publication_year=2014&amp;pages=960-968&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B19">19. <span class="mixed-citation">
Hollborn M., Stathopoulos C., Steffen A., Wiedemann P., Kohen L., Bringmann A. (2007) 
<span class="ref-title">Positive feedback regulation between MMP-9 and VEGF in human RPE cells</span>. <span class="ref-journal">Invest. Ophthalmol Vis. Sci</span>., <span class="ref-vol">48</span>, 4360–4367. [<a href="/pubmed/17724228" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol+Vis.+Sci&amp;title=Positive+feedback+regulation+between+MMP-9+and+VEGF+in+human+RPE+cells&amp;author=M.+Hollborn&amp;author=C.+Stathopoulos&amp;author=A.+Steffen&amp;author=P.+Wiedemann&amp;author=L.+Kohen&amp;volume=48&amp;publication_year=2007&amp;pages=4360-4367&amp;pmid=17724228&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B20">20. <span class="mixed-citation">
Joachim N., Colijn J.M., Kifley A., Lee K.E., Buitendijk G.H., Klein B.E., Myers C.E., Meuer S.M., Tan A.G., Holliday E.G.
et al. (2017) 
<span class="ref-title">Five-year progression of unilateral age-related macular degeneration to bilateral involvement: the Three Continent AMD Consortium report</span>. <span class="ref-journal">Br. J. Ophthalmol</span>., <span class="ref-vol">101</span>, 1185–1192. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5869687/">PMC free article</a>]</span> [<a href="/pubmed/28108569" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Ophthalmol&amp;title=Five-year+progression+of+unilateral+age-related+macular+degeneration+to+bilateral+involvement:+the+Three+Continent+AMD+Consortium+report&amp;author=N.+Joachim&amp;author=J.M.+Colijn&amp;author=A.+Kifley&amp;author=K.E.+Lee&amp;author=G.H.+Buitendijk&amp;volume=101&amp;publication_year=2017&amp;pages=1185-1192&amp;pmid=28108569&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B21">21. <span class="mixed-citation">
Sardell R.J., Persad P.J., Pan S.S., Whitehead P., Adams L.D., Laux R.A., Fortun J.A., Brantley M.A. Jr., Kovach J.L., Schwartz S.G.
et al. (2016) 
<span class="ref-title">Progression rate from intermediate to advanced age-related macular degeneration is correlated with the number of risk alleles at the CFH locus</span>. <span class="ref-journal">Invest. Ophthalmol. Vis. Sci</span>., <span class="ref-vol">57</span>, 6107–6115. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5104418/">PMC free article</a>]</span> [<a href="/pubmed/27832277" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest.+Ophthalmol.+Vis.+Sci&amp;title=Progression+rate+from+intermediate+to+advanced+age-related+macular+degeneration+is+correlated+with+the+number+of+risk+alleles+at+the+CFH+locus&amp;author=R.J.+Sardell&amp;author=P.J.+Persad&amp;author=S.S.+Pan&amp;author=P.+Whitehead&amp;author=L.D.+Adams&amp;volume=57&amp;publication_year=2016&amp;pages=6107-6115&amp;pmid=27832277&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B22">22. <span class="mixed-citation">
Davis M.D., Gangnon R.E., Lee L.Y., Hubbard L.D., Klein B.E., Klein R., Ferris F.L., Bressler S.B., Milton R.C., Age-Related Eye Disease Study G. (2005) 
<span class="ref-title">The Age-Related Eye Disease Study severity scale for age-related macular degeneration: AREDS Report No. 17</span>. <span class="ref-journal">Arch. Ophthalmol</span>., <span class="ref-vol">123</span>, 1484–1498. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1472813/">PMC free article</a>]</span> [<a href="/pubmed/16286610" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Ophthalmol&amp;title=The+Age-Related+Eye+Disease+Study+severity+scale+for+age-related+macular+degeneration:+AREDS+Report+No.+17&amp;author=M.D.+Davis&amp;author=R.E.+Gangnon&amp;author=L.Y.+Lee&amp;author=L.D.+Hubbard&amp;author=B.E.+Klein&amp;volume=123&amp;publication_year=2005&amp;pages=1484-1498&amp;pmid=16286610&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B23">23. <span class="mixed-citation">
Zhang M.J. (2002) 
<span class="ref-title">Cox proportional hazards regression models for survival data in cancer research</span>. <span class="ref-journal">Cancer. Treat. Res</span>., <span class="ref-vol">113</span>, 59–70. [<a href="/pubmed/12613350" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer.+Treat.+Res&amp;title=Cox+proportional+hazards+regression+models+for+survival+data+in+cancer+research&amp;author=M.J.+Zhang&amp;volume=113&amp;publication_year=2002&amp;pages=59-70&amp;pmid=12613350&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddy002-B24">24. <span class="mixed-citation">
Lee E.W., Wei L.J., Amato D. (1992), Cox-type regression analysis for large number of small groups of correlated failure time observations In Klein J. P., Goel P. K. (eds), <span class="ref-journal">Survival Analysis, State of the Art</span>. 
Dordrecht: 
Kluwer Academic Publishers, pp. 237–247. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Survival+Analysis,+State+of+the+Art&amp;author=E.W.+Lee&amp;author=L.J.+Wei&amp;author=D.+Amato&amp;publication_year=1992&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6059197&amp;issue-id=314458&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139999862093808title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Results</a></li><li><a href="#__sec7title">Discussion</a></li><li><a href="#__sec8title">Materials and Methods</a></li><li><a href="#__sec12title">Data availability</a></li><li><a href="#__sec13title">Supplementary Material</a></li><li><a href="#__sec14title">Supplementary Material</a></li><li><a href="#idm139999900081648title">Acknowledgements</a></li><li><a href="#idm139999900077168title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC6059197/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC6059197/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC6059197/pdf/ddy002.pdf">PDF (1.5M)</a></li> | <li><a href="#" data-citationid="PMC6059197" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6059197%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6059197%2F&amp;text=Genome-wide%20analysis%20of%20disease%20progression%20in%20age-related%20macular%20degeneration" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6059197%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="29346644" data-aiid="6059197" data-aid="6059197" data-iid="314458" data-domainid="740" data-domain="hmg" data-accid="PMC6059197" data-md5="3e7c4173f15d51a8e32e3427cd144b91"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T16:55:00-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BE73ADD5B4D010000000000CF0081&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6059197%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC6059197/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>