<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC6041180/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies a regulatory variant of RGMA associated with opioid dependence in European Americans</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6041180/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Biological psychiatry"><meta name="citation_title" content="Genome-wide association study identifies a regulatory variant of RGMA associated with opioid dependence in European Americans"><meta name="citation_authors" content="Zhongshan Cheng, Hang Zhou, Richard Sherva, Lindsay Farrer, Henry R. Kranzler, Joel Gelernter"><meta name="citation_date" content="15 November 2018"><meta name="citation_issue" content="10"><meta name="citation_volume" content="84"><meta name="citation_firstpage" content="762"><meta name="citation_doi" content="10.1016/j.biopsych.2017.12.016"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC6041180/?report=abstract"><meta name="citation_pmid" content="29478698"><meta name="DC.Title" content="Genome-wide association study identifies a regulatory variant of RGMA associated with opioid dependence in European Americans"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Zhongshan Cheng"><meta name="DC.Contributor" content="Hang Zhou"><meta name="DC.Contributor" content="Richard Sherva"><meta name="DC.Contributor" content="Lindsay Farrer"><meta name="DC.Contributor" content="Henry R. Kranzler"><meta name="DC.Contributor" content="Joel Gelernter"><meta name="DC.Date" content="2018 Nov 15"><meta name="DC.Identifier" content="10.1016/j.biopsych.2017.12.016"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies a regulatory variant of RGMA associated with opioid dependence in European Americans"><meta property="og:type" content="article"><meta property="og:description" content="Opioid dependence (OD) is at epidemic levels in the United States. Genetic studies can provide insight into its biology.We completed an OD genome-wide association study (GWAS) in 3058 opioid-exposed European-Americans, 1290 of whom met criteria for a ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6041180/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC6041180/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC6041180/pdf/nihms933995.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8847C0DD5A93A10000000008800505.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6041180%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC6041180</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Biol Psychiatry</a></span>. Author manuscript; available in PMC 2019 Nov 15.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=29478698" target="pmc_ext" ref="reftype=publisher&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Biol Psychiatry. 2018 Nov 15; 84(10): 762–770. </span></a></div><span class="fm-vol-iss-date">Published online 2018 Jan 11. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.biopsych.2017.12.016" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.biopsych.2017.12.016</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC6041180</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS933995</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/29478698">29478698</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies a regulatory variant of <em>RGMA</em> associated with opioid dependence in European Americans</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Cheng%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29478698" class="affpopup" co-rid="_co_idm140355638980416" co-class="co-affbox">Zhongshan Cheng</a>,<sup>1</sup> <a href="/pubmed/?term=Zhou%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29478698" class="affpopup" co-rid="_co_idm140355640183952" co-class="co-affbox">Hang Zhou</a>,<sup>1</sup> <a href="/pubmed/?term=Sherva%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29478698" class="affpopup" co-rid="_co_idm140355642286992" co-class="co-affbox">Richard Sherva</a>,<sup>3</sup> <a href="/pubmed/?term=Farrer%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29478698" class="affpopup" co-rid="_co_idm140355634994624" co-class="co-affbox">Lindsay Farrer</a>,<sup>3</sup> <a href="/pubmed/?term=Kranzler%20HR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29478698" class="affpopup" co-rid="_co_idm140355634992864" co-class="co-affbox">Henry R. Kranzler</a>,<sup>4</sup> and  <a href="/pubmed/?term=Gelernter%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29478698" class="affpopup" co-rid="_co_idm140355629630624" co-class="co-affbox">Joel Gelernter</a><sup>1,</sup><sup>2,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140355638980416"><h3 class="no_margin">Zhongshan Cheng</h3><p>  <sup>1</sup>Yale University School of Medicine, Department of Psychiatry, Division of Human Genetics, and VA CT Healthcare Center</p><div>Find articles by <a href="/pubmed/?term=Cheng%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29478698">Zhongshan Cheng</a></div></div><div id="_co_idm140355640183952"><h3 class="no_margin">Hang Zhou</h3><p>  <sup>1</sup>Yale University School of Medicine, Department of Psychiatry, Division of Human Genetics, and VA CT Healthcare Center</p><div>Find articles by <a href="/pubmed/?term=Zhou%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29478698">Hang Zhou</a></div></div><div id="_co_idm140355642286992"><h3 class="no_margin">Richard Sherva</h3><p>  <sup>3</sup>Boston University Schools of Medicine and Public Health, Departments of Neurology, Ophthalmology, Genetics &amp; Genomics, Epidemiology and Biostatistics</p><div>Find articles by <a href="/pubmed/?term=Sherva%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29478698">Richard Sherva</a></div></div><div id="_co_idm140355634994624"><h3 class="no_margin">Lindsay Farrer</h3><p>  <sup>3</sup>Boston University Schools of Medicine and Public Health, Departments of Neurology, Ophthalmology, Genetics &amp; Genomics, Epidemiology and Biostatistics</p><div>Find articles by <a href="/pubmed/?term=Farrer%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29478698">Lindsay Farrer</a></div></div><div id="_co_idm140355634992864"><h3 class="no_margin">Henry R. Kranzler</h3><p>  <sup>4</sup>University of Pennsylvania Perelman School of Medicine, Department of Psychiatry, Center for Studies of Addiction and Crescenz Veterans Affairs Medical Center</p><div>Find articles by <a href="/pubmed/?term=Kranzler%20HR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29478698">Henry R. Kranzler</a></div></div><div id="_co_idm140355629630624"><h3 class="no_margin">Joel Gelernter</h3><p>  <sup>1</sup>Yale University School of Medicine, Department of Psychiatry, Division of Human Genetics, and VA CT Healthcare Center</p><p>  <sup>2</sup>Yale University School of Medicine Departments of Genetics and Neuroscience</p><div>Find articles by <a href="/pubmed/?term=Gelernter%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29478698">Joel Gelernter</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140355649004320_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140355649004320_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140355649004320_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">  <sup>1</sup>Yale University School of Medicine, Department of Psychiatry, Division of Human Genetics, and VA CT Healthcare Center</div><div class="fm-affl" id="A2" lang="en">  <sup>2</sup>Yale University School of Medicine Departments of Genetics and Neuroscience</div><div class="fm-affl" id="A3" lang="en">  <sup>3</sup>Boston University Schools of Medicine and Public Health, Departments of Neurology, Ophthalmology, Genetics &amp; Genomics, Epidemiology and Biostatistics</div><div class="fm-affl" id="A4" lang="en">  <sup>4</sup>University of Pennsylvania Perelman School of Medicine, Department of Psychiatry, Center for Studies of Addiction and Crescenz Veterans Affairs Medical Center</div><div id="FN1"><sup>*</sup>Correspondence: Joel Gelernter, Yale University School of Medicine, Department of Psychiatry, VA CT 116A2; 950 Campbell Avenue; West Haven, CT 06516; <a href="mailto:dev@null" data-email="ude.elay@retnreleg.leoj" class="oemail">ude.elay@retnreleg.leoj</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140355649004320_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div><a href="#FN3">Publisher's Disclaimer</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=29478698" target="pmc_ext" ref="reftype=publisher&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Biol Psychiatry</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC6041180/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplemental: </strong>Supplementary Figure S1. Percent of opioid exposed samples grouped by different criterion counts among the four cohorts, including SAGE and Yale-Penn 1-3.<p>Supplementary Figure S2. QQ plot for meta-analysis of opioid dependence GWAS.</p><p><strong>Supplementary Figure S3. Sequence alignment of three RGMa isoforms</strong>. Three RGMa isoforms are encoded by six <em>RGMA</em> transcript variants, including <a class="tag_hotlink" href="/nuccore/NM_001166283" ref="/nuccore/NM_001166283"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166283","term_id":"261878462","term_text":"NM_001166283"}}</span>NM_001166283</a> (variant 1), <a class="tag_hotlink" href="/nuccore/NM_001166286" ref="/nuccore/NM_001166286"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166286","term_id":"1677531159","term_text":"NM_001166286"}}</span>NM_001166286</a> (variant 2), <a class="tag_hotlink" href="/nuccore/NM_001166287" ref="/nuccore/NM_001166287"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166287","term_id":"261878460","term_text":"NM_001166287"}}</span>NM_001166287</a> (variant 3), <a class="tag_hotlink" href="/nuccore/NM_020211" ref="/nuccore/NM_020211"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_020211","term_id":"1677530851","term_text":"NM_020211"}}</span>NM_020211</a> (variant 4), <a class="tag_hotlink" href="/nuccore/NM_001166288" ref="/nuccore/NM_001166288"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166288","term_id":"1677538601","term_text":"NM_001166288"}}</span>NM_001166288</a> (variant 5) and <a class="tag_hotlink" href="/nuccore/NM_001166289" ref="/nuccore/NM_001166289"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166289","term_id":"261878466","term_text":"NM_001166289"}}</span>NM_001166289</a> (variant 6) (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip">1</a>). The longest isoform 1 is derived from variant 1. Isoform 2 has the shortest N-terminus and is generated by variants 2, 3, 5 and 6. Isoform 3 is derived from variant 4 and harbors a distinct and shorter N-terminus.</p><p><strong>Supplementary Figure S4. mRNA co-expression analysis between <em>RGMA</em> and six genes affecting psychiatric disorder risk and two transcription factors in brain tissues (n=1301)</strong>., Pairwise Pearson correlation coefficients (r) for <em>RGMA</em> and 8 other genes are depicted in the heatmap (<strong>A</strong>), the -log10(P) correlation P values of which are indicated by different colors. Scatter plots and linear regression for <em>RGMA</em> with 8 other genes are depicted in (<strong>B</strong>). All expression data are normalized z-scores.</p><p>Supplementary Table S1. Sample comparison between Yale-Penn 1 and the Phase 1 and 3 samples published in the 2014 Biological Psychiatry paper (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip">2</a>).</p><p>Supplementary Table S2. Top 10 association signals in European-Americans OD GWAS, and results for those variants in African-Americans</p><p>Table S3. Cis-eQTL analysis of rs12442183 for 11 genes among 10 brain tissues.</p><p>Table S4. The top 10 observed exon probes associated with rs12442183 in Braineac database.</p><p>Table S4. Top 100 co-expressed genes of <em>RGMA</em> identified by COXPRESdb.</p><p>Table S5. The association P values for all exon probes of <em>RGMA</em> with rs12442183 in Braineac database.</p><p>Table S5. Disease enrichment among top 100 co-expressed genes with <em>RGMA</em>.</p><p>Table S6. Transcription factor motif enrichment among top 100 co-expressed genes with <em>RGMA</em>.</p></div><div><a href="/pmc/articles/PMC6041180/bin/NIHMS933995-supplement-Supplemental.pdf" data-ga-action="click_feat_suppl">NIHMS933995-supplement-Supplemental.pdf</a><span style="color:gray"> (629K)</span></div><div class="small guid">GUID:&nbsp;FC50DA32-5934-4324-ABFD-E8A0F581560A</div></div></dd></dl></div><div id="idm140355639567072" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140355639567072title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P1" class="p p-first-last">Opioid dependence (OD) is at epidemic levels in the United States. Genetic studies can provide insight into its biology.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P2" class="p p-first-last">We completed an OD genome-wide association study (GWAS) in 3058 opioid-exposed European-Americans, 1290 of whom met criteria for a DSM-IV diagnosis of OD. Analysis used DSM-IV criterion count.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last">By meta-analysis of four cohorts, Yale-Penn 1 (n=1388), Yale-Penn 2 (n=996), Yale-Penn 3 (n=98) and SAGE (n=576), we identified a variant on chromosome 15, rs12442183, near <em>RGMA</em> (repulsive guidance molecule A (RGMa)), associated with OD (P=1.3×10<sup>-8</sup>). The association was also genome-wide significant (GWS) in Yale-Penn 1 taken individually, and nominally significant in two of the other three samples. The finding was further supported in a meta-analysis of all available opioid-exposed African-Americans (n=2014, 1106 meeting DSM-IV OD criteria; P=3.0×10<sup>-3</sup>) from three cohorts; there was nominal significance in two of these samples. Thus, of the seven subsamples examined in two populations, one was GWS and four of six were nominally significant. RGMa is a CNS axon guidance protein. Risk allele rs12442183*T was correlated with higher expression of a specific <em>RGMA</em> transcript variant in frontal cortex (P=2×10<sup>-3</sup>). After chronic morphine injection, the homologous mouse gene (<em>Rgma</em>) was upregulated in C57BL/6J striatum. Co-expression analysis of 1301 brain samples revealed that <em>RGMA</em> mRNA expression was associated with that of four genes implicated in other psychiatric disorders, including <em>GRIN1</em>.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusion</h3><p id="P4" class="p p-first-last">This is the first study to demonstrate an association of <em>RGMA</em> with OD. It provides a new lead into our understanding of OD pathophysiology.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Genome-wide association, opioid dependence, <em>RGMA</em>, regulatory variant, European-Americans, psychiatric genetics</span></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Introduction</h2><p id="P5" class="p p-first">Opioid dependence (OD) is a major cause of medical, legal, and social problems, resulting in increased mortality and decreased productivity. It is presently at epidemic levels in the United States. OD is a genetically complex disorder that is influenced by complex environmental factors (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842692">1</a>), such as drug availability, making it difficult to identify genetic factors that underlie the disorder. Genome-wide association study (GWAS) is an unbiased way to search for novel genetic factors contributing to OD. We completed the first GWAS of OD (with DSM-IV criterion count as the phenotype definition, using all available subjects) and found two genome-wide significant single nucleotide polymorphisms (SNPs) in African-Americans (AAs) (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842720">2</a>). These two SNPs map to <em>KCNC1</em> (potassium voltage-gated channel subfamily C member 1) and <em>KCNG2</em> (potassium voltage-gated channel modifier subfamily G member 2), respectively, which are involved in potassium signaling pathways. Pathway-based GWAS analysis also demonstrated an enrichment of genes involved in calcium signaling and long-term potentiation (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842730">2</a>). There have been fewer findings in European-Americans (EAs), where the only GWAS to date to identify variants associated to OD in EAs showed an association with <em>CNIH3</em> (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842737">3</a>). Additionally, we used GWAS to identify a variant close to the <em>OPRM1</em> locus that affected opioid dose requirements in AAs (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842704">4</a>). These studies demonstrated that the use of an ordinal trait can provide more statistical power than a categorical diagnosis, as it can take account of affection severity.</p><p id="P6" class="p p-last">In the present study, we used an ordinal trait analysis that was limited to OD-exposed individuals in our GWAS. We identified a variant of <em>RGMA</em>, which encodes the RGM domain family member A (RGMa, which was associated with OD in EAs at a genome-wide significant (GWS) level. The opioid-exposed EA subjects we studied previously (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842756">2</a>) are included in the present study, augmented by additional subjects recruited more recently and we used a new analytic strategy that included only OD-exposed subjects. The finding was then replicated in both EAs and AAs.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Materials and Methods</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Subjects, Genotyping and Imputation</h3><p id="P7" class="p p-first">Subjects in our three cohorts, Yale-Penn 1-3, were recruited from five eastern US sites to participate in studies of the genetics of drug (opioid or cocaine) or alcohol dependence (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> for the specific relationship between the samples used in the present study and our previous study (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842719">2</a>)). These samples contain unrelated individuals and a small proportion of small nuclear families (SNFs) originally collected for linkage studies. All participants provided written informed consent as approved by the institutional review board at each site. Certificates of confidentiality were issued by the National Institute on Drug Abuse and the National Institute on Alcohol Abuse and Alcoholism.</p><p id="P8">Subjects are designated as Yale-Penn 1, 2, and 3 by the recruitment epoch and microarray platform used for genotyping. Samples for Yale-Penn 1 (n=5540) were genotyped on the Illumina (San Diego, CA, USA) HumanOmni1-Quad v1.0 microarray at the Center for Inherited Disease Research and the Yale Center for Genome Analysis. A total of ∼1M SNPs were targeted by microarray in Yale-Penn 1. Samples for Yale-Penn 2 (n=3675) were genotyped with the Illumina HumanCore Exome array, which includes a total of ∼0.5M SNPs, including exomic SNPs and tagging SNPs. Genotyping in Yale-Penn 3 (n=592) was performed with the Illumina Multi-Ethnic Genotyping Array, an array targeting ∼1.7M genome-wide markers and optimized for GWAS in populations of diverse ancestries. Yale-Penn 2 and Yale-Penn 3 genotyping data were obtained at the Gelernter Lab at Yale. Quality control (QC) for microarrays in each cohort were carried out using PLINK1.9 (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842725">5</a>) based on the following criteria: (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842732">1</a>) an individual genotype missing rate &lt;2%, (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842714">2</a>) a SNP genotype missing rate &lt;2%, (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842702">3</a>) a Hardy-Weinberg P&gt;1×10<sup>-6</sup>, and (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842740">4</a>) a minor allele frequency (MAF) &gt;1%. After QC, there were ∼0.70M, ∼0.27M, and ∼1.1M SNPs remaining in Yale-Penn 1-3 respectively, which were subjected to imputation and downstream analysis.</p><p id="P9">GWAS genotypes from all subjects taken separately in each sample group of Yale-Penn 1-3 were compared with the 1000 Genomes Project phase 1 reference panel (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842741">6</a>). Eigensoft (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842742">7</a>) was used for principal components analysis. The first 10 PC scores served to differentiate European-Americans (EAs) and African-Americans (AAs) through K-means clustering (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842760">8</a>). Here, we present GWAS data only for EAs (our previous study presented numerous positive GWAS results in AAs). For each Yale-Penn cohort, genotypes from EAs and AAs were imputed together with Minimac3 implemented in Michigan Imputation Server (<a href="https://imputationserver.sph.umich.edu/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://imputationserver.sph.umich.edu/index.html</a>) with genotyped SNPs and the 1000 Genomes reference phase 3 release that contains haplotypes on 2504 samples (n = 5008 haplotypes for a total of ∼81.2M polymorphic markers). The imputation data for Yale-Penn 1-3 were generated in dosage format separately. Under imputation R<sup>2</sup>&gt;0.3, there was an average of ∼47.1M variants included in dosage data from Yale-Penn 1-3. The dosage genotypes for each cohort were transformed into hard call genotypes with PLINK1.9 (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842707">5</a>). Using hard-call genotypes to analyze imputed GWAS genotypes has been reported previously (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842691">9</a>). In our study, the dosage genotypes for each cohort were transformed into hard call genotypes with PLINK1.9, the rationale for which was to keep only high-quality dosage data by filtering with stricter QC parameters of genotype probability (GP &gt;=0.9) using Plink1.9. The high-quality hard-call genotyping data are easier to QC with Plink1.9 than dosage data, and the genotype data can be directly used in association tests with GEMMA (Genome-wide Efficient Mixed Model Association) (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842744">10</a>) (improving the efficiency of computation), which incorporates and account for the relatedness among our family samples and model the relatedness to increase statistical power. Only genotypes with genotype imputation probability (GP) ≥0.9, an individual genotyping missing rate &lt;5%, MAF &gt;1% and missing call frequency &lt;5% were kept for the association analysis. Following these QC efforts, there were ∼13 M, ∼9.8 M, and ∼5.9 M variants left for Yale-Penn 1-3, respectively.</p><p id="P10" class="p p-last">We also included GWAS data from SAGE (n=4077) (dbGaP study id phs000092.v1.p1) (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842761">11</a>, <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_813751082">12</a>), which were genotyped by the Illumina Human 1M array covering ∼1 M total SNPs. SAGE genotypes were compared with the HapMap3 reference datasets, comprised of CEU, YRI, and CHB populations, and separated into EA and AA populations by using the first PC scores in principal components analysis with Eigensoft. The raw genotypes were imputed with IMPUTE2 (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842695">13</a>). The 1000 Genomes phase 1 reference panel (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842697">6</a>) containing phased haplotypes for 1092 individuals was used for imputation. Dosage genotypes were transformed into hard call genotypes with a custom perl script for heterozygous and homozygous genotypes, i.e., dosage probability for heterozygous genotypes were between 0.8 and 1.4 and homozygous genotypes of reference and alternative alleles ranged from 0 to 0.4 and 1.6 to 2, respectively, with others assigned as missing. The QC criteria for the individual genotyping missing rate &lt;5%, MAF &gt;1% and missing call frequency &lt;5% were also utilized to filter GWAS data. After QC, ∼14 M variants were retained for meta-analysis with Yale-Penn 1-3.</p></div><div id="S8" class="sec"><h3 id="S8title">Analytic Methods</h3><p id="P11" class="p p-first-last">The sum of DSM-IV criterion counts for all EA OD-exposed individuals from the Yale-Penn 1-3 samples were used as an ordinal (quantitative) trait in association analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary figure 1</a>). GEMMA (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842734">10</a>) was performed for each cohort separately, with an adjustment for sex, age, the first three PCs of ancestry, and the degree of relatedness among subjects. In our previous study, we also adjusted for the number of other substance dependence criteria, but did not do so here. This was intended to capture more OD information, at the cost of possible confounding by association driven in part by substance use comorbidities. Summary statistics from GEMMA were reformatted for PLINK1.9 for meta-analysis, and all variants were evaluated by matching their chromosomal positions and two alleles among the four GWAS datasets. Variants with conflicting allele information that could not be resolved by being flipped were excluded from the meta-analysis that used PLINK1.9. The inverse variance method implemented in PLINK1.9 was used to generate fixed-effects meta-analysis P-values (meta-P). In the above process, ∼8 M distinct variants present in at least in one cohort were reported in the meta-analysis; among these, ∼4.7 M variants were present among all four cohorts.</p></div><div id="S9" class="sec"><h3 id="S9title">Correlation between rs12442183 Genotype with <em>RGMA</em> Expression</h3><p id="P12" class="p p-first">The potential correlation between rs12442183 genotype and <em>RGMA</em> expression was tested with publicly available microarray data. First, R package ESLiMc (Exon Splicing by Linear Modeling Analysis core) (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842700">14</a>) was used to analyze alternative expression of <em>RGMA</em> transcripts among 11 different healthy human tissues, including breast, cerebellum, heart, kidney, liver, muscle, pancreas, prostate, spleen, testes, and thyroid, data from which were obtained from <a href="http://www.affymetrix.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.affymetrix.com</a> and included in ESLiMc analysis by its author (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842757">14</a>). The rationale for using ESLiMc was that if there are no alternative splicing events, an increase in the global expression of a gene should correlate with a higher expression of all of its exons. Linear regression was used by ESLiMc to establish the above relationship between each exon and its corresponding gene. The difference (referred to as gene-exon residual regression score) generated by ESLiMc between the observed exon expression and the predicted exon expression was interpreted as being due to an alternative splicing (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842709">14</a>). Next, we tested the correlation between rs12442183 genotype and the expression of <em>RGMA</em> transcript variants. We downloaded rs12442183 genotype information from BRAINEAC (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842759">15</a>) and reprocessed its corresponding microarray data from GEO (Gene Expression Omnibus) (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842753">16</a>) (accession No. <a id="__tag_760842764" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE60863" ref="/geo/query/acc.cgi?acc=GSE60863"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE60863","term_id":"60863"}}</span>GSE60863</a>) (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842718">17</a>) with R package ESLiMc. <a id="__tag_760842768" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE60863" ref="/geo/query/acc.cgi?acc=GSE60863"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE60863","term_id":"60863"}}</span>GSE60863</a> is comprised of 1231 samples originating from 10 brain tissues (occipital cortex, frontal cortex, temporal cortex, hippocampus, intralobular white matter, cerebellar cortex, thalamus, putamen, substantia nigra, and medulla (inferior olivary nucleus)) derived from 134 healthy human brains from UK Biobank (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842696">17</a>). These brain samples were confirmed to be free of neurodegenerative disorders and assayed by Affymetrix Human Exon 1.0 ST array. We also used PExFIns (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842746">18</a>), an expression quantitative locus (eQTL) analysis software package, to test the association of rs12442183 genotype with <em>RGMA</em> mRNA expression in 423 lymphoblastoid cell lines (LCLs) across 6 global populations, including 73 CEU (Utah residents with northern and western European ancestry), 77 CHB (Han Chinese in Beijing, China), 72 JPT (Japanese in Tokyo, Japan), 80 LWK (Luhya in Webuye, Kenya), 42 MEX (Mexican ancestry in Los Angeles) and 79 YRI (Yoruba in Ibadan, Nigeria).</p><p id="P13" class="p p-last">Besides <em>RGMA</em>, we also tested the association between rs12442183 genotype and the expression of other genes close to the SNP at both mRNA and exon level (see Integrative cis-eQTL Analysis in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Information</a>).</p></div><div id="S10" class="sec"><h3 id="S10title">Effect of Morphine Injection on Mouse <em>Rgma</em> Expression</h3><p id="P14" class="p p-first-last">The publicly available microarray experiment dataset <a id="__tag_760842765" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE7762" ref="/geo/query/acc.cgi?acc=GSE7762"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE7762","term_id":"7762"}}</span>GSE7762</a> was designed to determine the impact of genetic background on the transcriptional effects of morphine in mouse striatum (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842724">19</a>). It used different morphine treatments, including a single morphine injection of 20 mg/kg, repeated (3 times daily for 5 days) morphine injections of 10-40 mg/kg, and saline, in four inbred mouse strains (129P3/J, DBA/2J, C57BL/6J and SWR/J). We used the R package GEOquery (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842749">20</a>) to download the normalized <a id="__tag_760842766" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE7762" ref="/geo/query/acc.cgi?acc=GSE7762"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE7762","term_id":"7762"}}</span>GSE7762</a> microarray data from GEO and analyze the expression of <em>Rgma</em> (the mouse gene homologous to <em>RGMA</em>). One-way ANOVA implemented in MATLAB (Statistics Toolbox Release 2015b, MathWorks, Inc., Natick, Massachusetts, United States) was used to test the effect of morphine on <em>Rgma</em> expression in different mouse strains. Statistical significance was set at P&lt;0.05.</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">Co-expression Analysis of <em>RGMA</em></h3><p id="P15" class="p p-first-last">A novel bioinformatics pipeline was developed to search and prioritize genes co-expressed with <em>RGMA</em> in an unbiased way genome-wide. The correlations identified between <em>RGMA</em> and the identified co-expressed genes were subject to validation in brain tissue samples (n=1301) downloaded from GEO (see <em>RGMA</em> Co-Expression Analysis in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Information</a>).</p></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">Results</h2><p id="P16" class="p p-first">One GWS SNP was identified: rs12442183 (MAF 0.41 in EAs), which is ∼110 kb upstream of <em>RGMA</em>, in Yale-Penn 1 (P=4.7×10<sup>-8</sup>, n=1388). Meta-analysis of this sample with the other three samples (Yale-Penn 2 and 3, and SAGE) improved the GWS association signal (meta-P=1×10<sup>-8</sup>, n=3058) (<a href="/pmc/articles/PMC6041180/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> and <a href="/pmc/articles/PMC6041180/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The robust association signals around the top SNP (<a href="/pmc/articles/PMC6041180/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>) and the observed negligible inflation factor in the QQ plot (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary figure 2</a>) support the validity of the GWAS result. As noted above, the association was GWS in Yale-Penn 1 taken individually. We also observed P &lt;0.05 in both Yale-Penn 2 and Yale-Penn 3: the risk allele T of rs12442183 was significantly correlated with increased OD criterion counts in all cohorts, except for SAGE (<a href="/pmc/articles/PMC6041180/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="/pmc/articles/PMC6041180/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). SAGE has little variation in OD criterion count and &gt;50% of individuals have a criterion count of 0, yielding low power in that sample.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC6041180/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140355641165264" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms933995f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms933995f1.jpg" src="/pmc/articles/PMC6041180/bin/nihms933995f1.jpg"></div></a><div id="largeobj_idm140355641165264" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC6041180/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC6041180/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC6041180/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Manhattan plots showing genome-wide association signals of opioid dependence in opioid-exposed European-Americans by meta-analysis</strong><p id="__p4">(<strong>A</strong>) Manhattan plot with one significant variant, rs12442183, on chromosome 15, in meta-analysis of four OD cohorts. The line in the plot represents the genome-wide significance cutoff (5×10<sup>-8</sup>). (<strong>B</strong>) Regional Manhattan plot demonstrates rs12442183 is close to gene <em>RGMA</em> (and regulates <em>RGMA</em> expression; see <a href="/pmc/articles/PMC6041180/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> and text). The light blue line and right Y-axis show the observed recombination rate in the 1000 Genomes Project European samples (EUR, hg19). The SNPs are colored in accordance to R<sup>2</sup> with rs12442183 (in purple), except for rs113200177, which is specifically highlighted in dark. rs113200177 (T/TG, meta-P=2×10<sup>-3</sup>) is located in the promoter of <em>RGMA</em> and is predicted to be able to destroy the E-box motif (5′-CANNTG-3′) of transcription factor 3 or 4 (<em>TCF3</em> or <em>TCF4</em>) by Haploreg4 (<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842745">45</a>). By adjusting the effect of the indel in conditional association analysis for rs12442183, the association signal of rs12442183 became stronger in Yale-Penn 1 taken individually (P=1.9×10<sup>-8</sup>).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC6041180/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140355630885904" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms933995f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms933995f2.jpg" src="/pmc/articles/PMC6041180/bin/nihms933995f2.jpg"></div></a><div id="largeobj_idm140355630885904" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC6041180/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC6041180/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC6041180/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Forest plot for rs12442183 risk allele T in opioid dependence GWAS</strong><p id="__p5">Effect Beta and its 95% confidence interval are plotted with squares (proportional to sample size in each cohort) and horizontal lines with whiskers, respectively. The vertical line (beta=0) indicates no effect. Names of cohorts are shown on the left. Meta-analysis P-values and their corresponding effect betas (red diamonds) for European-American (EA) and African-American (AA) populations are also provided. The forest plot was generated by DistillerSR Forest Plot Generator from Evidence Partners (<a href="https://www.evidencepartners.com/resources/forest-plot-generator" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.evidencepartners.com/resources/forest-plot-generator</a>).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Meta-analysis of rs12442183 Risk Allele T in OD Exposed Individuals</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140355624544944" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center">Population</th><th rowspan="1" colspan="1" valign="middle" align="left">Cohort<sup>1</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">Total Subjects No.</th><th rowspan="1" colspan="1" valign="middle" align="center">Exposed Subject No.</th><th rowspan="1" colspan="1" valign="middle" align="center">Criterion Counts<sup>2</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">MAF<sup>3</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">Effect Beta (95% CI) <sup>4</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">Imputation R<sup>2</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left">Yale-Penn 1</td><td rowspan="1" colspan="1" valign="middle" align="center">1768</td><td rowspan="1" colspan="1" valign="middle" align="center">1388</td><td rowspan="1" colspan="1" valign="middle" align="center">5.0 ± 2.8</td><td rowspan="1" colspan="1" valign="middle" align="center">0.41</td><td rowspan="1" colspan="1" valign="middle" align="center">0.60 (0.39 - 0.82)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.98</td><td rowspan="1" colspan="1" valign="middle" align="center">4.7 × 10<sup>-8</sup></td></tr><tr><td rowspan="3" colspan="1" valign="middle" align="center">European-Americans (EA)</td><td rowspan="1" colspan="1" valign="middle" align="left">Yale-Penn 2</td><td rowspan="1" colspan="1" valign="middle" align="center">1725</td><td rowspan="1" colspan="1" valign="middle" align="center">996</td><td rowspan="1" colspan="1" valign="middle" align="center">4.7 ± 2.9</td><td rowspan="1" colspan="1" valign="middle" align="center">0.38</td><td rowspan="1" colspan="1" valign="middle" align="center">0.30 (0.02 - 0.58)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.82</td><td rowspan="1" colspan="1" valign="middle" align="center">3.8 × 10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Yale-Penn 3</td><td rowspan="1" colspan="1" valign="middle" align="center">403</td><td rowspan="1" colspan="1" valign="middle" align="center">98</td><td rowspan="1" colspan="1" valign="middle" align="center">3.6 ± 3.2</td><td rowspan="1" colspan="1" valign="middle" align="center">0.41</td><td rowspan="1" colspan="1" valign="middle" align="center">1.6 (0.52 - 2.8)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.81</td><td rowspan="1" colspan="1" valign="middle" align="center">5.3 × 10<sup>-3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">SAGE</td><td rowspan="1" colspan="1" valign="middle" align="center">2752</td><td rowspan="1" colspan="1" valign="middle" align="center">576</td><td rowspan="1" colspan="1" valign="middle" align="center">2.0 ± 2.8</td><td rowspan="1" colspan="1" valign="middle" align="center">0.38</td><td rowspan="1" colspan="1" valign="middle" align="center">-0.14 (-0.18 - 0.50)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.98</td><td rowspan="1" colspan="1" valign="middle" align="center">3.9 × 10<sup>-1</sup><sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong>Meta-analysis</strong></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>0.44</strong></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>1.3 × 10<sup>-8</sup></strong></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left">Yale-Penn 1</td><td rowspan="1" colspan="1" valign="middle" align="center">3289</td><td rowspan="1" colspan="1" valign="middle" align="center">1209</td><td rowspan="1" colspan="1" valign="middle" align="center">3.8 ± 3.1</td><td rowspan="1" colspan="1" valign="middle" align="center">0.33</td><td rowspan="1" colspan="1" valign="middle" align="center">0.14(0.04 - 0.6)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.98</td><td rowspan="1" colspan="1" valign="middle" align="center">2.3 × 10<sup>-2</sup></td></tr><tr><td rowspan="3" colspan="1" valign="middle" align="center">African-Americans (AA)</td><td rowspan="1" colspan="1" valign="middle" align="left">Yale-Penn 2</td><td rowspan="1" colspan="1" valign="middle" align="center">1744</td><td rowspan="1" colspan="1" valign="middle" align="center">245</td><td rowspan="1" colspan="1" valign="middle" align="center">3.4± 3.1</td><td rowspan="1" colspan="1" valign="middle" align="center">0.3</td><td rowspan="1" colspan="1" valign="middle" align="center">0.50 (-0.08 - 1.1)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.82</td><td rowspan="1" colspan="1" valign="middle" align="center">8.0 × 10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Yale-Penn 3</td><td rowspan="1" colspan="1" valign="middle" align="center">139</td><td rowspan="1" colspan="1" valign="middle" align="center">12</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">SAGE</td><td rowspan="1" colspan="1" valign="middle" align="center">1312</td><td rowspan="1" colspan="1" valign="middle" align="center">252</td><td rowspan="1" colspan="1" valign="middle" align="center">2.5 ± 3.0</td><td rowspan="1" colspan="1" valign="middle" align="center">0.34</td><td rowspan="1" colspan="1" valign="middle" align="center">0.25 (-0.31 - 0.81)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.98</td><td rowspan="1" colspan="1" valign="middle" align="center">3.7 × 10<sup>-1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong>Meta-analysis</strong></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>0.34</strong></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>3.0 × 10<sup>-3</sup></strong></td></tr></tbody></table></div><div id="largeobj_idm140355624544944" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6041180/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>1</sup>Illumina microarray HumanOmni1-Quad v1.0, HumanCore Exome array, and Human 1M were used to genotype samples for Yale-Penn 1-3, respectively.</div><div id="TFN2"><sup>2</sup>Mean ± standard deviation for criterion counts of ordinal traits.</div><div id="TFN3"><sup>3</sup>Minor allele frequency.</div><div id="TFN4"><sup>4</sup>95% confidence interval of continuous effect beta based on likelihood ratio test by using the software Genome-wide Efficient Mixed Model Association (GEMMA).</div><div id="TFN5"><sup>*</sup>indicates the opposite effect direction.</div></div></div><p id="P17">The association of the lead SNP rs12442183 with OD was tested further in opioid-exposed AAs from three cohorts (<a href="/pmc/articles/PMC6041180/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), including Yale-Penn 1 (n=1506), Yale-Penn 2 (n=245) and SAGE (n=263) (MAF 0.33 in AAs). Yale-Penn 3 was not included because of the small number of AAs (n=12). Rs12442183 was significantly (P=1.2×10<sup>-2</sup>) or marginally significantly (P=8×10<sup>-2</sup>) associated with OD in Yale-Penn 1 and Yale-Penn 2, respectively. Possibly due to the same low variation of OD criterion counts in SAGE AAs, the association of rs12442183 with OD was not significant. However, by meta-analysis of the 2014 AA samples, the association of rs12442183 with OD was replicated (meta-P=3×10<sup>-3</sup>) (<a href="/pmc/articles/PMC6041180/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="/pmc/articles/PMC6041180/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>).</p><p id="P18">The validity of the association of rs12442183 with OD in EAs and potentially also in AAs was demonstrated by the observation that 5 out of 7 cohorts (<a href="/pmc/articles/PMC6041180/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="/pmc/articles/PMC6041180/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>) show at least nominal significance at this marker. Rs12442183 and rs1872052 (hg19 position chr15: 93752878, ∼4.9 kb downstream of rs12442183) are the only two of the top 10 signals in the EA GWAS that were also nominally associated in AAs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). Many of the variants that were associated in EAs were not present or were uncommon in AAs, so there was no relevant information for them. The meta-analysis P-value for rs12442183 was also significant (3.0×10<sup>-8</sup>) across all cohorts (EA and AA), but the combined P-value is higher (i.e., less significant) than the European-specific P-value, which may be due to the different haplotype structures for EAs and AAs.</p><p id="P19">We next considered the effects of other substance use disorder criteria, including those for cocaine dependence (CD), alcohol dependence (AD), and nicotine dependence (ND), on the rs12442183 association signal. When we included CD, AD and ND criterion counts as covariates along with age, sex and first three PCs, there was decreased evidence of association of rs12442183 with OD in Yale-Penn 1-3 EAs, where nominal P-values were 3.7×10<sup>-7</sup>, 0.17, and 0.08, respectively (meta-analysis P=3.91×10<sup>-7</sup>). Thus, the effect of CD, AD and ND on the observed association of rs12442183 with OD was observable, but limited.</p><p id="P20">The association of a previously implicated OD locus, <em>CNIH3</em> (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842743">3</a>), was evaluated in the current GWAS meta-analysis of EAs. The lead SNP for <em>CNIH3</em>, rs10799590, was identified in a GWAS that compared opioid-dependent daily drug injectors with opioid misusers who never progressed to daily injection (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842711">5</a>), whereas in our OD GWAS, the phenotype was OD criterion count. With that in mind, we note that the meta-analysis P-value for rs10799590 with OD criterion count was 0.038 in EAs.</p><p id="P21">The regulatory relationship between rs12442183 and <em>RGMA</em> expression was studied using publicly available microarray data. <em>RGMA</em> encodes a glycosylphosphatidylinosital-anchored glycoprotein (RGMa). There are three RGMa isoforms (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary figure 3</a>) that are generated by six <em>RGMA</em> transcript variants (<a href="/pmc/articles/PMC6041180/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3A</span></a>), including <a id="__tag_760842843" class="tag_hotlink" href="/nuccore/NM_001166283" ref="/nuccore/NM_001166283"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166283","term_id":"261878462","term_text":"NM_001166283"}}</span>NM_001166283</a> (variant 1), <a id="__tag_813751089" class="tag_hotlink" href="/nuccore/NM_001166286" ref="/nuccore/NM_001166286"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166286","term_id":"1677531159","term_text":"NM_001166286"}}</span>NM_001166286</a> (variant 2), <a id="__tag_760842847" class="tag_hotlink" href="/nuccore/NM_001166287" ref="/nuccore/NM_001166287"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166287","term_id":"261878460","term_text":"NM_001166287"}}</span>NM_001166287</a> (variant 3), <a id="__tag_813751091" class="tag_hotlink" href="/nuccore/NM_020211" ref="/nuccore/NM_020211"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_020211","term_id":"1677530851","term_text":"NM_020211"}}</span>NM_020211</a> (variant 4), <a id="__tag_813751088" class="tag_hotlink" href="/nuccore/NM_001166288" ref="/nuccore/NM_001166288"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166288","term_id":"1677538601","term_text":"NM_001166288"}}</span>NM_001166288</a> (variant 5) and <a id="__tag_760842840" class="tag_hotlink" href="/nuccore/NM_001166289" ref="/nuccore/NM_001166289"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166289","term_id":"261878466","term_text":"NM_001166289"}}</span>NM_001166289</a> (variant 6). Transcript variant 1 and 4 encode RGMa isoforms 1 and 3, respectively, and all other transcripts produce the same RGMa isoform 2. To identify alternative expression of <em>RGMA</em> among 11 healthy human tissues, four <em>RGMA</em> exons, including ENSE00001325239, ENSE00001319132, ENSE00001304160 and ENSE00001532613, were analyzed via R package ESLiMc (<a href="/pmc/articles/PMC6041180/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3A</span></a>). ENSE00001325239 and ENSE00001319132 reside in the overlapping exonic regions in all 6 transcripts of <em>RGMA</em>, i.e., the expression signals of the two exons represent the total mRNA of <em>RGMA</em>. ENSE00001532613 is a specific exon representing transcript variant 4 (<a id="__tag_813751092" class="tag_hotlink" href="/nuccore/NM_020211" ref="/nuccore/NM_020211"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_020211","term_id":"1677530851","term_text":"NM_020211"}}</span>NM_020211</a>, encoding RGMa isoform 3). Similarly, exon ENSE00001304160, specifically represents transcript variant 2 (<a id="__tag_813751087" class="tag_hotlink" href="/nuccore/NM_001166286" ref="/nuccore/NM_001166286"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166286","term_id":"1677531159","term_text":"NM_001166286"}}</span>NM_001166286</a>, encoding RGMa isoform 2). Only ENSE00001532613, representing transcript variant 4, was variably expressed among the 11 tissues studied, and cerebellum was the only brain tissue with both the lowest expression and with the lowest gene-exon residual regression score for <em>RGMA</em> (<a href="/pmc/articles/PMC6041180/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3A and 3B</span></a>). Further correlation analysis between rs12442183 genotype and the gene-exon expression residual scores of the four exons revealed that only ENSE00001532613 (representative of transcript variant 4) was differentially expressed (P=2×10<sup>-3</sup>) among rs12442183*T risk allele carriers in frontal cortex (<a href="/pmc/articles/PMC6041180/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3C</span></a>). Among the other nine brain tissues, there were no significant correlations between rs12442183 genotype and gene-exon residual scores (data not shown). We also tested whether rs12442183 was an eQTL in LCLs across 6 global populations, including CEU, YRI, LWK, MEX, CHB and JPT, but found no significant associations (all linear model P-values &gt;0.05), indicating that rs12442183 may be a specific eQTL for <em>RGMA</em> transcript variant 4 (encoding RGMa isoform 3) in frontal cortex.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC6041180/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140355647135520" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms933995f3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms933995f3.jpg" src="/pmc/articles/PMC6041180/bin/nihms933995f3.jpg"></div></a><div id="largeobj_idm140355647135520" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC6041180/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC6041180/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC6041180/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>rs12442183 is a regulatory variant of <em>RGMA</em> and <em>Rgma</em> is an acute opioid response gene in mouse</strong><p id="__p6">(<strong>A</strong>) Six transcripts of <em>RGMA</em> and four exons used for alternative splicing analysis are mapped together. The relationship between the expression of these four exons and <em>RGMA</em> gene expression was determined by using a linear regression model implemented in R package ESLiMc. The difference between the observed exon expression and the predicted exon expression is named as residual regression score. The alternative splicing events of <em>RGMA</em> across 11 healthy human tissues are displayed in (<strong>B</strong>), with dot and error-bar represent mean and standard deviation of 3 replicates, respectively. Boxplots (<strong>C</strong>) are used to illustrate the association between rs12442183 genotype and the residual regression scores of four exons in brain tissue frontal cortex in European samples. Dot and line within each boxplot represent mean and median, respectively, and box region indicates the range from first quantile to third quantile. Up and down whiskers, as well as dots outside box region represent maximum value, minimum value, and outliers, respectively. (<strong>D</strong>) The expression of <em>Rgma</em> (homologous gene in mouse) among different mouse strains acutely after morphine injection (a single injection, 20 mg/kg) in comparison with chronic injections (repeated morphine administration, 10-40 mg/kg, 3 times daily for 5 days). X-axis shows the four inbred mouse strains (129P3/J, DBA/2J, C57BL/6J and SWR/J). Y-axis demonstrates the mRNA expression of mouse gene <em>Rgma</em>. The expression data were derived from NCBI Gene Expression Omnibus (GEO) dataset <a id="__tag_760842767" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE7762" ref="/geo/query/acc.cgi?acc=GSE7762"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE7762","term_id":"7762"}}</span>GSE7762</a>. The standard deviation is at the top of each bar.</p></div></div></div><p id="P22">We also tested whether the observed association between rs12442183 and OD was driven by the SNP's regulatory effect on other genes in the region (see Integrative cis-eQTL Analysis in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Information</a>). With the independent data from the R package ESLiM, only <em>RGMA</em> and <em>MCTP2</em> showed alternative splicing in healthy brain tissue. We further correlated the genotype of rs12442183 with available exon expression of <em>RGMA</em>, <em>CHD2</em>, <em>FAM174B</em> and <em>MCTP2</em> from BRAINEAC. There was a significant correlation of rs12442183 genotype with specific exons of <em>RGMA</em> and <em>MCTP2</em>. However, the expression of <em>MCTP2</em> is very low among 27 human tissues.</p><p id="P23">We re-analyzed publicly available data from a microarray experiment (GEO accession No. <a id="__tag_760842763" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE7762" ref="/geo/query/acc.cgi?acc=GSE7762"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE7762","term_id":"7762"}}</span>GSE7762</a>) designed to test the effects of different morphine treatments on gene expression in mouse striatum tissue. By comparison with control and chronic morphine treatment, we found that the expression of mouse <em>Rgma</em> was significantly decreased (all ANOVA P-values&lt;0.01) after acute morphine challenge in all 4 mouse strains, except for C57BL/6J. Compared to acute morphine challenge, significant upregulation of <em>Rgma</em> was found in C57BL/6J mouse striatum after chronic morphine injection (ANOVA P-value&lt;0.01). These results demonstrate that <em>Rgma</em> was differentially regulated in mouse striatum when comparing C57BL/6J with other mouse strains.</p><p id="P24" class="p p-last">The expression of six genes previously associated with psychiatric disorders and two potential regulators for <em>RGMA</em> were correlated with that of <em>RGMA</em> in brain tissue samples (n=1301). These genes are <em>GRIN1</em> (glutamate receptor, ionotropic, N-methyl D-aspartate 1), <em>KCNIP3</em> (Kv channel interacting protein 3, calsenilin), <em>PTCHD1</em> (patched domain containing 1), <em>FXYD6</em> (FXYD domain containing ion transport regulator 6), <em>DLG4</em> (discs, large homolog 4) and <em>NRG1</em> (neuregulin 1), as well as two potential regulators (<em>TCF3</em> and <em>TCF4</em>). <em>GRIN1</em>, <em>FXYD6</em>, <em>NRG1</em> and <em>DLG4</em> have been associated with schizophrenia (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842751">21</a>-<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842717">24</a>), <em>KCNIP3</em> with Alzheimer's disease (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842693">25</a>), <em>PTCHD1</em> with autism (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842721">26</a>), and <em>NRG1</em> with psychosis (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842739">27</a>). The E-box (5′-CANNTG-3′) motif of the two potential regulators, <em>TCF3</em> and <em>TCF4</em>, was found in the promoter region of <em>RGMA</em>, <em>GRIN1</em> and <em>PTCHD1</em>. We combined normalized expression data (z-scores) of 1301 brain tissue samples to test the pairwise correlation among the six genes. As shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary figure 4A</a>, the expressions of <em>GRIN1</em>, <em>KCNIP3</em>, <em>FXYD6</em>, <em>PTCHD1</em>, <em>RGMA</em>, <em>TCF3</em> and <em>TCF4</em> were highly inter-correlated. For <em>RGMA</em>, there was a strong positive correlation with <em>GRIN1</em> (Pearson correlation coefficient r=0.40 and P=7.0×10<sup>-50</sup>) and <em>KCNIP3</em> (r=0.38 and P=1.35×10<sup>-45</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary figure 4B</a>), with the latter two also significantly inter-correlated (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary figure 4A</a>: r=0.73 and P=1×10<sup>-200</sup>).</p></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Discussion</h2><p id="P25" class="p p-first">We report here results from a GWAS of opioid criterion count in EA subjects. There was a GWS association of an <em>RGMA</em> variant, rs12442183, in Yale-Penn 1. There was nominally significant evidence for association of the same variant in both Yale-Penn 2 and Yale-Penn 3, and when all EA samples (including SAGE, where no evidence of the association was obtained) were meta-analyzed together, the significance of the association finding increased. Because the minor allele frequency of rs12442183 was similar for the EA and AA populations, we further evaluated the association of rs12442183 with OD in the Yale-Penn AA sample. The nominally significant association in the AA meta-analysis (n=2014, meta-P=3×10<sup>-3</sup>) further supports the association. (AA replication of results from EA samples is not universally applicable, but in this case, we found evidence that we believe is of value.) Cis-eQTL analysis of <em>RGMA</em> in brain showed the rs12442183*T risk allele to be associated with higher expression of <em>RGMA</em> transcript variant 4 (encoding RGMa isoform 3) in frontal cortex. This observation is consistent with the upregulation of <em>Rgma</em> in C57BL/J6 mice treated with morphine chronically, which is not the case in three other mouse strains: DBA/2J, 129P3/J and SWR/J. C57BL/6J mice have a greater preference for oral self-administration of morphine than three other strains (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842758">19</a>). Further co-expression analysis in large brain tissue expression datasets demonstrated significant correlation between the expression of <em>RGMA</em> and that of four genes that have been shown to affect risk for other psychiatric disorders, <em>GRIN1</em> (schizophrenia) (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842728">23</a>), <em>KCNIP3</em> (Alzheimer's disease) (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842747">25</a>), <em>FXYD6</em> (schizophrenia) (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842722">21</a>) and <em>PTCHD1</em> (autism) (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842723">26</a>), as well as two transcription factors <em>TCF3</em> and <em>TCF4</em>. The latter gene TCF4 is a schizophrenia-risk gene (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842754">28</a>). These <em>RGMA</em> co-expressed genes are critical to normal brain function, supporting the potential involvement of <em>RGMA</em> in related biological processes.</p><p id="P26">The protein product of <em>RGMA</em>, RGMa, is a glycosylphosphatidylinosital-anchored glycoprotein that exists in membrane-bound and soluble forms (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842713">29</a>), both of which are potent regulators of cell death and inhibitors of nerve growth. RGMa was first reported to be a developmental neurite growth inhibitor (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842748">30</a>). It has since emerged as an important factor inhibiting neuronal regeneration and functional recovery (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842750">31</a>, <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842738">32</a>). RGMa inhibits neurite growth by binding to its neuronal receptor, neogenin, a member of the immunoglobulin (Ig) superfamily. Neogenin overexpression and RGMa down expression in the developing embryonic neural tube induce apoptosis (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842727">33</a>). RGMa may be involved in neuroplasticity consistent with increased secretion of RGMa in stroke- or injury-damaged human brain tissue (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842752">34</a>), in the plaque regions of Alzheimer's disease patients (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842729">35</a>), in the substantia nigra of Parkinson's disease patients (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842712">36</a>), and in multiple sclerosis (MS) patients (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842701">37</a>). Considering the example of progressive MS (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842716">38</a>), RGMa was found in active and chronic lesions, and in normal-appearing gray and white matter, and was expressed by cellular meningeal infiltrates in autopsy material. Levels of soluble RGMa in the cerebrospinal fluid were decreased in progressive MS patients for whom treatment resulted in functional improvement. Antibody-mediated neutralization of RGMa or downregulation of RGMa can result in neuroprotection and enhanced functional recovery (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842733">38</a>). Whether RGMa plays similar roles in OD patients and thereby modulates addiction risk warrants further study.</p><p id="P27">The mouse experiment that investigated the effects of morphine on the striatal transcriptome (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842736">19</a>) identified 661 morphine-responsive genes. <em>Rgma</em> was one of the top 98 genes that were down-regulated with <em>acute</em> morphine treatment. Consistent downregulation of <em>Rgma</em> was observed among 3 mouse strains, including DBA/2J, 129P3/J and SWR/J, but not C57BL/6J. However, <em>Rgma</em> was specifically upregulated in C57BL/6J mice (in contrast to the other 3 mouse strains) following <em>chronic</em> morphine treatment. The different responses of these mouse strains after chronic or acute morphine treatment may reflect genetic differences, as C57BL/6J mice have been reported to have the greatest preference for orally self-administered morphine of the strains studied (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842731">19</a>). Further, in the current study, risk rs12442183*T-allele carriers demonstrated a higher expression of one <em>RGMA</em> transcript in frontal cortex. As downregulation of RGMa can result in neuroprotection and enhanced functional recovery (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842715">38</a>) and increased RGMa in brain is associated with risk of multiple neurological disorders, including Alzheimer's disease, Parkinson's disease, and multiple sclerosis, downregulation of RGMa may have a protective effect on neurons after acute morphine treatment, while upregulation of RGMa may induce brain vulnerability with chronic morphine treatment.</p><p id="P28">In the analysis of several brain tissue samples, <em>RGMA</em> expression was associated with the expression of four other genes related to psychiatric traits, <em>GRIN1</em>, <em>KCNIP3</em>, <em>FXYD6</em> and <em>PTCHD1</em>. <em>GRIN1</em> was the gene that was most highly co-expressed (r=0.40) with <em>RGMA</em> in brain. It plays a key role in synaptic plasticity, synaptogenesis, excitotoxicity, memory acquisition and learning (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842710">39</a>). <em>GRIN1</em> can tether the μ-opioid receptor (the main biological target of opioid drugs) and subsequently immobilize the μ-opioid receptor within lipid rafts (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842694">40</a>). The overexpression of <em>GRIN1</em> increases the amount of μ-opioid receptor in lipid rafts and the magnitude of receptor signaling. The significant correlation between the expression of other genes (<em>PTCHD1</em>, <em>FXYD6</em>, and <em>KCNIP3</em>) and that of <em>RGMA</em>, may reflect their critical function in the brain (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842703">21</a>, <a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842706">26</a>, <a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842726">41</a>). These findings raise the question of whether, in response to opioid stimulation in brain, there is crosstalk between <em>RGMA</em> and <em>GRIN1</em>, and possibly <em>KCNIP3</em>, <em>FXYD6</em> and <em>PTCHD1</em> as well.</p><p id="P29">Different proportions of prescribed vs. illegal users of opioids in the Yale-Penn and the SAGE samples could have affected our results and may help to understand the different results from these samples. Neither the SSADDA (Yale-Penn) nor the SSAGA (SAGE) differentiates prescribed from illegal-recreational use of opioids, and therefore some subjects who endorsed DSM-IV OD criteria probably did so despite using opioids therapeutically. However, a majority (59.5%) of individuals in our samples who met criteria for OD endorsed at least five of the seven diagnostic criteria (mean criterion count = 6.6, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary figure 1</a>), suggesting that the GWS finding is attributable to moderate-to-severe OD, rather than being contributed to substantially by individuals who used opioids as prescribed, but required an increased dosage for pain control or who experienced withdrawal symptoms when opioid treatment was discontinued. The much greater proportion of participants in the Yale-Penn samples than SAGE who met the maximum number of (i.e., seven) OD criteria supports that interpretation. Also, compared to our (Yale-Penn) samples, the SAGE samples were recruited for alcoholism (<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842708">42</a>), cocaine dependence (<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842699">43</a>), or nicotine dependence (<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842755">44</a>), rather than OD. Therefore, a much higher percentage of opioid-exposed individuals in SAGE are “controls”: viz. individuals who used opioids fewer than 11 times lifetime and would have skipped out of the remainder of the SSAGA opioid section and had a null criterion count (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary figure 1</a>). Additionally, the relative percentage of opioid-exposed individuals varies considerably across our three Yale-Penn EA samples because of the different recruiting epochs, although the assessment protocol did not change: the earlier Yale-Penn samples were more focused on OD recruitment than the later ones.</p><p id="P30" class="p p-last">In summary, we report a GWS association of rs12442183, at the <em>RGMA</em> locus, with OD. In contrast to our earlier study, the present study includes only opioid-exposed subjects selected from an expanded sample; and our analyses for this study did not co-vary for the criterion count for other substance use disorder traits. We believe that the former two differences were key in our identification of the present very robust association finding. The finding was GWS in our largest sample taken separately, with nominally significant associations of the same variant in two of three other available samples of EAs. Further support was obtained in AA samples. Our current data support a model in which the regulatory variant rs12442183 affects expression of <em>RGMA</em> transcript variant 4 in frontal cortex, and the consequent variation in RGMa isoform 3 may alter the response to opioids, a potential underlying mechanism for its association with OD. This mechanism and locus merit further study, which could increase our understanding of the genetics and pathophysiology of OD.</p></div><div id="S14" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplemental</h4><p id="P31">Supplementary Figure S1. Percent of opioid exposed samples grouped by different criterion counts among the four cohorts, including SAGE and Yale-Penn 1-3.</p><p id="P32">Supplementary Figure S2. QQ plot for meta-analysis of opioid dependence GWAS.</p><p id="P33"><strong>Supplementary Figure S3. Sequence alignment of three RGMa isoforms</strong>. Three RGMa isoforms are encoded by six <em>RGMA</em> transcript variants, including <a id="__tag_760842845" class="tag_hotlink" href="/nuccore/NM_001166283" ref="/nuccore/NM_001166283"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166283","term_id":"261878462","term_text":"NM_001166283"}}</span>NM_001166283</a> (variant 1), <a id="__tag_813751085" class="tag_hotlink" href="/nuccore/NM_001166286" ref="/nuccore/NM_001166286"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166286","term_id":"1677531159","term_text":"NM_001166286"}}</span>NM_001166286</a> (variant 2), <a id="__tag_760842846" class="tag_hotlink" href="/nuccore/NM_001166287" ref="/nuccore/NM_001166287"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166287","term_id":"261878460","term_text":"NM_001166287"}}</span>NM_001166287</a> (variant 3), <a id="__tag_813751086" class="tag_hotlink" href="/nuccore/NM_020211" ref="/nuccore/NM_020211"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_020211","term_id":"1677530851","term_text":"NM_020211"}}</span>NM_020211</a> (variant 4), <a id="__tag_813751084" class="tag_hotlink" href="/nuccore/NM_001166288" ref="/nuccore/NM_001166288"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166288","term_id":"1677538601","term_text":"NM_001166288"}}</span>NM_001166288</a> (variant 5) and <a id="__tag_760842839" class="tag_hotlink" href="/nuccore/NM_001166289" ref="/nuccore/NM_001166289"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_001166289","term_id":"261878466","term_text":"NM_001166289"}}</span>NM_001166289</a> (variant 6) (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842705">1</a>). The longest isoform 1 is derived from variant 1. Isoform 2 has the shortest N-terminus and is generated by variants 2, 3, 5 and 6. Isoform 3 is derived from variant 4 and harbors a distinct and shorter N-terminus.</p><p id="P34"><strong>Supplementary Figure S4. mRNA co-expression analysis between <em>RGMA</em> and six genes affecting psychiatric disorder risk and two transcription factors in brain tissues (n=1301)</strong>., Pairwise Pearson correlation coefficients (r) for <em>RGMA</em> and 8 other genes are depicted in the heatmap (<strong>A</strong>), the -log10(P) correlation P values of which are indicated by different colors. Scatter plots and linear regression for <em>RGMA</em> with 8 other genes are depicted in (<strong>B</strong>). All expression data are normalized z-scores.</p><p id="P35">Supplementary Table S1. Sample comparison between Yale-Penn 1 and the Phase 1 and 3 samples published in the 2014 Biological Psychiatry paper (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_760842698">2</a>).</p><p id="P36">Supplementary Table S2. Top 10 association signals in European-Americans OD GWAS, and results for those variants in African-Americans</p><p id="P37">Table S3. Cis-eQTL analysis of rs12442183 for 11 genes among 10 brain tissues.</p><p id="P38">Table S4. The top 10 observed exon probes associated with rs12442183 in Braineac database.</p><p id="P39">Table S4. Top 100 co-expressed genes of <em>RGMA</em> identified by COXPRESdb.</p><p id="P40">Table S5. The association P values for all exon probes of <em>RGMA</em> with rs12442183 in Braineac database.</p><p id="P41">Table S5. Disease enrichment among top 100 co-expressed genes with <em>RGMA</em>.</p><p id="P42">Table S6. Transcription factor motif enrichment among top 100 co-expressed genes with <em>RGMA</em>.</p><div class="sup-box half_rhythm" id="d36e1073"><a href="/pmc/articles/PMC6041180/bin/NIHMS933995-supplement-Supplemental.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(629K, pdf)</sup></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Acknowledgments</h2><div class="sec"><p id="__p1">This study was supported by National Institutes of Health grants RC2 <a id="__tag_813751090" class="tag_hotlink" href="/nuccore/DA028909" ref="/nuccore/DA028909"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"DA028909","term_id":"79217476","term_text":"DA028909"}}</span>DA028909</a>, R01 DA12690, R01 DA12849, R01 DA18432, R01 AA11330, R01 AA017535, and the VA Connecticut Healthcare Center and Philadelphia VA MIRECCs. We appreciate the work in recruitment and assessment by James Poling, PhD at Yale University School of Medicine and the APT Foundation; Roger Weiss, MD at McLean Hospital; by Kathleen Brady, MD/PhD and Raymond Anton, MD at the Medical University of South Carolina; and David Oslin, MD at the University of Pennsylvania. Genotyping services for a part of our GWAS were provided by the Center for Inherited Disease Research and Yale University (Center for Genome Analysis), which is fully funded by Federal contract N01-HG-65403 from the NIH to The Johns Hopkins University. We thank Ann Marie Lacobelle, MS and Christa Robinson, BS, who provided technical assistance. We thank Yaira Nunez and Michelle Slivinsky who led the quality control effort for the Semi-Structured Assessment for Drug Dependence and Alcoholism interviews and phenotyping for the study sample. We thank John Farrell, PhD, Section of Biomedical Genetics, Boston University School of Medicine, who provided database management assistance. We also thank three anonymous reviewers for their constructive suggestions on the manuscript.</p></div></div><div id="idm140355627277536" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140355627277536title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="__p2" class="p p-first-last"><strong>Conflict of Interest</strong>: All authors have completed and submitted the ICMJE Form for Disclosure of Potential Conflicts of Interest. Dr. Kranzler has been a consultant or advisory board member for Alkermes, Indivior and Lundbeck. He is a member of the American Society of Clinical Psychopharmacology's Alcohol Clinical Trials Initiative, which was supported in the last three years by AbbVie, Alkermes, Ethypharm, Indivior, Lilly, Lundbeck, Otsuka, Pfizer, Arbor, and Amygdala Neurosciences. The other authors have no biomedical financial interests or potential conflicts of interest.</p><p></p><p class="fn sec" id="FN3"></p><p id="__p3" class="p p-first-last"><strong>Publisher's Disclaimer: </strong>This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p><p></p></div></div><div id="idm140355645664288" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140355645664288title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Mistry CJ, Bawor M, Desai D, Marsh DC, Samaan Z. Genetics of Opioid Dependence: A Review of the Genetic Contribution to Opioid Dependence. <span><span class="ref-journal">Curr Psychiatry Rev. </span>2014;<span class="ref-vol">10</span>:156–167.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4155832/">PMC free article</a>]</span> [<a href="/pubmed/25242908" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Psychiatry+Rev&amp;title=Genetics+of+Opioid+Dependence:+A+Review+of+the+Genetic+Contribution+to+Opioid+Dependence&amp;author=CJ+Mistry&amp;author=M+Bawor&amp;author=D+Desai&amp;author=DC+Marsh&amp;author=Z+Samaan&amp;volume=10&amp;publication_year=2014&amp;pages=156-167&amp;pmid=25242908&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Gelernter J, Kranzler HR, Sherva R, Koesterer R, Almasy L, Zhao H, et al.  Genome-wide association study of opioid dependence: multiple associations mapped to calcium and potassium pathways. <span><span class="ref-journal">Biol Psychiatry. </span>2014;<span class="ref-vol">76</span>:66–74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3992201/">PMC free article</a>]</span> [<a href="/pubmed/24143882" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=Genome-wide+association+study+of+opioid+dependence:+multiple+associations+mapped+to+calcium+and+potassium+pathways&amp;author=J+Gelernter&amp;author=HR+Kranzler&amp;author=R+Sherva&amp;author=R+Koesterer&amp;author=L+Almasy&amp;volume=76&amp;publication_year=2014&amp;pages=66-74&amp;pmid=24143882&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Nelson EC, Agrawal A, Heath AC, Bogdan R, Sherva R, Zhang B, et al.  Evidence of CNIH3 involvement in opioid dependence. <span><span class="ref-journal">Mol Psychiatry. </span>2016;<span class="ref-vol">21</span>:608–614.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4740268/">PMC free article</a>]</span> [<a href="/pubmed/26239289" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Evidence+of+CNIH3+involvement+in+opioid+dependence&amp;author=EC+Nelson&amp;author=A+Agrawal&amp;author=AC+Heath&amp;author=R+Bogdan&amp;author=R+Sherva&amp;volume=21&amp;publication_year=2016&amp;pages=608-614&amp;pmid=26239289&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Smith AH, Jensen KP, Li J, Nunez Y, Farrer LA, Hakonarson H, et al.  Genome-wide association study of therapeutic opioid dosing identifies a novel locus upstream of OPRM1. <span><span class="ref-journal">Mol Psychiatry. </span>2017;<span class="ref-vol">22</span>:346–352.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5407902/">PMC free article</a>]</span> [<a href="/pubmed/28115739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Genome-wide+association+study+of+therapeutic+opioid+dosing+identifies+a+novel+locus+upstream+of+OPRM1&amp;author=AH+Smith&amp;author=KP+Jensen&amp;author=J+Li&amp;author=Y+Nunez&amp;author=LA+Farrer&amp;volume=22&amp;publication_year=2017&amp;pages=346-352&amp;pmid=28115739&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second-generation PLINK: rising to the challenge of larger and richer datasets. <span><span class="ref-journal">Gigascience. </span>2015;<span class="ref-vol">4</span>:7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4342193/">PMC free article</a>]</span> [<a href="/pubmed/25722852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Second-generation+PLINK:+rising+to+the+challenge+of+larger+and+richer+datasets&amp;author=CC+Chang&amp;author=CC+Chow&amp;author=LC+Tellier&amp;author=S+Vattikuti&amp;author=SM+Purcell&amp;volume=4&amp;publication_year=2015&amp;pages=7&amp;pmid=25722852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">1000-Genomes-Project-Consortium. Abecasis GR, Altshuler D, Auton A, Brooks LD, Durbin RM, et al.  A map of human genome variation from population-scale sequencing. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:1061–1073.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042601/">PMC free article</a>]</span> [<a href="/pubmed/20981092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+variation+from+population-scale+sequencing&amp;author=GR+Abecasis&amp;author=D+Altshuler&amp;author=A+Auton&amp;author=LD+Brooks&amp;author=RM+Durbin&amp;volume=467&amp;publication_year=2010&amp;pages=1061-1073&amp;pmid=20981092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Sherva R, Wang Q, Kranzler H, Zhao H, Koesterer R, Herman A, et al.  Genome-wide association study of cannabis dependence severity, novel risk variants, and shared genetic risks. <span><span class="ref-journal">JAMA Psychiatry. </span>2016;<span class="ref-vol">73</span>:472–480.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4974817/">PMC free article</a>]</span> [<a href="/pubmed/27028160" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA+Psychiatry&amp;title=Genome-wide+association+study+of+cannabis+dependence+severity,+novel+risk+variants,+and+shared+genetic+risks&amp;author=R+Sherva&amp;author=Q+Wang&amp;author=H+Kranzler&amp;author=H+Zhao&amp;author=R+Koesterer&amp;volume=73&amp;publication_year=2016&amp;pages=472-480&amp;pmid=27028160&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Radmanesh F, Devan WJ, Anderson CD, Rosand J, Falcone GJ Alzheimer's Disease Neuroimaging I. Accuracy of imputation to infer unobserved APOE epsilon alleles in genome-wide genotyping data. <span><span class="ref-journal">Eur J Hum Genet. </span>2014;<span class="ref-vol">22</span>:1239–1242.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4169533/">PMC free article</a>]</span> [<a href="/pubmed/24448547" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Accuracy+of+imputation+to+infer+unobserved+APOE+epsilon+alleles+in+genome-wide+genotyping+data&amp;author=F+Radmanesh&amp;author=WJ+Devan&amp;author=CD+Anderson&amp;author=J+Rosand&amp;author=GJ+Falcone&amp;volume=22&amp;publication_year=2014&amp;pages=1239-1242&amp;pmid=24448547&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Zhou X, Stephens M. Genome-wide efficient mixed-model analysis for association studies. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:821–824.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3386377/">PMC free article</a>]</span> [<a href="/pubmed/22706312" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+efficient+mixed-model+analysis+for+association+studies&amp;author=X+Zhou&amp;author=M+Stephens&amp;volume=44&amp;publication_year=2012&amp;pages=821-824&amp;pmid=22706312&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Bierut LJ, Strickland JR, Thompson JR, Afful SE, Cottler LB. Drug use and dependence in cocaine dependent subjects, community-based individuals, and their siblings. <span><span class="ref-journal">Drug Alcohol Depen. </span>2008;<span class="ref-vol">95</span>:14–22.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2384165/">PMC free article</a>]</span> [<a href="/pubmed/18243582" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Alcohol+Depen&amp;title=Drug+use+and+dependence+in+cocaine+dependent+subjects,+community-based+individuals,+and+their+siblings&amp;author=LJ+Bierut&amp;author=JR+Strickland&amp;author=JR+Thompson&amp;author=SE+Afful&amp;author=LB+Cottler&amp;volume=95&amp;publication_year=2008&amp;pages=14-22&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Edenberg HJ. The collaborative study on the genetics of alcoholism: an update. <span><span class="ref-journal">Alcohol Res Health. </span>2002;<span class="ref-vol">26</span>:214–218.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6683843/">PMC free article</a>]</span> [<a href="/pubmed/12875050" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Alcohol+Res+Health&amp;title=The+collaborative+study+on+the+genetics+of+alcoholism:+an+update&amp;author=HJ+Edenberg&amp;volume=26&amp;publication_year=2002&amp;pages=214-218&amp;pmid=12875050&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Risueno A, Roson-Burgo B, Dolnik A, Hernandez-Rivas JM, Bullinger L, De Las Rivas J. A robust estimation of exon expression to identify alternative spliced genes applied to human tissues and cancer samples. <span><span class="ref-journal">BMC Genomics. </span>2014;<span class="ref-vol">15</span>:879.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4298068/">PMC free article</a>]</span> [<a href="/pubmed/25297679" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=A+robust+estimation+of+exon+expression+to+identify+alternative+spliced+genes+applied+to+human+tissues+and+cancer+samples&amp;author=A+Risueno&amp;author=B+Roson-Burgo&amp;author=A+Dolnik&amp;author=JM+Hernandez-Rivas&amp;author=L+Bullinger&amp;volume=15&amp;publication_year=2014&amp;pages=879&amp;pmid=25297679&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Hernandez DG, Nalls MA, Moore M, Chong S, Dillman A, Trabzuni D, et al.  Integration of GWAS SNPs and tissue specific expression profiling reveal discrete eQTLs for human traits in blood and brain. <span><span class="ref-journal">Neurobiol Dis. </span>2012;<span class="ref-vol">47</span>:20–28.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3358430/">PMC free article</a>]</span> [<a href="/pubmed/22433082" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol+Dis&amp;title=Integration+of+GWAS+SNPs+and+tissue+specific+expression+profiling+reveal+discrete+eQTLs+for+human+traits+in+blood+and+brain&amp;author=DG+Hernandez&amp;author=MA+Nalls&amp;author=M+Moore&amp;author=S+Chong&amp;author=A+Dillman&amp;volume=47&amp;publication_year=2012&amp;pages=20-28&amp;pmid=22433082&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Barrett T, Wilhite SE, Ledoux P, Evangelista C, Kim IF, Tomashevsky M, et al.  NCBI GEO: archive for functional genomics data sets-update. <span><span class="ref-journal">Nucleic Acids Res. </span>2013;<span class="ref-vol">41</span>:D991–D995.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3531084/">PMC free article</a>]</span> [<a href="/pubmed/23193258" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=NCBI+GEO:+archive+for+functional+genomics+data+sets-update&amp;author=T+Barrett&amp;author=SE+Wilhite&amp;author=P+Ledoux&amp;author=C+Evangelista&amp;author=IF+Kim&amp;volume=41&amp;publication_year=2013&amp;pages=D991-D995&amp;pmid=23193258&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Ramasamy A, Trabzuni D, Guelfi S, Varghese V, Smith C, Walker R, et al.  Genetic variability in the regulation of gene expression in ten regions of the human brain. <span><span class="ref-journal">Nat Neurosci. </span>2014;<span class="ref-vol">17</span>:1418–1428.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4208299/">PMC free article</a>]</span> [<a href="/pubmed/25174004" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Neurosci&amp;title=Genetic+variability+in+the+regulation+of+gene+expression+in+ten+regions+of+the+human+brain&amp;author=A+Ramasamy&amp;author=D+Trabzuni&amp;author=S+Guelfi&amp;author=V+Varghese&amp;author=C+Smith&amp;volume=17&amp;publication_year=2014&amp;pages=1418-1428&amp;pmid=25174004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Cheng Z, Chu H, Fan Y, Li C, Song YQ, Zhou J, et al.  PExFInS: an integrative post-GWAS explorer for functional indels and SNPs. <span><span class="ref-journal">Sci Rep. </span>2015;<span class="ref-vol">5</span>:17302.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4661514/">PMC free article</a>]</span> [<a href="/pubmed/26612672" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Rep&amp;title=PExFInS:+an+integrative+post-GWAS+explorer+for+functional+indels+and+SNPs&amp;author=Z+Cheng&amp;author=H+Chu&amp;author=Y+Fan&amp;author=C+Li&amp;author=YQ+Song&amp;volume=5&amp;publication_year=2015&amp;pages=17302&amp;pmid=26612672&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Korostynski M, Piechota M, Kaminska D, Solecki W, Przewlocki R. Morphine effects on striatal transcriptome in mice. <span><span class="ref-journal">Genome Biol. </span>2007;<span class="ref-vol">8</span></span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2394777/">PMC free article</a>]</span> [<a href="/pubmed/17598886" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Morphine+effects+on+striatal+transcriptome+in+mice&amp;author=M+Korostynski&amp;author=M+Piechota&amp;author=D+Kaminska&amp;author=W+Solecki&amp;author=R+Przewlocki&amp;volume=8&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Sean D, Meltzer PS. GEOquery: a bridge between the gene expression omnibus (GEO) and BioConductor. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:1846–1847.</span> [<a href="/pubmed/17496320" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GEOquery:+a+bridge+between+the+gene+expression+omnibus+(GEO)+and+BioConductor&amp;author=D+Sean&amp;author=PS+Meltzer&amp;volume=23&amp;publication_year=2007&amp;pages=1846-1847&amp;pmid=17496320&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Choudhury K, McQuillin A, Puri V, Pimm J, Datta S, Thirumalai S, et al.  A genetic association study of chromosome 11q22-24 in two different samples implicates the FXYD6 gene, encoding phosphohippolin, in susceptibility to schizophrenia. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">80</span>:664–672.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1852702/">PMC free article</a>]</span> [<a href="/pubmed/17357072" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+genetic+association+study+of+chromosome+11q22-24+in+two+different+samples+implicates+the+FXYD6+gene,+encoding+phosphohippolin,+in+susceptibility+to+schizophrenia&amp;author=K+Choudhury&amp;author=A+McQuillin&amp;author=V+Puri&amp;author=J+Pimm&amp;author=S+Datta&amp;volume=80&amp;publication_year=2007&amp;pages=664-672&amp;pmid=17357072&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Jonsson EG, Saetre P, Vares M, Andreou D, Larsson K, Timm S, et al.  DTNBP1, NRG1, DAOA, DAO and GRM3 polymorphisms and schizophrenia: an association study. <span><span class="ref-journal">Neuropsychobiology. </span>2009;<span class="ref-vol">59</span>:142–150.</span> [<a href="/pubmed/19439994" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuropsychobiology&amp;title=DTNBP1,+NRG1,+DAOA,+DAO+and+GRM3+polymorphisms+and+schizophrenia:+an+association+study&amp;author=EG+Jonsson&amp;author=P+Saetre&amp;author=M+Vares&amp;author=D+Andreou&amp;author=K+Larsson&amp;volume=59&amp;publication_year=2009&amp;pages=142-150&amp;pmid=19439994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Demontis D, Nyegaard M, Buttenschon HN, Hedemand A, Pedersen CB, Grove J, et al.  Association of GRIN1 and GRIN2A-D with schizophrenia and genetic interaction with maternal herpes simplex virus-2 infection affecting disease risk. <span><span class="ref-journal">Am J Med Genet B Neuropsychiatr Genet. </span>2011;<span class="ref-vol">156B</span>:913–922.</span> [<a href="/pubmed/21919190" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+B+Neuropsychiatr+Genet&amp;title=Association+of+GRIN1+and+GRIN2A-D+with+schizophrenia+and+genetic+interaction+with+maternal+herpes+simplex+virus-2+infection+affecting+disease+risk&amp;author=D+Demontis&amp;author=M+Nyegaard&amp;author=HN+Buttenschon&amp;author=A+Hedemand&amp;author=CB+Pedersen&amp;volume=156B&amp;publication_year=2011&amp;pages=913-922&amp;pmid=21919190&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Tsai SJ, Hong CJ, Cheng CY, Liao DL, Liou YJ. Association study of polymorphisms in post-synaptic density protein 95 (PSD-95) with schizophrenia. <span><span class="ref-journal">J Neural Transm (Vienna) </span>2007;<span class="ref-vol">114</span>:423–426.</span> [<a href="/pubmed/17093888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neural+Transm+(Vienna)&amp;title=Association+study+of+polymorphisms+in+post-synaptic+density+protein+95+(PSD-95)+with+schizophrenia&amp;author=SJ+Tsai&amp;author=CJ+Hong&amp;author=CY+Cheng&amp;author=DL+Liao&amp;author=YJ+Liou&amp;volume=114&amp;publication_year=2007&amp;pages=423-426&amp;pmid=17093888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Pereira AC, Gray JD, Kogan JF, Davidson RL, Rubin TG, Okamoto M, et al.  Age and Alzheimer/'s disease gene expression profiles reversed by the glutamate modulator riluzole. <span><span class="ref-journal">Mol Psychiatry. </span>2017;<span class="ref-vol">22</span>:296–305.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5042881/">PMC free article</a>]</span> [<a href="/pubmed/27021815" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Age+and+Alzheimer/'s+disease+gene+expression+profiles+reversed+by+the+glutamate+modulator+riluzole&amp;author=AC+Pereira&amp;author=JD+Gray&amp;author=JF+Kogan&amp;author=RL+Davidson&amp;author=TG+Rubin&amp;volume=22&amp;publication_year=2017&amp;pages=296-305&amp;pmid=27021815&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Torrico B, Fernandez-Castillo N, Hervas A, Mila M, Salgado M, Rueda I, et al.  Contribution of common and rare variants of the PTCHD1 gene to autism spectrum disorders and intellectual disability. <span><span class="ref-journal">Eur J Hum Genet. </span>2015;<span class="ref-vol">23</span>:1694–1701.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4795195/">PMC free article</a>]</span> [<a href="/pubmed/25782667" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Contribution+of+common+and+rare+variants+of+the+PTCHD1+gene+to+autism+spectrum+disorders+and+intellectual+disability&amp;author=B+Torrico&amp;author=N+Fernandez-Castillo&amp;author=A+Hervas&amp;author=M+Mila&amp;author=M+Salgado&amp;volume=23&amp;publication_year=2015&amp;pages=1694-1701&amp;pmid=25782667&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Okochi T, Kishi T, Ikeda M, Kitajima T, Kinoshita Y, Kawashima K, et al.  Genetic association analysis of NRG1 with methamphetamine-induced psychosis in a Japanese population. <span><span class="ref-journal">Prog Neuropsychopharmacol Biol Psychiatry. </span>2009;<span class="ref-vol">33</span>:903–905.</span> [<a href="/pubmed/19394386" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prog+Neuropsychopharmacol+Biol+Psychiatry&amp;title=Genetic+association+analysis+of+NRG1+with+methamphetamine-induced+psychosis+in+a+Japanese+population&amp;author=T+Okochi&amp;author=T+Kishi&amp;author=M+Ikeda&amp;author=T+Kitajima&amp;author=Y+Kinoshita&amp;volume=33&amp;publication_year=2009&amp;pages=903-905&amp;pmid=19394386&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Steinberg S, de Jong S, Andreassen OA, Werge T, Borglum AD, Mors O, et al.  Common variants at VRK2 and TCF4 conferring risk of schizophrenia. <span><span class="ref-journal">Hum Mol Genet. </span>2011;<span class="ref-vol">20</span>:4076–4081.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3298077/">PMC free article</a>]</span> [<a href="/pubmed/21791550" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Common+variants+at+VRK2+and+TCF4+conferring+risk+of+schizophrenia&amp;author=S+Steinberg&amp;author=S+de+Jong&amp;author=OA+Andreassen&amp;author=T+Werge&amp;author=AD+Borglum&amp;volume=20&amp;publication_year=2011&amp;pages=4076-4081&amp;pmid=21791550&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Tassew NG, Charish J, Seidah NG, Monnier PP. SKI-1 and Furin generate multiple RGMa fragments that regulate axonal growth. <span><span class="ref-journal">Dev Cell. </span>2012;<span class="ref-vol">22</span>:391–402.</span> [<a href="/pubmed/22340500" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Cell&amp;title=SKI-1+and+Furin+generate+multiple+RGMa+fragments+that+regulate+axonal+growth&amp;author=NG+Tassew&amp;author=J+Charish&amp;author=NG+Seidah&amp;author=PP+Monnier&amp;volume=22&amp;publication_year=2012&amp;pages=391-402&amp;pmid=22340500&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Monnier PP, Sierra A, Macchi P, Deitinghoff L, Andersen JS, Mann M, et al.  RGM is a repulsive guidance molecule for retinal axons. <span><span class="ref-journal">Nature. </span>2002;<span class="ref-vol">419</span>:392–395.</span> [<a href="/pubmed/12353034" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=RGM+is+a+repulsive+guidance+molecule+for+retinal+axons&amp;author=PP+Monnier&amp;author=A+Sierra&amp;author=P+Macchi&amp;author=L+Deitinghoff&amp;author=JS+Andersen&amp;volume=419&amp;publication_year=2002&amp;pages=392-395&amp;pmid=12353034&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Hata K, Fujitani M, Yasuda Y, Doya H, Saito T, Yamagishi S, et al.  RGMa inhibition promotes axonal growth and recovery after spinal cord injury. <span><span class="ref-journal">JCB. </span>2006;<span class="ref-vol">173</span>:47–58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2063787/">PMC free article</a>]</span> [<a href="/pubmed/16585268" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JCB&amp;title=RGMa+inhibition+promotes+axonal+growth+and+recovery+after+spinal+cord+injury&amp;author=K+Hata&amp;author=M+Fujitani&amp;author=Y+Yasuda&amp;author=H+Doya&amp;author=T+Saito&amp;volume=173&amp;publication_year=2006&amp;pages=47-58&amp;pmid=16585268&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Kyoto A, Hata K, Yamashita T. Synapse formation of the cortico-spinal axons is enhanced by RGMa inhibition after spinal cord injury. <span><span class="ref-journal">Brain Res. </span>2007;<span class="ref-vol">1186</span>:74–86.</span> [<a href="/pubmed/17996222" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Res&amp;title=Synapse+formation+of+the+cortico-spinal+axons+is+enhanced+by+RGMa+inhibition+after+spinal+cord+injury&amp;author=A+Kyoto&amp;author=K+Hata&amp;author=T+Yamashita&amp;volume=1186&amp;publication_year=2007&amp;pages=74-86&amp;pmid=17996222&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Matsunaga E, Tauszig-Delamasure S, Monnier PP, Mueller BK, Strittmatter SM, Mehlen P, et al.  RGM and its receptor neogenin regulate neuronal survival. <span><span class="ref-journal">Nat Cell Biol. </span>2004;<span class="ref-vol">6</span>:749–755.</span> [<a href="/pubmed/15258591" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Cell+Biol&amp;title=RGM+and+its+receptor+neogenin+regulate+neuronal+survival&amp;author=E+Matsunaga&amp;author=S+Tauszig-Delamasure&amp;author=PP+Monnier&amp;author=BK+Mueller&amp;author=SM+Strittmatter&amp;volume=6&amp;publication_year=2004&amp;pages=749-755&amp;pmid=15258591&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Schwab JM, Monnier PP, Schluesener HJ, Conrad S, Beschorner R, Chen L, et al.  Central nervous system injury-induced repulsive guidance molecule expression in the adult human brain. <span><span class="ref-journal">Arch Neurol. </span>2005;<span class="ref-vol">62</span>:1561–1568.</span> [<a href="/pubmed/16216939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Neurol&amp;title=Central+nervous+system+injury-induced+repulsive+guidance+molecule+expression+in+the+adult+human+brain&amp;author=JM+Schwab&amp;author=PP+Monnier&amp;author=HJ+Schluesener&amp;author=S+Conrad&amp;author=R+Beschorner&amp;volume=62&amp;publication_year=2005&amp;pages=1561-1568&amp;pmid=16216939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Satoh J, Tabunoki H, Ishida T, Saito Y, Arima K. Accumulation of a repulsive axonal guidance molecule RGMa in amyloid plaques: a possible hallmark of regenerative failure in Alzheimer's disease brains. <span><span class="ref-journal">Neuropathol Appl Neurobiol. </span>2013;<span class="ref-vol">39</span>:109–120.</span> [<a href="/pubmed/22582881" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuropathol+Appl+Neurobiol&amp;title=Accumulation+of+a+repulsive+axonal+guidance+molecule+RGMa+in+amyloid+plaques:+a+possible+hallmark+of+regenerative+failure+in+Alzheimer's+disease+brains&amp;author=J+Satoh&amp;author=H+Tabunoki&amp;author=T+Ishida&amp;author=Y+Saito&amp;author=K+Arima&amp;volume=39&amp;publication_year=2013&amp;pages=109-120&amp;pmid=22582881&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Bossers K, Meerhoff G, Balesar R, van Dongen JW, Kruse CG, Swaab DF, et al.  Analysis of gene expression in Parkinson's disease: possible involvement of neurotrophic support and axon guidance in dopaminergic cell death. <span><span class="ref-journal">Brain Pathol. </span>2009;<span class="ref-vol">19</span>:91–107.</span> [<a href="/pubmed/18462474" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Pathol&amp;title=Analysis+of+gene+expression+in+Parkinson's+disease:+possible+involvement+of+neurotrophic+support+and+axon+guidance+in+dopaminergic+cell+death&amp;author=K+Bossers&amp;author=G+Meerhoff&amp;author=R+Balesar&amp;author=JW+van+Dongen&amp;author=CG+Kruse&amp;volume=19&amp;publication_year=2009&amp;pages=91-107&amp;pmid=18462474&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Nohra R, Beyeen AD, Guo JP, Khademi M, Sundqvist E, Hedreul M, et al.  RGMA and IL21R show association with experimental inflammation and multiple sclerosis. <span><span class="ref-journal">Genes Immun. </span>2010;<span class="ref-vol">11</span>:279–293.</span> [<a href="/pubmed/20072140" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun&amp;title=RGMA+and+IL21R+show+association+with+experimental+inflammation+and+multiple+sclerosis&amp;author=R+Nohra&amp;author=AD+Beyeen&amp;author=JP+Guo&amp;author=M+Khademi&amp;author=E+Sundqvist&amp;volume=11&amp;publication_year=2010&amp;pages=279-293&amp;pmid=20072140&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Demicheva E, Cui YF, Bardwell P, Barghorn S, Kron M, Meyer AH, et al.  Targeting repulsive guidance molecule a to promote regeneration and neuroprotection in multiple sclerosis. <span><span class="ref-journal">Cell Rep. </span>2015;<span class="ref-vol">10</span>:1887–1898.</span> [<a href="/pubmed/25801027" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Rep&amp;title=Targeting+repulsive+guidance+molecule+a+to+promote+regeneration+and+neuroprotection+in+multiple+sclerosis&amp;author=E+Demicheva&amp;author=YF+Cui&amp;author=P+Bardwell&amp;author=S+Barghorn&amp;author=M+Kron&amp;volume=10&amp;publication_year=2015&amp;pages=1887-1898&amp;pmid=25801027&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Mundo E, Tharmalingham S, Neves-Pereira M, Dalton EJ, Macciardi F, Parikh SV, et al.  Evidence that the N-methyl-D-aspartate subunit 1 receptor gene (GRIN1) confers susceptibility to bipolar disorder. <span><span class="ref-journal">Mol Psychiatry. </span>2003;<span class="ref-vol">8</span>:241–245.</span> [<a href="/pubmed/12610658" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Evidence+that+the+N-methyl-D-aspartate+subunit+1+receptor+gene+(GRIN1)+confers+susceptibility+to+bipolar+disorder&amp;author=E+Mundo&amp;author=S+Tharmalingham&amp;author=M+Neves-Pereira&amp;author=EJ+Dalton&amp;author=F+Macciardi&amp;volume=8&amp;publication_year=2003&amp;pages=241-245&amp;pmid=12610658&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Ge X, Qiu Y, Loh HH, Law PY. GRIN1 regulates micro-opioid receptor activities by tethering the receptor and G protein in the lipid raft. <span><span class="ref-journal">J Biol Chem. </span>2009;<span class="ref-vol">284</span>:36521–36534.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2794768/">PMC free article</a>]</span> [<a href="/pubmed/19861419" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=GRIN1+regulates+micro-opioid+receptor+activities+by+tethering+the+receptor+and+G+protein+in+the+lipid+raft&amp;author=X+Ge&amp;author=Y+Qiu&amp;author=HH+Loh&amp;author=PY+Law&amp;volume=284&amp;publication_year=2009&amp;pages=36521-36534&amp;pmid=19861419&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Jo DG, Lee JY, Hong YM, Song S, Mook-Jung I, Koh JY, et al.  Induction of pro-apoptotic calsenilin/DREAM/KChIP3 in Alzheimer's disease and cultured neurons after amyloid-beta exposure. <span><span class="ref-journal">J Neurochem. </span>2004;<span class="ref-vol">88</span>:604–611.</span> [<a href="/pubmed/14720210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurochem&amp;title=Induction+of+pro-apoptotic+calsenilin/DREAM/KChIP3+in+Alzheimer's+disease+and+cultured+neurons+after+amyloid-beta+exposure&amp;author=DG+Jo&amp;author=JY+Lee&amp;author=YM+Hong&amp;author=S+Song&amp;author=I+Mook-Jung&amp;volume=88&amp;publication_year=2004&amp;pages=604-611&amp;pmid=14720210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Wang JC, Foroud T, Hinrichs AL, Le NX, Bertelsen S, Budde JP, et al.  A genome-wide association study of alcohol-dependence symptom counts in extended pedigrees identifies C15orf53. <span><span class="ref-journal">Mol Psychiatry. </span>2013;<span class="ref-vol">18</span>:1218–1224.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3752321/">PMC free article</a>]</span> [<a href="/pubmed/23089632" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=A+genome-wide+association+study+of+alcohol-dependence+symptom+counts+in+extended+pedigrees+identifies+C15orf53&amp;author=JC+Wang&amp;author=T+Foroud&amp;author=AL+Hinrichs&amp;author=NX+Le&amp;author=S+Bertelsen&amp;volume=18&amp;publication_year=2013&amp;pages=1218-1224&amp;pmid=23089632&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Bierut LJ, Strickland JR, Thompson JR, Afful SE, Cottler LB. Drug use and dependence in cocaine dependent subjects, community-based individuals, and their siblings. <span><span class="ref-journal">Drug Alcohol Depen. </span>2008;<span class="ref-vol">95</span>:14–22.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2384165/">PMC free article</a>]</span> [<a href="/pubmed/18243582" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Alcohol+Depen&amp;title=Drug+use+and+dependence+in+cocaine+dependent+subjects,+community-based+individuals,+and+their+siblings&amp;author=LJ+Bierut&amp;author=JR+Strickland&amp;author=JR+Thompson&amp;author=SE+Afful&amp;author=LB+Cottler&amp;volume=95&amp;publication_year=2008&amp;pages=14-22&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Luo ZH, Alvarado GF, Hatsukami DK, Johnson EO, Bierut LJ, Breslau N. Race differences in nicotine dependence in the Collaborative Genetic study of Nicotine Dependence (COGEND) <span><span class="ref-journal">Nicotine Tob Res. </span>2008;<span class="ref-vol">10</span>:1223–1230.</span> [<a href="/pubmed/18629733" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nicotine+Tob+Res&amp;title=Race+differences+in+nicotine+dependence+in+the+Collaborative+Genetic+study+of+Nicotine+Dependence+(COGEND)&amp;author=ZH+Luo&amp;author=GF+Alvarado&amp;author=DK+Hatsukami&amp;author=EO+Johnson&amp;author=LJ+Bierut&amp;volume=10&amp;publication_year=2008&amp;pages=1223-1230&amp;pmid=18629733&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Ward LD, Kellis M. HaploReg v4: systematic mining of putative causal variants, cell types, regulators and target genes for human complex traits and disease. <span><span class="ref-journal">Nucleic Acids Res. </span>2016;<span class="ref-vol">44</span>:D877–881.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4702929/">PMC free article</a>]</span> [<a href="/pubmed/26657631" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=HaploReg+v4:+systematic+mining+of+putative+causal+variants,+cell+types,+regulators+and+target+genes+for+human+complex+traits+and+disease&amp;author=LD+Ward&amp;author=M+Kellis&amp;volume=44&amp;publication_year=2016&amp;pages=D877-881&amp;pmid=26657631&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6041180&amp;issue-id=316700&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140355639567072title">Abstract</a></li><li><a href="#S5title">Introduction</a></li><li><a href="#S6title">Materials and Methods</a></li><li><a href="#S12title">Results</a></li><li><a href="#S13title">Discussion</a></li><li><a href="#S14title">Supplementary Material</a></li><li><a href="#S15title">Acknowledgments</a></li><li><a href="#idm140355627277536title">Footnotes</a></li><li><a href="#idm140355645664288title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC6041180/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC6041180/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC6041180/pdf/nihms933995.pdf">PDF (549K)</a></li> | <li><a href="#" data-citationid="PMC6041180" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6041180%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6041180%2F&amp;text=Genome-wide%20association%20study%20identifies%20a%20regulatory%20variant%20of%20RGMA%20associated%20with%20opioid%20dependence%20in%20European%20Americans" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6041180%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="29478698" data-aiid="6041180" data-aid="6041180" data-iid="316700" data-domainid="319" data-domain="nihpa" data-accid="PMC6041180" data-md5="8670b29b2c6532d022ad5d986a1820b6"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T17:00:21-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE8847C0DD5A93A10000000008800505&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6041180%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC6041180/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>