<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC6078638/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>
Trans-ethnic meta-analysis of genome-wide association studies for Hirschsprung disease</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6078638/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="
Trans-ethnic meta-analysis of genome-wide association studies for Hirschsprung disease"><meta name="citation_authors" content="Clara Sze-man Tang, Hongsheng Gui, Ashish Kapoor, Jeong-Hyun Kim, Berta Luzón-Toro, Anna Pelet, Grzegorz Burzynski, Francesca Lantieri, Man-ting So, Courtney Berrios, Hyoung Doo Shin, Raquel M. Fernández, Thuy-Linh Le, Joke B.G.M. Verheij, Ivana Matera, Stacey S. Cherny, Priyanka Nandakumar, Hyun Sub Cheong, Guillermo Antiñolo, Jeanne Amiel, Jeong-Meen Seo, Dae-Yeon Kim, Jung-Tak Oh, Stanislas Lyonnet, Salud Borrego, Isabella Ceccherini, Robert M.W. Hofstra, Aravinda Chakravarti, Hyun-Young Kim, Pak Chung Sham, Paul K.H. Tam, Maria-Mercè Garcia-Barceló"><meta name="citation_date" content="1 December 2016"><meta name="citation_issue" content="23"><meta name="citation_volume" content="25"><meta name="citation_firstpage" content="5265"><meta name="citation_doi" content="10.1093/hmg/ddw333"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC6078638/?report=abstract"><meta name="citation_pmid" content="27702942"><meta name="DC.Title" content="
Trans-ethnic meta-analysis of genome-wide association studies for Hirschsprung disease"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Clara Sze-man Tang"><meta name="DC.Contributor" content="Hongsheng Gui"><meta name="DC.Contributor" content="Ashish Kapoor"><meta name="DC.Contributor" content="Jeong-Hyun Kim"><meta name="DC.Contributor" content="Berta Luzón-Toro"><meta name="DC.Contributor" content="Anna Pelet"><meta name="DC.Contributor" content="Grzegorz Burzynski"><meta name="DC.Contributor" content="Francesca Lantieri"><meta name="DC.Contributor" content="Man-ting So"><meta name="DC.Contributor" content="Courtney Berrios"><meta name="DC.Contributor" content="Hyoung Doo Shin"><meta name="DC.Contributor" content="Raquel M. Fernández"><meta name="DC.Contributor" content="Thuy-Linh Le"><meta name="DC.Contributor" content="Joke B.G.M. Verheij"><meta name="DC.Contributor" content="Ivana Matera"><meta name="DC.Contributor" content="Stacey S. Cherny"><meta name="DC.Contributor" content="Priyanka Nandakumar"><meta name="DC.Contributor" content="Hyun Sub Cheong"><meta name="DC.Contributor" content="Guillermo Antiñolo"><meta name="DC.Contributor" content="Jeanne Amiel"><meta name="DC.Contributor" content="Jeong-Meen Seo"><meta name="DC.Contributor" content="Dae-Yeon Kim"><meta name="DC.Contributor" content="Jung-Tak Oh"><meta name="DC.Contributor" content="Stanislas Lyonnet"><meta name="DC.Contributor" content="Salud Borrego"><meta name="DC.Contributor" content="Isabella Ceccherini"><meta name="DC.Contributor" content="Robert M.W. Hofstra"><meta name="DC.Contributor" content="Aravinda Chakravarti"><meta name="DC.Contributor" content="Hyun-Young Kim"><meta name="DC.Contributor" content="Pak Chung Sham"><meta name="DC.Contributor" content="Paul K.H. Tam"><meta name="DC.Contributor" content="Maria-Mercè Garcia-Barceló"><meta name="DC.Date" content="2016 Dec 1"><meta name="DC.Identifier" content="10.1093/hmg/ddw333"><meta name="DC.Language" content="en"><meta property="og:title" content="
Trans-ethnic meta-analysis of genome-wide association studies for Hirschsprung disease"><meta property="og:type" content="article"><meta property="og:description" content="Hirschsprung disease (HSCR) is the most common cause of neonatal intestinal obstruction. It is characterized by the absence of ganglia in the nerve plexuses of the lower gastrointestinal tract. So far, three common disease-susceptibility variants at the ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6078638/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC6078638/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC6078638/pdf/ddw333.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8902FDDD595CB100000000063103D3.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6078638%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC6078638</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2016 Dec 1; 25(23): 5265–5275. </span></div><div><span class="fm-vol-iss-date">Published online 2016 Oct 3. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddw333" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddw333</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC6078638</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/27702942">27702942</a></div></div></div></div><h1 class="content-title">
<em>Trans</em>-ethnic meta-analysis of genome-wide association studies for Hirschsprung disease</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Tang%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663910206864" co-class="co-affbox">Clara Sze-man Tang</a>,<sup>1,</sup><sup>2,</sup><sup>5</sup> <a href="/pubmed/?term=Gui%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663910204240" co-class="co-affbox">Hongsheng Gui</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Kapoor%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663944321600" co-class="co-affbox">Ashish Kapoor</a>,<sup>6</sup> <a href="/pubmed/?term=Kim%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663944319616" co-class="co-affbox">Jeong-Hyun Kim</a>,<sup>7</sup> <a href="/pubmed/?term=Luz%26%23x000f3%3Bn-Toro%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663911600064" co-class="co-affbox">Berta Luzón-Toro</a>,<sup>9,</sup><sup>10</sup> <a href="/pubmed/?term=Pelet%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663984778464" co-class="co-affbox">Anna Pelet</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Burzynski%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663984775744" co-class="co-affbox">Grzegorz Burzynski</a>,<sup>13</sup> <a href="/pubmed/?term=Lantieri%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663944624704" co-class="co-affbox">Francesca Lantieri</a>,<sup>14</sup> <a href="/pubmed/?term=So%20Mt%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663944622720" co-class="co-affbox">Man-ting So</a>,<sup>1</sup> <a href="/pubmed/?term=Berrios%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663944620736" co-class="co-affbox">Courtney Berrios</a>,<sup>6</sup> <a href="/pubmed/?term=Shin%20HD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663906890144" co-class="co-affbox">Hyoung Doo Shin</a>,<sup>7,</sup><sup>8</sup> <a href="/pubmed/?term=Fern%26%23x000e1%3Bndez%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663908020464" co-class="co-affbox">Raquel M. Fernández</a>,<sup>9,</sup><sup>10</sup> <a href="/pubmed/?term=Le%20TL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663908017744" co-class="co-affbox">Thuy-Linh Le</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Verheij%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663948466496" co-class="co-affbox">Joke B.G.M. Verheij</a>,<sup>13</sup> <a href="/pubmed/?term=Matera%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663948464512" co-class="co-affbox">Ivana Matera</a>,<sup>14</sup> <a href="/pubmed/?term=Cherny%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663911231360" co-class="co-affbox">Stacey S. Cherny</a>,<sup>2,</sup><sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Nandakumar%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663911227904" co-class="co-affbox">Priyanka Nandakumar</a>,<sup>6</sup> <a href="/pubmed/?term=Cheong%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663947948608" co-class="co-affbox">Hyun Sub Cheong</a>,<sup>15</sup> <a href="/pubmed/?term=Anti%26%23x000f1%3Bolo%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663947946624" co-class="co-affbox">Guillermo Antiñolo</a>,<sup>9,</sup><sup>10</sup> <a href="/pubmed/?term=Amiel%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663987707584" co-class="co-affbox">Jeanne Amiel</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Seo%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663946617344" co-class="co-affbox">Jeong-Meen Seo</a>,<sup>16</sup> <a href="/pubmed/?term=Kim%20DY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663946615360" co-class="co-affbox">Dae-Yeon Kim</a>,<sup>17</sup> <a href="/pubmed/?term=Oh%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663946613376" co-class="co-affbox">Jung-Tak Oh</a>,<sup>18</sup> <a href="/pubmed/?term=Lyonnet%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663910697136" co-class="co-affbox">Stanislas Lyonnet</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Borrego%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663910694416" co-class="co-affbox">Salud Borrego</a>,<sup>9,</sup><sup>10</sup> <a href="/pubmed/?term=Ceccherini%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663948701792" co-class="co-affbox">Isabella Ceccherini</a>,<sup>14</sup> <a href="/pubmed/?term=Hofstra%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663906355984" co-class="co-affbox">Robert M.W. Hofstra</a>,<sup>13,</sup><sup>19</sup> <a href="/pubmed/?term=Chakravarti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663906353376" co-class="co-affbox">Aravinda Chakravarti</a>,<sup>6</sup> <a href="/pubmed/?term=Kim%20HY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663903749744" co-class="co-affbox">Hyun-Young Kim</a>,<sup>20</sup> <a href="/pubmed/?term=Sham%20PC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663903747760" co-class="co-affbox">Pak Chung Sham</a>,<sup>2,</sup><sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Tam%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663901826688" co-class="co-affbox">Paul K.H. Tam</a>,<sup>1,</sup><sup>5</sup><sup></sup> and  <a href="/pubmed/?term=Garcia-Barcel%26%23x000f3%3B%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942" class="affpopup" co-rid="_co_idm139663948318608" co-class="co-affbox">Maria-Mercè Garcia-Barceló</a><sup>1,</sup><sup>2</sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139663910206864"><h3 class="no_margin">Clara Sze-man Tang</h3><p>  <sup>1</sup>Department of Surgery,</p><p>  <sup>2</sup>Centre for Genomic Sciences,</p><p>  <sup>5</sup>Dr Li Dak-Sum Research Centre, The University of Hong Kong – Karolinska Institutet Collaboration in Regenerative Medicine, Hong Kong SAR, China,</p><div>Find articles by <a href="/pubmed/?term=Tang%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Clara Sze-man Tang</a></div></div><div id="_co_idm139663910204240"><h3 class="no_margin">Hongsheng Gui</h3><p>  <sup>1</sup>Department of Surgery,</p><p>  <sup>2</sup>Centre for Genomic Sciences,</p><div>Find articles by <a href="/pubmed/?term=Gui%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Hongsheng Gui</a></div></div><div id="_co_idm139663944321600"><h3 class="no_margin">Ashish Kapoor</h3><p>  <sup>6</sup>Center for Complex Disease Genomics, McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, MD 21205, USA,</p><div>Find articles by <a href="/pubmed/?term=Kapoor%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Ashish Kapoor</a></div></div><div id="_co_idm139663944319616"><h3 class="no_margin">Jeong-Hyun Kim</h3><p>  <sup>7</sup>Research Institute for Basic Science, Sogang University, Seoul 121-742, Republic of Korea,</p><div>Find articles by <a href="/pubmed/?term=Kim%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Jeong-Hyun Kim</a></div></div><div id="_co_idm139663911600064"><h3 class="no_margin">Berta Luzón-Toro</h3><p>  <sup>9</sup>Department of Genetics, Reproduction and Fetal Medicine, Institute of Biomedicine of Seville (IBIS), University Hospital Virgen del Rocío/CSIC/University of Seville, Seville, Spain,</p><p>  <sup>10</sup>Centre for Biomedical Network Research on Rare Diseases (CIBERER), Seville, Spain,</p><div>Find articles by <a href="/pubmed/?term=Luz%26%23x000f3%3Bn-Toro%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Berta Luzón-Toro</a></div></div><div id="_co_idm139663984778464"><h3 class="no_margin">Anna Pelet</h3><p>  <sup>11</sup>Laboratory of Embryology and Genetics of Congenital Malformations, Institut National de la Santé et de la Recherche Médicale (INSERM) U1163, Institut Imagine, Paris, France,</p><p>  <sup>12</sup>Paris Descartes-Sorbonne Paris Cité University, Institut Imagine, Paris, France,</p><div>Find articles by <a href="/pubmed/?term=Pelet%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Anna Pelet</a></div></div><div id="_co_idm139663984775744"><h3 class="no_margin">Grzegorz Burzynski</h3><p>  <sup>13</sup>Department of Clinical Genetic, Erasmus Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Burzynski%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Grzegorz Burzynski</a></div></div><div id="_co_idm139663944624704"><h3 class="no_margin">Francesca Lantieri</h3><p>  <sup>14</sup>UOC Genetica Medica, Istituto Giannina Gaslini, 16148 Genova, Italy,</p><div>Find articles by <a href="/pubmed/?term=Lantieri%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Francesca Lantieri</a></div></div><div id="_co_idm139663944622720"><h3 class="no_margin">Man-ting So</h3><p>  <sup>1</sup>Department of Surgery,</p><div>Find articles by <a href="/pubmed/?term=So%20Mt%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Man-ting So</a></div></div><div id="_co_idm139663944620736"><h3 class="no_margin">Courtney Berrios</h3><p>  <sup>6</sup>Center for Complex Disease Genomics, McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, MD 21205, USA,</p><div>Find articles by <a href="/pubmed/?term=Berrios%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Courtney Berrios</a></div></div><div id="_co_idm139663906890144"><h3 class="no_margin">Hyoung Doo Shin</h3><p>  <sup>7</sup>Research Institute for Basic Science, Sogang University, Seoul 121-742, Republic of Korea,</p><p>  <sup>8</sup>Department of Life Science, Sogang University, Seoul 121-742, Republic of Korea,</p><div>Find articles by <a href="/pubmed/?term=Shin%20HD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Hyoung Doo Shin</a></div></div><div id="_co_idm139663908020464"><h3 class="no_margin">Raquel M. Fernández</h3><p>  <sup>9</sup>Department of Genetics, Reproduction and Fetal Medicine, Institute of Biomedicine of Seville (IBIS), University Hospital Virgen del Rocío/CSIC/University of Seville, Seville, Spain,</p><p>  <sup>10</sup>Centre for Biomedical Network Research on Rare Diseases (CIBERER), Seville, Spain,</p><div>Find articles by <a href="/pubmed/?term=Fern%26%23x000e1%3Bndez%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Raquel M. Fernández</a></div></div><div id="_co_idm139663908017744"><h3 class="no_margin">Thuy-Linh Le</h3><p>  <sup>11</sup>Laboratory of Embryology and Genetics of Congenital Malformations, Institut National de la Santé et de la Recherche Médicale (INSERM) U1163, Institut Imagine, Paris, France,</p><p>  <sup>12</sup>Paris Descartes-Sorbonne Paris Cité University, Institut Imagine, Paris, France,</p><div>Find articles by <a href="/pubmed/?term=Le%20TL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Thuy-Linh Le</a></div></div><div id="_co_idm139663948466496"><h3 class="no_margin">Joke B.G.M. Verheij</h3><p>  <sup>13</sup>Department of Clinical Genetic, Erasmus Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Verheij%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Joke B.G.M. Verheij</a></div></div><div id="_co_idm139663948464512"><h3 class="no_margin">Ivana Matera</h3><p>  <sup>14</sup>UOC Genetica Medica, Istituto Giannina Gaslini, 16148 Genova, Italy,</p><div>Find articles by <a href="/pubmed/?term=Matera%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Ivana Matera</a></div></div><div id="_co_idm139663911231360"><h3 class="no_margin">Stacey S. Cherny</h3><p>  <sup>2</sup>Centre for Genomic Sciences,</p><p>  <sup>3</sup>Department of Psychiatry,</p><p>  <sup>4</sup>State Key Laboratory of Brain and Cognitive Sciences, The University of Hong Kong, Pokfulam, Hong Kong SAR, China,</p><div>Find articles by <a href="/pubmed/?term=Cherny%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Stacey S. Cherny</a></div></div><div id="_co_idm139663911227904"><h3 class="no_margin">Priyanka Nandakumar</h3><p>  <sup>6</sup>Center for Complex Disease Genomics, McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, MD 21205, USA,</p><div>Find articles by <a href="/pubmed/?term=Nandakumar%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Priyanka Nandakumar</a></div></div><div id="_co_idm139663947948608"><h3 class="no_margin">Hyun Sub Cheong</h3><p>  <sup>15</sup>Department of Genetic Epidemiology, SNP Genetics, Inc., Seoul 121-742, Republic of Korea,</p><div>Find articles by <a href="/pubmed/?term=Cheong%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Hyun Sub Cheong</a></div></div><div id="_co_idm139663947946624"><h3 class="no_margin">Guillermo Antiñolo</h3><p>  <sup>9</sup>Department of Genetics, Reproduction and Fetal Medicine, Institute of Biomedicine of Seville (IBIS), University Hospital Virgen del Rocío/CSIC/University of Seville, Seville, Spain,</p><p>  <sup>10</sup>Centre for Biomedical Network Research on Rare Diseases (CIBERER), Seville, Spain,</p><div>Find articles by <a href="/pubmed/?term=Anti%26%23x000f1%3Bolo%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Guillermo Antiñolo</a></div></div><div id="_co_idm139663987707584"><h3 class="no_margin">Jeanne Amiel</h3><p>  <sup>11</sup>Laboratory of Embryology and Genetics of Congenital Malformations, Institut National de la Santé et de la Recherche Médicale (INSERM) U1163, Institut Imagine, Paris, France,</p><p>  <sup>12</sup>Paris Descartes-Sorbonne Paris Cité University, Institut Imagine, Paris, France,</p><div>Find articles by <a href="/pubmed/?term=Amiel%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Jeanne Amiel</a></div></div><div id="_co_idm139663946617344"><h3 class="no_margin">Jeong-Meen Seo</h3><p>  <sup>16</sup>Division of Pediatric Surgery, Department of Surgery, Samsung Medical Center, Sungkyunkwan University School of Medicine, Seoul 135-710, Republic of Korea,</p><div>Find articles by <a href="/pubmed/?term=Seo%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Jeong-Meen Seo</a></div></div><div id="_co_idm139663946615360"><h3 class="no_margin">Dae-Yeon Kim</h3><p>  <sup>17</sup>Department of Pediatric Surgery, Asan Medical Center, University of Ulsan College of Medicine, Seoul 138-736, Republic of Korea,</p><div>Find articles by <a href="/pubmed/?term=Kim%20DY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Dae-Yeon Kim</a></div></div><div id="_co_idm139663946613376"><h3 class="no_margin">Jung-Tak Oh</h3><p>  <sup>18</sup>Department of Pediatric Surgery, Severance Children’s Hospital, Yonsei University College of Medicine, Seoul 120-752, Republic of Korea,</p><div>Find articles by <a href="/pubmed/?term=Oh%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Jung-Tak Oh</a></div></div><div id="_co_idm139663910697136"><h3 class="no_margin">Stanislas Lyonnet</h3><p>  <sup>11</sup>Laboratory of Embryology and Genetics of Congenital Malformations, Institut National de la Santé et de la Recherche Médicale (INSERM) U1163, Institut Imagine, Paris, France,</p><p>  <sup>12</sup>Paris Descartes-Sorbonne Paris Cité University, Institut Imagine, Paris, France,</p><div>Find articles by <a href="/pubmed/?term=Lyonnet%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Stanislas Lyonnet</a></div></div><div id="_co_idm139663910694416"><h3 class="no_margin">Salud Borrego</h3><p>  <sup>9</sup>Department of Genetics, Reproduction and Fetal Medicine, Institute of Biomedicine of Seville (IBIS), University Hospital Virgen del Rocío/CSIC/University of Seville, Seville, Spain,</p><p>  <sup>10</sup>Centre for Biomedical Network Research on Rare Diseases (CIBERER), Seville, Spain,</p><div>Find articles by <a href="/pubmed/?term=Borrego%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Salud Borrego</a></div></div><div id="_co_idm139663948701792"><h3 class="no_margin">Isabella Ceccherini</h3><p>  <sup>14</sup>UOC Genetica Medica, Istituto Giannina Gaslini, 16148 Genova, Italy,</p><div>Find articles by <a href="/pubmed/?term=Ceccherini%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Isabella Ceccherini</a></div></div><div id="_co_idm139663906355984"><h3 class="no_margin">Robert M.W. Hofstra</h3><p>  <sup>13</sup>Department of Clinical Genetic, Erasmus Medical Center, Rotterdam, The Netherlands</p><p>  <sup>19</sup>Stem Cells and Regenerative Medicine, Birth Defects Research Centre UCL Institute of Child Health, London, UK</p><div>Find articles by <a href="/pubmed/?term=Hofstra%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Robert M.W. Hofstra</a></div></div><div id="_co_idm139663906353376"><h3 class="no_margin">Aravinda Chakravarti</h3><p>  <sup>6</sup>Center for Complex Disease Genomics, McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, MD 21205, USA,</p><div>Find articles by <a href="/pubmed/?term=Chakravarti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Aravinda Chakravarti</a></div></div><div id="_co_idm139663903749744"><h3 class="no_margin">Hyun-Young Kim</h3><p>  <sup>20</sup>Department of Pediatric Surgery, Seoul National University Children’s Hospital, Seoul 110-744, Republic of Korea</p><div>Find articles by <a href="/pubmed/?term=Kim%20HY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Hyun-Young Kim</a></div></div><div id="_co_idm139663903747760"><h3 class="no_margin">Pak Chung Sham</h3><p>  <sup>2</sup>Centre for Genomic Sciences,</p><p>  <sup>3</sup>Department of Psychiatry,</p><p>  <sup>4</sup>State Key Laboratory of Brain and Cognitive Sciences, The University of Hong Kong, Pokfulam, Hong Kong SAR, China,</p><div>Find articles by <a href="/pubmed/?term=Sham%20PC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Pak Chung Sham</a></div></div><div id="_co_idm139663901826688"><h3 class="no_margin">Paul K.H. Tam</h3><p>  <sup>1</sup>Department of Surgery,</p><p>  <sup>5</sup>Dr Li Dak-Sum Research Centre, The University of Hong Kong – Karolinska Institutet Collaboration in Regenerative Medicine, Hong Kong SAR, China,</p><div>Find articles by <a href="/pubmed/?term=Tam%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Paul K.H. Tam</a></div></div><div id="_co_idm139663948318608"><h3 class="no_margin">Maria-Mercè Garcia-Barceló</h3><p>  <sup>1</sup>Department of Surgery,</p><p>  <sup>2</sup>Centre for Genomic Sciences,</p><div>Find articles by <a href="/pubmed/?term=Garcia-Barcel%26%23x000f3%3B%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27702942">Maria-Mercè Garcia-Barceló</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139663948933184_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139663948933184_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139663948933184_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139663948933184_ai" style="display:none"><div class="fm-affl" id="ddw333-aff1" lang="en">  <sup>1</sup>Department of Surgery,</div><div class="fm-affl" id="ddw333-aff2" lang="en">  <sup>2</sup>Centre for Genomic Sciences,</div><div class="fm-affl" id="ddw333-aff3" lang="en">  <sup>3</sup>Department of Psychiatry,</div><div class="fm-affl" id="ddw333-aff4" lang="en">  <sup>4</sup>State Key Laboratory of Brain and Cognitive Sciences, The University of Hong Kong, Pokfulam, Hong Kong SAR, China,</div><div class="fm-affl" id="ddw333-aff5" lang="en">  <sup>5</sup>Dr Li Dak-Sum Research Centre, The University of Hong Kong – Karolinska Institutet Collaboration in Regenerative Medicine, Hong Kong SAR, China,</div><div class="fm-affl" id="ddw333-aff6" lang="en">  <sup>6</sup>Center for Complex Disease Genomics, McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, MD 21205, USA,</div><div class="fm-affl" id="ddw333-aff7" lang="en">  <sup>7</sup>Research Institute for Basic Science, Sogang University, Seoul 121-742, Republic of Korea,</div><div class="fm-affl" id="ddw333-aff8" lang="en">  <sup>8</sup>Department of Life Science, Sogang University, Seoul 121-742, Republic of Korea,</div><div class="fm-affl" id="ddw333-aff9" lang="en">  <sup>9</sup>Department of Genetics, Reproduction and Fetal Medicine, Institute of Biomedicine of Seville (IBIS), University Hospital Virgen del Rocío/CSIC/University of Seville, Seville, Spain,</div><div class="fm-affl" id="ddw333-aff10" lang="en">  <sup>10</sup>Centre for Biomedical Network Research on Rare Diseases (CIBERER), Seville, Spain,</div><div class="fm-affl" id="ddw333-aff11" lang="en">  <sup>11</sup>Laboratory of Embryology and Genetics of Congenital Malformations, Institut National de la Santé et de la Recherche Médicale (INSERM) U1163, Institut Imagine, Paris, France,</div><div class="fm-affl" id="ddw333-aff12" lang="en">  <sup>12</sup>Paris Descartes-Sorbonne Paris Cité University, Institut Imagine, Paris, France,</div><div class="fm-affl" id="ddw333-aff13" lang="en">  <sup>13</sup>Department of Clinical Genetic, Erasmus Medical Center, Rotterdam, The Netherlands</div><div class="fm-affl" id="ddw333-aff14" lang="en">  <sup>14</sup>UOC Genetica Medica, Istituto Giannina Gaslini, 16148 Genova, Italy,</div><div class="fm-affl" id="ddw333-aff15" lang="en">  <sup>15</sup>Department of Genetic Epidemiology, SNP Genetics, Inc., Seoul 121-742, Republic of Korea,</div><div class="fm-affl" id="ddw333-aff16" lang="en">  <sup>16</sup>Division of Pediatric Surgery, Department of Surgery, Samsung Medical Center, Sungkyunkwan University School of Medicine, Seoul 135-710, Republic of Korea,</div><div class="fm-affl" id="ddw333-aff17" lang="en">  <sup>17</sup>Department of Pediatric Surgery, Asan Medical Center, University of Ulsan College of Medicine, Seoul 138-736, Republic of Korea,</div><div class="fm-affl" id="ddw333-aff18" lang="en">  <sup>18</sup>Department of Pediatric Surgery, Severance Children’s Hospital, Yonsei University College of Medicine, Seoul 120-752, Republic of Korea,</div><div class="fm-affl" id="ddw333-aff19" lang="en">  <sup>19</sup>Stem Cells and Regenerative Medicine, Birth Defects Research Centre UCL Institute of Child Health, London, UK</div><div class="fm-affl" id="ddw333-aff20" lang="en">  <sup>20</sup>Department of Pediatric Surgery, Seoul National University Children’s Hospital, Seoul 110-744, Republic of Korea</div><div id="ddw333-cor1"><sup>*</sup>To whom correspondence should be addressed at: Division of Paediatric Surgery, Department of Surgery, The University of Hong Kong, Hong Kong SAR, China. Tel: +852 28315073; Fax: +852 28199621; Email: <a href="mailto:dev@null" data-email="kh.ukh@aicragmm" class="oemail">kh.ukh@aicragmm</a> (M.M.G.B.), <a href="mailto:dev@null" data-email="kh.ukh@matluap" class="oemail">kh.ukh@matluap</a> (P.K.H.T.)</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139663948933184_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2016 May 16; Revised 2016 Aug 24; Accepted 2016 Sep 28.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139663948933184_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2016. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC6078638/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Data.</strong></div><div><a href="/pmc/articles/PMC6078638/bin/ddw333_supp.zip" data-ga-action="click_feat_suppl">ddw333_supp.zip</a><span style="color:gray"> (854K)</span></div><div class="small guid">GUID:&nbsp;9FBC1EBB-2990-4403-AE13-798E6CB4F4B0</div></div></dd></dl></div><div id="idm139663910190576" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139663910190576title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Hirschsprung disease (HSCR) is the most common cause of neonatal intestinal obstruction. It is characterized by the absence of ganglia in the nerve plexuses of the lower gastrointestinal tract. So far, three common disease-susceptibility variants at the <em>RET</em>, <em>SEMA3</em> and <em>NRG1</em> loci have been detected through genome-wide association studies (GWAS) in Europeans and Asians to understand its genetic etiologies. Here we present a <em>trans</em>-ethnic meta-analysis of 507 HSCR cases and 1191 controls, combining all published GWAS results on HSCR to fine-map these loci and narrow down the putatively causal variants to 99% credible sets. We also demonstrate that the effects of <em>RET</em> and <em>NRG1</em> are universal across European and Asian ancestries. In contrast, we detected a European-specific association of a low-frequency variant, rs80227144, in <em>SEMA3</em> [odds ratio (OR) = 5.2, <em>P </em>=<em> </em>4.7 × 10<sup>−10</sup>]. Conditional analyses on the lead SNPs revealed a secondary association signal, corresponding to an Asian-specific, low-frequency missense variant encoding RET p.Asp489Asn (rs9282834, conditional OR = 20.3, conditional <em>P </em>=<em> </em>4.1 × 10<sup>−14</sup>). When in <em>trans</em> with the <em>RET</em> intron 1 enhancer risk allele, rs9282834 increases the risk of HSCR from 1.1 to 26.7. Overall, our study provides further insights into the genetic architecture of HSCR and has profound implications for future study designs.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p2" class="p p-first">Hirschsprung disease (HSCR) is a rare, complex genetic disorder of the enteric nervous system (ENS). It is characterized by the absence of enteric ganglia in the myenteric and submucosal plexuses along a variable length of the hindgut (<a href="#ddw333-B1" rid="ddw333-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596608">1</a>). This developmental disorder is attributed to a failure in rostro-caudal migration, proliferation or differentiation of the enteric neural crest cells to the affected portion during embryogenesis, leading to contraction of the bowel and functional colonic obstruction. HSCR exhibits variable phenotypic expressivity and manifests with low, sex-dependent penetrance. Severity of the disease is determined by the extent of aganglionosis and can be classified into short-segment HSCR (S-HSCR; ∼80% of cases), long-segment HSCR (L-HSCR; ∼15%) and total colonic aganglionosis (TCA; ∼5%). Particularly for S-HSCR, gender bias is commonly observed, with males being affected four times more than females. The incidence of the disease (1.4–2.8 in 10&nbsp;000 newborns) also varies widely across ethnic groups, where Asia has the highest incidence rate reported (<a href="#ddw333-B1" rid="ddw333-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596626">1–3</a>).</p><p id="__p3">The mode of inheritance of HSCR varies, from dominant with reduced penetrance or recessive in familial cases to a more complex, non-Mendelian mode of inheritance in the sporadic cases. It has been long recognized as a highly heritable disorder (80–97% for S-HSCR and ∼100% for L-HSCR), although ∼80% of cases are considered sporadic (<a href="#ddw333-B4" rid="ddw333-B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596654">4</a>). To date, rare variants contributing to HSCR have been found in more than 15 genes, with <em>RET</em> being the most frequently mutated gene (<a href="#ddw333-B5" rid="ddw333-B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596598">5</a>); however, these rare variants typically have incomplete penetrance and, altogether, only explain a small proportion of the total heritability. Besides, a common variant in intron 1 of <em>RET</em> (rs2435357) also predisposes to HSCR. Despite its smaller effect size, this common, non-coding polymorphism explains 10–20 times more genetic variance than the rare, damaging <em>RET</em> variants (<a href="#ddw333-B6" rid="ddw333-B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596620">6</a>), implying that common variants might also contribute to a significant proportion of disease risk.</p><p id="__p4" class="p p-last">In order to find novel genetic variants associated with HSCR, three genome-wide association studies (GWAS) have been carried out thus far (<a href="#ddw333-B7" rid="ddw333-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596607">7–9</a>). In addition to <em>RET</em>, two novel HSCR-associated loci, semaphorin 3C/3D (<em>SEMA3</em>) at 7q21 and neuregulin-1 (<em>NRG1</em>) at 8p12, have been implicated in the GWAS of Europeans and East Asians, respectively. Semaphorin 3C and 3D are secreted glycoproteins involved in axon guidance and plexin/neuropilin-mediated neuronal migration. Likewise, neuregulin-1 also plays an important role in neural crest cell survival and differentiation through binding and interaction with ErbB tyrosine kinase receptors (<a href="#ddw333-B10" rid="ddw333-B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596646">10</a>). Follow-up mutational scans of these loci revealed similar genetic properties as those reported for <em>RET</em>, whereby both common and rare variants underlies the genetic risk to HSCR (<a href="#ddw333-B8" rid="ddw333-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596621">8</a>,<a href="#ddw333-B11" rid="ddw333-B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596633">11–13</a>). Unlike the <em>RET</em> common variant, the mechanisms by which these novel HSCR-associated alleles influence genetic susceptibility remains largely unknown. Lead SNPs of <em>NRG1</em> (rs7835688 and rs16879552) and <em>SEMA3</em> (rs12707682 and rs11766001) loci map to non-coding regions with unclear functional impact and are likely to represent surrogate markers in linkage disequilibrium (LD) with causal variants. To narrow down the potential causal variants for HSCR and to identify novel disease-susceptibility loci, we performed a <em>trans</em>-ethnic meta-analysis by aggregating the association results of all published GWAS on HSCR, totaling 507 HSCR cases and 1191 controls. By exploiting the allelic heterogeneity and difference in LD structure between populations, we aimed to (i) evaluate if the evidence of association is ancestry-specific or shared across ethnicities, and (ii) define credible sets that confidently contain the causal variants for functional interrogation.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Results</h2><p id="__p5" class="p p-first">To enhance the coverage of genetic variations in the meta-analysis, imputation was initially performed in each study with reference to the 1000 Genomes Project (Phase I Integrated, June 2014 haplotypes). An inverse-variance fixed-effects meta-analysis was carried out across Chinese and Korean studies (Asian ancestry) and the <em>trans</em>-ethnic meta-analysis was performed together with the European GWAS using a Bayesian approach—Meta-Analysis of <em>Trans</em>-ethnic association studies (MANTRA) (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Figs S2 and S3</a>) (<a href="#ddw333-B14" rid="ddw333-B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596615">14</a>). We first evaluated the potential of discovering novel HSCR-associated loci across ethnicities by assessing the concordance in the direction of effect between European GWAS and Asian ancestry meta-analysis. We observed an excess in directional concordance (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Fig. S4</a>) for SNPs showing weak to nominal evidence of association with HSCR, indicating some HSCR-associated loci are likely to be shared between ethnicities.</p><div id="__sec3" class="sec"><h3 id="__sec3title">Localizing variants causal to HSCR in known loci</h3><p id="__p6" class="p p-first">In the three original two-staged GWAS, <em>RET</em> was the only locus attaining genome-wide significance in the individual discovery stages, without the targeted genotyping in replication cohorts. After the <em>trans</em>-ethnic meta-analysis, all three known loci (<em>RET</em>, <em>NRG1</em> and <em>SEMA3</em>) showed strong evidence of association with HSCR [log<sub>10</sub>Bayes factor (BF) &gt;6.1; <a href="/pmc/articles/PMC6078638/table/ddw333-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw333-T1" rid-ob="ob-ddw333-T1" co-legend-rid=""><span>Table 1</span></a>]. We further constructed a 99% credible set for each locus, which represent the minimum set of variants accounting for 99% of the posterior probabilities of association in the region (<a href="#ddw333-B15" rid="ddw333-B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596643">15</a>). SNPs in the credible sets are most likely to be causal based on the statistical evidence from the <em>trans</em>-ethnic meta-analysis.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw333-T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p7"><em>Trans</em>-ethnic meta-analysis association results for three known HSCR-associated loci</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139663904588416" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="3" rowspan="1">European GWAS<hr></th><th colspan="3" rowspan="1">Asian-specific meta-analysis<hr></th><th colspan="3" rowspan="1"><em>Trans</em>-ethnic meta-analysis<hr></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="2" rowspan="1">99% credible set<hr></th></tr><tr><th rowspan="1" colspan="1">Locus</th><th rowspan="1" colspan="1">Lead SNP</th><th rowspan="1" colspan="1">Position</th><th rowspan="1" colspan="1">Allele<sup>a</sup></th><th rowspan="1" colspan="1">EAF<sup>b</sup></th><th rowspan="1" colspan="1">OR (95%CI)</th><th rowspan="1" colspan="1"><em>P</em></th><th rowspan="1" colspan="1">EAF</th><th rowspan="1" colspan="1">OR (95%CI)</th><th rowspan="1" colspan="1"><em>P</em></th><th rowspan="1" colspan="1">LogBF</th><th rowspan="1" colspan="1">Number of SNPs</th><th rowspan="1" colspan="1">Interval (bp)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>RET</em></td><td rowspan="1" colspan="1">rs2505998</td><td rowspan="1" colspan="1">10: 43570925</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.64/0.22</td><td rowspan="1" colspan="1">4.17 (3.23–5.26)</td><td rowspan="1" colspan="1">1.1 × 10<sup>−28</sup></td><td rowspan="1" colspan="1">0.82/0.48</td><td rowspan="1" colspan="1">3.57 (2.94–4.35)</td><td rowspan="1" colspan="1">4.44 × 10<sup>−43</sup></td><td rowspan="1" colspan="1">67.25</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">15&nbsp;913</td></tr><tr><td rowspan="1" colspan="1"><em>NRG1</em></td><td rowspan="1" colspan="1">rs7005606</td><td rowspan="1" colspan="1">8: 32401501</td><td rowspan="1" colspan="1">G/T</td><td rowspan="1" colspan="1">0.54/0.42</td><td rowspan="1" colspan="1">1.64 (1.25–2.15)</td><td rowspan="1" colspan="1">4.0 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.36/0.21</td><td rowspan="1" colspan="1">2.12 (1.70–2.63)</td><td rowspan="1" colspan="1">1.11 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">11.68</td><td rowspan="1" colspan="1">29</td><td rowspan="1" colspan="1">31&nbsp;993</td></tr><tr><td rowspan="1" colspan="1"><em>SEMA3C/3D</em></td><td rowspan="1" colspan="1">rs80227144</td><td rowspan="1" colspan="1">7: 84349842</td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1">0.14/0.03</td><td rowspan="1" colspan="1">5.2 (3.09–8.73)</td><td rowspan="1" colspan="1">4.7 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">0/0</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">7.99</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">603&nbsp;138</td></tr></tbody></table></div><div id="largeobj_idm139663904588416" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6078638/table/ddw333-T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="ddw333-TF1"><p id="__p8" class="p p-first-last"><sup>a</sup>Effect allele/other allele.</p></div><div id="ddw333-TF2"><p id="__p9" class="p p-first-last"><sup>b</sup>Effect allele frequency of cases/controls.</p></div></div></div><p id="__p10">For <em>RET</em> and <em>NRG1</em>, the 99% credible sets contained 13 and 29 non-coding SNPs each, narrowing down the association signals to 16 and 32 kb regions, respectively (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Tables S3 and S4</a>; <a href="/pmc/articles/PMC6078638/figure/ddw333-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw333-F1" rid-ob="ob-ddw333-F1" co-legend-rid="lgnd_ddw333-F1"><span>Fig. 1</span></a>, left and middle panels). Both credible set regions comprised promoter and intron 1. We observed little evidence of heterogeneity in allelic effects (posterior probability of heterogeneity &lt;0.5) between the Asian and European samples for the two loci, implying that both loci are robustly associated with HSCR irrespectively of the ancestry.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddw333-F1" co-legend-rid="lgnd_ddw333-F1"><a href="/pmc/articles/PMC6078638/figure/ddw333-F1/" target="figure" rid-figpopup="ddw333-F1" rid-ob="ob-ddw333-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139663911494416" class="figure"><a href="/pmc/articles/PMC6078638/figure/ddw333-F1/" target="figure" rid-figpopup="ddw333-F1" rid-ob="ob-ddw333-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6078638_ddw333f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddw333f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6078638/bin/ddw333f1.jpg"></a></div><div id="largeobj_idm139663911494416" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6078638/figure/ddw333-F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddw333-F1"><div><a class="figpopup" href="/pmc/articles/PMC6078638/figure/ddw333-F1/" target="figure" rid-figpopup="ddw333-F1" rid-ob="ob-ddw333-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p11">Regional plots of association for the Asian-specific and <em>trans</em>-ethnic meta-analyses at the three known HSCR-associated loci. Significance of association is reported in <em>P</em>-values for the fixed-effects Asian-specific meta-analysis (upper panel) and in Bayes factors for the <em>trans</em>-ethnic meta-analyses (lower panel). Genomic regions covered by the 99% credible set are highlighted by the purple boxes. The lead SNP of each locus is highlighted by purple diamond. Color indicates the LD (<em>r</em><sup>2</sup>) with the lead SNP.</p></div></div></div><p id="__p12">The new lead SNPs—rs2505998 for <em>RET</em> (OR = 3.6, <em>P</em><sub>Asian </sub>=<sub> </sub>4.4 × 10<sup>−43</sup>; <a href="/pmc/articles/PMC6078638/table/ddw333-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw333-T1" rid-ob="ob-ddw333-T1" co-legend-rid=""><span>Table 1</span></a>) and rs7005606 for <em>NRG1</em> (OR = 2.1, <em>P</em><sub>Asian </sub>=<sub> </sub>1.1 × 10<sup>−11</sup>)—in the Asian-specific meta-analysis also exhibited the strongest association in the <em>trans</em>-ethnic meta-analysis after incorporating the association results from Europeans (<em>RET</em>: rs2505998, log<sub>10</sub>BF = 67.3; <em>NRG1</em>: rs7005606, log<sub>10</sub>BF = 11.7). These two lead SNPs are common in both ancestries (minor allele frequency (MAF) &gt;0.2) and are in strong LD with the previously reported lead SNPs, both of which were also included in the 99% credible sets (rs2505998, <em>r</em><sup>2</sup><sub>rs2435357</sub> &gt;<sub> </sub>0.95; rs7005606, <em>r</em><sup>2</sup><sub>rs7835688 </sub>&gt;<sub> </sub>0.77). Because of the stochastic imputation or genotyping errors as well as statistical fluctuation, effect of SNPs within the same credible set is not easily distinguishable from one another.</p><p id="__p13">To identify the most plausible functional SNP and to better understand the regulatory mechanisms, we annotated the credible set variants against various regulatory databases. Functional annotation revealed that all SNPs in the two credible sets are indeed <em>cis</em>-acting expression quantitative loci (cis-eQTL; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Tables S3 and S4</a>), affecting <em>NRG1</em> expression as well as the expression of <em>RET</em> and the immediately downstream <em>CSGALNACT2</em> gene across multiple tissues in Genotype-Tissue Expression (GTEx) database. In addition, all credible set SNPs showed strong enhancer activities, marked by histone modification signatures of H3K27ac or H3K4me1 known to be enriched in active enhancers. Specifically, the <em>RET</em> lead SNP (rs2505998) overlaps with a CTCF binding site whereas the well-established functional <em>RET</em> enhancer rs2435357 has been shown to be a binding site for the transcriptional co-activator P300.</p><p id="__p14" class="p p-last">For <em>SEMA3</em> loci, 13 intergenic SNPs located downstream of <em>SEMA3D</em> were included in the 99% credible set (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S5</a>; <a href="/pmc/articles/PMC6078638/figure/ddw333-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw333-F1" rid-ob="ob-ddw333-F1" co-legend-rid="lgnd_ddw333-F1"><span>Fig. 1</span></a>, right panel). Most of the credible set SNPs (9 out of 13), including the lead SNP (rs80227144, MAF = 2.9%), are of low-frequency (MAF = 2.2–4.5%) in the European population but monomorphic in Asians, which plausibly explains why association was detected in neither Asian study. None of these 13 SNPs is in strong LD with the 2 SNPs implicated in the original European GWAS (<em>r</em><sup>2</sup><sub>rs11766001 </sub>&lt;<sub> </sub>0.67 and <em>r</em><sup>2</sup><sub>rs12707682 </sub>&lt;<sub> </sub>0.42). In the European data set, the lead SNP has a strong effect on disease risk (rs80227144, OR = 5.2, <em>P</em><sub>European </sub>=<sub> </sub>4.7 × 10<sup>−10</sup>; <a href="/pmc/articles/PMC6078638/table/ddw333-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw333-T1" rid-ob="ob-ddw333-T1" co-legend-rid=""><span>Table 1</span></a>) but it is only weakly linked to the reported HSCR-associated SNPs (<em>r</em><sup>2</sup><sub>rs11766001 </sub>=<sub> </sub>0.25 and <em>r</em><sup>2</sup><sub>rs12707682 </sub>=<sub> </sub>0.16). Nevertheless, according to the 1000 Genomes Project, its low frequency risk allele (A) is predominately in phase with the common risk alleles (C) of both rs11766001 (<em>D</em>′ = 0.97) and rs12707682 (<em>D</em>′ = 1). Moderate association was detected for rs80227144 (OR = 4.8, <em>P</em><sub>European </sub>=<sub> </sub>1.5 × 10<sup>−5</sup>) when conditioning on both rs11766001 and rs12707682. On the other hand, association of both reported SNPs was completely attenuated once accounting for the effect of rs80227144 (rs11766001, <em>P</em><sub>European </sub>=<sub> </sub>0.66; rs12707682, <em>P</em><sub>European </sub>=<sub> </sub>0.08). This implies that the previous finding might reflect an indirect association because of the strong effect of the low frequency rs80227144 variant. Interestingly, the genetic variation of rs80227144 alters a regulatory motif of <em>SOX7</em> and is reported as a binding site for <em>GATA-6</em>, where both are transcription factors important for cardiac development (<a href="#ddw333-B16" rid="ddw333-B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596619">16–18</a>). <em>GATA-6</em> is involved in semaphorin-plexin signaling and, in particular, may regulate the expression of semaphorin 3C in cardiac neural crest cells for cardiovascular morphogenesis (<a href="#ddw333-B19" rid="ddw333-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596642">19</a>,<a href="#ddw333-B20" rid="ddw333-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596647">20</a>). This putative long-range regulation might represent the missing link in the association of <em>SEMA3C</em> located ∼4Mb upstream of the credible region and the HSCR risk.</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Independent association of low frequency Asian-specific RET missense variants</h3><p id="__p15" class="p p-first">We also attempted to delineate if there are multiple independent association signals by conditioning on the reported lead SNP(s) in each known HSCR locus. A secondary association signal, corresponding to a low-frequency missense variant encoding RET p.Asp489Asn (rs9282834, conditional OR = 20.3, conditional <em>P</em><sub>Asian </sub>=<sub> </sub>4.1 × 10<sup>−14</sup>; MAF = 0.02 in Asian controls; <a href="/pmc/articles/PMC6078638/table/ddw333-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw333-T2" rid-ob="ob-ddw333-T2" co-legend-rid=""><span>Table 2</span></a>), was discovered from the Asian-specific meta-analysis after adjusting for the effect of the <em>RET</em> enhancer (rs2435357). Further conditioning on rs9282834 in <em>RET</em> or other reported SNPs in <em>NRG1</em> and <em>SEMA3</em> did not reveal additional independent association. This protein-altering HSCR-susceptibility allele (rs9282834 A allele) appears to be Asian-specific and is rarely found in non-Asian populations [MAF &lt; 0.001 according to the ExAC database (<a href="#ddw333-B21" rid="ddw333-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716916976">21</a>)]. Prior to the conditional analysis, the effect of <em>RET</em> p.Asp489Asn was masked as its risk allele (A) is out of phase with the rs2435357 risk allele (T), resulting in an underestimation of the effect sizes of both SNPs if considered individually (<a href="/pmc/articles/PMC6078638/table/ddw333-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw333-T2" rid-ob="ob-ddw333-T2" co-legend-rid=""><span>Table 2</span></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw333-T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p16">Association results for HSCR-associated SNPs in <em>RET</em> before and after conditional analysis on Asian samples</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139663944665360" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="2" rowspan="1">Univariate analysis<hr></th><th colspan="2" rowspan="1">Conditional analysis<hr></th></tr><tr><th rowspan="1" colspan="1">SNP</th><th rowspan="1" colspan="1">Position</th><th rowspan="1" colspan="1">Allele<sup>a</sup></th><th rowspan="1" colspan="1">EAF<sup>b</sup></th><th rowspan="1" colspan="1">OR (95%CI)</th><th rowspan="1" colspan="1"><em>P</em></th><th rowspan="1" colspan="1">OR (95%CI)</th><th rowspan="1" colspan="1"><em>P</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs2435357</td><td rowspan="1" colspan="1">10:43582056</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.82/0.48</td><td rowspan="1" colspan="1">3.54 (2.95–4.24)</td><td rowspan="1" colspan="1">1.36 × 10<sup>−42</sup></td><td rowspan="1" colspan="1">4.01 (3.33–4.84)</td><td rowspan="1" colspan="1">2.98 × 10<sup>−48</sup></td></tr><tr><td rowspan="1" colspan="1">rs9282834 (D489N)</td><td rowspan="1" colspan="1">10:43606856</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.05/0.03</td><td rowspan="1" colspan="1">1.80 (1.06–3.04)</td><td rowspan="1" colspan="1">0.029</td><td rowspan="1" colspan="1">20.3 (9.31–44.4)</td><td rowspan="1" colspan="1">4.09 × 10<sup>−14</sup></td></tr></tbody></table></div><div id="largeobj_idm139663944665360" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6078638/table/ddw333-T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="ddw333-TF3"><p id="__p17" class="p p-first-last"><sup>a</sup>Effect allele/Other allele.</p></div><div id="ddw333-TF4"><p id="__p18" class="p p-first-last"><sup>b</sup>Effect allele frequency of cases/controls.</p></div></div></div><p id="__p19" class="p p-last">While the OR for HSCR risk is 2.3 to 3.2 for carriers of the rs2435357 risk allele and ∼1.1 for carriers of the rs9282834 risk allele, the risk to HSCR dramatically increased 10-fold (OR = 16.7/26.7 in Chinese/Korean, respectively) in individuals compound heterozygous for both rs9282834 and rs2435357 risk alleles (<a href="/pmc/articles/PMC6078638/figure/ddw333-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="ddw333-F2" rid-ob="ob-ddw333-F2" co-legend-rid="lgnd_ddw333-F2"><span>Fig. 2</span></a>). Although rs9282834 is predicted to be non-damaging, this missense variant might have a synergistic effect with rs2435357 in gene regulation, possibly by disrupting the functionality of both copies of <em>RET</em>.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="ddw333-F2" co-legend-rid="lgnd_ddw333-F2"><a href="/pmc/articles/PMC6078638/figure/ddw333-F2/" target="figure" rid-figpopup="ddw333-F2" rid-ob="ob-ddw333-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139663948266032" class="figure"><a href="/pmc/articles/PMC6078638/figure/ddw333-F2/" target="figure" rid-figpopup="ddw333-F2" rid-ob="ob-ddw333-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6078638_ddw333f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddw333f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6078638/bin/ddw333f2.jpg"></a></div><div id="largeobj_idm139663948266032" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6078638/figure/ddw333-F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_ddw333-F2"><div><a class="figpopup" href="/pmc/articles/PMC6078638/figure/ddw333-F2/" target="figure" rid-figpopup="ddw333-F2" rid-ob="ob-ddw333-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><p id="__p20">Epistatic effect of the two <em>RET</em> SNPs independently associated with HSCR (rs9282834 and rs2435357) in (A) Korean and (B) Chinese GWAS. Information [proportion and odds ratio (OR)] regarding each genotype combination [rs9282834 (row) × rs2435357 (column)] is represented by cells. In each cell, left bar denotes the proportion of cases with the specified genotype combination among all cases while right bar denotes the same for controls. OR is defined as the odds of being HSCR with the combination relative to the baseline double homozygote carrying neither risk allele (i.e. GG for rs9282834 and CC for rs2435357).</p></div></div></div></div><div id="__sec5" class="sec"><h3 id="__sec5title">Characterizing the genetic basis of HSCR</h3><p id="__p21" class="p p-first-last">Apart from these known loci, no novel genome-wide significant association was detected under the additive model, raising the questions of whether a large proportion of disease risk can still be attributed to common variants and whether an increase in sample size can still discover more novel HSCR-associated loci. To address these questions, we sought to understand the underlying genetic basis of HSCR by evaluating the global contribution of common variants (MAF &gt; 5%) in disease risk using genome-wide complex trait analysis (GCTA) (<a href="#ddw333-B22" rid="ddw333-B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596603">22</a>). GCTA estimates narrow-sense heritability (ratio of additive genetic variance to total phenotypic variance) on the liability scale by computing the proportion of phenotypic variance captured by all genotyped markers in GWAS simultaneously (<a href="#ddw333-B23" rid="ddw333-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596602">23</a>). While the European GWAS is heterogeneous in terms of the genotyping array and the coverage is relatively low for majority of samples, we focused on the two Asian GWAS for the heritability estimation. Using the common SNPs typed in the two Asian GWAS, we estimated that common variants accounted for 0.34 (standard error (s.e.) = 0.11, <em>P </em>=<em> </em>3.0 × 10<sup>−3</sup>; <a href="/pmc/articles/PMC6078638/table/ddw333-T3/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw333-T3" rid-ob="ob-ddw333-T3" co-legend-rid=""><span>Table 3</span></a>) of variance in risk of HSCR. Compared with the low heritability (∼5.7–6.2%) associated with the three known HSCR-susceptibility variants and the high heritability (&gt;80%) estimated from family studies, the moderate total genetic variance explained implied (i) a non-negligible ‘hidden heritability’ captured by SNPs at other loci on the SNP genotyping arrays and, more importantly, (ii) a significant ‘missing heritability’ unexplained by the additive effects of all typed common variants (<a href="#ddw333-B24" rid="ddw333-B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596635">24</a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw333-T3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p22">Heritability estimates for the two Asian GWAS on HSCR</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139663944859872" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Study</th><th rowspan="1" colspan="1">Cases</th><th rowspan="1" colspan="1">Controls</th><th rowspan="1" colspan="1">Heritability Estimate (s.e.)</th><th rowspan="1" colspan="1"><em>P</em><sup>a</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Chinese</td><td rowspan="1" colspan="1">165</td><td rowspan="1" colspan="1">427</td><td rowspan="1" colspan="1">0.330 (0.138)</td><td rowspan="1" colspan="1">5.79 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1">Korean</td><td rowspan="1" colspan="1">121</td><td rowspan="1" colspan="1">366</td><td rowspan="1" colspan="1">0.369 (0.208)</td><td rowspan="1" colspan="1">7.11 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1">Combined</td><td rowspan="1" colspan="1">286</td><td rowspan="1" colspan="1">793</td><td rowspan="1" colspan="1">0.342 (0.115)</td><td rowspan="1" colspan="1">2.95 × 10<sup>−3</sup></td></tr></tbody></table></div><div id="largeobj_idm139663944859872" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC6078638/table/ddw333-T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="ddw333-TF5"><p id="__p23" class="p p-first-last"><sup>a</sup><em>P</em> value for likelihood ratio test against null hypothesis of zero heritability.</p></div></div></div></div><div id="__sec6" class="sec"><h3 id="__sec6title">Hints of the hidden heritability: genes involved in ENS development</h3><p id="__p24" class="p p-first-last">To explore the hidden heritability, we first assessed the association of other HSCR candidate genes (<em>n</em> = 109; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S6</a>) involved in ENS development, whereby their association could be hidden under the genome-wide significance threshold because of the lack of power attributed to the inadequate sample size. As shown in the quantile-quantile plot (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Fig. S5</a>), a significant excess of moderate association (<em>P </em>&lt;<em> </em>0.001) was observed than expected under the null hypothesis of no association (binomial test <em>P </em>=<em> </em>6.5 × 10<sup>−9</sup>), suggesting that some of these candidates might truly associate with risk of HSCR. Among the 109 candidate loci, 2 exhibited moderately strong association with HSCR (log<sub>10</sub>BF &gt; 4 and/or <em>P</em><sub>Asian </sub>&lt;<sub> </sub>1 × 10<sup>−5</sup>) (<a href="#ddw333-B25" rid="ddw333-B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596655">25</a>), which included <em>JAG1</em> of the Notch signaling pathway (rs6032942: log<sub>10</sub>BF = 4.9) (<a href="#ddw333-B26" rid="ddw333-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596638">26</a>) and the ENS-expressed transcriptional factor—<em>PHOX2B</em> (rs6826373: OR = 0.6, <em>P</em><sub>Asian </sub>=<sub> </sub>6.4 × 10<sup>−6</sup>) (<a href="#ddw333-B27" rid="ddw333-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596612">27</a>). While the causal role of <em>PHOX2B</em> rare variants in syndromic HSCR has been well-established, our study further provides further evidence in support of the contribution of <em>PHOX2B</em> common variants to HSCR susceptibility (<a href="#ddw333-B28" rid="ddw333-B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596605">28</a>).</p></div><div id="__sec7" class="sec sec-last"><h3 id="__sec7title">Marginal association at VRK2 under recessive model and other candidate genes</h3><p id="__p25" class="p p-first-last">Given the dose-dependent nature of the major gene, <em>RET</em>, and considering the fact that non-additive genetic variance is not considered in narrow-sense heritability, we hypothesized that the missing heritability may, in part, be explained by dominance effects. By assuming complete dominance, we therefore performed association tests for all imputed variants by fitting both recessive and dominant models. In addition to <em>RET</em>, which showed the strongest association, we detected, under the recessive model, a novel association at 2p16.1 (rs4672229, recessive log<sub>10</sub>BF = 6.2 compared with additive log<sub>10</sub>BF = 4.9; <a href="/pmc/articles/PMC6078638/table/ddw333-T4/" target="table" class="fig-table-link figpopup" rid-figpopup="ddw333-T4" rid-ob="ob-ddw333-T4" co-legend-rid=""><span>Table 4</span></a>) mapping to an intronic region of the vaccinia-related kinase 2 gene (<em>VRK2</em>). This lead SNP displayed a heterogeneous effect across ancestries, where the recessive association mainly driven by Asians (OR = 2.5, <em>P</em><sub>Asian </sub>=<sub> </sub>2.5×10<sup>−8</sup> in Asian-specific meta-analysis) was not observed in Europeans (OR = 0.8, <em>P</em><sub>European </sub>=<sub> </sub>0.24). Taking into account that we tested 3 different models of inheritance, the Asian-specific association at <em>VRK2</em> did not surpass the stringent Bonferroni correction of multiple testing and, consequently, a follow-up study is required to validate this finding. Although rs4672229 is located in intron 2 of <em>VRK2</em>, it has been reported as an eQTL in tibial nerves that increases the expression of a nearby gene, Fanconi-anemia complementation L polypeptide (<em>FANCL</em>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="ddw333-T4"><h3>Table 4.</h3><!--caption a7--><div class="caption"><p id="__p26">Marginal association for <em>VRK2</em> under recessive model</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139663901282608" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"><em>Trans</em>-ethnic meta-analysis</th><th colspan="3" rowspan="1">European GWAS<hr></th><th colspan="3" rowspan="1">Asian-specific meta-analysis<hr></th></tr><tr><th rowspan="1" colspan="1">Locus</th><th rowspan="1" colspan="1">Lead SNP</th><th rowspan="1" colspan="1">Position</th><th rowspan="1" colspan="1">Genotype<sup>a</sup></th><th rowspan="1" colspan="1">LogBF</th><th rowspan="1" colspan="1">EGF<sup>b</sup></th><th rowspan="1" colspan="1">OR (95% CI)</th><th rowspan="1" colspan="1"><em>P</em></th><th rowspan="1" colspan="1">EGF</th><th rowspan="1" colspan="1">OR (95% CI)</th><th rowspan="1" colspan="1"><em>P</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>VRK2</em></td><td rowspan="1" colspan="1">rs4672229</td><td rowspan="1" colspan="1">2:58276280</td><td rowspan="1" colspan="1">AA/AC + CC</td><td rowspan="1" colspan="1">6.20</td><td rowspan="1" colspan="1">0.21/0.26</td><td rowspan="1" colspan="1">0.75 (0.46–1.21)</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">0.36/0.19</td><td rowspan="1" colspan="1">2.48 (1.80–3.41)</td><td rowspan="1" colspan="1">2.52 × 10<sup>−8</sup></td></tr></tbody></table></div><div id="largeobj_idm139663901282608" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6078638/table/ddw333-T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="ddw333-TF6"><p id="__p27" class="p p-first-last"><sup>a</sup>Effect genotype under recessive model/other genotypes.</p></div><div id="ddw333-TF7"><p id="__p28" class="p p-first-last"><sup>b</sup>Effect genotype frequency.</p></div></div></div></div></div><div id="__sec8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec8title">Discussion</h2><p id="__p29" class="p p-first">Here, we present the largest genetic study and the first GWAS meta-analysis of HSCR, aggregating association results of all published GWAS on this disorder. In addition to reconfirming <em>RET</em> as the major gene for HSCR, we fine-mapped the three known loci through <em>trans</em>-ethnic meta-analysis and narrowed down the credible sets of SNPs that might be causal to HSCR. Moreover, we vindicated the important role of <em>NRG1</em>, wherein <em>NRG1</em> is the only locus other than <em>RET</em> having common variants robustly associated with HSCR across populations. On the other hand, the association of the <em>SEMA3</em> locus appeared to be European-specific and most likely driven by low-frequency regulatory variants that are monomorphic in Asians. By conditional analysis, we further detected another ancestry-specific association which corresponds to an Asian-specific secondary signal by a low-frequency, missense variant, encoding RET p.Asp489Asn and whose effect is independent of that of the reported common intron 1 <em>RET</em> enhancer.</p><p id="__p30">Like for other complex diseases or traits, <em>trans</em>-ethnic meta-analysis for HSCR did not reveal heterogeneous effects of common variants across populations (<a href="#ddw333-B29" rid="ddw333-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596610">29</a>), yet the two low-frequency variants found associated with HSCR are nearly ancestry-specific. This could be explained by the fact that rare variants tend to be more recent and may emerge after the divergence of different populations (<a href="#ddw333-B30" rid="ddw333-B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596613">30</a>). As a result, increasing sample size involving multiple ancestries by <em>trans</em>-ethnic meta-analysis might not facilitate the discovery of low-frequency variants of larger effect, where their association could be more readily detected by ancestry/population-specific meta-analysis. Indeed, the extent to which <em>trans</em>-ethnic meta-analysis can yield novel associations depends on the underlying genetic architecture of the disease. For HSCR, we estimated that a considerable amount of ‘hidden’ as well as ‘missing heritability’ exists. Proportion of phenotypic variance explained by common SNPs jointly was significantly greater than that of the known disease-susceptibility variants, but did not account for the totality of heritability inferred from family studies. Some of the genetic variance could be simply hidden below the threshold of genome-wide significance, hinting at unraveling more HSCR-associated common variants, for example, of genes involved in ENS development, by further increasing the sample size.</p><p id="__p31">Given our finding here of the two low-frequency HSCR-associated variants, together with reports of causative mutations in known genes and a global burden of copy number variants (CNVs) in HSCR cases, contributions of low-frequency, rare, or even structural variants can be the potential sources of ‘missing heritability’ in HSCR (<a href="#ddw333-B8" rid="ddw333-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596618">8</a>,<a href="#ddw333-B11" rid="ddw333-B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596644">11</a>,<a href="#ddw333-B31" rid="ddw333-B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596648">31</a>). While our GWAS employed old genotyping platforms, we acknowledge that the limitations on calling CNVs and imputing rare variants made them sub-optimal to evaluate the overall contribution of these types of variations to heritability. Genetic studies using more recent technologies (e.g. the latest SNP chips and/or next generation sequencing) and family-based study designs (<a href="#ddw333-B32" rid="ddw333-B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596639">32</a>) (e.g. parent-offspring trios) might resolve the mystery of their genetic contribution. Another prevailing hypothesis is that part of the ‘missing heritability’ is attributable to non-additive genetic variance, which includes interaction/epistatic (allelic interaction between loci) and dominance effects (allelic interaction within a locus) (<a href="#ddw333-B33" rid="ddw333-B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596657">33</a>,<a href="#ddw333-B34" rid="ddw333-B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596632">34</a>). In fact, the major gene—<em>RET</em>—serves as the best example for both types of non-additive contributions. Firstly, many studies have suggested the genetic interplay between variants in <em>RET</em> and other HSCR-associated loci. For instance, <em>NRG1</em> common variants have stronger effects on disease risk in the presence of the <em>RET</em> high-risk genotype (<a href="#ddw333-B7" rid="ddw333-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596645">7</a>,<a href="#ddw333-B35" rid="ddw333-B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596624">35</a>). On the other hand, chromosome 21 gene dosage, in which comorbidity of Down syndrome and HSCR is often observed, acts on a genetic background with a smaller contribution of <em>RET</em> intron 1 enhancer (<a href="#ddw333-B36" rid="ddw333-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596637">36</a>). Secondly, for the low frequency and common variants, <em>RET</em> appears to follow a recessive model either through homozygosity at the intron 1 enhancer or compound heterozygosity with the missense variant, rs9282834. Thus ‘recessive’ inheritance of gene disrupting alleles at one site (recessive inheritance), or at multiple sites of the same gene (compound heterozygous inheritance) or across multiple genes (oligogenic inheritance) may underlie the non-additive genetic contributions to HSCR. The existence of such non-additive genetic components may lead to confounded estimates of both heritability, the one estimated through family data and the one estimated through genetic variants (<a href="#ddw333-B37" rid="ddw333-B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596651">37</a>). Only until recently, methods to evaluate the relative importance of these non-additive variations in case–control studies were proposed (<a href="#ddw333-B34" rid="ddw333-B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596652">34</a>); however, these methods typically require a large sample size for partitioning into variance components or for establishing <em>trans</em>-ethnic genetic correlations and are therefore not feasible in our circumstances given the small sample size. Leveraging power by further increasing sample size would allow us to fully decipher the genetic architecture of HSCR as well as comparing and contrasting the <em>cis</em>- versus <em>trans</em>-haplotype effect of linked variants. Last but not least, we cannot negate the possibility of overestimation of heritability in the original family studies. A revisit of the epidemiology of HSCR, involving more HSCR twins or families, is warranted.</p><p id="__p32">Perhaps one of the most interesting candidates with non-additive effect is the <em>VRK2</em> locus, whose variants show a marginal association under a recessive model in the <em>trans</em>-ethnic meta-analysis. <em>VRK2</em> encodes a serine/threonine protein kinase implicated in neurological function. Variants upstream of <em>VRK2</em> were reported as one of the 108 loci significantly associated with schizophrenia (<a href="#ddw333-B38" rid="ddw333-B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596625">38</a>). Nearby variants in LD also showed moderate association with genetic generalized epilepsies (<a href="#ddw333-B39" rid="ddw333-B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596649">39</a>). In addition, <em>VRK2</em> together with the downstream gene, <em>FANCL</em>, falls within a common deletion region responsible for the 2p16.1-p15 microdeletion syndrome which is characterized by intellectual disability, microcephaly, and autistic behavior (OMIM:612513) (<a href="#ddw333-B40" rid="ddw333-B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596622">40</a>,<a href="#ddw333-B41" rid="ddw333-B41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596616">41</a>). Despite multiple lines of evidence supporting a role of <em>VRK2</em> in central nervous system and neurodevelopmental disorders, its involvement in ENS development remains poorly understood. One possible clue linking <em>VRK2</em> to HSCR may lie in its interaction with the co-receptor of NRG1, Erb-B2 Receptor Tyrosine Kinase 2 (ErbB2). A recent study of Fernandez <em>et al.</em> (<a href="#ddw333-B42" rid="ddw333-B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596623">42</a>) showed that, in breast cancer cells, high level of VRK2 inhibited ErbB2 activation of transcription, possibly by blocking phosphorylation of extracellular signal-regulated kinase (ERK) in response to ErbB2 stimulation. On the contrary, knockdown of <em>VRK2</em> increased the transcriptional response to stimulation. This inverse correlation between VRK2 and ErbB2 expression levels in breast carcinomas suggested a regulatory role of <em>VRK2</em> in neuregulin signaling, which may represent the missing link underlying the association of <em>VRK2</em> and HSCR.</p><p id="__p33" class="p p-last">With the recent progress of derivation and isolation of ENS progenitor cells, novel therapeutic opportunities, e.g. autologous transplantation of these ENS precursors to repopulate the colon and restore the peristalsis function, might arise as an alternative therapy in the future (<a href="#ddw333-B43" rid="ddw333-B43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596600">43</a>,<a href="#ddw333-B44" rid="ddw333-B44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596650">44</a>); however, the migration and differentiation abilities of these patient-derived precursors are often affected by the genetic background of the patients themselves. It is therefore of utmost importance to apprehend the pathophysiology contributing to, in general, HSCR and, specifically, to each individual in order to develop a patient-specific therapy. Overall, our study not only implicates putative HSCR-associated loci that might help modulate enteric neural crest cell properties, but also provides potential sources of the ‘hidden’ and ‘missing’ heritability. Understanding the genetic architecture of HSCR—the relative contribution of common versus rare variants and additive versus non-additive effects—may have profound implications on future study design.</p></div><div id="__sec9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec9title">Materials and Methods</h2><div id="__sec10" class="sec sec-first"><h3 id="__sec10title">Study subjects and quality controls</h3><p id="__p34" class="p p-first">The <em>trans</em>-ethnic meta-analysis comprised 507 HSCR cases and 1191 controls from 3 published GWAS on European (<a href="#ddw333-B8" rid="ddw333-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596641">8</a>), Chinese (<a href="#ddw333-B7" rid="ddw333-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596629">7</a>) and Korean (<a href="#ddw333-B9" rid="ddw333-B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596599">9</a>) populations. The phenotypic and demographic information for each study is summarized in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S1</a>. On top of the quality controls imposed by the original studies, individuals with low call rate (&lt;90% for Affymetrix 500K or &lt;98% for Illumina 1M arrays) or being an outlier in autosomal heterozygosity (4 SD from the mean) were further removed.</p><div id="__sec11" class="sec"><p></p><h4 id="__sec11title" class="inline">European </h4><p id="__p35" class="p p-first-last">Family-based GWAS undertaken by the International HSCR Disease Consortium was carried out on 220 trios (probands together with unaffected parents) of European descent, which included subjects from USA, Italy, The Netherlands, Spain and France (<a href="#ddw333-B8" rid="ddw333-B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596656">8</a>). Genotyping was performed on either or both of the 250 K arrays (Nsp and Sty) of the Affymetrix GeneChip Human Mapping 500 K array. To convert into case/pseudo-control design, we constructed pseudo-controls from the non-transmitted alleles of parents from each trio. While the resulting pseudo-controls are genetically unrelated to the proband, subsequent association analysis is equivalent to a matched case/control analysis. After quality control, a total of 25 samples genotyped on the Nsp arrays and 23 samples genotyped on the Sty arrays were dropped. The final dataset consisted of 179 cases and 172 controls genotyped on either array—Nsp (130&nbsp;873 SNPs) or Sty (115&nbsp;285 SNPs)—as well as 33 cases and 30 controls typed on both arrays (246&nbsp;158 SNPs) of the 500 K array were used.</p></div><div id="__sec12" class="sec"><p></p><h4 id="__sec12title" class="inline">Chinese </h4><p id="__p36" class="p p-first-last">The Chinese GWAS was originally performed on 181 sporadic cases and 346 controls of matched ancestry (<a href="#ddw333-B7" rid="ddw333-B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596634">7</a>) using Affymetrix 500K, which assays 493&nbsp;840 SNPs across the genome. We later included an additional set of 271 controls, of which 181 were genotyped by another array—Affymetrix Genome-Wide Human SNP Array 5.0 (<a href="#ddw333-B45" rid="ddw333-B45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596636">45</a>)—which includes all the SNPs of the original 500K array but differs by inclusion of extra non-polymorphic probes for copy number analysis. More stringent quality controls as described previously were applied on both individual and variant levels (<a href="#ddw333-B46" rid="ddw333-B46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596627">46</a>), resulting in a final set of 320&nbsp;613 SNPs for 173 cases and 615 controls included in the meta-analysis.</p></div><div id="__sec13" class="sec sec-last"><p></p><h4 id="__sec13title" class="inline">Korean </h4><p id="__p37" class="p p-first-last">The original Korean GWAS consisted of 123 sporadic HSCR cases and 432 controls (<a href="#ddw333-B9" rid="ddw333-B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596604">9</a>). Relatedness in controls was initially modeled and corrected by mixed model association in the original GWAS. To maintain consistency in statistical analysis with the other two studies, these related controls were removed, resulting in 714&nbsp;024 SNPs of 122 cases and 374 controls for meta-analysis.</p></div></div><div id="__sec14" class="sec"><h3 id="__sec14title">Genotype Imputation</h3><p id="__p38" class="p p-first-last">For each data set, individual genotypes first underwent pre-phasing using SHAPEIT (v2) to infer the underlying haplotypes (<a href="#ddw333-B47" rid="ddw333-B47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596640">47</a>). Whenever the parental genotypes were available, they were included into the pre-phasing using the SHAPEIT—duohmm module to improve accuracy but these genotypes were later excluded from association analysis. Imputation was then performed using IMPUTE2 with reference to the 1000 Genome Project multi-ancestry reference panel (Phase I Integrated, June 2014 haplotypes) (<a href="#ddw333-B48" rid="ddw333-B48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596601">48</a>,<a href="#ddw333-B49" rid="ddw333-B49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596628">49</a>). The reference panel contains 2184 haplotypes of 1092 samples, with ∼38 million (M) bi-allelic polymorphic but non-singleton markers (∼36.8M SNPs, 1.4 M short bi-allelic insertions/deletions and 14&nbsp;017 structural variations). Singleton variants—variants appearing only once in the whole data set—were not included in the imputation reference panel as the imputation quality is generally lower and we lack power to detect association for such low frequency variants.</p></div><div id="__sec15" class="sec"><h3 id="__sec15title">Association Tests</h3><p id="__p39" class="p p-first-last">Association analyses of all three imputed data sets were carried out using score tests (option-method <em>score</em> in SNPTEST v2) for three genetic models, (<a href="#ddw333-B1" rid="ddw333-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596606">1</a>) additive, (<a href="#ddw333-B2" rid="ddw333-B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596609">2</a>) recessive and (<a href="#ddw333-B3" rid="ddw333-B3" class=" bibr popnode">3</a>) dominant, based on the imputed genotype probabilities after adjustment by the study-specific covariates. For the Chinese GWAS, two principal components were used as covariates to control for population stratification and batch effect simultaneously, whereas, for the European GWAS, two covariates specifying the chip (Nsp only, Sty only and both Nsp and Sty) where genotyping was performed were included. No covariate was used for Korean data set as no population substructure was observed. Variants with low imputation quality (INFO ≤0.6), low population frequency (MAF ≤2%) or violating Hardy–Weinberg equilibrium (<em>P </em>&lt;<em> </em>10<sup>−4</sup>) were excluded (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Table S2</a>). The test statistics, as visualized in a quantile-quantile plot, appeared well calibrated (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material, Fig. S1</a>). To further delineate SNPs with independent association, we performed stepwise conditional analyses by SNPTEST using the reported lead SNP for each known HSCR locus as covariate (<em>RET</em>: rs2435357, <em>NRG1</em>: rs7835688 and SEMA3: rs11766001 and rs12707682). While the <em>SEMA3</em> association is European-specific and one of the reported lead SNP (rs11766001) is monomorphic in East Asians, conditional analysis on <em>SEMA3</em> was performed only in the European dataset. For <em>RET</em> and <em>NRG1</em> loci, SNPs with conditional test statistics reaching genome-wide significance (<em>P </em>&lt;<em> </em>5 × 10<sup>−8</sup> in the Asian/European-specific or log<sub>10</sub>BF &gt; 6.1 in the <em>trans</em>-ethnic meta-analyses) were considered as having independent association and the lead SNPs showing strongest evidence of independent association was later included in the regression model as covariate. The process was repeated until no variant reached genome-wide significance after adjustment.</p></div><div id="__sec16" class="sec"><h3 id="__sec16title">Asian-Specific and Trans-Ethnic Meta-Analyses</h3><div id="__sec17" class="sec sec-first"><p></p><h4 id="__sec17title" class="inline">Asian-specific meta-analysis </h4><p id="__p40" class="p p-first-last">Ethnic-specific meta-analysis was performed for East Asians only in METAL (<a href="#ddw333-B50" rid="ddw333-B50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596597">50</a>) under a fixed-effects model. Association summary statistics (effect size estimates and their standard error) of the Chinese and Korean GWAS were meta-analyzed by the inverse-variance approaches and Cochran's <em>Q</em> statistic was used to estimate the between-study heterogeneity.</p></div><div id="__sec18" class="sec sec-last"><p></p><h4 id="__sec18title" class="inline">Trans<em>-</em>ethnic meta-analysis </h4><p id="__p41" class="p p-first-last">We meta-analyzed the association results of all ancestry groups using MANTRA software (<a href="#ddw333-B14" rid="ddw333-B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596631">14</a>), which takes into account the expected similarity in allelic effects between closely related populations when compared with more diverse ethnic groups. Briefly, the estimated effect sizes, their standard errors and the allele frequencies of each study are used to model the relatedness between studies/populations, assign each population to ‘cluster’ according to the Bayesian partition model, and evaluate the statistical significance. We only considered variants tested in at least two GWAS, except for fine-mapping known loci. Evidence in favor of association was assessed by means of Bayes factor (BF), which represents the probability under the alternative hypothesis that the SNP is associated with the disease divided by the probability under the null hypothesis of no association. The larger the value of BF, the stronger is the evidence of association. An association with a log<sub>10</sub>BF of 6.1 or above is considered as genome-wide significant (comparable to GWAS significant threshold <em>P </em>&lt;<em> </em>5 × 10<sup>−8</sup> in the frequentist test) (<a href="#ddw333-B25" rid="ddw333-B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596630">25</a>). A full set of summary statistics of the meta-analysis can be obtained from <a href="http://www.surgery.hku.hk/trs/index.php/research-output/#" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.surgery.hku.hk/trs/index.php/research-output/#</a>transethnicMeta2016.</p></div></div><div id="__sec19" class="sec"><h3 id="__sec19title">Defining Credible Sets for Fine-Mapping Known HSCR Loci</h3><p id="__p42" class="p p-first-last">To fine-map the three known HSCR loci, we constructed the 99% credible sets which represent the minimum set of variants accounting for 99% of posterior probabilities in the region. For these loci with prior evidence of association with HSCR, we considered all variants tested for association in any GWAS. Assuming a single causal variant for each locus and that the true causal variant is either genotyped or well imputed, the probability that the 99% credible set contains the causal variant would be ∼0.99. For each locus, we first calculated the posterior probabilities for each variant located ±1 Mb from the top variant (variant showing the lowest <em>P</em>-value in the region), i.e. the corresponding BF divided by the summation of BF over all variants in the region. Variants were then ranked according to their BF and the ranked variants were combined to form the credible set until their cumulative posterior probability attains or exceeds 99%. For visualization purposes, the most distant SNPs within credible sets were defined as boundaries of the set.</p></div><div id="__sec20" class="sec"><h3 id="__sec20title">Functional Annotation</h3><p id="__p43" class="p p-first-last">We annotated the credible set variants for regulatory potentials against RegulomeDB (<a href="#ddw333-B51" rid="ddw333-B51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596653">51</a>) and HaploReg (<a href="#ddw333-B52" rid="ddw333-B52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596614">52</a>) databases. More specifically, we explored evidence of transcription factor binding, DNase hypersensitivity as well as promoter and enhancer activities via histone modification signals. Active promoter is marked by signature of H3K4me3, whereas H3K27ac and H3K4me1 define regions with enhancer activity. In addition, GTEx (<a href="#ddw333-B53" rid="ddw333-B53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596611">53</a>) resources were used to delineate if a variant is an eQTL that affects expression of nearby genes.</p></div><div id="__sec21" class="sec"><h3 id="__sec21title">Estimation of Variance Explained by Known HSCR SNPS</h3><p id="__p44" class="p p-first-last">The heritability contributed by known HSCR susceptibility loci was computed based on population-specific prevalence (0.0001 for Europeans and 0.00028 for Asians) together with allele frequencies and odds ratios (ORs) of the three previously reported HSCR-associated variants (<em>RET</em> rs2435357, <em>NRG1</em> rs7835688 and <em>SEMA3</em> rs12707682) based on the So <em>et al.</em> approach (<a href="#ddw333-B54" rid="ddw333-B54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_716596617">54</a>).</p></div><div id="__sec22" class="sec sec-last"><h3 id="__sec22title">Estimation of Variance Explained by GWAS</h3><p id="__p45" class="p p-first-last">To evaluate the narrow-sense heritability attributed to common variants (MAF &gt;5%), we carried out a GCTA analysis on the two Asian population-based GWAS. GCTA estimates heritability by studying the extent to which phenotypic variation can be explained by the genotypic similarity across individuals. The genetic relatedness estimated in terms of the genetic relationship matrix approximates to the coefficient of relationship (two times the kinship coefficient) obtained from pedigrees. For each study, cryptic relatedness was first removed by restricting the genetic relatedness to be &lt;0.025, a coefficient that approximates to fifth-degree relatives or less. A total of 17 Chinese and 9 Korean individuals were removed in this manner. Proportions of variance in disease liability captured by all genotyped SNPs were estimated by a mixed linear model, using a restricted maximum likelihood approach, while assuming the disease prevalence to be 0.0002. For the Chinese GWAS, only samples genotyped by the Affymetrix 500K platform were included to avoid spurious results. Also, two principal components were used as covariates to account for possible confounding effects because of population substructure. GCTA estimates from the two Asian studies were then meta-analyzed under a fixed-effects model to provide a more robust heritability estimate. Evidence for heterogeneity was assessed using Cochran’s <em>Q</em> test.</p></div></div><div id="__sec23" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec23title">Supplementary Material</h2><!--/article/body/sec/--><p id="__p46" class="p p-first-last">
<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Material</a> is available at <em>HMG</em> online.</p></div><div id="__sec24" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec24title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="sup1"><h4>Supplementary Data</h4><div class="sup-box half_rhythm" id="idm139663943832256"><a href="/pmc/articles/PMC6078638/bin/ddw333_supp.zip" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(854K, zip)</sup></div></div></div><div id="idm139663943830656" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139663943830656title">Acknowledgements</h2><div class="sec"><p id="__p48">We are grateful to the numerous patients, their families and referring physicians that have participated in these studies in our laboratories, and the numerous members of our laboratories for their valuable contributions over many years.</p><p id="__p49">
<em>Conflict of Interest statement</em>. None declared.</p><div id="__sec25" class="sec"><h3 id="__sec25title">Funding</h3><p id="__p50" class="p p-first-last">This work was supported by Health and Medical Research Fund [grant number 01121796 to C.S.T]; General Research Fund [grant number HKU777612 to P.T.]; Theme-based Research scheme [grant number T12C-714/14-R to P.T.]; Agence Nationale de la Recherche [grant number ANR-10-IAHU-01]; National Research Foundation of Korea funded by the Ministry of Education, Science and Technology [grant number 2010-0007857 and 2009-0093822]; National Institutes of Health [grant numbers HD28088 (Merit Award), R37HD28088]; Italian Ministry of Health (‘Cinque per mille’ and Ricerca Corrente to the Gaslini Institute); the Institute of Health Carlos III (ISCIII), Spanish Ministry of Economy and Competitiveness, Spain [grant number PI13/01560]; Andalusian Regional Ministry of Innovation, Science and Enterprise [grant number CTS-7447]; ZonMW [grant number TOP-subsidie 40-00812-98-10042 to R.M.W.H]; and the Maag Lever Darm stichting [grant number WO09-62 to R.M.W.H].</p></div></div></div><div id="idm139663909698880" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139663909698880title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="ddw333-B1">1. <span class="mixed-citation">
Amiel J., Sproat-Emison E., Garcia-Barcelo M., Lantieri F., Burzynski G., Borrego S., Pelet A., Arnold S., Miao X., Griseri P., et al. (2008) 
<span class="ref-title">Hirschsprung disease, associated syndromes and genetics: a review</span>. <span class="ref-journal">J. Med. Genet</span>., <span class="ref-vol">45</span>, 1–14. [<a href="/pubmed/17965226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Genet&amp;title=Hirschsprung+disease,+associated+syndromes+and+genetics:+a+review&amp;author=J.+Amiel&amp;author=E.+Sproat-Emison&amp;author=M.+Garcia-Barcelo&amp;author=F.+Lantieri&amp;author=G.+Burzynski&amp;volume=45&amp;publication_year=2008&amp;pages=1-14&amp;pmid=17965226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B2">2. <span class="mixed-citation">
Tam P.K., Garcia-Barcelo M. (2009) 
<span class="ref-title">Genetic basis of Hirschsprung's disease</span>. <span class="ref-journal">Pediatr. Surg. Int</span>., <span class="ref-vol">25</span>, 543–558. [<a href="/pubmed/19521704" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr.+Surg.+Int&amp;title=Genetic+basis+of+Hirschsprung's+disease&amp;author=P.K.+Tam&amp;author=M.+Garcia-Barcelo&amp;volume=25&amp;publication_year=2009&amp;pages=543-558&amp;pmid=19521704&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B3">3. <span class="mixed-citation">
Chakravarti A., McCallion A.S., Lyonnet S. (2014) In Beaudet A.L., Vogelstein B., Kinzler K.W., Antonarakis S.E., Ballabio A., Gibson K.M., Mitchell G., editors.  (eds), <span class="ref-journal">The Online Metabolic and Molecular Bases of Inherited Disease</span>. 
The McGraw-Hill Companies, Inc, 
New York, NY. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Online+Metabolic+and+Molecular+Bases+of+Inherited+Disease&amp;author=A.+Chakravarti&amp;author=A.S.+McCallion&amp;author=S.+Lyonnet&amp;publication_year=2014&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B4">4. <span class="mixed-citation">
Badner J.A., Sieber W.K., Garver K.L., Chakravarti A. (1990) 
<span class="ref-title">A genetic study of Hirschsprung disease</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">46</span>, 568–580. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1683643/">PMC free article</a>]</span> [<a href="/pubmed/2309705" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+genetic+study+of+Hirschsprung+disease&amp;author=J.A.+Badner&amp;author=W.K.+Sieber&amp;author=K.L.+Garver&amp;author=A.+Chakravarti&amp;volume=46&amp;publication_year=1990&amp;pages=568-580&amp;pmid=2309705&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B5">5. <span class="mixed-citation">
Alves M.M., Sribudiani Y., Brouwer R.W., Amiel J., Antinolo G., Borrego S., Ceccherini I., Chakravarti A., Fernandez R.M., Garcia-Barcelo M.M., et al. (2013) 
<span class="ref-title">Contribution of rare and common variants determine complex diseases-Hirschsprung disease as a model</span>. <span class="ref-journal">Dev. Biol</span>, <span class="ref-vol">382</span>, 320–329. [<a href="/pubmed/23707863" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Biol&amp;title=Contribution+of+rare+and+common+variants+determine+complex+diseases-Hirschsprung+disease+as+a+model&amp;author=M.M.+Alves&amp;author=Y.+Sribudiani&amp;author=R.W.+Brouwer&amp;author=J.+Amiel&amp;author=G.+Antinolo&amp;volume=382&amp;publication_year=2013&amp;pages=320-329&amp;pmid=23707863&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B6">6. <span class="mixed-citation">
Emison E.S., McCallion A.S., Kashuk C.S., Bush R.T., Grice E., Lin S., Portnoy M.E., Cutler D.J., Green E.D., Chakravarti A. (2005) 
<span class="ref-title">A common sex-dependent mutation in a RET enhancer underlies Hirschsprung disease risk</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">434</span>, 857–863. [<a href="/pubmed/15829955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+common+sex-dependent+mutation+in+a+RET+enhancer+underlies+Hirschsprung+disease+risk&amp;author=E.S.+Emison&amp;author=A.S.+McCallion&amp;author=C.S.+Kashuk&amp;author=R.T.+Bush&amp;author=E.+Grice&amp;volume=434&amp;publication_year=2005&amp;pages=857-863&amp;pmid=15829955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B7">7. <span class="mixed-citation">
Garcia-Barcelo M.M., Tang C.S., Ngan E.S., Lui V.C., Chen Y., So M.T., Leon T.Y., Miao X.P., Shum C.K., Liu F.Q., et al. (2009) 
<span class="ref-title">Genome-wide association study identifies NRG1 as a susceptibility locus for Hirschsprung's disease</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A</span>., <span class="ref-vol">106</span>, 2694–2699. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2650328/">PMC free article</a>]</span> [<a href="/pubmed/19196962" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A&amp;title=Genome-wide+association+study+identifies+NRG1+as+a+susceptibility+locus+for+Hirschsprung's+disease&amp;author=M.M.+Garcia-Barcelo&amp;author=C.S.+Tang&amp;author=E.S.+Ngan&amp;author=V.C.+Lui&amp;author=Y.+Chen&amp;volume=106&amp;publication_year=2009&amp;pages=2694-2699&amp;pmid=19196962&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B8">8. <span class="mixed-citation">
Jiang Q., Arnold S., Heanue T., Kilambi K.P., Doan B., Kapoor A., Ling A.Y., Sosa M.X., Guy M., Jiang Q., et al. (2015) 
<span class="ref-title">Functional loss of semaphorin 3C and/or semaphorin 3D and their epistatic interaction with ret are critical to Hirschsprung disease liability</span>. <span class="ref-journal">Am J Hum. Genet</span>., <span class="ref-vol">96</span>, 581–596. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4385176/">PMC free article</a>]</span> [<a href="/pubmed/25839327" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum.+Genet&amp;title=Functional+loss+of+semaphorin+3C+and/or+semaphorin+3D+and+their+epistatic+interaction+with+ret+are+critical+to+Hirschsprung+disease+liability&amp;author=Q.+Jiang&amp;author=S.+Arnold&amp;author=T.+Heanue&amp;author=K.P.+Kilambi&amp;author=B.+Doan&amp;volume=96&amp;publication_year=2015&amp;pages=581-596&amp;pmid=25839327&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B9">9. <span class="mixed-citation">
Kim J.H., Cheong H.S., Sul J.H., Seo J.M., Kim D.Y., Oh J.T., Park K.W., Kim H.Y., Jung S.M., Jung K., et al. (2014) 
<span class="ref-title">A genome-wide association study identifies potential susceptibility loci for Hirschsprung disease</span>. <span class="ref-journal">PLoS One</span>, <span class="ref-vol">9</span>, e110292.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4195606/">PMC free article</a>]</span> [<a href="/pubmed/25310821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=A+genome-wide+association+study+identifies+potential+susceptibility+loci+for+Hirschsprung+disease&amp;author=J.H.+Kim&amp;author=H.S.+Cheong&amp;author=J.H.+Sul&amp;author=J.M.+Seo&amp;author=D.Y.+Kim&amp;volume=9&amp;publication_year=2014&amp;pages=e110292.&amp;pmid=25310821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B10">10. <span class="mixed-citation">
Paratore C., Goerich D.E., Suter U., Wegner M., Sommer L. (2001) 
<span class="ref-title">Survival and glial fate acquisition of neural crest cells are regulated by an interplay between the transcription factor Sox10 and extrinsic combinatorial signaling</span>. <span class="ref-journal">Development</span>, <span class="ref-vol">128</span>, 3949–3961. [<a href="/pubmed/11641219" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Survival+and+glial+fate+acquisition+of+neural+crest+cells+are+regulated+by+an+interplay+between+the+transcription+factor+Sox10+and+extrinsic+combinatorial+signaling&amp;author=C.+Paratore&amp;author=D.E.+Goerich&amp;author=U.+Suter&amp;author=M.+Wegner&amp;author=L.+Sommer&amp;volume=128&amp;publication_year=2001&amp;pages=3949-3961&amp;pmid=11641219&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B11">11. <span class="mixed-citation">
Tang C.S., Ngan E.S., Tang W.K., So M.T., Cheng G., Miao X.P., Leon T.Y., Leung B.M., Hui K.J., Lui V.H., et al. (2012) 
<span class="ref-title">Mutations in the NRG1 gene are associated with Hirschsprung disease</span>. <span class="ref-journal">Hum. Genet</span>., <span class="ref-vol">131</span>, 67–76. [<a href="/pubmed/21706185" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet&amp;title=Mutations+in+the+NRG1+gene+are+associated+with+Hirschsprung+disease&amp;author=C.S.+Tang&amp;author=E.S.+Ngan&amp;author=W.K.+Tang&amp;author=M.T.+So&amp;author=G.+Cheng&amp;volume=131&amp;publication_year=2012&amp;pages=67-76&amp;pmid=21706185&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B12">12. <span class="mixed-citation">
Luzon-Toro B., Fernandez R.M., Torroglosa A., de Agustin J.C., Mendez-Vidal C., Segura D.I., Antinolo G., Borrego S. (2013) 
<span class="ref-title">Mutational spectrum of semaphorin 3A and semaphorin 3D genes in Spanish Hirschsprung patients</span>. <span class="ref-journal">PLoS One</span>, <span class="ref-vol">8</span>, e54800.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3553056/">PMC free article</a>]</span> [<a href="/pubmed/23372769" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Mutational+spectrum+of+semaphorin+3A+and+semaphorin+3D+genes+in+Spanish+Hirschsprung+patients&amp;author=B.+Luzon-Toro&amp;author=R.M.+Fernandez&amp;author=A.+Torroglosa&amp;author=J.C.+de+Agustin&amp;author=C.+Mendez-Vidal&amp;volume=8&amp;publication_year=2013&amp;pages=e54800.&amp;pmid=23372769&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B13">13. <span class="mixed-citation">
Luzon-Toro B., Torroglosa A., Nunez-Torres R., Enguix-Riego M.V., Fernandez R.M., de Agustin J.C., Antinolo G., Borrego S. (2012) 
<span class="ref-title">Comprehensive analysis of NRG1 common and rare variants in Hirschsprung patients</span>. <span class="ref-journal">PLoS One</span>, <span class="ref-vol">7</span>, e36524.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3344894/">PMC free article</a>]</span> [<a href="/pubmed/22574178" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Comprehensive+analysis+of+NRG1+common+and+rare+variants+in+Hirschsprung+patients&amp;author=B.+Luzon-Toro&amp;author=A.+Torroglosa&amp;author=R.+Nunez-Torres&amp;author=M.V.+Enguix-Riego&amp;author=R.M.+Fernandez&amp;volume=7&amp;publication_year=2012&amp;pages=e36524.&amp;pmid=22574178&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B14">14. <span class="mixed-citation">
Morris A.P. (2011) 
<span class="ref-title">Transethnic meta-analysis of genomewide association studies</span>. <span class="ref-journal">Genet. Epidemiol</span>., <span class="ref-vol">35</span>, 809–822. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3460225/">PMC free article</a>]</span> [<a href="/pubmed/22125221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Transethnic+meta-analysis+of+genomewide+association+studies&amp;author=A.P.+Morris&amp;volume=35&amp;publication_year=2011&amp;pages=809-822&amp;pmid=22125221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B15">15. <span class="mixed-citation">
Wellcome Trust Case Control Consortium, Maller J.B., McVean G., Byrnes J., Vukcevic D., Palin K., Su Z., Howson J.M., Auton A., Myers S., et al. (2012) 
<span class="ref-title">Bayesian refinement of association signals for 14 loci in 3 common diseases</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">44</span>, 1294–1301. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3791416/">PMC free article</a>]</span> [<a href="/pubmed/23104008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Bayesian+refinement+of+association+signals+for+14+loci+in+3+common+diseases&amp;author=J.B.+Maller&amp;author=G.+McVean&amp;author=J.+Byrnes&amp;author=D.+Vukcevic&amp;author=K.+Palin&amp;volume=44&amp;publication_year=2012&amp;pages=1294-1301&amp;pmid=23104008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B16">16. <span class="mixed-citation">
Kamachi Y., Kondoh H. (2013) 
<span class="ref-title">Sox proteins: regulators of cell fate specification and differentiation</span>. <span class="ref-journal">Development</span>, <span class="ref-vol">140</span>, 4129–4144. [<a href="/pubmed/24086078" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Sox+proteins:+regulators+of+cell+fate+specification+and+differentiation&amp;author=Y.+Kamachi&amp;author=H.+Kondoh&amp;volume=140&amp;publication_year=2013&amp;pages=4129-4144&amp;pmid=24086078&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B17">17. <span class="mixed-citation">
Hermkens D.M., van Impel A., Urasaki A., Bussmann J., Duckers H.J., Schulte-Merker S. (2015) 
<span class="ref-title">Sox7 controls arterial specification in conjunction with hey2 and efnb2 function</span>. <span class="ref-journal">Development</span>, <span class="ref-vol">142</span>, 1695–1704. [<a href="/pubmed/25834021" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Sox7+controls+arterial+specification+in+conjunction+with+hey2+and+efnb2+function&amp;author=D.M.+Hermkens&amp;author=A.+van+Impel&amp;author=A.+Urasaki&amp;author=J.+Bussmann&amp;author=H.J.+Duckers&amp;volume=142&amp;publication_year=2015&amp;pages=1695-1704&amp;pmid=25834021&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B18">18. <span class="mixed-citation">
Verzi M.P., Shin H., He H.H., Sulahian R., Meyer C.A., Montgomery R.K., Fleet J.C., Brown M., Liu X.S., Shivdasani R.A. (2010) 
<span class="ref-title">Differentiation-specific histone modifications reveal dynamic chromatin interactions and partners for the intestinal transcription factor CDX2</span>. <span class="ref-journal">Dev. Cell</span>, <span class="ref-vol">19</span>, 713–726. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3001591/">PMC free article</a>]</span> [<a href="/pubmed/21074721" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Cell&amp;title=Differentiation-specific+histone+modifications+reveal+dynamic+chromatin+interactions+and+partners+for+the+intestinal+transcription+factor+CDX2&amp;author=M.P.+Verzi&amp;author=H.+Shin&amp;author=H.H.+He&amp;author=R.+Sulahian&amp;author=C.A.+Meyer&amp;volume=19&amp;publication_year=2010&amp;pages=713-726&amp;pmid=21074721&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B19">19. <span class="mixed-citation">
Lepore J.J., Mericko P.A., Cheng L., Lu M.M., Morrisey E.E., Parmacek M.S. (2006) 
<span class="ref-title">GATA-6 regulates semaphorin 3C and is required in cardiac neural crest for cardiovascular morphogenesis</span>. <span class="ref-journal">J. Clin. Invest</span>., <span class="ref-vol">116</span>, 929–939. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1409743/">PMC free article</a>]</span> [<a href="/pubmed/16557299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=GATA-6+regulates+semaphorin+3C+and+is+required+in+cardiac+neural+crest+for+cardiovascular+morphogenesis&amp;author=J.J.+Lepore&amp;author=P.A.+Mericko&amp;author=L.+Cheng&amp;author=M.M.+Lu&amp;author=E.E.+Morrisey&amp;volume=116&amp;publication_year=2006&amp;pages=929-939&amp;pmid=16557299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B20">20. <span class="mixed-citation">
Kodo K., Nishizawa T., Furutani M., Arai S., Yamamura E., Joo K., Takahashi T., Matsuoka R., Yamagishi H. (2009) 
<span class="ref-title">GATA6 mutations cause human cardiac outflow tract defects by disrupting semaphorin-plexin signaling</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A</span>., <span class="ref-vol">106</span>, 13933–13938. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2728998/">PMC free article</a>]</span> [<a href="/pubmed/19666519" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A&amp;title=GATA6+mutations+cause+human+cardiac+outflow+tract+defects+by+disrupting+semaphorin-plexin+signaling&amp;author=K.+Kodo&amp;author=T.+Nishizawa&amp;author=M.+Furutani&amp;author=S.+Arai&amp;author=E.+Yamamura&amp;volume=106&amp;publication_year=2009&amp;pages=13933-13938&amp;pmid=19666519&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B21">21. <span class="mixed-citation">
Lek M., Karczewski K., Minikel E., Samocha K., Banks E., Fennell T., O'Donnell-Luria A., Ware J., Hill A., Cummings B., et al. (2015) 
<span class="ref-title">Analysis of protein-coding genetic variation in 60,706 humans</span>. <span class="ref-journal">bioRxiv</span>, in press. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5018207/">PMC free article</a>]</span> [<a href="/pubmed/27535533" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Analysis+of+protein-coding+genetic+variation+in+60,706+humans&amp;author=M.+Lek&amp;author=K.+Karczewski&amp;author=E.+Minikel&amp;author=K.+Samocha&amp;author=E.+Banks&amp;publication_year=2015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B22">22. <span class="mixed-citation">
Yang J., Lee S.H., Goddard M.E., Visscher P.M. (2011) 
<span class="ref-title">GCTA: a tool for genome-wide complex trait analysis</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">88</span>, 76–82. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J.+Yang&amp;author=S.H.+Lee&amp;author=M.E.+Goddard&amp;author=P.M.+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B23">23. <span class="mixed-citation">
Lee S.H., Wray N.R., Goddard M.E., Visscher P.M. (2011) 
<span class="ref-title">Estimating missing heritability for disease from genome-wide association studies</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">88</span>, 294–305. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3059431/">PMC free article</a>]</span> [<a href="/pubmed/21376301" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Estimating+missing+heritability+for+disease+from+genome-wide+association+studies&amp;author=S.H.+Lee&amp;author=N.R.+Wray&amp;author=M.E.+Goddard&amp;author=P.M.+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=294-305&amp;pmid=21376301&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B24">24. <span class="mixed-citation">
Manolio T.A., Collins F.S., Cox N.J., Goldstein D.B., Hindorff L.A., Hunter D.J., McCarthy M.I., Ramos E.M., Cardon L.R., Chakravarti A., et al. (2009) 
<span class="ref-title">Finding the missing heritability of complex diseases</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">461</span>, 747–753. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2831613/">PMC free article</a>]</span> [<a href="/pubmed/19812666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Finding+the+missing+heritability+of+complex+diseases&amp;author=T.A.+Manolio&amp;author=F.S.+Collins&amp;author=N.J.+Cox&amp;author=D.B.+Goldstein&amp;author=L.A.+Hindorff&amp;volume=461&amp;publication_year=2009&amp;pages=747-753&amp;pmid=19812666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B25">25. <span class="mixed-citation">
Wang X., Chua H.X., Chen P., Ong R.T., Sim X., Zhang W., Takeuchi F., Liu X., Khor C.C., Tay W.T., et al. (2013) 
<span class="ref-title">Comparing methods for performing trans-ethnic meta-&nbsp;analysis of genome-wide association studies</span>. <span class="ref-journal">Hum. Mol. Genet</span>., <span class="ref-vol">22</span>, 2303–2311. [<a href="/pubmed/23406875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Comparing+methods+for+performing+trans-ethnic+meta-&nbsp;analysis+of+genome-wide+association+studies&amp;author=X.+Wang&amp;author=H.X.+Chua&amp;author=P.+Chen&amp;author=R.T.+Ong&amp;author=X.+Sim&amp;volume=22&amp;publication_year=2013&amp;pages=2303-2311&amp;pmid=23406875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B26">26. <span class="mixed-citation">
Okamura Y., Saga Y. (2008) 
<span class="ref-title">Notch signaling is required for the maintenance of enteric neural crest progenitors</span>. <span class="ref-journal">Development</span>, <span class="ref-vol">135</span>, 3555–3565. [<a href="/pubmed/18832397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Notch+signaling+is+required+for+the+maintenance+of+enteric+neural+crest+progenitors&amp;author=Y.+Okamura&amp;author=Y.+Saga&amp;volume=135&amp;publication_year=2008&amp;pages=3555-3565&amp;pmid=18832397&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B27">27. <span class="mixed-citation">
Heanue T.A., Pachnis V. (2006) 
<span class="ref-title">Expression profiling the developing mammalian enteric nervous system identifies marker and candidate Hirschsprung disease genes</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A</span>., <span class="ref-vol">103</span>, 6919–6924. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1458994/">PMC free article</a>]</span> [<a href="/pubmed/16632597" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A&amp;title=Expression+profiling+the+developing+mammalian+enteric+nervous+system+identifies+marker+and+candidate+Hirschsprung+disease+genes&amp;author=T.A.+Heanue&amp;author=V.+Pachnis&amp;volume=103&amp;publication_year=2006&amp;pages=6919-6924&amp;pmid=16632597&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B28">28. <span class="mixed-citation">
Garcia-Barcelo M., Sham M.H., Lui V.C., Chen B.L., Ott J., Tam P.K. (2003) 
<span class="ref-title">Association study of PHOX2B as a candidate gene for Hirschsprung's disease</span>. <span class="ref-journal">Gut</span>, <span class="ref-vol">52</span>, 563–567. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1773584/">PMC free article</a>]</span> [<a href="/pubmed/12631670" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Association+study+of+PHOX2B+as+a+candidate+gene+for+Hirschsprung's+disease&amp;author=M.+Garcia-Barcelo&amp;author=M.H.+Sham&amp;author=V.C.+Lui&amp;author=B.L.+Chen&amp;author=J.+Ott&amp;volume=52&amp;publication_year=2003&amp;pages=563-567&amp;pmid=12631670&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B29">29. <span class="mixed-citation">
Marigorta U.M., Navarro A. (2013) 
<span class="ref-title">High trans-ethnic replicability of GWAS results implies common causal variants</span>. <span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">9</span>, e1003566.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3681663/">PMC free article</a>]</span> [<a href="/pubmed/23785302" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=High+trans-ethnic+replicability+of+GWAS+results+implies+common+causal+variants&amp;author=U.M.+Marigorta&amp;author=A.+Navarro&amp;volume=9&amp;publication_year=2013&amp;pages=e1003566.&amp;pmid=23785302&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B30">30. <span class="mixed-citation">
The 1000 Genomes Project Consortium, Auton A., Brooks L.D., Durbin R.M., Garrison E.P., Kang H.M., Korbel J.O., Marchini J.L., McCarthy S., McVean G.A., et al. (2015) 
<span class="ref-title">A global reference for human genetic variation</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">526</span>, 68–74. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4750478/">PMC free article</a>]</span> [<a href="/pubmed/26432245" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+global+reference+for+human+genetic+variation&amp;author=A.+Auton&amp;author=L.D.+Brooks&amp;author=R.M.+Durbin&amp;author=E.P.+Garrison&amp;author=H.M.+Kang&amp;volume=526&amp;publication_year=2015&amp;pages=68-74&amp;pmid=26432245&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B31">31. <span class="mixed-citation">
Tang C.S., Cheng G., So M.T., Yip B.H., Miao X.P., Wong E.H., Ngan E.S., Lui V.C., Song Y.Q., Chan D., et al. (2012) 
<span class="ref-title">Genome-wide copy number analysis uncovers a new HSCR gene: NRG3</span>. <span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">8</span>, e1002687.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3349728/">PMC free article</a>]</span> [<a href="/pubmed/22589734" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+copy+number+analysis+uncovers+a+new+HSCR+gene:+NRG3&amp;author=C.S.+Tang&amp;author=G.+Cheng&amp;author=M.T.+So&amp;author=B.H.+Yip&amp;author=X.P.+Miao&amp;volume=8&amp;publication_year=2012&amp;pages=e1002687.&amp;pmid=22589734&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B32">32. <span class="mixed-citation">
Luzon-Toro B., Gui H., Ruiz-Ferrer M., Sze-Man Tang C., Fernandez R.M., Sham P.C., Torroglosa A., Kwong-Hang Tam P., Espino-Paisan L., Cherny S.S., et al. (2015) 
<span class="ref-title">Exome sequencing reveals a high genetic heterogeneity on familial Hirschsprung disease</span>. <span class="ref-journal">Sci. Rep</span>., <span class="ref-vol">5</span>, 16473.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4642299/">PMC free article</a>]</span> [<a href="/pubmed/26559152" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep&amp;title=Exome+sequencing+reveals+a+high+genetic+heterogeneity+on+familial+Hirschsprung+disease&amp;author=B.+Luzon-Toro&amp;author=H.+Gui&amp;author=M.+Ruiz-Ferrer&amp;author=C.+Sze-Man+Tang&amp;author=R.M.+Fernandez&amp;volume=5&amp;publication_year=2015&amp;pages=16473.&amp;pmid=26559152&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B33">33. <span class="mixed-citation">
Chen X., Kuja-Halkola R., Rahman I., Arpegard J., Viktorin A., Karlsson R., Hagg S., Svensson P., Pedersen N.L., Magnusson P.K. (2015) 
<span class="ref-title">Dominant genetic variation and missing heritability for human complex traits: insights from twin versus genome-wide common SNP models</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">97</span>, 708–714. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4667127/">PMC free article</a>]</span> [<a href="/pubmed/26544805" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Dominant+genetic+variation+and+missing+heritability+for+human+complex+traits:+insights+from+twin+versus+genome-wide+common+SNP+models&amp;author=X.+Chen&amp;author=R.+Kuja-Halkola&amp;author=I.+Rahman&amp;author=J.+Arpegard&amp;author=A.+Viktorin&amp;volume=97&amp;publication_year=2015&amp;pages=708-714&amp;pmid=26544805&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B34">34. <span class="mixed-citation">
Brown B.C., Asian Genetic Epidemiology Network Type 2 Diabetes Consortium. Ye C.J., Price A.L., Zaitlen N. (2016) 
<span class="ref-title">Transethnic Genetic-Correlation Estimates from Summary Statistics</span>. <span class="ref-journal">Am. J. Hum. Genet.</span>, <span class="ref-vol">99</span>, 76-88. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5005434/">PMC free article</a>]</span> [<a href="/pubmed/27321947" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Transethnic+Genetic-Correlation+Estimates+from+Summary+Statistics&amp;author=B.C.+Brown&amp;author=C.J.+Ye&amp;author=A.L.+Price&amp;author=N.+Zaitlen&amp;volume=99&amp;publication_year=2016&amp;pages=76-88&amp;pmid=27321947&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B35">35. <span class="mixed-citation">
Gui H., Tang W.K., So M.T., Proitsi P., Sham P.C., Tam P.K., Ngan E.S., Cherny S.S., Garcia-Barcelo M.M. (2013) 
<span class="ref-title">RET and NRG1 interplay in Hirschsprung disease</span>. <span class="ref-journal">Hum. Genet</span>., <span class="ref-vol">132</span>, 591–600. [<a href="/pubmed/23400839" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet&amp;title=RET+and+NRG1+interplay+in+Hirschsprung+disease&amp;author=H.+Gui&amp;author=W.K.+Tang&amp;author=M.T.+So&amp;author=P.+Proitsi&amp;author=P.C.+Sham&amp;volume=132&amp;publication_year=2013&amp;pages=591-600&amp;pmid=23400839&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B36">36. <span class="mixed-citation">
Arnold S., Pelet A., Amiel J., Borrego S., Hofstra R., Tam P., Ceccherini I., Lyonnet S., Sherman S., Chakravarti A. (2009) 
<span class="ref-title">Interaction between a chromosome 10 RET enhancer and chromosome 21 in the Down syndrome-Hirschsprung disease association</span>. <span class="ref-journal">Hum. Mutat</span>., <span class="ref-vol">30</span>, 771–775. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2779545/">PMC free article</a>]</span> [<a href="/pubmed/19306335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mutat&amp;title=Interaction+between+a+chromosome+10+RET+enhancer+and+chromosome+21+in+the+Down+syndrome-Hirschsprung+disease+association&amp;author=S.+Arnold&amp;author=A.+Pelet&amp;author=J.+Amiel&amp;author=S.+Borrego&amp;author=R.+Hofstra&amp;volume=30&amp;publication_year=2009&amp;pages=771-775&amp;pmid=19306335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B37">37. <span class="mixed-citation">
Genin E., Clerget-Darpoux F. (2015) 
<span class="ref-title">The missing heritability paradigm: a dramatic resurgence of the GIGO syndrome in genetics</span>. <span class="ref-journal">Hum. Hered</span>., <span class="ref-vol">79</span>, 1–4. [<a href="/pubmed/25660130" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Hered&amp;title=The+missing+heritability+paradigm:+a+dramatic+resurgence+of+the+GIGO+syndrome+in+genetics&amp;author=E.+Genin&amp;author=F.+Clerget-Darpoux&amp;volume=79&amp;publication_year=2015&amp;pages=1-4&amp;pmid=25660130&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B38">38. <span class="mixed-citation">
Schizophrenia Working Group of the Psychiatric Genomics Consortium (2014) 
<span class="ref-title">Biological insights from 108 schizophrenia-associated genetic loci</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">511</span>, 421–427. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4112379/">PMC free article</a>]</span> [<a href="/pubmed/25056061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Biological+insights+from+108+schizophrenia-associated+genetic+loci&amp;volume=511&amp;publication_year=2014&amp;pages=421-427&amp;pmid=25056061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B39">39. <span class="mixed-citation">
EPICURE Consortium, EMINet Consortium, Steffens M., Leu C., Ruppert A.K., Zara F., Striano P., Robbiano A., Capovilla G., Tinuper P., et al. (2012) 
<span class="ref-title">Genome-wide association analysis of genetic generalized epilepsies implicates susceptibility loci at 1q43, 2p16.1, 2q22.3 and 17q21.32</span>. <span class="ref-journal">Hum. Mol. Genet</span>., <span class="ref-vol">21</span>, 5359–5372. [<a href="/pubmed/22949513" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Genome-wide+association+analysis+of+genetic+generalized+epilepsies+implicates+susceptibility+loci+at+1q43,+2p16.1,+2q22.3+and+17q21.32&amp;author=M.+Steffens&amp;author=C.+Leu&amp;author=A.K.+Ruppert&amp;author=F.+Zara&amp;author=P.+Striano&amp;volume=21&amp;publication_year=2012&amp;pages=5359-5372&amp;pmid=22949513&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B40">40. <span class="mixed-citation">
Rajcan-Separovic E., Harvard C., Liu X., McGillivray B., Hall J.G., Qiao Y., Hurlburt J., Hildebrand J., Mickelson E.C., Holden J.J., et al. (2007) 
<span class="ref-title">Clinical and molecular cytogenetic characterisation of a newly recognised micro&nbsp;deletion syndrome involving 2p15-16.1</span>. <span class="ref-journal">J. Med. Genet</span>., <span class="ref-vol">44</span>, 269–276. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2598046/">PMC free article</a>]</span> [<a href="/pubmed/16963482" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Genet&amp;title=Clinical+and+molecular+cytogenetic+characterisation+of+a+newly+recognised+micro&nbsp;deletion+syndrome+involving+2p15-16.1&amp;author=E.+Rajcan-Separovic&amp;author=C.+Harvard&amp;author=X.+Liu&amp;author=B.+McGillivray&amp;author=J.G.+Hall&amp;volume=44&amp;publication_year=2007&amp;pages=269-276&amp;pmid=16963482&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B41">41. <span class="mixed-citation">
de Leeuw N., Pfundt R., Koolen D.A., Neefs I., Scheltinga I., Mieloo H., Sistermans E.A., Nillesen W., Smeets D.F., de Vries B.B., et al. (2008) 
<span class="ref-title">A newly recognised microdeletion syndrome involving 2p15p16.1: narrowing down the critical region by adding another patient detected by genome wide tiling path array comparative genomic hybridisation analysis</span>. <span class="ref-journal">J. Med. Genet</span>., <span class="ref-vol">45</span>, 122–124. [<a href="/pubmed/18245392" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Genet&amp;title=A+newly+recognised+microdeletion+syndrome+involving+2p15p16.1:+narrowing+down+the+critical+region+by+adding+another+patient+detected+by+genome+wide+tiling+path+array+comparative+genomic+hybridisation+analysis&amp;author=N.+de+Leeuw&amp;author=R.+Pfundt&amp;author=D.A.+Koolen&amp;author=I.+Neefs&amp;author=I.+Scheltinga&amp;volume=45&amp;publication_year=2008&amp;pages=122-124&amp;pmid=18245392&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B42">42. <span class="mixed-citation">
Fernandez I.F., Blanco S., Lozano J., Lazo P.A. (2010) 
<span class="ref-title">VRK2 inhibits mitogen-activated protein kinase signaling and inversely correlates with ErbB2 in human breast cancer</span>. <span class="ref-journal">Mol. Cell. Biol</span>., <span class="ref-vol">30</span>, 4687–4697. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2950518/">PMC free article</a>]</span> [<a href="/pubmed/20679487" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol&amp;title=VRK2+inhibits+mitogen-activated+protein+kinase+signaling+and+inversely+correlates+with+ErbB2+in+human+breast+cancer&amp;author=I.F.+Fernandez&amp;author=S.+Blanco&amp;author=J.+Lozano&amp;author=P.A.+Lazo&amp;volume=30&amp;publication_year=2010&amp;pages=4687-4697&amp;pmid=20679487&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B43">43. <span class="mixed-citation">
Hotta R., Stamp L.A., Foong J.P., McConnell S.N., Bergner A.J., Anderson R.B., Enomoto H., Newgreen D.F., Obermayr F., Furness J.B., et al. (2013) 
<span class="ref-title">Transplanted progenitors generate functional enteric neurons in the postnatal colon</span>. <span class="ref-journal">J. Clin. Invest</span>., <span class="ref-vol">123</span>, 1182–1191. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3582137/">PMC free article</a>]</span> [<a href="/pubmed/23454768" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=Transplanted+progenitors+generate+functional+enteric+neurons+in+the+postnatal+colon&amp;author=R.+Hotta&amp;author=L.A.+Stamp&amp;author=J.P.+Foong&amp;author=S.N.+McConnell&amp;author=A.J.+Bergner&amp;volume=123&amp;publication_year=2013&amp;pages=1182-1191&amp;pmid=23454768&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B44">44. <span class="mixed-citation">
Fattahi F., Steinbeck J.A., Kriks S., Tchieu J., Zimmer B., Kishinevsky S., Zeltner N., Mica Y., El-Nachef W., Zhao H., et al. (2016) 
<span class="ref-title">Deriving human ENS lineages for cell therapy and drug discovery in Hirschsprung disease</span>. <span class="ref-journal">Nature</span> <span class="ref-vol">531</span>, 105-109. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4846424/">PMC free article</a>]</span> [<a href="/pubmed/26863197" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Deriving+human+ENS+lineages+for+cell+therapy+and+drug+discovery+in+Hirschsprung+disease&amp;author=F.+Fattahi&amp;author=J.A.+Steinbeck&amp;author=S.+Kriks&amp;author=J.+Tchieu&amp;author=B.+Zimmer&amp;volume=531&amp;publication_year=2016&amp;pages=105-109&amp;pmid=26863197&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B45">45. <span class="mixed-citation">
Garcia-Barcelo M.M., Yeung M.Y., Miao X.P., Tang C.S., Cheng G., So M.T., Ngan E.S., Lui V.C., Chen Y., Liu X.L., et al. (2010) 
<span class="ref-title">Genome-wide association study identifies a susceptibility locus for biliary atresia on 10q24.2</span>. <span class="ref-journal">Hum. Mol. Genet</span>., <span class="ref-vol">19</span>, 2917–2925. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2893814/">PMC free article</a>]</span> [<a href="/pubmed/20460270" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Genome-wide+association+study+identifies+a+susceptibility+locus+for+biliary+atresia+on+10q24.2&amp;author=M.M.+Garcia-Barcelo&amp;author=M.Y.+Yeung&amp;author=X.P.+Miao&amp;author=C.S.+Tang&amp;author=G.+Cheng&amp;volume=19&amp;publication_year=2010&amp;pages=2917-2925&amp;pmid=20460270&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B46">46. <span class="mixed-citation">
Tang C.S., Tang W.K., So M.T., Miao X.P., Leung B.M., Yip B.H., Leon T.Y., Ngan E.S., Lui V.C., Chen Y., et al. (2011) 
<span class="ref-title">Fine mapping of the NRG1 Hirschsprung's disease locus</span>. <span class="ref-journal">PLoS One</span>, <span class="ref-vol">6</span>, e16181.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3024406/">PMC free article</a>]</span> [<a href="/pubmed/21283760" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Fine+mapping+of+the+NRG1+Hirschsprung's+disease+locus&amp;author=C.S.+Tang&amp;author=W.K.+Tang&amp;author=M.T.+So&amp;author=X.P.+Miao&amp;author=B.M.+Leung&amp;volume=6&amp;publication_year=2011&amp;pages=e16181.&amp;pmid=21283760&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B47">47. <span class="mixed-citation">
O'Connell J., Gurdasani D., Delaneau O., Pirastu N., Ulivi S., Cocca M., Traglia M., Huang J., Huffman J.E., Rudan I., et al. (2014) 
<span class="ref-title">A general approach for haplotype phasing across the full spectrum of relatedness</span>. <span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">10</span>, e1004234.. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3990520/">PMC free article</a>]</span> [<a href="/pubmed/24743097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+general+approach+for+haplotype+phasing+across+the+full+spectrum+of+relatedness&amp;author=J.+O'Connell&amp;author=D.+Gurdasani&amp;author=O.+Delaneau&amp;author=N.+Pirastu&amp;author=S.+Ulivi&amp;volume=10&amp;publication_year=2014&amp;pages=e1004234.&amp;pmid=24743097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B48">48. <span class="mixed-citation">
Howie B., Fuchsberger C., Stephens M., Marchini J., Abecasis G.R. (2012) 
<span class="ref-title">Fast and accurate genotype imputation in genome-wide association studies through pre-phasing</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">44</span>, 955–959. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B.+Howie&amp;author=C.+Fuchsberger&amp;author=M.+Stephens&amp;author=J.+Marchini&amp;author=G.R.+Abecasis&amp;volume=44&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B49">49. <span class="mixed-citation">
Marchini J., Howie B., Myers S., McVean G., Donnelly P. (2007) 
<span class="ref-title">A new multipoint method for genome-wide association studies by imputation of genotypes</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">39</span>, 906–913. [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J.+Marchini&amp;author=B.+Howie&amp;author=S.+Myers&amp;author=G.+McVean&amp;author=P.+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B50">50. <span class="mixed-citation">
Willer C.J., Li Y., Abecasis G.R. (2010) 
<span class="ref-title">METAL: fast and efficient meta-analysis of genomewide association scans</span>. <span class="ref-journal">Bioinformatics</span>, <span class="ref-vol">26</span>, 2190–2191. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B51">51. <span class="mixed-citation">
Boyle A.P., Hong E.L., Hariharan M., Cheng Y., Schaub M.A., Kasowski M., Karczewski K.J., Park J., Hitz B.C., Weng S., et al. (2012) 
<span class="ref-title">Annotation of functional variation in personal genomes using RegulomeDB</span>. <span class="ref-journal">Genome Res</span>., <span class="ref-vol">22</span>, 1790–1797. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3431494/">PMC free article</a>]</span> [<a href="/pubmed/22955989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Annotation+of+functional+variation+in+personal+genomes+using+RegulomeDB&amp;author=A.P.+Boyle&amp;author=E.L.+Hong&amp;author=M.+Hariharan&amp;author=Y.+Cheng&amp;author=M.A.+Schaub&amp;volume=22&amp;publication_year=2012&amp;pages=1790-1797&amp;pmid=22955989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B52">52. <span class="mixed-citation">
Ward L.D., Kellis M. (2016) 
<span class="ref-title">HaploReg v4: systematic mining of putative causal variants, cell types, regulators and target genes for human complex traits and disease</span>. <span class="ref-journal">Nucleic. Acids. Res</span>., <span class="ref-vol">44</span>, D877–D881. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4702929/">PMC free article</a>]</span> [<a href="/pubmed/26657631" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic.+Acids.+Res&amp;title=HaploReg+v4:+systematic+mining+of+putative+causal+variants,+cell+types,+regulators+and+target+genes+for+human+complex+traits+and+disease&amp;author=L.D.+Ward&amp;author=M.+Kellis&amp;volume=44&amp;publication_year=2016&amp;pages=D877-D881&amp;pmid=26657631&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B53">53. <span class="mixed-citation">
GTEx Consortium. (2013) 
<span class="ref-title">The Genotype-Tissue Expression (GTEx) project</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">45</span>, 580–585. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4010069/">PMC free article</a>]</span> [<a href="/pubmed/23715323" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=The+Genotype-Tissue+Expression+(GTEx)+project&amp;volume=45&amp;publication_year=2013&amp;pages=580-585&amp;pmid=23715323&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="ddw333-B54">54. <span class="mixed-citation">
So H.C., Gui A.H., Cherny S.S., Sham P.C. (2011) 
<span class="ref-title">Evaluating the heritability explained by known susceptibility variants: a survey of ten complex diseases</span>. <span class="ref-journal">Genet. Epidemiol</span>., <span class="ref-vol">35</span>, 310–317. [<a href="/pubmed/21374718" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Evaluating+the+heritability+explained+by+known+susceptibility+variants:+a+survey+of+ten+complex+diseases&amp;author=H.C.+So&amp;author=A.H.+Gui&amp;author=S.S.+Cherny&amp;author=P.C.+Sham&amp;volume=35&amp;publication_year=2011&amp;pages=310-317&amp;pmid=21374718&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6078638&amp;issue-id=309841&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139663910190576title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Results</a></li><li><a href="#__sec8title">Discussion</a></li><li><a href="#__sec9title">Materials and Methods</a></li><li><a href="#__sec23title">Supplementary Material</a></li><li><a href="#__sec24title">Supplementary Material</a></li><li><a href="#idm139663943830656title">Acknowledgements</a></li><li><a href="#idm139663909698880title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC6078638/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC6078638/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC6078638/pdf/ddw333.pdf">PDF (481K)</a></li> | <li><a href="#" data-citationid="PMC6078638" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6078638%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6078638%2F&amp;text=%0ATrans-ethnic%20meta-analysis%20of%20genome-wide%20association%20studies%20for%20Hirschsprung%20disease" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6078638%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="27702942" data-aiid="6078638" data-aid="6078638" data-iid="309841" data-domainid="740" data-domain="hmg" data-accid="PMC6078638" data-md5="04eb993dbc6085647df079584e84124e"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T15:15:40-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE8902FDDD595CB100000000063103D3&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6078638%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC6078638/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>