<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="carcin"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC6148967/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study of familial lung cancer</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148967/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Carcinogenesis"><meta name="citation_title" content="Genome-wide association study of familial lung cancer"><meta name="citation_authors" content="Jinyoung Byun, Ann G Schwartz, Christine Lusk, Angela S Wenzlaff, Mariza de Andrade, Diptasri Mandal, Colette Gaba, Ping Yang, Ming You, Elena Y Kupert, Marshall W Anderson, Younghun Han, Yafang Li, David Qian, Adrienne Stilp, Cathy Laurie, Sarah Nelson, Wenying Zheng, Rayjean J Hung, Valerie Gaborieau, James Mckay, Paul Brennan, Neil E Caporaso, Maria Teresa Landi, Xifeng Wu, John R McLaughlin, Yonathan Brhane, Yohan Bossé, Susan M Pinney, Joan E Bailey-Wilson, Christopher I Amos"><meta name="citation_date" content="September 2018"><meta name="citation_issue" content="9"><meta name="citation_volume" content="39"><meta name="citation_firstpage" content="1135"><meta name="citation_doi" content="10.1093/carcin/bgy080"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC6148967/?report=abstract"><meta name="citation_pmid" content="29924316"><meta name="DC.Title" content="Genome-wide association study of familial lung cancer"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Jinyoung Byun"><meta name="DC.Contributor" content="Ann G Schwartz"><meta name="DC.Contributor" content="Christine Lusk"><meta name="DC.Contributor" content="Angela S Wenzlaff"><meta name="DC.Contributor" content="Mariza de Andrade"><meta name="DC.Contributor" content="Diptasri Mandal"><meta name="DC.Contributor" content="Colette Gaba"><meta name="DC.Contributor" content="Ping Yang"><meta name="DC.Contributor" content="Ming You"><meta name="DC.Contributor" content="Elena Y Kupert"><meta name="DC.Contributor" content="Marshall W Anderson"><meta name="DC.Contributor" content="Younghun Han"><meta name="DC.Contributor" content="Yafang Li"><meta name="DC.Contributor" content="David Qian"><meta name="DC.Contributor" content="Adrienne Stilp"><meta name="DC.Contributor" content="Cathy Laurie"><meta name="DC.Contributor" content="Sarah Nelson"><meta name="DC.Contributor" content="Wenying Zheng"><meta name="DC.Contributor" content="Rayjean J Hung"><meta name="DC.Contributor" content="Valerie Gaborieau"><meta name="DC.Contributor" content="James Mckay"><meta name="DC.Contributor" content="Paul Brennan"><meta name="DC.Contributor" content="Neil E Caporaso"><meta name="DC.Contributor" content="Maria Teresa Landi"><meta name="DC.Contributor" content="Xifeng Wu"><meta name="DC.Contributor" content="John R McLaughlin"><meta name="DC.Contributor" content="Yonathan Brhane"><meta name="DC.Contributor" content="Yohan Bossé"><meta name="DC.Contributor" content="Susan M Pinney"><meta name="DC.Contributor" content="Joan E Bailey-Wilson"><meta name="DC.Contributor" content="Christopher I Amos"><meta name="DC.Date" content="2018 Sep"><meta name="DC.Identifier" content="10.1093/carcin/bgy080"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study of familial lung cancer"><meta property="og:type" content="article"><meta property="og:description" content="To identify genetic variation associated with lung cancer risk, we performed a genome-wide association analysis of 685 lung cancer cases that had a family history of two or more first or second degree relatives compared with 744 controls without lung ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148967/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-carcin.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC6148967/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC6148967/pdf/bgy080.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C41D5DD5AC8C100000000082104BC.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6148967%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="mesh" class="last">MeSH</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Carcinogenesis%22[journal]" class="navlink">Carcinogenesis</a></li><li class="accid">PMC6148967</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-carcin.png" alt="Logo of carcin" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/carcin" target="pmc_ext" ref="reftype=publisher&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Carcinogenesis</a></span>. 2018 Sep; 39(9): 1135–1140. </span></div><div><span class="fm-vol-iss-date">Published online 2018 Jun 19. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fcarcin%2Fbgy080" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/carcin/bgy080</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC6148967</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/29924316">29924316</a></div></div></div></div><h1 class="content-title">Genome-wide association study of familial lung cancer</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Byun%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670473539440" co-class="co-affbox">Jinyoung Byun</a>,<sup>1</sup><sup></sup> <a href="/pubmed/?term=Schwartz%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670473524896" co-class="co-affbox">Ann G Schwartz</a>,<sup>2</sup><sup></sup><sup></sup> <a href="/pubmed/?term=Lusk%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670473521184" co-class="co-affbox">Christine Lusk</a>,<sup>2</sup> <a href="/pubmed/?term=Wenzlaff%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670473519200" co-class="co-affbox">Angela S Wenzlaff</a>,<sup>2</sup> <a href="/pubmed/?term=de%20Andrade%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670473517168" co-class="co-affbox">Mariza de Andrade</a>,<sup>3</sup> <a href="/pubmed/?term=Mandal%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670473515088" co-class="co-affbox">Diptasri Mandal</a>,<sup>4</sup> <a href="/pubmed/?term=Gaba%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670473513008" co-class="co-affbox">Colette Gaba</a>,<sup>5</sup> <a href="/pubmed/?term=Yang%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670473510880" co-class="co-affbox">Ping Yang</a>,<sup>3</sup> <a href="/pubmed/?term=You%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670473508848" co-class="co-affbox">Ming You</a>,<sup>6</sup> <a href="/pubmed/?term=Kupert%20EY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476605808" co-class="co-affbox">Elena Y Kupert</a>,<sup>6</sup> <a href="/pubmed/?term=Anderson%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476603776" co-class="co-affbox">Marshall W Anderson</a>,<sup>6</sup> <a href="/pubmed/?term=Han%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476601744" co-class="co-affbox">Younghun Han</a>,<sup>1</sup> <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476599712" co-class="co-affbox">Yafang Li</a>,<sup>1</sup> <a href="/pubmed/?term=Qian%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476597680" co-class="co-affbox">David Qian</a>,<sup>1</sup> <a href="/pubmed/?term=Stilp%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476595648" co-class="co-affbox">Adrienne Stilp</a>,<sup>7</sup> <a href="/pubmed/?term=Laurie%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476593568" co-class="co-affbox">Cathy Laurie</a>,<sup>7</sup> <a href="/pubmed/?term=Nelson%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476591536" co-class="co-affbox">Sarah Nelson</a>,<sup>7</sup> <a href="/pubmed/?term=Zheng%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476589504" co-class="co-affbox">Wenying Zheng</a>,<sup>7</sup> <a href="/pubmed/?term=Hung%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476587472" co-class="co-affbox">Rayjean J Hung</a>,<sup>8</sup> <a href="/pubmed/?term=Gaborieau%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476585392" co-class="co-affbox">Valerie Gaborieau</a>,<sup>9</sup> <a href="/pubmed/?term=Mckay%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476583312" co-class="co-affbox">James Mckay</a>,<sup>9</sup> <a href="/pubmed/?term=Brennan%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476581280" co-class="co-affbox">Paul Brennan</a>,<sup>9</sup> <a href="/pubmed/?term=Caporaso%20NE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476579248" co-class="co-affbox">Neil E Caporaso</a>,<sup>10</sup> <a href="/pubmed/?term=Landi%20MT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476577216" co-class="co-affbox">Maria Teresa Landi</a>,<sup>10</sup> <a href="/pubmed/?term=Wu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476575184" co-class="co-affbox">Xifeng Wu</a>,<sup>11</sup> <a href="/pubmed/?term=McLaughlin%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476573056" co-class="co-affbox">John R McLaughlin</a>,<sup>12</sup> <a href="/pubmed/?term=Brhane%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476570928" co-class="co-affbox">Yonathan Brhane</a>,<sup>7</sup> <a href="/pubmed/?term=Boss%26%23x000e9%3B%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476568896" co-class="co-affbox">Yohan Bossé</a>,<sup>13</sup> <a href="/pubmed/?term=Pinney%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670476566768" co-class="co-affbox">Susan M Pinney</a>,<sup>14</sup> <a href="/pubmed/?term=Bailey-Wilson%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670518057520" co-class="co-affbox">Joan E Bailey-Wilson</a>,<sup>15</sup> and  <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316" class="affpopup" co-rid="_co_idm139670518055440" co-class="co-affbox">Christopher I Amos</a><sup>1,</sup><sup>16</sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139670473539440"><h3 class="no_margin">Jinyoung Byun</h3><p><sup>1</sup>Department of Biomedical Data Science, Dartmouth Geisel School of Medicine, Lebanon, NH, USA</p><div>Find articles by <a href="/pubmed/?term=Byun%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Jinyoung Byun</a></div></div><div id="_co_idm139670473524896"><h3 class="no_margin">Ann G Schwartz</h3><p><sup>2</sup>Karmanos Cancer Institute, Wayne State University, Detroit, MI, USA</p><div>Find articles by <a href="/pubmed/?term=Schwartz%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Ann G Schwartz</a></div></div><div id="_co_idm139670473521184"><h3 class="no_margin">Christine Lusk</h3><p><sup>2</sup>Karmanos Cancer Institute, Wayne State University, Detroit, MI, USA</p><div>Find articles by <a href="/pubmed/?term=Lusk%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Christine Lusk</a></div></div><div id="_co_idm139670473519200"><h3 class="no_margin">Angela S Wenzlaff</h3><p><sup>2</sup>Karmanos Cancer Institute, Wayne State University, Detroit, MI, USA</p><div>Find articles by <a href="/pubmed/?term=Wenzlaff%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Angela S Wenzlaff</a></div></div><div id="_co_idm139670473517168"><h3 class="no_margin">Mariza de Andrade</h3><p><sup>3</sup>Department of Health Sciences Research, Mayo Clinic, Rochester, MN, USA</p><div>Find articles by <a href="/pubmed/?term=de%20Andrade%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Mariza de Andrade</a></div></div><div id="_co_idm139670473515088"><h3 class="no_margin">Diptasri Mandal</h3><p><sup>4</sup>Louisiana State University Health Sciences Center, New Orleans, LA, USA</p><div>Find articles by <a href="/pubmed/?term=Mandal%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Diptasri Mandal</a></div></div><div id="_co_idm139670473513008"><h3 class="no_margin">Colette Gaba</h3><p><sup>5</sup>University of Toledo Dana Cancer Center, Toledo, OH, USA</p><div>Find articles by <a href="/pubmed/?term=Gaba%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Colette Gaba</a></div></div><div id="_co_idm139670473510880"><h3 class="no_margin">Ping Yang</h3><p><sup>3</sup>Department of Health Sciences Research, Mayo Clinic, Rochester, MN, USA</p><div>Find articles by <a href="/pubmed/?term=Yang%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Ping Yang</a></div></div><div id="_co_idm139670473508848"><h3 class="no_margin">Ming You</h3><p><sup>6</sup>Medical College of Wisconsin, Milwaukee, WI, USA</p><div>Find articles by <a href="/pubmed/?term=You%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Ming You</a></div></div><div id="_co_idm139670476605808"><h3 class="no_margin">Elena Y Kupert</h3><p><sup>6</sup>Medical College of Wisconsin, Milwaukee, WI, USA</p><div>Find articles by <a href="/pubmed/?term=Kupert%20EY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Elena Y Kupert</a></div></div><div id="_co_idm139670476603776"><h3 class="no_margin">Marshall W Anderson</h3><p><sup>6</sup>Medical College of Wisconsin, Milwaukee, WI, USA</p><div>Find articles by <a href="/pubmed/?term=Anderson%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Marshall W Anderson</a></div></div><div id="_co_idm139670476601744"><h3 class="no_margin">Younghun Han</h3><p><sup>1</sup>Department of Biomedical Data Science, Dartmouth Geisel School of Medicine, Lebanon, NH, USA</p><div>Find articles by <a href="/pubmed/?term=Han%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Younghun Han</a></div></div><div id="_co_idm139670476599712"><h3 class="no_margin">Yafang Li</h3><p><sup>1</sup>Department of Biomedical Data Science, Dartmouth Geisel School of Medicine, Lebanon, NH, USA</p><div>Find articles by <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Yafang Li</a></div></div><div id="_co_idm139670476597680"><h3 class="no_margin">David Qian</h3><p><sup>1</sup>Department of Biomedical Data Science, Dartmouth Geisel School of Medicine, Lebanon, NH, USA</p><div>Find articles by <a href="/pubmed/?term=Qian%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">David Qian</a></div></div><div id="_co_idm139670476595648"><h3 class="no_margin">Adrienne Stilp</h3><p><sup>7</sup>Genetic Analysis Center, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Stilp%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Adrienne Stilp</a></div></div><div id="_co_idm139670476593568"><h3 class="no_margin">Cathy Laurie</h3><p><sup>7</sup>Genetic Analysis Center, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Laurie%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Cathy Laurie</a></div></div><div id="_co_idm139670476591536"><h3 class="no_margin">Sarah Nelson</h3><p><sup>7</sup>Genetic Analysis Center, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Nelson%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Sarah Nelson</a></div></div><div id="_co_idm139670476589504"><h3 class="no_margin">Wenying Zheng</h3><p><sup>7</sup>Genetic Analysis Center, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Zheng%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Wenying Zheng</a></div></div><div id="_co_idm139670476587472"><h3 class="no_margin">Rayjean J Hung</h3><p><sup>8</sup>Lunenfeld-Tanenbaum Research Institute, Sinai Health System, Toronto, Ontario, Canada</p><div>Find articles by <a href="/pubmed/?term=Hung%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Rayjean J Hung</a></div></div><div id="_co_idm139670476585392"><h3 class="no_margin">Valerie Gaborieau</h3><p><sup>9</sup>Genetic Epidemiology Group, International Agency for Research on Cancer (IARC), Lyon, France</p><div>Find articles by <a href="/pubmed/?term=Gaborieau%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Valerie Gaborieau</a></div></div><div id="_co_idm139670476583312"><h3 class="no_margin">James Mckay</h3><p><sup>9</sup>Genetic Epidemiology Group, International Agency for Research on Cancer (IARC), Lyon, France</p><div>Find articles by <a href="/pubmed/?term=Mckay%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">James Mckay</a></div></div><div id="_co_idm139670476581280"><h3 class="no_margin">Paul Brennan</h3><p><sup>9</sup>Genetic Epidemiology Group, International Agency for Research on Cancer (IARC), Lyon, France</p><div>Find articles by <a href="/pubmed/?term=Brennan%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Paul Brennan</a></div></div><div id="_co_idm139670476579248"><h3 class="no_margin">Neil E Caporaso</h3><p><sup>10</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Caporaso%20NE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Neil E Caporaso</a></div></div><div id="_co_idm139670476577216"><h3 class="no_margin">Maria Teresa Landi</h3><p><sup>10</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Landi%20MT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Maria Teresa Landi</a></div></div><div id="_co_idm139670476575184"><h3 class="no_margin">Xifeng Wu</h3><p><sup>11</sup>Department of Epidemiology, University of Texas MD Anderson Cancer Center, Houston, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Wu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Xifeng Wu</a></div></div><div id="_co_idm139670476573056"><h3 class="no_margin">John R McLaughlin</h3><p><sup>12</sup>Public Health Ontario, Toronto, Canada</p><div>Find articles by <a href="/pubmed/?term=McLaughlin%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">John R McLaughlin</a></div></div><div id="_co_idm139670476570928"><h3 class="no_margin">Yonathan Brhane</h3><p><sup>7</sup>Genetic Analysis Center, University of Washington, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Brhane%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Yonathan Brhane</a></div></div><div id="_co_idm139670476568896"><h3 class="no_margin">Yohan Bossé</h3><p><sup>13</sup>Institut Universitaire de Cardiologie et de Pneumologie de Québec, Department of Molecular Medicine, Laval University, Québec, Canada</p><div>Find articles by <a href="/pubmed/?term=Boss%26%23x000e9%3B%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Yohan Bossé</a></div></div><div id="_co_idm139670476566768"><h3 class="no_margin">Susan M Pinney</h3><p><sup>14</sup> University of Cincinnati College of Medicine, Cincinnati, OH, USA</p><div>Find articles by <a href="/pubmed/?term=Pinney%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Susan M Pinney</a></div></div><div id="_co_idm139670518057520"><h3 class="no_margin">Joan E Bailey-Wilson</h3><p><sup>15</sup>National Human Genome Research Institute, National Institutes of Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Bailey-Wilson%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Joan E Bailey-Wilson</a></div></div><div id="_co_idm139670518055440"><h3 class="no_margin">Christopher I Amos</h3><p><sup>1</sup>Department of Biomedical Data Science, Dartmouth Geisel School of Medicine, Lebanon, NH, USA</p><p><sup>16</sup>Institute for Clinical and Translational Research, Baylor College of Medicine, Houston, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=29924316">Christopher I Amos</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139670473503200_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139670473503200_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139670473503200_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139670473503200_ai" style="display:none"><div class="fm-affl" id="AF0001" lang="en"><sup>1</sup>Department of Biomedical Data Science, Dartmouth Geisel School of Medicine, Lebanon, NH, USA</div><div class="fm-affl" id="AF0002" lang="en"><sup>2</sup>Karmanos Cancer Institute, Wayne State University, Detroit, MI, USA</div><div class="fm-affl" id="AF0003" lang="en"><sup>3</sup>Department of Health Sciences Research, Mayo Clinic, Rochester, MN, USA</div><div class="fm-affl" id="AF0004" lang="en"><sup>4</sup>Louisiana State University Health Sciences Center, New Orleans, LA, USA</div><div class="fm-affl" id="AF0005" lang="en"><sup>5</sup>University of Toledo Dana Cancer Center, Toledo, OH, USA</div><div class="fm-affl" id="AF0006" lang="en"><sup>6</sup>Medical College of Wisconsin, Milwaukee, WI, USA</div><div class="fm-affl" id="AF0007" lang="en"><sup>7</sup>Genetic Analysis Center, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="AF0008" lang="en"><sup>8</sup>Lunenfeld-Tanenbaum Research Institute, Sinai Health System, Toronto, Ontario, Canada</div><div class="fm-affl" id="AF0009" lang="en"><sup>9</sup>Genetic Epidemiology Group, International Agency for Research on Cancer (IARC), Lyon, France</div><div class="fm-affl" id="AF0010" lang="en"><sup>10</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Bethesda, MD, USA</div><div class="fm-affl" id="AF0011" lang="en"><sup>11</sup>Department of Epidemiology, University of Texas MD Anderson Cancer Center, Houston, TX, USA</div><div class="fm-affl" id="AF0012" lang="en"><sup>12</sup>Public Health Ontario, Toronto, Canada</div><div class="fm-affl" id="AF0013" lang="en"><sup>13</sup>Institut Universitaire de Cardiologie et de Pneumologie de Québec, Department of Molecular Medicine, Laval University, Québec, Canada</div><div class="fm-affl" id="AF0014" lang="en"><sup>14</sup> University of Cincinnati College of Medicine, Cincinnati, OH, USA</div><div class="fm-affl" id="AF0015" lang="en"><sup>15</sup>National Human Genome Research Institute, National Institutes of Health, Baltimore, MD, USA</div><div class="fm-affl" id="AF0016" lang="en"><sup>16</sup>Institute for Clinical and Translational Research, Baylor College of Medicine, Houston, TX, USA</div><div id="c1">To whom correspondence should be addressed. Tel: +1 603 650 1972; Fax: +1 603 653 6696; Email:<a href="mailto:dev@null" data-email="ude.mcb@asirhc" class="oemail">ude.mcb@asirhc</a></div><div id="c2">Correspondence may also be addressed to Ann Schwartz Tel: +1 313 578 4201; Fax: +1 313 578 4306; Email:<a href="mailto:dev@null" data-email="gro.sonamrak@atrawhcs" class="oemail">gro.sonamrak@atrawhcs</a></div><div id="fn-0001">These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139670473503200_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2017 Dec 20; Revised 2018 Apr 12; Accepted 2018 Jun 18.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139670473503200_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author(s) 2018. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com.</div><div class="license half_rhythm">This article is published and distributed under the terms of the Oxford University Press, Standard Journals Publication Model (<a href="https://academic.oup.com/journals/pages/open_access/funder_policies/chorus/standard_publication_model" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">https://academic.oup.com/journals/pages/open_access/funder_policies/chorus/standard_publication_model</a>)</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC6148967/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="3" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary_Table_6.</strong></div><div><a href="/pmc/articles/PMC6148967/bin/bgy080_suppl_supplementary_table_6.xlsx" data-ga-action="click_feat_suppl">bgy080_suppl_supplementary_table_6.xlsx</a><span style="color:gray"> (206K)</span></div><div class="small guid">GUID:&nbsp;AAE6005A-0744-42D9-B636-33EDB8C29882</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary_Table_7.</strong></div><div><a href="/pmc/articles/PMC6148967/bin/bgy080_suppl_supplementary_table_7.xlsx" data-ga-action="click_feat_suppl">bgy080_suppl_supplementary_table_7.xlsx</a><span style="color:gray"> (21K)</span></div><div class="small guid">GUID:&nbsp;8C55248A-165A-497B-B5C4-D297E5377D6D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary_Materials.</strong></div><div><a href="/pmc/articles/PMC6148967/bin/bgy080_suppl_supplementary_materials.docx" data-ga-action="click_feat_suppl">bgy080_suppl_supplementary_materials.docx</a><span style="color:gray"> (1.6M)</span></div><div class="small guid">GUID:&nbsp;E7180ED1-A2B0-45D6-A0AB-890791DAFC1E</div></div></dd></dl></div><div id="idm139670518010320" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139670518010320title">Abstract</h2><!--article-meta--><div><p id="__p3" class="p p-first-last">To identify genetic variation associated with lung cancer risk, we performed a genome-wide association analysis of 685 lung cancer cases that had a family history of two or more first or second degree relatives compared with 744 controls without lung cancer that were genotyped on an Illumina Human OmniExpressExome-8v1 array. To ensure robust results, we further evaluated these findings using data from six additional studies that were assembled through the Transdisciplinary Research on Cancer of the Lung Consortium comprising 1993 familial cases and 33 690 controls. We performed a meta-analysis after imputation of all variants using the 1000 Genomes Project Phase 1 (version 3 release date September 2013). Analyses were conducted for 9 327 222 SNPs integrating data from the two sources. A novel variant on chromosome 4p15.31 near the <em>LCORL</em> gene and an imputed rare variant intergenic between <em>CDKN2A</em> and <em>IFNA8</em> on chromosome 9p21.3 were identified at a genome-wide level of significance for squamous cell carcinomas. Additionally, associations of <em>CHRNA3</em> and <em>CHRNA5</em> on chromosome 15q25.1 in sporadic lung cancer were confirmed at a genome-wide level of significance in familial lung cancer. Previously identified variants in or near <em>CHRNA2, BRCA2, CYP2A6</em> for overall lung cancer, <em>TERT, SECISPB2L</em> and <em>RTEL1</em> for adenocarcinoma and <em>RAD52</em> and <em>MHC</em> for squamous carcinoma were significantly associated with lung cancer.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">Introduction</h2><p id="__p4" class="p p-first">Lung cancer is estimated to account for over 13% of all new cancer diagnoses and almost 27% of all cancer deaths (<a href="http://SEER.cancer.gov/statfacts/html.lungb.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://SEER.cancer.gov/statfacts/html.lungb.html</a>). Despite diagnostic advances in recent years, lung cancer usually presents at advanced stages and mortality remains high. While most lung cancer is attributable to tobacco smoke exposure, many studies have identified genetic variation that influences lung cancer risk (<a href="#CIT0001" rid="CIT0001" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588875">1–12</a>). Because cancers arise through an accumulation of deleterious somatic mutations, environmental or inherited risk factors due to inherited deleterious germline genetic variants are critical (<a href="#CIT0012" rid="CIT0012" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588872">12</a>).</p><p id="__p5">The Genetic Epidemiology of Lung Cancer Consortium (GELCC) has conducted the only family-based lung cancer linkage study to identify rare, highly penetrant lung cancer susceptibility genes, providing evidence of linkage on chromosomal regions 6q23-25, 18p11.23, 2p22.2, 14q13.1, 16p13 and 20q13.11 (<a href="#CIT0004" rid="CIT0004" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588867">4</a>,<a href="#CIT0005" rid="CIT0005" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588870">5</a>,<a href="#CIT0010" rid="CIT0010" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588863">10–13</a>). Linkage analysis is designed to identify highly penetrant uncommon variants but does not identify more common variants. Genome-wide association studies (GWAS) have provided highly significant and reproducible results associating common genetic variations to lung cancer. For example, a locus on chromosome 15q25, spanning a neuronal nicotinic acetylcholine receptor gene cluster comprising <em>CHRNA3, CHRNA5</em> and <em>CHRNB4</em> subunits, is significantly associated with lung cancer risk (<a href="#CIT0005" rid="CIT0005" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588864">5</a>,<a href="#CIT0007" rid="CIT0007" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588849">7</a>,<a href="#CIT0014" rid="CIT0014" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588858">14</a>). Regions on chromosomes 6p21 (<em>BAG6, MSH5</em>), 5p15 (<em>TERT, CLPTM1L),</em> 13q <em>(BRCA2</em>) and 22q <em>(CHEK2</em>) have also been identified from GWAS as being consistently associated with histology-specific lung cancer risk (<a href="#CIT0005" rid="CIT0005" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588859">5</a>,<a href="#CIT0006" rid="CIT0006" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588847">6</a>,<a href="#CIT0011" rid="CIT0011" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588837">11</a>,<a href="#CIT0015" rid="CIT0015" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588874">15</a>).</p><p id="__p6" class="p p-last">To further identify genetic variation associated with familial lung cancer risk, we conducted a GWAS meta-analysis focused on familial lung cancer cases. Familial lung cancer cases used in the analysis came from the GELCC family-based study and from the Transdisciplinary Research in Cancer of the Lung (<a href="#CIT0016" rid="CIT0016" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588850">16</a>) and the International Lung Cancer Consortium (TRICL-ILCCO) (<a href="#CIT0011" rid="CIT0011" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588865">11</a>). Lung cancer genome-wide association studies enriched for familial lung cancer cases represent a novel application of GWAS methodology to a population most likely to have a genetic component to their risk. Selecting cases with a family history improves the power to detect uncommon variants that are likely to have a larger inherited impact on lung cancer risk (<a href="#CIT0012" rid="CIT0012" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588845">12</a>,<a href="#CIT0017" rid="CIT0017" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588851">17</a>).</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">Materials and methods</h2><div id="s3" class="sec sec-first"><h3 id="s3title">GELCC study subjects</h3><p id="__p7" class="p p-first">The GELCC enrolled lung cancer cases with a family history of lung cancer from the following data collection sites: Karmanos Cancer Institute/Wayne State University, University of Cincinnati College of Medicine, Mayo Clinic, Medical College of Toledo, and Louisiana State University Health Sciences Center—New Orleans. Local Institutional Review Boards approved all protocols and each participant provided written informed consent. Familial lung cancer cases (ICD-9 162.0–162.9) were defined as individuals with a histologically confirmed diagnosis of lung cancer and at least one first or second degree relative with lung cancer. Family history information was collected by interview. The diagnosis of lung cancer in both the index case and relatives was verified with medical records, tumor registries and death certificates when possible. For the GWAS, one lung cancer case from each of 685 families of European descent with at least two first or second-degree relatives with lung cancer was selected. 744 controls of European descent were selected from persons related by marriage to lung cancer cases or from the general population in which the cases were identified, and did not have a lung cancer diagnosis and had no first or second-degree relatives with lung cancer (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 1</a>, available at <em>Carcinogenesis</em> Online). Controls were frequency matched to cases on age (5-year groups), sex and smoking status as closely as possible. The detailed demographic information on GELCC study is presented in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplemented Table 2</a>, available at <em>Carcinogenesis</em> Online.</p><p id="__p8" class="p p-last">Participation included completing a questionnaire (either by phone, in the clinic, or by mail), providing a blood or buccal/saliva sample for DNA, and providing access to medical records for confirmation of diagnosis and histology type. The study questionnaire included basic demographics, health history including history of other lung diseases, smoking history, and family history of cancer. Clinical data about diagnostic procedures, stage of disease, and histology type were obtained from medical records.</p></div><div id="s4" class="sec"><h3 id="s4title">GELCC study genotyping</h3><p id="__p9" class="p p-first-last">DNA samples were obtained from several sources including blood (68.2%), mouthwash (0.3%), saliva (2.3%) and whole genome amplification from blood (29.2%). Germline DNA was genotyped at the Center for Inherited Disease Research (CIDR) using the Illumina Human OmniExpressExome-8v1 array, designed to human genotype build 37. The GenTrain Version 1.0 calling algorithm in Genome Studio version 2011.1 was used, as was zCall for post-processing rare variant calling. Subsequent quality control steps were conducted collaboratively between the Genetic Analysis Center at the University of Washington in Seattle and the statistical group at Dartmouth College. The array included 18 642 pairs and 51 trios of duplicated SNPs. Where SNPs were duplicated, the SNP with the lowest missing rate was retained for analysis. In the analysis of the array, 49 blind duplicate samples were analyzed of which 31 were study-specific duplicates and 18 were derived from HapMap referent panel. One HapMap control and one internal control were included on each 96-well plate that was analyzed. The median discordance rate for genotypes for these 49 blind duplicates was 1.7 × 10<sup>–5</sup>. The genotyped HapMap samples included five trios, and of 936 945 unique markers, there were 1008 markers with one Mendelian error and 75 with two or more errors all of which were excluded from further analysis. A selected set of 907 markers from the whole genome amplified samples analyzed at the Mayo Clinic, one of the participating sites, provided different genotype frequencies from other participating sites. Results from the Mayo Clinic for those specific SNPs were removed from analyses. The median call rate for genotypes on this array was 99.9% and the error rate assessed from 31 pairs of study sample duplicates was 8.5 × 10<sup>–6</sup>. All genotyped samples had a call rate of 97% or higher and were retained for further quality control checks. Relatively few SNPs were removed due to quality control steps, but on this array 14.47% of SNPs were monomorphic and were therefore removed from analyses (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 3</a>, available at <em>Carcinogenesis</em> Online). Of the 951,117 SNPs assayed, 29% had minor allele frequencies of 1% or less. The detailed QC processes are presented in <a href="#sup3" rid="sup3" class=" supplementary-material">Supplementary Figure 1</a>, available at <em>Carcinogenesis</em> Online.</p></div><div id="s5" class="sec"><h3 id="s5title">Imputation in GELCC study</h3><p id="__p10" class="p p-first-last">We performed genotype imputation in the familial lung cancer study, using the software package IMPUTE2 v2.3.0 (<a href="#CIT0018" rid="CIT0018" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588877">18</a>) and 1000 Genomes Project Phase 1 as a reference panel, which contains whole genome sequence data from 1092 individuals including approximately 39 million variants, insertions/deletions, and structural variants. Imputation was done to 1000 Genomes Project Phase 1 version 3 integrated haplotypes produced with SHAPEIT2 for chromosome 1–22; for chromosome X, the previous version was used. The study samples were imputed in a two-stage procedure using SHAPEIT2 (version 2.r644, c2011-2012) to derive phased genotypes from the array SNPs and IMPUTE2 (version 2.3.0) to perform imputation of the phased data. We used the default parameters to derive phased genotypes with SHAPEIT2, increasing the number of conditioning haplotypes to be used during the phasing process to 200 (“--states 200 or –S 200”). The imputation was performed using 5MB segments over the length of each chromosome from the first to last position appearing in the reference panel. The flag “-use_prephased_g” was provided to indicate that prephased haplotypes were being used and the buffer region was increased to 500kb (“-buffer 500”). Imputation target variants were limited to those with at least two copies of the minor allele observed in any of the four reference panel super-populations, yielding 27 728, 264 total variants in the imputed dataset.</p></div><div id="s6" class="sec"><h3 id="s6title">TRICL-ILCCO Data</h3><p id="__p11" class="p p-first-last">The Transdisciplinary Research in Cancer of the Lung (<a href="#CIT0016" rid="CIT0016" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588854">16</a>) and the International Lung Cancer Consortium (TRICL-ILCCO) is an international consortium that has assembled and analyzed GWAS data from lung cancer cases and controls. Among the studies that have collected data for TRICL-ILCCO, a subset had information about the presence of lung cancer in a first degree relative. From TRICL-ILCCO, 1,308 cases with a family history of lung cancer in a first degree relative from six studies were included (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 2</a>, available at <em>Carcinogenesis</em> Online). An additional sample of 32 946 controls was available for analysis from the TRICL data. In all (GELCC plus TRICL-ILCCO), 1993 familial cases and 33 690 controls were included (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 1</a>, available at <em>Carcinogenesis</em> Online). The same referent panel and procedures were used for the United States, Canadian and French sites for TRICL-ILCCO as were used for GELCC. The deCODE analysis included a referent set derived from Icelandic individuals who have been sequenced; it therefore provides more accurate imputation for rare variants than imputation analysis used by other sites.</p></div><div id="s7" class="sec"><h3 id="s7title">Imputation in TRICL studies</h3><p id="__p12" class="p p-first-last">Whole genome imputations for the MD Anderson Cancer Center (MDACC), deCODE, Samuel Lunenfeld Research Institute (SLRI) Study, Environmental and Genetics in Lung Cancer Etiology (EAGLE), and Prostate, Lung, Colon, and Ovary (PLCO) screening studies in TRICL-ILCCO consortium were conducted using IMPUTE2 v2.5 based on the 1000 Genomes Project Phase 1. For the International Agency for Research on Cancer (IARC) study imputation was performed using minimac (version 2012.10.3) based on 1000 Genome Project Phase 1 For MDACC, SLRI, EAGLE and PLCO studies, we used the default parameters to derive phased genotypes with SHAPEIT, increasing: the number of burn-in iterations used by the algorithm to reach a good starting point to 10 (“--burn 10”), the number of pruning iterations to find a parsimonious graph for each individual to 10 (“--prune 10”), and the number of iterations to compute transition probabilities in haplotype graphs to 50 (“--main 50”). We conducted imputation with IMPUTE2 using 5MB non-overlapping intervals. The flag “-use_prephased_g” was applied to indicate the prephased haplotypes were being used. The missing genotypes were replaced with imputed genotypes using the option “-pgs_miss” and the buffer region was increased to 500KB (“-buffer 500”). <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 4</a>, available at <em>Carcinogenesis</em> Online, provides the details for the number of SNPs that passed quality control on each study.</p></div><div id="s8" class="sec"><h3 id="s8title">Statistical analysis</h3><p id="__p13" class="p p-first-last">Using genotyped data from 1429 subjects of European descent in the GELCC study, we applied the EIGENSTRAT method to carry out principal components analysis. To assure robustness of our analytical approach, we performed association analyses adjusting for the top three eigenvectors based on the principal components analysis. Logistic regression was applied to evaluate the odds ratio per allele of each SNP, adjusting for age, sex and eigenvectors. Analyses were conducted using PLINK.</p></div><div id="s9" class="sec"><h3 id="s9title">Meta-Analysis</h3><p id="__p14" class="p p-first-last">Meta-analysis of the GELCC and TRICL-ILCCO studies was conducted using METAL software (<a href="http://csg.sph.umich.edu/abecasis/metal/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://csg.sph.umich.edu/abecasis/metal/index.html</a>) that combined test statistics and standard errors across studies (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 5</a>, available at <em>Carcinogenesis</em> Online). In addition, we determined whether there were differences between case groups according to the most common histologic subtypes of lung cancer: squamous (<em>n</em> = 367) and adenocarcinoma (<em>n</em> = 775). For the overall meta-analysis, GELCC and 6 TRICL-ILCCO studies were used (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 6A</a>, available at <em>Carcinogenesis</em> Online). Histology specific meta-analysis was performed on GELCC and 5 TRICL-ILCCO studies (excluding the IARC study) (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Tables 6B</a> and <a href="#sup1" rid="sup1" class=" supplementary-material">C</a>, available at <em>Carcinogenesis</em> Online). For some histology specific analyses, some populations contributed small numbers of cases, a situation where rare variants can create unstable estimates of the effect size. Therefore, we excluded markers with imputation quality info score less than 0.5 and removed results when Odd Ratios (OR) were greater than 100 or less than 0.01 for each study to reduce false positive findings. Common SNPs that were present in at least five studies including GELCC plus at least four studies out of five TRICL-ILCCO studies were used for histology specific analyses. We also did not consider SNPs when the Mantel–Haenszel heterogeneity <em>P</em>-value test was ≤ 0.05. The lambda values showed minimal inflation except for squamous carcinoma, 1.015 and 1.004, respectively, before and after adjustment to reflect a standardized sample size of 1000 cases and 1000 controls (since lambda increases with sample size). Lambda values for histology-specific analyses were 1.040 and 1.026 (adenocarcinoma) and 1.079 and 1.108 (squamous cell) before and after correction based on 1000 cases and 1000 controls, respectively. Q-Q plots are presented in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure 2</a>, available at <em>Carcinogenesis</em> Online. We used GCTA to evaluate joint and conditional models with chromosome 15q21.1 region markers (<a href="#CIT0019" rid="CIT0019" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588879">19</a>,<a href="#CIT0020" rid="CIT0020" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588842">20</a>).</p></div><div id="s10" class="sec sec-last"><h3 id="s10title">Lung expression QTL analysis</h3><p id="__p15" class="p p-first-last">Whole-genome gene expression profiling in 409 adjacent normal lung samples obtained from patients undergoing resection for lung cancer therapy was performed on a custom Affymetrix array (<a id="__tag_724588881" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GPL10379" ref="/geo/query/acc.cgi?acc=GPL10379"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GPL10379","term_id":"10379"}}</span>GPL10379</a>). Microarray pre-processing and quality controls were described previously (<a href="#CIT0021" rid="CIT0021" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588868">21</a>,<a href="#CIT0022" rid="CIT0022" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588857">22</a>). Genotyping was carried out on the Illumina Human 1M-Duo BeadChip array. Expression traits were adjusted for age, sex and smoking status. <em>Cis</em>-eQTL were evaluated for transcripts (probe sets) located 1 Megabase up- and downstream of the sentinel lung cancer associated SNPs. Association tests were carried out using PLINK version 1.9. For the chromosome 9 region transcripts that were evaluated included <em>IFNA</em> gene cluster and the <em>CDKN2A/CDKN2B</em> region.</p></div></div><div id="s11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s11title">Results</h2><p id="__p16" class="p p-first">In the meta-analysis of the GELCC and 6 TRICL-ILCCO studies, 44 and 41 variants (<a href="#sup2" rid="sup2" class=" supplementary-material">Supplementary Tables 7A</a> and <a href="#sup2" rid="sup2" class=" supplementary-material">B</a>, available at <em>Carcinogenesis</em> Online) were identified as associated with overall lung cancer and squamous cell carcinoma lung cancer (367 cases) in family history positive lung cancer cases, respectively, at a genome-wide level of significance (<em>P</em> &lt; 5 × 10<sup>–8</sup>) (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure 3</a>, available at <em>Carcinogenesis</em> Online). No variants were identified for adenocarcinoma (775 cases) at a genome-wide significant level. In the analysis of overall lung cancers, the top hits (<em>P</em> &lt; 5 × 10<sup>-8</sup>) fall primarily within the <em>CHRNA3/CHRNA5/CHRNB4</em> nicotinic acetylcholine receptor gene region on chromosome 15 (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure 4A</a>, available at <em>Carcinogenesis</em> Online) (<a href="#CIT0023" rid="CIT0023" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588853">23</a>). The most significant SNP on chromosome15, rs8040868, is in the region of <em>CHRNA3.</em> As presented in <a href="/pmc/articles/PMC6148967/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, the estimate of risk associated with this relatively common SNP (MAF = 0.42) in familial lung cancer is OR = 1.33 (<em>P</em> = 4.38 × 10<sup>−13</sup>). Even though this is a synonymous change, statistically significant changes in gene expression for <em>CHRNA3</em>, as well as <em>CHRNA5</em> and <em>PSMA4</em>, are associated with this polymorphism, based on expression quantitative trait analysis (eQTL) from 409 noncancerous lung tissues removed for lung cancer resections, providing additional support for the role of this polymorphism or another variant in linkage disequilibrium with it in influencing lung cancer risk (<a href="/pmc/articles/PMC6148967/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure 5</a>, available at <em>Carcinogenesis</em> Online).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p17">Meta-analysis association test results for top SNPs, by histology type</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139670476024208" class="xtable"><table frame="vsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Strata</th><th rowspan="1" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="left">Chr</th><th rowspan="1" colspan="1" valign="bottom" align="left">Position</th><th rowspan="1" colspan="1" valign="bottom" align="left">Nearest Gene</th><th rowspan="1" colspan="1" valign="bottom" align="left">Function</th><th rowspan="1" colspan="1" valign="bottom" align="left">Allele<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">EAF</th><th rowspan="1" colspan="1" valign="bottom" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">
<em>P</em>-value</th><th rowspan="1" colspan="1" valign="bottom" align="left">Mean of QC Info Score</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Overall</td><td rowspan="1" colspan="1">rs8040868</td><td rowspan="1" colspan="1">15q25.1</td><td rowspan="1" colspan="1">78911181</td><td rowspan="1" colspan="1">
<em>CHRNA3</em>
</td><td rowspan="1" colspan="1">Synonymous (V-V)</td><td rowspan="1" colspan="1">T_C</td><td rowspan="1" colspan="1">0.42</td><td rowspan="1" colspan="1">1.33 (1.23, 1.43)</td><td rowspan="1" colspan="1">4.38 × 10<sup>−13</sup></td><td rowspan="1" colspan="1">0.99</td></tr><tr><td rowspan="2" colspan="1">SQC</td><td rowspan="1" colspan="1">rs71603396</td><td rowspan="1" colspan="1">4p15.31</td><td rowspan="1" colspan="1">18012035</td><td rowspan="1" colspan="1">
<em>LCORL</em>
</td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1">G_A</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">1.92 (1.55, 2.37)</td><td rowspan="1" colspan="1">1.73 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">0.88</td></tr><tr><td rowspan="1" colspan="1">rs12686364</td><td rowspan="1" colspan="1">9p21.3</td><td rowspan="1" colspan="1">21670087</td><td rowspan="1" colspan="1">
<em>CDKN2A</em>
</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">A_G</td><td rowspan="1" colspan="1">0.004</td><td rowspan="1" colspan="1">8.49 (4.13, 17.46)</td><td rowspan="1" colspan="1">6.21 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">0.96</td></tr></tbody></table></div><div id="largeobj_idm139670476024208" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6148967/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="fn-00011"><p id="__p18" class="p p-first-last">OR, odds (log additive) ratio; 95% CI, 95% confidence interval; SQC, squamous cell carcinoma.</p></div><div id="fn-0002"><p id="__p19" class="p p-first-last">
<sup>a</sup>Denotes reference_effect(coded) allele; Genome positions relative to GRCh37; EAF, effective allele frequency.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p20">Gene expression QTL analysis in the lung for the three top SNPs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139670515805088" class="xtable"><table frame="vsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="left">Chr</th><th rowspan="1" colspan="1" valign="bottom" align="left">Major allele</th><th rowspan="1" colspan="1" valign="bottom" align="left">Minor allele</th><th rowspan="1" colspan="1" valign="bottom" align="left">MAF</th><th rowspan="1" colspan="1" valign="bottom" align="left"># probes tested</th><th rowspan="1" colspan="1" valign="bottom" align="left">ProbeSet</th><th rowspan="1" colspan="1" valign="bottom" align="left">Gene</th><th rowspan="1" colspan="1" valign="bottom" align="left">BETA</th><th rowspan="1" colspan="1" valign="bottom" align="left">SE</th><th rowspan="1" colspan="1" valign="bottom" align="left">
<em>P</em>-value<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">% variance</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs8040868</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1">0.474</td><td rowspan="1" colspan="1">45</td><td rowspan="1" colspan="1">100154936_IF147302_TGI_at</td><td rowspan="1" colspan="1">
<em><a id="__tag_724588940" class="tag_hotlink" href="/nuccore/AF147302" ref="/nuccore/AF147302"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AF147302","term_id":"4761653","term_text":"AF147302"}}</span>AF147302</a></em>
</td><td rowspan="1" colspan="1">−0.235</td><td rowspan="1" colspan="1">0.023</td><td rowspan="1" colspan="1">7.84 × 10<sup>−22</sup></td><td rowspan="1" colspan="1">20.3</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1">100154936_IREB2_TGI_at</td><td rowspan="1" colspan="1">
<em>IREB2</em>
</td><td rowspan="1" colspan="1">−0.244</td><td rowspan="1" colspan="1">0.027</td><td rowspan="1" colspan="1">4.23 × 10<sup>−18</sup></td><td rowspan="1" colspan="1">16.9</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1">100156434_TGI_at</td><td rowspan="1" colspan="1">
<em>CHRNA5</em>
</td><td rowspan="1" colspan="1">−0.121</td><td rowspan="1" colspan="1">0.019</td><td rowspan="1" colspan="1">3.43 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">9.3</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1">100148404_TGI_at</td><td rowspan="1" colspan="1">
<em>PSMA4</em>
</td><td rowspan="1" colspan="1">0.059</td><td rowspan="1" colspan="1">0.014</td><td rowspan="1" colspan="1">1.9 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">4.4</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1">100123649_TGI_at</td><td rowspan="1" colspan="1">
<em>CHRNA3</em>
</td><td rowspan="1" colspan="1">−0.151</td><td rowspan="1" colspan="1">0.039</td><td rowspan="1" colspan="1">1.5 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">3.5</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1">100312267_TGI_at</td><td rowspan="1" colspan="1">
<em><a id="__tag_724588941" class="tag_hotlink" href="/nuccore/AJ584709" ref="/nuccore/AJ584709"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AJ584709","term_id":"50199108","term_text":"AJ584709"}}</span>AJ584709</a></em>
</td><td rowspan="1" colspan="1">−0.109</td><td rowspan="1" colspan="1">0.032</td><td rowspan="1" colspan="1">0.001</td><td rowspan="1" colspan="1">2.7</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1">100143446_TGI_at</td><td rowspan="1" colspan="1">
<em>IREB2</em>
</td><td rowspan="1" colspan="1">−0.066</td><td rowspan="1" colspan="1">0.023</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">1.9</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1">100145993_TGI_at</td><td rowspan="1" colspan="1">
<em>CHRNA3</em>
</td><td rowspan="1" colspan="1">0.037</td><td rowspan="1" colspan="1">0.015</td><td rowspan="1" colspan="1">0.012</td><td rowspan="1" colspan="1">1.5</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1">100154605_TGI_at</td><td rowspan="1" colspan="1">
<em>CRABP1</em>
</td><td rowspan="1" colspan="1">0.078</td><td rowspan="1" colspan="1">0.035</td><td rowspan="1" colspan="1">0.025</td><td rowspan="1" colspan="1">1.3</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1">100142282_TGI_at</td><td rowspan="1" colspan="1">
<em>TBC1D2B</em>
</td><td rowspan="1" colspan="1">−0.027</td><td rowspan="1" colspan="1">0.013</td><td rowspan="1" colspan="1">0.031</td><td rowspan="1" colspan="1">1.2</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1">100135897_TGI_at</td><td rowspan="1" colspan="1">Not known</td><td rowspan="1" colspan="1">0.071</td><td rowspan="1" colspan="1">0.033</td><td rowspan="1" colspan="1">0.032</td><td rowspan="1" colspan="1">1.1</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1">100126596_TGI_at</td><td rowspan="1" colspan="1">
<em>DNAJA4</em>
</td><td rowspan="1" colspan="1">0.057</td><td rowspan="1" colspan="1">0.028</td><td rowspan="1" colspan="1">0.044</td><td rowspan="1" colspan="1">1.0</td></tr><tr><td rowspan="1" colspan="1">rs71603396</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1">0.113</td><td rowspan="1" colspan="1">21</td><td rowspan="1" colspan="1">100160526_TGI_at</td><td rowspan="1" colspan="1">
<em>MED28</em>
</td><td rowspan="1" colspan="1">−0.063</td><td rowspan="1" colspan="1">0.026</td><td rowspan="1" colspan="1">0.01498</td><td rowspan="1" colspan="1">1.5</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1">100142682_TGI_at</td><td rowspan="1" colspan="1">
<em><a id="__tag_724588938" class="tag_hotlink" href="/nuccore/LF383968" ref="/nuccore/LF383968"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"LF383968","term_id":"1093726742","term_text":"LF383968"}}</span>LF383968</a></em>
</td><td rowspan="1" colspan="1">−0.128</td><td rowspan="1" colspan="1">0.054</td><td rowspan="1" colspan="1">0.01769</td><td rowspan="1" colspan="1">1.4</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1">100158710_TGI_at</td><td rowspan="1" colspan="1">
<em><a id="__tag_724588939" class="tag_hotlink" href="/nuccore/BX648830" ref="/nuccore/BX648830"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"BX648830","term_id":"34367995","term_text":"BX648830"}}</span>BX648830</a></em>
</td><td rowspan="1" colspan="1">0.082</td><td rowspan="1" colspan="1">0.039</td><td rowspan="1" colspan="1">0.03589</td><td rowspan="1" colspan="1">1.1</td></tr><tr><td rowspan="1" colspan="1">rs12686364</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1">0.0049</td><td rowspan="1" colspan="1">38</td><td rowspan="1" colspan="1">100144400_TGI_at</td><td rowspan="1" colspan="1">
<em>IFNA8</em>
</td><td rowspan="1" colspan="1">−0.390</td><td rowspan="1" colspan="1">0.191</td><td rowspan="1" colspan="1">0.04117</td><td rowspan="1" colspan="1">1.0</td></tr></tbody></table></div><div id="largeobj_idm139670515805088" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6148967/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="fn-0003"><p id="__p21" class="p p-first-last">
<sup>a</sup>Only genes with statistically significant changes in gene expression associated with SNP genotypes are presented.</p></div></div></div><p id="__p22">In histology-specific analyses, two of the most significant associations were with familial lung squamous cell carcinoma. The first, rs71603396 on chromosome 4p15.31 is in the region of the ligand dependent nuclear receptor corepressor (<em>LCORL</em>) gene (MAF = 0.14, OR = 1.92, <em>P</em> = 1.73 × 10<sup>−9</sup>) (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure 4B</a>, available at <em>Carcinogenesis</em> Online). This SNP rs71603396 in the <em>LCORL</em> gene was associated with gene expression in eQTL analysis (<a href="/pmc/articles/PMC6148967/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The second significant SNP, rs12686364, is located in an intergenic region (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure 4C</a>, available at <em>Carcinogenesis</em> Online, on chromosome 9p21.3 (MAF = 0.004, OR = 8.49, <em>P</em> = 6.21 × 10<sup>−9</sup>) that is upstream of <em>IFNA8</em> and downstream of <em>CDKN2A</em>). eQTL analysis demonstrated an association between this SNP and expression of interferon-α8 (<em>IFNA8</em>) (<a href="/pmc/articles/PMC6148967/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Results by study center are presented in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 8</a>, available at <em>Carcinogenesis</em> Online.</p><p id="__p23">
<a href="/pmc/articles/PMC6148967/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> presents results for previously identified SNPs that influence lung cancer risk in European descent individuals and that had previously reached genome-wide levels of significance. Of the previously identified SNPs, only the rs55781567 variant on chromosome 15q25.1 reached a genome-wide level of significance. The nicotinic acetylcholine region of chromosome 15q25.1 has consistently been the most strongly associated region for lung cancer risk among unselected cases of European descent lung cancer. We further examined the two SNPs on 15q25.1; rs8040868 from familial lung cancer and rs55781567 from unselected lung cancer, which are 53,195 bp apart. The linkage disequilibrium between the two SNPs is D’ = 0.95 (<em>R</em><sup>2</sup> = 0.73), indicated that these SNPs are in high linkage disequilibrium. We, therefore, performed the conditional analysis to evaluate whether rs8040868 (<em>P</em> =4.38 × 10<sup>−13</sup>) has association signal conditioning on rs55781567 (<em>P</em> =1.69 × 10<sup>−11</sup>) (<a href="#CIT0022" rid="CIT0022" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588838">22</a>). The conditional test of rs8040868 after adjusting for rs55781567 yielded <em>P</em> = 3.4 × 10<sup>−3</sup>. This indicates that rs8040868 still has association signal for familial lung cancer risk. While rs8040868 provided an independent signal, many SNPs in this region are strongly correlated and the signal from rs804868 could reflect effects from another causal variant in the region such as rs16969968, a causal SNP in CHRNA5. Conditional analysis of rs804868 allowing for effects from rs16969968 also showed a residual effect with <em>P</em> = 0.002. On chromosome 9p21.3, the effects between the previously identified SNP rs885518 and the newly identified SNP rs12686364 was very low (<em>R</em><sup>2</sup> = 0.007, D’ = 0.452), with the very low <em>R</em><sup>2</sup> value reflecting very different allele frequencies of the two variants. After conditioning on the previously identified SNP rs885518, which predisposes to adenocarcinoma, the association of rs12686364 remained highly significant (<em>P</em> = 5.93 × 10<sup>−9</sup>), supporting completely independent effects of these two SNPs on lung cancer risk.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p24">Comparison to familial lung cancer study using the recently published susceptible loci in OncoArray lung cancer</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139670483457168" class="xtable"><table frame="vsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Strata</th><th rowspan="1" colspan="1" valign="bottom" align="left">Chromosome</th><th rowspan="1" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="left">Position</th><th rowspan="1" colspan="1" valign="bottom" align="left">Gene</th><th rowspan="1" colspan="1" valign="bottom" align="left">Allele<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">EAF</th><th rowspan="1" colspan="1" valign="bottom" align="left">OR (95%CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">
<em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Lung</td><td rowspan="1" colspan="1">1p31.1</td><td rowspan="1" colspan="1">rs71658797</td><td rowspan="1" colspan="1">77967507</td><td rowspan="1" colspan="1">
<em>FUBP1</em>
</td><td rowspan="1" colspan="1">T_A</td><td rowspan="1" colspan="1">0.099</td><td rowspan="1" colspan="1">1.12 (0.99–1.27)</td><td rowspan="1" colspan="1">7.54 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1">Lung</td><td rowspan="1" colspan="1">6q27</td><td rowspan="1" colspan="1">rs6920364</td><td rowspan="1" colspan="1">167376466</td><td rowspan="1" colspan="1">
<em>RNASET2</em>
</td><td rowspan="1" colspan="1">G_C</td><td rowspan="1" colspan="1">0.449</td><td rowspan="1" colspan="1">1.08 (1.00–1.17)</td><td rowspan="1" colspan="1">5.25 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1">
<strong>Lung</strong>
</td><td rowspan="1" colspan="1">
<strong>8p21.1</strong>
</td><td rowspan="1" colspan="1">
<strong>rs11780471</strong>
</td><td rowspan="1" colspan="1">
<strong>27344719</strong>
</td><td rowspan="1" colspan="1">
<strong><em>EPHX2, CHRNA2</em></strong>
</td><td rowspan="1" colspan="1">
<strong>G_A</strong>
</td><td rowspan="1" colspan="1">
<strong>0.063</strong>
</td><td rowspan="1" colspan="1">
<strong>0.80 (0.67</strong>–<strong>0.95</strong>)</td><td rowspan="1" colspan="1">
<strong>1.00</strong> × <strong>10</strong><sup>−<strong>2</strong></sup></td></tr><tr><td rowspan="1" colspan="1">
<strong>Lung</strong>
</td><td rowspan="1" colspan="1">
<strong>13q13.1</strong>
</td><td rowspan="1" colspan="1">
<strong>rs11571833</strong>
</td><td rowspan="1" colspan="1">
<strong>32972626</strong>
</td><td rowspan="1" colspan="1">
<strong><em>BRCA2</em></strong>
</td><td rowspan="1" colspan="1">
<strong>A_T</strong>
</td><td rowspan="1" colspan="1">
<strong>0.010</strong>
</td><td rowspan="1" colspan="1">
<strong>1.66 (1.14</strong>–<strong>2.41</strong>)</td><td rowspan="1" colspan="1">
<strong>7.78</strong> × <strong>10</strong><sup>−<strong>3</strong></sup></td></tr><tr><td rowspan="1" colspan="1">Lung</td><td rowspan="1" colspan="1">15q21.1</td><td rowspan="1" colspan="1">rs66759488</td><td rowspan="1" colspan="1">47577451</td><td rowspan="1" colspan="1">
<em>SEMA6D</em>
</td><td rowspan="1" colspan="1">G_A</td><td rowspan="1" colspan="1">0.362</td><td rowspan="1" colspan="1">1.05 (0.97–1.14)</td><td rowspan="1" colspan="1">1.91 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">
<strong>Lung</strong>
</td><td rowspan="1" colspan="1">
<strong>15q25.1</strong>
</td><td rowspan="1" colspan="1">
<strong>rs55781567</strong>
</td><td rowspan="1" colspan="1">
<strong>78857986</strong>
</td><td rowspan="1" colspan="1">
<strong><em>CHRNA5</em></strong>
</td><td rowspan="1" colspan="1">
<strong>C_G</strong>
</td><td rowspan="1" colspan="1">
<strong>0.366</strong>
</td><td rowspan="1" colspan="1">
<strong>1.31 (1.21</strong>–<strong>1.41</strong>)</td><td rowspan="1" colspan="1">
<strong>1.69</strong> × <strong>10</strong><sup>−<strong>11</strong></sup></td></tr><tr><td rowspan="1" colspan="1">
<strong>Lung</strong>
</td><td rowspan="1" colspan="1">
<strong>19q13.2</strong>
</td><td rowspan="1" colspan="1">
<strong>rs56113850</strong>
</td><td rowspan="1" colspan="1">
<strong>41353107</strong>
</td><td rowspan="1" colspan="1">
<strong><em>CYP2A6</em></strong>
</td><td rowspan="1" colspan="1">
<strong>C_T</strong>
</td><td rowspan="1" colspan="1">
<strong>0.429</strong>
</td><td rowspan="1" colspan="1">
<strong>0.91 (0.83</strong>–<strong>0.99</strong>)</td><td rowspan="1" colspan="1">
<strong>2.86</strong> × <strong>10</strong><sup>−<strong>2</strong></sup></td></tr><tr><td rowspan="1" colspan="1">Adeno</td><td rowspan="1" colspan="1">3q28</td><td rowspan="1" colspan="1">rs13080835</td><td rowspan="1" colspan="1">189357199</td><td rowspan="1" colspan="1">
<em>TP63</em>
</td><td rowspan="1" colspan="1">G_T</td><td rowspan="1" colspan="1">0.489</td><td rowspan="1" colspan="1">0.96 (0.86–1.08)</td><td rowspan="1" colspan="1">5.28 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">
<strong>Adeno</strong>
</td><td rowspan="1" colspan="1">
<strong>5p15.33</strong>
</td><td rowspan="1" colspan="1">
<strong>rs7705526</strong>
</td><td rowspan="1" colspan="1">
<strong>1285974</strong>
</td><td rowspan="1" colspan="1">
<strong><em>TERT</em></strong>
</td><td rowspan="1" colspan="1">
<strong>C_A</strong>
</td><td rowspan="1" colspan="1">
<strong>0.343</strong>
</td><td rowspan="1" colspan="1">
<strong>1.28 (1.13</strong>–<strong>1.46</strong>)</td><td rowspan="1" colspan="1">
<strong>1.45</strong> × <strong>10</strong><sup>−<strong>4</strong></sup></td></tr><tr><td rowspan="1" colspan="1">Adeno</td><td rowspan="1" colspan="1">8p12</td><td rowspan="1" colspan="1">rs4236709</td><td rowspan="1" colspan="1">32410110</td><td rowspan="1" colspan="1">
<em>NRG1</em>
</td><td rowspan="1" colspan="1">A_G</td><td rowspan="1" colspan="1">0.225</td><td rowspan="1" colspan="1">1.10 (0.96–1.26)</td><td rowspan="1" colspan="1">1.56 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">
<strong>Adeno</strong>
</td><td rowspan="1" colspan="1">
<strong>9p21.3</strong>
</td><td rowspan="1" colspan="1">
<strong>rs885518</strong>
</td><td rowspan="1" colspan="1">
<strong>21830157</strong>
</td><td rowspan="1" colspan="1">
<strong><em>MTAP/CDNK2A</em></strong>
</td><td rowspan="1" colspan="1">
<strong>A_G</strong>
</td><td rowspan="1" colspan="1">
<strong>0.110</strong>
</td><td rowspan="1" colspan="1">
<strong>1.37 (1.16</strong>–<strong>1.62</strong>)</td><td rowspan="1" colspan="1">
<strong>2.20</strong> × <strong>10</strong><sup>−<strong>4</strong></sup></td></tr><tr><td rowspan="1" colspan="1">Adeno</td><td rowspan="1" colspan="1">10q24.3</td><td rowspan="1" colspan="1">rs11591710</td><td rowspan="1" colspan="1">1.06E+08</td><td rowspan="1" colspan="1">
<em>OBFC1</em>
</td><td rowspan="1" colspan="1">A_C</td><td rowspan="1" colspan="1">0.143</td><td rowspan="1" colspan="1">1.14 (0.97–1.33)</td><td rowspan="1" colspan="1">1.21 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">Adeno</td><td rowspan="1" colspan="1">11q23.3</td><td rowspan="1" colspan="1">rs1056562</td><td rowspan="1" colspan="1">1.18E+08</td><td rowspan="1" colspan="1">
<em>MPZL3/AMICA1</em>
</td><td rowspan="1" colspan="1">C_T</td><td rowspan="1" colspan="1">0.471</td><td rowspan="1" colspan="1">1.08 (0.96–1.21)</td><td rowspan="1" colspan="1">1.89 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">
<strong>Adeno</strong>
</td><td rowspan="1" colspan="1">
<strong>15q21.1</strong>
</td><td rowspan="1" colspan="1">
<strong>rs77468143</strong>
</td><td rowspan="1" colspan="1">
<strong>49376624</strong>
</td><td rowspan="1" colspan="1">
<strong><em>SECISBP2L</em></strong>
</td><td rowspan="1" colspan="1">
<strong>T_G</strong>
</td><td rowspan="1" colspan="1">
<strong>0.249</strong>
</td><td rowspan="1" colspan="1">
<strong>0.87 (0.76</strong>–<strong>0.99</strong>)</td><td rowspan="1" colspan="1">
<strong>4.00</strong> × <strong>10</strong><sup>−<strong>2</strong></sup></td></tr><tr><td rowspan="1" colspan="1">
<strong>Adeno</strong>
</td><td rowspan="1" colspan="1">
<strong>20q13.33</strong>
</td><td rowspan="1" colspan="1">
<strong>rs41309931</strong>
</td><td rowspan="1" colspan="1">
<strong>62326579</strong>
</td><td rowspan="1" colspan="1">
<strong><em>RTEL1</em></strong>
</td><td rowspan="1" colspan="1">
<strong>G_T</strong>
</td><td rowspan="1" colspan="1">
<strong>0.119</strong>
</td><td rowspan="1" colspan="1">
<strong>1.35 (1.14</strong>–<strong>1.60</strong>)</td><td rowspan="1" colspan="1">
<strong>6.14</strong> × <strong>10</strong><sup>−<strong>4</strong></sup></td></tr><tr><td rowspan="1" colspan="1">
<strong>SQC</strong>
</td><td rowspan="1" colspan="1">
<strong>6p21.33</strong>
</td><td rowspan="1" colspan="1">
<strong>rs116822326</strong>
</td><td rowspan="1" colspan="1">
<strong>31434111</strong>
</td><td rowspan="1" colspan="1">
<strong><em>MHC</em></strong>
</td><td rowspan="1" colspan="1">
<strong>A_G</strong>
</td><td rowspan="1" colspan="1">
<strong>0.161</strong>
</td><td rowspan="1" colspan="1">
<strong>1.26 (1.03</strong>–<strong>1.54</strong>)</td><td rowspan="1" colspan="1">
<strong>2.36</strong> × <strong>10</strong><sup>−<strong>2</strong></sup></td></tr><tr><td rowspan="1" colspan="1">
<strong>SQC</strong>
</td><td rowspan="1" colspan="1">
<strong>12p13.33</strong>
</td><td rowspan="1" colspan="1">
<strong>rs7953330</strong>
</td><td rowspan="1" colspan="1">
<strong>998819</strong>
</td><td rowspan="1" colspan="1">
<strong><em>RAD52</em></strong>
</td><td rowspan="1" colspan="1">
<strong>G_C</strong>
</td><td rowspan="1" colspan="1">
<strong>0.313</strong>
</td><td rowspan="1" colspan="1">
<strong>0.79 (0.66</strong>–<strong>0.94</strong>)</td><td rowspan="1" colspan="1">
<strong>7.05</strong> × <strong>10</strong><sup>−<strong>3</strong></sup></td></tr><tr><td rowspan="1" colspan="1">SQC</td><td rowspan="1" colspan="1">22q12.1</td><td rowspan="1" colspan="1">rs17879961</td><td rowspan="1" colspan="1">29121087</td><td rowspan="1" colspan="1">
<em>CHEK2</em>
</td><td rowspan="1" colspan="1">A_G</td><td rowspan="1" colspan="1">0.004</td><td rowspan="1" colspan="1">2.50 (0.76–8.15)</td><td rowspan="1" colspan="1">1.29 × 10<sup>−1</sup></td></tr></tbody></table></div><div id="largeobj_idm139670483457168" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC6148967/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="fn-0004"><p id="__p25" class="p p-first-last">EAF, effective allele frequency; OR, odds (log additive) ratio; 95% CI, 95% confidence interval; Adeno, adenocarcinoma; SQC, squamous cell carcinoma.</p></div><div id="fn-0005"><p id="__p26" class="p p-first-last">
<sup>a</sup>Denotes reference_effect(coded) allele. Variants in bold are associated with familial lung cancer at nominal significance level of 0.05. Genome positions relative to GRCh37.</p></div></div></div><p id="__p27" class="p p-last">
<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Tables 6A–C</a>, available at <em>Carcinogenesis</em> Online, present results stratified by analysis in GELCC alone, TRICL alone or combined. Results for the GELCC study only identified several additional variants that were not replicated on analysis of the data from the TRICL studies. Variants in <em>BCHE, PDGFRA</em> and <em>CFTR</em> genes were highly significant (<em>P</em> &lt; 5 × 10<sup>−7</sup>) for overall lung cancer, but all protective of developing lung cancer and did not replicate in the TRICL analysis. Similarly, highly significant associations (<em>P</em> &lt; 5 × 10<sup>−7</sup>) were noted for adenocarcinoma including the <em>GSG1L</em> and <em>ZC3H12D</em> genes and for squamous associations were noted for RP11-335M9.1, EPDR1, CNTN4 and a gene desert on chromosome 14, but none of these associations were supported when combined with the TRICL data.</p></div><div id="s12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s12title">Discussion</h2><p id="__p28" class="p p-first">Prior studies of familial lung cancer have identified rare variants that substantially increase lung cancer risk (<a href="#CIT0024" rid="CIT0024" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588841">24</a>,<a href="#CIT0025" rid="CIT0025" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588840">25</a>). Common genetic variants have been associated with sporadic lung cancer risk in GWAS studies (<a href="#CIT0026" rid="CIT0026" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_801873829">26</a>). The study presented is the largest study to use a GWAS approach to more broadly identify genetic risk in familial lung cancer. Selecting cases with a family history of disease increases the power to detect rare variants that increase cancer risk (<a href="#CIT0027" rid="CIT0027" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588856">27</a>). This study of familial lung cancer confirmed genetic variation on <em>CHRNA3</em> and <em>CHRNA5</em> on chromosome 15q25.1 previously reported in sporadic lung cancer. For all lung cancers, the top hits (<em>P</em> &lt; 5 × 10<sup>−8</sup>) fall primarily within the <em>CHRNA3/CHRNA5/CHRNB4</em> nicotinic acetylcholine receptor regions on chromosome 15. The most significant SNP on chromosome15, rs8040868, has shown to be associated with lung cancer risk regardless of family history (<a href="#CIT0016" rid="CIT0016" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588839">16</a>) (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Figure 4A</a>, available at <em>Carcinogenesis</em> Online). In European descent individuals, rs8040868 is very strongly associated (<em>R</em><sup>2</sup> = 0.78) with the causal variant rs16969968 that changes signal transduction of <em>CHRNA5</em> (<a href="#CIT0028" rid="CIT0028" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588871">28</a>,<a href="#CIT0029" rid="CIT0029" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588843">29</a>) The variant rs55781567 that was previously identified in unselected lung cancer cases (<a href="#CIT0011" rid="CIT0011" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588861">11</a>) as most strongly associated with lung cancer SNP was also highly significant in our study (MAF = 0.37; OR = 1.31, <em>P</em> = 1.69 × 10<sup>−11</sup>). This variant was previously shown to tag an insertion/deletion polymorphism in the promoter of <em>CHRNA5</em> that affects its expression (<a href="#CIT0030" rid="CIT0030" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588869">30</a>,<a href="#CIT0031" rid="CIT0031" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588846">31</a>). The estimate of risk we report in familial lung cancer is similar to risk estimates for SNPs in the chromosome 15q25.1 region reported for sporadic lung cancer (<a href="#CIT0005" rid="CIT0005" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588873">5</a>,<a href="#CIT0007" rid="CIT0007" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588844">7</a>).</p><p id="__p29">Two of the most significant associations were with familial lung squamous cell carcinoma. The first SNP, rs71603396 on chromosome 4p15.31, is in the region of the <em>LCORL</em> gene. The association between familial lung cancer risk and this SNP in the <em>LCORL</em> gene was also supported by the eQTL analysis. <em>LCORL</em> regulates transcription from RNA polymerase II and mutations in <em>LCORL</em> have frequently been noted in all histologies of lung cancer. In TCGA data, deletions of <em>LCORL</em> were observed in 0.5% of squamous lung cancers (<a href="#CIT0032" rid="CIT0032" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588876">32</a>) and <em>LCORL</em> is often deleted in pancreatic cancers (5.5% of cases) (<a href="#CIT0033" rid="CIT0033" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588852">33</a>). The second significant SNP associated with familial squamous cell lung cancer, rs12686364, is located in an intergenic region on chromosome 9p21.3, upstream of <em>IFNA8</em> and downstream of <em>CDKN2A.</em> eQTL analysis again provided support for this association. Published results show that <em>IFNA8</em> was deleted in 30% of glioblastomas (<a href="#CIT0034" rid="CIT0034" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588862">34</a>), 24.8% of pancreatic cancers (<a href="#CIT0033" rid="CIT0033" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588878">33</a>), and deleted or mutated in 12.9% of squamous lung cancers (<a href="#CIT0032" rid="CIT0032" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588848">32</a>). Deletions of <em>IFNA8</em> are associated with improved survival in glioma patients (<a href="#CIT0035" rid="CIT0035" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588860">35</a>,<a href="#CIT0036" rid="CIT0036" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588866">36</a>).</p><p id="__p30">Our results were also compared to the recently published variants associated with risk of sporadic lung cancer from a very large study (<a href="/pmc/articles/PMC6148967/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>) (<a href="#CIT0011" rid="CIT0011" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_724588855">11</a>). There is some heterogeneity in findings for familial lung cancer compared with those previously reported for the largest study to date of unselected lung cancer cases. In particular, variants in <em>TP63</em> have been reproducibly associated with lung adenocarcinoma risk, but showed no association in this study. The rare variant rs17879961 of <em>CHEK2</em> in sporadic lung cancer is highly protective from lung cancer development but in these familial cancers, the association analysis yielded strongly positive association (odds ratio of 2.5). Given the small sample size, this association was not significant and larger studies are needed to evaluate this finding further. While this is the largest study to date of a GWAS for familial lung cancer, the sample size is still quite limited, which may largely explain why many known associations for sporadic lung cancer were not associated with significant findings in this study. Another limitation in this study is the difference in family history between the Genetic Epidemiology of Lung Cancer Consortium, which recruited families with two or more first or second degree relatives and the restriction to families with one or more first degree relatives for the other studies. However, in conducting this study we sought to identify reliable findings that met genome wide significance and to be able to evaluate the findings in multiple independent studies.</p><p id="__p31" class="p p-last">In summary, a novel variant rs71603396 on chromosome 4p15.31 and an imputed rare variant rs12686364 on 9p21.3 were identified at a genome-wide level of significance for familial squamous cell carcinomas. Also, the associations of <em>CHRNA3</em> and <em>CHRNA5</em> on chromosome 15q25.1 were confirmed in this familial lung cancer data. The identification of high risk variants in individuals selected through a positive family history provides insights into the etiology of lung cancer and may improve our ability to target selected individuals for lung cancer screening.</p></div><div id="s13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s13title">Supplementary material</h2><p id="__p32" class="p p-first-last">Supplementary materials can be found at <em>Carcinogenesis</em> online.</p></div><div id="s14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s14title">Funding</h2><p id="__p33" class="p p-first-last">
National Institute of Health (U01CA76293, U19CA148127, P30CA22453, HHSN26820100007C); The Intramural Research Program of the National Human Genome Research Institute; National Institute of Health (U19CA148127, CA148127S1) (to the Transdisciplinary Research for Cancer in Lung (TRICL)); Cancer Care Ontario Research Chair of Population Studies (to RH); Canadian Cancer Society Research Institute (no. 020214) (to the SLRI GWAS); Ontario Institute for Cancer Research (to RH); National Institute of Health (R01CA060691, R01CA87895) (to Detroit GELCC study). National Institute of Health -National Cancer Institute (CA77118, CA80127, CA84353) (to Mayo Clinic Lung Study); National Institute of Health (P30ES006096) (to Cincinnati study); Genotyping services for the GELCC study were provided by the Center for Inherited Disease Research (CIDR). CIDR is fully funded through a federal contract from the National Institutes of Health to The Johns Hopkins University, contract numbers HHSN268200782096C and HHSN268201100011.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="sup1"><h4>Supplementary_Table_6</h4><div class="sup-box half_rhythm" id="idm139670479259600"><a href="/pmc/articles/PMC6148967/bin/bgy080_suppl_supplementary_table_6.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(206K, xlsx)</sup></div></div><div class="sec suppmat" id="sup2"><h4>Supplementary_Table_7</h4><div class="sup-box half_rhythm" id="idm139670479256784"><a href="/pmc/articles/PMC6148967/bin/bgy080_suppl_supplementary_table_7.xlsx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(21K, xlsx)</sup></div></div><div class="sec suppmat" id="sup3"><h4>Supplementary_Materials</h4><div class="sup-box half_rhythm" id="idm139670479253968"><a href="/pmc/articles/PMC6148967/bin/bgy080_suppl_supplementary_materials.docx" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(1.6M, docx)</sup></div></div></div><div id="idm139670479252368" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139670479252368title">Acknowledgements</h2><div class="sec"><p id="__p37">The authors would like to thank the staff at the Respiratory Health Network Tissue Bank of the FRQS for their valuable assistance with the lung eQTL dataset at Laval University.</p><p id="__p38">
<em>Conflict of Interest Statement:</em> None declared.</p></div></div><div id="idm139670479250176" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139670479250176title">Abbreviations</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td id="term1">eQTL</td><td>expression quantitative trait loci</td></tr><tr><td id="term2">EAGLE</td><td>Environment And Genetics in Lung cancer Etiology</td></tr><tr><td id="term3">GELCC</td><td>The Genetic Epidemiology of Lung Cancer Consortium</td></tr><tr><td id="term4">GWAS</td><td>Genome-wide association studies</td></tr><tr><td id="term5">ILCCO</td><td>International Lung Cancer Consortium</td></tr><tr><td id="term6">IARC</td><td>International Agency for Research on Cancer</td></tr><tr><td id="term7">MDACC</td><td>MD Anderson Cancer Center</td></tr><tr><td id="term8">PLCO</td><td>Prostate, Lung, Colon, and Ovary screening studies</td></tr><tr><td id="term9">SNP</td><td>single nucleotide polymorphism</td></tr><tr><td id="term10">TRICL</td><td>Transdisciplinary Research in Cancer of the Lung</td></tr></tbody></table></div></div></div><div id="idm139670479236944" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139670479236944title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="CIT0001">1. <span class="mixed-citation">
Sellers T.A., et al. (1990)<span class="ref-title">Evidence for mendelian inheritance in the pathogenesis of lung cancer</span>. <span class="ref-journal">J. Natl. Cancer Inst</span>., <span class="ref-vol">82</span>, 1272–1279. [<a href="/pubmed/2374177" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Natl.+Cancer+Inst&amp;title=Evidence+for+mendelian+inheritance+in+the+pathogenesis+of+lung+cancer&amp;author=T.A.+Sellers&amp;volume=82&amp;publication_year=1990&amp;pages=1272-1279&amp;pmid=2374177&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0002">2. <span class="mixed-citation">
Schwartz A.G., et al. (1996)<span class="ref-title">Familial risk of lung cancer among nonsmokers and their relatives</span>. <span class="ref-journal">Am. J. Epidemiol</span>., <span class="ref-vol">144</span>, 554–562. [<a href="/pubmed/8797515" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol&amp;title=Familial+risk+of+lung+cancer+among+nonsmokers+and+their+relatives&amp;author=A.G.+Schwartz&amp;volume=144&amp;publication_year=1996&amp;pages=554-562&amp;pmid=8797515&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0003">3. <span class="mixed-citation">
Sellers T.A., et al. (1994)<span class="ref-title">Segregation analysis of smoking-associated malignancies: evidence for Mendelian inheritance</span>. <span class="ref-journal">Am. J. Med. Genet</span>., <span class="ref-vol">52</span>, 308–314. [<a href="/pubmed/7810562" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet&amp;title=Segregation+analysis+of+smoking-associated+malignancies:+evidence+for+Mendelian+inheritance&amp;author=T.A.+Sellers&amp;volume=52&amp;publication_year=1994&amp;pages=308-314&amp;pmid=7810562&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0004">4. <span class="mixed-citation">
Bailey-Wilson J.E., et al. (2004)<span class="ref-title">A major lung cancer susceptibility locus maps to chromosome 6q23-25</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">75</span>, 460–474. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182024/">PMC free article</a>]</span> [<a href="/pubmed/15272417" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+major+lung+cancer+susceptibility+locus+maps+to+chromosome+6q23-25&amp;author=J.E.+Bailey-Wilson&amp;volume=75&amp;publication_year=2004&amp;pages=460-474&amp;pmid=15272417&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0005">5. <span class="mixed-citation">
Amos C.I., et al. (2008)<span class="ref-title">Genome-wide association scan of tag SNPs identifies a susceptibility locus for lung cancer at 15q25.1</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">40</span>, 616–622. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2713680/">PMC free article</a>]</span> [<a href="/pubmed/18385676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+scan+of+tag+SNPs+identifies+a+susceptibility+locus+for+lung+cancer+at+15q25.1&amp;author=C.I.+Amos&amp;volume=40&amp;publication_year=2008&amp;pages=616-622&amp;pmid=18385676&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0006">6. <span class="mixed-citation">
Wang Y., et al. (2014)<span class="ref-title">Rare variants of large effect in BRCA2 and CHEK2 affect risk of lung cancer</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">46</span>, 736–741. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4074058/">PMC free article</a>]</span> [<a href="/pubmed/24880342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Rare+variants+of+large+effect+in+BRCA2+and+CHEK2+affect+risk+of+lung+cancer&amp;author=Y.+Wang&amp;volume=46&amp;publication_year=2014&amp;pages=736-741&amp;pmid=24880342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0007">7. <span class="mixed-citation">
Hung R.J., et al. (2008)<span class="ref-title">A susceptibility locus for lung cancer maps to nicotinic acetylcholine receptor subunit genes on 15q25</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">452</span>, 633–637. [<a href="/pubmed/18385738" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+susceptibility+locus+for+lung+cancer+maps+to+nicotinic+acetylcholine+receptor+subunit+genes+on+15q25&amp;author=R.J.+Hung&amp;volume=452&amp;publication_year=2008&amp;pages=633-637&amp;pmid=18385738&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0008">8. <span class="mixed-citation">
Landi M.T., et al. (2009)<span class="ref-title">A genome-wide association study of lung cancer identifies a region of chromosome 5p15 associated with risk for adenocarcinoma</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">85</span>, 679–691. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2775843/">PMC free article</a>]</span> [<a href="/pubmed/19836008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+genome-wide+association+study+of+lung+cancer+identifies+a+region+of+chromosome+5p15+associated+with+risk+for+adenocarcinoma&amp;author=M.T.+Landi&amp;volume=85&amp;publication_year=2009&amp;pages=679-691&amp;pmid=19836008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0009">9. <span class="mixed-citation">
Shi J., et al. (2012)<span class="ref-title">Inherited variation at chromosome 12p13.33, including RAD52, influences the risk of squamous cell lung carcinoma</span>. <span class="ref-journal">Cancer Discov</span>., <span class="ref-vol">2</span>, 131–139. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3354721/">PMC free article</a>]</span> [<a href="/pubmed/22585858" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Discov&amp;title=Inherited+variation+at+chromosome+12p13.33,+including+RAD52,+influences+the+risk+of+squamous+cell+lung+carcinoma&amp;author=J.+Shi&amp;volume=2&amp;publication_year=2012&amp;pages=131-139&amp;pmid=22585858&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0010">10. <span class="mixed-citation">
Amos C.I., et al. (2010)<span class="ref-title">A susceptibility locus on chromosome 6q greatly increases lung cancer risk among light and never smokers</span>. <span class="ref-journal">Cancer Res</span>., <span class="ref-vol">70</span>, 2359–2367. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2855643/">PMC free article</a>]</span> [<a href="/pubmed/20215501" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=A+susceptibility+locus+on+chromosome+6q+greatly+increases+lung+cancer+risk+among+light+and+never+smokers&amp;author=C.I.+Amos&amp;volume=70&amp;publication_year=2010&amp;pages=2359-2367&amp;pmid=20215501&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0011">11. <span class="mixed-citation">
McKay J.D., et al. ; SpiroMeta Consortium  (2017)<span class="ref-title">Large-scale association analysis identifies new lung cancer susceptibility loci and heterogeneity in genetic susceptibility across histological subtypes</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">49</span>, 1126–1132. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5510465/">PMC free article</a>]</span> [<a href="/pubmed/28604730" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Large-scale+association+analysis+identifies+new+lung+cancer+susceptibility+loci+and+heterogeneity+in+genetic+susceptibility+across+histological+subtypes&amp;author=J.D.+McKay&amp;volume=49&amp;publication_year=2017&amp;pages=1126-1132&amp;pmid=28604730&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0012">12. <span class="mixed-citation">
Musolf A.M., et al. (2017)<span class="ref-title">Familial lung cancer: a brief history from the earliest work to the most recent studies</span>. <span class="ref-journal">Genes (Basel)</span>, <span class="ref-vol">8</span>, E36. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5295030/">PMC free article</a>]</span> [<a href="/pubmed/28106732" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+(Basel)&amp;title=Familial+lung+cancer:+a+brief+history+from+the+earliest+work+to+the+most+recent+studies&amp;author=A.M.+Musolf&amp;volume=8&amp;publication_year=2017&amp;pages=E36&amp;pmid=28106732&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0013">13. <span class="mixed-citation">
Musolf A.M., et al. (2016)<span class="ref-title">Parametric linkage analysis identifies five novel genome-wide significant loci for familial lung cancer</span>. <span class="ref-journal">Hum. Hered</span>., <span class="ref-vol">82</span>, 64–74. [<a href="/pubmed/28817824" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Hered&amp;title=Parametric+linkage+analysis+identifies+five+novel+genome-wide+significant+loci+for+familial+lung+cancer&amp;author=A.M.+Musolf&amp;volume=82&amp;publication_year=2016&amp;pages=64-74&amp;pmid=28817824&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0014">14. <span class="mixed-citation">
Thorgeirsson T.E., et al. (2008)<span class="ref-title">A variant associated with nicotine dependence, lung cancer and peripheral arterial disease</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">452</span>, 638–642. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4539558/">PMC free article</a>]</span> [<a href="/pubmed/18385739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+variant+associated+with+nicotine+dependence,+lung+cancer+and+peripheral+arterial+disease&amp;author=T.E.+Thorgeirsson&amp;volume=452&amp;publication_year=2008&amp;pages=638-642&amp;pmid=18385739&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0015">15. <span class="mixed-citation">
McKay J.D., et al. ; EPIC Study  (2008)<span class="ref-title">Lung cancer susceptibility locus at 5p15.33</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">40</span>, 1404–1406. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2748187/">PMC free article</a>]</span> [<a href="/pubmed/18978790" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Lung+cancer+susceptibility+locus+at+5p15.33&amp;author=J.D.+McKay&amp;volume=40&amp;publication_year=2008&amp;pages=1404-1406&amp;pmid=18978790&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0016">16. <span class="mixed-citation">
Timofeeva M.N., et al. ; Transdisciplinary Research in Cancer of the Lung (TRICL) Research Team  (2012)<span class="ref-title">Influence of common genetic variation on lung cancer risk: meta-analysis of 14 900 cases and 29 485 controls</span>. <span class="ref-journal">Hum. Mol. Genet</span>., <span class="ref-vol">21</span>, 4980–4995. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3607485/">PMC free article</a>]</span> [<a href="/pubmed/22899653" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Influence+of+common+genetic+variation+on+lung+cancer+risk:+meta-analysis+of+14+900+cases+and+29+485+controls&amp;author=M.N.+Timofeeva&amp;volume=21&amp;publication_year=2012&amp;pages=4980-4995&amp;pmid=22899653&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0017">17. <span class="mixed-citation">
Peng G., et al. (2010)<span class="ref-title">Gene and pathway-based second-wave analysis of genome-wide association studies</span>. <span class="ref-journal">Eur. J. Hum. Genet</span>., <span class="ref-vol">18</span>, 111–117. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2987176/">PMC free article</a>]</span> [<a href="/pubmed/19584899" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet&amp;title=Gene+and+pathway-based+second-wave+analysis+of+genome-wide+association+studies&amp;author=G.+Peng&amp;volume=18&amp;publication_year=2010&amp;pages=111-117&amp;pmid=19584899&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0018">18. <span class="mixed-citation">
Howie B., et al. (2012)<span class="ref-title">Fast and accurate genotype imputation in genome-wide association studies through pre-phasing</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">44</span>, 955–959. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B.+Howie&amp;volume=44&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0019">19. <span class="mixed-citation">
Yang J., et al. (2011)<span class="ref-title">GCTA: a tool for genome-wide complex trait analysis</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">88</span>, 76–82. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J.+Yang&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0020">20. <span class="mixed-citation">
Yang J., et al. ; Genetic Investigation of ANthropometric Traits (GIANT) Consortium; DIAbetes Genetics Replication And Meta-analysis (DIAGRAM) Consortium  (2012)<span class="ref-title">Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits</span>. <span class="ref-journal">Nat. Genet</span>., <span class="ref-vol">44</span>, 369–75, S1. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3593158/">PMC free article</a>]</span> [<a href="/pubmed/22426310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Conditional+and+joint+multiple-SNP+analysis+of+GWAS+summary+statistics+identifies+additional+variants+influencing+complex+traits&amp;author=J.+Yang&amp;volume=44&amp;publication_year=2012&amp;pages=369-75,+S1&amp;pmid=22426310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0021">21. <span class="mixed-citation">
Hao K., et al. (2012)<span class="ref-title">Lung eQTLs to help reveal the molecular underpinnings of asthma</span>. <span class="ref-journal">PLoS Genet</span>., <span class="ref-vol">8</span>, e1003029. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3510026/">PMC free article</a>]</span> [<a href="/pubmed/23209423" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Lung+eQTLs+to+help+reveal+the+molecular+underpinnings+of+asthma&amp;author=K.+Hao&amp;volume=8&amp;publication_year=2012&amp;pages=e1003029&amp;pmid=23209423&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0022">22. <span class="mixed-citation">
Nguyen J.D., et al. (2014)<span class="ref-title">Susceptibility loci for lung cancer are associated with mRNA levels of nearby genes in the lung</span>. <span class="ref-journal">Carcinogenesis</span>, <span class="ref-vol">35</span>, 2653–2659. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4247514/">PMC free article</a>]</span> [<a href="/pubmed/25187487" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Susceptibility+loci+for+lung+cancer+are+associated+with+mRNA+levels+of+nearby+genes+in+the+lung&amp;author=J.D.+Nguyen&amp;volume=35&amp;publication_year=2014&amp;pages=2653-2659&amp;pmid=25187487&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0023">23. <span class="mixed-citation">
Johnson A.D., et al. (2008)<span class="ref-title">SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap</span>. <span class="ref-journal">Bioinformatics</span>, <span class="ref-vol">24</span>, 2938–2939. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2720775/">PMC free article</a>]</span> [<a href="/pubmed/18974171" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SNAP:+a+web-based+tool+for+identification+and+annotation+of+proxy+SNPs+using+HapMap&amp;author=A.D.+Johnson&amp;volume=24&amp;publication_year=2008&amp;pages=2938-2939&amp;pmid=18974171&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0024">24. <span class="mixed-citation">
Xiong D., et al. (2015)<span class="ref-title">A recurrent mutation in PARK2 is associated with familial lung cancer</span>. <span class="ref-journal">Am. J. Hum. Genet</span>., <span class="ref-vol">96</span>, 301–308. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4320264/">PMC free article</a>]</span> [<a href="/pubmed/25640678" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+recurrent+mutation+in+PARK2+is+associated+with+familial+lung+cancer&amp;author=D.+Xiong&amp;volume=96&amp;publication_year=2015&amp;pages=301-308&amp;pmid=25640678&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0025">25. <span class="mixed-citation">
Hwang S.J., et al. (2003)<span class="ref-title">Lung cancer risk in germline p53 mutation carriers: association between an inherited cancer predisposition, cigarette smoking, and cancer risk</span>. <span class="ref-journal">Hum. Genet</span>., <span class="ref-vol">113</span>, 238–243. [<a href="/pubmed/12802680" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet&amp;title=Lung+cancer+risk+in+germline+p53+mutation+carriers:+association+between+an+inherited+cancer+predisposition,+cigarette+smoking,+and+cancer+risk&amp;author=S.J.+Hwang&amp;volume=113&amp;publication_year=2003&amp;pages=238-243&amp;pmid=12802680&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0026">26. <span class="mixed-citation">
Bossé Y., et al. (2018)<span class="ref-title">A decade of GWAS results in lung cancer</span>. <span class="ref-journal">Cancer Epidemiol. Biomarkers Prev</span>., <span class="ref-vol">27</span>, 363–379. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6464125/">PMC free article</a>]</span> [<a href="/pubmed/28615365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev&amp;title=A+decade+of+GWAS+results+in+lung+cancer&amp;author=Y.+Bossé&amp;volume=27&amp;publication_year=2018&amp;pages=363-379&amp;pmid=28615365&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0027">27. <span class="mixed-citation">
Peng B., et al. (2010)<span class="ref-title">Power analysis for case-control association studies of samples with known family histories</span>. <span class="ref-journal">Hum. Genet</span>., <span class="ref-vol">127</span>, 699–704. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3914772/">PMC free article</a>]</span> [<a href="/pubmed/20383776" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet&amp;title=Power+analysis+for+case-control+association+studies+of+samples+with+known+family+histories&amp;author=B.+Peng&amp;volume=127&amp;publication_year=2010&amp;pages=699-704&amp;pmid=20383776&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0028">28. <span class="mixed-citation">
Doyle G.A., et al. (2011)<span class="ref-title"><em>In vitro</em> and ex vivo analysis of CHRNA3 and CHRNA5 haplotype expression</span>. <span class="ref-journal">PLoS One</span>, <span class="ref-vol">6</span>, e23373. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3155531/">PMC free article</a>]</span> [<a href="/pubmed/21858091" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=In+vitro+and+ex+vivo+analysis+of+CHRNA3+and+CHRNA5+haplotype+expression&amp;author=G.A.+Doyle&amp;volume=6&amp;publication_year=2011&amp;pages=e23373&amp;pmid=21858091&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0029">29. <span class="mixed-citation">
Bierut L.J., et al. (2008)<span class="ref-title">Variants in nicotinic receptors and risk for nicotine dependence</span>. <span class="ref-journal">Am. J. Psychiatry</span>, <span class="ref-vol">165</span>, 1163–1171. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2574742/">PMC free article</a>]</span> [<a href="/pubmed/18519524" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Psychiatry&amp;title=Variants+in+nicotinic+receptors+and+risk+for+nicotine+dependence&amp;author=L.J.+Bierut&amp;volume=165&amp;publication_year=2008&amp;pages=1163-1171&amp;pmid=18519524&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0030">30. <span class="mixed-citation">
Falvella F.S., et al. (2013)<span class="ref-title">Multiple isoforms and differential allelic expression of CHRNA5 in lung tissue and lung adenocarcinoma</span>. <span class="ref-journal">Carcinogenesis</span>, <span class="ref-vol">34</span>, 1281–1285. [<a href="/pubmed/23430818" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Multiple+isoforms+and+differential+allelic+expression+of+CHRNA5+in+lung+tissue+and+lung+adenocarcinoma&amp;author=F.S.+Falvella&amp;volume=34&amp;publication_year=2013&amp;pages=1281-1285&amp;pmid=23430818&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0031">31. <span class="mixed-citation">
Chen D., et al. (2011)<span class="ref-title">A sex-specific association between a 15q25 variant and upper aerodigestive tract cancers</span>. <span class="ref-journal">Cancer Epidemiol. Biomarkers Prev</span>., <span class="ref-vol">20</span>, 658–664. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3070066/">PMC free article</a>]</span> [<a href="/pubmed/21335511" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev&amp;title=A+sex-specific+association+between+a+15q25+variant+and+upper+aerodigestive+tract+cancers&amp;author=D.+Chen&amp;volume=20&amp;publication_year=2011&amp;pages=658-664&amp;pmid=21335511&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0032">32. <span class="mixed-citation">
Cancer Genome Atlas Network(2015)<span class="ref-title">Comprehensive genomic characterization of head and neck squamous cell carcinomas</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">517</span>, 576–582. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4311405/">PMC free article</a>]</span> [<a href="/pubmed/25631445" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Comprehensive+genomic+characterization+of+head+and+neck+squamous+cell+carcinomas&amp;volume=517&amp;publication_year=2015&amp;pages=576-582&amp;pmid=25631445&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0033">33. <span class="mixed-citation">
Witkiewicz A.K., et al. (2015)<span class="ref-title">Whole-exome sequencing of pancreatic cancer defines genetic diversity and therapeutic targets</span>. <span class="ref-journal">Nat. Commun</span>., <span class="ref-vol">6</span>, 6744. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4403382/">PMC free article</a>]</span> [<a href="/pubmed/25855536" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun&amp;title=Whole-exome+sequencing+of+pancreatic+cancer+defines+genetic+diversity+and+therapeutic+targets&amp;author=A.K.+Witkiewicz&amp;volume=6&amp;publication_year=2015&amp;pages=6744&amp;pmid=25855536&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0034">34. <span class="mixed-citation">
Brennan C.W., et al. ; TCGA Research Network  (2013)<span class="ref-title">The somatic genomic landscape of glioblastoma</span>. <span class="ref-journal">Cell</span>, <span class="ref-vol">155</span>, 462–477. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3910500/">PMC free article</a>]</span> [<a href="/pubmed/24120142" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The+somatic+genomic+landscape+of+glioblastoma&amp;author=C.W.+Brennan&amp;volume=155&amp;publication_year=2013&amp;pages=462-477&amp;pmid=24120142&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0035">35. <span class="mixed-citation">
Kohanbash G., et al. (2012)<span class="ref-title">Differential activity of interferon-α8 promoter is regulated by Oct-1 and a SNP that dictates prognosis of glioma</span>. <span class="ref-journal">Oncoimmunology</span>, <span class="ref-vol">1</span>, 487–492. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3382910/">PMC free article</a>]</span> [<a href="/pubmed/22754767" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncoimmunology&amp;title=Differential+activity+of+interferon-α8+promoter+is+regulated+by+Oct-1+and+a+SNP+that+dictates+prognosis+of+glioma&amp;author=G.+Kohanbash&amp;volume=1&amp;publication_year=2012&amp;pages=487-492&amp;pmid=22754767&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0036">36. <span class="mixed-citation">
Okada H., et al. (2013)<span class="ref-title">Integration of epidemiology, immunobiology, and translational research for brain tumors</span>. <span class="ref-journal">Ann. N. Y. Acad. Sci</span>., <span class="ref-vol">1284</span>, 17–23. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3648859/">PMC free article</a>]</span> [<a href="/pubmed/23651189" target="pmc_ext" ref="reftype=pubmed&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+N.+Y.+Acad.+Sci&amp;title=Integration+of+epidemiology,+immunobiology,+and+translational+research+for+brain+tumors&amp;author=H.+Okada&amp;volume=1284&amp;publication_year=2013&amp;pages=17-23&amp;pmid=23651189&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=6148967&amp;issue-id=321087&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139670518010320title">Abstract</a></li><li><a href="#s1title">Introduction</a></li><li><a href="#s2title">Materials and methods</a></li><li><a href="#s11title">Results</a></li><li><a href="#s12title">Discussion</a></li><li><a href="#s13title">Supplementary material</a></li><li><a href="#s14title">Funding</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm139670479252368title">Acknowledgements</a></li><li><a href="#idm139670479250176title">Abbreviations</a></li><li><a href="#idm139670479236944title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Carcinogenesis</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC6148967/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC6148967/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC6148967/pdf/bgy080.pdf">PDF (126K)</a></li> | <li><a href="#" data-citationid="PMC6148967" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6148967%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6148967%2F&amp;text=Genome-wide%20association%20study%20of%20familial%20lung%20cancer" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6148967%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="29924316" data-aiid="6148967" data-aid="6148967" data-iid="321087" data-domainid="724" data-domain="carcin" data-accid="PMC6148967" data-md5="04f3f658a9f7ac52c140d748040fccf5"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T17:15:00-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C41D5DD5AC8C100000000082104BC&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6148967%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=carcin&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC6148967/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>